<html xml:lang="en" xmlns="http://www.w3.org/1999/xhtml" lang="en"><head xmlns="">
































	
		
						
																																																																																												
																					
																					
 												 		
					




<!-- super head top --><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
                





































        
               
      








        
                
 	



 










	
				

        
        
  

 


 

 
 
 


 
 





<title>Proteomics: the first decade and beyond - Nature Genetics</title><meta content="noarchive" name="robots"><link href="http://www.nature.com.www.lib.ncsu.edu:2048/favicon.ico" rel="shortcut icon"><link type="text/css" href="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/style_older.css" rel="stylesheet"><style media="all" type="text/css">@import "/ng/style.css";</style>








<script type="text/javascript"><!--//--><![CDATA[//><!--
	var include = '';

	var wp = '';
	if (wp != '') include += 'wp='+wp;
	include += ';';

	var rf = '';
	if (rf != '') include += "rf="+rf;
	include += ';';

	var sp = '';
	if (sp != '') include += "sp="+sp;
	include += ";";

	var ps = '';
	if (ps != '') include += "ps="+ps;
	include += ";";

	var adinf = "lg=no;";

	if(include != ";;;;")
	{
		adinf = include;
	}

	var time = new Date();
	ordval= (time.getTime());

		ord=Math.random()*10000000000000000;
//--><!]]></script>

<script type="text/javascript" language="javascript">// provides cross-browser functionality
//======================================
function getObject(name) {
  if(document.getElementById) return document.getElementById(name);
  else if(document.all) return document.all[name];
  else return null;
}

</script><script src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/mailafriend.js" language="javascript" type="text/javascript"></script><!-- super head bottom -->
	<!-- resource metadata -->
				
				 																							
			
				
											
			<link title="schema(PRISM)" rel="schema.prism" href="http://prismstandard.org/namespaces/1.2/basic/">
																																																												<meta name="prism.copyright" content="© 2003 Nature Publishing Group">
					<meta name="prism.rightsAgent" content="permissions@nature.com">
					<meta name="prism.publicationDate" content="2003-03-01">
					<meta name="prism.volume" content="33">
					<meta name="prism.number" content="">
					<meta name="prism.startingPage" content="311">
					<meta name="prism.endingPage" content="323">
					<meta name="prism.publicationName" content="Nature Genetics">
					<meta name="prism.issn" content="ERROR! NO ISSN">
					<meta name="prism.eIssn" content="ERROR! NO EISSN">
				
			<link title="schema(DC)" rel="schema.dc" href="http://purl.org/dc/elements/1.1/">
																																																							<meta name="dc.publisher" content="Nature Publishing Group">
					<meta name="dc.language" content="en">
					<meta name="dc.rights" content="© 2003 Nature Publishing Group">
					<meta name="dc.title" content="Proteomics: the first decade and beyond">
					<meta name="dc.creator" content="Scott D. Patterson">
					<meta name="dc.creator" content="Ruedi H. Aebersold">
					<meta name="dc.identifier" content="doi:10.1038/ng1106">
					<meta name="dc.date" content="2003-03-01">
					<meta name="dc.source" content="Nature Genetics, Published online: 01 March 2003; | doi:10.1038/ng1106">
				
																																												<meta name="citation_publisher" content="Nature Publishing Group">
					<meta name="citation_authors" content="Scott D. Patterson, Ruedi H. Aebersold">
					<meta name="citation_title" content="Proteomics: the first decade and beyond">
					<meta name="citation_date" content="2003-03-01">
					<meta name="citation_firstpage" content="311">
					<meta name="citation_doi" content="doi:10.1038/ng1106">
					<meta name="citation_journal_title" content="Nature Genetics">
			
																			<meta name="Access" content="Yes">
					<meta name="WT.cg_n" content="Nature Genetics">
			</head><div FirebugVersion="1.3.3" style="display: none;" id="_firebugConsole"></div><body topmargin="0" leftmargin="0" bgcolor="#ffffff" marginheight="0" marginwidth="0" text="#000000">
<!-- super body top --><a name="top">


																		

<!-- header -->
</a><div id="head-er">

<div class="header-nonad-wrapper" id="top">
<a name="top">				</a><div class="accessibility-login-wrap">
<a name="top">	</a><div class="accessibility-login">
<a name="top">				<!--<a href="/" class="rollover"><img src="/common/includes/header/images/nature.com.gif" alt="Nature.com homepage" class="nature-logo" /></a>-->
		</a><a href="http://www.nature.com.www.lib.ncsu.edu:2048/" class="nature-link">nature.com<span class="hidden"> homepage</span></a>
									

			<div class="logon">
		<p><a href="http://www.nature.com.www.lib.ncsu.edu:2048/foxtrot/svc/login" class="login" title="login to Nature.com">Login<!-- following space necessary for Mac IE --> </a><span class="onlyformacie">&nbsp;</span></p>
	</div>
						<div class="cleardiv"><!--  --></div>
	</div>
</div>

<div class="cleardiv"><!--  --></div>
<hr>
		
			<div class="image-user-services-wrap">
	<div class="image-user-services">
					<a href="http://www.nature.com.www.lib.ncsu.edu:2048/ng"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/ng_logo_old.gif" class="journal-header" alt="Nature Genetics homepage"></a>		
								<div class="user-services div1">
								<!-- whitespace removed to deal with IE 5.0 bug --><ul class="list1"><li class="li1"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/myaccount/">My account</a></li><li class="li2"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/nams/svc/myaccount/save/ealert?list_id=4" title="sign up to receive Nature Genetics email alerts">E-alert sign up</a></li></ul>
							</div>
						<div class="user-services div2">
								<!-- whitespace removed to deal with IE 5.0 bug --><ul class="list2"><li class="li1"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/register/" title="Register with Nature.com">Register</a></li><li class="li2"><a href="https://secure-nature-com.www.lib.ncsu.edu:2433/subscribe/ng" title="Subscribe to Nature Genetics">Subscribe</a></li></ul>
							</div>
						<div class="cleardiv"><!--  --></div>
			</div>
	<div class="cleardiv"><!--  --></div>
</div>

<div class="cleardiv"><!--  --></div>
<hr>		
			<div class="publications-search-wrap clearfix">
	<div class="publications-search">
				<ul class="publications">
			<li class="first"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/siteindex/">Publications&nbsp;A-Z&nbsp;index</a></li><li><a href="http://www.nature.com.www.lib.ncsu.edu:2048/browse/">Browse&nbsp;by&nbsp;subject</a></li>		</ul>

		<hr>
		
				<div class="search">
			<form class="search" action="/search/executeSearch?" method="get">
				<div>
					<label for="searchtext">Search</label>
																	<select name="sp-q-1">
														<option value="NG">This journal</option>
														<option value="">All of nature.com</option>
													</select>
																<input id="searchtext" name="sp-q" value="" type="text">
															<input name="sp-c" value="25" type="hidden">
															<input name="sp-m" value="0" type="hidden">
															<input name="sp-s" value="date_descending" type="hidden">
															<input name="include-collections" value="journals_nature,crawled_content" type="hidden">
															<input name="exclude-collections" value="journals_palgrave,lab_animal" type="hidden">
																				
																									<input name="sp-a" value="sp1001702d" type="hidden">
																											<input name="sp-sfvl-field" value="subject|ujournal" type="hidden">
																											<input name="sp-x-1" value="ujournal" type="hidden">
																											<input name="sp-p-1" value="phrase" type="hidden">
																											<input name="sp-p" value="all" type="hidden">
																							<input name="submit" class="button" value="go" title="search now" type="submit"> 						
									</div>
			</form>
		</div>
				<div class="cleardiv"><!--  --></div>
	</div>
</div>		
		</div>

	<!-- leaderboard ad -->
<div class="header-leaderboard-wrap">











		
<div class="header-leaderboard adunit-leaderboard">
<script type="text/javascript"><!--//--><![CDATA[//><!--
		document.write('<script type="text/javascript" src="http://ad.doubleclick.net/adj/genetics.nature.com/;abr=!webtv;type=rv;artid=ng1106;pos=top;dcopt=ist;sz=728x90;ptile=1;ord=' + ord + '?" ><\/script>');

	if ((!document.images && navigator.userAgent.indexOf('Mozilla/2.') >= 0) || (navigator.userAgent.indexOf("WebTV") >= 0)) {
		document.write('<a href="http://ad.doubleclick.net/jump/genetics.nature.com/;sz=728x90;ord=' + ord + '?">');
		document.write('<img src="http://ad.doubleclick.net/ad/genetics.nature.com/;sz=728x90;ord=' + ord + '?" width="728" height="90" border="0"></a>');
	}
//--><!]]></script><script type="text/javascript" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/a.js"></script><!-- Template Id = 1 Template Name = Banner Creative (Flash) -->
<!-- Copyright 2002 DoubleClick Inc., All rights reserved. --><script src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/flashwrite_1_2.js"></script>
<object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" id="FLASH_AD" height="90" width="728"><param name="movie" value="http://m1.2mdn.net/488537/FLASH_banner_ABI_728x90B_world.swf"><param name="flashvars" value="clickTag=http%3A%2F%2Fad.doubleclick.net%2Fclick%253Bh%3Dv8%2F3812%2F3%2F0%2F%252a%2Fk%253B208558981%253B0-0%253B0%253B2638843%253B3454-728%2F90%253B30478841%2F30496718%2F1%253B%253B%257Esscs%253D%253fhttp%3A%2F%2Fmarketing.appliedbiosystems.com%2Fmk%2Fget%2FOA_LANDING%3Futm_source%3Dnrg%26utm_medium%3Dbanner%26utm_campaign%3Dopenarray"><param name="quality" value="high"><param name="bgcolor" value="#"><param name="wmode" value="opaque"><param name="AllowScriptAccess" value="never"><embed src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/FLASH_banner_ABI_728x90B_world.swf" quality="high" wmode="opaque" swliveconnect="TRUE" bgcolor="#" type="application/x-shockwave-flash" allowscriptaccess="never" height="90" width="728"></object><noscript><A
TARGET="_blank"
HREF="http://ad.doubleclick.net/click%3Bh=v8/3812/3/0/%2a/k%3B208558981%3B0-0%3B0%3B2638843%3B3454-728/90%3B30478841/30496718/1%3B%3B%7Esscs%3D%3fhttp://marketing.appliedbiosystems.com/mk/get/OA_LANDING?utm_source=nrg&utm_medium=banner&utm_campaign=openarray"><IMG
SRC="http://m1.2mdn.net/488537/AB_728x90_noflashB.gif" alt=""
BORDER=0></A></noscript>
<noscript> <div><a
href="http://ad.doubleclick.net/jump/genetics.nature.com/;abr=!NN2;tile=1;type=rv;artid=ng1106;pos=top;dcopt=ist;sz=728x90;ptile=1;ord=123456789?"><img
src="http://ad.doubleclick.net/ad/genetics.nature.com/;abr=!NN2;tile=1;type=rv;artid=ng1106;pos=top;dcopt=ist;sz=728x90;ptile=1;ord=123456789?"
alt="Advertisement" style="border:0" /></a></div>
</noscript>

</div>


</div>
<!-- end leaderboard ad -->

</div>
<!-- end header -->

			<div id="constrain"><div class="constrain">
		<table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr xmlns=""><td align="center" height="22" valign="top"><table border="0" cellpadding="0" cellspacing="0" width="760"><tbody><tr><td bgcolor="#efefdb" height="22" width="145">&nbsp;</td><td class="breadcrumb" align="left" bgcolor="#efefdb" height="22" width="630"><a class="breadcrumblink" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/index.html">Journal home</a>&nbsp;&gt;
<a class="breadcrumblink" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/archive/index.html">Archive</a>&nbsp;&gt;
<a class="breadcrumblink" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/index.html">

 Table of Contents</a>&nbsp;&gt;&nbsp;<a class="breadcrumblink" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/index.html#rv">Review</a> &gt; Full text</td></tr></tbody></table></td></tr><tr><td align="center"><table border="0" cellpadding="0" cellspacing="0" width="760"><tbody><tr><td valign="top" width="130"><input xmlns="" value="archive" id="leftnav_type" type="hidden"><input xmlns="" value="rv" id="categ" type="hidden"><input xmlns="" value="" id="subcateg" type="hidden"><input xmlns="" value="" id="arch_categ" type="hidden">
<script language="JavaScript">
var bgColorOn = "#4D876E";
var bgColor2 = "#EAEAEA";
//var bgClassOn = "leftnavOn_bg";
var imageOn = "/includes/rj_globnavimages/nav_dots1.gif";
var imageOff = "/includes/rj_globnavimages/nav_dots2.gif";
var imageSpacer = "/includes/rj_globnavimages/spacer.gif";



var children = new Array();

function addSubLevel(parent, child){
	children[parent] = child;
}

function getObjectRef(name) {
  if(document.getElementById) return document.getElementById(name);
  else if(document.all) return document.all[name];
  else return null;
}

var thisPageType = '';

function setPageType(){
	var type = getObjectRef("leftnav_type").value;
	//alert(type);

	if (type == null){
		type = '';
	}


	thisPageType = type;
}

function getPageType(){
	return thisPageType;
}

setPageType();

function isCurrentType(thisType){
	if (thisType == getPageType()){
		return true;
	}else{
		var isChild = false;
		if (typeof children[thisType] != "undefined"){
			var child = children[thisType];
			for (var i=0; i<child.length; i++){
				if (getPageType() == child[i]){
					isChild = true;
				}
			}
		}
		return isChild;
	}
}

function getNewSubRow(name, linkhref, text){
	return '<tr><td width="5" style="background-color:'+bgColor2+'" id="'+name+'_1"><img src="'+imageSpacer+'" width="5" height="15" alt=""></td><td style="background-color:'+bgColor2+'" id="'+name+'_2" valign="top"><img src="'+imageOff+'" id="'+name+'_image" width="5" height="7" alt="" vspace="2"></td><td style="background-color:'+bgColor2+'" id="'+name+'_3"><img src="'+imageSpacer+'" width="5" height="15" alt=""></td><td style="background-color:'+bgColor2+'" id="'+name+'_4"><a href="'+linkhref+'" class="leftnav3" id="'+name+'_link">'+text+'</a></td><td width="5" style="background-color:'+bgColor2+'" id="'+name+'_5" valign="top"><img src="'+imageSpacer+'" width="5" height="15" alt=""></td></tr>';
}

function getSubRow(newRows){
	if (newRows != ''){
		return '<img src="'+imageSpacer+'" width="5" height="1" alt=""></td></tr><tr><td height="1" colspan="3"><table width="100%" cellpadding="0" cellspacing="0" border="0">'+newRows+'</table><tr><td colspan="3" height="1"><img src="'+imageSpacer+'" width="5" height="1" alt=""></td></tr>'
	} else {
		//return '<img src="'+imageSpacer+'" width="5" height="1" alt="">';
		return '';
	}
}

function hilite(text) {
// set colors

//alert(getObjectRef(text+"_1").style.backgroundColor);
	if (getObjectRef(text+"_link") != null){

		var num = 1;
		while (getObjectRef(text+"_"+num)){
			//getObjectRef(text+"_"+num).className  = bgClassOn;
			getObjectRef(text+"_"+num).style.backgroundColor = bgColorOn;
			num++;
		}

		getObjectRef(text+"_link").className = getObjectRef(text+"_link").className+"On";
	}

	if (getObjectRef(text+"_image") != null){
		//alert(getObjectRef(text+"_image").src);
		getObjectRef(text+"_image").src = imageOn;
	}

}

</script>
		<table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr><tr><td id="home_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="home_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/index.html" id="home_link">Journal home</a></td><td id="home_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="aop_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="aop_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/vaop/ncurrent/index.html" id="aop_link">Advance online publication</a></td><td id="aop_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><script language="JavaScript">

var thisType = 'aop';

addSubLevel(thisType, new Array("About_AOP"));

var newRows = '';
if (isCurrentType(thisType) || '' == 'true'){

	
	newRows += getNewSubRow('About_AOP', '/ng/aop/index.html', 'About AOP');
	

	if (isCurrentType(thisType) && thisType != getPageType()){
		hilite(thisType);
	}
	
	
	

}

document.write (getSubRow(newRows));
</script></td></tr><tr><td id="issue_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="issue_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v41/n4/index.html" id="issue_link">
Current issue
</a></td><td id="issue_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="archive_1" style="background-color: rgb(77, 135, 110);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="archive_2" style="background-color: rgb(77, 135, 110);"><a class="leftnavOn" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/archive/index.html" id="archive_link">Archive</a></td><td id="archive_3" style="background-color: rgb(77, 135, 110);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><script language="JavaScript">

var thisType = 'archive';

addSubLevel(thisType, new Array("supp_info"));

var newRows = '';
if (isCurrentType(thisType) || '' == 'true'){

	
	newRows += getNewSubRow('supp_info', '/ng/archive/suppinfo/index.html', 'Supplementary info');
	

	if (isCurrentType(thisType) && thisType != getPageType()){
		hilite(thisType);
	}
	
	
	

}

document.write (getSubRow(newRows));
</script><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="1" width="5"></td></tr><tr><td colspan="3" height="1"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td style="background-color: rgb(234, 234, 234);" id="supp_info_1" width="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="supp_info_2" valign="top"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/nav_dots2.gif" id="supp_info_image" alt="" height="7" vspace="2" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="supp_info_3"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="supp_info_4"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/archive/suppinfo/index.html" class="leftnav3" id="supp_info_link">Supplementary info</a></td><td style="background-color: rgb(234, 234, 234);" id="supp_info_5" valign="top" width="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="15" width="5"></td></tr></tbody></table></td></tr><tr><td colspan="3" height="1"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="1" width="5"></td></tr><tr><td id="Press_releases_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="Press_releases_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/press_release/index.html" id="Press_releases_link">Press releases</a></td><td id="Press_releases_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr></tbody></table>	
		<table border="0" cellpadding="0" cellspacing="0" width="130">
<tbody><tr>
<td colspan="3" height="1">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="1" width="5"></td></tr>
<tr>
<td colspan="3" height="12">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="12" width="5"></td></tr>
<tr>
<td colspan="3" height="1"></td></tr>
<tr>
<td id="Blog_1" style="background-color: rgb(239, 239, 219);" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td>
<td id="Blog_2" style="background-color: rgb(239, 239, 219);">
<a class="leftnav2" href="http://blogs.nature.com.www.lib.ncsu.edu:2048/ng/freeassociation/" id="Blog_link">Free Association (blog)</a></td>
<td id="Blog_3" style="background-color: rgb(239, 239, 219);" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr>
<tr>
<td colspan="3" height="1"></td></tr>
<tr>
<td id="Supplements_1" style="background-color: rgb(239, 239, 219);" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td>
<td id="Supplements_2" style="background-color: rgb(239, 239, 219);">
<a class="leftnav2" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/focsup_index.html" id="Supplements_link">Supplements</a></td>
<td id="Supplements_3" style="background-color: rgb(239, 239, 219);" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr>
<tr>
<td colspan="3" height="1"></td></tr>
<tr>
<td id="Web_Focuses_1" style="background-color: rgb(239, 239, 219);" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td>
<td id="Web_Focuses_2" style="background-color: rgb(239, 239, 219);">
<a class="leftnav2" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/focsup_index.html" id="Web_Focuses_link">Focuses</a></td>
<td id="Web_Focuses_3" style="background-color: rgb(239, 239, 219);" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr>
<tr>
<td colspan="3" height="1"></td></tr></tbody></table>	
		<table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td colspan="3" height="12"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="12" width="5"></td></tr><tr><td id="For_authors_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="For_authors_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/authors/index.html" id="For_authors_link">Guide to authors</a></td><td id="For_authors_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><script language="JavaScript">

var thisType = 'For_authors';

addSubLevel(thisType, new Array("How_to_submit","Content_types","Editorial_process"));

var newRows = '';
if (isCurrentType(thisType) || '' == 'true'){

	
	newRows += getNewSubRow('Editorial_policies', '/authors/editorial_policies/index.html', 'Editorial policies');

	newRows += getNewSubRow('How_to_submit', '/ng/authors/submit/index.html', 'How to submit');
	
	newRows += getNewSubRow('Content_types', '/ng/authors/article_types/index.html', 'Content types');
	
	newRows += getNewSubRow('Editorial_process', '/ng/authors/ed_process/index.html', 'Manuscript decisions');
	
	newRows += getNewSubRow('About_the_editors', '/ng/about/about_eds/index.html', 'About the editors');

	newRows += getNewSubRow('Masthead', '/ng/about/masthead/index.html', 'Masthead');
	

	if (isCurrentType(thisType) && thisType != getPageType()){
		hilite(thisType);
	}
	
	
	

}

document.write (getSubRow(newRows));
</script></td></tr><tr><td id="Online submission_1" style="background-color: rgb(239, 239, 219);" colspan="3"><table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td height="20"><img id="Online_submission_navimage" alt="Online submission" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/ng_icon_onlinesub.gif" align="absmiddle" border="0"></td><td><a class="leftnav2" href="http://mts-ng.nature.com.www.lib.ncsu.edu:2048/" target="_blank" id="Online_submission_link">Online submission</a></td></tr></tbody></table></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td colspan="3" height="12"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="12" width="5"></td></tr><tr><td id="For_referees_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="For_referees_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="http://www.nature.com.www.lib.ncsu.edu:2048/authors/editorial_policies/peer_review.html" id="For_referees_link">For referees</a></td><td id="For_referees_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td colspan="3" height="12"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="12" width="5"></td></tr><tr><td id="sample_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="sample_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v37/n1/index.html" id="sample_link">Free online issue</a></td><td id="sample_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td colspan="3" height="12"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="12" width="5"></td></tr>
<!--
<tr><td id="About_the_journal_1" width="5" style="background-color:
#EFEFDB"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="About_the_journal_2" style="background-color:
#EFEFDB"><a class="leftnav2" href="/ng/about/index.html" id="About_the_journal_link">About the journal</a></td><td id="About_the_journal_3" width="5" style="background-color:
#EFEFDB"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"><script LANGUAGE="JavaScript">

var thisType = 'About_the_journal';

addSubLevel(thisType, new Array("About_the_editors","Editorial_policies","Masthead"));

var newRows = '';
if (isCurrentType(thisType) || '' == 'true'){

	
	newRows += getNewSubRow('About_the_editors', '/ng/about/about_eds/index.html', 'About the editors');
	
	newRows += getNewSubRow('Editorial_policies', '/ng/about/ed_policies/index.html', 'Editorial policies');
	
	newRows += getNewSubRow('Masthead', '/ng/about/masthead/index.html', 'Masthead');
	

	if (isCurrentType(thisType) && thisType != getPageType()){
		hilite(thisType);
	}
	
	
	

}

document.write (getSubRow(newRows));
</script>-->

<tr><td id="Contact_the_journal_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="Contact_the_journal_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/contact/index.html" id="Contact_the_journal_link">Contact the journal</a></td><td id="Contact_the_journal_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Subscribe_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="Subscribe_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="https://secure-nature-com.www.lib.ncsu.edu:2433/subscribe/ng" target="_blank" id="Subscribe_link">Subscribe</a></td><td id="Subscribe_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Advertising_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="Advertising_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="http://advertising.nature.com.www.lib.ncsu.edu:2048/" target="_blank" id="Advertising_link">Advertising</a></td><td id="Advertising_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Work_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="Work_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="http://npg.nature.com.www.lib.ncsu.edu:2048/npg/forms/03_current.jsp" id="Work_link">work@npg</a></td><td id="Work_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Reprints_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="Reprints_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="http://www.nature.com.www.lib.ncsu.edu:2048/reprints" target="_blank" id="Reprints_link">Reprints and permissions</a></td><td id="Reprints_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="About_this_site_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="About_this_site_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/site/index.html" id="About_this_site_link">About this site</a></td><td id="About_this_site_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td colspan="3" height="12"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="12" width="5"></td></tr><tr><td id="For_librarians_1" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td><td id="For_librarians_2" style="background-color: rgb(239, 239, 219);"><a class="leftnav2" href="http://npg.nature.com.www.lib.ncsu.edu:2048/npg/servlet/Content?data=xml/07_home.xml&amp;style=xml/07_home.xsl" target="_blank" id="For_librarians_link">For librarians</a></td><td id="For_librarians_3" style="background-color: rgb(239, 239, 219);" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr></tbody></table>	<script language="JavaScript">
	hilite(getPageType());
</script>

<table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td colspan="3" height="20">&nbsp;</td></tr><tr><td bgcolor="#cc0033" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td><td class="leftnavOn" bgcolor="#cc0033">NPG Resources</td><td bgcolor="#cc0033" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>


<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr>
<tr><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/nature/">Nature</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>





<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/nbt">Nature Biotechnology</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>

<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr>
<tr><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/ncb/">Nature Cell Biology</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>


<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/nm/">Nature Medicine</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>


<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/nmeth/">Nature Methods</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>


<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/nrc/">Nature Reviews Cancer</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>



<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/nrg/">Nature Reviews Genetics</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>


<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/nrm/">Nature Reviews Molecular Cell Biology</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>

<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr>
<tr>
<td bgcolor="#eaeaea" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td>
<td bgcolor="#eaeaea">
<a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/news/">news@nature.com</a></td>
<td bgcolor="#eaeaea" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>


<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr>
<tr>
<td bgcolor="#eaeaea" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td>
<td bgcolor="#eaeaea">
<a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/natureconferences/">Nature Conferences</a></td>
<td bgcolor="#eaeaea" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>

<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr>

<tr>
<td bgcolor="#eaeaea" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td>
<td bgcolor="#eaeaea">
<a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/stemcells/">Nature Reports Stem Cells</a></td>
<td bgcolor="#eaeaea" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>

<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr>

<tr>
<td bgcolor="#eaeaea" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td>
<td bgcolor="#eaeaea">
<a target="_blank" class="leftnav3" href="http://www.nature.com.www.lib.ncsu.edu:2048/rnai/index.html">RNAi Gateway</a></td>
<td bgcolor="#eaeaea" width="5">
<img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="5"></td></tr>

<tr><td colspan="3" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="5"></td></tr>

</tbody></table>

<table border="0" cellpadding="0" cellspacing="0" width="130">
<tbody><tr>
<td colspan="2" height="20"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="20" width="5"></td>
</tr>
<tr>
<td colspan="2" height="20">
<table bgcolor="#cc0033" border="0" cellpadding="0" cellspacing="0" width="130">
<tbody><tr>
<td width="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="20" width="5"></td>
<td><a href="http:///" class="leftnavOn">NPG Subject areas</a></td>
</tr>
</tbody></table>
</td>
</tr>
<tr>
<td colspan="2" height="8"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="8" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_bio.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/biotech/" class="subject">Biotechnology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_can.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/cancer/" class="subject">Cancer</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_chem.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/chemistry/" class="subject">Chemistry</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_cli.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/clinicalmedicine/" class="subject">Clinical
Medicine</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_den.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/dentistry/" class="subject">Dentistry</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_dev.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/development/" class="subject">Development</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_dru.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/drugdisc/" class="subject">Drug Discovery</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_ear.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/earthsciences/" class="subject">Earth Sciences</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_evo.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/evoeco/" class="subject">Evolution &amp; Ecology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_gen.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/genetics/" class="subject">Genetics</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_imm.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/immuno/" class="subject">Immunology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_mat.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/materials/" class="subject">Materials Science</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_med.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/medicalresearch/" class="subject">Medical Research</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_mic.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/micro/" class="subject">Microbiology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_mol.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/molcellbio/" class="subject">Molecular Cell
Biology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_neu.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/neurosci/" class="subject">Neuroscience</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_pha.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/pharma/" class="subject">Pharmacology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/areas_phy.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.www.lib.ncsu.edu:2048/physics/" class="subject">Physics</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td colspan="2" height="6">
<table border="0" cellpadding="0" cellspacing="0" width="130">
<tbody><tr>
<td width="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_red.gif" alt="" height="5" width="3"></td>
<td><a href="http://npg.nature.com.www.lib.ncsu.edu:2048/npg/forms/04_cat.jsp" class="subjectred">Browse
all publications</a></td>
</tr>
</tbody></table>
</td>
</tr>
<tr>
<td colspan="2" height="5"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td colspan="2" height="1"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" alt="" height="1" width="5"></td>
</tr>
</tbody></table>
</td><td width="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="15"></td><td valign="top" width="470"><table border="0" cellpadding="0" cellspacing="0" width="470"><tbody><tr xmlns=""><td height="21" valign="top"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="21" width="1"></td></tr><tr xmlns=""><td height="22" valign="top"><span class="categ_large">Review</span></td></tr><tr xmlns=""><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_grey.gif" height="1" width="470"></td></tr><tr xmlns=""><td height="14"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="14" width="1"></td></tr><tr><td><span xmlns="" class="blacksml"><i>Nature Genetics</i>
&nbsp;<b>33</b>, 311 - 323 (2003)
 <br>doi:10.1038/ng1106
</span>
<h2 xmlns="">Proteomics: the first decade and beyond</h2><span xmlns="" class="author">Scott D. Patterson<sup>1,&nbsp;</sup><sup>3</sup>
&amp;&nbsp;Ruedi H. Aebersold<sup>2</sup><p class="blacksml"><sup>1</sup>&nbsp;Celera Genomics Corporation, 45 West Gude Drive, Rockville, Maryland 20850, USA.</p><p class="blacksml"><sup>2</sup>&nbsp;Institute for Systems Biology, 1441 North 34th Street, Seattle, Washington 98103, USA.</p><p class="blacksml"><sup>3</sup>&nbsp;Present address: Farmal Biomedicines, LLC, 129 North Hill Avenue, Suite #107, Pasadena, California 91106, USA.</p><span class="blacksml">
Correspondence should be addressed to Scott D. Patterson <a href="mailto:scottp@farmalbiomed.com">scottp@farmalbiomed.com</a></span></span><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="450"><tbody><tr><td height="20"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="1"></td></tr><tr><td height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_grey.gif" height="1" width="450"></td></tr><tr><td height="20"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="1"></td></tr></tbody></table><a xmlns="" name="abstract"></a><span xmlns="" class="articletext"><b>Proteomics
is the systematic study of the many and diverse properties of proteins
in a parallel manner with the aim of providing detailed descriptions of
the structure, function and control of biological systems in health and
disease. Advances in methods and technologies have catalyzed an
expansion of the scope of biological studies from the reductionist
biochemical analysis of single proteins to proteome-wide measurements.
Proteomics and other complementary analysis methods are essential
components of the emerging 'systems biology' approach that seeks to
comprehensively describe biological systems through integration of
diverse types of data and, in the future, to ultimately allow
computational simulations of complex biological systems.</b></span><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="470"><tbody><tr><td height="20"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="1"></td></tr><tr><td height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_grey.gif" height="1" width="470"></td></tr><tr><td height="20"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">Proteomics, like other 'discovery science' technologies (<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>)<sup><a href="#B1">1</a></sup>,
such as genomic sequencing, microarray analysis and metabolite
profiling, is the direct consequence of both the results obtained from
ambitious projects aimed at mapping and sequencing the complete genome
of many species and the changes to our models that are catalyzed by
such projects. The essence of this emerging 'systems biology' approach (<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>)
is that, for any given species, the space of possible biomolecules and
their organization into pathways and processes is large but finite. In
theory, therefore, the biological systems operating in a species can be
described comprehensively if a sufficient density of observations on
all of the elements that constitute the system can be obtained.
Proteomics is a particularly rich source of biological information
because proteins are involved in almost all biological activities and
they also have diverse properties, which collectively contribute
greatly to our understanding of biological systems. These properties
are summarized in <a href="#f1">Fig. 1</a>.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f1"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr><tr><td bgcolor="#efefdb"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F1.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F1.html">Figure&nbsp;1.</a>&nbsp;Representation of a eukaryotic cell.</b></td></tr><tr><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="160"></td><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F1.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F1.html"><img alt="Figure&nbsp;1 thumbnail" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/ng1106-F1.gif" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">A
section through a eukaryotic cell is shown, highlighting the diverse
properties of proteins. The systematic investigation of these
properties constitutes the field of proteomics. The subcellular
distribution, quantity, modification and interaction state, catalytic
activity and structure are particularly informative for describing
biological systems. Representative examples of protein properties are
shown, including the subcellular distribution of proteins to specific
compartments and organelles; the interaction of proteins with DNA,
other proteins or small molecules to form functional complexes (or
'machines') with diverse functions; and protein modifications such as
carbohydrate (CHO), phosphate (PO<sub>4</sub>) or lipid (lip).</p><br><br><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F1.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F1.html"><img alt="Full Figure" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F1.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F1.html">Full Figure and legend (71K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">Genome
sequencing, although technically complex, is conceptually simple and
has a defined end point: the conclusive determination of the complete
genome sequence of the species in question. Discovery science projects
aimed at assaying the function and control of biological systems are
based on less well-defined technologies and are devoid of clear end
points. An initial goal of proteomics was the rapid identification of
all of the proteins expressed by a cell or tissue—a goal that has yet
to be achieved for any species. Current goals of proteomic research are
more varied and directed toward the systematic determination of diverse
properties of proteins. These include sequence, quantity, state of
modification, interactions with other proteins, activity, subcellular
distribution and structure (<a href="#f1">Fig. 1</a>).</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Many
different technologies have been and are still being developed to
collect the information contained in the properties of proteins. <a href="#f2">Figure 2</a>
summarizes the current state of these technologies and how they relate
to other discovery science tools. Three characteristics of these
proteomic technologies are immediately apparent: first, there is no
single technology platform that can satisfy all of the desired
proteomic measurements; second, the closer the measurement to protein
function, the less mature the technology; and third, there is no
mature, 'true' proteomic technology as yet.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f2"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr><tr><td bgcolor="#efefdb"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F2.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F2.html">Figure&nbsp;2.</a>&nbsp;The current status of proteomic technologies.</b></td></tr><tr><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="160"></td><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F2.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F2.html"><img alt="Figure&nbsp;2 thumbnail" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/ng1106-F2.gif" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">The
different data typically collected in proteomic research and the
available technologies are listed. The relative maturity of the
proteomic technologies and other key discovery science tools is
apparent from the position of the respective technology on the graph.</p><br><br><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F2.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F2.html"><img alt="Full Figure" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F2.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F2.html">Full Figure and legend (33K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">In
this review, we do not discuss the three-dimensional structural
analysis of proteins (structural genomics), which is a large field in
its own right<sup><a href="#B2">2</a></sup>. Instead, we discuss three
phases of the emergence and maturation of proteomic concepts and
technology. The first spans the emergence from protein chemistry to
proteomics as a coordinated platform for discovery science. The second
is the current diversification of proteomic technologies and platforms,
which aim to capture the many properties of proteins. The third is a
window into the future, in which the variety of data obtained by
proteomics and other discovery science technologies will be integrated
and collectively interpreted to achieve a comprehensive understanding
of the workings of biological systems. We focus predominantly on
developments in proteomics in the past decade.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>From protein chemistry to proteomics</b></span><br xmlns=""><span xmlns="" class="articletext">Protein
chemistry was a key element of the reductionist research approaches
that were a mainstay of biology in the 1980s. Also called 'forward', as
in 'forward genetics', these approaches attempted to move from an
observed phenotype or function to the relevant genes and their products
that caused that phenotype. Together with rapidly advancing methods in
molecular biology such as gene cloning, sequencing and expression
analysis, protein chemistry provided the link between the observed
activity or function of a biochemically isolated protein and the gene
that encoded it. A key objective was therefore the development of
ever-more sensitive and reliable methods for protein sequencing to make
ever-smaller amounts of purified protein accessible to identification.
The long-term goals of this type of research were to reassemble <i>in vitro</i>
the system under study from its isolated components and to test whether
this reconstituted system would recapitulate functions observed <i>in vivo</i><sup><a href="#B3">3</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The advent of large-scale sequencing projects and their results<sup><a href="#B4">4</a></sup>
catalyzed the development of 'reverse' approaches, which attempted to
move from the gene sequence to function and phenotype. Such approaches
included the observation of clusters of mRNA species showing
coordinated expression patterns in different cellular states, either by
expression arrays or by serial analysis of gene expression (SAGE)<sup><a href="#B5">5,   
</a></sup><sup><a href="#B6">6</a></sup>. The idea of defining
functionally relevant patterns of gene expression by comparative
pattern analysis was also applied, and in fact pioneered, in the
protein science field through attempts to develop global approaches to
the quantitative analysis of protein expression patterns generated by
high-resolution two-dimensional gel electrophoresis or '2DE' (<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>)<sup><a href="#B7">7,   
</a></sup><sup><a href="#B8">8</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Below, we describe the evolution of protein chemistry methods and their adaptation from forward to reverse research strategies.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>From isolated proteins to gene sequence</b></span><br xmlns=""><span xmlns="" class="articletext">Technological
developments for analytical protein chemistry in the 1980s and early
1990s were primarily directed at improving the sensitivity of
techniques for identifying proteins separated by gels. Protein
sequencing often provided the crucial link between the activity of a
purified protein and its amino acid sequence or the sequence of the
corresponding gene. Edman sequencing of the amino terminus of intact
proteins or enzymatically digested fragments was the most common
approach to identify gel-separated proteins<sup><a href="#B9">9,   
</a></sup><sup><a href="#B10">10</a></sup>. In the likely case that the
sequence did not exist in the rather small sequence databases that
existed then, degenerate oligonucleotide primers could be synthesized
based on segments of the determined sequences and the corresponding
gene could be cloned using methods based on PCR<sup><a href="#B11">11</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Although
protein sequencing by Edman degradation was mature, reliable and
automated, it also had relatively poor sensitivity and was slow. With
the rapid increase in the size of sequence databases, which was fueled
by systematic sequencing programs, the chance that a particular protein
and/or gene sequence was already represented in a sequence database,
either in total or in part, also increased. This greatly facilitated
gene cloning and sometimes the complete sequence of a gene of interest
could be found by database searching, without the need for further
experimentation. Thus, the idea that protein sequences did not always
have to be determined <i>de novo</i> began to take shape.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>DNA sequencing rules the day</b></span><br xmlns=""><span xmlns="" class="articletext">In
the early 1990s, mass DNA sequencing of cDNAs derived from pools of
mRNA generated large numbers of expressed sequence tags (ESTs)<sup><a href="#B12">12</a></sup>.
These stretches of sequence provided an unprecedented window into the
transcripts present in particular types of cell and tissue, and
therefore a powerful tool for gene discovery<sup><a href="#B4">4</a></sup>.
Gene sequences (ESTs and others) also provided a resource that could
greatly accelerate protein identification by correlating experimentally
derived sequence segments with sequences in databases. At the time, it
was expected that eventually every gene of a species would be
represented in sequence databases and it would be possible to identify
proteins and/or genes simply by looking up the gene of interest using
experimentally derived data.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Large-scale EST sequencing represented the first real approach to the systematic sequencing of expressed genes<sup><a href="#B13">13</a></sup>. Although ESTs did provide much useful data, they did not produce the depth of analysis that had been anticipated<sup><a href="#B14">14</a></sup>.
In part, this was due to the dynamic range of transcript numbers
expressed in cells, which complicated the detection of low-abundance
species. This problem could be alleviated partially through the use of
normalized libraries<sup><a href="#B15">15</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The
ultimate normalized sequence libraries—that is, the complete genomic
sequences—were established only a few years later for yeast<sup><a href="#B16">16</a></sup> and a decade later for human<sup><a href="#B17">17,   
</a></sup><sup><a href="#B18">18</a></sup>. With such complete
libraries on hand, the rapid identification of proteins was limited
only by our capacity to extract sequence information from proteins and
peptides, and to correlate this information with the sequence
databases. 'Mass spectrometry' (<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>) and database search algorithms rapidly filled this gap.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>Protein and peptide mass spectrometry</b></span><br xmlns=""><span xmlns="" class="articletext">For
years, mass spectrometry has been the analytical chemist's workhorse
for analyzing small molecules. The high precision of mass spectrometric
measurements can distinguish closely related species, and tandem mass
spectrometry or 'MS/MS' can provide structural information on molecular
ions that can be isolated and fragmented within the instrument (<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>).</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">To measure the mass or, more specifically, the mass-to-charge ratio (<i>m</i>/<i>z</i>)
of a molecule in a mass spectrometer, the analyte must first be ionized
and transferred into the high vacuum system of the instrument. Peptides
and proteins, like other large molecules, proved difficult to ionize
under conditions that did not destroy the molecule. In the late 1980s,
two methods were developed that allowed the 'ionization' of peptides
and proteins at high sensitivity and without excessive fragmentation.
These breakthroughs were electrospray ionization ('ESI')<sup><a href="#B19">19</a></sup> and matrix-assisted laser desorption ionization ('MALDI')<sup><a href="#B20">20</a></sup>, which had closely followed the development of laser desorption<sup><a href="#B21">21,   
</a></sup><sup><a href="#B22">22</a></sup>(<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>).
The success of these ionization methods in analytical protein chemistry
led to the development of commercial mass spectrometers equipped with
robust ESI or MALDI 'ion source' instruments, which rapidly penetrated
the protein chemistry community.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">MALDI
ion sources were most commonly coupled with time-of-flight (TOF) 'mass
analyzers', whereas ESI was most often coupled with ion-trap or
triple-quadrupole MS/MS spectrometers (<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>).
Although MALDI-TOF mass spectrometers can determine the mass of a
protein or peptide with a high degree of accuracy, the intrinsic mass
of a eukaryotic protein is not a uniquely identifying feature. It was
quickly recognized, however, that the masses of the various peptides
generated by fragmentation of an isolated protein with an enzyme of
known cleavage specificity could uniquely identify a protein.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">In 1993, five independent reports were published that described the implementation of this insight in database search algorithms<sup><a href="#B23">23,   
</a></sup><sup><a href="#B24">24,   
</a></sup><sup><a href="#B25">25,   
</a></sup><sup><a href="#B26">26,   
</a></sup><sup><a href="#B27">27</a></sup>. These algorithms, together
with MALDI-TOF mass spectrometry peptide analysis, constituted a
'protein identification' method that is now known as peptide mass
mapping (or peptide mass fingerprinting). In this type of analysis, the
collected 'MS spectra' (<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>)
are used to generate a list of proteolytic (peptide) fragment masses,
which are then matched to the masses calculated from the same
proteo-lytic digestion of each entry in a sequence database, resulting
in identification of the target protein. The success of this type of
analysis is dependent on the specificity of the enzyme used (most
frequently trypsin), the number of peptides identified from each
protein species, and the mass accuracy of the mass spectrometer<sup><a href="#B28">28</a></sup>.
Owing to its increasing sensitivity and ease of use, MALDI-TOF mass
spectrometry has become the method of choice for protein identification
by peptide mass mapping and is commonly used for identifying proteins
separated by 2DE.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">As
discussed above, ESI ion sources were originally coupled mostly with
triple-quadrupole or ion-trap instruments. More recently,
hybrid-quadrupole TOF MS/MS spectrometers have become available and are
also used frequently with ESI. In addition to measuring peptide mass,
all of these instruments can isolate specific ions from a mixture on
the basis of their <i>m</i>/<i>z</i> ratio and fragment these ions in
the gas phase within the instrument, allowing the recording of MS/MS
spectra. Because peptide ions fragment in a sequence-dependent manner,
the MS/MS spectrum of a peptide, in principle, represents its amino
acid sequence.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Algorithms that match MS/MS spectra to sequence databases<sup><a href="#B29">29,   
</a></sup><sup><a href="#B30">30</a></sup> have greatly facilitated mass spectrometric protein identification by this approach<sup><a href="#B31">31</a></sup>.
Because a peptide sequence, and thus the MS/MS spectrum of a peptide,
can uniquely identify a protein, the specificity of MS/MS-based protein
identifications is often much higher than that of peptide mass mapping.
MS/MS spectra are also ideally suited to search translated EST and
other sequence databases containing incomplete sequences. ESI-MS/MS is
popular because it can be combined easily with standard peptide
separation tools, such as chromatography, and because it is directly
compatible with the solvents that are used to solubilize peptides.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The
method of nanospray-ESI, in which unseparated peptide mixtures were
sprayed into the mass spectrometer at very low flow rates and detected
at sensitivities not previously achieved by ESI-MS, was developed
subsequently<sup><a href="#B32">32,   
</a></sup><sup><a href="#B33">33</a></sup>. The very slow sample
consumption afforded by the low flow rate provided the opportunity to
generate fragment ion spectra of several of the observed precursor
ions. This was achieved by the operator, who manually selected
precursor ions. Subsequent developments in instrument control software
facilitated computer-controlled ion selection, such that MS/MS spectra
could be generated from many peptide ions in a given sample without the
need for operator intervention, effectively automating the process.
This was developed mostly for analyses in which mixtures of peptides
were supplied to the mass spectrometer from an online, capillary,
high-pressure liquid chromatography (HPLC) system—an approach referred
to as LC-MS/MS.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">As summarized in <a href="#f3">Fig. 3</a>,
peptide and protein separation technologies, advanced mass spectrometry
and MS/MS instrumentation and algorithms for searching mass
spectrometric data against sequence databases have been combined in
different ways to create a set of technologies for protein and proteome
analyses (<a href="#f4">Fig. 4</a>).</span><br xmlns=""><br xmlns=""><a xmlns="" name="f3"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr><tr><td bgcolor="#efefdb"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F3.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F3.html">Figure&nbsp;3.</a>&nbsp;Quantitative protein analysis from the cell to the identified protein.</b></td></tr><tr><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="160"></td><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F3.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F3.html"><img alt="Figure&nbsp;3 thumbnail" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/ng1106-F3.gif" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">The
two most common processes for quantitative proteome analysis are shown.
In the first (top), 2DE is used to separate and to quantify proteins,
and selected proteins are then isolated and identified by mass
spectrometry. In the second (bottom), LC-MS/MS is used to analyze
enzyme digests of unseparated protein mixtures, and accurate
quantification is achieved by labeling the peptides with stable
isotope. Both processes are compatible with protein fractionation or
separation methods, such as subcellular fractionation, protein complex
isolation and electrophoresis and chromatography, thereby providing
additional biological context to the protein samples being analyzed.</p><br><br><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F3.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F3.html"><img alt="Full Figure" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F3.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F3.html">Full Figure and legend (22K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr></tbody></table><a xmlns="" name="f4"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr><tr><td bgcolor="#efefdb"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F4.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F4.html">Figure&nbsp;4.</a>&nbsp;Time line indicating the convergence of different technologies and resources into a proteomic process.</b></td></tr><tr><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="160"></td><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F4.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F4.html"><img alt="Figure&nbsp;4 thumbnail" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/ng1106-F4.gif" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">Advances
in mass spectrometry and the generation of large quantities of
nucleotide sequence information, combined with computational algorithms
that could correlate the two, led to the emergence of proteomics as a
field.</p><br><br><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F4.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F4.html"><img alt="Full Figure" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F4.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F4.html">Full Figure and legend (33K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext"><b>The 'proteome' is born</b></span><br xmlns=""><span xmlns="" class="articletext">Long
before global differential analysis of mRNA expression was possible,
protein science relied on 2DE for generating reproducible protein
arrays, displaying large numbers of separated features and indicating
their quantities<sup><a href="#B34">34,   
</a></sup><sup><a href="#B35">35,   
</a></sup><sup><a href="#B36">36</a></sup>. On the strength of such 2DE
protein profiles, in the 1970s and 1980s ideas were proposed to build
protein databases (such as the human protein index<sup><a href="#B37">37</a></sup>) and to apply reverse strategies based on subtractive pattern analysis<sup><a href="#B38">38,   
</a></sup><sup><a href="#B39">39,   
</a></sup><sup><a href="#B40">40</a></sup>, similar to current popular
strategies for analyzing data obtained from gene expression
microarrays. In fact, at that time many of the principles now commonly
used for the global, quantitative analysis of gene expression patterns,
such as the use of clustering algorithms and multivariate statistics,
were developed in the context of 2DE<sup><a href="#B7">7,   
</a></sup><sup><a href="#B8">8</a></sup>. These ideas were not
substantially implemented then, however, mainly because 2DE by itself
is an essentially descriptive technique that does not indicate the
identity of the separated proteins and because the technique had been
plagued by reproducibility and other technical problems.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">With
the rapid advances in protein analytical technologies, fueled by the
addition of mass spectrometry, sequence databases and database search
tools, in the early 1990s it became possible for protein chemists to
identify and to examine the expression of many, if not all, of the
proteins resolvable by 2DE, and the possibility for large-scale protein
studies seemed attainable<sup><a href="#B41">41</a></sup>. It was in this context that in 1994, at the first 2DE meeting in Siena, Italy, the term 'proteome' was coined<sup><a href="#B42">42</a></sup>. The proteome was defined as the protein complement of the ge<u>nome</u>, and the process of studying the proteome became known as 'proteomics'.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">In
comparison to its nucleic acid−based counterpart, genomics, the
experimental complexity of proteomics is far greater. The technology is
also not as mature and, owing to the lack of amplification schemes akin
to PCR, only proteins isolated from a natural source can be analyzed.
Proteomic analyses are therefore generally limited by substrate. The
complexities of the proteome arise because most proteins seem to be
processed and modified in complex ways and can be the products of
differential splicing; in addition, protein abundance spans a range
estimated at five to six orders of magnitude for yeast cells<sup><a href="#B43">43</a></sup> and more than ten orders of magnitude for human blood serum—for example, from interleukin-6 at <img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/glyph.gif" alt="approx" align="bottom" border="0">2 pg/ml (ref. <a href="#B44">44</a>) to albumin at 50 mg/ml (ref. <a href="#B45">45</a>). Thus, the relatively low number of human genes predicted from the genome sequence<sup><a href="#B17">17,   
</a></sup><sup><a href="#B18">18</a></sup> has the potential to generate a proteome of enormous and as yet undetermined complexity.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">It
became rapidly apparent that a proteomic technology based strictly on
2DE was technically complex, labor- and therefore cost-intensive, and
fundamentally limited. The increased use of MALDI-MS and ESI-MS/MS in
the identification of 2DE-separated proteins showed that the incidence
of comigrating proteins even in this, the highest resolving protein
method known, was more prevalent than had been originally thought<sup><a href="#B46">46</a></sup>.
Because quantification in 2DE relies on the assumption that one protein
is present in each spot, comigration compromises such analyses. It was
also observed that with conventional protein staining methods only a
relatively small subset of a cellular proteome was apparent when
unfractionated cell lysates were separated by 2DE<sup><a href="#B46">46,   
</a></sup><sup><a href="#B47">47</a></sup>. Thus, despite the maturity
and unmatched performance of 2DE for separating intricate patterns of
differentially modified and processed proteins<sup><a href="#B48">48</a></sup>, and despite the continuing evolution of 2DE separation<sup><a href="#B49">49,   
</a></sup><sup><a href="#B50">50,   
</a></sup><sup><a href="#B51">51,   
</a></sup><sup><a href="#B52">52</a></sup> and detection technology<sup><a href="#B53">53</a></sup>, alternative methods for large-scale protein expression analysis began to be investigated more vigorously.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Hunt and co-workers<sup><a href="#B54">54</a></sup>
laid the groundwork for a gel-independent approach to proteomics by
demonstrating the ability of LC-MS/MS systems to handle extremely
complex peptide mixtures. Antigen-presenting lymphocytes continually
digest proteins and present some of the resulting peptides bound to
major histocompatibility complex (MHC) proteins for immune
surveillance. Hunt and co-workers<sup><a href="#B54">54</a></sup> used
immunoprecipitation to isolate the peptide-MHC complexes, extracted the
antigenic peptides and subjected the complex peptide mixtures to
successive LC-MS/MS analyses. They also used a specific cytotoxic T
cell response as a bioassay to confirm the presence of antigenic
peptides in each fraction and correlated this functional data with the
mass spectrometric data, thereby identifying the sequence of the
antigenic peptides<sup><a href="#B55">55,   
</a></sup><sup><a href="#B56">56,   
</a></sup><sup><a href="#B57">57</a></sup>. This approach clearly
established LC-MS/MS as a powerful tool for analyzing complex peptide
mixtures, and the application of this method to the analysis of peptide
mixtures generated by the proteolysis of complex protein samples was a
considerable step toward gel-independent proteomic technologies<sup><a href="#B58">58</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>Gel-independent quantitative profiling of the proteome</b></span><br xmlns=""><span xmlns="" class="articletext">The
combination of LC-MS/MS and sequence database searching has been widely
adopted for the analysis of complex peptide mixtures generated from the
proteolysis of samples containing several proteins. This approach is
often referred to as 'shotgun' proteomics<sup><a href="#B59">59,   
</a></sup><sup><a href="#B60">60</a></sup> and has the ability to
catalog hundreds, or even thousands, of components contained in samples
isolated from very different sources. Specific examples include the
identification of proteins in the periplasmic space of bacteria<sup><a href="#B61">61</a></sup>, yeast ribosomal complexes<sup><a href="#B62">62</a></sup>, murine nuclear interchromatin granule clusters (nuclear speckles)<sup><a href="#B63">63</a></sup>, murine mitochondrial soluble intermembrane proteins<sup><a href="#B64">64</a></sup>, human urinary proteins<sup><a href="#B65">65</a></sup>, yeast TFIID-associated proteins<sup><a href="#B66">66</a></sup>, proteasomal proteins<sup><a href="#B67">67</a></sup>, human microsomal proteins<sup><a href="#B68">68</a></sup>, human membrane proteins<sup><a href="#B69">69</a></sup> and yeast nuclear pore proteins<sup><a href="#B70">70</a></sup> pre-fractionated by SDS polyacrylamide gel electrophoresis, and proteins from yeast lysates<sup><a href="#B71">71</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Such
studies have also highlighted the limitations of shotgun proteomics,
including the difficulty of detecting and analyzing by
collision-induced dissociation (CID) mass spectrometry all of the
peptides in a sample, the qualitative nature of data-dependent
experiments, and the challenge of processing the tens of thousands of
CID spectra generated in a typical experiment—one of the many
informatics challenges that still faces scientists in this field. On
average, a protein digested with trypsin will generate 30−50 different
peptides. A tryptic digest of the proteome of a typical human cell will
therefore generate a peptide mixture containing at least hundreds of
thousands of peptides. Even the most advanced LC-MS/MS systems cannot
resolve and analyze such complexity in a reasonable amount of time.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">To
use LC-MS/MS for the analysis of most proteomes, therefore, a form of
complexity reduction (fractionation) is required. Two approaches have
been developed to tackle this problem. The first approach is the
selective enrichment of a subset of peptides from a complex mixture.
This has been mostly achieved by specifically targeting peptides that
contain a distinguishing feature, such as chemically reactive
sulfhydryl groups (cysteine residues)<sup><a href="#B59">59,   
</a></sup><sup><a href="#B72">72</a></sup> or residues that have been modified post-translationally with phosphate<sup><a href="#B73">73,   
</a></sup><sup><a href="#B74">74</a></sup> or carbohydrate<sup><a href="#B75">75,   
</a></sup><sup><a href="#B76">76</a></sup>. Ideally, such strategies
would select precisely one idiotypic peptide per protein. Although this
has not been realized, substantial reductions in sample complexity have
been achieved. But the trade-off is the loss of those proteins that do
not contain the selected feature.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The
second approach relies on extended upstream fractionation of the
complex peptide mixtures, with the aim of increasing the potential of
the mass spectrometer to detect and to sequence all of the components
of the sample. This has been implemented by using two or three
orthogonal peptide separation methods in sequence, the most successful
of which have involved cation exchange and capillary reverse-phase
chromatography<sup><a href="#B68">68,   
</a></sup><sup><a href="#B71">71,   
</a></sup><sup><a href="#B77">77</a></sup>. Although both
approaches—either by themselves or in combination—have produced
evidence that proteome cataloguing using such shotgun proteomic
strategies is feasible<sup><a href="#B71">71,   
</a></sup><sup><a href="#B78">78,   
</a></sup><sup><a href="#B79">79</a></sup>, a complete map of the proteome of any species has yet to be produced by any method.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">For
proteomic studies applying a forward (function to sequence) approach,
determination of the sequence of the target proteins is usually a
defined end point, because detailed functional analyses of the isolated
species precede sequence analysis. For studies that apply reverse
(sequence to function) approaches, knowing the sequence of the proteins
in a sample is necessary but not sufficient. Reverse approaches, which
are used in many proteomic studies, typically involve quantitative
comparison of the protein profiles expressed by cells or tissues in
different states.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The
most valuable information on the system being studied is obtained from
those proteins that are expressed differentially in a matrix of
proteins of unchanged expression; therefore, proteomic technologies
detecting differences in protein profiles need to be quantitative.
Unfortunately, peptides analyzed in a mass spectrometer will produce
different specific signal intensities depending on their chemical
composition, on the matrix in which they are present and on other
poorly understood variables. Thus, the intensity of a peptide ion
signal does not accurately reflect the amount of peptide in a sample;
in other words, mass spectrometry is inherently not a quantitative
technique. However, two peptides of identical chemical structure that
differ in mass because they differ in isotopic composition are
expected, according to stable isotope dilution theory<sup><a href="#B80">80</a></sup>,
to generate identical specific signals in a mass spectrometer. To turn
shotgun proteomics into a quantitative protein profiling method,
therefore, stable isotope dilution has been combined with the
complexity reduction techniques described above.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The first such approach was based on a class of reagents termed isotope-coded affinity tags or 'ICAT' (<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>), LC-MS/MS and sequence database searching<sup><a href="#B81">81</a></sup>.
The reagents consist of an alkylating group (iodoacetic acid) that
covalently attaches the reagent to reduced cysteine residues, a
polyether mass-encoded linker containing either eight hydrogens (d0) or
eight deuteriums (d8) that represents the isotope dilution and a biotin
affinity tag through which tagged peptides are selectively isolated.
For quantitative protoemics, the ICAT reagent approach, or similar
techniques, now provides an alternative method to the subtractive
2DE-based approaches discussed above.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">As shown in <a href="#f5">Fig. 5</a>,
the ICAT reagent approach involves labeling the cysteine residues in
one sample with d0-ICAT reagent and the cysteine residues in a second
sample with the d8-ICAT reagent. The samples are then combined. After
optional protein enrichment and enzymatic digestion of the combined
samples, the biotinylated ICAT-labeled peptides are enriched by means
of avidin affinity chromatography and analyzed by LC-MS/MS. Each
cysteinyl peptide appears as a pair of signals differing by the mass
differential encoded in the mass tag. The ratio of these signal
intensities precisely indicates the ratio of abundance of the protein
from which the peptide originates, and the MS/MS spectrum of either
isotopic form of the peptide allows the protein to be identified. Thus,
in a single, automated operation this method identifies the proteins
present in two related samples and determines the ratio of relative
abundance.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f5"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr><tr><td bgcolor="#efefdb"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F5.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F5.html">Figure&nbsp;5.</a>&nbsp;Quantitative proteomics using ICAT reagents.</b></td></tr><tr><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="160"></td><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F5.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F5.html"><img alt="Figure&nbsp;5 thumbnail" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/ng1106-F5.gif" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">The
ICAT reagent comprises a protein reactive group (such as a
sulfhydryl-specific reactivity), a mass-encoded linker and an affinity
tag (such as biotin). Variations of any of the three can be used to
facilitate the quantification of many different modifications or
activities. The general scheme used for this reagent is shown: first,
the protein reactive groups (such as cysteine residues) are labeled
separately with either light or heavy reagent, and then the proteins
are mixed and digested by enzymes; second, the labeled peptides are
captured and then quantified and identified by LC-MS/MS.</p><br><br><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F5.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F5.html"><img alt="Full Figure" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F5.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F5.html">Full Figure and legend (45K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">Variations of this approach have been described. For example, alternative labeling chemistries have been explored<sup><a href="#B82">82,   
</a></sup><sup><a href="#B83">83</a></sup>; stable isotope labeling has been achieved by a solid-phase isotope tag transfer method<sup><a href="#B84">84</a></sup>; <sup>16</sup>O or <sup>18</sup>O has been incorporated from H<sub>2</sub><sup>16</sup>O or H<sub>2</sub><sup>18</sup>O, respectively, at the carboxy terminus of peptides during proteolytic cleavage by trypsin<sup><a href="#B85">85,   
</a></sup><sup><a href="#B86">86,   
</a></sup><sup><a href="#B87">87</a></sup>; and stable-isotope
metabolic protein labeling has been attempted before mass spectrometric
analysis of intact proteins or proteolytic peptides in a high LC-MS/MS
system<sup><a href="#B88">88,   
</a></sup><sup><a href="#B89">89,   
</a></sup><sup><a href="#B90">90,   
</a></sup><sup><a href="#B91">91</a></sup>. Although isotope-tagging
methods based on chemical labeling after isolation are compatible with
essentially any protein sample, including organelles, body fluids or
subcellular and biochemical fractions, the application of metabolic
labeling is limited to those situations in which cells can be cultured
in isotopically defined media.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>Contemporary proteomics: a bewildering array of tools</b></span><br xmlns=""><span xmlns="" class="articletext">From its inception to the present day, proteomics has evolved substantially (<a href="#f4">Fig. 4</a>).
Conceptually, proteomics has become a biological assay for the
quantitative, subtractive analysis of complex protein samples.
Technologically, proteomics has become a suite of relatively mature
tools that support protein cataloguing and quantitative proteome
measurements reliably, sensitively and at high throughput. The
impressive successes of gene expression profiling ('DNA microarrays'; <a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>) for classifying cell types and cellular states in health and disease<sup><a href="#B92">92,   
</a></sup><sup><a href="#B93">93,   
</a></sup><sup><a href="#B94">94,   
</a></sup><sup><a href="#B95">95</a></sup> and for dissecting cellular pathways<sup><a href="#B96">96,   
</a></sup><sup><a href="#B97">97</a></sup> have illustrated the value
of the information obtained by systematic, quantitative expression
profiling experiments, and several insights have become apparent from
such studies.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">First,
global data sets are rich in information but difficult to analyze using
traditional knowledge-based interpretation. Second, the more data the
better: in comparison to interpretations of one or a few expression
profiles, it has been much more informative to collect several global
data sets on the same, differentially perturbed system, and to use
mathematical tools such as cluster analysis or singular value
decomposition<sup><a href="#B98">98,   
</a></sup><sup><a href="#B99">99</a></sup> to extract biological
insights or to formulate hypotheses. Third, proteomic and genomic
measurements done on the same system provide complementary information<sup><a href="#B100">100</a></sup> because neither the steady-state quantity of<sup><a href="#B43">43,   
</a></sup><sup><a href="#B101">101,   
</a></sup><sup><a href="#B102">102</a></sup> nor the response to perturbation-induced changes in mRNA or protein is mutually predictable<sup><a href="#B103">103,   
</a></sup><sup><a href="#B104">104</a></sup>. Extrapolating from
insights gained from the comparison of mRNA and protein expression
profiles, it is expected that additional systematic proteomic data,
including activity profiles, interaction maps and profiles of
(regulatory) modifications, will provide yet further insights into the
structure, function and control of biological systems.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Much
of the effort in contemporary proteomics is therefore directed toward
the development of suitable platforms on which to generate these data.
These efforts can be broadly grouped into three categories based on
mass spectrometry, microchips and genetics (<a href="#f4">Fig. 4</a>).</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>Mass spectrometry−based methods for studying the proteome</b></span><br xmlns=""><span xmlns="" class="articletext">The
success of combining the selective chemical labeling of proteins,
stable isotope tagging, LC-MS/MS and database searching as the basis
for quantitative protein profiling suggested that the same general
approach might be adapted to other types of quantitative proteomic
measurement. Over the years, protein chemistry has provided many
reagents for selectively capturing classes of protein with specific
biochemical properties, including lectins to capture glycoproteins,
immobilized metal ions to affinity-capture phosphorylated peptides, and
suicide enzyme substrates to capture specific enzymes. If such reagents
could be adapted to interact with their targets specifically and
tightly (ideally covalently), and if they could be made compatible with
stable isotope tagging methods, they might be used to profile
quantitatively the targeted functional group, activity or other
property on a proteome-wide scale.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Activity-based
reagents covalently label only enzymatically active forms of proteins
and can thus directly determine activity as opposed to total protein<sup><a href="#B105">105,   
</a></sup><sup><a href="#B106">106,   
</a></sup><sup><a href="#B107">107</a></sup>. For example, the
conversion of enzyme substrates into enzyme suicide inhibitors has been
achieved for serine hydrolases and cysteine proteases. In fact, the
study of proteases and their substrates and inhibitors in an organism
has been called "degradomics"<sup><a href="#B108">108</a></sup>, and
several studies on the cysteine and serine protease families have
provided 'proof of principle' for the idea. These enzymes form
covalently bound acyl intermediates during their catalytic cycle. This
property can be used to introduce tagged chemical probes by means of a
structure that mimics a general inhibitor and captures the active form
of the proteases covalently. Fluorescently tagged probes<sup><a href="#B107">107,   
</a></sup><sup><a href="#B109">109,   
</a></sup><sup><a href="#B110">110,   
</a></sup><sup><a href="#B111">111</a></sup> or biotinylated probes<sup><a href="#B112">112,   
</a></sup><sup><a href="#B113">113</a></sup> have been used successfully, and isotope-tagged versions of these reagents are being developed.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Although
the range of enzyme families to which this strategy has been applied so
far is limited, additional enzyme families such as glycosidases also
seem amenable to activity profiling<sup><a href="#B114">114</a></sup>.
In addition, drug screening projects that have been aborted because of
poor compound specificity provide a rich source of potential leads for
the development of class-specific reagents. For example, methods for
capturing ATP-binding proteins have been developed to enrich a range of
proteins, including protein kinases, with the potential for subsequent
(activity) profiling of the enriched proteins by quantitative proteomic
strategies<sup><a href="#B115">115</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">When
enzyme activities cannot be measured directly, they may be inferred
from the analysis of converted substrates. This has been achieved by
the quantitative measurement of substrate conversion using
isotope-tagged substrates and mass spectrometry<sup><a href="#B116">116</a></sup>,
or by trapping enzyme substrates on chemically or genetically altered
enzymes and then analyzing the isolated enzyme-substrate conjugates by
mass spectrometry. For example, chemically modified trypsin<sup><a href="#B117">117</a></sup> has been used to identify incompletely processed neuropeptides<sup><a href="#B118">118,   
</a></sup><sup><a href="#B119">119</a></sup>, and protein tyrosine
phosphatases in which the invariant catalytic amino acid aspartic acid
is mutated to alanine have been used as 'substrate traps', allowing the
identification of their physiological substrates<sup><a href="#B120">120,   
</a></sup><sup><a href="#B121">121</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Selective
reagents or chemical reactions have been used to extract phosphorylated
peptides from complex mixtures. Immobilized metal affinity
chromatography<sup><a href="#B73">73,   
</a></sup><sup><a href="#B122">122,   
</a></sup><sup><a href="#B123">123</a></sup> has been used successfully
to enrich phosphopeptides from relatively simple mixtures, but it lacks
the specificity to be effective with more-complex peptide samples. To
increase the specificity of metal affinity chromatography for
phosphopeptides, Ficarro <i>et al</i>.<sup><a href="#B74">74</a></sup>
eliminated interactions between the resin and carboxyl groups by
capping the carboxyl groups with methyl ester groups. This allowed them
to extract phosphopeptides selectively from the tryptic digest of a
yeast lysate. Subsequent LC-MS/MS analysis of the selected peptides
indicated the activation state of some yeast kinases from their
phosphorylation state—information that is not accessible by most
conventional proteomic approaches. Two covalent chemistries, one based
on <img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/glyph_003.gif" alt="beta" align="middle" border="0">-elimination reactions<sup><a href="#B124">124,   
</a></sup><sup><a href="#B125">125</a></sup> and one based on the formation of phosphoramidates<sup><a href="#B126">126</a></sup>,
have been developed with the same objective. Although none of these
methods is currently capable of quantitative phosphoprotein profiling
on a proteome-wide scale, these early studies represent a path to such
a technology.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Mass spectrometry−based proteomics is also rapidly becoming the method of choice for analyzing functional protein complexes<sup><a href="#B62">62,   
</a></sup><sup><a href="#B66">66,   
</a></sup><sup><a href="#B70">70,   
</a></sup><sup><a href="#B127">127,   
</a></sup><sup><a href="#B128">128</a></sup>. By providing a means to
identify multiple members of complexes, this approach complements the
view recently emphasized by Hartwell <i>et al</i>.<sup><a href="#B129">129</a></sup> that a cell is a collection of interconnected modules (<a href="#f1">Fig. 1</a>)—that
is, groups of proteins with many network interconnections that act
synergistically to execute a particular cellular function.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Indeed,
two large-scale 'protein complex' studies recently used affinity-tagged
bait proteins expressed in yeast cells to isolate the bait protein
together with its associated proteins<sup><a href="#B130">130,   
</a></sup><sup><a href="#B131">131</a></sup>. The composition of the
isolated complexes was analyzed by gel electrophoresis and mass
spectrometry, and thousands of protein interactions—many previously
unknown—were identified. But the poor overlap between the two data sets
when the same bait proteins were used, and between either data set and
results obtained by a genetic yeast two-hybrid screen (see below)<sup><a href="#B132">132</a></sup>, suggests that the conditions and parameters for such experiments need further optimization and standardization.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">In
an alternative approach using ICAT reagents to label the components of
a target protein complex and a suitable control isolate before
quantitative LC-MS/MS analysis, specific components of the complex
could be distinguished from nonspecifically associated proteins and
changes in the composition of protein complexes isolated from cells at
different states could be observed readily<sup><a href="#B133">133</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">In
the near future, it can be expected that quantitative proteomic
technologies will mature to become the obvious choice for
systematically determining the important, diverse properties of
proteins, including their activity and state of modification, and the
composition and dynamics of their functional modules.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>Chip-based methods for studying the proteome</b></span><br xmlns=""><span xmlns="" class="articletext">Complementary
DNA or oligonucleotide microarrays have proved invaluable for analyzing
transcript levels in several biological systems<sup><a href="#B5">5,   
</a></sup><sup><a href="#B134">134,   
</a></sup><sup><a href="#B135">135</a></sup>, and technological improvements continue to increase their utility<sup><a href="#B136">136,   
</a></sup><sup><a href="#B137">137</a></sup>. Array-based profiling
techniques are conceptually simple. A probe that is specific for a
particular analyte is placed at a defined position in a two-dimensional
array, and the interaction of the probe with its target molecule is
detected. Signals indicate that interactions have occurred, their
intensity and position on the array are recorded, and thus the probed
molecules and their quantity can be identified.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">It
therefore seemed obvious to apply similar principles to proteomic
analyses—in particular to protein expression profiling—by generating
ordered arrays of specific protein-binding modules<sup><a href="#B138">138,   
</a></sup><sup><a href="#B139">139,   
</a></sup><sup><a href="#B140">140</a></sup>. Such modules have included phage library−selected scFv antibodies<sup><a href="#B141">141</a></sup>, minibodies<sup><a href="#B142">142,   
</a></sup><sup><a href="#B143">143</a></sup>, cyclic peptides<sup><a href="#B144">144</a></sup>, reagents resulting from the scaffold engineering of various proteins<sup><a href="#B145">145,   
</a></sup><sup><a href="#B146">146,   
</a></sup><sup><a href="#B147">147</a></sup>, aptamers<sup><a href="#B148">148</a></sup>, antibodies<sup><a href="#B149">149</a></sup> and antibody mimics<sup><a href="#B150">150</a></sup>. In practice, however, the translation of array-based profiling from nucleic acids to proteins has faced many difficulties<sup><a href="#B151">151</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">First,
unlike cDNAs, proteins usually need to be captured in their native
conformation. This restricts the range of conditions that can be
applied to maintain solubility, to optimize interactions and to remove
nonspecific contaminants. Second, because at present proteins cannot be
amplified before analysis, detection methods must be very sensitive.
Third, proteins have no inherent properties that make them measurable
at high sensitivity, and the attachment of a detectable tag is prone to
interfere with a protein's interactions. Fourth, the interactions
between proteins and their binding reagents are less specific and of
lower and more variable affinity than those between Watson-Crick
base-paired nucleic acids. This increases the potential for
crossreactivity and complicates quantification, because substantial
differences in the dissociation constants for each protein-protein
interaction must be taken into account.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The
challenge of obtaining antibodies of sufficient specificity to make
protein expression arrays possible was recently demonstrated in studies
in which monoclonal antibodies that were probed against numerous
expressed proteins showed a considerable propensity to crossreact with
proteins other than the intended target protein<sup><a href="#B152">152,   
</a></sup><sup><a href="#B153">153</a></sup>. In spite of advances such as the development of high-sensitivity detection methods<sup><a href="#B154">154</a></sup> and chip surface engineering<sup><a href="#B155">155</a></sup>,
it is therefore unsurprising that the most successful protein
microarrays for measuring an analyte in a complex protein mixture use
relatively simple arrays of well-characterized antibodies<sup><a href="#B138">138</a></sup>, most notably those directed towards cytokines<sup><a href="#B154">154</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">At
present, the use of 'protein chips' has been more successful for
systematically measuring or inferring functional properties of
proteins. Direct functional measurements include the activities of
diverse enzymes<sup><a href="#B156">156</a></sup> such as protein kinases<sup><a href="#B157">157,   
</a></sup><sup><a href="#B158">158</a></sup>, protein-DNA interactions<sup><a href="#B159">159,   
</a></sup><sup><a href="#B160">160</a></sup>, profiles of reaction antibodies (autoantibodies) in blood serum and other clinical samples<sup><a href="#B152">152</a></sup>, and the interactions of proteins with small molecules<sup><a href="#B161">161</a></sup>. Using cDNA arrays assayed with probes generated from mRNAs extracted from membrane-associated polysomes, Diehn <i>et al</i>.<sup><a href="#B162">162</a></sup>
could infer the identity of secreted and membrane-associated proteins.
Similarly, in a technique that they call 'translation state array
analysis', Morris and co-workers<sup><a href="#B163">163</a></sup> have
profiled polysome-associated mRNAs using conventional cDNA array
technology to distinguish actively translated from non-translated
mRNAs. As the chemistry of protein attachment continues to improve, the
size of the arrays decreases<sup><a href="#B164">164</a></sup>; and, as different types of array are developed, additional uses for this technology will undoubtedly emerge.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>Genetic methods for studying the proteome</b></span><br xmlns=""><span xmlns="" class="articletext">Genetic
methods for interrogating the proteome are generally based on
recombinant DNA techniques that introduce different tags into all or
selected proteins expressed by a cell. These tags can be used to
observe directly or to infer specific properties of the tagged proteins.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The
strengths of genetic methods are their ability to target potentially
every protein, to use selection for inferring function, to assay the
protein in a cellular environment and to engineer cell strains with
specific properties, as well as the ease of assay automation<sup><a href="#B165">165</a></sup>.
The weaknesses of genetic methods are that tagging itself can
potentially interfere with the observed function and that the range of
species amenable to rigorous genetic engineering is limited. This means
that most observations—particularly those on mammalian proteins—are
made in a heterologous environment and that most readouts, with the
exception of certain fluorescence measurements, are indirect.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The prototypical genetic proteomic assay is the 'yeast two-hybrid system' (<a href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/box/ng1106_BX1.html','box','width=600,height=400,scrollbars=yes')">Box 1</a>), which was initially developed for detecting protein-protein interactions<sup><a href="#B166">166</a></sup> and has been reviewed recently<sup><a href="#B167">167</a></sup>. This method has been used for large-scale protein interaction screens in, for example, bacteriophage<sup><a href="#B168">168</a></sup>, vaccinia<sup><a href="#B169">169</a></sup>, yeast<sup><a href="#B170">170,   
</a></sup><sup><a href="#B171">171,   
</a></sup><sup><a href="#B172">172</a></sup>, <i>Helicobacter pylori</i><sup><a href="#B173">173</a></sup> and <i>Caenorhabditis elegans</i><sup><a href="#B174">174,   
</a></sup><sup><a href="#B175">175</a></sup>. The relative merits of
the systematic protein interaction maps generated by two-hybrid assay
and those generated by mass spectrometry<sup><a href="#B130">130,   
</a></sup><sup><a href="#B131">131</a></sup>, together with the matter
of which one, if any, best represents the true space of protein
interactions in a cell, is an important and currently unresolved issue<sup><a href="#B133">133</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">This type of method has also been extended to facilitate the determination of protein-mRNA interactions<sup><a href="#B176">176</a></sup> and for sensing protein-ligand binding<sup><a href="#B177">177</a></sup>. Related approaches in which proteins are expressed as fusions of green fluorescent protein or one of its variants<sup><a href="#B178">178,   
</a></sup><sup><a href="#B179">179</a></sup> are proving invaluable for probing several functional properties of proteins by direct <i>in vivo</i> observation<sup><a href="#B180">180</a></sup>.
Although such observations have been made mostly during specific
research projects, an initial compendium of the subcellular
localization of the yeast proteome<sup><a href="#B181">181</a></sup> clearly shows the potential of this approach as a proteomic assay.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><b>A window to the future</b></span><br xmlns=""><span xmlns="" class="articletext">The
scope of proteomic investigations has considerably broadened in the
past few years. Whereas initial efforts were focused on determining the
identity and quantity of proteins using a narrow selection of methods,
many emerging technologies now attempt to measure systematically all of
the biologically important properties of proteins. Although few, if
any, of these methods have reached the status of validated proteomic
tools, the rapid pace at which they are developing suggests that the
rich and varied sources of information contained in the proteome will
become increasingly accessible. The main challenges of the future will
be the validation, visualization, integration and interpretation, in a
biological context, of the vast amounts of diverse data generated by
the application of proteomic and genomic discovery science tools.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">In 1996, the first complete genomic sequence of a eukaryotic species, <i>Saccharomyces cerevisiae</i>, was published<sup><a href="#B16">16</a></sup>.
The early availability of this resource, the richness of knowledge
already acquired through decades of hypothesis-driven research and the
ease with which it could be experimentally manipulated made yeast the
model organism in which to test and to validate most of the
technologies and approaches discussed above. Because in many respects
the analysis of mammalian cells is more complex and technically
challenging, yeast can be viewed as a window into the future of
genomics-based and proteomics-based biological research. What insights
might this view provide?</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">First,
there will be a convergence of discovery science and hypothesis-driven
research. The beginning of such a convergence is already apparent in
information resources such as the yeast protein database<sup><a href="#B182">182</a></sup> and the Saccharomyces Genome Database<sup><a href="#B183">183</a></sup>,
in which systematically collected data and the results from
hypothesis-driven research published in the scientific literature have
been combined into highly useful resources for experimental biologists.
In addition, a recent study has shown the value of this union of data
as an experimental strategy to gain insights into the physiology of a
cell<sup><a href="#B100">100</a></sup>. In this study, both genomic and
proteomic data were collected from yeast cells in which all of the
known components of the galactose induction pathway had been perturbed
systematically. The different data were integrated into a mathematical
model consistent with the available information. This model was then
used to predict previously unknown interactions within the pathway and
between the galactose induction pathway and other cellular processes.
Some of these predictions were then verified experimentally<sup><a href="#B100">100</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Second,
systems biology approaches will detect connections between broad
cellular functions and pathways that were neither apparent nor
predictable despite decades of biochemical and genetic analysis of the
biological system in question. This has been validated broadly by
numerous mRNA expression studies, and also by studies based on
quantitative protein profiling (refs. <a href="#B184">184</a>, <a href="#B185">185</a>).
The rich and diverse information represented in large proteomic data
sets is expected to accelerate our understanding of the interdependence
of cellular processes.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Third,
our ability to collect large proteomic data sets already outstrips our
ability to validate, to interpret and to integrate such data for the
purpose of creating biological knowledge. Therefore, software tools
will be developed to help manage, interpret, integrate and understand
proteomic data. The lack of suitable software tools currently limits
essentially all areas of proteomic data analysis, from database
searching using MS/MS spectra to the assembly of large data sets
containing different types of data in relational databases (<a href="#f6">Fig. 6</a>).
To derive value from the data that goes beyond an initial scan for
'interesting observations' and to make data portable and comparable, it
will be necessary to develop algorithms that assign a score to each
data point that estimates the probability that the observation is
correct. Just as the assignment of quality scores to each base in DNA
sequencing using the algorithm Phred<sup><a href="#B186">186,   
</a></sup><sup><a href="#B187">187</a></sup> was essential for the
success of genome sequencing programs, it can be expected that
probability-based scores calculated for proteomic data<sup><a href="#B188">188</a></sup> will have a similar impact on proteomics.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f6"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr><tr><td bgcolor="#efefdb"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F6.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F6.html">Figure&nbsp;6.</a>&nbsp;Quantitative proteomics and informatics.</b></td></tr><tr><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="160"></td><td height="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F6.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F6.html"><img alt="Figure&nbsp;6 thumbnail" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/ng1106-F6.gif" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">Brief
descriptions of the informatics requirements for each of the processes
of biological analysis are listed. Handling these data requires
significant computational infrastructure if it is to be carried out
repeatedly on a large scale. Many of the algorithms used in the process
are still not mature.</p><br><br><a onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F6.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F6.html"><img alt="Full Figure" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_F6.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/fig_tab/ng1106_F6.html">Full Figure and legend (33K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">It
may seem that these trends might be realized with incremental
improvements of current proteomic technologies, and this may be largely
true for unicellular organisms. But multicellular organisms contain
levels of organization, such as the arrangement of cells in tissues,
and carry out processes, such as memory and immunity, for which there
are no counterparts in yeast or other unicellular organisms. The
transition from the application of proteomic strategies in unicellular
organisms to their application in higher eukaryotes is therefore much
more complex than one could estimate from a simple comparison of the
numbers of genes in the respective genomes.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Some
of these complexities, in particular the organization of cells into
tissues and to some extent the topology of the proteins contained in
them, are being investigated by an innovative emerging technique called
imaging mass spectrometry<sup><a href="#B189">189,   
</a></sup><sup><a href="#B190">190</a></sup>. Although at present this
technique does not afford the sensitivity and resolution to study
biological processes, it is immediately useful for generating
diagnostic patterns. Despite these advances and substantial increases
in the sensitivity, resolution and mass accuracy of new types of mass
spectrometer such as linear ion traps<sup><a href="#B191">191</a></sup>, MALDI-TOF-TOF<sup><a href="#B192">192</a></sup> and FT-ICR-MS<sup><a href="#B79">79,   
</a></sup><sup><a href="#B193">193</a></sup>, proteomics—especially when applied to higher eukaryotic species—will remain limited by technology for the foreseeable future.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">We
see four main challenges to be addressed in order for proteomics to
have a substantial impact on eukaryotic biology within the systems
biology model. The first challenge is the enormous complexity of the
proteome. For some proteins, in excess of 1,000 variants (splice and
translation isoforms, differentially modified and processed species)
have been described<sup><a href="#B194">194</a></sup>. The detection,
and particularly the molecular analysis of this complexity, remains an
unmatched task. The second challenge is the need for a general
technology for the targeted manipulation of gene expression in
eukaryotic cells. An approach that has proved successful for the
systematic analysis of biological systems relies on iterative cycles of
targeted perturbations of the system under study and the systematic
analysis of the consequences of each perturbation<sup><a href="#B100">100</a></sup>.
Although recent advances in using RNA interference in higher eukaryotic
cells open up exciting possibilities, the general targeted manipulation
of biological systems in these species remains unsolved. The third
challenge is the limited throughput of today's proteomic platforms:
iterative, systematic measurements on differentially perturbed systems
demand a sample throughput that is not matched by current proteomic
platforms. The fourth challenge is the lack of a general technique for
the absolute quantification of proteins. The ability to quantify
proteins absolutely, thereby eliminating the need for a reference
sample, would have far-reaching implications for proteomics—from the
determination of the stoichiometry of protein complexes to the design
of clinical studies aimed at discovering diagnostic markers.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Fortunately,
proteomics will have an impact on clinical and biological research well
before these challenges are met. We expect that precise clinical
diagnosis based on highly discriminating patterns of proteins in easily
accessible samples, particularly body fluids, may be the area in which
proteomics will make its first significant contribution<sup><a href="#B195">195,   
</a></sup><sup><a href="#B196">196</a></sup>. In the short term,
proteomics also can be expected to provide partial data sets of
sufficient quality, density and information content to provide the
basis for generating sophisticated mathematical models of biological
processes that will be able to simulate system properties such as
adaptation or robustness<sup><a href="#B197">197,   
</a></sup><sup><a href="#B198">198</a></sup>, which may not be apparent from the analysis of isolated elements of a system.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">In
its first decade, the field of proteomics has grown rapidly to
encompass numerous advanced technologies that strive to provide the
molecular data necessary for a comprehensive understanding of
biological processes. Although much ground has been covered, continued
advances in methods, instrumentation and computational analysis will be
needed to get closer to the workings of biology through the analysis of
these systems.</span><br xmlns=""><br xmlns=""><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="470"><tbody><tr><td rowspan="2" height="20"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="1"></td><td align="right"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black_up.gif" height="3" width="5">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td align="right" height="7"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="7" width="1"></td></tr><tr><td colspan="2" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_grey.gif" height="1" width="470"></td></tr><tr><td colspan="2" height="20"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="1"></td></tr></tbody></table><a xmlns="" name="References"></a><span xmlns="" class="heading3">REFERENCES </span><ol xmlns="" class="articletext">
<li><a name="B1"></a>Aebersold, R., Hood, L.E. &amp; Watts, J.D. Equipping scientists for the new biology. <i>Nat. Biotechnol.</i> <b>18</b>, 359 (2000).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F74325" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10748470&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B2"></a>Thornton, J. Structural genomics takes off. <i>Trends Biochem. Sci.</i> <b>26</b>, 88-99 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0968-0004%2800%2901765-5" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11327043&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B3"></a>Aebersold, R. &amp; Patterson, S.D. Current problems and technical solutions in protein biochemistry. In <i>PROTEINS: Analysis &amp; Design</i> (ed. Angeletti, R.H.) 3-120 (Academic, San Diego, 1998).</li>
<li><a name="B4"></a>Adams, M.D. <i>et al</i>. Initial assessment of human gene diversity and expression patterns based upon 83 million nucleotides of cDNA sequence. <i>Nature</i> <b>377</b>, 3-174 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7566098&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B5"></a>Schena, M., Shalon, D., Davis, R.W. &amp; Brown,
P.O. Quantitative monitoring of gene expression patterns with a
complementary DNA microarray. <i>Science</i> <b>270</b>, 467-470 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7569999&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B6"></a>Velculescu, V.E., Zhang, L., Vogelstein, B. &amp; Kinzler, K.W. Serial analysis of gene expression. <i>Science</i> <b>270</b>, 484-487 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7570003&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B7"></a>Anderson, N.L., Hofmann, J.P., Gemmell, A. &amp;
Taylor, J. Global approaches to quantitative analysis of
gene-expression patterns observed by use of two-dimensional gel
electrophoresis. <i>Clin. Chem.</i> <b>30</b>, 2031-2036 (1984).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=6499176&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B8"></a>Tarroux, P., Vincens, P. &amp; Rabilloud, T.
HERMeS: A second generation approach to the automatic analysis of
two-dimensional electrophoresis gels. Part V: Data analysis. <i>Electrophoresis</i> <b>8</b>, 187-199 (1987).</li>
<li><a name="B9"></a>Aebersold, R.H., Leavitt, J., Saavedra, R.A.,
Hood, L.E. &amp; Kent, S.B. Internal amino acid sequence analysis of
proteins separated by one- or two-dimensional gel electrophoresis after
<i>in situ</i> protease digestion on nitrocellulose. <i>Proc. Natl. Acad. Sci.</i> <b>84</b>, 6970-6974 (1987).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=3313383&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B10"></a>Vandekerckhove, J., Bauw, G., Puype, M., Van
Damme, J. &amp; Van Montagu, M. Protein-blotting on polybrene-coated
glass-fiber sheets. <i>Eur. J. Biochem.</i> <b>152</b>, 9-19 (1985).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=3899644&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B11"></a>Tempst, P., Link, A.J., Riviere, L.R., Fleming,
M. &amp; Elicone, C. Internal sequence analysis of proteins separated
on polyacrylamide gels at the submicrogram level: improved methods,
applications and gene cloning strategies. <i>Electrophoresis</i> <b>11</b>, 537-553 (1990).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2226409&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B12"></a>Adams, M.D. <i>et al</i>. Complementary DNA sequencing: expressed sequence tags and human genome project. <i>Science</i> <b>252</b>, 1651-1656 (1991).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2047873&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B13"></a>Adams, M.D., Kerlavage, A.R., Fields, C. &amp;
Venter, J.C. 3,400 new expressed sequence tags identify diversity of
transcripts in human brain. <i>Nat. Genet.</i> <b>4</b>, 256-267 (1993).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8358434&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B14"></a>Zhang, L. <i>et al</i>. Gene expression profiles in normal and cancer cells. <i>Science</i> <b>276</b>, 1268-1272 (1997).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.276.5316.1268" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9157888&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B15"></a>Bonaldo, M.F., Lennon, G. &amp; Soares, M.B. Normalization and subtraction: two approaches to facilitate gene discovery. <i>Genome Res.</i> <b>6</b>, 791-806 (1996).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8889548&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B16"></a>Goffeau, A. <i>et al</i>. Life with 6000 genes. <i>Science</i> <b>274</b>, 563-567 (1996).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2FSCIENCE.274.5287.546" target="external">Article</a>&nbsp;|</li>
<li><a name="B17"></a>Venter, J.C. <i>et al</i>. The sequence of the human genome. <i>Science</i> <b>291</b>, 1304-1351 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.1058040" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11181995&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B18"></a>Lander, E.S. <i>et al</i>. Initial sequencing and analysis of the human genome. <i>Nature</i> <b>409</b>, 860-921 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F35057062" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11237011&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B19"></a>Fenn, J.B., Mann, M., Meng, C.K., Wong, S.F.
&amp; Whitehouse, C.M. Electrospray ionization for mass spectrometry of
large biomolecules. <i>Science</i> <b>246</b>, 64-71 (1989).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2675315&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B20"></a>Karas, M. &amp; Hillenkamp, F. Laser desorption ionization of proteins with molecular masses exceeding 10000 daltons. <i>Anal. Chem.</i> <b>60</b>, 2299-2301 (1988).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=3239801&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B21"></a>Tanaka, K., Ido, Y., Akita, S., Yoshida, Y. &amp;
Yoshida, T. Detection of high mass molecules by laser desorption
time-of-flight mass spectrometry. In <i>Proc. 2nd Japan-China Joint Symp. Mass Spectrom</i>. (eds. Matsuda, H. &amp; Xiao-tian, L.) 185-188 (Osaka, Japan, 1987).</li>
<li><a name="B22"></a>Tanaka, K. <i>et al</i>. Protein and polymer analyses up to <i>m</i>/<i>z</i> 100,000 by laser ionization TOF-MS. <i>Rapid Commun. Mass Spectrom.</i> <b>2</b>, 151-153 (1988).</li>
<li><a name="B23"></a>Henzel, W.J. <i>et al</i>. Identifying proteins
from two-dimensional gels by molecular mass searching of peptide
fragments in protein sequence databases. <i>Proc. Natl. Acad. Sci. USA</i> <b>90</b>, 5011-5015 (1993).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8506346&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B24"></a>Mann, M., Hojrup, P. &amp; Roepstorff, P. Use of
mass spectrometric molecular weight information to identify proteins in
sequence databases. <i>Biol. Mass Spectrom.</i> <b>22</b>, 338-345 (1993).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8329463&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B25"></a>Pappin, D.J.C., Hojrup, P. &amp; Bleasby, A.J. Rapid identification of proteins by peptide-mass fingerprinting. <i>Curr. Biol.</i> <b>3</b>, 327-332 (1993).</li>
<li><a name="B26"></a>James, P., Quadroni, M., Carafoli, E. &amp; Gonnet, G. Protein identification by mass profile fingerprinting. <i>Biochem. Biophys. Res. Commun.</i> <b>195</b>, 58-64 (1993).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1006%2Fbbrc.1993.2009" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8363627&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B27"></a>Yates, J.R., III, Speicher, S., Griffin, P.R.
&amp; Hunkapiller, T. Peptide mass maps: a highly informative approach
to protein identification. <i>Anal. Biochem.</i> <b>214</b>, 397-408 (1993).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1006%2Fabio.1993.1514" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8109726&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B28"></a>Patterson, S.D. &amp; Aebersold, R. Mass spectrometric approaches for the identification of gel-separated proteins. <i>Electrophoresis</i> <b>16</b>, 1791-1814 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8586048&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B29"></a>Eng, J.K., McCormack, A.L. &amp; Yates, J.R.,
III. An approach to correlate tandem mass spectral data pf peptides
with amino acid sequences in a protein database. <i>J. Am. Soc. Mass Spectrom.</i> <b>5</b>, 976-989 (1994).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2F1044-0305%2894%2980016-2" target="external">Article</a>&nbsp;|</li>
<li><a name="B30"></a>Mann, M. Sequence database searching by mass spectrometric data. In <i>Microcharacterization of Proteins</i> (eds. Kellner, R., Lottspeich, F. &amp; Meyer, H.E.) 223-245 (VCH, Weinheim, 1994).</li>
<li><a name="B31"></a>Gras, R. &amp; Muller, M. Computational aspects of protein identification by mass spectrometry. <i>Curr. Opin. Mol. Ther.</i> <b>3</b>, 526-532 (2001).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11804266&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B32"></a>Wilm, M.S. &amp; Mann, M. Electrospray and Taylor-Cone theory, Dole's beam of macromolecules at last? <i>Int. J. Mass Spectrom. Ion Proc.</i> <b>136</b>, 167-180 (1994).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2F0168-1176%2894%2904024-9" target="external">Article</a>&nbsp;|</li>
<li><a name="B33"></a>Wilm, M. <i>et al</i>. Femtomole sequencing of proteins from polyacrylamide gels by nano-electrospray mass-spectrometry. <i>Nature</i> <b>379</b>, 466-469 (1996).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8559255&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B34"></a>Scheele, G.A. Two-dimensional gel analysis of soluble proteins. Characterization of guinea pig exocrine pancreatic proteins. <i>J. Biol. Chem.</i> <b>250</b>, 5375-5385 (1975).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1141235&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B35"></a>Klose, J. Protein mapping by combined isoelectric
focusing and electrophoresis of mouse tissues: a novel approach to
testing for induced point mutations in mammals. <i>Humangenetik</i> <b>26</b>, 231-243 (1975).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1093965&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B36"></a>O'Farrell, P.H. High resolution two-dimensional gel electrophoresis of proteins. <i>J. Biol. Chem.</i> <b>250</b>, 4007-4021 (1975).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=236308&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B37"></a>Anderson, N.G. &amp; Anderson, L. The human protein index. <i>Clin Chem.</i> <b>28</b>, 739-748 (1982).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7074867&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B38"></a>Garrels, J.I. The QUEST system for quantitative analysis of two-dimensional gels. <i>J. Biol. Chem.</i> <b>264</b>, 5269-5282 (1989).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2925692&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B39"></a>Garrels, J.I. &amp; Franza, B.R. Jr.
Transformation-sensitive and growth-related changes of protein
synthesis in REF52 cells. A two-dimensional gel analysis of SV40-,
adenovirus-, and Kirsten murine sarcoma virus-transformed rat cells
using the REF52 protein database. <i>J. Biol. Chem.</i> <b>264</b>, 5299-5312 (1989).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2564397&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B40"></a>Garrels, J.I. &amp; Franza, B.R., Jr. The REF52
protein database. Methods of database construction and analysis using
the QUEST system and characterizations of protein patterns from
proliferating and quiescent REF52 cells. <i>J. Biol. Chem.</i> <b>264</b>, 5283-5298 (1989).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2925693&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B41"></a>Anderson, N.L., Matheson, A.D. &amp; Steiner, S. Proteomics: applications in basic and applied biology. <i>Curr. Opin. Biotechnol.</i> <b>11</b>, 408-412 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0958-1669%2800%2900118-X" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10975462&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B42"></a>Wilkins, M.R. <i>et al</i>. Progress with proteome projects: why all proteins expressed by a genome should be identified and how to do it. <i>Biotech. Gen. Eng. Rev.</i> <b>13</b>, 19-50 (1995).</li>
<li><a name="B43"></a>Gygi, S.P., Rochon, Y., Franza, B.R. &amp; Aebersold, R. Correlation between protein and mRNA abundance in yeast. <i>Mol. Cell. Biol.</i> <b>19</b>, 1720-1730 (1999).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10022859&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B44"></a>Lai, R. <i>et al</i>. Prognostic value of plasma interleukin-6 levels in patients with chronic lymphocytic leukemia. <i>Cancer</i> <b>95</b>, 1071-1075 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2Fcncr.10772" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12209693&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B45"></a>Ritchie, R.F., Palomaki, G.E., Neveux, L.M. &amp;
Navolotskaia, O. Reference distributions for the negative acute-phase
proteins, albumin, transferrin, and transthyretin: a comparison of a
large cohort to the world's literature. <i>J. Clin. Lab. Anal.</i> <b>13</b>, 280-286 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F%28SICI%291098-2825%281999%2913%3A6%3C280%3A%3AAID-JCLA5%3E3.0.CO%3B2-U" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10633295&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B46"></a>Gygi, S.P., Corthals, G.L., Zhang, Y., Rochon, Y.
&amp; Aebersold, R. Evaluation of two-dimensional gel
electrophoresis-based proteome analysis technology. <i>Proc. Natl. Acad. Sci. USA</i> <b>97</b>, 9390-9395 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2FPNAS.160270797" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10920198&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B47"></a>Corthals, G.L., Wasinger, V.C., Hochstrasser,
D.F. &amp; Sanchez, J.C. The dynamic range of protein expression: a
challenge for proteomic research. <i>Electrophoresis</i> <b>21</b>, 1104-1115 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1104%3A%3AAID-ELPS1104%3E3.3.CO%3B2-3" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10786884&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B48"></a>Gauss, C., Kalkum, M., Lowe, M., Lehrach, H.
&amp; Klose, J. Analysis of the mouse proteome. I. Brain proteins:
separation by two-dimensional electrophoresis and identification by
mass spectrometry and genetic variation. <i>Electrophoresis</i> <b>20</b>, 575-600 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F%28SICI%291522-2683%2819990301%2920%3A3%3C575%3A%3AAID-ELPS575%3E3.3.CO%3B2-V" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10217174&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B49"></a>Rabilloud, T. Two-dimensional gel electrophoresis in proteomics: old, old fashioned, but it still climbs up the mountains. <i>Proteomics</i> <b>2</b>, 3-10 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F1615-9861%28200201%292%3A1%3C3%3A%3AAID-PROT3%3E3.0.CO%3B2-R" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11788986&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B50"></a>Herbert, B. Advances in protein solubilisation for two-dimensional electrophoresis. <i>Electrophoresis</i> <b>20</b>, 660-663 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F%28SICI%291522-2683%2819990101%2920%3A4%2F5%3C660%3A%3AAID-ELPS660%3E3.0.CO%3B2-Q" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10344231&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B51"></a>Gorg, A. <i>et al</i>. The current state of two-dimensional electrophoresis with immobilized pH gradients. <i>Electrophoresis</i> <b>21</b>, 1037-1053 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F%28SICI%291522-2683%2820000401%2921%3A6%3C1037%3A%3AAID-ELPS1037%3E3.0.CO%3B2-V" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10786879&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B52"></a>Unlu, M., Morgan, M.E. &amp; Minden, J.S.
Difference gel electrophoresis: a single gel method for detecting
changes in protein extracts. <i>Electrophoresis</i> <b>18</b>, 2071-2077 (1997).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9420172&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B53"></a>Rabilloud, T., Strub, J.M., Luche, S., van
Dorsselaer, A. &amp; Lunardi, J. A comparison between Sypro Ruby and
ruthenium II tris (bathophenanthroline disulfonate) as fluorescent
stains for protein detection in gels. <i>Proteomics</i> <b>1</b>, 699-704 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F1615-9861%28200104%291%3A5%3C699%3A%3AAID-PROT699%3E3.0.CO%3B2-C" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11678039&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B54"></a>Appella, E., Padlan, E.A. &amp; Hunt, D.F.
Analysis of the structure of naturally processed peptides bound by
class I and class II major histocompatibility complex molecules. <i>EXS</i> <b>73</b>, 105-119 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7579970&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B55"></a>Hunt, D.F. <i>et al</i>. Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass spectrometry. <i>Science</i> <b>255</b>, 1261-1263 (1992).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1546328&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B56"></a>Henderson, R.A. <i>et al</i>. HLA-A2.1-associated peptides from a mutant cell line: a second pathway of antigen presentation. <i>Science</i> <b>255</b>, 1264-1266 (1992).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1546329&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B57"></a>Hunt, D.F. <i>et al</i>. Peptides presented to the immune system by the murine class II major histocompatibility complex molecule I-Ad. <i>Science</i> <b>256</b>, 1817-1820 (1992).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1319610&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B58"></a>Yates, J.R., III, McCormack, A.L., Schieltz, D.,
Carmack, E. &amp; Link, A. Direct analysis of protein mixtures by
tandem mass spectrometry. <i>J. Prot. Chem.</i> <b>16</b>, 495-497 (1997).</li>
<li><a name="B59"></a>Spahr, C.S. <i>et al</i>. Simplification of complex peptide mixtures for proteomic analysis: reversible biotinylation of cysteinyl peptides. <i>Electrophoresis</i> <b>21</b>, 1635-1650 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F%28SICI%291522-2683%2820000501%2921%3A9%3C1635%3A%3AAID-ELPS1635%3E3.0.CO%3B2-1" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10870950&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B60"></a>Wolters, D.A., Washburn, M.P. &amp; Yates, J.R.
III. An automated multidimensional protein identification technology
for shotgun proteomics. <i>Anal. Chem.</i> <b>73</b>, 5683-5690 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fac010617e" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11774908&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B61"></a>Link, A.J., Carmack, E. &amp; Yates, J.R. III. A
strategy for the identification of proteins localized to subcellular
spaces: Application to <i>E-coli</i> periplasmic proteins. <i>Int. J. Mass Spectrom. Ion Proc.</i> <b>160</b>, 303-316 (1997).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0168-1176%2896%2904489-8" target="external">Article</a>&nbsp;|</li>
<li><a name="B62"></a>Link, A.J. <i>et al</i>. Direct analysis of protein complexes using mass spectrometry. <i>Nat. Biotechnol.</i> <b>17</b>, 676-682 (1999).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F10890" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10404161&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B63"></a>Mintz, P.J., Patterson, S.D., Neuwald, A.F.,
Spahr, C.S. &amp; Spector, D.L. Purification and biochemical
characterization of interchromatin granule clusters. <i>EMBO J.</i> <b>18</b>, 4308-4320 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1093%2Femboj%2F18.15.4308" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10428969&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B64"></a>Patterson, S.D. <i>et al</i>. Mass spectrometric identification of proteins released from mitochondria undergoing permeability transition. <i>Cell Death Diff.</i> <b>7</b>, 137-144 (2000).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fsj.cdd.4400640" target="external">Article</a>&nbsp;|</li>
<li><a name="B65"></a>Spahr, C.S. <i>et al</i>. Towards defining the
urinary proteome using liquid chromatography-tandem mass spectrometry
I. Profiling an unfractionated tryptic digest. <i>Proteomics</i> <b>1</b>, 93-107 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F1615-9861%28200101%291%3A1%3C93%3A%3AAID-PROT93%3E3.0.CO%3B2-3" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11680902&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B66"></a>Sanders, S.L., Jennings, J., Canutescu, A., Link,
A.J. &amp; Weil, P.A. Proteomics of the eukaryotic transcription
machinery: identification of proteins associated with components of
yeast TFIID by multidimensional mass spectrometry. <i>Mol. Cell. Biol.</i> <b>22</b>, 4723-4738 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1128%2FMCB.22.13.4723-4738.2002" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12052880&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B67"></a>Verma, R. <i>et al</i>. Proteasomal proteomics:
identification of nucleotide-sensitive proteasome-interacting proteins
by mass spectrometric analysis of affinity-purified proteasomes. <i>Mol. Biol. Cell</i> <b>11</b>, 3425-3439 (2000).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11029046&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B68"></a>Han, D.K., Eng, J., Zhou, H. &amp; Aebersold, R.
Quantitative profiling of differentiation-induced microsomal proteins
using isotope-coded affinity tags and mass spectrometry. <i>Nat. Biotechnol.</i> <b>19</b>, 946-951 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fnbt1001-946" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11581660&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B69"></a>Simpson, R.J. <i>et al</i>. Proteomic analysis of the human colon carcinoma cell line (LIM 1215): Development of a membrane protein database. <i>Electrophoresis</i> <b>21</b>, 1707-1732 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F%28SICI%291522-2683%2820000501%2921%3A9%3C1707%3A%3AAID-ELPS1707%3E3.0.CO%3B2-Q" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10870958&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B70"></a>Rout, M.P. <i>et al</i>. The yeast nuclear pore complex: composition, architecture, and transport mechanism. <i>J. Cell Biol.</i> <b>148</b>, 635-651 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1083%2FJCB.148.4.635" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10684247&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B71"></a>Washburn, M.P., Wolters, D. &amp; Yates, J.R.r
Large-scale analysis of the yeast proteome by multidimensional protein
identification technology. <i>Nat. Biotechnol.</i> <b>19</b>, 242-247 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F85686" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11231557&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B72"></a>Patterson, S.D. Using MS fragment-ion data to identify proteins from large sequence databases. In <i>Proteomics, Integrating Protein-based Tools and Applications for Drug Discovery</i> (ed. Savage, L.M.) 127-135 (International Business Communications, Southborough, 1998).</li>
<li><a name="B73"></a>Nuwaysir, L.M. &amp; Stults, J.T. Electrospray
ionization mass spectrometry of phosphopeptides isolated by on-line
immobilized metal-ion affinity chromatography. <i>J. Am. Soc. Mass Spectrom.</i> <b>4</b>, 662-669 (1993).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2F1044-0305%2893%2985031-R" target="external">Article</a>&nbsp;|</li>
<li><a name="B74"></a>Ficarro, S.B. <i>et al</i>. Phosphoproteome analysis by mass spectrometry and its application to <i>Saccharomyces cerevisiae</i>. <i>Nat. Biotechnol.</i> <b>20</b>, 301-305 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fnbt0302-301" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11875433&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B75"></a>Hayes, B.K., Greis, K.D. &amp; Hart, G.W. Specific isolation of <i>O</i>-Linked <i>N</i>-acetylglucosamine glycopeptides from complex mixtures. <i>Anal. Biochem.</i> <b>228</b>, 115-122 (1995).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1006%2Fabio.1995.1322" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8572267&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B76"></a>Greis, K.D. <i>et al</i>. Selective detection and site-analysis of <i>O</i>-GlcNAc-modified glycopeptides by <img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/glyph_003.gif" alt="beta" align="middle" border="0">-elimination and tandem electrospray mass spectrometry. <i>Anal. Biochem.</i> <b>234</b>, 38-49 (1996).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1006%2Fabio.1996.0047" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8742080&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B77"></a>Davis, M.T. <i>et al</i>. Automated LC-LC-MS-MS
platform using binary ion-exchange and gradient reversed-phase
chromatography for improved proteomic analyses. <i>J. Chromatogr. B</i> <b>752</b>, 281-291 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0378-4347%2800%2900547-8" target="external">Article</a>&nbsp;|</li>
<li><a name="B78"></a>Gygi, S.P., Rist, B., Griffin, T.J., Eng, J.
&amp; Aebersold, R. Proteome analysis of low-abundance proteins using
multidimensional chromatography and isotope-coded affinity tags. <i>J. Proteome Res.</i> <b>1</b>, 47-54 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fpr015509n" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12643526&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B79"></a>Smith, R.D. <i>et al</i>. An accurate mass tag strategy for quantitative and high-throughput proteome measurements. <i>Proteomics</i> <b>2</b>, 513-523 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F1615-9861%28200205%292%3A5%3C513%3A%3AAID-PROT513%3E3.0.CO%3B2-W" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11987125&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B80"></a>De Leenheer, A.P. &amp; Thienpont, L.M.
Application of isotope dilution-mass spectrometry in clinical
chemistry, pharmacokinetics, and toxicology. <i>Mass Spectrom. Rev.</i> <b>11</b>, 249-307 (1992).</li>
<li><a name="B81"></a>Gygi, S.P. <i>et al</i>. Quantitative analysis of complex protein mixtures using isotope-coded affinity tags. <i>Nat. Biotechnol.</i> <b>17</b>, 994-999 (1999).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F13690" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10504701&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B82"></a>Munchbach, M., Quadroni, M., Miotto, G. &amp; James, P. Quantitation and facilitated <i>de novo</i> sequencing of proteins by isotopic N-terminal labeling of peptides with a fragmentation-directing moiety. <i>Anal. Chem.</i> <b>72</b>, 4047-4057 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fac000265w" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10994964&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B83"></a>Cagney, G. &amp; Emili, A. <i>De novo</i> peptide sequencing and quantitative profiling of complex protein mixtures using mass-coded abundance tagging. <i>Nat. Biotechnol.</i> <b>20</b>, 163-170 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fnbt0202-163" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11821862&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B84"></a>Zhou, H., Ranish, J.A., Watts, J.D. &amp;
Aebersold, R. Quantitative proteome analysis by solid-phase isotope
tagging and mass spectrometry. <i>Nat. Biotechnol.</i> <b>20</b>, 512-515 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fnbt0502-512" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11981568&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B85"></a>Mirgorodskaya, O.A. <i>et al</i>. Quantitation of peptides and proteins by matrix-assisted laser desorption/ionization mass spectrometry using <sup>18</sup>O-labeled internal standards. <i>Rapid Commun. Mass Spectrom.</i> <b>14</b>, 1226-1232 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F1097-0231%2820000730%2914%3A14%3C1226%3A%3AAID-RCM14%3E3.0.CO%3B2-V" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10918372&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B86"></a>Yao, X., Freas, A., Ramirez, J., Demirev, P.A. &amp; Fenselau, C. Proteolytic <sup>18</sup>O labeling for comparative proteomics: model studies with two serotypes of adenovirus. <i>Anal. Chem.</i> <b>73</b>, 2836-2842 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fac001404c" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11467524&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B87"></a>Uttenweiler-Joseph, S., Neubauer, G., Christoforidis, S., Zerial, M. &amp; Wilm, M. Automated <i>de novo</i> sequencing of proteins using the differential scanning technique. <i>Proteomics</i> <b>1</b>, 668-682 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F1615-9861%28200104%291%3A5%3C668%3A%3AAID-PROT668%3E3.0.CO%3B2-S" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11678036&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B88"></a>Oda, Y., Huang, K., Cross, F.R., Cowburn, D.
&amp; Chait, B.T. Accurate quantitation of protein expression and
site-specific phosphorylation. <i>Proc. Natl. Acad. Sci. USA</i> <b>96</b>, 6591-6596 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2Fpnas.96.12.6591" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10359756&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B89"></a>Conrads, T.P. <i>et al</i>. Quantitative analysis of bacterial and mammalian proteomes using a combination of cysteine affinity tags and <sup>15</sup>N-metabolic labeling. <i>Anal. Chem.</i> <b>73</b>, 2132-2139 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fac001487x" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11354501&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B90"></a>Smith, R.D. <i>et al</i>. Rapid quantitative measurements of proteomes by Fourier transform ion cyclotron resonance mass spectrometry. <i>Electrophoresis</i> <b>22</b>, 1652-1668 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F1522-2683%28200105%2922%3A9%3C1652%3A%3AAID-ELPS1652%3E3.0.CO%3B2-E" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11425221&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B91"></a>Washburn, M.P., Ulaszek, R., Deciu, C., Schieltz,
D.M. &amp; Yates, J.R. III. Analysis of quantitative proteomic data
generated via multidimensional protein identification technology. <i>Anal. Chem.</i> <b>74</b>, 1650-1657 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fac015704l" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12043600&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B92"></a>Siebert, R., Rosenwald, A., Staudt, L.M. &amp; Morris, S.W. Molecular features of B-cell lymphoma. <i>Curr. Opin. Oncol.</i> <b>13</b>, 316-324 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1097%2F00001622-200109000-00002" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11555707&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B93"></a>Bangur, C.S. <i>et al</i>. Identification of
genes over-expressed in small cell lung carcinoma using suppression
subtractive hybridization and cDNA microarray expression analysis. <i>Oncogene</i> <b>21</b>, 3814-3825 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fsj.onc.1205480" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12032850&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B94"></a>van't Veer, L.J. <i>et al</i>. Gene expression profiling predicts clinical outcome of breast cancer. <i>Nature</i> <b>415</b>, 530-536 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F415530a" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11823860&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B95"></a>Spellman, P.T. <i>et al</i>. Comprehensive identification of cell cycle-regulated genes of the yeast <i>Saccharomyces cerevisiae</i> by microarray hybridization. <i>Mol. Biol. Cell</i> <b>9</b>, 3273-3297 (1998).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9843569&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B96"></a>Hughes, T.R. <i>et al</i>. Functional discovery via a compendium of expression profiles. <i>Cell</i> <b>102</b>, 109-126 (2000).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10929718&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B97"></a>Roberts, C.J. <i>et al</i>. Signaling and circuitry of multiple MAPK pathways revealed by a matrix of global gene expression profiles. <i>Science</i> <b>287</b>, 873-880 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.287.5454.873" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10657304&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B98"></a>Eisen, M.B., Spellman, P.T., Brown, P.O. &amp; Botstein, D. Cluster analysis and display of genome-wide expression patterns. <i>Proc. Natl. Acad. Sci. USA</i> <b>95</b>, 14863-14868 (1998).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2Fpnas.95.25.14863" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9843981&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B99"></a>Alter, O., Brown, P.O. &amp; Botstein, D. Singular value decomposition for genome-wide expression data processing and modeling. <i>Proc. Natl. Acad. Sci. USA</i> <b>97</b>, 10101-10106 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2Fpnas.97.18.10101" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10963673&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B100"></a>Ideker, T. <i>et al</i>. Integrated genomic and proteomic analyses of a systematically perturbed metabolic network. <i>Science</i> <b>292</b>, 929-934 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.292.5518.929" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11340206&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B101"></a>Anderson, L. &amp; Seilhamer, J. A comparison of selected mRNA and protein abundances in human liver. <i>Electrophoresis</i> <b>18</b>, 533-537 (1997).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9150937&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B102"></a>Futcher, B., Latter, G.I., Monardo, P., McLaughlin, C.S. &amp; Garrels, J.I. A sampling of the yeast proteome. <i>Mol. Cell. Biol.</i> <b>19</b>, 7357-7368 (1999).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10523624&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B103"></a>Betts, J.C., Lukey, P.T., Robb, L.C., McAdam, R.A. &amp; Duncan, K. Evaluation of a nutrient starvation model of <i>Mycobacterium tuberculosis</i> persistence by gene and protein expression profiling. <i>Mol. Microbiol.</i> <b>43</b>, 717-731 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1046%2Fj.1365-2958.2002.02779.x" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11929527&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B104"></a>Griffin, T.J. <i>et al</i>. Complementary profiling of gene expression at the transcriptome and proteome levels in <i>Saccharomyces cerevisiae</i>. <i>Mol. Cell. Proteomics</i> <b>1</b>, 323-333 (2002).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12096114&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B105"></a>Adam, G.C., Cravatt, B.F. &amp; Sorensen, E.J.
Profiling the specific reactivity of the proteome with non-directed
activity-based probes. <i>Chem. Biol.</i> <b>8</b>, 81-95 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS1074-5521%2800%2990060-7" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11182321&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B106"></a>Adam, G.C., Sorensen, E.J. &amp; Cravatt, B.F.
Proteomic profiling of mechanistically distinct enzyme classes using a
common chemotype. <i>Nat. Biotechnol.</i> <b>20</b>, 805-809 (2002).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12091914&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B107"></a>Greenbaum, D., Medzihradszky, K.F., Burlingame,
A. &amp; Bogyo, M. Epoxide electrophiles as activity-dependent cysteine
protease profiling and discovery tools. <i>Chem. Biol.</i> <b>7</b>, 569-581 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS1074-5521%2800%2900014-4" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11048948&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B108"></a>Lopez-Otin, C. &amp; Overall, C.M. Protease degradomics: a new challenge for proteomics. <i>Nat. Rev. Mol. Cell Biol.</i> <b>3</b>, 509-519 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fnrm858" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12094217&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B109"></a>Bogyo, M., Shin, S., McMaster, J.S. &amp;
Ploegh, H.L. Substrate binding and sequence preference of the
proteasome revealed by active-site-directed affinity probes. <i>Chem. Biol.</i> <b>5</b>, 307-320 (1998).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9653549&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B110"></a>Cravatt, B.F. &amp; Sorensen, E.J. Chemical strategies for the global analysis of protein function. <i>Curr. Opin. Chem. Biol.</i> <b>4</b>, 663-668 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS1367-5931%2800%2900147-2" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11102872&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B111"></a>Greenbaum, D. <i>et al</i>. Chemical approaches for functionally probing the proteome. <i>Mol. Cell. Proteomics</i> <b>1</b>, 60-68 (2002).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12096141&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B112"></a>Kidd, D., Liu, Y. &amp; Cravatt, B.F. Profiling serine hydrolase activities in complex proteomes. <i>Biochemistry</i> <b>40</b>, 4005-4015 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fbi002579j" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11300781&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B113"></a>Liu, Y., Patricelli, M.P. &amp; Cravatt, B.F. Activity-based protein profiling: the serine hydrolases. <i>Proc. Natl. Acad. Sci. USA</i> <b>96</b>, 14694-14699 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2FPNAS.96.26.14694" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10611275&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B114"></a>Withers, S.G. &amp; Aebersold, R. Approaches to labeling and identification of active-site residues in glycosidases. <i>Protein Sci.</i> <b>4</b>, 361-372 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7795519&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B115"></a>Haystead, C.M., Gregory, P., Sturgill, T.W. &amp; Haystead, T.A. <img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/glyph_002.gif" alt="gamma" align="middle" border="0">-Phosphate-linked
ATP-sepharose for the affinity purification of protein kinases. Rapid
purification to homogeneity of skeletal muscle mitogen-activated
protein kinase kinase. <i>Eur. J. Biochem.</i> <b>214</b>, 459-467 (1993).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8513796&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B116"></a>Turecek, F. Mass spectrometry in coupling with
affinity capture-release and isotope-coded affinity tags for
quantitative protein analysis. <i>J. Mass Spectrom.</i> <b>37</b>, 1-14 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2Fjms.275" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11813306&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B117"></a>Kumazaki, T., Terasawa, K. &amp; Ishii, S.
Affinity chromatography on immobilized anhydrotrypsin: general utility
for selective isolation of C-terminal peptides from protease digests of
proteins. <i>J. Biochem.</i> <b>102</b>, 1539-1546 (1987).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=3329196&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B118"></a>Fricker, L.D. <i>et al</i>. Identification and
characterization of proSAAS, a granin-like neuroendocrine peptide
precursor that inhibits prohormone processing. <i>J. Neurosci.</i> <b>20</b>, 639-648 (2000).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10632593&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B119"></a>Bures, E.J. <i>et al</i>. Identification of incompletely processed potential carboxypeptidase E substrates from CpE<sup><i>fat</i></sup>/CpE<sup><i>fat</i></sup> mice. <i>Proteomics</i> <b>1</b>, 79-92 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F1615-9861%28200101%291%3A1%3C79%3A%3AAID-PROT79%3E3.0.CO%3B2-8" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11680901&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B120"></a>Flint, A.J., Tiganis, T., Barford, D. &amp;
Tonks, N.K. Development of 'substrate-trapping' mutants to identify
physiological substrates of protein tyrosine phosphatases. <i>Proc. Natl. Acad. Sci. USA</i> <b>94</b>, 1680-1685 (1997).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2Fpnas.94.5.1680" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9050838&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B121"></a>Zhang, S.H., Liu, J., Kobayashi, R. &amp; Tonks,
N.K. Identification of the cell cycle regulator VCP (p97/CDC48) as a
substrate of the band 4.1-related protein-tyrosine phosphatase PTPH1. <i>J. Biol. Chem.</i> <b>274</b>, 17806-17812 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1074%2Fjbc.274.25.17806" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10364224&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B122"></a>Belew, M. &amp; Porath, J. Immobilized metal ion
affinity chromatography. Effect of solute structure, ligand density and
salt concentration on the retention of peptides. <i>J. Chromatogr.</i> <b>516</b>, 333-354 (1990).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0021-9673%2801%2989275-X" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2079494&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B123"></a>Posewitz, M.C. &amp; Tempst, P. Immobilized gallium(iii) affinity chromatography of phosphopeptides. <i>Anal. Chem.</i> <b>71</b>, 2883-2892 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fac981409y" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10424175&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B124"></a>Oda, Y., Nagasu, T. &amp; Chait, B.T. Enrichment analysis of phosphorylated proteins as a tool for probing the phosphoproteome. <i>Nat. Biotechnol.</i> <b>19</b>, 379-382 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F86783" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11283599&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B125"></a>Goshe, M.B. <i>et al</i>. Phosphoprotein isotope-coded affinity tag approach for isolating and quantitating phosphopeptides in proteome-wide analyses. <i>Anal. Chem.</i> <b>73</b>, 2578-2586 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fac010081x" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11403303&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B126"></a>Zhou, H., Watts, J.D. &amp; Aebersold, R. A systematic approach to the analysis of protein phosphorylation. <i>Nat. Biotechnol.</i> <b>19</b>, 375-378 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F86777" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11283598&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B127"></a>Pandey, A. &amp; Mann, M. Proteomics to study genes and genomes. <i>Nature</i> <b>405</b>, 837-846 (2000).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F35015709" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10866210&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B128"></a>Deshaies, R.J. <i>et al</i>. Charting the protein complexome in yeast by mass spectrometry. <i>Mol. Cell. Proteomics</i> <b>1</b>, 3-10 (2002).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12096135&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B129"></a>Hartwell, L.H., Hopfield, J.J., Leibler, S. &amp; Murray, A.W. From molecular to modular cell biology. <i>Nature</i> <b>402</b>, C47-C52 (1999).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F35011540" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10591225&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B130"></a>Gavin, A.C. <i>et al</i>. Functional organization of the yeast proteome by systematic analysis of protein complexes. <i>Nature</i> <b>415</b>, 141-147 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F415141a" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11805826&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B131"></a>Ho, Y. <i>et al</i>. Systematic identification of protein complexes in <i>Saccharomyces cerevisiae</i> by mass spectrometry. <i>Nature</i> <b>415</b>, 180-183 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F415180a" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11805837&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B132"></a>von Mering, C. <i>et al</i>. Comparative assessment of large-scale data sets of protein-protein interactions. <i>Nature</i> <b>417</b>, 399-403 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2FNATURE750" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12000970&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B133"></a>Ranish, J.A. <i>et al</i>. The study of macromolecular complexes by quantitative proteomics. <i>Nat. Genet.</i> (in the press).</li>
<li><a name="B134"></a>Pease, A.C. <i>et al</i>. Light-generated oligonucleotide arrays for rapid DNA sequence analysis. <i>Proc. Natl. Acad. Sci. USA</i> <b>91</b>, 5022-5026 (1994).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8197176&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B135"></a>Hughes, T.R. <i>et al</i>. Expression profiling using microarrays fabricated by an ink-jet oligonucleotide synthesizer. <i>Nat. Biotechnol.</i> <b>19</b>, 342-347 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F86730" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11283592&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B136"></a>van Berkum, N.L. &amp; Holstege, F.C. DNA microarrays: raising the profile. <i>Curr. Opin. Biotechnol.</i> <b>12</b>, 48-52 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0958-1669%2800%2900173-7" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11167072&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B137"></a>Nadon, R. &amp; Shoemaker, J. Statistical issues with microarrays: processing and analysis. <i>Trends Genet.</i> <b>18</b>, 265-271 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0168-9525%2802%2902665-3" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12047952&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B138"></a>Haab, B.B., Dunham, M.J. &amp; Brown, P.O.
Protein microarrays for highly parallel detection and quantitation of
specific proteins and antibodies in complex solutions. <i>Genome Biol.</i> <b>2</b>, RESEARCH0004 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1186%2Fgb-2001-2-2-research0004" target="external">Article</a>&nbsp;|</li>
<li><a name="B139"></a>Jenkins, R.E. &amp; Pennington, S.R. Arrays for
protein expression profiling: towards a viable alternative to
two-dimensional gel electrophoresis? <i>Proteomics</i> <b>1</b>, 13-29 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F1615-9861%28200101%291%3A1%3C13%3A%3AAID-PROT13%3E3.0.CO%3B2-J" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11680892&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B140"></a>Zhou, H., Roy, S., Schulman, H. &amp; Natan, M.J. Solution and chip arrays in protein profiling. <i>Trends Biotechnol.</i> <b>19</b>, S34-S39 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0167-7799%2801%2901798-X" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11780969&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B141"></a>McCafferty, J., Griffiths, A.D., Winter, G.
&amp; Chiswell, D.J. Phage antibodies: filamentous phage displaying
antibody variable domains. <i>Nature</i> <b>348</b>, 552-554 (1990).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2247164&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B142"></a>Tramontano, A. <i>et al</i>. The making of the minibody: an engineered <img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/glyph_003.gif" alt="beta" align="middle" border="0">-protein for the display of conformationally constrained peptides. <i>J. Mol. Recognit.</i> <b>7</b>, 9-24 (1994).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7986569&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B143"></a>Martin, F. <i>et al</i>. Coupling protein design and <i>in vitro</i> selection strategies: improving specificity and affinity of a designed <img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/glyph_003.gif" alt="beta" align="middle" border="0">-protein IL-6 antagonist. <i>J. Mol. Biol.</i> <b>255</b>, 86-97 (1996).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1006%2Fjmbi.1996.0008" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8568877&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B144"></a>Koivunen, E., Wang, B. &amp; Ruoslahti, E. Phage
libraries displaying cyclic peptides with different ring sizes: ligand
specificities of the RGD-directed integrins. <i>Biotechnology</i> <b>13</b>, 265-270 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9634769&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B145"></a>McConnell, S.J. &amp; Hoess, R.H. Tendamistat as a scaffold for conformationally constrained phage peptide libraries. <i>J. Mol. Biol.</i> <b>250</b>, 460-470 (1995).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1006%2Fjmbi.1995.0390" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7542349&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B146"></a>Nord, K., Nilsson, J., Nilsson, B., Uhlen, M. &amp; Nygren, P.A. A combinatorial library of an <img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/glyph_004.gif" alt="alpha" align="bottom" border="0">-helical bacterial receptor domain. <i>Protein Eng.</i> <b>8</b>, 601-608 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8532685&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B147"></a>Choo, Y. &amp; Klug, A. Designing DNA-binding proteins on the surface of filamentous phage. <i>Curr. Opin. Biotechnol.</i> <b>6</b>, 431-436 (1995).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2F0958-1669%2895%2980073-5" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7579654&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B148"></a>Brody, E.N. <i>et al</i>. The use of aptamers in large arrays for molecular diagnostics. <i>Mol. Diagn.</i> <b>4</b>, 381-388 (1999).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10671648&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B149"></a>Holt, L.J., Enever, C., de Wildt, R.M. &amp; Tomlinson, I.M. The use of recombinant antibodies in proteomics. <i>Curr. Opin. Biotechnol.</i> <b>11</b>, 445-449 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0958-1669%2800%2900133-6" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11024361&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B150"></a>Eklund, M., Axelsson, L., Uhlen, M. &amp;
Nygren, P.A. Anti-idiotypic protein domains selected from protein
A-based affibody libraries. <i>Proteins</i> <b>48</b>, 454-462 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2Fprot.10169" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12112671&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B151"></a>Mitchell, P. A perspective on protein microarrays. <i>Nat. Biotechnol.</i> <b>20</b>, 225-229 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fnbt0302-225" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11875416&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B152"></a>Cahill, D.J. Protein and antibody arrays and their medical applications. <i>J. Immunol. Methods</i> <b>250</b>, 81-91 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0022-1759%2801%2900325-8" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11251223&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B153"></a>Walter, G., Bussow, K., Cahill, D., Lueking, A.
&amp; Lehrach, H. Protein arrays for gene expression and molecular
interaction screening. <i>Curr. Opin. Microbiol.</i> <b>3</b>, 298-302 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS1369-5274%2800%2900093-X" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10851162&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B154"></a>Schweitzer, B. <i>et al</i>. Multiplexed protein profiling on microarrays by rolling-circle amplification. <i>Nat. Biotechnol.</i> <b>20</b>, 359-365 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fnbt0402-359" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11923841&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B155"></a>Houseman, B.T. &amp; Mrksich, M. Towards quantitative assays with peptide chips: a surface engineering approach. <i>Trends Biotechnol.</i> <b>20</b>, 279-281 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0167-7799%2802%2901984-4" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12062966&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B156"></a>Martzen, M.R. <i>et al</i>. A biochemical genomics approach for identifying genes by the activity of their products. <i>Science</i> <b>286</b>, 1153-1155 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.286.5442.1153" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10550052&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B157"></a>Houseman, B.T., Huh, J.H., Kron, S.J. &amp;
Mrksich, M. Peptide chips for the quantitative evaluation of protein
kinase activity. <i>Nat. Biotechnol.</i> <b>20</b>, 270-274 (2002).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fnbt0302-270" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11875428&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B158"></a>Zhu, H. <i>et al</i>. Analysis of yeast protein kinases using protein chips. <i>Nat. Genet.</i> <b>26</b>, 283-289 (2000).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F81576" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11062466&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B159"></a>Ren, B. <i>et al</i>. Genome-wide location and function of DNA binding proteins. <i>Science</i> <b>290</b>, 2306-2309 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.290.5500.2306" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11125145&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B160"></a>Krylov, A.S., Zasedateleva, O.A., Prokopenko,
D.V., Rouviere-Yaniv, J. &amp; Mirzabekov, A.D. Massive parallel
analysis of the binding specificity of histone-like protein HU to
single- and double-stranded DNA with generic oligodeoxyribonucleotide
microchips. <i>Nucleic Acids Res.</i> <b>29</b>, 2654-2660 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1093%2Fnar%2F29.12.2654" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11410675&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B161"></a>MacBeath, G. &amp; Schreiber, S.L. Printing proteins as microarrays for high-throughput function determination. <i>Science</i> <b>289</b>, 1760-1763 (2000).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10976071&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B162"></a>Diehn, M., Eisen, M.B., Botstein, D. &amp;
Brown, P.O. Large-scale identification of secreted and
membrane-associated gene products using DNA microarrays. <i>Nat. Genet.</i> <b>25</b>, 58-62 (2000).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F75603" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10802657&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B163"></a>Zong, Q., Schummer, M., Hood, L. &amp; Morris,
D.R. Messenger RNA translation state: the second dimension of
high-throughput expression screening. <i>Proc. Natl. Acad. Sci. USA</i> <b>96</b>, 10632-10636 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2Fpnas.96.19.10632" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10485877&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B164"></a>Lee, K.B., Park, S.J., Mirkin, C.A., Smith, J.C. &amp; Mrksich, M. Protein nanoarrays generated by dip-pen nanolithography. <i>Science</i> <b>295</b>, 1702-1705 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.1067172" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11834780&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B165"></a>Winzeler, E.A. <i>et al</i>. Functional characterization of the <i>S. cerevisiae</i> genome by gene deletion and parallel analysis. <i>Science</i> <b>285</b>, 901-906 (1999).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.285.5429.901" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10436161&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B166"></a>Fields, S. &amp; Song, O. A novel genetic system to detect protein-protein interactions. <i>Nature</i> <b>340</b>, 245-246 (1989).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2547163&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B167"></a>Tucker, C.L., Gera, J.F. &amp; Uetz, P. Towards an understanding of complex protein networks. <i>Trends Cell Biol.</i> <b>11</b>, 102-106 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0962-8924%2800%2901902-4" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11306254&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B168"></a>Bartel, P.L., Roecklein, J.A., SenGupta, D. &amp; Fields, S. A protein linkage map of <i>Escherichia coli</i> bacteriophage T7. <i>Nat. Genet.</i> <b>12</b>, 72-77 (1996).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8528255&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B169"></a>McCraith, S., Holtzman, T., Moss, B. &amp; Fields, S. Genome-wide analysis of vaccinia virus protein-protein interactions. <i>Proc. Natl. Acad. Sci. USA</i> <b>97</b>, 4879-4884 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2Fpnas.080078197" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10781095&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B170"></a>Uetz, P. <i>et al</i>. A comprehensive analysis of protein-protein interactions in <i>Saccharomyces cerevisiae</i>. <i>Nature</i> <b>403</b>, 623-627 (2000).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F35001009" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10688190&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B171"></a>Ito, T. <i>et al</i>. Toward a protein-protein
interaction map of the budding yeast: A comprehensive system to examine
two-hybrid interactions in all possible combinations between the yeast
proteins. <i>Proc. Natl. Acad. Sci. USA</i> <b>97</b>, 1143-1147 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2Fpnas.97.3.1143" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10655498&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B172"></a>Ito, T. <i>et al</i>. A comprehensive two-hybrid analysis to explore the yeast protein interactome. <i>Proc. Natl. Acad. Sci. USA</i> <b>98</b>, 4569-4574 (2001).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1073%2Fpnas.061034498" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11283351&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B173"></a>Rain, J.C. <i>et al</i>. The protein-protein interaction map of <i>Helicobacter pylori</i>. <i>Nature</i> <b>409</b>, 211-215 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F35051615" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11196647&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B174"></a>Walhout, A.J. <i>et al</i>. Protein interaction mapping in <i>C. elegans</i> using proteins involved in vulval development. <i>Science</i> <b>287</b>, 116-122 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.287.5450.116" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10615043&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B175"></a>Davy, A. <i>et al</i>. A protein-protein interaction map of the <i>Caenorhabditis elegans</i> 26S proteasome. <i>EMBO Rep.</i> <b>2</b>, 821-828 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fembor337" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11559592&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B176"></a>Zhang, B., Kraemer, B., SenGupta, D., Fields, S.
&amp; Wickens, M. Yeast three-hybrid system to detect and analyze
RNA-protein interactions. <i>Methods Enzymol.</i> <b>318</b>, 399-419 (2000).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10890002&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B177"></a>Tucker, C.L. &amp; Fields, S. A yeast sensor of ligand binding. <i>Nat. Biotechnol.</i> <b>19</b>, 1042-1046 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2Fnbt1101-1042" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11689849&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B178"></a>Heim, R., Prasher, D.C. &amp; Tsien, R.Y. Wavelength mutations and posttranslational autoxidation of green fluorescent protein. <i>Proc. Natl. Acad. Sci. USA</i> <b>91</b>, 12501-12504 (1994).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7809066&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B179"></a>Tsien, R.Y. The green fluorescent protein. <i>Annu. Rev. Biochem.</i> <b>67</b>, 509-544 (1998).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1146%2FANNUREV.BIOCHEM.67.1.509" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9759496&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B180"></a>Tsien, R.Y. &amp; Miyawaki, A. Seeing the machinery of live cells. <i>Science</i> <b>280</b>, 1954-1955 (1998).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.280.5371.1954" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9669950&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B181"></a>Kumar, A. <i>et al</i>. Subcellular localization of the yeast proteome. <i>Genes Dev.</i> <b>16</b>, 707-719 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1101%2FGAD.970902" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11914276&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B182"></a>Garrels, J.I. YPD--a database for the proteins of <i>Saccharomyces cerevisiae</i>. <i>Nucleic Acids Res.</i> <b>24</b>, 46-49 (1996).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1093%2Fnar%2F24.1.46" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8594598&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B183"></a>Cherry, J.M. <i>et al</i>. SGD: Saccharomyces Genome Database. <i>Nucleic Acids Res.</i> <b>26</b>, 73-79 (1998).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1093%2Fnar%2F26.1.73" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9399804&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B184"></a>Shiio, Y. <i>et al</i>. Quantitative proteomic analysis of Myc oncoprotein function. <i>EMBO J.</i> <b>21</b>, 5088-5096 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1093%2Femboj%2Fcdf525" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12356725&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B185"></a>Guina, T. <i>et al</i>. Quantitative proteomic analysis of <i>Pseudomonas aeruginosa</i> indicates synthesis of quinolone signal in adaptation to cystic fibrosis airways. <i>Proc. Natl. Acad. Sci. USA</i> (in the press).</li>
<li><a name="B186"></a>Ewing, B., Hillier, L., Wendl, M.C. &amp; Green,
P. Base-calling of automated sequencer traces using phred. I. Accuracy
assessment. <i>Genome Res.</i> <b>8</b>, 175-185 (1998).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9521921&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B187"></a>Ewing, B. &amp; Green, P. Base-calling of automated sequencer traces using phred. II. Error probabilities. <i>Genome Res.</i> <b>8</b>, 186-194 (1998).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9521922&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B188"></a>Keller, A. <i>et al</i>. Experimental protein mixture for validating tandem mass spectral analysis. <i>Omics</i> <b>6</b>, 207-212 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1089%2F153623102760092805" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12143966&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B189"></a>Caprioli, R.M., Farmer, T.B. &amp; Gile, J.
Molecular imaging of biological samples: localization of peptides and
proteins using MALDI-TOF MS. <i>Anal. Chem.</i> <b>69</b>, 4751-4760 (1997).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fac970888i" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9406525&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B190"></a>Stoeckli, M., Chaurand, P., Hallahan, D.E. &amp;
Caprioli, R.M. Imaging mass spectrometry: a new technology for the
analysis of protein expression in mammalian tissues. <i>Nat. Med.</i> <b>7</b>, 493-496 (2001).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F86573" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11283679&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B191"></a>Collings, B.A., Sudakov, M. &amp; Londry, F.A. Resonance shifts in the excitation of the <i>n</i> = 0, <i>K</i> = 1 to 6 quadrupolar resonances for ions confined in a linear ion trap. <i>J. Am. Soc. Mass Spectrom.</i> <b>13</b>, 577-586 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS1044-0305%2802%2900361-6" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12019981&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B192"></a>Medzihradszky, K.F. <i>et al</i>. The
characteristics of peptide collision-induced dissociation using a
high-performance MALDI-TOF/TOF tandem mass spectrometer. <i>Anal. Chem.</i> <b>72</b>, 552-558 (2000).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1021%2Fac990809y" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10695141&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B193"></a>Marshall, A.G., Hendrickson, C.L. &amp; Jackson, G.S. Fourier transform ion cyclotron resonance mass spectrometry: a primer. <i>Mass Spectrom. Rev.</i> <b>17</b>, 1-35 (1998).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1002%2F%28SICI%291098-2787%281998%2917%3A1%3C1%3A%3AAID-MAS1%3E3.0.CO%3B2-K" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9768511&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B194"></a>Ullrich, B., Ushkaryov, Y.A. &amp; Sudhof, T.C.
Cartography of neurexins: more than 1000 isoforms generated by
alternative splicing and expressed in distinct subsets of neurons. <i>Neuron</i> <b>14</b>, 497-507 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7695896&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B195"></a>Petricoin, E.F. <i>et al</i>. Use of proteomic patterns in serum to identify ovarian cancer. <i>Lancet</i> <b>359</b>, 572-577 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1016%2FS0140-6736%2802%2907746-2" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11867112&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B196"></a>Adam, B.L. <i>et al</i>. Serum protein
fingerprinting coupled with a pattern-matching algorithm distinguishes
prostate cancer from benign prostate hyperplasia and healthy men. <i>Cancer Res.</i> <b>62</b>, 3609-3614 (2002).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12097261&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B197"></a>Alon, U., Surette, M.G., Barkai, N. &amp; Leibler, S. Robustness in bacterial chemotaxis. <i>Nature</i> <b>397</b>, 168-171 (1999).&nbsp;|&nbsp;<a href="http://www.nature.com.www.lib.ncsu.edu:2048/doifinder/10.1038%2F16483" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9923680&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li>
<li><a name="B198"></a>Guet, C.C., Elowitz, M.B., Hsing, W. &amp; Leibler, S. Combinatorial synthesis of genetic networks. <i>Science</i> <b>296</b>, 1466-1470 (2002).&nbsp;|&nbsp;<a href="http://dx.doi.org.www.lib.ncsu.edu:2048/10.1126%2Fscience.1067407" target="external">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.www.lib.ncsu.edu:2048/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12029133&amp;dopt=Abstract" target="external">PubMed</a> &nbsp;|</li></ol><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="470"><tbody><tr><td rowspan="2" height="20"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="1"></td><td align="right"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black_up.gif" height="3" width="5">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td align="right" height="7"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="7" width="1"></td></tr><tr><td colspan="2" height="1"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_grey.gif" height="1" width="470"></td></tr><tr><td colspan="2" height="20"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="20" width="1"></td></tr></tbody></table><a xmlns="" name="Ack"></a><span xmlns="" class="heading3">Acknowledgments</span><br xmlns=""><span xmlns="" class="articletext">We would like to thank L. Feltz for administrative assistance and J. Watts for review of the manuscript.</span><br xmlns=""><br xmlns=""></td></tr></tbody></table></td><td width="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="1" width="15"></td><td valign="top" width="130"><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="34" width="1"></td></tr><tr><td colspan="2" height="22"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="22" width="1"></td></tr><tr><td colspan="2" class="whitesml" align="center" bgcolor="#4d876e" height="20"><b>FULL  TEXT</b></td></tr><tr><td colspan="2" class="blacksml2" align="center" bgcolor="#efefdb" height="17" valign="middle"><table border="0" cellpadding="1" cellspacing="0" width="106"><tbody><tr><td width="3"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black_prev.gif" border="0" height="5" width="3"></td><td align="center" width="100"><a class="breadcrumblink" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/full/ng1109.html">Previous</a></td></tr></tbody></table></td></tr><tr><td colspan="2"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_003.gif" height="1" width="130"></td></tr><tr><td class="blacksml2" align="center" bgcolor="#efefdb" height="15" width="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black_002.gif" border="0" height="5" width="3"></td><td class="blacksml2" bgcolor="#efefdb" height="17" width="115"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/index.html" class="breadcrumblink">Table of contents</a></td></tr><tr><td colspan="2" class="blacksml2" align="center" height="15"><table border="0" cellpadding="1" cellspacing="0" width="130"><tbody><tr><td bgcolor="#efefdb"><table border="0" cellpadding="0" cellspacing="0" width="128"><tbody><tr><td colspan="2" align="center" bgcolor="#ffffff"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="5" width="1"></td></tr><tr><td align="center" bgcolor="#ffffff" width="23"><img alt="Download PDF" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/icon_pdf.gif" height="14" width="14"></td><td bgcolor="#ffffff" height="14" width="105"><a class="breadcrumblink" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/pdf/ng1106.pdf">Download PDF</a></td></tr><tr><td colspan="2" align="center" bgcolor="#ffffff" valign="middle"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="6" width="1"></td></tr><tr><td align="center" bgcolor="#ffffff" valign="middle" width="23"><img alt="Send to a friend" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/icon_email.gif" height="10" width="13"></td><td bgcolor="#ffffff" height="12" valign="middle" width="105"><a class="breadcrumblink" onclick="window.open('/foxtrot/svc/mailform?file=/ng/journal/v33/n3s/full/ng1106.html','mailafriend','width=600,height=600,resizable=yes,location=no,menubar=no,scrollbars=yes,personalbar=no,status=yes,top=10,left=10');" href="javascript:void(0)">Send 
to a friend</a></td></tr><tr><td colspan="2" align="center" bgcolor="#ffffff" valign="middle"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_003.gif" height="5" width="1"></td></tr><tr><td align="center" bgcolor="#ffffff" valign="middle" width="23"><img alt="Save this link" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/connotea.gif" height="10" width="13"></td><td bgcolor="#ffffff" height="12" valign="middle" width="105"><a class="breadcrumblink" href="http://www.connotea.org/add?uri=http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/full/ng1106.html">Save this link</a></td></tr><tr><td colspan="2" align="center" bgcolor="#ffffff" valign="middle"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_003.gif" height="5" width="1"></td></tr></tbody></table></td></tr></tbody></table></td></tr></tbody></table><div class="box" id="natjob">
<h2 class="natjob"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/naturejobs/"><span class="first">nature</span><span class="hidden"> </span><span class="second">jobs</span></a></h2>

	<ul class="natjob">

	<li><h3 class="first"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/naturejobs/science/jobs/89812-Project-Assistant">Project Assistant</a></h3>
	<ul>
	<li>National Brain Research Centre</li>
	<li>Manesar, Gurgaon, Haryana 122 050 India</li>
	</ul>
	</li>

			<li><h3><a href="http://www.nature.com.www.lib.ncsu.edu:2048/naturejobs/science/jobs/92207-Post-Doctoral-Research-Associates">Post Doctoral Research Associates</a></h3>
		<ul>
		<li>The University of Manchester</li>
		<li>Manchester United Kingdom</li>
		</ul>
		</li>
	</ul>


<div class="natjobmore"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/naturejobs/science/jobs">More science jobs</a></div>
<div class="natjobmore"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/naturejobs/science/jobs/new">Post a job for free</a></div>
</div>

<table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td height="10"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_003.gif" height="1" width="15"></td><td height="10"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_003.gif" height="1" width="115"></td></tr><tr><td class="blacksml2" align="center" width="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black_dwn.gif" border="0" height="3" width="5"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="#abstract">Abstract</a></td></tr><tr><td class="blacksml2" align="center" valign="top" width="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black_002.gif" border="0" height="5" vspace="4" width="3"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="javascript:void(0)" onclick="window.open('/ng/journal/v33/n3s/fig_tab/ng1106_ft.html','fig_tab','width=600,height=400,scrollbars=yes')">Figures &amp; Tables</a></td></tr><tr><td class="blacksml2" align="center" width="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black_dwn.gif" border="0" height="3" width="5"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="#Ack">Acknowledgments</a></td></tr><tr><td class="blacksml2" align="center" width="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black_dwn.gif" border="0" height="3" width="5"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="#References">References</a></td></tr><tr><td colspan="2" class="blacksml2" align="center"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="5" width="1"></td></tr><tr><td colspan="2" class="blacksml2" align="center"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_grey.gif" height="1" width="130"></td></tr><tr><td class="blacksml2" align="center" width="15"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black.gif" alt="" border="0" height="5" width="3"></td><td class="blacksml2" height="15" width="115"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/ris/ng1106.ris" target="_blank" class="breadcrumblink">Export citation</a></td></tr><tr><td class="blacksml2" align="center" width="15"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black.gif" alt="" border="0" height="5" width="3"></td><td class="blacksml2" height="15" width="115"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v33/n3s/ris/ng1106refs.ris" target="_blank" class="breadcrumblink">Export references</a></td></tr><tr><td colspan="2" class="blacksml2" align="center"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="5" width="1"></td></tr>
 <script>
 var show = true;
 if(show)
 {
 document.write('<tr>');
 document.write('<td class="blacksml2" width="15" align="center">');
 document.write('<img src="/ng/images/arrow_black.gif" width="3" height="5" border="0" alt="">');
 document.write('</td>');
 document.write('<td class="blacksml2" width="115" height="15">');
 }
 </script><tr><td class="blacksml2" align="center" width="15"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/arrow_black_002.gif" alt="" border="0" height="5" width="3"></td><td class="blacksml2" height="15" width="115">
<a class="breadcrumblink" href="javascript:void(0)" onclick="window.open('http://sherman.nature.com.www.lib.ncsu.edu:2048/arls/Query?prod=NG&amp;query=http%3A//www.nature.com/doifinder/10.1038/ng1106&amp;type=RV&amp;src_title=Proteomics%3A%20the%20first%20decade%20and%20beyond&amp;src_jrnl=Nature%20Genetics&amp;src_vol=33&amp;src_pp=311-323&amp;src_date=2003&amp;target=popup', 'autonomypopup', 'resizeable=yes,scrollbars=yes,scrolling=yes,statusbar=no,navbar=no,width=550,height=550');return false;">More articles like this</a><script>
	if(show){
	   document.write('</td>');
	   document.write('</tr>');
	   document.write('<tr>');
	   document.write('<td height="5" align="center" colspan="2" class="blacksml2"><img alt="" height="5" width="1" src="/nsmb/images/spacer.gif"></td>');
	   document.write('</tr>');
	   document.write('<tr>');
	   document.write('<td align="center" colspan="2" class="blacksml2"><img alt="" height="1" width="130" src="/ng/images/spacer_grey.gif"></td>');
	   document.write('</tr>');

	}
   </script></td></tr><tr><td colspan="2" class="blacksml2" align="center" height="5"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_005.gif" height="5" width="1"></td></tr><tr><td colspan="2" class="blacksml2" align="center"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_grey.gif" height="1" width="130"></td></tr>
				
<form method="post" action="http://www.biocompare.com/nature/search.asp" name="search"></form>
</tbody></table><table bgcolor="#000000" border="0" cellpadding="1" cellspacing="0" width="100%">
<tbody><tr><td valign="top">
<table bgcolor="#ffffff" border="0" cellpadding="0" cellspacing="0" width="100%">
<tbody><tr><td colspan="3"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_004.gif" alt="" height="3" width="5"></td></tr>
<tr><td><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_004.gif" alt="" height="1" width="5"></td>
<td valign="top"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/logo.gif" alt="natureproducts" border="0" height="16" width="88"></td>
<td bgcolor="#ffffff"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_004.gif" alt="" height="1" width="5"></td></tr>
<tr><td colspan="3"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_004.gif" alt="" height="2" width="1"></td></tr>
<tr><td colspan="3" bgcolor="#000000" valign="top"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_004.gif" alt="" border="0" height="1" width="1"></td></tr>
<tr><td colspan="3" bgcolor="#f3f2f2"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_004.gif" alt="" height="9" width="5"></td></tr>
<tr><td bgcolor="#f3f2f2"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_004.gif" alt="" height="1" width="5"></td>
<td bgcolor="#f3f2f2" valign="top">
<p class="prdnorm">Search buyers guide:<br><br>
<input name="search" size="10" type="text">
<input src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/go.gif" type="image"></p></td><td bgcolor="#f3f2f2"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_004.gif" alt="" height="1" width="5"></td></tr>
<tr><td colspan="3" bgcolor="#f3f2f2"><img src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_004.gif" alt="" height="9" width="5"></td></tr>

</tbody></table>
</td></tr></tbody></table>




<table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td align="center" height="30">&nbsp;</td></tr><tr><td align="center"><span class="greysml">ADVERTISEMENT</span></td></tr><tr><td><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_003.gif" height="3" width="1"></td></tr><tr><td align="center"><script type="text/javascript" language="javascript">
<!--
document.write ('<ILAYER HEIGHT="600" WIDTH="120" VISIBILITY="hidden" ID="layer2"></ILAYER>');
document.write ('<NOLAYER>');
document.write ('<IFRAME scrolling="no" frameborder="0" vspace="0" hspace="0" marginheight="0" marginwidth="0" height="600" width="120" SRC="http://ad.doubleclick.net/adi/genetics.nature.com/curr_back;pos=right;issue=$!issue;' + adinf + ';sz=120x600;ord=' + ordval + '?">');
document.write ('<A HREF="http://ad.doubleclick.net/jump/genetics.nature.com/curr_back;pos=right;issue=$!issue;abr=!ie4;abr=!ie5;' + adinf + 'sz=120x600;ord=' + ordval + '?"></A>');
document.write ('<IMG alt="advertisement" align="center" HSPACE="0" HEIGHT="600" WIDTH="120" BORDER="0" SRC="http://ad.doubleclick.net/ad/genetics.nature.com/curr_back;pos=right;issue=$!issue;abr=!ie4;abr=!ie5;' + adinf + 'sz=120x600;ord=' + ordval + '?"></IFRAME>');
document.write ('</NOLAYER>');
// --></script><ilayer height="600" width="120" visibility="hidden" id="layer2"></ilayer><nolayer><iframe vspace="0" hspace="0" marginheight="0" marginwidth="0" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/curr_back.htm" frameborder="0" height="600" scrolling="no" width="120">&lt;A
HREF="http://ad.doubleclick.net/jump/genetics.nature.com/curr_back;pos=right;issue=$!issue;abr=!ie4;abr=!ie5;lg=no;sz=120x600;ord=1239938424990?"&gt;&lt;/A&gt;&lt;IMG
alt="advertisement" align="center" HSPACE="0" HEIGHT="600" WIDTH="120"
BORDER="0"
SRC="http://ad.doubleclick.net/ad/genetics.nature.com/curr_back;pos=right;issue=$!issue;abr=!ie4;abr=!ie5;lg=no;sz=120x600;ord=1239938424990?"&gt;</iframe></nolayer><noscript><a
href="http://ad.doubleclick.net/jump/genetics.nature.com/curr_back;pos=right;issue=$!issue;abr=!NN2;sz=120x600;ord=07893195?"><img
alt="advertisement" height="600" width="120" border="0"
src="http://ad.doubleclick.net/ad/genetics.nature.com/curr_back;pos=right;abr=!NN2;sz=120x600;ord=07893195?"></a></noscript></td></tr></tbody></table>
</td></tr></tbody></table></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td align="center">&nbsp;</td></tr>
<tr><td align="center" bgcolor="#4d876e" height="4"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="4" width="1"></td></tr><tr><td align="center"><table border="0" cellpadding="0" cellspacing="0" width="760"><tbody><tr><td colspan="2" valign="top"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="10" width="1"></td></tr><tr><td valign="top" width="145"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/"><img alt="Nature Genetics" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/ng_footer_logo.gif" border="0" height="38" width="84"></a><br><span class="blacksmltext">
ISSN:&nbsp;1061-4036<br>
EISSN:&nbsp;1546-1718</span></td><td class="blacknorm" valign="top" width="615">
<a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/index.html">Journal home</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/vaop/ncurrent/index.html">Advance online publication</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v39/n2/index.html">
Current issue
</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/archive/index.html">Archive</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/press_release/index.html">Press releases</a> | 
<a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/supplements/index.html">Supplements</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/webfocus/index.html">Focuses</a> | 
<a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/authors/index.html">For authors</a> | <a class="footer" href="http://mts-ng.nature.com.www.lib.ncsu.edu:2048/" target="_blank">Online submission</a> | <a class="footer" href="http://npg.nature.com.www.lib.ncsu.edu:2048/npg/servlet/Content?data=xml/01_reprints.xml&amp;style=xml/01_reprints.xsl" target="_blank">Permissions</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/referees/index.html">For referees</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/journal/v37/n1/index.html">Free online issue</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/about/index.html">About the journal</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/contact/index.html">Contact the journal</a> | <a class="footer" href="https://secure-nature-com.www.lib.ncsu.edu:2433/subscribe/ng" target="_blank">Subscribe</a> | <a class="footer" href="http://advertising.nature.com.www.lib.ncsu.edu:2048/" target="_blank">Advertising</a> | <a class="footer" href="http://npg.nature.com.www.lib.ncsu.edu:2048/npg/forms/03_current.jsp">work@npg</a> | <a class="footer" href="http://www.naturereprints.com/" target="_blank">naturereprints</a> | <a class="footer" href="http://www.nature.com.www.lib.ncsu.edu:2048/ng/site/index.html">About this site</a> | <a class="footer" href="http://npg.nature.com.www.lib.ncsu.edu:2048/npg/servlet/Content?data=xml/07_home.xml&amp;style=xml/07_home.xsl" target="_blank">For librarians</a></td></tr><tr><td colspan="2" valign="top"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="10" width="1"></td></tr></tbody></table></td></tr><tr><td align="center" bgcolor="#4d876e" height="2"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer_002.gif" height="2" width="1"></td></tr><tr><td align="center"><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="760"><tbody><tr><td colspan="3" valign="top"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="10" width="1"></td></tr><tr><td align="right" valign="top" width="130"><a href="http://npg.nature.com.www.lib.ncsu.edu:2048/npg/servlet/Content?data=xml/02_welcome.xml&amp;style=xml/02_welcome.xsl"><img alt="Nature Publishing Group, publisher of Nature, and other science journals and reference works" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/npg_footer_logo.gif" border="0" height="26" width="26"></a></td><td align="right" valign="top" width="15"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="1" width="15"></td><td class="blacksmltext" valign="middle" width="615"><a href="http://www.nature.com.www.lib.ncsu.edu:2048/copyright/copyright_ng.html" onclick="window.open('/copyright/copyright_ng.html','copyright_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=no,resizable=no,copyhistory=yes,width=450,height=280,top=380,left=20');return false;" class="copyright">©2003&nbsp;Nature Publishing Group</a> | <a target="_blank" class="privacy" href="http://npg.nature.com.www.lib.ncsu.edu:2048/npg/servlet/Content?data=xml/08_privacy.xml&amp;style=xml/08_privacy.xsl">Privacy policy</a></td></tr><tr><td colspan="3" valign="top"><img alt="" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/spacer.gif" height="10" width="1"></td></tr></tbody></table></td></tr></tbody></table>
<!-- super body bottom -->
<!-- super javascript -->
															
																			
						<script type="text/javascript" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/jquery.js"></script>
								<script type="text/javascript" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/runOnLoad.js"></script>
								<script type="text/javascript" src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/group.js"></script>
			

<script type="text/javascript"><!--//--><![CDATA[//><!--
(function($) {
	$('a.popup-this').bind('click', function() {
		window.open(this.href);
		return false;
	});
})(jQuery);
//--><!]]></script>
<!--[if IE 6]>
<![endif]--><!-- webtrends Version: 8.6.0 -->
	 <script src="Aebersold%202003%20Proteomics%20-%20the%20first%20decade%20and%20beyond_files/webtrends.js" type="text/javascript"></script>
	<script type="text/javascript"><!--//--><![CDATA[//><!--
		var _tag=new WebTrends();
		_tag.dcsGetId();
	//--><!]]></script>
	<script type="text/javascript"><!--//--><![CDATA[//><!--
		// Add custom parameters here.
		// _tag.DCSext.param_name=param_value;
		_tag.dcsCollect();
	//--><!]]></script>
	<noscript>
<div><img alt="" id="DCSIMG" width="1" height="1"
src="http://statse.webtrendslive.com/dcs0zztfg00000s969s37qoal_2f6z/njs.gif?dcsuri=/nojavascript&amp;WT.js=No&amp;WT.tv=8.6.0"
/></div> </noscript>
	<script type="text/javascript"><!--//--><![CDATA[//><!--
	(function($) {
		$(function() {
			var getMeta = function(name) {
				return $('meta[name="' + name + '"]').attr('content');
			};
			var applyParams = function(wt) {
				var params = [];
				for (var prop in wt) {
					params.push(prop);
					params.push(wt[prop]);
				}
				_tag.dcsMultiTrack.apply(_tag, params);
			};

			$('a[href$=".pdf"], a[href*=".pdf?"]').bind('click', function() {
				var wt = {};
				wt['DCS.dcssip'] = this.hostname ? this.hostname.split(':')[0] : '';
				wt['DCS.dcsuri'] = this.pathname ? this.pathname.indexOf('/') !== 0 ? '/' + this.pathname : this.pathname : '/';
				wt['DCS.dcsqry'] = this.search ? this.search.substring(1) : '';
				wt['WT.cg_n'] = 'Nature Genetics';
				wt['WT.cg_s'] = '';
				wt['WT.art_journal_title'] = getMeta('citation_journal_title');
				wt['WT.art_volume'] = getMeta('citation_volume');
				wt['WT.art_issue'] = getMeta('citation_issue');
				wt['WT.art_title'] = getMeta('citation_title');
				wt['WT.art_doi'] = getMeta('citation_doi');
				wt['WT.art_authors'] = getMeta('citation_authors').replace(/;/g, '%3B').replace(/\s*,\s*/g, ';');
				wt['WT.art_date'] = getMeta('citation_date');
				applyParams(wt);
			});
			
			$('a[href*="/rss"]').each(function() {
				if (this.href.match(/\/rss\/?(\?.*)?$/)) {
					$(this).bind('click', function() {
						var wt = {};
						wt['DCS.dcsuri'] = this.pathname ? this.pathname.indexOf('/') !== 0 ? '/' + this.pathname : this.pathname : '/';
						wt['DCS.dcsqry'] = this.search ? this.search.substring(1) : '';
						wt['WT.ti'] = 'Download:' + (this.hostname ? this.hostname.split(':')[0] : '') + wt['DCS.dcsuri'] + (wt['DCS.dcsqry'] ? '?' + wt['DCS.dcsqry'] : '');
						wt['WT.dl'] = '20';
						applyParams(wt);
					});
				}
			});

			$('div[class*="adunit-"] a').bind('click', function() {
				var classes = $(this).parents('div[class*="adunit-"]:first').attr('class');
				if (classes) {
					var match = /(^|\s)adunit-([^\s]+)/.exec(classes);
					_tag.dcsMultiTrack('WT.ac', (match) ? match[2] : '');
				}
			});
		});	
	})(jQuery);
	//--><!]]></script>
	<!-- end webtrends Version: 8.6.0 -->
</div></div></body></html>