<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="BMC Bioinformatics"><meta name="citation_title" content="The Firegoose: two-way integration of diverse data from different bioinformatics web resources with desktop applications"><meta name="citation_authors" content="J Christopher Bare, Paul T Shannon, Amy K Schmid, and  Nitin S Baliga"><meta name="citation_date" content="2007"><meta name="citation_volume" content="8"><meta name="citation_firstpage" content="456"><meta name="citation_doi" content="10.1186/1471-2105-8-456"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2211326&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326"><meta name="citation_pmid" content="18021453"><meta name="DC.Title" content="The Firegoose: two-way integration of diverse data from different bioinformatics web resources with desktop applications"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="BioMed Central"><meta name="DC.Contributor" content="J Christopher Bare"><meta name="DC.Contributor" content="Paul T Shannon"><meta name="DC.Contributor" content="Amy K Schmid"><meta name="DC.Contributor" content="Nitin S Baliga"><meta name="DC.Date" content="2007"><meta name="DC.Identifier" content="10.1186/1471-2105-8-456"><meta name="DC.Language" content="en"><link rel="stylesheet" href="Bare%202007%20Firegoose_files/pmcstatic.css" type="text/css"><link rel="stylesheet" href="Bare%202007%20Firegoose_files/pmcbase1.css" type="text/css"><link rel="stylesheet" href="Bare%202007%20Firegoose_files/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="Bare%202007%20Firegoose_files/pmcbody4.css" type="text/css"><link rel="stylesheet" href="Bare%202007%20Firegoose_files/pmcrefs3.css" type="text/css"><script type="text/javascript" src="Bare%202007%20Firegoose_files/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="Bare%202007%20Firegoose_files/tileshop_pmc.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="Bare%202007%20Firegoose_files/utils.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="Bare%202007%20Firegoose_files/tileshop_pmc.js"></script><script src="Bare%202007%20Firegoose_files/firebugx.js" type="text/javascript"></script><script src="Bare%202007%20Firegoose_files/tile.js" type="text/javascript"></script><script src="Bare%202007%20Firegoose_files/notify.js" type="text/javascript"></script><script src="Bare%202007%20Firegoose_files/drag_n_drop.js" type="text/javascript"></script><script src="Bare%202007%20Firegoose_files/scale_pmc.js" type="text/javascript"></script><script type="text/javascript" src="Bare%202007%20Firegoose_files/tileshop_data_db.js"></script><script src="Bare%202007%20Firegoose_files/remote_data_provider.js" type="text/javascript"></script><title>The Firegoose: two-way integration of diverse data from different bioinformatics web resources with desktop applications</title></head><div FirebugVersion="1.3.3" style="display: none;" id="_firebugConsole"></div><body class="pmc-body"><div style="height: 0px;"><a id="top" name="top"></a></div><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr valign="top"><td colspan="3"><table style="border-collapse: collapse;" border="0" cellpadding="0" cellspacing="0"><tbody><tr><td valign="top" width="145"><img style="border-style: none;" usemap="#pmclogo" alt="pmc logo image" src="Bare%202007%20Firegoose_files/pmc3_logo_v5.gif"><map name="pmclogo"><area shape="rect" coords="65,52,137,67" href="http://www.pubmedcentral.nih.gov/fprender.fcgi" alt="Journal List"><area shape="rect" coords="7,52,52,67" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Jqw3ls5r-jdIpUcxYoEhX8qYDv49uNW1d6Mk4k-T&amp;reftype=other&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?db=pmc" alt="Search"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="http://www.pubmedcentral.nih.gov/"></map></td><td style="padding-left: 4px;" height="75" valign="top"><div><img style="border-style: none;" src="Bare%202007%20Firegoose_files/logo-bmcbioi.gif" alt="Logo of bmcbioi" usemap="#logo-imagemap" align="top"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="BioMed Central web site" coords="0,0,99,74" onclick="focuswin('pmc_ext')" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0JuXvneyzYMRM9_iJHkT_Zn7lFWhk5PsaJS2S_QUD&amp;reftype=publisher&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com" target="pmc_ext"><area shape="rect" coords="100,54,172,74" alt="Reference to the article." onclick="focuswin('pmc_ext') " href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0hE6Sm0VCr7jJO4xnvckwjteEOkyHZlGvjHIttQrO&amp;reftype=publisher&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7CArticle%7CFullText&amp;rendering-type=normal&amp;&amp;http://dx.doi.org/10.1186/1471-2105-8-456" target="pmc_ext"><area shape="rect" coords="173,54,223,74" alt="Search." onclick="focuswin('pmc_ext')" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ZM02mhMGy119Tlxu8pBHhaP_BP2utQc4tx_BD5zH&amp;reftype=publisher&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/search/bmcbioinformatics" target="pmc_ext"><area shape="rect" coords="224,55,342,74" alt="Manuscript submission." onclick="focuswin('pmc_ext')" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0xwn1Yja8YEzNn8AIZ3J4Ct4G3HGJpl8xyWEkbkid&amp;reftype=publisher&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/manuscript/bmcbioinformatics" target="pmc_ext"><area shape="rect" coords="343,53,480,74" alt="Registration." onclick="focuswin('pmc_ext')" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0x1PjkaKd0YaXOLa9EM03v9ACiFx9XVSSiBzC3kEf&amp;reftype=publisher&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/registration" target="pmc_ext"><area shape="rect" coords="0,0,480,74" alt="Journal front page." onclick="focuswin('pmc_ext') " href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0nS9sK51NVhXWeP8dXNnNJ31HfLIRLbXEPY0IQ5ib&amp;reftype=publisher&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/bmcbioinformatics" target="pmc_ext"></map></div></td></tr></tbody></table></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="3" width="100%"><tbody><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="http://www.pubmedcentral.nih.gov/fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="http://www.pubmedcentral.nih.gov/tocrender.fcgi?journal=13&amp;action=archive">BMC Bioinformatics</a><span> &gt; </span><a class="navlink" href="http://www.pubmedcentral.nih.gov/tocrender.fcgi?iid=140969">v.8; 2007</a></div></td></tr><tr><td class="sidebar-cell" valign="top" width="145"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img src="Bare%202007%20Firegoose_files/square.gif" alt="&gt;" style="border-style: none;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="#supplementary-material-sec" class="sidefm-supplink">Supplementary Material</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2211326&amp;blobtype=pdf" class="sidefm-pmclink">PDF (718K)</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/tocrender.fcgi?iid=140969" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/tocrender.fcgi?journal=13&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0ts9XkvZcLOQwOeWLkX3Z-Owizn-F4IVqKb8LoQMj&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18021453">PubMed record</option><option selected="selected" value="redirect3.cgi?&amp;&amp;auth=0daxNuPeoZdB74ZAheVRKS4p9QOzN_AfNT-4KKAqC&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=18021453">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0JkcoIOBm8rVxlPIfduxRmjnQ32zJExNhoMdxMQXu&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=18021453">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0bQlhUhPj-sh26ThNR5mvVa0zMLcbUnQK5jvj0QYm&amp;reftype=Protein&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Protein&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=link&amp;linkname=pmc_protein&amp;from_uid=2211326">Protein</option><option value="redirect3.cgi?&amp;&amp;auth=04kFMvqQ4vKdQjfFTe-dbIK9Kwfdtyep9DBUWEQqA&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=link&amp;linkname=pmc_taxonomy&amp;from_uid=2211326">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0tWMlO03Z2n-Gmj7DsO9oivFKrZiQnvRDN1l5SQJE&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=2211326">Taxonomy tree</option></select><br><input class="sidefm-pmart" alt="GO" src="Bare%202007%20Firegoose_files/go.gif" type="image"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0FIqM1XqlFwUM04gfOW0zMiucIaut2v45SFpwxQx6&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Bare+JC%5Bauth%5D" class="sidefm-pmclink"> Bare, J.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0-9Dr936sQH_spWX6gBL0xvHTSBNE7PZoDKxvRAdj&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Shannon+PT%5Bauth%5D" class="sidefm-pmclink"> Shannon, P.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=00wmdFpvrJw-hzcxErGMbUcfZkhWSH3fea04vwAsJ&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Schmid+AK%5Bauth%5D" class="sidefm-pmclink"> Schmid, A.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0t1Fui1sGzG9mDuIi6Up1vRglt-pTDoHb7DyqCkvP&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Baliga+NS%5Bauth%5D" class="sidefm-pmclink"> Baliga, N.</a></div></div></div></td><td rowspan="107" background="Bare%202007%20Firegoose_files/wm-bmcbioi.gif" height="300" width="20"><div style="width: 20px;"></div></td><td class="content-cell" valign="top"><div class="front-matter-section"><table cellpadding="0" cellspacing="0" width="100%"><tbody><tr style="vertical-align: top;"><td><div class="fm-citation"><div><span class="citation-abbreviation">BMC Bioinformatics. </span><span class="citation-publication-date">2007; </span><span class="citation-volume">8</span><span class="citation-issue"></span><span class="citation-flpages">: 456. </span></div><div><span class="fm-vol-iss-date">Published online 2007 November 19. </span><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1186/1471-2105-8-456.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC2211326</div></td></tr></tbody></table><div class="fm-copyright"><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/about/copyright.html">Copyright</a>  © 2007 Bare et al; licensee BioMed Central Ltd.</div><div class="fm-title">The Firegoose: two-way integration of diverse data from different bioinformatics web resources with desktop applications</div><div class="fm-author contrib-group">Reviewed by J Christopher Bare,<sup>1</sup> Paul T Shannon,<sup>1</sup> Amy K Schmid,<sup>1</sup> and  Nitin S Baliga<sup><img src="Bare%202007%20Firegoose_files/corrauth.gif" alt="corresponding author"></sup><sup>1</sup><div class="fm-affl"><sup>1</sup>Institute for Systems Biology, 1441 N 34th Street, Seattle, WA 98103, USA</div></div><div class="fm-footnote"><sup><img src="Bare%202007%20Firegoose_files/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div class="fm-footnote"><span style="white-space: nowrap;" id="id551815">J Christopher Bare: <span class="e_id551815"><a class="ext-reflink" href="mailto:cbare@systemsbiology.org">cbare@systemsbiology.org</a></span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id551815", '<a class="ext-reflink" href="' + reverseAndReplaceString('gro.ygoloibsmetsys/ta/erabc:otliam', '/at/', '@') + '">' + reverseAndReplaceString('gro.ygoloibsmetsys/ta/erabc', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id594576">Paul T Shannon: <span class="e_id594576"><a class="ext-reflink" href="mailto:pshannon@systemsbiology.org">pshannon@systemsbiology.org</a></span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id594576", '<a class="ext-reflink" href="' + reverseAndReplaceString('gro.ygoloibsmetsys/ta/nonnahsp:otliam', '/at/', '@') + '">' + reverseAndReplaceString('gro.ygoloibsmetsys/ta/nonnahsp', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id362879">Amy K Schmid: <span class="e_id362879"><a class="ext-reflink" href="mailto:aschmid@systemsbiology.org">aschmid@systemsbiology.org</a></span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id362879", '<a class="ext-reflink" href="' + reverseAndReplaceString('gro.ygoloibsmetsys/ta/dimhcsa:otliam', '/at/', '@') + '">' + reverseAndReplaceString('gro.ygoloibsmetsys/ta/dimhcsa', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id532251">Nitin S Baliga: <span class="e_id532251"><a class="ext-reflink" href="mailto:nbaliga@systemsbiology.org">nbaliga@systemsbiology.org</a></span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id532251", '<a class="ext-reflink" href="' + reverseAndReplaceString('gro.ygoloibsmetsys/ta/agilabn:otliam', '/at/', '@') + '">' + reverseAndReplaceString('gro.ygoloibsmetsys/ta/agilabn', '/at/','@') + '</a>')}catch(e){}
                                //--></script></span> </div><div class="fm-pubdate">Received August 4, 2007; Accepted November 19, 2007.</div><div class="fm-copyright"><div class="p" id="id532338">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=06_oSTVIsaaGy-SlKSjTeHOs39uS8VNBBU9TCEA6T&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://creativecommons.org/licenses/by/2.0" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://creativecommons.org/licenses/by/2.0</a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><div class="links-box"><div class="fm-footnote"><img src="Bare%202007%20Firegoose_files/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="http://www.pubmedcentral.nih.gov/tocrender.fcgi?artid=2211326&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Bare%202007%20Firegoose_files/square.gif" alt="&gt;" style="border-style: none;"></span>Abstract</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532409" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id544990" class="sidefm-pmclink" style="text-transform: none;">Implementation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545440" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545999" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546026" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546099" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546119" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546178" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id532350">Abstract</div><div class="section-content"><!--article-meta--><div class="sec sec-first"><div class="abs-head2 head-separate">Background</div><div class="p p-first-last" id="id532356">Information
resources on the World Wide Web play an indispensable role in modern
biology. But integrating data from multiple sources is often encumbered
by the need to reformat data files, convert between naming systems, or
perform ongoing maintenance of local copies of public databases.
Opportunities for new ways of combining and re-using data are arising
as a result of the increasing use of web protocols to transmit
structured data.</div></div><div class="sec"><div class="abs-head2 head-separate">Results</div><div class="p p-first" id="id532380">The
Firegoose, an extension to the Mozilla Firefox web browser, enables
data transfer between web sites and desktop tools. As a component of
the Gaggle integration framework, Firegoose can also exchange data with
Cytoscape, the R statistical package, Multiexperiment Viewer (MeV), and
several other popular desktop software tools. Firegoose adds the
capability to easily use local data to query KEGG, EMBL STRING, DAVID,
and other widely-used bioinformatics web sites. Query results from
these web sites can be transferred to desktop tools for further
analysis with a few clicks.</div><div class="p" id="id532387">Firegoose
acquires data from the web by screen scraping, microformats, embedded
XML, or web services. We define a microformat, which allows structured
information compatible with the Gaggle to be embedded in HTML documents.</div><div class="p p-last" id="id532391">We demonstrate the capabilities of this software by performing an analysis of the genes activated in the microbe <em>Halobacterium salinarum NRC-1 </em>in
response to anaerobic environments. Starting with microarray data, we
explore functions of differentially expressed genes by combining data
from several public web resources and construct an integrated view of
the cellular processes involved.</div></div><div class="sec sec-last"><div class="abs-head2 head-separate">Conclusion</div><div class="p p-first-last" id="id532405">The
Firegoose incorporates Mozilla Firefox into the Gaggle environment and
enables interactive sharing of data between diverse web resources and
desktop software tools without maintaining local copies. Additional web
sites can be incorporated easily into the framework using the scripting
platform of the Firefox browser. Performing data integration in the
browser allows the excellent search and navigation capabilities of the
browser to be used in combination with powerful desktop tools.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532350" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Bare%202007%20Firegoose_files/square.gif" alt="&gt;" style="border-style: none;"></span>Background</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id544990" class="sidefm-pmclink" style="text-transform: none;">Implementation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545440" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545999" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546026" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546099" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546119" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546178" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id532409">Background</div><div class="section-content"><div class="p p-first" id="id544799" cite="cite-id544799" pmids="12728276 14668224 12511062 16569235">Information
resources on the World Wide Web play an indispensable role in modern
biology. Using a web browser, the biologist can easily access a wealth
of information such as sequences, biochemical pathways, protein
interactions, functional domains, annotations, and gene expression
data. Yet, integrating data from diverse sources remains challenging.
Biologists wishing to analyze their own experimental data in
combination with publicly available data face the cumbersome tasks of
converting file formats, reconciling incompatible schemas, and mapping
between inconsistent naming systems. Several strategies for data
integration have been developed in the context of biology [<a href="#B1" rid="B1" class="cite-reflink bibr popnode">1</a>].
Data warehousing and web services are two general approaches. Semantic
mapping between different data sources has been accomplished by
describing biological entities in terms of shared object models [<a href="#B2" rid="B2" class="cite-reflink bibr popnode">2</a>] or curated ontologies [<a href="#B3" rid="B3" class="cite-reflink bibr popnode">3</a>].
Our team's previous work in this area resulted in the Gaggle, a
software environment that integrates databases and software tools based
on a few simple data types and the principle of semantic flexibility [<a href="#B4" rid="B4" class="cite-reflink bibr popnode">4</a>].
Previously, the Gaggle supported integration with web resources through
a special purpose browser of limited capabilities. An improved solution
was needed due to the ubiquitous role of the browser and the increasing
relevance of the web to data integration.</div><div class="p" id="id544855">The
web is rapidly becoming a channel for structured data as evidenced by
the increasing adoption of web services. Rich internet applications,
microformats [<a href="#B5" rid="B5" class="cite-reflink bibr popnode">5</a>], and the Semantic Web [<a href="#B6" rid="B6" class="cite-reflink bibr popnode">6</a>] are pushing the browser into a central role as an information broker [<a href="#B7" rid="B7" class="cite-reflink bibr popnode">7</a>]
working with data in human and machine readable form side-by-side.
Several tools outside the domain of biology have augmented the browser
with these types of capabilities, including Greasemonkey [<a href="#B8" rid="B8" class="cite-reflink bibr popnode">8</a>], Piggy Bank [<a href="#B9" rid="B9" class="cite-reflink bibr popnode">9</a>], and Operator [<a href="#B10" rid="B10" class="cite-reflink bibr popnode">10</a>].
These technologies offer the ability to perform ad-hoc manipulation of
data from multiple sources on the web using familiar browser based
interfaces.</div><div class="p p-last" id="id544927" cite="cite-id544927" pmids="12539951 17098935 15608232 12734009">We
developed the Firegoose to bring a full-featured browser into the
Gaggle environment and to allow easy transfer of data between the web
and the desktop. The Firegoose is a toolbar for the Mozilla Firefox [<a href="#B11" rid="B11" class="cite-reflink bibr popnode">11</a>]
browser that makes use of a diverse array of web communication
protocols to seamlessly query and retrieve data from custom databases
as well as popular bioinformatics resources including KEGG [<a href="#B12" rid="B12" class="cite-reflink bibr popnode">12</a>], EMBL STRING [<a href="#B13" rid="B13" class="cite-reflink bibr popnode">13</a>,<a href="#B14" rid="B14" class="cite-reflink bibr popnode">14</a>], and DAVID [<a href="#B15" rid="B15" class="cite-reflink bibr popnode">15</a>].
As a member of the Gaggle software environment, the Firegoose can also
exchange data with a growing collection of desktop tools. Bringing the
data integration techniques of the Gaggle into the browser, the
Firegoose facilitates exploratory analysis of systems biology data
using web resources and desktop software tools.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532350" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532409" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Bare%202007%20Firegoose_files/square.gif" alt="&gt;" style="border-style: none;"></span>Implementation</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545440" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545999" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546026" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546099" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546119" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546178" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id544990">Implementation</div><div class="section-content"><div class="p p-first-last" id="id544994">The
Firegoose is implemented as an add-on extension to Mozilla Firefox, a
popular opensource web browser. Its user interface is a toolbar defined
in XUL, an XML dialect for describing user interface components, and
its actions are scripted in Javascript. The Firegoose communicates with
the Gaggle via the Java RMI protocol and supports several modes of
information exchange over the web.</div><div class="sec"><div class="head2 head-separate">The Gaggle approach</div><div class="p p-first" id="id545003">While
it can be useful outside of the Gaggle, the Firegoose is a component of
the Gaggle software environment. The goal of the Gaggle is to enable
interactive exploration of interrelated biological data by making data
transfer between analysis tools, databases, and now web resources,
quick and easy. Integration is achieved by supporting the interchange
of five simple data types, which cover a wide variety of use-cases
within the domain of systems biology.</div><div class="p" id="id545010"><strong>Name list</strong>: a list of identifiers.</div><div class="p" id="id545015"><strong>Associative array</strong>: a set of key/value pairs.</div><div class="p" id="id545021"><strong>Data matrix</strong>: a 2 dimensional grid of floating point numbers, with labels for each row and column.</div><div class="p" id="id545026"><strong>Network</strong>: a graph in which attributes may be assigned to nodes and edges.</div><div class="p" id="id545032"><strong>Cluster</strong>: a set of identifiers of interest under a set of conditions.</div><div class="p" id="id545037">The central feature of the Gaggle is the <em>broadcast</em>.
Applications become part of the Gaggle by implementing an interface
that allows them to accept broadcasts. A broadcast consists of sending
a message holding one of the five data types to a program called the
Gaggle Boss, which then relays the message to one or more receiving
applications or targets.</div><div class="p" id="id545047">Using the
simple Gaggle data types as an intermediary has the advantage that
applications can share data without sharing common object models. Each
program is free to handle broadcasts in a way that makes sense in its
own context. This is <em>semantic flexibility</em>, the first of the
Gaggle's guiding principles. The second is to keep the set of data
types as small as possible minimizing the programming effort required
to integrate a new application into the Gaggle environment.</div><div class="p" id="id545057" cite="cite-id545057" pmids="14597658 12613259">The Gaggle environment currently includes Cytoscape [<a href="#B16" rid="B16" class="cite-reflink bibr popnode">16</a>] for viewing networks, the R statistical package [<a href="#B17" rid="B17" class="cite-reflink bibr popnode">17</a>], Multi-experiment Viewer (MeV) [<a href="#B18" rid="B18" class="cite-reflink bibr popnode">18</a>]
for microarray analysis, the spreadsheet-like Data Matrix Viewer (DMV),
and several other programs. The term "gaggled" is used to indicate that
a software application or database has been configured to exchange
messages with the Boss; for example, Cytoscape has been gaggled through
the Cygoose plug-in.</div><div class="p p-last" id="id545095">The
Firegoose uses the power of the browser to extend the reach of the
Gaggle to target web resources based on widely supported protocols. The
Firegoose communicates with the Gaggle Boss via Java RMI. To
communicate with web resources, the Firegoose supports standard web
protocols and data encoding schemes such as HTTP, HTML, XML, and SOAP.
A schematic view (Fig. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>)
shows the flow of messages over various protocols between desktop
applications, the Gaggle Boss, the Firegoose, and several websites.</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F1" name="F1"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Bare%202007%20Firegoose_files/picrender_004.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span><strong>Communication in the Gaggle</strong>.
Software and databases shown as red dots send and receive broadcasts
via Java RMI. The blue nodes are web resources connected to the Gaggle
through the Firegoose and accessed using HTTP with other protocols and
formats such</span><a class="side-caption" style="font-size: 100%;" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></tbody></table></div></div><div style="clear: both;"></div></div><div class="sec"><div class="head2 head-separate">Exchanging data with web resources</div><div class="p p-first" id="id545140">When
data is broadcast to the Firegoose, it can be rebroadcast, under user
control, to any supported website. A broadcast to a web resource is
typically used to build a query against an underlying database. For
example, a set of nodes representing genes may be selected in a
Cytoscape network. Broadcasting this data from Cytoscape to the
Firegoose causes a list of gene identifiers to appear in the
Firegoose's broadcast data menu. To broadcast to the KEGG Pathway
database, the user would select the gene list and then select the
target "KEGG Pathway". Clicking "Broadcast" submits a request to KEGG
for the pathways in which those genes participate (Fig. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>).</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F2" name="F2"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Bare%202007%20Firegoose_files/picrender.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span><strong>The Firegoose toolbar for Mozilla Firefox</strong>.
A broadcast is sent by three steps: select data to be broadcast (i),
select the target of the broadcast (ii), and click 'Broadcast' to send
(iii). In this example, a list of genes has been broadcast to KEGG.</span><a class="side-caption" style="font-size: 100%;" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p p-last" id="id545184">More
importantly, the Firegoose also brings data back from the web in a
usable form. Traditionally, a web server responds to a request by
sending an HTML document, which the browser renders for display. The
HTML format in which web pages are encoded describes presentation but
is poorly suited for representing the underlying structure, or
metadata, necessary for integration and re-use by automated processes.
In response to this shortcoming, a variety of encodings have been
developed including XML, SOAP, and microformats. Firegoose supports
several means of data interchange, described in detail below, in order
to interoperate with a wide variety of data providers.</div><div class="sec"><div class="head3 head-separate">Screen Scraping</div><div class="p p-first" id="id545196">HTML
does not represent structured data well. However, by making some
assumptions about how the data are formatted, it is possible to
reconstruct the structure that is lost in HTML. This process, known as
screen scraping, has a long history on the web.</div><div class="p" id="id545200">The
Firegoose scrapes pages generated by the KEGG Pathways database to
acquire a list of genes present in a given pathway. The pathway
diagrams generated by KEGG contain links from each gene to its sequence
and annotations. The links take the following form: <em>href="/dbget-bin/www_bget?eco+b0728+b0729"</em>.</div><div class="p" id="id545209">This allows a simple script to glean the information that the two <em>Escherichia coli </em>genes
b0728 and b0729 play a role in the current pathway, in this case the
citrate cycle. By scanning all such links, a list can be compiled of
all genes that are members of the pathway in a selected organism, which
fits nicely into the name-list data type of the Gaggle. A biologist
may, for example, broadcast this list to a visualization tool for
microarray data and check for evidence of differential expression under
specific conditions of genes whose products play a role in the same
metabolic pathway.</div><div class="p p-last" id="id545220">While it
can be effective, screen scraping is inelegant and prone to breakage.
It requires code to be written that is specific for an individual site
and will likely require maintenance whenever the site makes formatting
changes. For the Firegoose, screen scraping is one means of acquiring
data, but is not the preferred option. One simple solution to the lack
of structure inherent in web pages is found in microformats.</div></div><div class="sec"><div class="head3 head-separate">Microformats and Embedded XML</div><div class="p p-first" id="id545230">Microformats
embed machine-readable data within valid HTML. The structure missing in
HTML is supplied in CSS class attributes, giving the parser the
necessary clues to extract data systematically from the page. The same
data elements may do double duty accommodating both on-screen display
and machine-readability. A browser extension like the Firegoose is
ideally positioned to augment web browsing with new capabilities
enabled by data embedded in the web page.</div><div class="p" id="id545237">We
have defined a microformat to represent the Gaggle data types,
specifications for which can be found in the Gaggle microformat
reference [<a href="#B19" rid="B19" class="cite-reflink bibr popnode">19</a>].
This format allows any of the Gaggle data types to be embedded directly
into a web page, either displayed in the page or hidden from the user.
The toolbar detects embedded data and makes that data available for
broadcast to other web sites or desktop applications. An example of the
Gaggle microformat is shown (Fig. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">3</a>) which encodes a list of three genes involved in signal transduction.</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F3" name="F3"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Bare%202007%20Firegoose_files/picrender_002.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 3" title="Figure 3"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 3</strong></a><div class="figure-table-caption-in-article"><span><strong>Example of the Gaggle Microformat</strong>. Microformats embed structured data in web pages by using CSS tags as markup.</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p" id="id545288">As
with microformats, XML can be embedded directly in web pages. XML has
the advantages of a well-defined syntax and good support by parsers and
tools. Adoption of this technique has been hindered by the fact that
web pages containing embedded XML fail HTML and XHTML validation. This
may not be an obstacle in practice since browsers do not perform
validation and simply ignore unrecognized tags. We implemented an XML
format for Gaggle name lists to communicate lists of genes from a web
view of a database of gene annotations for <em>H. salinarum NRC-1 </em>[<a href="#B20" rid="B20" class="cite-reflink bibr popnode">20</a>].</div><div class="p" id="id545310">The
benefits of these two approaches are the same. Their value is that they
allow data to be captured without requiring site-specific screen
scraping code to be written and maintained. By adding a small amount of
standard markup to their web pages, website providers make their site
accessible to any tools that recognize that format. In the case of the
Gaggle microformat, websites can be gaggled without additional code in
the Firegoose and a very modest effort on the server side.</div><div class="p p-last" id="id545316">Embedding
data directly in HTML works well for moderate amounts of data. Where
the size of the data is prohibitive, a microformat embedded in the page
could provide the parameters needed by tools like Firegoose to access
related data through external channels such as web services or direct
downloads.</div></div><div class="sec sec-last"><div class="head3 head-separate">Web Services</div><div class="p p-first" id="id545325">A
web service is a programmatic interface allowing applications running
on heterogeneous platforms to interoperate over a network. Typically,
the messages passed back and forth are XML. The Firegoose can access
web services from within the browser forging a direct link from the
presentation of a web site to its underlying data.</div><div class="p" id="id545331">For example, KEGG offers extensive web services. The list of genes in a pathway can be requested by calling the <em>get_genes_by_pathway </em>SOAP
method, which returns the same list of genes acquired above by screen
scraping, but in an easily processed XML format. A parameter specifies
the pathway and organism in which we are interested. For example, the
string "path:<span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0j5SDzLtaIQRmzj0xAHHVs5WASD9qzMU-C_-MIAor&amp;reftype=extlink-entrez-protein&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CProtein&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=Retrieve&amp;db=protein&amp;dopt=GenPept&amp;list_uids=140861114" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">eco00020</a></span>" denotes the tricarboxylic acid (TCA) cycle pathway in <em>E. coli</em>. In order to find these parameters, a nominal amount of screen scraping and case-specific scripting is still necessary.</div><div class="p" id="id545357">Accessing
web services from within the browser is not typical, but offers some
compelling advantages. The familiar user-friendly interface of a web
application can be used to navigate through a database and then the web
service can be invoked to acquire the desired records in structured
form amenable to further computation. The browser supports an
interactive style of usage without the need to build a customized
client to the web service.</div><div class="p p-last" id="id545363">Currently,
there is no standard way of linking data displayed in a web page with a
query to a web service that will return the same data in a structured
form. The Firegoose uses screen scraping to fill in this gap. A
standardized microformat for this purpose could facilitate this kind of
interactive access to structured data.</div></div></div><div class="sec sec-last"><div class="head2 head-separate">Website Handlers</div><div class="p p-first" id="id545373">A
website may provide data via a web service, an embedded microformat, or
other channels. If not, screen scraping may be required. For the sake
of modularity, we encapsulate the details of interacting with each
individual site or specific format into a separate component called a
"handler".</div><div class="p" id="id545378">For each target website or
format, a handler is written and packaged in a separate Javascript
source file that contains code specific to that target. The handler
implements a common interface (Fig. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">4</a>)
and is responsible for recognizing web pages with which it can interact
and transferring data to and from its target. Recognition can be based
either on the URL or the contents of the page, which means we can write
a handler for a particular web site or a handler that understands a
data format embedded in pages from several sources.</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F4" name="F4"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F4" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Bare%202007%20Firegoose_files/picrender_003.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 4" title="Figure 4"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 4</strong></a><div class="figure-table-caption-in-article"><span><strong>Interface of a handler script</strong>. The handler interface provides an extension point for adding support for new websites, web services, and protocols.</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p" id="id545417">When a document is loaded in the browser, the toolbar calls the <em>recognize </em>method
for each handler until a handler recognizes the document. Then the
getPageData method is called for that handler. The handler then
inspects the document and constructs a list of GaggleData objects that
can be acquired from the page. This, in turn, is used to populate the
broadcast menu.</div><div class="p" id="id545426">The GaggleData object
represents data from the website in the form of one of the Gaggle data
types. Support for lazy instantiation and asynchronous access to web
services can be neatly hidden behind its simple interface. Typically,
getPageData performs only enough work to generate descriptive
information for each data object. The actual parsing or issuing a
request to a web service can be deferred until the user requests a
broadcast. This minimizes processing overhead and prevents unnecessary
network traffic.</div><div class="p p-last" id="id545433">Any of the
Gaggle data types may be returned from the getPageData method, but so
far we have only implemented broadcasting lists of names to web
resources. Typically, a broadcast to a website is transformed into a
query, which retrieves information relevant to a list of genes,
proteins, or other identifiers. In principle, methods could easily be
defined for each of the other data types: handleNetwork, handleMatrix,
etc. In keeping with the nature of dynamic languages like Javascript,
all of the "handle..." methods are optional.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532350" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532409" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id544990" class="sidefm-pmclink" style="text-transform: none;">Implementation</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Bare%202007%20Firegoose_files/square.gif" alt="&gt;" style="border-style: none;"></span>Results and discussion</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545999" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546026" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546099" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546119" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546178" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id545440">Results and discussion</div><div class="section-content"><div class="sec sec-first"><div class="head2 head-separate">Case Study: synthesis of a model of anaerobic physiology in H. salinarum NRC-1 using Firegoose</div><div class="p p-first" id="id545449" cite="cite-id545449" pmids="17785531 11016950">To demonstrate the effectiveness of the Firegoose in a typical systems biology type of investigation, we explore expression in <em>Halobacterium salinarum NRC-1 </em>in response to fluctuating oxygen concentration [<a href="#B21" rid="B21" class="cite-reflink bibr popnode">21</a>]. Briefly, <em>H. salinarum NRC-1 </em>is a halophilic archaeon with a small genome of 2.6 Mb that encodes ~2,400 protein-coding genes [<a href="#B22" rid="B22" class="cite-reflink bibr popnode">22</a>].
This organism is most prolific in aerobic conditions but switches
facultatively to other modes of energy production in anoxic
environments. We will take as our starting point a list of genes that
were found in microarray experiments to be actively expressed under low
oxygen conditions. To understand the physiological changes associated
with the anoxic state as completely as possible, we need to understand
the functions of individual proteins, metabolic pathways encoded by
genes of known function, and functional associations among genes
through evolutionary and literature analysis. Not all of this
information is contained within one resource; for instance whereas KEGG
specializes in information regarding metabolic pathways, STRING
calculates functional associations among proteins through comparative
analysis of sequence, literature and publicly available experimental
data, and DAVID classifies proteins into enriched functional clusters.
Finally, using our own expertise we have curated function assignments
to many proteins in <em>H. salinarum NRC-1 </em>using a combination of
sequence and structure-based approaches. In this example, we will
demonstrate how the Firegoose can significantly aid in functionally
characterizing a set of genes by enabling seamless exploration and
integration of several web-based tools from separate providers
including KEGG, EMBL STRING, DAVID, and an in-house annotation database
for <em>H. salinarum NRC-1</em>. First, we will briefly explain how these genes were identified.</div><div class="p" id="id545503" cite="cite-id545503" pmids="11382363">Halobacterial
cells were subjected to differing levels of oxygen and samples were
collected at varying oxygen concentrations and time. Total RNA from
these samples was analyzed using microarray analysis [<a href="#B23" rid="B23" class="cite-reflink bibr popnode">23</a>].
The DMV (Data Matrix Viewer) and the R statistical package were used to
normalize and select genes whose expression was significantly changed
in response to the perturbations in oxygen. MeV, a microarray data
analysis tool, was then used to cluster the differentially expressed
genes by their expression profiles. Two sets of genes emerged, one
activated in the presence of oxygen and another activated in absence of
oxygen. Additional details about this part of the analysis are included
in the tutorial web page [<a href="#B24" rid="B24" class="cite-reflink bibr popnode">24</a>]. We will concentrate on the anaerobically induced genes.</div><div class="p p-last" id="id545534">The
remainder of the case study traces our inquiry into the functional
roles these genes may be playing. Using Firegoose, we consult multiple
remote data sources and transfer data between them and local desktop
tools. For demonstration purposes, the lists of genes in both aerobic
and anaerobic clusters were encoded in the Gaggle microformat and
embedded in the tutorial [<a href="#B24" rid="B24" class="cite-reflink bibr popnode">24</a>],
which documents the steps of the analysis. The reader can install the
Firegoose, browse to the tutorial web page, and easily reproduce the
analysis that follows and is encouraged to do so. The analysis is
broken into numbered steps to correspond with data transfers labeled in
red in the diagram (Fig. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F5" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">5</a>).</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F5" name="F5"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F5" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Bare%202007%20Firegoose_files/picrender_005.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 5" title="Figure 5"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F5" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 5</strong></a><div class="figure-table-caption-in-article"><span><strong>Using Firegoose to investigate transcription response of <em>H. salinarum </em>to anoxic conditions</strong>.
A list of genes with similar expression profiles is found by microarray
analysis. (1) We broadcast these genes to KEGG to query for known
biochemical pathways,</span><a class="side-caption" style="font-size: 100%;" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F5" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="sec"><div class="head3 head-separate">Step 1</div><div class="p p-first" id="id545598">We
first consult KEGG, a database of curated biochemical pathways.
Starting at the tutorial web page, we broadcast the embedded anaerobic
gene list to the KEGG Pathway target. This has the same result as
cutting and pasting into the KEGG interface; KEGG performs a query for
biochemical pathways in which these genes participate. Thirty-one of
the 222 genes that were queried matched KEGG pathways including amino
acid metabolism, ABC transporters, and active potassium transport. Some
of these pathway matches are consistent with known physiological
properties of <em>H. salinarum NRC-1 </em>such as to facultatively
derive energy by fermenting arginine under anaerobic conditions.
Further, this analysis also suggests that uptake systems for several
alternate nutritional sources (e.g. glycerol) may also be utilized
under anaerobic conditions. Interestingly, KEGG also finds two
transcription factors in this anaerobic set, <em>tfbA </em>and <em>tfbE</em>, providing clues to possible regulatory mechanisms.</div><div class="p p-last" id="id545620">Although
the information gathered from this first line of analysis has yielded
considerable insight into the physiological adjustment to an anoxic
environment, 191 genes did not match any enzymes within the KEGG
catalog of pathways. At this point, we could narrow our investigation
to genes in a particular pathway or to the unmatched genes, which
Firegoose can capture and broadcast onward. But, for the next step,
we'll stick with the full set of anaerobically induced genes.</div></div><div class="sec"><div class="head3 head-separate">Step 2</div><div class="p p-first" id="id545631">To
continue our analysis, we use EMBL STRING, a powerful tool focused on
protein interactions computed from supporting evidence such as sequence
homology, journal abstracts, and protein domains. Specifically, we are
interested in finding functional associations among genes already
classified into metabolic pathways by KEGG and genes of unknown
function. We broadcast our list of anaerobically induced genes from the
tutorial page to STRING, which responds by displaying a network of
nodes (proteins) connected by colored edges representing functional
relationships.</div><div class="p p-last" id="id545638" cite="cite-id545638" pmids="15716436">We
find strongly interconnected components of the network that are
associated with specialized metabolism and do not have corresponding
pathways in KEGG. For instance, a cluster of six genes appears in the
network connected by edges indicating chromosomal proximity,
co-occurrence across genomes, and text mining results. STRING gives
annotations and protein domains that show these genes to be involved in
the use of dimethyl sulfoxide (DMSO) as an alternative electron
acceptor [<a href="#B25" rid="B25" class="cite-reflink bibr popnode">25</a>].
A second gene cluster, connected through chromosomal proximity
associations, encodes gas vesicles used by the organism to vertically
orient itself in the water column [<a href="#B26" rid="B26" class="cite-reflink bibr popnode">26</a>].</div></div><div class="sec"><div class="head3 head-separate">Step 3</div><div class="p p-first-last" id="id545672">Another interesting grouping links four genes of unknown function <em>VNG1183H</em>, <em>VNG1184G</em>, <em>VNG1185G</em>, and <em>VNG1187G</em>.
VNG1187G contains two multicopper oxidase domains. VNG1185G is
annotated as a Coenzyme PQQ synthesis protein and VNG1184G is annotated
as a heme biosynthesis protein. Recentering the network on VNG1187G and
expanding twice (to reveal second neighbors in the network) shows
functional relationships to several other proteins including more heme
biosynthesis proteins. Because not all of these genes were
differentially regulated in oxygen the initial seed generated from
expression analysis may have captured some incomplete pathways
including this particular one. However, by combining, filtering and
expanding that seed using a variety of resources we can navigate
towards a comprehensive picture.</div></div><div class="sec"><div class="head3 head-separate">Step 4</div><div class="p p-first-last" id="id545701">STRING
makes its networks available in an XML format, which the Firegoose can
parse and broadcast to other Gaggle tools. Broadcasting the network to
Cytoscape allows the user to work with the network more interactively.
For example, subsets of nodes can be selected and broadcast, or
highlighted in response to broadcasts from other tools. We select the
four-gene cluster noted above and broadcast to the Firegoose for
further investigation.</div></div><div class="sec"><div class="head3 head-separate">Step 5</div><div class="p p-first-last" id="id545711">The <em>H. salinarum NRC-1 </em>proteome
has been recently re-annotated using a combination of sequence and
structure-based approaches. These functional annotations are curated
and made publicly available on the web [<a href="#B27" rid="B27" class="cite-reflink bibr popnode">27</a>] through SBEAMS[<a href="#B20" rid="B20" class="cite-reflink bibr popnode">20</a>],
an open source data management system. In this database, VNG1187G is
annotated as a putative copper containing nitrite reductase. Drilling
down using the links provided within the SBEAMS database we see that
the supporting evidence is a match to entry <em>1KBV </em>(the membrane protein AniA, a copper-containing nitrite reductase from <em>Neisseria Gonorrhoeae</em>; e-value &lt; 10<sup>-166</sup>)
in the Protein Data Bank. This function in conjunction with the
functional associations with putative heme biosynthesis proteins and a
putative coenzyme PQQ synthesis protein form the basis of a hypothesis
that <em>H. salinarum NRC-1 </em>may possess a nitrite reducing pathway.</div></div><div class="sec"><div class="head3 head-separate">Step 6</div><div class="p p-first" id="id545764">DAVID
is a functional annotation tool that integrates many of the same
primary sources as KEGG and STRING but displays its results in tabular
format rather than as a network.</div><div class="p" id="id545768">DAVID
differs from the other resources we have used in that it doesn't work
well with the VNG naming system used to identify genes in <em>H. salinarum NRC-1</em>.
We use a Gaggle integrated translator utility to translate gene
identifiers between the VNG nomenclature and GI accession numbers
suitable to be broadcast to DAVID. The issue of multiple naming schemes
is a major bottleneck for data integration in biology. Passing
broadcasts through a simple synonym-mapping translator within the
Gaggle helps overcome this hurdle.</div><div class="p p-last" id="id545779">DAVID
also clusters genes with related annotations, conveniently summarizing
the kinds of cellular processes for which a set of genes is enriched.
Our genes of interest divide into 17 clusters — several of which were
consistent with the results from previous steps. For example, seven
genes, including one member of the DMSO cluster, have products
annotated as being involved in electron transport.</div></div><div class="sec sec-last"><div class="head3 head-separate">Step 7</div><div class="p p-first-last" id="id545789" cite="cite-id545789" pmids="9847187 9600884">DAVID
produces another cluster containing signal transduction proteins. Back
in STRING, we can click on these proteins to view functional domains
provided by the SMART database [<a href="#B28" rid="B28" class="cite-reflink bibr popnode">28</a>,<a href="#B29" rid="B29" class="cite-reflink bibr popnode">29</a>].
VNG0355G (Htr14) contains sensory domains associated with chemotaxis.
Both VNG0716G (AfsQ2) and VNG1175G (PhoR) contain domains implicated in
signal transduction and light sensitivity. It is notable that this is
biologically meaningful because light and oxygen physiology in <em>H. salinarum NRC-1 </em>are tightly coupled with one another and also with physical relocation (taxis).</div></div></div><div class="sec"><div class="head2 head-separate">Summary of results</div><div class="p p-first" id="id545825">In
this example we have combined four web-based resources with several
desktop applications to build a unified understanding of the physiology
of <em>H. salinarum NRC-1 </em>under anaerobic conditions. The
depletion of oxygen seems to activate certain aspects of amino acid
metabolism, and alternate energy transduction pathways such as
phototrophy, arginine fermentation and DMSO respiration. The
simultaneous induction of gas vesicle synthesis is consistent with the
anaerobic physiological behavior of <em>H. salinarum NRC-1 </em>to move towards the surface in search of alternate energy sources including light. The data also suggest that <em>H. salinarum NRC-1 </em>may
have some specific nutritional requirements in this anoxic environment
that cause the induction of an array of membrane transport systems and
possibly previously uncharacterized functions such as dissimilatory
nitrite reduction. In addition, putative components of the signaling
and regulatory mechanisms that may mediate the transition to this
metabolic state were also detected.</div><div class="p p-last" id="id545847">It
is important to recognize that the information resources, while
overlapping to some degree, provide complementary perspectives. KEGG
provides information regarding biochemical pathways but does not draw
associations between these pathways and genes of unknown function. This
information can be obtained from STRING, which on its own is not
cognizant of the metabolic pathways. Likewise, DAVID includes
information on functional domains within proteins and integrates many
of the same primary sources as KEGG and STRING. But, DAVID presents the
information differently with its unique clustering feature to provide a
statistical evaluation of the enrichment of particular functions among
the queried genes. Finally, our local database provides a source of
data curated manually by experts through years of careful literature
surveys and experimentation. Integration of these web-based resources
with desktop applications through the Gaggle and the Firegoose enables
the type of analysis that is necessary to understand the complex
dynamic regulation of cellular responses (Fig. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F6" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">6</a>). Complete data from each of our four sources can be found in the supplementary table (Additional file <a href="#S1" rid="S1" class="side-supplink supplementary-material">1</a>).</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F6" name="F6"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F6" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Bare%202007%20Firegoose_files/picrender_006.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 6" title="Figure 6"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F6" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 6</strong></a><div class="figure-table-caption-in-article"><span><strong>Integrated visualization of <em>H. salinarum </em>response to anoxic conditions</strong>.
The transcriptional response to anoxia was characterized using several
data sources. Edges represent several types of evidence for functional
association provided by STRING. Yellow</span><a class="side-caption" style="font-size: 100%;" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2211326&amp;rendertype=figure&amp;id=F6" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></tbody></table></div></div><div style="clear: both;"></div></div><div class="sec"><div class="head2 head-separate">Comparison with workflow software</div><div class="p p-first" id="id545915" cite="cite-id545915" pmids="16845108 15201187">Firegoose, along with the Gaggle framework, shares several features in common with workflow tools such as Taverna[<a href="#B30" rid="B30" class="cite-reflink bibr popnode">30</a>,<a href="#B31" rid="B31" class="cite-reflink bibr popnode">31</a>].
They share the strategy of composing distinct programs and data sources
to build larger systems with rich capabilities. To this end, both
Firegoose and workflow tools benefit greatly from the availability of
programmatic access to structured data and computational services over
common web protocols.</div><div class="p p-last" id="id545942">Workflow
tools enable a user to automate a well-defined and repeatable process,
often using web services or message queues for interprocess
communication. But, before a well-defined analysis process exists,
there is a need for exploratory analysis, which is necessarily ad-hoc.
Gaggle and the Firegoose seek to enable this kind of interactive
exploration by exploiting the flexibility of web-based tools in
combination with desktop analysis and visualization tools. Scripting an
interaction with a web site is, of course, possible using a browser
extension, but the emphasis in the Firegoose is on automating the
exchange of data, leaving the direction of the analysis up to the user.
The difference in emphasis does not rule out using workflow engines and
Firegoose together. Invoking workflows on a remote server from within
Firegoose is one potentially valuable example.</div></div><div class="sec sec-last"><div class="head2 head-separate">Future Work</div><div class="p p-first" id="id545956">Additional
functionality could be added to the Firegoose in a number of ways, most
easily through the addition of more handlers for biological websites.
We also considered that users might want to develop their own handler
scripts. We prototyped code for dynamically importing custom scripts
into the Firegoose. Other projects such as Greasemonkey [<a href="#B8" rid="B8" class="cite-reflink bibr popnode">8</a>]
have had success with similar capabilities. If further developed, this
feature would allow a straightforward mechanism for users or data
providers to contribute scripts.</div><div class="p" id="id545973">Supporting
the RMI communications protocol requires Java. Using Java within a
Firefox extension is something of a challenge and code from MIT's
Simile project [<a href="#B9" rid="B9" class="cite-reflink bibr popnode">9</a>]
was extremely helpful in this area. An alternative under consideration
is to communicate with the Boss using an XML based protocol over
sockets eliminating the need to run a Java virtual machine in the
browser's process.</div><div class="p" id="id545989">We plan to extend
the Gaggle microformat to express links to data in addition to
embedding data directly in the page. This allows large data structures
to be transmitted independently of the page while preserving the
linkage between presentation in the browser and the underlying
structured data. A standard format would decrease the need for
customized coding for each web site.</div><div class="p p-last" id="id545994">RDF
(Resource Description Framework) is a data model designed to represent
meta-data for Semantic Web applications. Incorporating support for RDF
into the Firegoose would allow the Gaggle to exchange data with the
semantically rich resources envisioned by proponents of the Semantic
Web project.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532350" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532409" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id544990" class="sidefm-pmclink" style="text-transform: none;">Implementation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545440" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Bare%202007%20Firegoose_files/square.gif" alt="&gt;" style="border-style: none;"></span>Conclusion</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546026" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546099" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546119" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546178" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id545999">Conclusion</div><div class="section-content"><div class="p p-first" id="id546003">The
Firegoose incorporates Mozilla Firefox into the Gaggle environment
providing coordinated access to web applications and programmatic data
sources. Performing data integration in the browser has several
advantages and is perhaps the most interesting feature of the
Firegoose. Browsers excel at search and navigation. Using the
Firegoose, a biologist can search and navigate web resources using
familiar browser-based interfaces with the additional capability of
easily moving data from one web-based resource to another as well as
between the web and the desktop. Interactively integrating specific
information as needed replaces the cumbersome process of maintaining
local copies of large databases and manually coercing data from diverse
sources into a compatible format. Using the Gaggle data types as
intermediaries lowers the barrier between web resources and desktop
tools, allowing the scientist to creatively combine and re-use data in
ways that go beyond those provided by the curators of individual data
sources.</div><div class="p" id="id546014">The Firegoose positions the
Gaggle to take advantage of increasing use of web protocols to transmit
structured data. The Firegoose provides a framework in which new web
resources can be integrated into the Gaggle in a straightforward and
easily implemented manner, accommodating a variety of protocols. In
supporting a number of protocols, we hope to encourage data providers
to make available structured data in the format of their choice and to
provide the necessary information to link web interfaces with the
underlying data allowing browsing and programmatic access to become
seamlessly integrated.</div><div class="p p-last" id="id546022">If the
web is becoming a channel for structured data, applications that share
data between diverse web resources and software tools will be of
increasing importance. The Firegoose aims to fill this role for the
systems biology domain.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532350" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532409" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id544990" class="sidefm-pmclink" style="text-transform: none;">Implementation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545440" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545999" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Bare%202007%20Firegoose_files/square.gif" alt="&gt;" style="border-style: none;"></span>Availability and requirements</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546099" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546119" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546178" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id546026">Availability and requirements</div><div class="section-content"><div class="p p-first" id="id546031">Source code for the Firegoose, along with that of the other components of the Gaggle, is available at the Gaggle website [<a href="#B32" rid="B32" class="cite-reflink bibr popnode">32</a>]. Also available are instructions for installing and uninstalling the Firegoose toolbar [<a href="#B33" rid="B33" class="cite-reflink bibr popnode">33</a>] and documentation [<a href="#B34" rid="B34" class="cite-reflink bibr popnode">34</a>].
Most of the desktop components of the Gaggle are deployed as Java
webstarts, which can be launched by clicking a link in the browser.</div><div class="p" id="id546068">The
toolbar is compatible with versions 1.5.x and 2.0.x of Mozilla Firefox.
We anticipate maintaining compatibility with Firefox 3.x when released.</div><div class="p" id="id546072">Java version 5 [<a href="#B35" rid="B35" class="cite-reflink bibr popnode">35</a>]
or higher runtime environment is required and the Java browser plug in
for Firefox must be installed. Extra attention is often required to
install the Java browser plug-in on Linux. Specific instructions for
most distributions are available on the web.</div><div class="p p-last" id="id546088">The source code is distributed under the GNU Lesser General Public License, the text of which is available at: <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=081s9L2gF_wcN36VhsNX51fCSdXfgAgw9lv8GaQQn&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.gnu.org/copyleft/lesser.html" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.gnu.org/copyleft/lesser.html</a></span>.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id546099">Authors' contributions</div><div class="section-content"><div class="p p-first" id="id546103">JCB Wrote the manuscript and implemented software.</div><div class="p" id="id546106">PTS Conceived and initiated the project, implemented prototype and provided feedback on the written manuscript.</div><div class="p" id="id546109">AKS Assisted in the conception and implementation of the case study and provided feedback on the written manuscript.</div><div class="p" id="id546113">NSB
Conceived and initiated the project, wrote the manuscript and provided
direction and feedback on quality of results and software design.</div><div class="p p-last" id="id546116">All authors read and approved the final manuscript.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id546119"><a id="supplementary-material-sec"></a>Supplementary Material</div><div class="section-content"><!--body--><div id="S1"><div><strong>Additional file 1</strong></div><div class="p" id="id546138"><strong>Characterization of <em>H. salinarum </em>anoxic transcription response by four data sources</strong>.
Data about our genes of interest from KEGG, STRING, DAVID, and the
Halobacterium genome annotations database are presented here in tabular
form.</div><div class="media"><blockquote><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2211326&amp;blobname=1471-2105-8-456-S1.xls">Click here for file</a><sup> (590K)</sup></blockquote></div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145">&nbsp;</td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id546164">Acknowledgements</div><div class="section-content"><div class="sec"><div class="sec sec-first"><div class="p p-first" id="id546170">We
thank Dan Tenenbaum and Ricardo Vencio for critical reading of the
manuscript and helpful suggestions. We also thank John Boyle, Chris
Cavnor, David Shteynberg, and Neils Gelenburg for thoughtful
discussions.</div><div class="p p-last" id="id546174">This work was supported by the following grants: NSF: DBI-0640950; DoE: DE-FG02-07ER64327; and NIH: P50 GM076547.</div></div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532350" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id532409" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id544990" class="sidefm-pmclink" style="text-transform: none;">Implementation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545440" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id545999" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546026" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546099" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id546119" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Bare%202007%20Firegoose_files/square.gif" alt="&gt;" style="border-style: none;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id546178">References</div><div class="section-content"><div class="back-matter-section" id="reference-list"><ul class="back-ref-list" style="list-style-type: decimal;"><li id="B1"><div class="citation">Stein LD. Integrating biological databases. <span><span class="ref-journal">Nature reviews. </span>2003;<span class="ref-vol">4</span>:337–345. doi: 10.1038/nrg1065.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=02SF0SitxNlstR0XY_K42smDmqPIn5Rli-m0NCQgh&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12728276">PubMed</a>]</span></div></li><li id="B2"><div class="citation">Covitz PA, Hartel F, Schaefer C, De Coronado S, Fragoso G, Sahni H, Gustafson S, Buetow KH. <span class="ref-journal">Bioinformatics.</span> Vol. 19. Oxford, England; 2003. caCORE: a common infrastructure for cancer informatics; pp. 2404–2412.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0lme6obhINUpHgvxnq7b0w9TbIcncg1Hk4FKBCk6q&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14668224">PubMed</a>]</span></div></li><li id="B3"><div class="citation">Wilkinson MD, Links M. BioMOBY: an open source biological web services proposal. <span><span class="ref-journal">Briefings in bioinformatics. </span>2002;<span class="ref-vol">3</span>:331–341. doi: 10.1093/bib/3.4.331.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0R5IXdYhXm4OzNpr8wtEG5LVAZ2FY8WjMIpoK7QUY&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12511062">PubMed</a>]</span></div></li><li id="B4"><div class="citation">Shannon
PT, Reiss DJ, Bonneau R, Baliga NS. The Gaggle: an open-source software
system for integrating bioinformatics software and data sources. <span><span class="ref-journal">BMC bioinformatics. </span>2006;<span class="ref-vol">7</span>:176. doi: 10.1186/1471-2105-7-176.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0bmWNIMXwnJzuhB0BE9UmuE04Tgmh8vUK7g99q5zQ&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16569235">PubMed</a>]</span></div></li><li id="B5"><div class="citation">Microformats.org. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0N3GSkbBiJu2V6Nkm4SxPx2Mxd_618k6-yf7ox5n_&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://microformats.org/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://microformats.org/</a></span></div></li><li id="B6"><div class="citation">Semantic Web. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0HENQY0sTUDCTo3UU23ffbAhnzZrNSt58SDa8E5AU&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.w3.org/2001/sw/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.w3.org/2001/sw/</a></span></div></li><li id="B7"><div class="citation">Microformats. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ZAJcLTRWAHiNr5ZoANCsiZ2GMQKIkoAypTP-pkDS&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://blog.mozilla.com/faaborg/2006/12/11/microformats-part-0-introduction/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://blog.mozilla.com/faaborg/2006/12/11/microformats-part-0-introduction/</a></span></div></li><li id="B8"><div class="citation">Greasemonkey. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0BATX64DyKJCskEbVIVKzLXP9f76dvJLGusfE1QPH&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.greasespot.net/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.greasespot.net/</a></span></div></li><li id="B9"><div class="citation">Huynh D, Mazzocchi S, Karger D. Piggy Bank: Experience the Semantic Web Inside Your Web Browser. <span class="ref-journal">International Semantic Web Conference: 2005.</span> 2005. </div></li><li id="B10"><div class="citation">Operator. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Bm0ZCdol4p8kN9qLMXA_QMRMN8MhKuqpIRRylQ00&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;https://addons.mozilla.org/en-US/firefox/addon/4106" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">https://addons.mozilla.org/en-US/firefox/addon/4106</a></span></div></li><li id="B11"><div class="citation">Mozilla Firefox. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0r1_fgY1SLhiIQxno7rGptlRGdSnvNmN64_fPRAN0&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.mozilla.com" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.mozilla.com</a></span></div></li><li id="B12"><div class="citation">Kanehisa M. The KEGG database. <span><span class="ref-journal">Novartis Foundation symposium. </span>2002;<span class="ref-vol">247</span>:91–101.</span> discussion 101–103, 119–128, 244–152.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0PGur4SwejupOHmqLt1JoEa0TOBtyAeqhs4EdvAPO&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12539951">PubMed</a>]</span></div></li><li id="B13"><div class="citation">von
Mering C, Jensen LJ, Kuhn M, Chaffron S, Doerks T, Kruger B, Snel B,
Bork P. STRING 7 – recent developments in the integration and
prediction of protein interactions. <span><span class="ref-journal">Nucleic acids research. </span>2007:D358–362. doi: 10.1093/nar/gkl825.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0-Kwi92oCtHOmeLrmOF-vXDa6wgsSo2Bf47Mv_kBy&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17098935">PubMed</a>]</span></div></li><li id="B14"><div class="citation">von
Mering C, Jensen LJ, Snel B, Hooper SD, Krupp M, Foglierini M, Jouffre
N, Huynen MA, Bork P. STRING: known and predicted protein-protein
associations, integrated and transferred across organisms. <span><span class="ref-journal">Nucleic acids research. </span>2005:D433–437.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Vq2MpumTTuEHpWHcN74-QuoBLL44lDqJut2NmAMn&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15608232">PubMed</a>]</span></div></li><li id="B15"><div class="citation">Dennis
G, Jr, Sherman BT, Hosack DA, Yang J, Gao W, Lane HC, Lempicki RA.
DAVID: Database for Annotation, Visualization, and Integrated
Discovery. <span><span class="ref-journal">Genome biology. </span>2003;<span class="ref-vol">4</span>:P3. doi: 10.1186/gb-2003-4-5-p3.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0HeegtMjuKScxlSfUTwpDZvxnwwtrI5fuVfQV957i&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12734009">PubMed</a>]</span></div></li><li id="B16"><div class="citation">Shannon
P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N,
Schwikowski B, Ideker T. Cytoscape: a software environment for
integrated models of biomolecular interaction networks. <span><span class="ref-journal">Genome research. </span>2003;<span class="ref-vol">13</span>:2498–2504. doi: 10.1101/gr.1239303.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0fs3kFuRih7OBXePG7fgWQDGTaze1PXKISekmpAt8&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14597658">PubMed</a>]</span></div></li><li id="B17"><div class="citation">R Statistical Package. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0zUWAOms7jOwdEDU8JJHpv9jScOTweU00LdjUVQWg&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.r-project.org" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.r-project.org</a></span></div></li><li id="B18"><div class="citation">Saeed
AI, Sharov V, White J, Li J, Liang W, Bhagabati N, Braisted J, Klapa M,
Currier T, Thiagarajan M, et al. TM4: a free, open-source system for
microarray data management and analysis. <span><span class="ref-journal">BioTechniques. </span>2003;<span class="ref-vol">34</span>:374–378.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0jcIohBOxYjvnafgAJ9PRXlkmRjC3kMr7sWmjI5yN&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12613259">PubMed</a>]</span></div></li><li id="B19"><div class="citation">Gaggle Microformat. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0lgUt5tY1_i8Balvd_eC_osqkv6lN27dmokozWA2G&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://gaggle.systemsbiology.net/docs/geese/firegoose/microformat/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://gaggle.systemsbiology.net/docs/geese/firegoose/microformat/</a></span></div></li><li id="B20"><div class="citation">SBEAMS. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Dg8ZjarQfG29_dQlirp-3Fgeo0ikUoizUUkILk9Z&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.sbeams.org/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.sbeams.org/</a></span></div></li><li id="B21"><div class="citation">Schmid
A, Reiss DJ, Kaur A, Pan M, King N, Van PT, Hohmann L, Martin DB,
Baliga NS. The anatomy of microbial cell state transitions in response
to oxygen. <span><span class="ref-journal">Genome research. </span>2007;<span class="ref-vol">17</span>:1399–1413. doi: 10.1101/gr.6728007.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0hkFewSRrdTfVG1B-pD10259gmWgR_T3EoBoV5k99&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17785531">PubMed</a>]</span></div></li><li id="B22"><div class="citation">Ng
WV, Kennedy SP, Mahairas GG, Berquist B, Pan M, Shukla HD, Lasky SR,
Baliga NS, Thorsson V, Sbrogna J, et al. Genome sequence of
Halobacterium species NRC-1. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the United States of America. </span>2000;<span class="ref-vol">97</span>:12176–12181. doi: 10.1073/pnas.190337797.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Xy6e7dfZIYUTmbuJnYe5O8opLgVkQ89xjOGKOk2Q&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11016950">PubMed</a>]</span></div></li><li id="B23"><div class="citation">Ideker
T, Thorsson V, Siegel AF, Hood LE. Testing for differentially-expressed
genes by maximum-likelihood analysis of microarray data. <span><span class="ref-journal">J Comput Biol. </span>2000;<span class="ref-vol">7</span>:805–817. doi: 10.1089/10665270050514945.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=023HWjivT9nmzscOurmEm_PYlnJfXfBWl_oTnJQwQ&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11382363">PubMed</a>]</span></div></li><li id="B24"><div class="citation">Gaggle and Firegoose Oxygen Demo. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0XA5Gz9F80AbCHQLgsPOArTtDxpMXK_iIuUBNLQSX&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://gaggle.systemsbiology.net/projects/demos/halo_oxygen_analysis/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://gaggle.systemsbiology.net/projects/demos/halo_oxygen_analysis/</a></span></div></li><li id="B25"><div class="citation">Muller
JA, DasSarma S. Genomic analysis of anaerobic respiration in the
archaeon Halobacterium sp. strain NRC-1: dimethyl sulfoxide and
trimethylamine N-oxide as terminal electron acceptors. <span><span class="ref-journal">Journal of bacteriology. </span>2005;<span class="ref-vol">187</span>:1659–1667. doi: 10.1128/JB.187.5.1659-1667.2005.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0voW5JJ_oQVTa33Tjgz3yrOmIaAQRCb_St39HukKu&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15716436">PubMed</a>]</span></div></li><li id="B26"><div class="citation">Robb FT, Place AR, Sowers KR, Schreier HJ, DasSarma S, Fleischmann EM. <span class="ref-journal">Archaea: A laboratory manual.</span> Cold Spring Harbor, New York.: Cold Spring Harbor Laboratory Press; 1995. </div></li><li id="B27"><div class="citation">Halobacterium genome annotations. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0NGtYh2MgxAPUm1ovDjJUk8LUN8Vl8h3iP6qxEkCr&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://baliga.systemsbiology.net/halobacterium/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://baliga.systemsbiology.net/halobacterium/</a></span></div></li><li id="B28"><div class="citation">Ponting
CP, Schultz J, Milpetz F, Bork P. SMART: identification and annotation
of domains from signalling and extracellular protein sequences. <span><span class="ref-journal">Nucleic acids research. </span>1999;<span class="ref-vol">27</span>:229–232. doi: 10.1093/nar/27.1.229.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0bE56KCwXPBf4lHrBoWLisAeAL334HVRY0WrRl57b&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9847187">PubMed</a>]</span></div></li><li id="B29"><div class="citation">Schultz
J, Milpetz F, Bork P, Ponting CP. SMART, a simple modular architecture
research tool: identification of signaling domains. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the United States of America. </span>1998;<span class="ref-vol">95</span>:5857–5864. doi: 10.1073/pnas.95.11.5857.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0dSrpC0YALsgKzP_F8EgiO2QnUGBCDaAkuTUGJkcH&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9600884">PubMed</a>]</span></div></li><li id="B30"><div class="citation">Hull
D, Wolstencroft K, Stevens R, Goble C, Pocock MR, Li P, Oinn T.
Taverna: a tool for building and running workflows of services. <span><span class="ref-journal">Nucleic acids research. </span>2006:W729–732. doi: 10.1093/nar/gkl320.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0TCeQ7JInARAViSydcBTgPg0Ly6VdppnBDZtZjA-Y&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16845108">PubMed</a>]</span></div></li><li id="B31"><div class="citation">Oinn T, Addis M, Ferris J, Marvin D, Senger M, Greenwood M, Carver T, Glover K, Pocock MR, Wipat A, et al.  <span class="ref-journal">Bioinformatics.</span> Vol. 20. Oxford, England; 2004. Taverna: a tool for the composition and enactment of bioinformatics workflows; pp. 3045–3054.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0nAMDNa3lLBmgemb0SNGNeB_BloiK8NvYP1vzJAw4&amp;reftype=pubmed&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15201187">PubMed</a>]</span></div></li><li id="B32"><div class="citation">The Gaggle website. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=08U0YYqaQJfhTAu35cfnVsG8UymHPOu2brW2QoAVu&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://gaggle.systemsbiology.net" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://gaggle.systemsbiology.net</a></span></div></li><li id="B33"><div class="citation">Firegoose Installation Help. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0dAKFRLwvQ1y2Z8geU8WOKDwSfj6snGc1p--VZArJ&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://gaggle.systemsbiology.net/docs/geese/firegoose/install/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://gaggle.systemsbiology.net/docs/geese/firegoose/install/</a></span></div></li><li id="B34"><div class="citation">Firegoose. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0-oo62qeNFTqFHZD2sm10nfdIi2VReZiGJIiE4Adh&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://gaggle.systemsbiology.net/docs/geese/firegoose/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://gaggle.systemsbiology.net/docs/geese/firegoose/</a></span></div></li><li id="B35"><div class="citation">Download Java. <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0JcU_kKezl2V6y2mSl5Ov_KQW_sLcmnCga0KXo5n_&amp;reftype=extlink&amp;artid=2211326&amp;article-id=2211326&amp;iid=140969&amp;issue-id=140969&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.java.com/download/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.java.com/download/</a></span></div></li></ul></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">BMC Bioinformatics</span> are provided here courtesy of <br><b>BioMed Central</b></div>
    <div class="footer-link" id="fp-footer">
      <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.pubmedcentral.nih.gov/">PMC Home</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov/">NCBI</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov/">NIH</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="http://www.pubmedcentral.nih.gov/about/privacypolicy.html">Privacy Policy</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></tbody></table><!--host:ipmc2-->

</body></html>