<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN">
<html xmlns:m="http://www.w3.org/1998/Math/MathML"><head><title>Source
Code for Biology and Medicine | Full text | Bioinformatics process
management: information flow via a computational journal</title><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

<meta name="citation_pdf_url" content="http://www.scfbm.org/content/pdf/1751-0473-2-9.pdf">

<meta name="citation_abstract_html_url" content="http://www.scfbm.org/content/2/1/9/abstract/">
<meta name="citation_fulltext_html_url" content="http://www.scfbm.org/content/2/1/9">

<script src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/__utm.js" type="text/javascript"></script><script language="JavaScript1.5">_u5_ujv='1.5';</script><script src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/viewingoptions.js" type="text/javascript"></script><link rel="stylesheet" type="text/css" href="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/screen_002.css" title="Default Styles"><!--[if gte IE 7]><link rel="stylesheet" type="text/css" href="/stylesheets/ie7.css" /><![endif]--><!--[if lte IE 6]><link rel="stylesheet" type="text/css" href="/stylesheets/ie56.css" /><![endif]-->

<meta name="dc.title" content="Bioinformatics process management: information flow via a computational journal">
<meta name="dc.creator" content="Lance Feagan">
<meta name="dc.creator" content="Justin Rohrer">
<meta name="dc.creator" content="Alexander Garrett">
<meta name="dc.creator" content="Heather Amthauer">
<meta name="dc.creator" content="Ed Komp">
<meta name="dc.creator" content="David Johnson">
<meta name="dc.creator" content="Adam Hock">
<meta name="dc.creator" content="Terry Clark">
<meta name="dc.creator" content="Gerald Lushington">
<meta name="dc.creator" content="Gary Minden">
<meta name="dc.creator" content="Victor Frost">
<meta name="dc.type" content="Research">
<meta name="dc.source" content="Source Code for Biology and Medicine 2007 2:9">
<meta name="dc.date" content="2007-12-03">
<meta name="dc.identifier" content="10.1186/1751-0473-2-9">
<meta name="dc.publisher" content="BioMed Central Ltd">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/">
<meta name="dc.format" content="text/html">
<meta name="dc.language" content="en">

<meta name="prism.publicationName" content="Source Code for Biology and Medicine">
<meta name="prism.issn" content="1751-0473">
<meta name="prism.publicationDate" content="2007-12-03">
<meta name="prism.volume" content="2">
<meta name="prism.number" content="1">
<meta name="prism.section" content="Research">
<meta name="prism.startingPage" content="9">
<meta name="prism.copyright" content="2007 Feagan et al; ">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

<meta name="citation_journal_title" content="Source Code for Biology and Medicine">
<meta name="citation_publisher" content="BioMed Central Ltd">
<meta name="citation_authors" content="Lance Feagan, Justin Rohrer, Alexander Garrett, Heather Amthauer, Ed Komp, David Johnson, Adam Hock, Terry Clark, Gerald Lushington, Gary Minden, Victor Frost">
<meta name="citation_title" content="Bioinformatics process management: information flow via a computational journal">
<meta name="citation_volume" content="2">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="9">
<meta name="citation_doi" content="10.1186/1751-0473-2-9">

<link rel="stylesheet" type="text/css" href="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/screen.css" title="Default Styles">

<link rel="alternate" type="application/pdf" title="PDF" href="http://www.scfbm.org/content/pdf/1751-0473-2-9.pdf">
<link rel="alternate" type="text/xml" title="XML version" href="http://www.scfbm.org/content/xml/1751-0473-2-9.xml">

</head><div FirebugVersion="1.3.3" style="display: none;" id="_firebugConsole"></div><body class="bgimage" alink="#6699ff" link="#003399" vlink="#3366ff"><center><div id="outerframe"><div class="skip"><a href="#content">Skip to main content</a></div><div id="newad"><table align="center" border="0" cellpadding="0" cellspacing="0"><tbody><tr><td class="topad" align="center">

<center>

<!------ OAS SETUP begin ------>
<script language="JavaScript">
<!--
OAS_version = 10;
function OAS_NORMAL(pos)
{
   document.write('<a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.scfbm.org/article/sourcecodebiolmed/1491372061@Top,Bottom!' + pos + '?%22bioinformatics+workflows%22+pise+review" target=_top>');
   document.write('<img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.scfbm.org/article/sourcecodebiolmed/1491372061@Top,Bottom!' + pos + '?%22bioinformatics+workflows%22+pise+review" border=0></a>');
}
//-->
</script>
<script language="JavaScript1.1">
<!--
OAS_version = 11;
if (navigator.userAgent.indexOf('Mozilla/3') > 1)
   OAS_version = 10;
if (navigator.userAgent.indexOf('Netscape/7.0') > 1)
   OAS_version = 10;
if (OAS_version >= 11)
   document.write('<scr' + 'ipt language="JavaScript1.1" src="http://oas.biomedcentral.com/RealMedia/ads/adstream_mjx.ads/www.scfbm.org/article/sourcecodebiolmed/1491372061@Top,Bottom?%22bioinformatics+workflows%22+pise+review"></scr' + 'ipt>');
//-->
</script><script language="JavaScript1.1" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1491372061TopBottom.js"></script>
<script language="JavaScript">
<!--
document.write('');
function OAS_AD(pos)
{
   if (OAS_version >= 11)
      OAS_RICH(pos);
   else
      OAS_NORMAL(pos);
}
//-->
</script>
<!------ OAS SETUP end ------>
<div id="dropdown" onmouseover="testMouse()" onmouseout="testMouse()">
<!------ OAS AD 'Top' begin ------>
<script language="JavaScript">
<!--
OAS_AD('Top');
//-->
</script><!-- 
Support: http://oas.biomedcentral.com#biomed/Alma-Biomed-Lead-2#Alma-Biomed-Lead-2#Alma-Biomed-Lead-2.html#c7b8b#1239108553#437#S#Top#www.scfbm.org/article/sourcecodebiolmed##
--><script id="extFlashTop1" type="text/javascript" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/TFSMflashobject.js"></script>
<script language="JavaScript">
<!--
if(!document.body)
document.write("<html><body>");
OASd = document;
var plug = false;
var flashVersion = -1;
var minFlashVersion = 6;
if(navigator.plugins != null && navigator.plugins.length > 0){flashVersion =(navigator.plugins["Shockwave Flash 2.0"] || navigator.plugins["Shockwave Flash"]) ? navigator.plugins["Shockwave Flash" +(navigator.plugins["Shockwave Flash 2.0"] ? " 2.0" : "")].description.split(" ")[2].split(".")[0] : -1;
plug = flashVersion >= minFlashVersion;}
else if(navigator.userAgent.toLowerCase().indexOf("webtv/2.6") != -1){flashVersion = 4;plug = flashVersion >= minFlashVersion;}
else if(navigator.userAgent.toLowerCase().indexOf("webtv/2.5") != -1){flashVersion = 3;plug = flashVersion >= minFlashVersion;}
else if(navigator.userAgent.toLowerCase().indexOf("webtv") != -1){flashVersion = 2;plug = flashVersion >= minFlashVersion;}
else if((navigator.appVersion.indexOf("MSIE") != -1) &&(navigator.appVersion.toLowerCase().indexOf("win") != -1) &&(navigator.userAgent.indexOf("Opera") == -1)){var oasobj;
var exc;
try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.7");
flashVersion = oasobj.GetVariable("$version");} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.6");
version = "WIN 6,0,21,0"; 
oasobj.AllowScriptAccess = "always";
flashVersion = oasobj.GetVariable("$version");} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.3");
flashVersion = oasobj.GetVariable("$version");} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.3");
flashVersion = "WIN 3,0,18,0";} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash");
flashVersion = "WIN 2,0,0,11";} catch(exc){flashVersion = -1;}}}}}
plug =(flashVersion != -1)? flashVersion.split(" ")[1].split(",")[0] >= minFlashVersion : false;}

if(plug)
{
document.write("<div id=\"FinContentTop1\"></div>");
function loadFlashTop1(){
	if(navigator.userAgent.indexOf("MSIE") != -1 && navigator.userAgent.indexOf("Opera") == -1){
	
		if (extFlashTop1.readyState == "complete")
  		{
	   		FlashObject("http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-2/novoseek_728x90_02_flash08.swf?clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.scfbm.org/article/sourcecodebiolmed/1418541634/Top/biomed/Alma-Biomed-Lead-2/Alma-Biomed-Lead-2.html/34386665393030393439386238646230?%22bioinformatics+workflows%22+pise+review", "OAS_AD_Top", "width=728 height=90", "opaque", "clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" ,"6", "FinContentTop1");
			extFlashTop1.onreadystatechange = "";
		}
	
		extFlashTop1.onreadystatechange = loadFlashTop1;
	}
	else
	{
           OASfp=" Menu=FALSE swModifyReport=TRUE width=728 height=90 ";
           OASd.write("<object id=\"techsource_Top\" classid=\"clsid:D27CDB6E-AE6D-11cf-96B8-444553540000\" codebase=\"http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,0,0\""+OASfp+">");
           OASd.write("<param name=src value=\"http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-2/novoseek_728x90_02_flash08.swf\"><param name=allowScriptAccess value=always><param name=quality value=autohigh><param name=loop value=true><param name=play value=true><param name=menu value=false><param name=wmode value=opaque><param name=FlashVars value=\"clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.scfbm.org/article/sourcecodebiolmed/1418541634/Top/biomed/Alma-Biomed-Lead-2/Alma-Biomed-Lead-2.html/34386665393030393439386238646230?%22bioinformatics+workflows%22+pise+review\">");
           OASd.write("<embed src=\"http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-2/novoseek_728x90_02_flash08.swf\""+OASfp+"pluginspage=\"http://www.macromedia.com/shockwave\" type=\"application/x-shockwave-flash\" width=728 height=90 PLAY=true LOOP=true QUALITY=autohigh WMODE=opaque FlashVars=\"clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.scfbm.org/article/sourcecodebiolmed/1418541634/Top/biomed/Alma-Biomed-Lead-2/Alma-Biomed-Lead-2.html/34386665393030393439386238646230?%22bioinformatics+workflows%22+pise+review\" allowScriptAccess=always>");
           OASd.write("</embed></object>");
        }

}
	loadFlashTop1();
}
else
{
OASd.write("<a href=\"http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.scfbm.org/article/sourcecodebiolmed/1418541634/Top/biomed/Alma-Biomed-Lead-2/Alma-Biomed-Lead-2.html/34386665393030393439386238646230?%22bioinformatics+workflows%22+pise+review\" target=\"_blank\"><img src=\"http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-2/banner_alternativo2.gif\" width=728 height=90 border=0 alt=\"\"></a>");}
if(!document.body)
document.write("</body></html>");
//-->
</script><div id="FinContentTop1"></div><object id="techsource_Top" classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,0,0" menu="FALSE" swmodifyreport="TRUE" height="90" width="728"><param name="src" value="http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-2/novoseek_728x90_02_flash08.swf"><param name="allowScriptAccess" value="always"><param name="quality" value="autohigh"><param name="loop" value="true"><param name="play" value="true"><param name="menu" value="false"><param name="wmode" value="opaque"><param name="FlashVars" value="clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.scfbm.org/article/sourcecodebiolmed/1418541634/Top/biomed/Alma-Biomed-Lead-2/Alma-Biomed-Lead-2.html/34386665393030393439386238646230?%22bioinformatics+workflows%22+pise+review"><embed src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/novoseek_728x90_02_flash08.swf" menu="FALSE" swmodifyreport="TRUE" pluginspage="http://www.macromedia.com/shockwave" type="application/x-shockwave-flash" play="true" loop="true" quality="autohigh" wmode="opaque" flashvars="clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.scfbm.org/article/sourcecodebiolmed/1418541634/Top/biomed/Alma-Biomed-Lead-2/Alma-Biomed-Lead-2.html/34386665393030393439386238646230?%22bioinformatics+workflows%22+pise+review" allowscriptaccess="always" height="90" width="728"></object><noscript>
<a
href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.scfbm.org/article/sourcecodebiolmed/1491372061@Top,Bottom!Top?%22bioinformatics+workflows%22+pise+review"
target=_top>
<img
src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.scfbm.org/article/sourcecodebiolmed/1491372061@Top,Bottom!Top?%22bioinformatics+workflows%22+pise+review"
border=0></a>
</noscript></div></center></td><td valign="top"><table height="95"><tbody><tr><td style="vertical-align: middle;"><a href="http://www.biomedcentral.com/info/mediainformation/"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/advhere.gif" alt="" border="0"></a></td></tr></tbody></table>
<!------ OAS AD end ------>


</td></tr></tbody></table></div><div id="innerframe"><table border="0" cellpadding="5" cellspacing="0" width="100%"><tbody><tr valign="bottom"><td width="50"><a href="http://www.scfbm.org/"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/headsquare.gif" alt="" border="0" hspace="3"></a></td><td align="left"><a href="http://www.scfbm.org/"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/type.gif" alt="" border="0"></a><a href="http://www.biomedcentral.com/info/about/faq?name=impactfactor"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/10118.gif" alt="" border="0" hspace="3"></a></td><td id="topnav" align="right"><a href="http://www.scfbm.org/registration/technical.asp?process=default">Log on / register</a><br><a href="http://www.biomedcentral.com/">BioMed Central home</a> | <a href="http://www.biomedcentral.com/browse/journals/">Journals A-Z</a> | <a href="http://www.scfbm.org/feedback.asp?referedPage=%2Frender%2Frender%2Easp%3Fvol%3D2%26iss%3D1%26page%3D9%26type%3D%26item%3D%26pubtype%3Dcontent%26" onclick="popup(this.href,'Feedback',650,500); return false;">Feedback</a> | <a href="http://www.scfbm.org/support/">Support</a> | <a href="http://www.scfbm.org/my/">My details</a></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr valign="top"><td colspan="3" bgcolor="#003399"><div id="TopBar"><font color="#ffffff">&nbsp;<nobr><a class="xtoplinks" href="http://www.scfbm.org/home/">Home</a> | <a class="xtoplinks" href="http://www.scfbm.org/articles/browse.asp">Browse articles</a> | <a class="xtoplinks" href="http://www.scfbm.org/search/">Search</a> | <a class="xtoplinks" href="http://www.scfbm.org/weblinks/?link=General%20resources">Weblinks</a> | <a class="xtoplinks" id="submitArticle" href="http://www.scfbm.org/info/instructions/">Submit article</a> | <a class="xtoplinks" href="http://www.scfbm.org/my/">My Source Code for Biology and Medicine</a></nobr> | <a class="xtoplinks" href="http://www.scfbm.org/info/about/">About Source Code for Biology and Medicine</a></font></div></td></tr></tbody></table><div id="article"><script src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/fulltext.js" type="text/javascript"></script><table cellpadding="0" cellspacing="0" width="100%"><tbody valign="top"><tr><td id="leftcolumn"><ul style="position: static; top: 0px; left: 120px;" id="outline">
<li><a href="#">Top</a></li>
<li><a href="#IDA5AO32">Abstract</a></li>
<li><a href="#IDAFAO32">Introduction</a></li><li><a href="#IDAIAT22">Background</a></li><li class="long"><a href="#IDATBT22" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="Computational journal overview">Computational...</a></li>
<li class="long"><a href="#IDAILM22" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="Design and implementation of the BCJ">Design and im...</a></li>
<li class="long"><a href="#IDA2JN22" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="Computational journal experiment examples">Computational...</a></li>
<li><a href="#IDANPN22">Conclusion</a></li><li><a href="#IDARPN22">Competing interests</a></li><li><a href="#IDAVPN22">Authors' contributions</a></li><li class="long"><a href="#IDAZPN22" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="Appendix of computational environments">Appendix of c...</a></li>
<li><a href="#IDA30O22">Acknowledgements</a></li>
<li class="lowest"><a href="#refs">References</a></li>
</ul>
<table width="130"><tbody><tr><td>&nbsp;</td></tr></tbody></table></td><td id="content"><div id="navbox">
<table cellpadding="5" cellspacing="2">
<tbody><tr>
<td><a href="http://www.scfbm.org/home"><strong>Source Code for Biology and Medicine</strong></a><br><a href="http://www.scfbm.org/articles/browse.asp?volume=2">Volume 2</a><br></td>
</tr><tr><td><strong>Viewing options:</strong><ul>
<li>&nbsp;<a href="http://www.scfbm.org/content/2/1/9/abstract/">Abstract</a></li>
<li>&nbsp;<strong>Full text</strong></li>
<li>&nbsp;<a href="http://www.scfbm.org/content/pdf/1751-0473-2-9.pdf">PDF </a>
		(1.1MB)</li>
</ul><strong>Associated material:</strong><ul><li>&nbsp;<a href="http://www.scfbm.org/content/2/1/9/comments">Readers' comments</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/pubmed/18053179" onclick="largepopup(this.href,'pubmed');return false">PubMed record</a></li></ul><strong>Related literature:</strong><ul><li>&nbsp;Articles citing this article<div><a href="http://scholar.google.com/scholar?hl=en&amp;lr=&amp;safe=off&amp;c2coff=1&amp;q=link%3Ahttp://www.scfbm.org/content/2/1/9&amp;btnG=Search" onclick="largepopup(this.href,'on_google');return false">on Google Scholar</a></div><div><a href="http://www.biomedcentral.com/search/citedby/pmc/18053179">on PubMed Central</a></div></li>
<li>&nbsp;Other articles by authors<br><div><a href="javascript://" onclick="toggleauthors('authg','authbuttong'); return false"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/expand2.gif" id="authbuttong" alt="">on Google Scholar
				</a></div>
<div style="display: none;" id="authg"><a href="http://scholar.google.com/scholar?q=author%3A%22L+Feagan%22">Feagan L</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22J+Rohrer%22">Rohrer J</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22A+Garrett%22">Garrett A</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22H+Amthauer%22">Amthauer H</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22E+Komp%22">Komp E</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22D+Johnson%22">Johnson D</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22A+Hock%22">Hock A</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22T+Clark%22">Clark T</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22G+Lushington%22">Lushington G</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22G+Minden%22">Minden G</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22V+Frost%22">Frost V</a></div>
<div><a href="javascript://" onclick="toggleauthors('authpm','authbuttonpm'); return false"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/expand2.gif" id="authbuttonpm" alt="">on PubMed
				</a></div>
<div style="display: none;" id="authpm"><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Feagan_L&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Feagan L</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Rohrer_J&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Rohrer J</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Garrett_A&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Garrett A</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Amthauer_H&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Amthauer H</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Komp_E&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Komp E</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Johnson_D&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Johnson D</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Hock_A&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Hock A</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Clark_T&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Clark T</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Lushington_G&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Lushington G</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Minden_G&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Minden G</a><br><a href="http://www.scfbm.org/search/results.asp?db=pm&amp;terms=Frost_V&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Frost V</a></div>
</li>
<li>&nbsp;Related articles/pages<br><div><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.scfbm.org/content/2/1/9">on Google</a></div>
<div><a href="http://scholar.google.com/scholar?q=related:http://www.scfbm.org/content/2/1/9">on Google Scholar</a></div>
<div><a href="http://www.scfbm.org/pubmed/related/18053179">on PubMed</a></div>
</li></ul><strong>Tools:</strong><ul><li>&nbsp;<a href="http://www.scfbm.org/content/2/1/9/citation">Download citation(s)</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/content/download/xml/1751-0473-2-9.xml">Download XML</a></li><li>&nbsp;<a href="http://www.scfbm.org/content/2/1/9/email">Email to a friend</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/info/about/reprintsandperm">Order reprints</a></li><li>&nbsp;<a href="http://www.scfbm.org/content/2/1/9/postcomment">Post a comment</a></li>
<li>&nbsp;<a href="http://www.scfbm.org/alerts/">Sign up for article alerts</a></li></ul><script type="text/javascript">toggleauthors('authpm','authbuttonpm'); toggleauthors('authg','authbuttong'); toggleauthors('authos','authbuttonos'); toggleauthors('autha','authbuttona');</script><strong>Post to:</strong><br><ul class="iconlist">
<li><a href="http://www.citeulike.org/posturl?url=http://www.scfbm.org/content/2/1/9" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false"><img alt="" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/citeulink.gif"></a>&nbsp;<a href="http://www.citeulike.org/posturl?url=http://www.scfbm.org/content/2/1/9" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false">Citeulike				
				</a></li>
<li><a onclick="largepopup(this.href,'Connotea',800,600);return false" href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1751-0473-2-9"><img alt="" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/connotea.gif"></a>&nbsp;<a href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1751-0473-2-9" onclick="largepopup(this.href,'Connotea',800,600);return false">Connotea
					</a></li>
<li><a href="http://del.icio.us/post?url=http://www.scfbm.org/content/2/1/9&amp;title=Bioinformatics%20process%20management:%20information%20flow%20via%20a%20computational%20journal" onclick="largepopup(this.href,'Del.icio.us',800,600);return false"><img alt="" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/delicious.gif"></a>&nbsp;<a href="http://del.icio.us/post?url=http://www.scfbm.org/content/2/1/9&amp;title=Bioinformatics%20process%20management:%20information%20flow%20via%20a%20computational%20journal" onclick="largepopup(this.href,'Del.icio.us',800,600);return false">Del.icio.us</a></li>
<li><a href="http://digg.com/submit?phase=2&amp;url=http://www.scfbm.org/content/2/1/9&amp;title=Bioinformatics%20process%20management:%20information%20flow%20via%20a%20computational%20journal&amp;topic=general_sciences" onclick="largepopup(this.href,'Digg',800,600);return false"><img alt="" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/digg.gif"></a>&nbsp;<a href="http://digg.com/submit?phase=2&amp;url=http://www.scfbm.org/content/2/1/9&amp;title=Bioinformatics%20process%20management:%20information%20flow%20via%20a%20computational%20journal&amp;topic=general_sciences" onclick="largepopup(this.href,'Digg',800,600);return false">Digg
				</a></li>
<li><a href="http://www.facebook.com/sharer.php?u=http://www.scfbm.org/content/2/1/9" onclick="largepopup(this.href,'facebook',800,600);return false"><img alt="" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/facebook.gif"></a>&nbsp;<a href="http://www.facebook.com/sharer.php?u=http://www.scfbm.org/content/2/1/9" onclick="largepopup(this.href,'facebook',800,600);return false">Facebook</a></li>
</ul></td></tr></tbody></table>
</div><a name="maincontent"></a><div id="topmatter"><a name="text"></a><a href="http://www.biomedcentral.com/info/about/openaccess/"><img class="access" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/openaccess-large.gif" alt="Open Access" height="24" width="89"></a><a href="http://www.biomedcentral.com/info/about/mostviewed/"><img class="access" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/highlyaccessed-large.gif" alt="Highly Access" height="24" width="124"></a><!-- <rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns="http://purl.org/rss/1.0/">
<cc:Work rdf:about="http://www.scfbm.org/content/2/1/9">
<cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/>
</cc:Work>
<cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/">
<cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
<cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Notice"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/>
<cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/>
</cc:License>
<item rdf:about="http://www.scfbm.org/content/2/1/9">
<title>Bioinformatics process management: information flow via a computational journal</title>
<dc:title>Bioinformatics process management: information flow via a computational journal</dc:title>
<dc:creator>Feagan, Lance</dc:creator>
<dc:creator>Rohrer, Justin</dc:creator>
<dc:creator>Garrett, Alexander</dc:creator>
<dc:creator>Amthauer, Heather</dc:creator>
<dc:creator>Komp, Ed</dc:creator>
<dc:creator>Johnson, David</dc:creator>
<dc:creator>Hock, Adam</dc:creator>
<dc:creator>Clark, Terry</dc:creator>
<dc:creator>Lushington, Gerald</dc:creator>
<dc:creator>Minden, Gary</dc:creator>
<dc:creator>Frost, Victor</dc:creator>
<dc:identifier>info:doi/10.1186/1751-0473-2-9</dc:identifier>
<dc:identifier>info:pmid/18053179</dc:identifier>
<dc:source>Source Code for Biology and Medicine 2007, 2:9</dc:source>
<dc:date>2007-12-03</dc:date>
<prism:publicationName>Source Code for Biology and Medicine</prism:publicationName>
<prism:publicationDate>2007-12-03</prism:publicationDate>
<prism:volume>2</prism:volume>
<prism:number>1</prism:number>
<prism:section>Research</prism:section>
<prism:startingPage>9</prism:startingPage>
</item>
</rdf:RDF> --><span class="articletype">Research</span><h1>Bioinformatics process management: information flow via a computational journal</h1><div class="singleins"><p class="authors"><strong>Lance Feagan</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a>, <strong>Justin Rohrer</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a>, <strong>Alexander Garrett</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a>, <strong>Heather Amthauer</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a>, <strong>Ed Komp</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a>, <strong>David Johnson</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a>, <strong>Adam Hock</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a>, <strong>Terry Clark</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a>, <strong>Gerald Lushington</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a>, <strong>Gary Minden</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email_002.gif" alt="email" title="Email"></a> and <strong>Victor Frost</strong><sup>* </sup> <a href="http://www.scfbm.org/registration/technical.asp?process=default&amp;msg=ce"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email-ca_002.gif" title="Corresponding author" alt="email"></a></p><p>Information and Telecommunication Technology Center, University of Kansas, Lawrence, Kansas, USA</p></div><p><span class="pseudotabau"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email.gif" border="0"> author email</span><span class="pseudotabau"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/email-ca.gif"> corresponding author email</span><span class="pseudotabau">* Contributed equally</span></p><p><em>Source Code for Biology and Medicine</em> 2007,
			 <strong>2</strong><strong>:</strong>9<span class="pseudotab">doi:10.1186/1751-0473-2-9</span></p><p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.scfbm.org/content/2/1/9">http://www.scfbm.org/content/2/1/9</a></p>
<table cellpadding="0" cellspacing="0">
<tbody>
<tr>
<td>Received:</td>
<td>21&nbsp;January&nbsp;2007</td>
</tr>
<tr>
<td>Accepted:</td>
<td>3&nbsp;December&nbsp;2007</td>
</tr>
<tr>
<td>Published:</td>
<td>3&nbsp;December&nbsp;2007</td>
</tr>
</tbody>
</table>
<p>
						©
						 2007 Feagan et al; licensee BioMed Central Ltd.<br>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><a name="IDA5AO32"></a><h3>Abstract</h3><p>This
paper presents the Bioinformatics Computational Journal (BCJ), a
framework for conducting and managing computational experiments in
bioinformatics and computational biology. These experiments often
involve series of computations, data searches, filters, and annotations
which can benefit from a structured environment. Systems to manage
computational experiments exist, ranging from libraries with standard
data models to elaborate schemes to chain together input and output
between applications. Yet, although such frameworks are available,
their use is not widespread–<em>ad hoc </em>scripts are often required
to bind applications together. The BCJ explores another solution to
this problem through a computer based environment suitable for on-site
use, which builds on the traditional laboratory notebook paradigm. It
provides an intuitive, extensible paradigm designed for expressive
composition of applications. Extensive features facilitate sharing
data, computational methods, and entire experiments. By focusing on the
bioinformatics and computational biology domain, the scope of the
computational framework was narrowed, permitting us to implement a
capable set of features for this domain. This report discusses the
features determined critical by our system and other projects, along
with design issues. We illustrate the use of our implementation of the
BCJ on two domain-specific examples.</p><a name="IDAFAO32"></a><h3>Introduction</h3><p>The
Bioinformatics Computational Journal (BCJ) is an extensible environment
that integrates computational resources, methods, and data.
Bioinformatics and computational biology span a wide variety of
applications ranging from interpretation of gene expression data to
protein structure prediction (For brevity, in this report we often
describe <em>bioinformatics and computational biology </em>as
bioinformatics.). However, within this range of applications there is
considerable common ground that is pivotal to a domain-oriented
approach. Many bioinformatics applications are centered on the rapidly
growing sequence data archived at national centers <a name="IDAKCO32"></a>[<a href="#B1">1</a>].
These data can be seen as enablers of bioinformatics approaches. As a
result, standard approaches to process these data have appeared, but
specifics of the applications can vary widely. Foremost in common is
sequence alignment <a name="IDAFCO32"></a>[<a href="#B2">2</a>]. Many other applications are widely used (for example, see EMBOSS <a name="IDACAT22"></a>[<a href="#B3">3</a>]) and more are under development.</p>
<p>The BCJ is designed around characteristics of bioinformatics
research, which often involves: (1) stepwise application of a series of
methods, (2) re-application of the series of methods in search and
retry iterations using different parameters, (3) development of new
procedures and combinations in the rapidly growing discipline, (4) an
open environment crucially dependent on web-based application servers
and public databases, and (5) rapidly growing data collections and
evolving vocabularies. The volume of data and the number of
computational experiments create significant processing and management
complications.</p>
<p>The BCJ framework reported here addresses these problems through
data and process management, with mechanisms for experiment
specification and composing computations, tracking data provenance and
retaining the context of computations. Activities such as entering and
locating data, applications, and results are facilitated by the
framework. The BCJ fosters collaboration among project teams through
structured privacy mechanisms and naming constructs, increasing
productivity in computational experiments for both computationally
expert and non-expert researchers.</p><a name="IDAIAT22"></a><h3>Background</h3><p>A
computational environment is a coherent interface to a set of
applications that integrates them under a single paradigm. The design
goal is to simplify the use and management of applications and their
output. Here, we are concerned about bioinformatics applications. Many
bioinformatics applications are based on scanning sequences for
features relative to some database. This activity is a significant
application in the field spurred by an exponentially growing sequence
collection archived at national centers, now at around 65 billion bases
at the National Center for Biotechnology Information (NCBI) <a name="IDAOAT22"></a>[<a href="#B1">1</a>]. Numerous applications are standard in these studies, for example, BLAST for sequence alignment <a name="IDATAT22"></a>[<a href="#B4">4</a>], Markov models for profiling related sequences <a name="IDAYAT22"></a>[<a href="#B5">5</a>], and GrailEXP for searching for predicting genome features <a name="IDA3AT22"></a>[<a href="#B6">6</a>].
These and other programs are often integrated together in different
ways, raising the challenge in bioinformatics of managing trails of
data and methods applied to them. Often programs are composed together
in <em>ad hoc </em>ways. This can lead to errors in interpreting results, lost time in rewriting the software <em>glue </em>to
compose the programs, and overall poor solutions. The problem is not
insignificant, and includes scientists from numerous disciplines
involved in the interdisciplinary bioinformatics field. There are
intermediate solutions to unified approaches to using bioinformatics
software such as the somewhat standardized interfaces and data models
provided by Bioperl and Biojava <a name="IDAGBT22"></a>[<a href="#B4">4</a>], but these appropriately do not attempt a user paradigm.</p>
<p>The BCJ is geared for bioinformatics applications where computations
are often organized as workflows. A workflow consists of a series of
computations connected by a flow of information between them (cf.
task/control-based workflows <a name="IDANBT22"></a>[<a href="#B5">5</a>]).
Computational experiments are composed of workflows. As in the
laboratory, computations are often repeated or modified to create new
experiments to test additional hypotheses. When considering
computational environments, there are several distinct factors to be
considered: the execution environment, computation design, and the
management of experiments and data.</p>
<p>A number of execution environment issues need to be considered.
First, the target computing platform may be a standalone workstation, a
local-area cluster, or a wide-area grid. In the wide-area grid
scenario, integration of resources may be difficult because of
decentralized security and administrative tasks. Second, the native
execution environment may support simple, linear execution pipelines
where the output of each stage drives only one input, or more
complicated execution graphs where one output can branch to provide
multiple inputs. Iteration and conditional branching are challenging
execution features to include because there are many iteration and
conditional branching strategies to choose from. Third, extension of
the environment may be done with internal tools that are part of the
(target) execution environment, or with external tools that a user runs
on his/her local workstation separately from the execution environment.
Fourth, web services can form the basis for an execution environment.</p>
<p>The graphical user interface paradigm for specifying the experiment
design is central to the usability of the environment. Ease of use is
achieved with an intuitive and consistent interface to applications,
data, and results. Experiment-management issues including archiving,
searching, modifying and retrieving existing workflows, searching
stored data, searching annotated workflows/experiments, and the ability
to export data to and import data from external sources impact the
usability of the environment. Finally, other miscellaneous features,
such as access controls on data, data provenance, experiment
repeatability, collaboration, and annotation of experiments are
important.</p><a name="IDATBT22"></a><h3>Computational journal overview</h3><p>The
BCJ documents the entire computational experiment. The BCJ has
mechanisms to record the complete context of the experiment to enable
its review and re-execution. Furthermore, by recording this
information, the ability to interpret results is enhanced. Although the
range of computational experiments in bioinformatics research is
extensive, the experimental processes have much in common. We leveraged
the similarities to arrive at our framework for this domain. We chose
the traditional laboratory notebook, long used by biologists to record
wet lab experiments, as the base paradigm. However, the BCJ is distinct
from a lab notebook. The BCJ's task is limited to providing support for
managing and performing computational experiments with a complete
electronic record to enable advanced functions like searching and
collaboration. From the user perspective, the top-level structure of
the computational journal is a collection of journal entries and
relationships among these entries. More formally, an <em>entry </em>is the basic unit of information provided by the end user for the BCJ. Each entry is characterized by a single <em>content type </em>that
identifies the format of this information. The number of content types
is user extensible, allowing the BCJ to support new data formats and
programs. Relations among entries provide an organizational structure
to enable tracking entry dependencies, e.g., entry A "is an execution
input" for entry B. These relationships are defined automatically as
the entries are created and then they are maintained by the BCJ
environment.</p>
<p>Our implementation of the BCJ uses the Eclipse infrastructure <a name="IDA5BT22"></a>[<a href="#B7">7</a>].
The Eclipse project is an open source software framework that provides
a development platform for building applications. Eclipse is based on a
plug-in framework that supports the requirementsdiscussed above.
Eclipse provides a rigorously tested, refined, and documented
infrastructure for the management of plug-ins. In addition, we were
able touse several Eclipse sub-projects, such as the Graphical Editor
Framework (GEF), to provide infrastructure for BCJ – specific editors.
By using Eclipse, we have been able to focus directly on the
development of a set of plug-ins and functionality particular to theBCJ.</p>
<p>The BCJ is composed of a two-tier architecture with a single server
and multiple clients. The server is located on a cluster while clients
run a multi-platform application supported on Windows, Linux, or OS X.
The security model provides encryption between the client's computer,
the BCJ database, and the cluster, as well as the concept of
group-based access control. Workflows are defined graphically with the
client, and the requested computations are loaded into the
computational resources for execution. Tools can be added using XML and
an execution wrapper script (for example, specific bioinformatics
applications like BLAST). The BCJ, similar to Wildfire (see Appendix),
offers a drag-and-drop interface for workflow creation (Figure <a name="IDAFCT22"></a><a onclick="popup('/content/2/1/9/figure/F1','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F1">1</a>). The BCJ does not provide iterations and conditional branches, but these are planned.</p><div class="figs"><div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F1','F1',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F1"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-1.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 1.</strong></a> Sample BCJ Session.</p></div></div><p>In
addition to providing program integration, the BCJ manages workflows,
data, and experiment definitions. Data that are input to and output
from workflows are stored and linked with the corresponding execution
specification, thus tracking all information related to a computational
experiment. For example, users can search for experiments that used
specific data as an input or created specific data as an output. Each
execution of a workflow can be identified, along with the input,
output, and parameters. There are also options to search based on
content type, owner, journal, and user annotations of workflows and
experiments to facilitate experiment creation and management. The BCJ
can import data from external sources such as local files, program
output, or web resources; and it can export data to a file, for other
use.</p>
<p>The BCJ provides four of the five types of provenance described in the taxonomy of data provenance techniques developed in <a name="IDAXCT22"></a>[<a href="#B8">8</a>].
These four types are: (1) audit trail (complete traces of the execution
can be stored); (2) replication recipes (the steps, parameters, and
other transformations are recorded in experiments); (3) attribution
(authentication and database control over ownership metadata ensures
proper pedigree can be established); and (4) informational (annotations
of varying types along with indexing of data). (The fifth type of
provenance is data quality, i.e., fitness of the data for an
application.) Data, workflows, and experiments can be shared through
group-level access control lists. Scientists working on the same
project can publish their work to other users through the BCJ.
Annotation of experiments, workflow, and data can be textual or may be
of other types, for instance, a Microsoft Word<sup>® </sup>document or a picture of a plot related to an experiment.</p>
<h4>Comparisons</h4>
<p>Four aspects were used to analyze the environments discussed above:
execution environment, workflow design, data management, and additional
features. A comparison based on these aspects with other computations
is discussed below. (See the appendix for introductions to the systems
discussed in this section: PathPort, Pegasys, PISE, BioCoRE, MIGenAS,
Taverna, and Wildfire.)</p>
<h4>Execution</h4><p>Most surveyed solutions were designed to be run
locally or on the grid–few were designed for cluster-based systems.
Neither PathPort nor PISE were implemented for cluster-based execution.
MIGenAS was written to be run on only one specific cluster. BioCoRE,
Pegasys, Wildfire, Taverna, and BCJ allowed execution on a local
cluster. Only Pegasys, Wildfire, Taverna, and the BCJ offer workflow
capabilities. PISE and MIGenAS allow for linking program input and
output, but neither goes beyond a serial pipeline. Furthermore,
BioCoRE, MIGenAS, and Wildfire do not offer the ability to add tools to
the server's execution environment, while only PathPort, Wildfire, and
the BCJ allow client-side, external tools to be added. Only the BCJ and
PathPort allow for both internal and external extensibility. Table <a name="IDAGDT22"></a><a onclick="popup('/content/2/1/9/table/T1','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T1">1</a> summarizes the execution capabilities of the environments.</p><div class="figs"><div class="table"><p><a onclick="popup('/content/2/1/9/table/T1','T1',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T1"><strong>Table 1.</strong></a> Execution capabilities of analyzed systems</p></div></div><h4>Workflow Design</h4><p>All
of the surveyed solutions provide a GUI, however only Pegasys,
Wildfire, and the BCJ provide an interactive, drag-and-drop interface.
Unlike the BCJ, all surveyed solutions use input forms that are loosely
coupled to the workflow on a separate page from the workflow
specification. The input of parameters for each element in the BCJ is
closely coupled to the associated workflow, as is discussed below.
Wildfire and Taverna provide iterations and branching based on user
specifications. Wildfire iterations are essentially while loops based
on user-supplied imperative programs; Taverna iterations are defined
graphically. Table <a name="IDAJJT22"></a><a onclick="popup('/content/2/1/9/table/T2','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T2">2</a> summarizes the workflow design capabilities of these environments.</p><div class="figs"><div class="table"><p><a onclick="popup('/content/2/1/9/table/T2','T2',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T2"><strong>Table 2.</strong></a> Workflow design of analyzed systems</p></div></div><h4>Data Management</h4><p>PathPort,
BioCoRE, and the BCJ are the only environments that provide data
management. Additionally, only Pegasys, Wildfire, Taverna, and the BCJ
provide workflow management. The combination of workflow and data
management increases the usefulness of the environment. Table <a name="IDATNT22"></a><a onclick="popup('/content/2/1/9/table/T3','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T3">3</a> summarizes the data management capabilities.</p><div class="figs"><div class="table"><p><a onclick="popup('/content/2/1/9/table/T3','T3',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T3"><strong>Table 3.</strong></a> Data management properties of analyzed systems</p></div></div><h4>Additional Features</h4><p>Of
the systems reviewed, only the BCJ and Taverna provided the ability to
track the provenance of data created within it and provided the ability
to repeat an experiment under the same conditions. Note that for the
BCJ to support experiment repeatability, program versioning needs to be
implemented. This provides the ability to re-run computational
experiments. The BCJ and BioCoRE encrypt information exchanges between
the client and the server. Additionally, the collaborative features in
several reviewed systems consisted of data sharing by exporting
data/workflows to a shared file and sharing the file directly.
Collaboration at the system level is offered by the BCJ. BioCoRE
offered many collaborative features, including message boards and
chatting, while the BCJ natively handled collaboration through the
sharing of data, workflows, and experiment journals. Annotation of data
and experiments, like in BioCoRE, Wildfire, Taverna, and the BCJ, is a
valuable feature and is an integral part of workflow and data
management. Table <a name="IDAP2T22"></a><a onclick="popup('/content/2/1/9/table/T4','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T4">4</a> summarizes these features.</p><div class="figs"><div class="table"><p><a onclick="popup('/content/2/1/9/table/T4','T4',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T4"><strong>Table 4.</strong></a> Additional features of analyzed systems</p></div></div><a name="IDAILM22"></a><h3>Design and implementation of the BCJ</h3><h4>Abstract data interface</h4>
<p>There has been a proliferation of specialized tools, interfaces, and <em>ad hoc </em>scripts
to handle specific, repetitive tasks in narrow domains. The generality
of these tools is sometimes limited by implicit data formatting,
organization, and naming conventions. The presence of these low-level
details can impede a system's usefulness. In the BCJ environment, the
interface has been designed to shield the user from these details;
every data element is simply an entry, regardless of its specific
content type, providing a uniform, high-level abstraction to the user.
Users are insulated from tedious low-level abstractions such as files
systems and their computational hardware resources.</p>
<p>The BCJ environment maintains every entry created by every user. An
entry can be a bioinformatics application, a workflow, an experiment,
or a data set. The underlying framework collects and maintains a set of
meta-data for each entry, including its name, author, creation and
modification dates, and content type. Every application that plugs into
the BCJ environment is an entry and is accessible through the standard
entry interface. Many BCJ functions manipulate only meta-data fields of
an entry, other BCJ functions are designed to manipulate only a
specific content type, and are enabled only when an entry containing
this content type is selected.</p>
<p>The BCJ environment provides a variety of <em>navigators</em>. These
are functions that provide a hierarchical view of a collection of
entries. Navigators establish a virtual organization of the entries.
This allows the user to see the data organized to match a specific
activity, by selecting the navigator most appropriate for the task. The
most basic navigator, the Journal navigator, displays entries according
to the journal in which entries are created. This navigator provides a
view of entries, similar to a common file hierarchy with journals
corresponding to file directories. Other navigators display entries
related to a specific entry. For example, the dependency navigator
displays entries that depend on the selected entry.</p>
<p>The BCJ can be extended through the addition of navigators. As new
tools and techniques are incorporated into the environment, it is
likely that some will implement novel data organization schemes
requiring a new navigator.</p>
<p>Shared access to all entries must be balanced with the individual's
need for security and privacy. The BCJ security model only allows the
creator of an entry to modify it. This simple model is consistent with
the larger computational journal paradigm. The journal is primarily a
record of experimental steps. Once completed, a step may no longer be
modified, however, it may be followed by additional experimental steps
to refine results and investigate variations on the base experiment.
This characteristic aids in maintaining the <em>replication recipe </em>provenance.
If another user wants to make changes to data or an experiment created
by another user, the user may create a new entry as a copy of the
original one using the SaveAs operation, thereby maintaining data
pedigree.</p>
<p>Read access to every entry is controlled by the <em>group access </em>attribute
of the entry, where a group is a set of users. Only members of the
associated group are able to view an entry and its contents. All manner
of access to an entry is controlled by this attribute. Search results
will only contain entries for which the initiator has group access. The
BCJ environment provides administrative tools to manage groups.</p>
<p>The BCJ environment provides another form of access control. Every entry contains a <em>committed </em>attribute.
Once set, the content of the entry may no longer be modified by any
user, including the entry's creator. The committed attribute of an
entry can never be reset. The committed attribute is fundamental both
to encourage sharing among users and to ensure reproducibility of
experimental results.</p>
<h4>Single framework for diverse programs</h4>
<p>A key feature of bioinformatics research is the breadth of programs
that are being used by team members to address a shared goal.
Frequently, specialized user interfaces are constructed around a set of
closely related programs to facilitate their usage and to help perform
repetitive tasks. However, integration of the process and results among
these specialized interfaces is often difficult. The BCJ fundamental
paradigm leverages the similar structures of computational experiments
in biological experiments. These often are chained computations where
the output of one is the input of another. However, the program's data
formats and graphical interfaces can vary widely. Thus, within its
common framework, the BCJ provides the flexibility for the problem –
specific components. The BCJ environment provides common support for
the <em>experimental process</em>, and <em>flexibility </em>and <em>extensibility </em>in the framework in these three key areas.</p>
<p>Designing a database schema broad enough to handle all forms of
biological data can be a daunting undertaking. Specialized systems such
as the Genomic Unified Schema (GUS) are designed for this <a name="IDASMM22"></a>[<a href="#B9">9</a>].
For the BCJ role in performing and recording computational experiments,
this generality is not a critical requirement. We have chosen a data
model that simply stores the content of an entry as an uninterpreted
stream of bytes. The BCJ domain does not attempt a structured
biological database schema. Other systems exist for managing this
problem with issues discussed elsewhere <a name="IDAXMM22"></a>[<a href="#B9">9</a>].
Instead, the BCJ uses a simple generic entry. Every entry is stored in
the same table with a very simple schema. This schema consists of a
fixed number of fields such as name, author, creation time, data type
information, and a pointer to actual content which is stored as a
vector of bytes. With this simple structure, support for a new data
format simply requires the definition of a new content type.</p>
<p>No update or extension of a database schema is required. All BCJ
environment functions that operate on the abstract entry interface,
such as navigators and search tools, implicitly handle entries with
content in the new format.</p>
<p>The BCJ supports a wide variety of raw data types. This is achieved in the storage architecture through an extensible set of <em>content types</em>. There are several parallels between requirements for Internet browsers and the BCJ environment:</p>
<p>1. The number of content types will grow after the product is
delivered; and reissuing the product to handle new content type(s) is
undesirable;</p>
<p>2. Each user does not require handlers for all content types;</p>
<p>3. There may be multiple handlers for the same content type;</p>
<p>4. It is desirable to distribute development of content
type-specific handlers to external groups with specific interest and
knowledge related to the content type.</p>
<p>The content types are presented and manipulated through a plug-in
framework that has become ubiquitous in Internet browsers. Although a
BCJ environment may contain many plug-ins, the user sees a relatively
simple, coherent interface (Figure <a name="IDAGNM22"></a><a onclick="popup('/content/2/1/9/figure/F1','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F1">1</a>).
The functionality that a user interface plug-in provides fits into one
of four categories where each member of a category adheres to a common
interface: entry navigation, entry editing, auxiliary views, and
experiment execution.</p>
<p>From the user perspective, the BCJ provides a mechanism to navigate
among the potentially large number of entries. Each navigator provides
a virtual organization of the entries, based on one or more relations
among the entries. This allows the user to see the data organized to
match a specific activity, by selecting the navigator most appropriate
for the task. At creation time, each entry is assigned a parent
journal. This provides an immediate and natural hierarchical view of
all entries. This journal relationship provides a view of the entries
that closely resembles the file directory hierarchy familiar to all
computer users. This is the most basic of the BCJ navigators.
Graphically, it uses a tree-list display that is widely used on
computers to present such hierarchies, but with annotations that
provide more detail than a basic hierarchical file system interface.</p>
<p>When an entry is selected, a plug-in associated with the entry's
content type is selected to edit or view the content. Graphical editors
are provided for workflow definitions and experiment definitions. When
a new content type is defined in the BCJ environment, it is natural to
incorporate a new editor plug-in tailored for the type. Many of the
program inputs and outputs in the bioinformatics domain are textual.
For these types, a generic text editor plug-in can be used. The editing
area in the BCJ environment is subdivided into multiple panes so that
the user can conveniently compare the content of two or more entries.</p>
<p>Some data formats, particularly binary data formats, are supported
by sophisticated and optimized viewer programs. For example,
three-dimensional graphical viewers are available for molecular
dynamics simulation outputs. It is not practical to convert these
programs to operate directly inside the framework. The BCJ environment
provides an interface that allows a user to associate an external
viewer, a program installed on his/her client machine, as the default
viewer for a specified content type. Whenever an entry of one of these
content types is selected for viewing, the BCJ environment
automatically generates a local copy of the data on the client machine
and invokes the associated program with this input.</p>
<p>In addition to editors and navigators, the BCJ environment provides
a variety of additional views that present auxiliary information to
support these primary tasks, such as the Queue Manager view, which
displays the progress of currently executing experiments.</p>
<p>Precise descriptions of BCJ programs are kept in XML documents. This description is called a <em>resource definition </em>in the BCJ environment. The Document Type Definition (DTD) for the XML data can be found in <a name="IDATNM22"></a>[<a href="#B10">10</a>]. The BCJ environment currently contains about sixty programs/resource definitions (see Table <a name="IDAXNM22"></a><a onclick="popup('/content/2/1/9/table/T5','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T5">5</a>). Many resource definitions were derived from the XML descriptions for the corresponding programs in PISE <a name="IDA2NM22"></a>[<a href="#B11">11</a>].</p><div class="figs"><div class="table"><p><a onclick="popup('/content/2/1/9/table/T5','T5',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/table/T5"><strong>Table 5.</strong></a> Resources currently defined in BCJ</p></div></div><h4>Graphical interface for experiment definition</h4>
<p>Experiments are defined with simple block diagrams in a hierarchical
fashion. Each block represents a program, or sequence of programs, to
be executed. Input and output connections are made between blocks. In
the BCJ environment, a block diagram is called a <em>workflow</em>. Figure <a name="IDAJIN22"></a><a onclick="popup('/content/2/1/9/figure/F2','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F2">2</a>
is an example of a simple workflow. Workflows provide a convenient and
intuitive way for any user to define a customized task. Users are not
required to be familiar with programming languages to define complex
tasks. Once defined, a workflow may be reused. Workflows are just
another entry content type, so they can be shared among all BCJ users.
Experiments can be joined by connecting resources to form workflows. A
data source can be used as an input to one BCJ resource with the output
directed to another resource. The BCJ also provides a mechanism to
annotate workflows. Figure <a name="IDANIN22"></a><a onclick="popup('/content/2/1/9/figure/F3','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F3">3</a> provides an example of a simple workflow definition.</p><div class="figs"><div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F2','F2',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F2"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-2.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 2.</strong></a> Complete Experimental Workflow.</p></div>
<div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F3','F3',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F3"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-3.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 3.</strong></a> HMMER Workflow.</p></div></div><h4>Management of data provenance</h4>
<p>The details of the experimental process are nearly as important as
the experimental results themselves. The BCJ environment collects and
maintains this information transparently, providing organizational
views of data and experiments that focus on data provenance. With the
provenance data maintained in the BCJ environment users can:</p>
<p>1. Review all the details of an experiment at anytime,</p>
<p>2. Regenerate the results by re-executing the experiment,</p>
<p>3. Run similar experiments and compare with the original results.</p>
<p>When reviewing an experiment, the dependency navigator in the BCJ
environment is particularly useful. The dependency navigator collects
the set of entries upon which a selected entry is dependent, in a
concise hierarchical view. The first level of the hierarchy contains
the immediate dependencies of the selected entry. The second level of
the hierarchy contains the dependencies for each of these entries, and
so on. Figure <a name="IDAOJN22"></a><a onclick="popup('/content/2/1/9/figure/F4','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F4">4</a> displays a sample dependency view.</p><div class="figs"><div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F4','F4',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F4"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-4.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 4.</strong></a> Dependency View.</p></div></div><a name="IDA2JN22"></a><h3>Computational journal experiment examples</h3><p>In
this section we describe two computational experiments performed in the
BCJ environment, highlighting many of the features described above. The
two examples were chosen from significantly different domains in order
to demonstrate that the BCJ environment is sufficiently flexible to
support a wide variety of applications. A user manual for the BCJ can
be found in <a name="IDACKN22"></a>[<a href="#B12">12</a>].</p>
<h4>Sequence-based investigation</h4>
<p>To demonstrate sequence analysis in the BCJ environment, a simple
example experiment was implemented that tries to identify a protein
that may be a potential target for drug therapy in treating malaria. If
there is sufficient divergence between some protein of the parasite and
the host, then inhibition of the parasite protein is a potential target
for drug therapy. In this experiment, we are interested in the
divergence between the mosquito <em>Plasmodium falciparum </em>casein
kinase and its vertebrate hosts. There are three applications used by
the experiment: BLAST, CLUSTALW, and HMMER. The experiment involves the
following computations: (1) identify similar proteins to the human
casein kinase enzyme using BLAST, (2) create a global alignment using
the sequences of the similar proteins that were extracted from the
BLAST report using CLUSTALW, (3) construct a hidden Markov model (HMM)
based on this multiple alignment, using components of the HMMER program
suite, (4) run the model against the proteins in the <em>Plasmodium falciparum </em>genomic database (PlasmoDB) to identify proteins that may be functionally equivalent to human casein kinase.</p>
<p>In this experiment, we build two workflows. The first workflow
contains the HMMER components. The second workflow, which will contain
the complete experiment, will incorporate the HMMER workflow and the
BLAST and CLUSTALW components.</p>
<p>The HMMER <a name="IDATKN22"></a>[<a href="#B9">9</a>] program suite
for biosequence analysis using profile HMM contains several components
that are frequently used in a fixed order. In this workflow, we define
a reusable component that may find value in other studies involving
HMM. This workflow will construct an HMM based on the multiple
alignment input, calibrate the model, and finally search a sequence
database using this model.</p>
<p>To increase the flexibility of this computational component, the
database is provided as a second input. As a design alternative, we
could have chosen to specifically include PlasmoDB in this workflow (as
a data source block) and have a single input to this workflow. A
picture of the final workflow can be seen in Figure <a name="IDAZKN22"></a><a onclick="popup('/content/2/1/9/figure/F3','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F3">3</a>.</p>
<p>This workflow could be useful to a wide variety of researchers and
does not contain any sensitive data or novel computation techniques,
thus we set the group access control for this entry to ALL, so any user
in the BCJ environment will have access to this workflow. To make the
entry easier for others to find we could place the entry in a journal
designated for shared workflows, and/or add an annotation briefly
describing its functionality, so that other users can locate it with a
search query.</p>
<p>In the complete workflow, we use the previously defined HMMER workflow as a <em>resource</em>.
It also includes BLAST, to find proteins similar to the input protein,
and CLUSTALW, to perform a multiple alignment of these similar
proteins. The CLUSTALW program does not accept input in the format
generated by the BLAST program, so the block, Extract Sequence, was
inserted between them to perform the appropriate data conversion.
Resource definitions include type information for each of their ports;
the Workflow Editor does not permit connections between ports with
incompatible types. These constraints help users construct only
meaningful and valid workflows. Resources such as Extract Sequences
serve the necessary role of automatic resolution of data reformatting.
In addition, users can define new resource definitions that perform
data format conversions to meet their specific requirements. The
completed workflow is shown in Figure <a name="IDACLN22"></a><a onclick="popup('/content/2/1/9/figure/F2','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F2">2</a>.</p>
<p>To define an experiment based on this workflow, the user provides a
protein sequence input and a database to be searched with HMMER. For
this example, Amino Acid Sequence was used, for the input is human
casein kinase. The Search DB input is PlasmoDB. The output file,
Related Proteins, will contain the sequences from PlasmoDB that are
most similar to the profile HMM built from the human casein kinase
related proteins. Figure <a name="IDAILN22"></a><a onclick="popup('/content/2/1/9/figure/F5','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F5">5</a> shows the completed experiment definition.</p><div class="figs"><div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F5','F5',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F5"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-5.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 5.</strong></a> Experiment Definition.</p></div></div><p>When
defining the experiment, the user is queried to specify access control
for the experiment definition. For example, the user may want to
restrict any other access to the experiment and results, by setting the
access control to NONE, pending validation of results. Later, the owner
can modify access control to allow access by a wider range of users. At
the completion of the experiment definition, a collection of jobs to
perform the tasks defined by the workflow are submitted to the
computational cluster for execution. The user tracks the progress of
the experiment using the Queue Manager view.</p>
<p>The output entry generated by the experiment contains the top
sequences from PlasmoDB that match the profile created by HMMER. From
the results (Figure <a name="IDAZLN22"></a><a onclick="popup('/content/2/1/9/figure/F6','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F6">6</a>), we see that the top result has the gene ID of PF11_0377. This gene happens to encode <em>P. falciparum's </em>casein kinase 1. Based on these results, we see that this kinase would not be a good candidate for drug therapy.</p><div class="figs"><div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F6','F6',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F6"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-6.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 6.</strong></a> Experiment Results.</p></div></div><p>The user can summarize what has been learned and annotate the experiment as shown in Figure <a name="IDALMN22"></a><a onclick="popup('/content/2/1/9/figure/F7','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F7">7</a>.
These annotations communicate reviews to other users and provide a log
for the author. Annotations also enable search capabilities, since
neither the experiment definition nor the output entry directly contain
the lessons learned from the experiment.</p><div class="figs"><div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F7','F7',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F7"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-7.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 7.</strong></a> Adding an Annotation to the Experiment.</p></div></div><p>If
the researcher has other candidate proteins to study, another
experiment can reuse this workflow definition. Here, the user simply
repeats the experiment definition step, and supplies another protein
for the Amino Acid Sequence input. The BCJ environment emphasizes that
providing a different sequence input is a new experiment definition,
rather than a modification of the original experiment. It will be
useful to keep the original experiment, even though it was not
successful in locating a promising drug target. These results may be
helpful months later if someone asks, "Have you considered casein
kinase as a possible target for malaria?" The BCJ environment provides
three alternatives to find an answer for this question:</p>
<p>1. The dependent viewer in the BCJ environment is particularly
helpful for answering such questions. The researcher could select the
casein kinase sequence entry, and find all experiments in which it is
used as an input;</p>
<p>2. Alternatively, the researcher could use the dependency viewer,
which is effectively the inverse of the dependent viewer. With this
viewer, the user can select the workflow definition and immediately
find all experiments, based on the workflow, that have been executed;</p>
<p>3. Perform a content-based search to locate an annotation that summarizes the result of a previous experiment.</p>
<h4>Molecular dynamics experiment</h4>
<p>Typically, molecular dynamics (MD) simulations involve a large
number of simulations, where each varies from the previous only in the
input data. The organization of the computations is often tailored to
some specific application such as drug design or methodology
development. Analysis of the simulation results will vary widely
between simulation goal and investigators' styles. In addition,
although the simulations are repetitive, there are numerous places
during the study where interactions with the process and output are
needed. Here, we demonstrate the use of the BCJ on such a
representative MD simulation. For this experiment, we use the GROMACS
package <a name="IDADNN22"></a>[<a href="#B13">13</a>] and an example based on a tutorial <a name="IDAINN22"></a>[<a href="#B14">14</a>].</p>
<p>There are five major steps in this experiment: (1) converting the
protein structure from a common database format, PDB (Protein Data
Bank), into a format suitable for GROMACS, (2) minimizing the energy of
the system, (3) solvating the protein in a simulation box with water,
(4) performing an MD simulation from the initial state, and (5)
analyzing the simulation results. (Often, proteins are solvated then
minimized, but for purposes of demonstration, the protein is minimized
first.)</p>
<p>Each step can be run as an independent experiment with the outputs
of each step appearing as entries in the user's journal. This stepwise
procedure is useful for reviewing intermediate results. Alternatively,
the steps can be incorporated into a single workflow, which is useful
for a tested simulation procedure. The current example represents an
intermediate approach in order to demonstrate aspects of the BCJ
environment; two sub-workflows are combined into another workflow to
perform the end-to-end experiment.</p>
<p>The workflow "Energy Min" shown in Figure <a name="IDAQNN22"></a><a onclick="popup('/content/2/1/9/figure/F1','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F1">1</a>
combines the first two steps of the procedure, since these two steps
are standard initialization steps. This workflow has been designed with
reuse in mind. The precise parameter settings for minimization may
depend on the simulated molecule (one of the inputs); thus, a second
input is defined for this workflow "MD params," which will translate
the simulation parameters into GROMACS format. In addition, several
output ports are defined for this workflow that are not specifically
required in the example here. However, these additional outputs may be
of use in another application that chooses to reuse this workflow.</p>
<p>The workflow "MD Sim" shown in Figure <a name="IDAWNN22"></a><a onclick="popup('/content/2/1/9/figure/F8','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F8">8</a>
combines steps three through five of the procedure outlined above. As
with the previous workflow, care was taken so that the workflow could
be reused. The "Solvent" input for the resource "genbox" is provided by
the data source labeled "spc216.gro." An entry for this data in the BCJ
environment was created by using the "Create New Entry" interface to
import this information. It contains a GROMACS-specific description of
"Simple Point Charge water" that is often used as the solvent in
GROMACS experiments.</p><div class="figs"><div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F8','F8',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F8"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-8.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 8.</strong></a> MD Simulation Workflow.</p></div></div><p>The workflow "Tutorial Example" shown in Figure <a name="IDAGON22"></a><a onclick="popup('/content/2/1/9/figure/F9','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F9">9</a>
represents the complete process from the GROMACS tutorial. It makes use
of the two supporting workflows described in the previous sections,
demonstrating the hierarchical workflow definition integral to the BCJ
environment. As described above, the supporting workflows were not
designed only for this example tutorial. Each was designed generally,
so that it could be used as a component in other molecular dynamics
studies. Outputs not needed can be discarded by connecting each of them
to a "Sink" block. The Workflow Editor requires these explicit
connections since the ports to which they are connected were required.
This consistency check by the Workflow Editor helps to ensure workflow
correctness.</p><div class="figs"><div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F9','F9',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F9"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-9.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 9.</strong></a> Tutorial Experiment Workflow.</p></div></div><p>The workflow "Tutorial Example" defines only the <em>process </em>abstracted
from the GROMACS tutorial. To begin a new project (or experiment), the
user must import the original input data from an external source such
as a public database. When data is imported to the BCJ, an entry is
created and its content is initialized to that of a file provided by
the user. The BCJ physically copies the data into the environment with
subsequent references to the data in the BCJ through the entry. Thus,
this experiment began by having the user import a number of files into
the BCJ. Consequently, the workflow, Tutorial Example, allows one to
perform this experiment for any protein from the PDB. Figure <a name="IDAYON22"></a><a onclick="popup('/content/2/1/9/figure/F10','',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F10">10</a>
displays the specific experiment definition corresponding to the
example in the tutorial. The protein supplied as input is lysozyme, a
serine protease. The MD parameters for the energy minimization are
those supplied in the tutorial.</p><div class="figs"><div class="fig"><p><a onclick="popup('/content/2/1/9/figure/F10','F10',800,470); return false;" href="http://www.scfbm.org/content/2/1/9/figure/F10"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/1751-0473-2-9-10.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 10.</strong></a> Experiment Definition.</p></div></div><p>Numerous
specialized programs have been developed for molecular visualization
and interaction with molecular dynamics simulation. Rather than attempt
to develop another molecular visualization subsystem directly in the
BCJ environment (and then require BCJ users to become familiar with a
new user interface), the BCJ environment invokes an existing tool,
integrated as an external viewer, to display these results. For
example, VMD, Visual Molecular Dynamics <a name="IDAJPN22"></a>[<a href="#B15">15</a>], can be used to display the output structure.</p><a name="IDANPN22"></a><h3>Conclusion</h3><p>Bioinformatics
research involves computational processing of multi faceted biological
data at scales from atomic level to cellular. Sophisticated
environments are needed to support the diverse and expanding range of
computational methods, scales, models, and data. The inherently
collaborative work requires managing data and methods between
colleagues and public-domain databases. Complex experiments are often
involved, which need to be checked, altered, and documented. The
Bioinformatics Computational Journal (BCJ) reported here provides
support for these activities. The BCJ handles data management and
process specification in a single framework, demonstrated here with two
representative bioinformatics problems. The system was founded on
Eclipse, providing a rigorously tested and maintained software
infrastructure. The user paradigm was designed to resemble a laboratory
notebook, inline with the target users. We found this paradigm a
natural basis for describing computational experiments. Features are
provided in the BCJ to authenticate and establish pedigree of
experiments. This is a crucial feature toward verifying and documenting
computational results. Future efforts include adding iteration and
branching functionality and enabling for grid collaborations.</p><a name="IDARPN22"></a><h3>Competing interests</h3><p>The author(s) declare that they have no competing interests.</p><a name="IDAVPN22"></a><h3>Authors' contributions</h3><p>LF
designed and developed the data management architecture for the BCJ. JR
contributed to database development. AG and HA contributed the
computational examples. EK was senior software architect for the BCJ.
DJ contributed to the database and computing architecture. AH provided
cluster computing support for the BCJ. TC and GL provided
bioinformatics direction. GM conceived of the Computational Journal
architecture and directed the software development team. VF was the
project principal investigator, and coordinated and contributed to the
development of the BCJ and the writing of this paper. All authors read
and approved the final manuscript.</p><a name="IDAZPN22"></a><h3>Appendix of computational environments</h3><p><strong>PathPort </strong><a name="IDACYO22"></a>[<a href="#B16">16</a>],
developed by the Virginia Bioinformatics Institute (VBI), is designed
to combine information about pathogens with analysis and visualization
tools. It uses web services to provide remote execution of independent,
single process jobs in a non-cluster environment (it is grid-based,
hence designed for execution over a heterogeneous, geographically
distributed network of individual computers shared by users).</p>
<p><strong>BioCoRE </strong><a name="IDAJYO22"></a>[<a href="#B17">17</a>],
developed by the Theoretical and Computational Biophysics Group at the
University of Illinois at Urbana-Champlain, is designed as a biological
collaborative environment. Through a combination of web pages and Java
applets, BioCoRE combines the capabilities of project management,
instant message, message board, lab notebook, and file sharing software
with several specific applications, all designed for the simulation and
visualization of molecular dynamics experiments, including NAMD <a name="IDAOYO22"></a>[<a href="#B18">18</a>], VMD (Visual Molecular Dynamics) <a name="IDATYO22"></a>[<a href="#B19">19</a>], and MDTools <a name="IDAYYO22"></a>[<a href="#B11">11</a>]. BioCoRE also allows the submission of jobs to a cluster through a web page.</p>
<p><strong>PISE </strong><a name="IDA5YO22"></a>[<a href="#B11">11</a>],
developed by the Pasteur Institute, is designed to offer a web
interface to bioinformatics programs described in XML. The tools in
PISE were not designed for execution on a cluster. (However, as part of
this work some of the PISE applications have been modified to
facilitate cluster computing.) The basic functionality of PISE is
designed to execute programs sequentially. However, there is a plug-in
called G-Pipe, which allows pipelines to be defined.</p>
<p><strong>MIGenAS </strong><a name="IDAGZO22"></a>[<a href="#B20">20</a>],
the Max-Planck Integrated Gene Analysis System, is similar to PISE in
that it offers a web page providing the ability to graphically enter
program parameters and execute linear pipelines. Unlike PISE, which can
be downloaded and run locally, MIGenAS performs all operations on
servers at the Max-Planck Institute. It comes with a set of tools, and
at this time does not provide a mechanism to add external or internal
tools.</p>
<p><strong>Pegasys </strong><a name="IDANZO22"></a>[<a href="#B21">21</a>],
developed by the University of British Columbia's Bioinformatics
Center, is designed to graphically create and execute bioinformatics
workflows for high-throughput analysis. It offers clients for Linux,
Windows, and Macintosh. It comes with a set of analysis programs, but
allows internal tool expansion through the creation of XML tool
definitions.</p>
<p><strong>Wildfire </strong><a name="IDAUZO22"></a>[<a href="#B22">22</a>],
like Pegasys, is developed to create and execute bioinformatics
workflows. Wildfire is available as both a standalone and a web-based
workflow application that runs on a local machine, on the Grid, or on a
cluster. It allows drag-and-drop creation of both simple linear
pipelines and complex branching workflows. Wildfire provides some forms
of iteration and conditional branching.</p>
<p><strong>Taverna </strong><a name="IDA1ZO22"></a>[<a href="#B23">23</a>],
a commonly used standalone workflow system, was developed to provide
language and software tools for workflows as a component of the myGrid
project. Taverna provides the ability to create complex workflows
including iteration and conditional branching, as well as a scripting
language.</p>
<p><strong>Other systems </strong>include the commercial software Pipeline Pilot <a name="IDAD0O22"></a>[<a href="#B24">24</a>], Ergatis <a name="IDAI0O22"></a>[<a href="#B25">25</a>] (which is still in development), and Biopipe <a name="IDAN0O22"></a>[<a href="#B26">26</a>]
(which is no longer actively supported). The SciRun software has
capabilities that are conceptually relevant to this work (workflow
editing, procedure replication, job control, data management, etc.),
but is designed for imaging rather than molecular computational biology
<a name="IDAS0O22"></a>[<a href="#B27">27</a>]. A commercial data mining workbench from SPSS that includes a workflow editor has also been developed<a name="IDAX0O22"></a>[<a href="#B28">28</a>].</p><a name="IDA30O22"></a><h3>Acknowledgements</h3><p>This
work was supported in part by the U.S. Department of Commerce under
Grant No. BS123456 and by U.S. Army Contract W911SR-04-C-0087.</p><a name="refs"></a><h3>References</h3><ol id="references">
<li id="B1">
<p><a name="B1"></a> Benson DA,  Karsch-Mizrachi I,  Lipman DJ,  Ostell J,  Wheeler DL: <strong> GenBank. </strong></p><p><em>Nucleic Acids Research</em> 2007,  <strong>35</strong>(suppl_1)<strong>:</strong>D21-25.  <a target="_blank" href="http://dx.doi.org/10.1093/nar/gkl986">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B1" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B1','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B2">
<p><a name="B2"></a> Altschul SF,  Gish W,  Miller W,  Myers EW,  Lipman DJ: <strong> Basic local alignment search tool. </strong></p><p><em>Journal of Molecular Biology</em> 1990,  <strong>215</strong>(3)<strong>:</strong>403-410. <a target="_blank" href="http://www.scfbm.org/pubmed/2231712">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2231712">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B2" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B2','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B3">
<p><a name="B3"></a> Rice P,  Longden I,  Bleasby A: <strong> EMBOSS: the European Molecular Biology Open Software Suite. </strong></p><p><em>Trends Genet</em> 2000,  <strong>16</strong>(6)<strong>:</strong>276-277. <a target="_blank" href="http://www.scfbm.org/pubmed/10827456">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10827456">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B3" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B3','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B4">
<p><a name="B4"></a> Mangalam H: <strong> The Bio* toolkits--a brief overview. </strong></p><p><em>Briefings in Bioinformatics</em> 2003,  <strong>3</strong>(3)<strong>:</strong>296-302.  <a target="_blank" href="http://dx.doi.org/10.1093/bib/3.3.296">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B4" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B4','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B5">
<p><a name="B5"></a> Ludascher B and Groble: <strong> Introduction to the Special Section on Scientific Workflows. </strong></p><p><em>Sigmod Record</em> 2005,  <strong>34</strong>(3)<strong>:</strong>3-4.  <a target="_blank" href="http://dx.doi.org/10.1145/1084805.1084807">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B5" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B5','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B6">
<p><a name="B6"></a> Xu Y,  Uberbacher EC: <strong> Gene prediction by pattern recognition and homology search. </strong></p><p><em>Proceedings of the Fourth International Conference on Intelligent Systems for Molecular Biology; ISMB</em> 1996,  <strong>4</strong><strong>:</strong>241-251.  <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B6" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B6','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B7">
<p><a name="B7"></a> D'Anjou J,  Fairbrother S,  Kehn D,  Kellerman J,  McCarthy P: <em>The Java Developer's Guide to ELIPSE. </em> 2nd edition.  Boston , Addison-Wesley;  2005. <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B7" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B7','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B8">
<p><a name="B8"></a> Simmhan YL,  Plale B,  Gannon D: <strong> A survey of data provenance in e-science. </strong>In <em>SIGMOD Record</em>. <em> Volume 34</em>.   ACM;  2005:31-36.  <a target="_blank" href="http://dx.doi.org/10.1145/1084805.1084812">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B8" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B8','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B9">
<p><a name="B9"></a> Davidson SB,  Crabtree J,  Brunk BP,  Schug J,  Tannen V,  Overton GC,  Stoeckert CJ Jr: <strong> K2/Kleisli and GUS: experiments in integrated access to genomic data sources. </strong></p><p><em>IBM Systems Journal</em> 2001,  <strong>40</strong>(2)<strong>:</strong>512-531.  <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B9" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B9','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B10">
<p><a name="B10"></a><strong> Document Type Definition for XML data </strong>
		[<a target="_blank" href="http://silk.bioinfo.ittc.ku.edu/cj/schema/ResourceDef.dtd">http://silk.bioinfo.ittc.ku.edu/cj/schema/ResourceDef.dtd</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://silk.bioinfo.ittc.ku.edu/cj/schema/ResourceDef.dtd&amp;refdoi=10.1186/1751-0473-2-9" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B10" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B10','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B11">
<p><a name="B11"></a> Letondal C: <strong> A Web interface generator for molecular biology programs in Unix. </strong></p><p><em>Bioinformatics</em> 2001,  <strong>17</strong>(1)<strong>:</strong>73-82. <a target="_blank" href="http://www.scfbm.org/pubmed/11222264">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11222264">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B11" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B11','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B12">
<p><a name="B12"></a> Frost V, Clark T, Gauch S, Lushington G, Minden
G, Komp E, Hock A, Johnson D, Feagan L, Garrett A, Rohrer J, Amthauer
H, Ozor A: <strong> Bioinformatics Computational Journal: User Guide, Technical Report ITTC-FY2008-TR-38270-04. </strong>
			[<a class="longstring" target="_blank" href="http://www.ittc.ku.edu/publications/documents/Bioinfo_Comp_Journal_User_Manual.pdf">http:/ / www.ittc.ku.edu/ publications/ documents/ Bioinfo_Comp_Journal_User_Manual.pd f</a>]
		 <a href="http://www.webcitation.org/query.php?url=http://www.ittc.ku.edu/publications/documents/Bioinfo_Comp_Journal_User_Manual.pdf&amp;refdoi=10.1186/1751-0473-2-9" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> 2007. <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B12" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B12','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B13">
<p><a name="B13"></a><strong> GROMACS: Fast, Free and Flexible MD </strong>
		[<a target="_blank" href="http://www.gromacs.org/">http://www.gromacs.org</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.gromacs.org&amp;refdoi=10.1186/1751-0473-2-9" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B13" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B13','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B14">
<p><a name="B14"></a><strong> Molecular Dynamics Tutorial (GROMACS) </strong>
		[<a target="_blank" href="http://md.chem.rug.nl/education/mdcourse/index.html">http://md.chem.rug.nl/education/mdcourse/index.html</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://md.chem.rug.nl/education/mdcourse/index.html&amp;refdoi=10.1186/1751-0473-2-9" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B14" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B14','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B15">
<p><a name="B15"></a> Humphrey W,  Dalke A,  Schulten K: <strong> VMD: visual molecular dynamics. </strong></p><p><em>Journal of Molecular Graphics</em> 1996,  <strong>14</strong>(1)<strong>:</strong>33-38. <a target="_blank" href="http://www.scfbm.org/pubmed/8744570">PubMed Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1016/0263-7855%2896%2900018-5">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B15" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B15','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B16">
<p><a name="B16"></a> Eckart JD,  Sobral B: <strong> A Life Scientists Gateway to Distributed Data Management and Computing: The PathPort/ToolBus Framework. </strong></p><p><em>OMICS: A Journal of Integrative Biology</em> 2003,  <strong>7</strong>(1)<strong>:</strong>79-88. <a target="_blank" href="http://www.scfbm.org/pubmed/12831562">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12831562">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B16" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B16','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B17">
<p><a name="B17"></a> Bhandarkar M,  Budescu G,  Humphrey WF,  Izaguirre JA,  Izrailev S,  Kal LV,  Kosztin D,  Molnar F,  Phillips JC,  Schulten K: <em>BioCoRE: a collaboratory for structural biology: San Francisco, California.
					 </em> ;  1999:242-251.  <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B17" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B17','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B18">
<p><a name="B18"></a> Phillips JC,  Braun R,  Wang W,  Gumbart J,  Tajkhorshid E,  Villa E,  Chipot C,  Skeel RD,  Kale L,  Schulten K: <strong> Scalable molecular dynamics with NAMD. </strong></p><p><em>J Comput Chem</em> 2005,  <strong>26</strong>(16)<strong>:</strong>1781-1802. <a target="_blank" href="http://www.scfbm.org/pubmed/16222654">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16222654">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B18" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B18','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B19">
<p><a name="B19"></a><strong> VMD - Visual Molecular Dynamics </strong>
		[<a target="_blank" href="http://www.ks.uiuc.edu/Research/vmd/">http://www.ks.uiuc.edu/Research/vmd/</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.ks.uiuc.edu/Research/vmd/&amp;refdoi=10.1186/1751-0473-2-9" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B19" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B19','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B20">
<p><a name="B20"></a> Letondal C: <strong> A web interface generator for molecular biology programs in Unix. </strong></p><p><em>Bioinformatics (Oxford, England)</em> 2001,  <strong>17</strong>(1)<strong>:</strong>73-82. <a target="_blank" href="http://www.scfbm.org/pubmed/11222264">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11222264">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B20" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B20','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B21">
<p><a name="B21"></a> Rampp M Soddemann T, Lederer H.: <strong> The MIGenAS integrated bioinformatics toolkit for web-based sequence analysis. </strong></p><p><em>Nucleic Acids Research</em> 2006,  <strong>1</strong>(34W1)<strong>:</strong>5-9.  <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B21" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B21','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B22">
<p><a name="B22"></a> Shah SP,  He DY,  Sawkins JN,  Druce JC,  Quon G,  Lett D,  Zheng GX,  Xu T,  Ouellette BF: <strong> Pegasys: software for executing and integrating analyses of biological sequences. </strong></p><p><em>BMC Bioinformatics</em> 2004,  <strong>5</strong><strong>:</strong>40. <a target="_blank" href="http://www.scfbm.org/pubmed/15096276">PubMed Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-5-40">BioMed Central Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15096276">PubMed Central Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B22" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B22','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B23">
<p><a name="B23"></a> Stevens RD,  Robinson AJ,  Goble CA: <strong> myGrid: personalised bioinformatics on the information grid. </strong></p><p><em>Bioinformatics (Oxford, England)</em> 2003,  <strong>19 Suppl 1</strong><strong>:</strong>i302-4. <a target="_blank" href="http://www.scfbm.org/pubmed/12855473">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12855473">Publisher Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B23" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B23','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B24">
<p><a name="B24"></a>SciTegic: <strong> SciTegic - PP Overview. </strong>
		[<a target="_blank" href="http://www.scitegic.com/products/overview/">http://www.scitegic.com/products/overview/</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.scitegic.com/products/overview/&amp;refdoi=10.1186/1751-0473-2-9" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B24" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B24','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B25">
<p><a name="B25"></a> Angiuol SV: <strong> Workflow / Ergatis: A Data Management System for Automated Genome Annotation and Comparative Analysis (poster). </strong></p><p><em>Genome Informatics 2005 (Cold Spring Harbor Laboratory, New York)</em> 2006, Poster.  <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B25" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B25','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B26">
<p><a name="B26"></a> Hoon S,  Ratnapu KK,  Chia JM,  Kumarasamy B,  Juguang X,  Clamp M,  Stabenau A,  Potter S,  Clarke L,  Stupka E: <strong> Biopipe: a flexible framework for protocol-based bioinformatics analysis. </strong></p><p><em>Genome Res</em> 2003,  <strong>13</strong>(8)<strong>:</strong>1904-1915. <a target="_blank" href="http://www.scfbm.org/pubmed/12869579">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12869579">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12869579">PubMed Central Full Text</a> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B26" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B26','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B27">
<p><a name="B27"></a><strong> SCIRun - SCI Software - Scientific Computing and Imaging Institute </strong>
		[<a target="_blank" href="http://software.sci.utah.edu/scirun.html">http://software.sci.utah.edu/scirun.html</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://software.sci.utah.edu/scirun.html&amp;refdoi=10.1186/1751-0473-2-9" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B27" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B27','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B28">
<p><a name="B28"></a><strong> Data mining, Clementine, Predictive Modeling, Predictive Analytics, Predictive Analysis </strong>
		[<a target="_blank" href="http://www.spss.com/clementine">http://www.spss.com/clementine</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.spss.com/clementine&amp;refdoi=10.1186/1751-0473-2-9" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.scfbm.org/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B28" onclick="popup('/sfx_links.asp?ui=1751-0473-2-9&amp;bibl=B28','SFXMenu','460','420'); return false;"><img src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
</ol>
<h4 class="bottommessage"><a href="http://www.scfbm.org/content/2/1/9/postcomment">Have something to say? Post a comment on this article!</a></h4><script type="text/javascript">initializeOutline();</script></td></tr></tbody></table></div><hr color="#cccccc" noshade="noshade" size="1">
<table cellpadding="3" width="100%"><tbody><tr valign="middle"><td align="center">


<center>

<!------ OAS AD 'Bottom' begin ------>
<script language="JavaScript">
<!--
OAS_AD('Bottom');
//-->
</script><script type="text/javascript"><!--
google_ad_client = "pub-8287243681099367";
/* 728x90, created 3/5/09 */
google_ad_slot = "2716523086";
google_ad_width = 728;
google_ad_height = 90;
//-->
</script>
<script type="text/javascript" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/show_ads.js">
</script><script src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/expansion_embed.js"></script><script src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/test_domain.js"></script><script>window.google_render_ad();</script><ins style="border: medium none ; margin: 0pt; padding: 0pt; display: inline-table; height: 90px; position: relative; visibility: visible; width: 728px;"><ins style="border: medium none ; margin: 0pt; padding: 0pt; display: block; height: 90px; position: relative; visibility: visible; width: 728px;"><iframe allowtransparency="true" hspace="0" id="google_ads_frame1" marginheight="0" marginwidth="0" name="google_ads_frame" src="Feagan%202007%20Bioinformatics%20process%20management%20-%20information%20flow%20via%20a%20computational%20journal_files/ads.htm" style="left: 0pt; position: absolute; top: 0pt;" vspace="0" frameborder="0" height="90" scrolling="no" width="728"></iframe></ins></ins><noscript>
<a
href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.scfbm.org/article/sourcecodebiolmed/1491372061@Top,Bottom!Bottom?%22bioinformatics+workflows%22+pise+review"
target=_top>
<img
src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.scfbm.org/article/sourcecodebiolmed/1491372061@Top,Bottom!Bottom?%22bioinformatics+workflows%22+pise+review"
border=0></a>
</noscript>
<!------ OAS AD end ------>
</center>

</td></tr>
<tr><td align="center"><div class="smalltext">
<a href="http://www.scfbm.org/info/terms">Terms and Conditions</a> 
<a href="http://www.scfbm.org/info/privacy">Privacy statement</a> 
<a href="http://www.biomedcentral.com/info/mediainformation/">Information for advertisers</a> 
<a href="http://www.biomedcentral.com/info/about/bmcjobs">Jobs at BMC</a> 
<a href="mailto:info@biomedcentral.com">Contact us</a> 
</div>
</td></tr></tbody></table>
<table class="footer2t" border="0" cellpadding="0" cellspacing="0" width="100%">
<tbody><tr>
<td colspan="3" align="center" bgcolor="#003399">
© 1999-2009 BioMed Central Ltd unless otherwise stated.
 Part of <a href="http://www.springer.com/">Springer Science+Business Media</a>.
</td>
</tr>
</tbody></table>
</div>
</div>
</center>
</body></html>