<html><head>
<meta http-equiv="content-type" content="text/html; charset=windows-1252">

<title>CRISP - Computer Retrieval of Information on Scientific Projects, Abstract Display</title>
</head><body alink="#990000" background="Haft%202008%20NIH%20Grant%20title%20Algorithmically-Tuned%20Protein%20Families,%20Rule_Base%20and%20Characterized%20Protein_files/back.gif" link="#009909" text="#000000" vlink="#990000">
<table width="100%"><tbody><tr valign="middle">
<td valign="top">
 Version 2.5.2.0
</td>
<td align="left">
<img src="Haft%202008%20NIH%20Grant%20title%20Algorithmically-Tuned%20Protein%20Families,%20Rule_Base%20and%20Characterized%20Protein_files/crisp_a.gif" alt="CRISP Logo"></td>
<td align="left">
<a href="http://crisp.cit.nih.gov/" onmouseover="status='Return to the CRISP Homepage';return true;">
         <img src="Haft%202008%20NIH%20Grant%20title%20Algorithmically-Tuned%20Protein%20Families,%20Rule_Base%20and%20Characterized%20Protein_files/home.gif" alt="CRISP Homepage" border="0"></a>
<a href="http://crisp.cit.nih.gov/crisp/CRISP_Help.help" onmouseover="status='Go to the CRISP Help Overview';return true;"><img src="Haft%202008%20NIH%20Grant%20title%20Algorithmically-Tuned%20Protein%20Families,%20Rule_Base%20and%20Characterized%20Protein_files/help.gif" alt="Help for CRISP" border="0"></a>
<a href="mailto:commons@od.nih.gov" onmouseover="status='Email your comments and questions regarding CRISP';return true;"><img src="Haft%202008%20NIH%20Grant%20title%20Algorithmically-Tuned%20Protein%20Families,%20Rule_Base%20and%20Characterized%20Protein_files/mailbox.gif" alt="Email Us" border="0"></a>
</td>
</tr></tbody></table>
<hr>
<blockquote><h2 align="center"> <i>Abstract</i></h2>
<form action="" method="get" name="abstract">
<input value="Back to Hit List" onclick="history.back()" type="button">
</form>
<table cellpadding="1" cellspacing="1" frame="void">
<colgroup span="2"> <col align="left"><col align="left">
</colgroup><tbody><tr><td><b>Grant Number: </b></td><td>1R01HG004881-01</td></tr>
<tr><td><b>Project Title: </b></td><td>Algorithmically-Tuned Protein Families, Rule_Base and Characterized Proteins</td></tr>
</tbody></table>
<table cellpadding="1" cellspacing="1" frame="void">
<col align="left"><col align="left"><col align="left"><col align="left">
<tbody><tr><td><b>PI Information:</b></td><td><b>Name</b></td><td><b>Email</b></td><td><b>Title</b></td></tr>
<tr><td></td><td>HAFT, DANIEL H.</td>
<td> <a href="mailto:"> </a></td>
<td></td></tr>
</tbody></table>
<p><b>Abstract: </b> DESCRIPTION (provided by applicant): Analysis of
the microbial communities present in or on the human body holds promise
for explaining the dynamic basis of host-microbiome symbiosis and the
contribution of these communities (the human microbiome) to health and
disease. Vast amounts of metagenomic DNA sequence can be collected.
However, current bioinformatics tools limit our ability to translate
sequence into fundamentally new biomedical knowledge. There is a great
need to improve existing tools and develop computational methods to
address the complexity of data generated by human microbiome projects
(HMP). This proposal takes a three-pronged approach to dramatically
improve methods for extracting meaning from HMP sequence data. The
first is to develop algorithms that build protein families, each family
just inclusive enough that checking a genome for some cohort of
families tells whether or not a pathway is present. These algorithms
resemble Phylogenetic Profiling, a data mining technique, but go
through optimization steps that guide the building of each family.
Pre-built families are not required. The result is new descriptive
power that can discover and describe new systems and pathways.
Thousands of new families will be created. The second is a new way to
apply annotation rules. Large numbers of rules created automatically,
each of which works on fairly small numbers of proteins, can apply very
exacting tests to determine whether one protein should be expected to
have the same function as another that is already characterized. By
deriving support from comparing gene regions or metabolic backgrounds
in ways made possible only by having large numbers of complete genomes,
these rules can achieve much greater confidence than more simplistic
annotation techniques. The third is a systematic compilation of the
right starting points for annotation. Annotation methods today are
built to achieve maximum leverage from those few proteins whose
functions are known for sure, but searching for those good anchors is
surprisingly difficult, and searching repeatedly wasteful. The CHAR
database will collect experimentally characterized proteins and make
them "rule-ready" and universally available. All of the resources
developed through this proposal will be made publicly available. These
approaches combine to let us read metabolic properties from microbial
genome sequences more accurately, and figure out better ways to fight
disease. PUBLIC HEALTH RELEVANCE: The massive numbers of microbial
species living in and on the human organism vary greatly from person to
person, and transform our metabolism enough to impact our health. The
work we propose reads patterns of DNA differences from microbe to
microbe as a means to figure out which species do what inside the human
gut, and therefore how we can make changes to treat or prevent disease.
</p><p><b>Public Health Relevance: </b>
The massive numbers of microbial species living in and on the human organism vary
greatly from person to person, and transform our metabolism enough to impact our
health. The work we propose reads patterns of DNA differences from microbe to microbe
as a means to figure out which species do what inside the human gut, and therefore how
we can make changes to treat or prevent disease.
</p><p><b>Thesaurus Terms:</b><br>
</p><p>There are no thesaurus terms on file for this project.
<br>
<br>
<table cellpadding="1" cellspacing="1" frame="void">
<colgroup span="2"> <col align="left"><col align="left">
</colgroup><tbody><tr><td><b>Institution: </b></td><td>J. CRAIG VENTER INSTITUTE, INC.</td></tr>
<tr><td></td><td>9704 MEDICAL CENTER DR, 4TH FL</td></tr>
<tr><td></td><td>ROCKVILLE, MD 20850</td></tr>
<tr><td><b>Fiscal Year: </b></td><td>2008</td></tr>
<tr><td valign="top"><b>Department: </b></td><td></td></tr>
<tr><td><b>Project Start: </b></td><td>26-SEP-2008</td></tr>
<tr><td><b>Project End: </b></td><td>31-JUL-2011</td></tr>
<tr><td><b>ICD: </b></td><td>NATIONAL HUMAN GENOME RESEARCH INSTITUTE</td></tr>
<tr><td><b>IRG: </b></td><td>ZRG1</td></tr>
</tbody></table>
</p><hr>
 <p align="center">

<a href="http://crisp.cit.nih.gov/" onmouseover="status='Return to the CRISP Homepage';return true;">
         <img src="Haft%202008%20NIH%20Grant%20title%20Algorithmically-Tuned%20Protein%20Families,%20Rule_Base%20and%20Characterized%20Protein_files/home.gif" alt="CRISP Homepage" border="0"></a>
<a href="http://crisp.cit.nih.gov/crisp/CRISP_Help.help" onmouseover="status='Go to the CRISP Help Overview';return true;"><img src="Haft%202008%20NIH%20Grant%20title%20Algorithmically-Tuned%20Protein%20Families,%20Rule_Base%20and%20Characterized%20Protein_files/help.gif" alt="Help for CRISP" border="0"></a>
<a href="mailto:commons@od.nih.gov" onmouseover="status='Email your comments and questions regarding CRISP';return true;"><img src="Haft%202008%20NIH%20Grant%20title%20Algorithmically-Tuned%20Protein%20Families,%20Rule_Base%20and%20Characterized%20Protein_files/mailbox.gif" alt="Email Us" border="0"></a>

<br>
</p></blockquote></body></html>