
































	
		
						
																																																																																																
																																																																											
																					
 														 		
					



<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
	

	

	
<head xmlns="">
<!-- super head top --><META http-equiv="Content-Type" content="text/html; charset=utf-8">
                





































        
               
      








        
                
 	



 










	
				

        
        
  

 


 

 

 


 
 









	
	
<title>Whole-genome sequencing and variant discovery in C. elegans - Nature Methods</title><meta content="noarchive" name="robots"><link href="http://www.nature.com/favicon.ico" rel="shortcut icon"><link type="text/css" href="/nmeth/style_older.css" rel="stylesheet"><style media="all" type="text/css">@import "/nmeth/style.css";</style>
		







<script type="text/javascript"><!--//--><![CDATA[//><!--
	var include = '';

	var wp = '';
	if (wp != '') include += 'wp='+wp;
	include += ';';

	var rf = '';
	if (rf != '') include += "rf="+rf;
	include += ';';

	var sp = '';
	if (sp != '') include += "sp="+sp;
	include += ";";

	var ps = '';
	if (ps != '') include += "ps="+ps;
	include += ";";

	var adinf = "lg=no;";

	if(include != ";;;;")
	{
		adinf = include;
	}

	var time = new Date();
	ordval= (time.getTime());

		ord=Math.random()*10000000000000000;
//--><!]]></script>

<script type="text/javascript" language="javascript">// provides cross-browser functionality
//======================================
function getObject(name) {
  if(document.getElementById) return document.getElementById(name);
  else if(document.all) return document.all[name];
  else return null;
}

</script><script src="/mailafriend/mailafriend.js" language="javascript" type="text/javascript"></script><!-- super head bottom -->
	<!-- resource metadata -->
				
				 																							
			
				
											
			<link title="schema(PRISM)" rel="schema.prism" href="http://prismstandard.org/namespaces/1.2/basic/" />
																																																												<meta name="prism.copyright" content="&#169; 2008 Nature Publishing Group" />
					<meta name="prism.rightsAgent" content="permissions@nature.com" />
					<meta name="prism.publicationDate" content="2008-01-20" />
					<meta name="prism.volume" content="5" />
					<meta name="prism.number" content="2" />
					<meta name="prism.startingPage" content="183" />
					<meta name="prism.endingPage" content="188" />
					<meta name="prism.publicationName" content="Nature Methods" />
					<meta name="prism.issn" content="ERROR! NO ISSN" />
					<meta name="prism.eIssn" content="ERROR! NO EISSN" />
				
			<link title="schema(DC)" rel="schema.dc" href="http://purl.org/dc/elements/1.1/" />
																																																																																																																																																	<meta name="dc.publisher" content="Nature Publishing Group" />
					<meta name="dc.language" content="en" />
					<meta name="dc.rights" content="&#169; 2008 Nature Publishing Group" />
					<meta name="dc.title" content="Whole-genome sequencing and variant discovery in C. elegans" />
					<meta name="dc.creator" content="LaDeana W Hillier" />
					<meta name="dc.creator" content="Gabor T Marth" />
					<meta name="dc.creator" content="Aaron R Quinlan" />
					<meta name="dc.creator" content="David Dooling" />
					<meta name="dc.creator" content="Ginger Fewell" />
					<meta name="dc.creator" content="Derek Barnett" />
					<meta name="dc.creator" content="Paul Fox" />
					<meta name="dc.creator" content="Jarret I Glasscock" />
					<meta name="dc.creator" content="Matthew Hickenbotham" />
					<meta name="dc.creator" content="Weichun Huang" />
					<meta name="dc.creator" content="Vincent J Magrini" />
					<meta name="dc.creator" content="Ryan J Richt" />
					<meta name="dc.creator" content="Sacha N Sander" />
					<meta name="dc.creator" content="Donald A Stewart" />
					<meta name="dc.creator" content="Michael Stromberg" />
					<meta name="dc.creator" content="Eric F Tsung" />
					<meta name="dc.creator" content="Todd Wylie" />
					<meta name="dc.creator" content="Tim Schedl" />
					<meta name="dc.creator" content="Richard K Wilson" />
					<meta name="dc.creator" content="Elaine R Mardis" />
					<meta name="dc.identifier" content="doi:10.1038/nmeth.1179" />
					<meta name="dc.date" content="2008-01-20" />
					<meta name="dc.source" content="Nature Methods, Published online: 20 January 2008; | doi:10.1038/nmeth.1179" />
				
																																																						<meta name="citation_publisher" content="Nature Publishing Group" />
					<meta name="citation_authors" content="LaDeana W Hillier, Gabor T Marth, Aaron R Quinlan, David Dooling, Ginger Fewell, Derek Barnett, Paul Fox, Jarret I Glasscock, Matthew Hickenbotham, Weichun Huang, Vincent J Magrini, Ryan J Richt, Sacha N Sander, Donald A Stewart, Michael Stromberg, Eric F Tsung, Todd Wylie, Tim Schedl, Richard K Wilson, Elaine R Mardis" />
					<meta name="citation_title" content="Whole-genome sequencing and variant discovery in C. elegans" />
					<meta name="citation_date" content="2008-01-20" />
					<meta name="citation_volume" content="5" />
					<meta name="citation_issue" content="2" />
					<meta name="citation_firstpage" content="183" />
					<meta name="citation_doi" content="doi:10.1038/nmeth.1179" />
					<meta name="citation_journal_title" content="Nature Methods" />
			
																			<meta name="Access" content="Yes" />
					<meta name="WT.cg_n" content="Nature Methods" />
			</head><body marginheight="0" marginwidth="0" topmargin="0" leftmargin="0" text="#000000" bgcolor="#FFFFFF">
<!-- super body top --><a name="top"/>


																		

<!-- header -->
<div id="head-er">

<div class="header-nonad-wrapper" id="top">
				<div class="accessibility-login-wrap">
	<div class="accessibility-login">
				<!--<a href="/" class="rollover"><img src="/common/includes/header/images/nature.com.gif" alt="Nature.com homepage" class="nature-logo" /></a>-->
		<a href="/" class="nature-link">nature.com<span class="hidden"> homepage</span></a>
									

			<div class="logon">
		<p><a href="/foxtrot/svc/login" class="login" title="login to Nature.com">Login<!-- following space necessary for Mac IE --> </a><span class="onlyformacie">&nbsp;</span></p>
	</div>
						<div class="cleardiv"><!--  --></div>
	</div>
</div>

<div class="cleardiv"><!--  --></div>
<hr />
		
			<div class="image-user-services-wrap">
	<div class="image-user-services">
					<a href="/nmeth"><img src="/includes/rj_globnavimages/nmeth_logo.gif" class="journal-header" alt="Nature Methods homepage" /></a>		
								<div class="user-services div1">
								<!-- whitespace removed to deal with IE 5.0 bug --><ul class="list1"><li class="li1"><a href="/myaccount/">My account</a></li><li class="li2"><a href="http://mts-nmeth.nature.com/" title="Submit  manuscript">Submit</a></li></ul>
							</div>
						<div class="user-services div2">
								<!-- whitespace removed to deal with IE 5.0 bug --><ul class="list2"><li class="li1"><a href="/register/" title="Register with Nature.com">Register</a></li><li class="li2"><a href="https://secure.nature.com/subscribe/nmeth" title="Subscribe to Nature Methods">Subscribe</a></li></ul>
							</div>
						<div class="cleardiv"><!--  --></div>
			</div>
	<div class="cleardiv"><!--  --></div>
</div>

<div class="cleardiv"><!--  --></div>
<hr />		
			<div class="publications-search-wrap clearfix">
	<div class="publications-search">
				<ul class="publications">
			<li class="first"><a href="/siteindex/">Publications&nbsp;A-Z&nbsp;index</a></li><li><a href="/browse/">Browse&nbsp;by&nbsp;subject</a></li>		</ul>

		<hr />
		
				<div class="search">
			<form class="search" action="/search/executeSearch?" method="get">
				<div>
					<label for="searchtext">Search</label>
																	<select name="sp-q-1">
														<option value="NMETH">This journal</option>
														<option value="">All of nature.com</option>
													</select>
																<input type="text" id="searchtext" name="sp-q" value="" />
															<input type="hidden" name="sp-c" value="25" />
															<input type="hidden" name="sp-m" value="0" />
															<input type="hidden" name="sp-s" value="date_descending" />
															<input type="hidden" name="include-collections" value="journals_nature,crawled_content" />
															<input type="hidden" name="exclude-collections" value="journals_palgrave,lab_animal" />
																				
																									<input type="hidden" name="sp-a" value="sp1001702d" />
																											<input type="hidden" name="sp-sfvl-field" value="subject|ujournal" />
																											<input type="hidden" name="sp-x-1" value="ujournal" />
																											<input type="hidden" name="sp-p-1" value="phrase" />
																											<input type="hidden" name="sp-p" value="all" />
																							<input type="submit" name="submit" class="button" value="go" title="search now" /> 						
									</div>
			</form>
		</div>
				<div class="cleardiv"><!--  --></div>
	</div>
</div>		
		</div>

	<!-- leaderboard ad -->
<div class="header-leaderboard-wrap">











		
<div class="header-leaderboard adunit-leaderboard">
<script type="text/javascript"><!--//--><![CDATA[//><!--
		document.write('<script type="text/javascript" src="http://ad.doubleclick.net/adj/nmeth.nature.com/article;abr=!webtv;type=af;artid=nmeth.1179;date=20080120;issue=2;pos=top;dcopt=ist;sz=728x90;ptile=1;ord=' + ord + '?" ><\/script>');

	if ((!document.images && navigator.userAgent.indexOf('Mozilla/2.') >= 0) || (navigator.userAgent.indexOf("WebTV") >= 0)) {
		document.write('<a href="http://ad.doubleclick.net/jump/nmeth.nature.com/article;sz=728x90;ord=' + ord + '?">');
		document.write('<img src="http://ad.doubleclick.net/ad/nmeth.nature.com/article;sz=728x90;ord=' + ord + '?" width="728" height="90" border="0"></a>');
	}
//--><!]]></script>
<noscript>
	<div><a href="http://ad.doubleclick.net/jump/nmeth.nature.com/article;abr=!NN2;tile=1;type=af;artid=nmeth.1179;date=20080120;issue=2;pos=top;dcopt=ist;sz=728x90;ptile=1;ord=123456789?"><img src="http://ad.doubleclick.net/ad/nmeth.nature.com/article;abr=!NN2;tile=1;type=af;artid=nmeth.1179;date=20080120;issue=2;pos=top;dcopt=ist;sz=728x90;ptile=1;ord=123456789?" alt="Advertisement" style="border:0" /></a></div>
</noscript>

</div>


</div>
<!-- end leaderboard ad -->

</div>
<!-- end header -->

			<div id="constrain"><div class="constrain">
		<table cellpadding="0" cellspacing="0" border="0" width="100%"><tr xmlns=""><td valign="top" height="22" align="center"><table cellpadding="0" cellspacing="0" border="0" width="760"><tr><td height="22" bgcolor="#D1E1EA" width="145">&nbsp;</td><td align="left" width="630" height="22" bgcolor="#D1E1EA" class="breadcrumb"><a class="breadcrumblink" href="/nmeth/index.html">Journal home</a>&nbsp;&gt;
<a class="breadcrumblink" href="/nmeth/archive/index.html">Archive</a>&nbsp;&gt;
<a class="breadcrumblink" href="/nmeth/journal/v5/n2/index.html">
					
 Table of Contents</a>&nbsp;&gt;&nbsp;<a class="breadcrumblink" href="/nmeth/journal/v5/n2/index.html#af">Article</a> &gt;Full text</td></tr></table></td></tr><tr><td align="center"><table cellpadding="0" cellspacing="0" border="0" width="760"><tr><td valign="top" width="130"><input xmlns="" value="archive" id="leftnav_type" type="hidden"><input xmlns="" value="af" id="categ" type="hidden"><input xmlns="" value="" id="subcateg" type="hidden"><input xmlns="" value="orig" id="arch_categ" type="hidden">
<script LANGUAGE="JavaScript">
var bgColorOn = "#6E83A0";
var bgColor2 = "#EAEAEA";
//var bgClassOn = "leftnavOn_bg";
var imageOn = "/includes/rj_globnavimages/nav_dots1.gif";
var imageOff = "/includes/rj_globnavimages/nav_dots2.gif";
var imageSpacer = "/includes/rj_globnavimages/spacer.gif";



var children = new Array();

function addSubLevel(parent, child){
	children[parent] = child;
}

function getObjectRef(name) {
  if(document.getElementById) return document.getElementById(name);
  else if(document.all) return document.all[name];
  else return null;
}

var thisPageType = '';

function setPageType(){
	var type = getObjectRef("leftnav_type").value;
	//alert(type);

	if (type == null){
		type = '';
	}


	thisPageType = type;
}

function getPageType(){
	return thisPageType;
}

setPageType();

function isCurrentType(thisType){
	if (thisType == getPageType()){
		return true;
	}else{
		var isChild = false;
		if (typeof children[thisType] != "undefined"){
			var child = children[thisType];
			for (var i=0; i<child.length; i++){
				if (getPageType() == child[i]){
					isChild = true;
				}
			}
		}
		return isChild;
	}
}

function getNewSubRow(name, linkhref, text){
	return '<tr><td width="5" style="background-color:'+bgColor2+'" id="'+name+'_1"><img src="'+imageSpacer+'" width="5" height="15" alt=""></td><td style="background-color:'+bgColor2+'" id="'+name+'_2" valign="top"><img src="'+imageOff+'" id="'+name+'_image" width="5" height="7" alt="" vspace="2"></td><td style="background-color:'+bgColor2+'" id="'+name+'_3"><img src="'+imageSpacer+'" width="5" height="15" alt=""></td><td style="background-color:'+bgColor2+'" id="'+name+'_4"><a href="'+linkhref+'" class="leftnav3" id="'+name+'_link">'+text+'</a></td><td width="5" style="background-color:'+bgColor2+'" id="'+name+'_5" valign="top"><img src="'+imageSpacer+'" width="5" height="15" alt=""></td></tr>';
}

function getSubRow(newRows){
	if (newRows != ''){
		return '<img src="'+imageSpacer+'" width="5" height="1" alt=""></td></tr><tr><td height="1" colspan="3"><table width="100%" cellpadding="0" cellspacing="0" border="0">'+newRows+'</table><tr><td colspan="3" height="1"><img src="'+imageSpacer+'" width="5" height="1" alt=""></td></tr>'
	} else {
		//return '<img src="'+imageSpacer+'" width="5" height="1" alt="">';
		return '';
	}
}

function hilite(text) {
// set colors

//alert(getObjectRef(text+"_1").style.backgroundColor);
	if (getObjectRef(text+"_link") != null){

		var num = 1;
		while (getObjectRef(text+"_"+num)){
			//getObjectRef(text+"_"+num).className  = bgClassOn;
			getObjectRef(text+"_"+num).style.backgroundColor = bgColorOn;
			num++;
		}

		getObjectRef(text+"_link").className = getObjectRef(text+"_link").className+"On";
	}

	if (getObjectRef(text+"_image") != null){
		//alert(getObjectRef(text+"_image").src);
		getObjectRef(text+"_image").src = imageOn;
	}

}

</script>
		<table cellpadding="0" cellspacing="0" border="0" width="130"><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td id="home_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="home_2" style="background-color:
#D1E1EA"><a class="leftnav" href="/nmeth/index.html" id="home_link">Journal home</a></td><td id="home_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="aop_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="aop_2" style="background-color:
#D1E1EA"><a class="leftnav" href="/nmeth/journal/vaop/ncurrent/index.html" id="aop_link">Advance online publication</a></td><td id="aop_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"><script LANGUAGE="JavaScript">

var thisType = 'aop';

addSubLevel(thisType, new Array("About_AOP"));

var newRows = '';
if (isCurrentType(thisType) || '' == 'true'){

	
	newRows += getNewSubRow('About_AOP', '/nmeth/aop/index.html', 'About AOP');
	

	if (isCurrentType(thisType) && thisType != getPageType()){
		hilite(thisType);
	}
	
	
	

}

document.write (getSubRow(newRows));
</script></td></tr><tr><td id="issue_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="issue_2" style="background-color:
#D1E1EA"><a class="leftnav" href="/nmeth/journal/v5/n11/index.html" id="issue_link">
Current issue
</a></td><td id="issue_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="archive_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="archive_2" style="background-color:
#D1E1EA"><a class="leftnav" href="/nmeth/archive/index.html" id="archive_link">Archive</a></td><td id="archive_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="Press_releases_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Press_releases_2" style="background-color:
#D1E1EA"><a class="leftnav" href="/nmeth/press_release/index.html" id="Press_releases_link">Press releases</a></td><td id="Press_releases_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr></table>
		<table cellpadding="0" cellspacing="0" border="0" width="130"><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="12" colspan="3"><img alt="" height="12" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td id="Supplements_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Supplements_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="http://blogs.nature.com/nmeth/methagora/" id="Supplements_link">Methagora</a></td><td id="Supplements_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="Web_Focuses_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Web_Focuses_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/nmeth/focsup_index.html" id="Web_Focuses_link">Focuses</a></td><td id="Web_Focuses_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr></table>
	
								
			
		<table cellpadding="0" cellspacing="0" border="0" width="130"><tr><td height="12" colspan="3"><img alt="" height="12" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td id="For_authors_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="For_authors_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/nmeth/authors/index.html" id="For_authors_link">Guide to authors</a></td><td id="For_authors_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"><script LANGUAGE="JavaScript">

var thisType = 'For_authors';

addSubLevel(thisType, new Array("Call_papers","How_to_submit","Content_types","Editorial_process"));

var newRows = '';
if (isCurrentType(thisType) || '' == 'true'){

	
	newRows += getNewSubRow('Call_papers', '/nmeth/authors/call/', 'Call for Papers');
	
	newRows += getNewSubRow('Editorial_policies', '/authors/editorial_policies/index.html', 'Editorial policies');
	
	newRows += getNewSubRow('How_to_submit', '/nmeth/authors/submit/index.html', 'How to submit');
	
	newRows += getNewSubRow('Content_types', '/nmeth/authors/article_types/index.html', 'Content types');
	
	newRows += getNewSubRow('Editorial_process', '/nmeth/authors/ed_process/index.html', 'Manuscript decisions');
	
	newRows += getNewSubRow('About_the_editors', '/nmeth/about/about_eds/index.html', 'About the editors');
			
	newRows += getNewSubRow('Masthead', '/nmeth/about/masthead/index.html', 'Masthead');
	

	if (isCurrentType(thisType) && thisType != getPageType()){
		hilite(thisType);
	}
	
	
	

}

document.write (getSubRow(newRows));
</script></td></tr><tr><td id="Online submission_1" style="background-color:#D1E1EA" colspan="3"><table cellpadding="0" cellspacing="0" border="0" width="130"><tr><td height="20"><img id="Online_submission_navimage" alt="Online submission" border="0" align="absmiddle" src="/includes/rj_globnavimages/nmeth_icon_onlinesub.gif"></td><td><a class="leftnav2" href="http://mts-nmeth.nature.com" target="_blank" id="Online_submission_link">Online submission</a></td></tr></table></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="Permissions_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Permissions_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="http://npg.nature.com/npg/servlet/Content?data=xml/01_reprints.xml&amp;style=xml/01_reprints.xsl" target="_blank" id="Permissions_link">Permissions</a></td><td id="Permissions_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td height="12" colspan="3"><img alt="" height="12" width="5" src="/images/spacer.gif"></td></tr><tr><td id="For_referees_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="For_referees_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/authors/editorial_policies/peer_review.html" id="For_referees_link">For referees</a></td><td id="For_referees_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td height="12" colspan="3"><img alt="" height="12" width="5" src="/images/spacer.gif"></td></tr><tr><td id="sample_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="sample_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/nmeth/journal/v4/n1/index.html" id="sample_link">Free online issue</a></td><td id="sample_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td height="12" colspan="3"><img alt="" height="12" width="5" src="/images/spacer.gif"></td></tr>

<!--

<tr><td id="About_the_journal_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="About_the_journal_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/nmeth/about/index.html" id="About_the_journal_link">About the journal</a></td><td id="About_the_journal_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"><script LANGUAGE="JavaScript">

var thisType = 'About_the_journal';

addSubLevel(thisType, new Array("About_the_editors","Editorial_policies","Masthead"));

var newRows = '';
if (isCurrentType(thisType) || '' == 'true'){

	
	newRows += getNewSubRow('About_the_editors', '/nmeth/about/about_eds/index.html', 'About the editors');
	
	newRows += getNewSubRow('Editorial_policies', '/nmeth/about/ed_policies/index.html', 'Editorial policies');
	
	newRows += getNewSubRow('Masthead', '/nmeth/about/masthead/index.html', 'Masthead');
	

	if (isCurrentType(thisType) && thisType != getPageType()){
		hilite(thisType);
	}
	
	
	

}

document.write (getSubRow(newRows));
</script>-->


</td></tr><tr><td id="Contact_the_journal_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Contact_the_journal_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/nmeth/contact/index.html" id="Contact_the_journal_link">Contact the journal</a></td><td id="Contact_the_journal_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="Subscribe_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Subscribe_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="https://www.sunbeltfs.com/forms/nh/subscribe.asp?eid=D50401" target="_blank" id="Subscribe_link">Subscribe</a></td><td id="Subscribe_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"><script LANGUAGE="JavaScript">

var thisType = 'Subscribe';

addSubLevel(thisType, new Array("new-sub","renew-sub","paid-sub","change-address"));

var newRows = '';
if (isCurrentType(thisType) || 'true' == 'true'){

	
	newRows += getNewSubRow('new-sub', 'http://www.sunbeltfs.com/forms/select.asp?pid=NH&arq=subscribe&addurl=eid=D50401', 'New Subscription');
	
	newRows += getNewSubRow('renew-sub', 'http://www.sunbeltfs.com/forms/select.asp?pid=NH&arq=renew&addurl=eid=D50402', 'Renew Subscription');
	
	newRows += getNewSubRow('paid-sub', 'https://secure.nature.com/subscribe/nmeth', 'Paid Subscriptions');
	
	newRows += getNewSubRow('change-address', 'http://www.sunbeltfs.com/forms/select.asp?pid=NH&arq=renew&addurl=eid=D50403', 'Change of Address');
	

	if (isCurrentType(thisType) && thisType != getPageType()){
		hilite(thisType);
	}
	
	
	

}

document.write (getSubRow(newRows));
</script></td></tr><tr><td id="jobs_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="jobs_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="http://www.nature.com/naturejobs/science/jobs/" target="_blank" id="jobs_link">naturejobs</a></td><td id="jobs_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="Advertising_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Advertising_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/advertising/index.html" target="_blank" id="Advertising_link">For Advertisers</a></td><td id="Advertising_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="Work_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Work_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="http://www.nature.com/npg_/work/index.html" id="Work_link">work@npg</a></td><td id="Work_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="Reprints_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Reprints_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="http://www.naturereprints.com" target="_blank" id="Reprints_link">naturereprints</a></td><td id="Reprints_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td id="About_this_site_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="About_this_site_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/nmeth/site/index.html" id="About_this_site_link">About this site</a></td><td id="About_this_site_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td height="12" colspan="3"><img alt="" height="12" width="5" src="/images/spacer.gif"></td></tr><tr><td id="For_librarians_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="For_librarians_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="http://npg.nature.com/npg/servlet/Content?data=xml/07_home.xml&amp;style=xml/07_home.xsl" target="_blank" id="For_librarians_link">For librarians</a></td><td id="For_librarians_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr><tr><td height="12" colspan="3"><img alt="" height="12" width="5" src="/images/spacer.gif"></td></tr><tr><td id="Application_notes_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="Application_notes_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/nmeth/appnotes/submit/index.html" target="_blank" id="Application_notes_link">Application notes</a></td><td id="Application_notes_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"></td></tr></table>	<script LANGUAGE="JavaScript">
	hilite(getPageType());
</script>
		
<table cellpadding="0" cellspacing="0" border="0" width="130"><tr><td height="20" colspan="3">&nbsp;</td></tr><tr><td bgcolor="#cc0033" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td class="leftnavOn" bgcolor="#cc0033">NPG Resources</td><td bgcolor="#cc0033" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/nature/">Nature</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/nbt/">Nature Biotechnology</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/nprot/">Nature Protocols</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/ng/">Nature Genetics</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/nchembio/">Nature Chemical Biology</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/ncb/">Nature Cell Biology</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/neuro/">Nature Neuroscience</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/nrg/">Nature Reviews Genetics</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/nrm/">Nature Reviews Molecular Cell Biology</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/nrd/">Nature Reviews Drug Discovery</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td><td bgcolor="#EAEAEA"><a target="_blank" class="leftnav3" href="http://www.nature.com/natureconferences/">Nature Conferences</a></td><td bgcolor="#EAEAEA" width="5"><img alt="" height="20" width="5" src="/images/spacer.gif"></td></tr><tr><td height="1" colspan="3"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr></table>
		
		
<table width="130" border="0" cellspacing="0" cellpadding="0">
<tr>
<td colspan="2" height="20"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="20" alt="" /></td>
</tr>
<tr>
<td colspan="2" height="20">
<table width="130" border="0" cellspacing="0" cellpadding="0" bgcolor="#cc0033">
<tr>
<td width="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="20" alt="" /></td>
<td><a href="//" class="leftnavOn">NPG Subject areas</a></td>
</tr>
</table>
</td>
</tr>
<tr>
<td colspan="2" height="8"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="8" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_bio.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/biotech/" class="subject">Biotechnology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_can.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/cancer/" class="subject">Cancer</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_chem.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/chemistry/" class="subject">Chemistry</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_cli.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/clinicalmedicine/" class="subject">Clinical
Medicine</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_den.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/dentistry/" class="subject">Dentistry</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_dev.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/development/" class="subject">Development</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_dru.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/drugdisc/" class="subject">Drug Discovery</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_ear.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/earthsciences/" class="subject">Earth Sciences</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_evo.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/evoeco/" class="subject">Evolution &amp; Ecology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_gen.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/genetics/" class="subject">Genetics</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_imm.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/immuno/" class="subject">Immunology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_mat.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/materials/" class="subject">Materials Science</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_med.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/medicalresearch/" class="subject">Medical Research</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_mic.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/micro/" class="subject">Microbiology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_mol.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/molcellbio/" class="subject">Molecular Cell
Biology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_neu.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/neurosci/" class="subject">Neuroscience</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_pha.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/pharma/" class="subject">Pharmacology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td><img src="/includes/rj_globnavimages/areas_phy.gif"  width="8" height="8" alt="" /></td>
<td><a href="http://www.nature.com/physics/" class="subject">Physics</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td colspan="2" height="6">
<table width="130" border="0" cellspacing="0" cellpadding="0">
<tr>
<td width="5"><img src="/includes/rj_globnavimages/arrow_red.gif" width="3" height="5" alt="" /></td>
<td><a href="http://npg.nature.com/npg/forms/04_cat.jsp" class="subjectred">Browse
all publications</a></td>
</tr>
</table>
</td>
</tr>
<tr>
<td colspan="2" height="5"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="5" alt="" /></td>
</tr>
<tr>
<td colspan="2" height="1"><img src="/includes/rj_globnavimages/spacer.gif" width="5" height="1" alt="" /></td>
</tr>
</table>
		
	</td><td width="15"><img alt="" height="1" width="15" src="/images/spacer.gif"/></td><td width="470" valign="top"><table cellpadding="0" cellspacing="0" border="0" width="470"><tr xmlns=""><td valign="top" height="21"><img alt="" height="21" width="1" src="/images/spacer.gif"></td></tr><tr xmlns=""><td valign="top" height="22"><span class="categ_large">Article</span></td></tr><tr xmlns=""><td><img alt="" height="1" width="470" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr xmlns=""><td height="14"><img alt="" height="14" width="1" src="/images/spacer.gif"></td></tr><tr><td><span xmlns="" class="blacksml"><i>Nature Methods</i> 
		- <b>5</b>, 183 - 188 (2008)
 <br>Published online: 20 January 2008;  | doi:10.1038/nmeth.1179
				
			</span>
<h2 xmlns="">Whole-genome sequencing and variant discovery in <i>C. elegans</i></h2><p xmlns=""><span class="author">LaDeana W Hillier<sup>1,&nbsp;</sup><sup>3</sup>, Gabor T Marth<sup>2,&nbsp;</sup><sup>3</sup>, Aaron R Quinlan<sup>2</sup>, David Dooling<sup>1</sup>, Ginger Fewell<sup>1</sup>, Derek Barnett<sup>2</sup>, Paul Fox<sup>1</sup>, Jarret I Glasscock<sup>1</sup>, Matthew Hickenbotham<sup>1</sup>, Weichun Huang<sup>2</sup>, Vincent J Magrini<sup>1</sup>, Ryan J Richt<sup>1</sup>, Sacha N Sander<sup>1</sup>, Donald A Stewart<sup>2</sup>, Michael Stromberg<sup>2</sup>, Eric F Tsung<sup>2</sup>, Todd Wylie<sup>1</sup>, Tim Schedl<sup>1</sup>, Richard K Wilson<sup>1</sup>&nbsp;&amp;&nbsp;Elaine R Mardis<sup>1</sup><p class="blacksml"><sup>1</sup>&nbsp;Washington University School of Medicine, Department of Genetics and Genome Sequencing Center, 4444 Forest Park Blvd., St. Louis, Missouri 63108, USA.</p><p class="blacksml"><sup>2</sup>&nbsp;Boston College, Department of Biology, 140 Commonwealth Ave., Chestnut Hill, Massachusetts 02467, USA.</p><p class="blacksml"><sup>3</sup>&nbsp;These authors contributed equally to this work.</p><span class="blacksml">
Correspondence should be addressed to Elaine R Mardis <a href="mailto:emardis@watson.wustl.edu">emardis@watson.wustl.edu</a></span></span></p><table xmlns="" cellpadding="0" cellspacing="0" border="0" width="450"><tr><td height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr><tr><td height="1"><img alt="" height="1" width="450" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr><td height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr></table><a xmlns="" name="abstract"></a><span xmlns="" class="articletext"><b>Massively parallel sequencing instruments enable rapid and inexpensive DNA sequence data production. Because these instruments are new, their data require characterization with respect to accuracy and utility. To address this, we sequenced a <i>Caernohabditis elegans</i> N2 Bristol strain isolate using the Solexa Sequence Analyzer, and compared the reads to the reference genome to characterize the data and to evaluate coverage and representation. Massively parallel sequencing facilitates strain-to-reference comparison for genome-wide sequence variant discovery. Owing to the short-read-length sequences produced, we developed a revised approach to determine the regions of the genome to which short reads could be uniquely mapped. We then aligned Solexa reads from <i>C. elegans</i> strain CB4858 to the reference, and screened for single-nucleotide polymorphisms (SNPs) and small indels. This study demonstrates the utility of massively parallel short read sequencing for whole genome resequencing and for accurate discovery of genome-wide polymorphisms.</b></span><table xmlns="" cellpadding="0" cellspacing="0" border="0" width="470"><tr><td height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr><tr><td height="1"><img alt="" height="1" width="470" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr><td height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr></table><span xmlns="" class="articletext">In 1998 the decoding of the first animal genome sequence, that of <i>C. elegans</i>, was published<sup><a href="#B1">1</a></sup>. <i>C. elegans</i> was first suggested as a model organism in the 1960s by Sydney Brenner, and subsequent work produced a physical map of its genome<sup><a href="#B2">2</a></sup>. As a result, the <i>C. elegans</i> genome sequencing project formed the cornerstone of efforts ultimately aimed at decoding the human genome<sup><a href="#B3">3,   
</a></sup><sup><a href="#B4">4</a></sup>. The entire <i>C. elegans</i> biology community has benefited enormously from the availability of the genome sequence and the ever-improving genome annotation<sup><a href="#B5">5</a></sup>, and from the comparative power of the availability of sequenced genomes for <i>C. elegans</i>' relatives such as <i>C. briggsae</i><sup><a href="#B6">6</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The emerging availability of massively parallel sequencing instrumentation provides the capability to resequence genomes in a fraction of the time, effort and expense than ever before. Compared to capillary sequencing, these instruments produce relatively short-read-length sequences that require characterization, including read error profiles and base call accuracy (which we refer to as base quality) values. Furthermore, the general utility of short read sequences, coverage models for resequencing and approaches for read mapping to reference genomes requires investigation. To address these, we sequenced an isolate of the <i>C. elegans</i> N2 Bristol strain using the <a target="_blank" href="http://www.biocompare.com/natureproducts/go.asp?id=nmeth.1179_p_p1">Solexa Sequence Analyzer (Illumina Inc.)</a>. Our analyses of these sequences included (i) an elucidation of the Solexa read error model, (ii) an evaluation of sequence differences between the two isolates and (iii) identification and investigation of representational biases in Solexa data. We revealed possible sequencing errors in the <i>C. elegans</i> reference genome, and putative variants that had occurred in our passaged N2 Bristol strain.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Massively parallel sequencing can be applied to strain-to-reference comparisons that reveal genome-wide sequence differences, either for evolutionary studies or for discovering genetic variation that may explain phenotypic variation. Implementing this application requires a new approach that assesses the fraction of a genome to which short read sequences can be uniquely mapped because they are more susceptible to multiple placements than are longer capillary instrument&#8211;derived sequences. Computational identification and markup of these 'microrepeats' is therefore an important precursor to accurate short-read analysis, and must allow for mismatches resulting from sequencing errors or polymorphisms. We aligned Solexa sequence reads from the <i>C. elegans</i> strain CB4858 (originally isolated in Pasadena, California, USA)<sup><a href="#B7">7</a></sup> to the microrepeat masked N2 Bristol reference sequence, and identified SNPs and small indels with a modified PolyBayes<sup><a href="#B8">8</a></sup> version. Orthologous validation yielded a high validation rate.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Results"></a><span xmlns="" class="heading3">Results</span><br xmlns=""><a xmlns="" name="Experimental design"></a><span xmlns="" class="articletext"><b>Experimental design</b></span><br xmlns=""><span xmlns="" class="articletext">In this study we explored two applications of Solexa sequencing: (i) genome resequencing and (ii) genome-wide polymorphism discovery. For the first application, we sequenced an isolate of the <i>C. elegans</i> N2 Bristol strain at a high coverage depth with single-end reads and at a much lower coverage depth with paired-end reads. Using the reference genome as our alignment target, we determined an accuracy estimate and an error model for Solexa reads. We next aligned all reads possible using a tiered approach (<a href="#f1">Fig. 1</a>), identified sequence differences between the two isolates and evaluated both representational bias and copy-number detection.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f1"></a><table xmlns="" cellpadding="10" cellspacing="0" border="0" width="470"><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#D1E1EA"><table cellpadding="0" cellspacing="0" border="0" width="100%"><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F1.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F1.html">Figure&nbsp;1.</a>&nbsp;N2 Bristol Solexa read analysis.</b></td></tr><tr><td height="15"><img alt="" height="1" width="160" src="/images/spacer.gif"></td><td height="15"><img alt="" height="1" width="100%" src="/images/spacer.gif"></td></tr><tr><td width="160" valign="top"><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F1.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F1.html"><img border="0" alt="Figure&nbsp;1 thumbnail" src="/nmeth/journal/v5/n2/thumbs/nmeth.1179-F1.jpg"></a></td><td class="blacksml" width="270" valign="top"><p class="blacksml">The diagram shows the processing steps used to evaluate Solexa single-end reads from the N2 Bristol isolate. The majority of reads mapped exactly to the reference genome.</p><br><br><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F1.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F1.html"><img border="0" align="bottom" alt="Full Figure" height="11" width="17" src="/nmeth/images/fullsize_icon.gif"></a><a class="sml" onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F1.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F1.html">Full Figure and legend (49K)</a></td></tr></table></td></tr><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr></table><span xmlns="" class="articletext">We developed a genome-wide polymorphism discovery approach by first sequencing <i>C. elegans</i> strain CB4858, using Solexa single-end reads of about ninefold coverage. To decrease the possibility of erroneous variant detection because of paralogous read placements, we identified and masked 'microrepeat' regions in the genome based on a 32-bp read length. We then aligned CB4858 reads to the reference genome using Mosaik and applied a modified PolyBayes version to detect variants. Our predicted polymorphic sites were validated by PCR amplification and Sanger sequencing at a high rate.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Resequencing a C. elegans N2 Bristol strain isolate"></a><span xmlns="" class="articletext"><b>Resequencing a <i>C. elegans</i> N2 Bristol strain isolate</b></span><br xmlns=""><span xmlns="" class="articletext">We used the single-end <i>C. elegans</i> N2 Bristol reads to evaluate the overall accuracy and quality of Solexa pipeline passed reads. <a href="#t1">Table 1</a> provides several metrics of our Solexa single-end read dataset, including Eland alignment results to the ws170 release of the <i>C</i>. <i>elegans</i> reference genome<sup><a href="#B9">9</a></sup> (<a target="_blank" href="http://www.wormbase.org">http://www.wormbase.org</a>). Based on the Eland metrics, we estimated 20-fold coverage of N2 Bristol for the quality passed and aligned single-end reads.</span><br xmlns=""><br xmlns=""><a xmlns="" name="t1"></a><table xmlns="" cellpadding="10" cellspacing="0" border="0" width="470"><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#D1E1EA"><table cellpadding="0" cellspacing="0" border="0" width="100%"><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T1.html','Table_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T1.html">Table&nbsp;1.</a>&nbsp;Solexa run metrics for N2 Bristol and CB 4858 single-end reads</b></td></tr><tr><td height="15"><img alt="" height="1" width="160" src="/images/spacer.gif"></td><td height="15"><img alt="" height="1" width="100%" src="/images/spacer.gif"></td></tr><tr><td width="160" valign="top"><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T1.html','Table_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T1.html"><img border="0" alt="Table&nbsp;1 thumbnail" src="/includes/rj_globnavimages/table_thumb.gif"></a></td><td class="blacksml" width="270" valign="top"><br><br><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T1.html','Table_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T1.html"><img border="0" align="bottom" alt="Full Table" height="11" width="17" src="/nmeth/images/fullsize_icon.gif"></a><a class="sml" onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T1.html','Table_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T1.html">Full Table</a></td></tr></table></td></tr><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr></table><span xmlns="" class="articletext">We performed read alignment with EagleDiscoverer, and subsequent error analysis revealed that 57.2% of the uniquely mapping single-end reads contained zero mismatches and 79.9% had 0 or 1 mismatch. We determined the full distribution of mismatches for the Solexa N2 Bristol reads (<a href="#f2">Fig. 2</a>), and the position-specific dependency of Solexa base calls at phred qualities of 25 and 30 (<a href="#f3">Fig. 3</a>), which illustrates that the base accuracy for a given base quality value depends upon that base's position in the sequence read.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f2"></a><table xmlns="" cellpadding="10" cellspacing="0" border="0" width="470"><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#D1E1EA"><table cellpadding="0" cellspacing="0" border="0" width="100%"><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F2.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F2.html">Figure&nbsp;2.</a>&nbsp;Accuracy distribution of N2 Bristol Solexa single-end reads.</b></td></tr><tr><td height="15"><img alt="" height="1" width="160" src="/images/spacer.gif"></td><td height="15"><img alt="" height="1" width="100%" src="/images/spacer.gif"></td></tr><tr><td width="160" valign="top"><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F2.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F2.html"><img border="0" alt="Figure&nbsp;2 thumbnail" src="/nmeth/journal/v5/n2/thumbs/nmeth.1179-F2.jpg"></a></td><td class="blacksml" width="270" valign="top"><p class="blacksml">As described in the text, after alignment of N2 Bristol Solexa reads to the reference genome sequence using EagleDiscoverer and tabulating any differences between the two sequences, we determined that <img src="/__chars/math/special/sim/black/med/base/glyph.gif" style="border:0; vertical-align:baseline;" alt="approx" />80% of the reads exhibited 0 or 1 mismatch when uniquely aligned to the reference genome.</p><br><br><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F2.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F2.html"><img border="0" align="bottom" alt="Full Figure" height="11" width="17" src="/nmeth/images/fullsize_icon.gif"></a><a class="sml" onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F2.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F2.html">Full Figure and legend (18K)</a></td></tr></table></td></tr><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr></table><a xmlns="" name="f3"></a><table xmlns="" cellpadding="10" cellspacing="0" border="0" width="470"><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#D1E1EA"><table cellpadding="0" cellspacing="0" border="0" width="100%"><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F3.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F3.html">Figure&nbsp;3.</a>&nbsp;Position dependency of base calling accuracy for N2 Bristol Solexa single-end reads.</b></td></tr><tr><td height="15"><img alt="" height="1" width="160" src="/images/spacer.gif"></td><td height="15"><img alt="" height="1" width="100%" src="/images/spacer.gif"></td></tr><tr><td width="160" valign="top"><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F3.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F3.html"><img border="0" alt="Figure&nbsp;3 thumbnail" src="/nmeth/journal/v5/n2/thumbs/nmeth.1179-F3.jpg"></a></td><td class="blacksml" width="270" valign="top"><p class="blacksml">The calculated based quality is shown as a distribution at phred base quality 25 or at base quality 30.</p><br><br><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F3.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F3.html"><img border="0" align="bottom" alt="Full Figure" height="11" width="17" src="/nmeth/images/fullsize_icon.gif"></a><a class="sml" onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F3.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F3.html">Full Figure and legend (62K)</a></td></tr></table></td></tr><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr></table><span xmlns="" class="articletext">To determine Solexa N2 Bristol single-end read coverage, we first devised an iterative read-alignment strategy for these reads (<a href="#f1">Fig. 1</a> and <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Methods</a> online). We then determined the average coverage of the genome to be 19.2 (s.d. = 9.0; <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Fig. 1</a> online).</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">We examined the genome for over-represented regions and found <img src="/__chars/math/special/sim/black/med/base/glyph.gif" style="border:0; vertical-align:baseline;" alt="approx" />1.7% of the genome had &gt;40-fold coverage in unique 32-mers. We expected ribosomal DNA genes to have higher than average coverage because the reference represents these as single copies but they actually exist in multiple copies. By examining unique 32-mers within rDNA segments, we found evidence of excess coverage (&gt;100 <img src="/__chars/math/special/times/black/med/base/glyph.gif" style="border:0; vertical-align:middle;" alt="times" /> for the chromosome 1 rDNA unique 32-mers). This finding lends credibility to the use of read coverage as a quantitative metric of region-specific copy number. Based on our analysis of regions with higher than average coverage, combined with the assembly and analysis of unmapped reads (see <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Methods</a>), we estimate a maximum of <img src="/__chars/math/special/sim/black/med/base/glyph.gif" style="border:0; vertical-align:baseline;" alt="approx" />0.5 Mb of repetitive sequence is missing from the <i>C. elegans</i> reference genome.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">To determine the genome coverage of the sequence reads we proceeded as follows. After aligning exactly matching 32-bp reads, exactly matching quality-trimmed reads (that is, reads with at least 20 consecutive base pairs of quality score <img src="/__chars/greater/special/ge/black/med/base/glyph.gif" style="border:0; vertical-align:baseline;" alt="greater than or equal to" />20) and quality-trimmed reads with 2 or more mismatches to the reference genome, we found that 99.9% of the unique <i>C. elegans</i> genome was covered, mostly in large spans; the longest was 194 kb on chromosome V. Of the regions left uncovered by Solexa reads, there were 9,492 gaps comprising 95,913 bp. These coverage gaps ranged in size from 1 base to over 1,000 bases; 77.9% were 1&#8211;9 bp and another 36.8% were 10&#8211;50 bp. If we only consider 32-mer exact mapping reads, the largest coverage gap was a 4,601-bp region on chromosome X (5907815&#8211;5912415). Notably, the entire 4,600-bp region is bounded by a transposon (TC5A&#035;DNA/Tc4) in the reference sequence and is completely contained within a single fosmid clone (H05L03) that extends into an overlapping yeast artificial chromosome (YAC; Y23B4A).</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">There were a total of 1,728 zero-base-pair gaps discovered&#8212;areas of the genome where two adjacent reference genome 32-mers were covered by reads that aligned exactly to each 32-mer but across which no exactly matching spanning read exists. Of these, 1,564 (90%) had single read representation of the flanking 32-mer, consistent with the notion that these regions are under-represented by Solexa reads. Further investigation of these coverage gaps revealed that (i) they are located primarily in noncoding sequence (2% are in exons), (ii) only a few regions could be explained by hairpin formation (see <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Data</a> and <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Table 1</a> online) and (iii) the A+T content in these regions is substantially higher than the genome average (85% versus 65%, respectively; <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Fig. 2</a> online). Furthermore, this A+T bias more likely occurs during amplification than during sequencing (see <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Data</a>). We identified 125 zero-base-pair gaps with a non-identical spanning Solexa read, suggesting an insertion in the Solexa-sequenced strain. We resequenced 22 of these and validated them as true differences between the two N2 Bristol isolates.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Genomic alignment of nonexact match reads (that is, N2 Bristol single-end reads without an exact match to <i>C. elegans</i>; <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Methods</a>) allowed us determine differences between the two N2 Bristol isolates and to identify possible errors in the reference sequence as these reads are highly similar but contain inserted or deleted bases that preclude an exact match. Here three or more Solexa reads were required to predict a reference error, to reduce contributions from Solexa base calling errors. Such alignments putatively identified 2,981 insertions, deletions and indels of 1&#8211;20 bp. Of these, 2,082 occurred at positions also having exactly matching Solexa reads, thus confirming the reference sequence and indicating an allelic polymorphism between the two N2 Bristol isolates. By contrast, 235 of the indels occurred in regions with no perfectly aligning Solexa read, suggesting a possible error in the <i>C. elegans</i> reference genome, and indicating a potential indel error rate of 1 in 373 kb. We detected 56 different putative deletion events, for which a Solexa read spanned one or more bases in the reference genome, aligning immediately on either side. Alternatively, these could be insertions in the reference genome. Lastly, 53 different putative insertions were suggested (by 502 reads) for which a Solexa read had extra bases relative to the reference. These also could be deletions in the reference.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">We identified 1,396 nonrepetitive, uncovered regions with at least one read having an unaligned or mismatched base, suggesting a Solexa base-calling error, a polymorphism in the Solexa-sequenced N2 Bristol isolate or a substitution error in the reference genome. Of these, 1,011 were covered by more than one read, and 544 were covered by more than two reads. These suggest a maximum substitution error rate in the <i>C. elegans</i> reference sequence of 1 in 99 kb. We included a limited number of these putative errors in our validation efforts, described below.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">We were able to produce and analyze limited numbers of paired-end reads for <i>C. elegans</i> N2 Bristol, providing an average coverage of 0.84 <img src="/__chars/math/special/times/black/med/base/glyph.gif" style="border:0; vertical-align:middle;" alt="times" /> with a mean physical coverage (measured by the span of matching paired ends) of 3.08 <img src="/__chars/math/special/times/black/med/base/glyph.gif" style="border:0; vertical-align:middle;" alt="times" />. Because paired-end reads are used to evaluate structural variation based on deviations in end read distance from expectation<sup><a href="#B10">10</a></sup>, we determined that 37,352 read pairs had a mapped distance &gt;3 s.d. from the 218-bp average (<a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Data</a>; 36,209 were &lt;104 bp and 1,143 were &gt;332 bp). If we required more than one read pair placement to confirm an event, only 5,908 pairs remained (5,670 were &lt;104 bp). Notably, these 5,670 read pairs spaced <img src="/__chars/math/special/sim/black/med/base/glyph.gif" style="border:0; vertical-align:baseline;" alt="approx" />100 bp closer than expected support our estimate that <img src="/__chars/math/special/sim/black/med/base/glyph.gif" style="border:0; vertical-align:baseline;" alt="approx" />0.5 Mb of the <i>C. elegans</i> repetitive genome is missing from the reference genome. The vast majority of multiple read pairs that confirmed a structural variation event were within introns and/or were annotated as repetitive. Many such read pairs confirmed regions already annotated as "difficult to sequence"; about 1.5 times as many fell in genomic regions sequenced from YACs or plasmids (both clone types were used to sequence regions unclonable in cosmids for the reference genome sequencing). For example, on chromosome III, a complex tandem repeat annotated as "restriction digest data indicate 3 kb is missing from the assembly of this region" was identified by 238 Solexa paired ends placed at &gt;3 s.d. apart (50% were 332&#8211;400 base pairs apart), further substantiating the initial suspicion of misassembly. Hence, paired-end data enhance the utility of Solexa reads, providing an important tool for identifying putative structural variation.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Polymorphism discovery in C. elegans strain CB4858"></a><span xmlns="" class="articletext"><b>Polymorphism discovery in <i>C. elegans</i> strain CB4858</b></span><br xmlns=""><span xmlns="" class="articletext">We sequenced an isolate of the CB4858 strain using the Solexa technology to produce <img src="/__chars/math/special/sim/black/med/base/glyph.gif" style="border:0; vertical-align:baseline;" alt="approx" />ninefold coverage in single-end reads. This strain was selected because previous work had suggested a polymorphism rate of 1:1,600 (ref. <a href="#B11">11</a>). The Solexa analysis pipeline produced metrics of our Solexa single-end read data for CB4858 (<a href="#t1">Table 1</a>).</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">As a precursor to variant discovery in CB4858, we identified regions of the reference genome with a high potential for ambiguous read alignment, based on the Solexa 32-bp read length. First, we identified all unique 32-mers in the reference sequence, but as our error rate analysis (<a href="#f2">Fig. 2</a>) indicated a drop-off in the error rate beyond 2 errors per read, we defined a repetitive 32-mer as one that appears in the genome more than once, allowing 0&#8211;2 mismatched bases (substitutions, insertions or deletions). We called these 'microrepeats' to distinguish them from repeats marked by the RepeatMasker program<sup><a href="#B12">12</a></sup>, which masks 14.5% of the bases in the genome. The fraction of the genome comprising perfect and near-perfect microrepeats totaled 19.8%. We illustrate the relationship between RepeatMasker-masked bases and microrepeat bases identified by our methods as a Venn diagram (<a href="#f4">Fig. 4</a>). Although there is a substantial overlap (11.11%) between the regions masked by both methods, 8.7% of the genome that we identified as microrepeats was not masked by RepeatMasker. Conversely, 3.4% of the genome was masked by RepeatMasker only, indicating that some fraction of <i>C. elegans</i> repeat elements can be uniquely sequenced with 32-bp reads. Taken together, RepeatMasker repeats and microrepeats cover 23.2% the genome.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f4"></a><table xmlns="" cellpadding="10" cellspacing="0" border="0" width="470"><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#D1E1EA"><table cellpadding="0" cellspacing="0" border="0" width="100%"><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F4.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F4.html">Figure&nbsp;4.</a>&nbsp;Repetitive content in <i>C. elegans</i>.</b></td></tr><tr><td height="15"><img alt="" height="1" width="160" src="/images/spacer.gif"></td><td height="15"><img alt="" height="1" width="100%" src="/images/spacer.gif"></td></tr><tr><td width="160" valign="top"><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F4.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F4.html"><img border="0" alt="Figure&nbsp;4 thumbnail" src="/nmeth/journal/v5/n2/thumbs/nmeth.1179-F4.jpg"></a></td><td class="blacksml" width="270" valign="top"><p class="blacksml">Venn diagram depicting the fraction of bases in the genome covered by microrepeats and by RepeatMasker, and the overlapping set.</p><br><br><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F4.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F4.html"><img border="0" align="bottom" alt="Full Figure" height="11" width="17" src="/nmeth/images/fullsize_icon.gif"></a><a class="sml" onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F4.html','Figure_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_F4.html">Full Figure and legend (28K)</a></td></tr></table></td></tr><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr></table><span xmlns="" class="articletext">Once we aligned CB4858 Solexa reads to the conservatively masked <i>C. elegans</i> genome, we applied our combined repeat masking to filter the alignments, identified high-quality sequence differences with PolyBayes, and finalized a set of 45,539 SNPs and 7,353 single-base-pair indels. This yields a rate of one SNP per 1,629.81 bp and one indel per 9,894.99 bp. Hence the pair-wise nucleotide diversity (theta) between the CB4858 and the N2 Bristol strains is 6.136 <img src="/__chars/math/special/times/black/med/base/glyph.gif" style="border:0; vertical-align:middle;" alt="times" /> 10<sup>-4</sup>, in good agreement with the <img src="/__chars/math/special/sim/black/med/base/glyph.gif" style="border:0; vertical-align:baseline;" alt="approx" />1:1,500 rate posited in a previous description of CB4858 (ref. <a href="#B11">11</a>). As 37,856,444 CB4858 Solexa reads yielded a total number of 45,539 SNPs, the 'read-per-SNP' yield was 831. All confirmed CB4858 sequence variants are available in Wormbase.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">We orthologously validated roughly 1,000 candidate SNPs and indels by PCR-directed capillary sequencing to gauge the performance of our Mosaik-PolyBayes approach. After sequencing and evaluation, we determined a SNP validation rate of 96.3% (438/455) and an 89.0% conversion rate (438/492) for candidates identified by PolyBayes (<a href="#t2">Table 2</a>). We sequenced 239 of our putative single-base indels, finding they validated (93.8%) and converted (87.7%) at practically the same rates as SNPs. Both insertions and deletions predicted in the reference genome sequence were represented (insertions: 2,948 or 47.1%, and deletions: 3,316 or 52.9%). Many of the indels were variable numbers of bases in mono-nucleotide repeats, for example, 5 versus 4 adenosines. Although mononucleotide runs are typically very difficult areas for indel detection, our high validation rate indicates that Solexa reads resolve base numbers in these runs very well. Microrepeat masking has a marked impact on accurate SNP discovery by eliminating putative SNPs and indels resulting from paralogous read mapping (<a href="#t2">Table 2</a>).</span><br xmlns=""><br xmlns=""><a xmlns="" name="t2"></a><table xmlns="" cellpadding="10" cellspacing="0" border="0" width="470"><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#D1E1EA"><table cellpadding="0" cellspacing="0" border="0" width="100%"><tr><td class="articletext" colspan="2"><b><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T2.html','Table_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T2.html">Table&nbsp;2.</a>&nbsp;PolyBayes SNP and indel validation data</b></td></tr><tr><td height="15"><img alt="" height="1" width="160" src="/images/spacer.gif"></td><td height="15"><img alt="" height="1" width="100%" src="/images/spacer.gif"></td></tr><tr><td width="160" valign="top"><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T2.html','Table_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T2.html"><img border="0" alt="Table&nbsp;2 thumbnail" src="/includes/rj_globnavimages/table_thumb.gif"></a></td><td class="blacksml" width="270" valign="top"><br><br><a onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T2.html','Table_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T2.html"><img border="0" align="bottom" alt="Full Table" height="11" width="17" src="/nmeth/images/fullsize_icon.gif"></a><a class="sml" onclick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T2.html','Table_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=yes,resizable=yes,copyhistory=yes,width=617,top=20,left=20');return false;" href="/nmeth/journal/v5/n2/fig_tab/nmeth.1179_T2.html">Full Table</a></td></tr></table></td></tr><tr><td><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr></table><span xmlns="" class="articletext">We estimated false negative rates for PolyBayes by running PolyPhred<sup><a href="#B13">13,   
</a></sup><sup><a href="#B14">14,   
</a></sup><sup><a href="#B15">15</a></sup> (version 5.0) on the validation trace data. This algorithm indicated PolyBayes had missed 26 SNPs, for a false negative rate of 3.75%.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">To determine the chromosomal distribution of CB4858 polymorphisms, we placed CB4858 SNPs and indels along the six <i>C. elegans</i> chromosomes, and identified both chromosome-wide and chromosomal position&#8211;specific differences (<a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Data</a> and <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Fig. 3</a> online). Our data confirmed an earlier study in <i>C. elegans</i><sup><a href="#B16">16</a></sup> suggesting that nonsynonymous substitution rates are higher in the first and second codon positions than in the third (<a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Fig. 4</a> online). Furthermore, over half of CB4858 SNPs positioned in exons putatively introduce an amino acid change.</span><br xmlns=""><br xmlns=""><table xmlns="" cellpadding="0" cellspacing="0" border="0" width="470"><tr><td rowspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td><td align="right"><img alt="" height="3" width="5" src="/nmeth/images/arrow_black_up.gif">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td height="7" align="right"><img alt="" height="7" width="1" src="/images/spacer.gif"></td></tr><tr><td colspan="2" height="1"><img alt="" height="1" width="470" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr><td colspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr></table><a xmlns="" name="Discussion"></a><span xmlns="" class="heading3">Discussion</span><br xmlns=""><span xmlns="" class="articletext">Massively parallel sequencing approaches hold great promise for genome-wide discovery of sequence variation, when comparing different isolates or strains to reference genomes. It is apparent that short-read technologies must initially be characterized with respect to their quality and accuracy, providing a baseline for devising analytical methods. Dramatically shorter read lengths also increase the coverage level needed for adequate depth and breadth of reads to predict variation with high confidence, when compared to capillary sequencing reads. Although these short reads presently are too short for <i>de novo</i> assembly, producing regional assemblies of resequencing reads, followed by reference genome alignment, apparently has merit for detecting insertions and deletions, and should be pursued in future resequencing efforts.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Paired-end reads clearly increase the power to properly interpret problematic areas of the genome, including collapsed or misassembled repeats, and to detect structural variations. As genomes increase in size and complexity, paired ends will also be more efficiently placed than single-end reads, as only one end of each read pair needs a unique genome placement to properly place most reads, given that a precise paired-end read distance has been achieved in library construction.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Solexa reads provide a rapid vehicle for genome-wide SNP and small indel discovery, once additional masking of 'microrepeat' sequences is achieved. Aside from SNP or indel discovery, whole-genome resequencing also can be used after random mutagenesis to identify and characterize each mutagenized base. Our results establish the utility of short-read-length massively parallel sequencing for the accurate discovery of both single-nucleotide and small insertion-deletion polymorphisms, and establishes a framework for human genome resequencing toward similar discovery aims.</span><br xmlns=""><br xmlns=""><table xmlns="" cellpadding="0" cellspacing="0" border="0" width="470"><tr><td rowspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td><td align="right"><img alt="" height="3" width="5" src="/nmeth/images/arrow_black_up.gif">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td height="7" align="right"><img alt="" height="7" width="1" src="/images/spacer.gif"></td></tr><tr><td colspan="2" height="1"><img alt="" height="1" width="470" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr><td colspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr></table><a xmlns="" name="Methods"></a><span xmlns="" class="heading3">Methods</span><br xmlns=""><a xmlns="" name="Determining Solexa single-end read accuracy."></a><span xmlns="" class="articletext"><b>Determining Solexa single-end read accuracy.</b></span><br xmlns=""><span xmlns="" class="articletext">To isolate sequencing errors from simple alignment errors, we used a version of the Smith-Waterman&#8211;based global alignment algorithm that reports all optimal and suboptimal alignments above a prespecified alignment score (EagleDiscover; W.H., unpublished data). Although time-intensive, this algorithm identifies all alignable positions in the <i>C. elegans</i> genome for a 32-bp read. Here we generated three random sample sets of 20,000 Solexa N2 Bristol single-end reads and aligned each read set to the unmasked reference genome, allowing up to 4 mismatches (substitution, insertion or deletion). For further consideration of accuracy, we kept only reads that aligned at a single locus in the genome. For each of the three read sets we tabulated the number of sequence differences between each read and the reference sequence, and combined the results to make a histogram of reads (<a href="#f2">Fig. 2</a>). Then we evaluated unique alignments to calculate the observed error rate at each base position for a given Solexa base quality score. We converted these rates to phred scores (Solexa base qualities are expressed as a probability of each of the 4 bases being the correct call rather than as a single phred-like probability of correctness) and graphed the dependence of observed base quality on base position (<a href="#f3">Fig. 3</a>).</span><br xmlns=""><br xmlns=""><a xmlns="" name="Alignment and analysis of Solexa single-end reads."></a><span xmlns="" class="articletext"><b>Alignment and analysis of Solexa single-end reads.</b></span><br xmlns=""><span xmlns="" class="articletext">We compared Solexa N2 Bristol reads to the reference genome to identify sequence variants, to analyze coverage and to evaluate representational bias. These alignments consisted of a combination of exact hash-match based comparisons, followed by BLAST-like alignment tool (BLAT)-based comparisons. Our methods are detailed below, and are presented in a flowchart format (<a href="#f1">Fig. 1</a> and <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Data</a>).</span><br xmlns=""><br xmlns=""><a xmlns="" name="Paired-end read evaluation."></a><span xmlns="" class="articletext"><b>Paired-end read evaluation.</b></span><br xmlns=""><span xmlns="" class="articletext">We mapped paired-end reads (for example, a 25&#8211;35 bp read from each end of a <img src="/__chars/math/special/sim/black/med/base/glyph.gif" style="border:0; vertical-align:baseline;" alt="approx" />200-bp genomic fragment) from the N2 Bristol isolate to the <i>C. elegans</i> genome using the exact hash-match based method described above. After read mapping of individual paired ends, we determined final placements by asking that the 'forward' and 'reverse' read of the pair match on the same chromosome and within 1,000 bases of each other.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Mosaik alignment of CB4858 Solexa reads."></a><span xmlns="" class="articletext"><b>Mosaik alignment of CB4858 Solexa reads.</b></span><br xmlns=""><span xmlns="" class="articletext">We identified both perfect microrepeats and microrepeats with up to two mismatches (substitutions, deletions or insertions) to encompass the possibility of sequencing errors (nucleotide misincorporation or base calling) in the reads or of polymorphism in the genomes being compared. Custom scripts then produced a microrepeat-masked reference genome.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">We next aligned the Solexa CB4858 single-end reads to the microrepeat-masked <i>C. elegans</i> reference genome with our Mosaik program. Mosaik consists of two parts: the aligner (aligns each read to the reference genome separately in a pair-wise fashion) and the assembler (pads the individual reads and the reference genome sequence so that every aligned base within each read remains in register in the resulting multiple read alignment). The details of Mosaik processing are described in <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Methods</a>. The resulting multiple read alignments were then reported either in ACE<sup><a href="#B17">17</a></sup> or in binary formats used by downstream analysis software.</span><br xmlns=""><br xmlns=""><a xmlns="" name="SNP and indel discovery in strain CB4858."></a><span xmlns="" class="articletext"><b>SNP and indel discovery in strain CB4858.</b></span><br xmlns=""><span xmlns="" class="articletext">Starting with the multiple read alignments produced by the Mosaik aligner and assembler, we analyzed the resulting alignments using a version of PolyBayes<sup><a href="#B8">8</a></sup> that was completely reengineered to enable efficient analyses of millions of aligned short-read sequences. The program evaluated each aligned base and its base quality value at each position, to indicate putative SNPs and small (1&#8211;3 bp) putative indels, and their corresponding SNP probability value (P<sub>SNP</sub>). Base quality values were converted to base probabilities corresponding to every one of the four possible nucleotides (and to the probability that the nucleotide in question was an actual insertion error in the sequence). Using a Bayesian formulation<sup><a href="#B8">8</a></sup>, a P<sub>SNP</sub> (or indel probability value, as appropriate) was calculated as the likelihood that multiple different alleles are present between the reference genome sequence and the reads aligned at that position. If the probability value exceeds a prespecified threshold, the SNP or indel candidate is reported in the output. For the collection of bases contributed by such reads, a single 'consensus' base call and its base quality value are computed. The corresponding base probabilities are then used in the Bayesian P<sub>SNP</sub> calculation. In this study, we used a P<sub>SNP</sub> cutoff value of 0.7 to define a high-certainty SNP or small indel site. Validated CB4858 SNPs and indels were assigned Wormbase accession numbers pas1&#8211; pas50906.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Software availability."></a><span xmlns="" class="articletext"><b>Software availability.</b></span><br xmlns=""><span xmlns="" class="articletext">The combined microrepeat plus RepeatMasker masked genome sequence annotations and FASTA files are available at <a target="_blank" href="http://bioinformatics.bc.edu/microrepeats/elegans/">http://bioinformatics.bc.edu/microrepeats/elegans/</a>. Mosaik and the updated version of PolyBayes are now in beta release and available for users to wish to participate in software testing (<a target="_blank" href="http://bioinformatics.bc.edu/marthlab/Beta_Release">http://bioinformatics.bc.edu/marthlab/Beta_Release</a>). After the testing period, both programs will be released for public use, free of charge for academic users.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Additional methods."></a><span xmlns="" class="articletext"><b>Additional methods.</b></span><br xmlns=""><span xmlns="" class="articletext">Details of Solexa library construction and sequencing, data analysis of primary sequence data and its alignment to the <i>C. elegans</i> reference genome (both single and paired-end reads) as well as detailed descriptions of Mosaik and PolyBayes analysis of CB4858 read data and its validation are available in <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary Methods</a>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><i>Note: <a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html">Supplementary information</a> is available on the Nature Methods website</i>.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Author contributions"></a><span xmlns="" class="articletext"><b>Author contributions</b></span><br xmlns=""><span xmlns="" class="articletext">L.W.H., N2 Bristol read, coverage, variant and gap analyses; G.T.M., CB4858 SNP discovery and N2 Bristol error profile analysis; A.R.Q., CB4858 SNP discovery and validation analysis; D.D., Solexa analysis pipeline; G.F., validation assay design and analysis, D.B., Solexa base quality value analysis, P.F., preparation of N2 Bristol and CB4858 DNA, J.I.G., N2 Bristol read analysis; M.H., Solexa libraries and sequencing, W.H., microrepeat analysis, V.J.M., Solexa libraries and sequencing, R.J.R., N2 Bristol analysis; S.N.S., validation assays; D.A.S., microrepeat masking of C. elegans; M.S., Mosaik adaptation; E.F.T., microrepeat finding; T.W., N2 Bristol analysis, T.S., C. elegans strain selection; R.K.W., project origination; E.R.M., project coordination and manuscript preparation.</span><br xmlns=""><br xmlns=""><table xmlns="" cellpadding="0" cellspacing="0" border="0" width="470"><tr><td rowspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td><td align="right"><img alt="" height="3" width="5" src="/nmeth/images/arrow_black_up.gif">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td height="7" align="right"><img alt="" height="7" width="1" src="/images/spacer.gif"></td></tr><tr><td colspan="2" height="1"><img alt="" height="1" width="470" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr><td colspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr></table><span xmlns="" class="articletext">Received 19 September 2007; </span><span xmlns="" class="articletext">Accepted 21 December 2007; </span><span xmlns="" class="articletext">Published online: 20 January 2008.</span><br xmlns=""><br xmlns=""><a xmlns="" name="References"></a><span xmlns="" class="heading3">REFERENCES </span><ol xmlns="" class="articletext">
<li id="B1"><a name="B1"><!-- . --></a><i>C. elegans</i> Sequencing Consortium. <span class="atl"> Genome sequence of the nematode <i>C. elegans</i>: a platform for investigating biology</span>. <span class="journalname">Science</span> <span class="journalnumber">282</span>, 2012&#8211;2018 (<span class="cite-month-year">1998</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.282.5386.53" class="reftxt" title="Genome sequence of the nematode C. elegans: a platform for investigating biology">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851916&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Genome sequence of the nematode C. elegans: a platform for investigating biology">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000077467100034&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Genome sequence of the nematode C. elegans: a platform for investigating biology">ISI</a>&nbsp;|</li><li id="B2"><a name="B2"><!-- . --></a>Waterston, R. <i>et al</i>. <span class="atl"> The genome of the nematode <i>Caenorhabditis elegans</i></span>. <span class="journalname">Cold Spring Harb. Symp. Quant. Biol.</span> <span class="journalnumber">58</span>, 367&#8211;376 (<span class="cite-month-year">1993</span>).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7956049&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - The genome of the nematode Caenorhabditis elegans">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2MXmtFyrtw%3D%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=6ddd52484edabc4057c3886e8d493135" class="reftxt" title="Article on ChemPort - The genome of the nematode Caenorhabditis elegans">ChemPort</a>&nbsp;|</li><li id="B3"><a name="B3"><!-- . --></a>Lander, E.S. <i>et al</i>. <span class="atl"> Initial sequencing and analysis of the human genome</span>. <span class="journalname">Nature</span> <span class="journalnumber">409</span>, 860&#8211;921 (<span class="cite-month-year">2001</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/35057062" class="reftxt" title="Article on Article - Initial sequencing and analysis of the human genome">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11237011&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Initial sequencing and analysis of the human genome">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000166938800058&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Initial sequencing and analysis of the human genome">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3MXhsFCjtLc%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=bc88cf8abfcdbce03f91f62cdb886a0d" class="reftxt" title="Article on ChemPort - Initial sequencing and analysis of the human genome">ChemPort</a>&nbsp;|</li><li id="B4"><a name="B4"><!-- . --></a>International Human Genome Sequencing Consortium. <span class="atl"> Finishing the euchromatic sequence of the human genome</span>. <span class="journalname">Nature</span> <span class="journalnumber">431</span>, 931&#8211;945 (<span class="cite-month-year">2004</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nature03001" class="reftxt" title="Article on Article - Finishing the euchromatic sequence of the human genome">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15496913&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Finishing the euchromatic sequence of the human genome">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000224585600034&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Finishing the euchromatic sequence of the human genome">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXoslSlt7o%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=650beb8a132f497902e4cb8de60d7017" class="reftxt" title="Article on ChemPort - Finishing the euchromatic sequence of the human genome">ChemPort</a>&nbsp;|</li><li id="B5"><a name="B5"><!-- . --></a>Harris, T.W. <i>et al</i>. <span class="atl"> WormBase: a multi-species resource for nematode biology and genomics</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">32</span>, D411&#8211;D417 (<span class="cite-month-year">2004</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/gkh066" class="reftxt" title="WormBase: a multi-species resource for nematode biology and genomics">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=14681445&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - WormBase: a multi-species resource for nematode biology and genomics">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000188079000097&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - WormBase: a multi-species resource for nematode biology and genomics">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXhtVSrurvJ&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=abeda45104f1f84ef71511da31dbe4f3" class="reftxt" title="Article on ChemPort - WormBase: a multi-species resource for nematode biology and genomics">ChemPort</a>&nbsp;|</li><li id="B6"><a name="B6"><!-- . --></a>Stein, L.D. <i>et al</i>. <span class="atl"> The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics</span>. <span class="journalname">PLoS Biol.</span> <span class="journalnumber">1</span>, e45 (<span class="cite-month-year">2003</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1371/journal.pbio.0000045" class="reftxt" title="The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=14624247&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXptV2mtrk%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=8367f07caa277d8c82fdf3efcd2dd896" class="reftxt" title="Article on ChemPort - The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics">ChemPort</a>&nbsp;|</li><li id="B7"><a name="B7"><!-- . --></a>Hodgkin, J. &amp; Doniach, T. <span class="atl"> Natural variation and copulatory plug formation in <i>Caenorhabditis elegans</i></span>. <span class="journalname">Genetics</span> <span class="journalnumber">146</span>, 149&#8211;164 (<span class="cite-month-year">1997</span>).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9136008&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Natural variation and copulatory plug formation in Caenorhabditis elegans">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1997WW88000013&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Natural variation and copulatory plug formation in Caenorhabditis elegans">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXmsVarsLk%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=8f0e3a23801413cf687a5115979f49f9" class="reftxt" title="Article on ChemPort - Natural variation and copulatory plug formation in Caenorhabditis elegans">ChemPort</a>&nbsp;|</li><li id="B8"><a name="B8"><!-- . --></a>Marth, G.T. <i>et al</i>. <span class="atl"> A general approach to single-nucleotide polymorphism discovery</span>. <span class="journalname">Nat. Genet.</span> <span class="journalnumber">23</span>, 452&#8211;456 (<span class="cite-month-year">1999</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/70570" class="reftxt" title="Article on Article - A general approach to single-nucleotide polymorphism discovery">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10581034&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - A general approach to single-nucleotide polymorphism discovery">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000084023900023&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - A general approach to single-nucleotide polymorphism discovery">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1MXnvFOkur4%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=efc0fbc26157841b428f42d92c1af91b" class="reftxt" title="Article on ChemPort - A general approach to single-nucleotide polymorphism discovery">ChemPort</a>&nbsp;|</li><li id="B9"><a name="B9"><!-- . --></a>Bieri, T. <i>et al</i>. <span class="atl"> WormBase: new content and better access</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">35</span>, D506&#8211;D510 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/gkl818" class="reftxt" title="WormBase: new content and better access">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17099234&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - WormBase: new content and better access">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXivFGltA%3D%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=a3e04888ba877836313541a7e506a572" class="reftxt" title="Article on ChemPort - WormBase: new content and better access">ChemPort</a>&nbsp;|</li><li id="B10"><a name="B10"><!-- . --></a>Tuzun, E. <i>et al</i>. <span class="atl"> Fine-scale structural variation of the human genome</span>. <span class="journalname">Nat. Genet.</span> <span class="journalnumber">37</span>, 727&#8211;732 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/ng1562" class="reftxt" title="Article on Article - Fine-scale structural variation of the human genome">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15895083&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Fine-scale structural variation of the human genome">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000230196400019&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Fine-scale structural variation of the human genome">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXlslWhsrk%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=378250271baf8fd840168bb3b4ec2ef8" class="reftxt" title="Article on ChemPort - Fine-scale structural variation of the human genome">ChemPort</a>&nbsp;|</li><li id="B11"><a name="B11"><!-- . --></a>Denver, D.R., Morris, K. &amp; Thomas, W.K. <span class="atl"> Phylogenetics in <i>Caenorhabditis elegans</i>: an analysis of divergence and outcrossing</span>. <span class="journalname">Mol. Biol. Evol.</span> <span class="journalnumber">20</span>, 393&#8211;400 (<span class="cite-month-year">2003</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/molbev/msg044" class="reftxt" title="Phylogenetics in Caenorhabditis elegans: an analysis of divergence and outcrossing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12644560&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Phylogenetics in Caenorhabditis elegans: an analysis of divergence and outcrossing">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000181913900010&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Phylogenetics in Caenorhabditis elegans: an analysis of divergence and outcrossing">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXisFaqs7c%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=e4f2987ca33d81ad18f6c73050e07095" class="reftxt" title="Article on ChemPort - Phylogenetics in Caenorhabditis elegans: an analysis of divergence and outcrossing">ChemPort</a>&nbsp;|</li><li id="B12"><a name="B12"><!-- . --></a>Smit, A.F. <span class="atl"> The origin of interspersed repeats in the human genome</span>. <span class="journalname">Curr. Opin. Genet. Dev.</span> <span class="journalnumber">6</span>, 743&#8211;748 (<span class="cite-month-year">1996</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1016/S0959-437X(96)80030-X" class="reftxt" title="The origin of interspersed repeats in the human genome">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8994846&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - The origin of interspersed repeats in the human genome">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1996VZ21500011&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - The origin of interspersed repeats in the human genome">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXhsVWmtA%3D%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=9b9d187b948e856c883476cf496fd1b6" class="reftxt" title="Article on ChemPort - The origin of interspersed repeats in the human genome">ChemPort</a>&nbsp;|</li><li id="B13"><a name="B13"><!-- . --></a>Bhangale, T.R., Stephens, M. &amp; Nickerson, D.A. <span class="atl"> Automating resequencing-based detection of insertion-deletion polymorphisms</span>. <span class="journalname">Nat. Genet.</span> <span class="journalnumber">38</span>, 1457&#8211;1462 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/ng1925" class="reftxt" title="Article on Article - Automating resequencing-based detection of insertion-deletion polymorphisms">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17115056&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Automating resequencing-based detection of insertion-deletion polymorphisms">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28Xht1CntrfL&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=ae6e8adf5e85c892c96028c51a46b45a" class="reftxt" title="Article on ChemPort - Automating resequencing-based detection of insertion-deletion polymorphisms">ChemPort</a>&nbsp;|</li><li id="B14"><a name="B14"><!-- . --></a>Stephens, M., Sloan, J.S., Robertson, P.D., Scheet, P. &amp; Nickerson, D.A. <span class="atl"> Automating sequence-based detection and genotyping of SNPs from diploid samples</span>. <span class="journalname">Nat. Genet.</span> <span class="journalnumber">38</span>, 375&#8211;381 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/ng1746" class="reftxt" title="Article on Article - Automating sequence-based detection and genotyping of SNPs from diploid samples">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16493422&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Automating sequence-based detection and genotyping of SNPs from diploid samples">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000235589600022&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Automating sequence-based detection and genotyping of SNPs from diploid samples">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XhslCjt7w%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=ade6298a3c25f1722fe37a0b40903be6" class="reftxt" title="Article on ChemPort - Automating sequence-based detection and genotyping of SNPs from diploid samples">ChemPort</a>&nbsp;|</li><li id="B15"><a name="B15"><!-- . --></a>Nickerson, D.A., Kolker, N., Taylor, S.L. &amp; Rieder, M.J. <span class="atl"> Sequence-based detection of single nucleotide polymorphisms</span>. <span class="journalname">Methods Mol. Biol.</span> <span class="journalnumber">175</span>, 29&#8211;35 (<span class="cite-month-year">2001</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1385/1-59259-235-X:029" class="reftxt" title="Sequence-based detection of single nucleotide polymorphisms">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11462842&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Sequence-based detection of single nucleotide polymorphisms">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3MXltFajt7o%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=bd25cb775375c6d10d90844d077946b7" class="reftxt" title="Article on ChemPort - Sequence-based detection of single nucleotide polymorphisms">ChemPort</a>&nbsp;|</li><li id="B16"><a name="B16"><!-- . --></a>Koch, R., van Luenen, H.G., van der Horst, M., Thijssen, K.L. &amp; Plasterk, R.H. <span class="atl"> Single nucleotide polymorphisms in wild isolates of <i>Caenorhabditis elegans</i></span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">10</span>, 1690&#8211;1696 (<span class="cite-month-year">2000</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.GR-1471R" class="reftxt" title="Single nucleotide polymorphisms in wild isolates of Caenorhabditis elegans">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11076854&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Single nucleotide polymorphisms in wild isolates of Caenorhabditis elegans">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000165364700007&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Single nucleotide polymorphisms in wild isolates of Caenorhabditis elegans">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3cXosVWjt7c%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=dfdbe990afc3c3eb89548de02c2d0ed0" class="reftxt" title="Article on ChemPort - Single nucleotide polymorphisms in wild isolates of Caenorhabditis elegans">ChemPort</a>&nbsp;|</li><li id="B17"><a name="B17"><!-- . --></a>Gordon, D., Abajian, C. &amp; Green, P. <span class="atl"> Consed: a graphical tool for sequence finishing</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">8</span>, 195&#8211;202 (<span class="cite-month-year">1998</span>).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9521923&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Consed: a graphical tool for sequence finishing">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000072838200008&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Consed: a graphical tool for sequence finishing">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1cXitlWksr0%3D&amp;pissn=1548-7091&amp;pyear=2008&amp;md5=92811b9619900324b342352b6673ed3a" class="reftxt" title="Article on ChemPort - Consed: a graphical tool for sequence finishing">ChemPort</a>&nbsp;|</li></ol><table xmlns="" cellpadding="0" cellspacing="0" border="0" width="470"><tr><td rowspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td><td align="right"><img alt="" height="3" width="5" src="/nmeth/images/arrow_black_up.gif">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td height="7" align="right"><img alt="" height="7" width="1" src="/images/spacer.gif"></td></tr><tr><td colspan="2" height="1"><img alt="" height="1" width="470" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr><td colspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr></table><a xmlns="" name="Ack"></a><span xmlns="" class="heading3">Acknowledgments</span><br xmlns=""><span xmlns="" class="articletext">We acknowledge National Human Genome Research Institute funding (HG003079-04 to R.K.W. and HG003698 to G.T.M.). We thank K. Hall and D. Bentley of Illumina, Inc. for generously producing the paired-end read data described in the manuscript, M. Wendl for careful reading of the manuscript and T. Bieri for submitting the CB4858 variants to Wormbase.</span><br xmlns=""><br xmlns=""><a name="products"></a>
<table cellpadding="0" cellspacing="0" border="0" width="450"><tr><td rowspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td><td align="right"><img alt="" height="3" width="5" src="/nmeth/images/arrow_black_up.gif">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td height="7" align="right"><img alt="" height="7" width="1" src="/images/spacer.gif"></td></tr><tr><td colspan="2" height="1"><img alt="" height="1" width="450" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr><td colspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr></table><table border="0" cellspacing="0" cellpadding="1" width="450"><tr><td valign="top" bgcolor="#000000"><table border="0" cellspacing="0" cellpadding="0" width="450"><tr><td colspan="3" bgcolor="#ffffff"><a href="http://www.biocompare.com/nature/"><img border="0" alt="natureproducts" height="31" width="110" src="/images/natureproducts_logo.gif"></a></td></tr><tr><td colspan="3" bgcolor="#000000"><img alt="" width="1" height="1" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#eaeaea"><img alt="" height="1" width="10" src="/images/spacer.gif"></td><td class="doi" bgcolor="#eaeaea" valign="top"><br><p><a class="linksm" target="_blank" href="http://www.biocompare.com/nature/">
Natureproducts</a> is an online service detailing information about specific 
products used in this article, you can view the product descriptions, request 
information and compare with other similar products. The products
used are listed in alphabetical order.</p><table bgcolor="#eaeaea" border="0" cellspacing="0" cellpadding="0" width="450"><tr><td class="norm">A-Z product listing</td><td valign="bottom" align="right"><a href="http://www.biocompare.com/nature/"><img border="0" alt="biocompare" height="23" width="218" src="/nmeth/images/biocompare_logo.gif"></a></td></tr><tr><td bgcolor="#666666" height="1" colspan="2"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr></table></td><td bgcolor="#eaeaea"><img alt="" height="1" width="5" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#eaeaea" colspan="3"><img alt="" height="5" width="1" src="/images/spacer.gif"></td></tr></table><table border="0" cellspacing="0" cellpadding="0" width="100%"><tr><td width="10" bgcolor="#eaeaea"><img alt="" width="10" height="1" src="/images/spacer.gif"></td><td valign="top" width="7" bgcolor="#eaeaea"><img border="0" vspace="5" alt="" height="5" width="3" src="/nmeth/images/arrow_blue.gif"></td><td width="437" colspan="2" bgcolor="#eaeaea" valign="top"><a class="latestadvances" href="http://www.biocompare.com/natureproducts/go.asp?id=nmeth.1179_p_p1">Solexa Sequence Analyzer&nbsp;(Illumina Inc.)</a></td><td width="5" bgcolor="#eaeaea" valign="top"><img alt="" width="5" height="1" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#eaeaea" colspan="5"><img border="0" alt="" height="3" width="3" src="/images/spacer.gif"></td></tr><tr><td width="10" bgcolor="#eaeaea"><img alt="" width="10" height="1" src="/images/spacer.gif"></td><td valign="top" width="7" bgcolor="#eaeaea"><img border="0" alt="" vspace="5" height="5" width="3" src="/images/spacer.gif"></td><td align="right" width="437" colspan="2" bgcolor="#eaeaea" valign="top"><img alt="" height="5" width="3" src="/images/arrow_red.gif"><a target="_blank" class="redsml2" href="/nmeth/journal/v5/n2/products/nmeth.1179.html">See more natureproducts</a></td><td width="5" bgcolor="#eaeaea" valign="top"><img alt="" width="5" height="1" src="/images/spacer.gif"></td></tr><tr><td bgcolor="#eaeaea" colspan="5"><img border="0" alt="" height="5" width="3" src="/images/spacer.gif"></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="0" border="0" width="450"><tr><td rowspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td><td align="right"><img alt="" height="3" width="5" src="/nmeth/images/arrow_black_up.gif">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td height="7" align="right"><img alt="" height="7" width="1" src="/images/spacer.gif"></td></tr><tr><td colspan="2" height="1"><img alt="" height="1" width="450" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr><td colspan="2" height="20"><img alt="" height="20" width="1" src="/images/spacer.gif"></td></tr></table></td></tr></table></td><td width="15"><img alt="" height="1" width="15" src="/images/spacer.gif"/></td><td valign="top" width="130"><table xmlns="" cellpadding="0" cellspacing="0" border="0" width="130"><tr><td><img alt="" height="34" width="1" src="/images/spacer.gif"></td></tr><tr><td colspan="2" height="22"><img alt="" height="22" width="1" src="/images/spacer.gif"></td></tr></table>
<table xmlns="" cellpadding="0" cellspacing="0" border="0" width="130"><tr><td colspan="2" class="whitesml" height="20" bgcolor="#6E83A0" align="center"><b>FULL  TEXT</b></td></tr><tr><td valign="middle" colspan="2" bgcolor="#D1E1EA" height="17" align="center" class="blacksml2"><table cellpadding="1" cellspacing="0" border="0" width="106"><tr><td width="3"><img border="0" alt="" height="5" width="3" src="/nmeth/images/arrow_black_prev.gif"></td><td width="100" align="center"><a class="breadcrumblink" href="/nmeth/journal/v5/n2/full/nmeth.1172.html">
									Previous							
								</a><a class="breadcrumblink" href="/nmeth/journal/v5/n2/full/nmeth.1174.html">
										 | 
																		
									Next							
								</a></td><td height="10" width="3"><img alt="" height="5" width="3" src="/nmeth/images/arrow_black.gif"></td></tr></table></td></tr><tr><td colspan="2"><img alt="" height="1" width="130" src="/ni/images/spacer.gif"></td></tr><tr><td width="15" bgcolor="#D1E1EA" height="15" align="center" class="blacksml2"><img alt="" border="0" height="5" width="3" src="/nmeth/images/arrow_black.gif"></td><td width="115" bgcolor="#D1E1EA" height="17" class="blacksml2"><a href="/nmeth/journal/v5/n2/index.html" class="breadcrumblink">Table of contents</a></td></tr><tr><td colspan="2" height="15" align="center" class="blacksml2"><table cellpadding="1" cellspacing="0" border="0" width="130"><tr><td bgcolor="#D1E1EA"><table cellpadding="0" cellspacing="0" border="0" width="128"><tr><td colspan="2" align="center" bgcolor="#FFFFFF"><img alt="" height="5" width="1" src="/images/spacer.gif"></td></tr><tr><td width="23" align="center" bgcolor="#FFFFFF"><img alt="Download PDF" height="14" width="14" src="/nmeth/images/icon_pdf.gif"></td><td height="14" width="105" bgcolor="#FFFFFF"><a class="breadcrumblink" href="/nmeth/journal/v5/n2/pdf/nmeth.1179.pdf">Download PDF</a></td></tr><tr><td valign="middle" colspan="2" align="center" bgcolor="#FFFFFF"><img alt="" height="6" width="1" src="/images/spacer.gif"></td></tr><tr><td valign="middle" width="23" align="center" bgcolor="#FFFFFF"><img alt="Send to a friend" height="10" width="13" src="/nmeth/images/icon_email.gif"></td><td valign="middle" height="12" width="105" bgcolor="#FFFFFF"><a class="breadcrumblink" onclick="window.open('/foxtrot/svc/mailform?file=/nmeth/journal/v5/n2/full/nmeth.1179.html','mailafriend','width=600,height=600,resizable=yes,location=no,menubar=no,scrollbars=yes,personalbar=no,status=yes,top=10,left=10');" href="javascript:void(0)">Send to a friend</a></td></tr>
		
		




<tr><td valign="middle" colspan="2" align="center" bgcolor="#FFFFFF"><img alt="" height="5" width="1" src="/ni/images/spacer.gif"></td></tr>
<tr><td valign="middle" width="23" align="center" bgcolor="#FFFFFF"><img alt="rights and permissions" height="10" width="13" src="/common/images/icons/fff/permissions.gif"></td><td valign="middle" height="12" width="105" bgcolor="#FFFFFF"><a class="breadcrumblink" href="http://s100.copyright.com/AppDispatchServlet?publisherName=NPG&publication=Nature+Methods&title=Whole-genome sequencing and variant discovery in C. elegans&author=					LaDeana W Hillier  
						,
				Gabor T Marth  
						,
				Aaron R Quinlan  
						,
				David Dooling  
						,
				Ginger Fewell  
		<i> et al.</i>
&contentID=10.1038/nmeth.1179&publicationDate=02/01/2008&Vol=5&Issue=2">Rights and permissions</a></td></tr>
<tr id="ocr"><td valign="middle" width="23" align="center" bgcolor="#FFFFFF"><img alt="Order commercial reprints" height="10" width="13" src="/common/images/icons/fff/ocr.gif"></td><td valign="middle" height="12" width="105" bgcolor="#FFFFFF"><a class="breadcrumblink" href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPGR&publication=Nature+Methods&title=Whole-genome sequencing and variant discovery in C. elegans&author=					LaDeana W Hillier  
						,
				Gabor T Marth  
						,
				Aaron R Quinlan  
						,
				David Dooling  
						,
				Ginger Fewell  
		<i> et al.</i>
&contentID=10.1038/nmeth.1179&publicationDate=02/01/2008&volumeNum=5&issueNum=2&numPages=6&pageNumbers=pp183-188&orderBeanReset=true">Order commercial reprints</a></td></tr>
<tr id="crossref"><td valign="middle" width="23" align="center" bgcolor="#FFFFFF"><img alt="CrossRef lists 25 articles citing this article" src="/common/images/icons/cited.gif"></td><td valign="middle" height="12" width="105" bgcolor="#FFFFFF"><a class="breadcrumblink" href="/cited/cited.html?doi=10.1038/nmeth.1179">CrossRef lists 25 articles citing this article</a></td></tr>	<tr><td valign="middle" colspan="2" align="center" bgcolor="#FFFFFF"><img alt="" height="5" width="1" src="/ni/images/spacer.gif"></td></tr><tr><td valign="middle" width="23" align="center" bgcolor="#FFFFFF"><img alt="Save this link" height="10" width="13" src="/common/images/icons/fff/connotea.gif"></td><td valign="middle" height="12" width="105" bgcolor="#FFFFFF"><a class="breadcrumblink" href="http://www.connotea.org/add?uri=http://www.nature.com/nmeth/journal/v5/n2/full/nmeth.1179.html">Save this link</a></td></tr><tr><td valign="middle" colspan="2" align="center" bgcolor="#FFFFFF"><img alt="" height="5" width="1" src="/ni/images/spacer.gif"></td></tr></table></td></tr></table></td></tr><tr><td colspan="2"><table cellpadding="0" cellspacing="0" border="0" width="130"><tr><td height="10"><img alt="" height="1" width="15" src="/ni/images/spacer.gif"></td><td height="10"><img alt="" height="1" width="115" src="/ni/images/spacer.gif"></td></tr><tr><td align="center" width="15" class="blacksml2"><img alt="" border="0" height="3" width="5" src="/nmeth/images/arrow_black_dwn.gif"></td><td height="15" width="115" class="blacksml2"><a class="breadcrumblink" href="#abstract">Abstract</a></td></tr><tr><td align="center" width="15" class="blacksml2"><img alt="" border="0" height="3" width="5" src="/nmeth/images/arrow_black_dwn.gif"></td><td height="15" width="115" class="blacksml2"><a class="breadcrumblink" href="#Results">Results</a></td></tr><tr><td align="center" width="15" class="blacksml2"><img alt="" border="0" height="3" width="5" src="/nmeth/images/arrow_black_dwn.gif"></td><td height="15" width="115" class="blacksml2"><a class="breadcrumblink" href="#Discussion">Discussion</a></td></tr><tr><td align="center" width="15" class="blacksml2"><img alt="" border="0" height="3" width="5" src="/nmeth/images/arrow_black_dwn.gif"></td><td height="15" width="115" class="blacksml2"><a class="breadcrumblink" href="#Methods">Methods</a></td></tr><tr><td align="center" width="15" class="blacksml2" valign="top"><img alt="" vspace="4" border="0" height="5" width="3" src="/nmeth/images/arrow_black.gif"></td><td height="15" width="115" class="blacksml2"><a class="breadcrumblink" href="javascript:void(0)" onClick="window.open('/nmeth/journal/v5/n2/fig_tab/nmeth.1179_ft.html','fig_tab','width=600,height=400,scrollbars=yes')">Figures &amp; Tables</a></td></tr><tr><td align="center" width="15" class="blacksml2"><img alt="" border="0" height="3" width="5" src="/nmeth/images/arrow_black_dwn.gif"></td><td height="15" width="115" class="blacksml2"><a class="breadcrumblink" href="#Ack">Acknowledgments</a></td></tr><tr><td align="center" width="15" class="blacksml2"><img alt="" border="0" height="3" width="5" src="/nmeth/images/arrow_black_dwn.gif"></td><td height="15" width="115" class="blacksml2"><a class="breadcrumblink" href="#References">References</a></td></tr><tr><td valign="top" align="center" width="15" class="blacksml2"><img alt="" vspace="4" border="0" height="5" width="3" src="/nmeth/images/arrow_black.gif"></td><td height="15" width="115" class="blacksml2"><a href="/nmeth/journal/v5/n2/suppinfo/nmeth.1179_S1.html" class="breadcrumblink">Supplementary info</a></td></tr><tr><td align="center" width="15" class="blacksml2"><img alt="" border="0" height="3" width="5" src="/images/arrow_black_dwn.gif"></td><td height="15" width="115" class="blacksml2"><a href="#products" class="breadcrumblink">Products</a></td></tr><tr><td align="center" colspan="2" class="blacksml2"><img alt="" height="5" width="1" src="/images/spacer.gif"></td></tr><tr><td align="center" colspan="2" class="blacksml2"><img alt="" height="1" width="130" src="/nmeth/images/spacer_dotted.gif"></td></tr><tr><td class="blacksml2" width="15" align="center"><img src="/images/arrow_black.gif" width="3" height="5" border="0" alt=""></td><td class="blacksml2" width="115" height="15"><a href="/nmeth/journal/v5/n2/ris/nmeth.1179.ris" target="_blank" class="breadcrumblink">Export citation</a></td></tr><tr><td class="blacksml2" width="15" align="center"><img src="/images/arrow_black.gif" width="3" height="5" border="0" alt=""></td><td class="blacksml2" width="115" height="15"><a href="/nmeth/journal/v5/n2/ris/nmeth.1179refs.ris" target="_blank" class="breadcrumblink">Export references</a></td></tr><tr><td align="center" colspan="2" class="blacksml2"><img alt="" height="5" width="1" src="/images/spacer.gif"></td></tr>
			 <SCRIPT>
			 var show = true;
			 if(show)
			 {
			 document.write('<tr>');
			 document.write('<td class="blacksml2" width="15" align="center">');
			 document.write('<img src="/nmeth/images/arrow_black.gif" width="3" height="5" border="0" alt="">');
			 document.write('</td>');
			 document.write('<td class="blacksml2" width="115" height="15">');
			 }
			 </SCRIPT>
<A class="" HREF="javascript:void(0)" onclick="window.open('http://sherlock.nature.com/arls/Query?prod=NMETH&query=http%3A%2F%2Fxml.nature.com%2Fnmeth%2Fjournal%2Fv5%2Fn2%2Fxml%2Fnmeth.1179.xml&type=AF&src_title=Whole-genome+sequencing+and+variant+discovery+in+%3Ci%3EC.+elegans%3C%2Fi%3E&src_jrnl=Nature+Methods&src_vol=5&src_pp=183-188&src_date=2008&target=popup', 'autonomypopup', 'resizeable=yes,scrollbars=yes,scrolling=yes,statusbar=no,navbar=no,width=550,height=550');return false;">More articles like this</A><SCRIPT>
				if(show){
				   document.write('</td>');
				   document.write('</tr>');
				   document.write('<tr>');
				   document.write('<td height="5" align="center" colspan="2" class="blacksml2"><img alt="" height="5" width="1" src="/nsmb/images/spacer.gif"></td>');
				   document.write('</tr>');
				   document.write('<tr>');
				   document.write('<td align="center" colspan="2" class="blacksml2"><img alt="" height="1" width="130" src="/nmeth/images/spacer_dotted.gif"></td>');
				   document.write('</tr>');

				}
			   </SCRIPT></table></td></tr></table><table xmlns="" cellpadding="0" cellspacing="0" border="0" width="130"><tr><td>
				
<div class="box" id="natjob">
<h2 class="natjob"><a href="/naturejobs/"><span class="first">nature</span><span class="hidden"> </span><span class="second">jobs</span></a></h2>

	<ul class="natjob">

	<li><h3 class="first"><a href="http://www.nature.com/naturejobs/science/jobs/92066-Chemist-R-">Chemist - R&D</a></h3>
	<ul>
	<li>Suven Life Sciences Ltd.</li>
	<li>Hyderabad, A.P. 500 034 India</li>
	</ul>
	</li>

			<li><h3><a href="http://www.nature.com/naturejobs/science/jobs/92745-Research-Assistant-Associate-and-Lab-Leaders">Research Assistant / Associate and Lab Leaders</a></h3>
		<ul>
		<li>Aurigene Discovery Technologies Limited</li>
		<li>Bangalore, Karnataka 560 100 India</li>
		</ul>
		</li>
	</ul>


<div class="natjobmore"><a href="/naturejobs/science/jobs">More science jobs</a></div>
<div class="natjobmore"><a href="/naturejobs/science/jobs/new">Post a job for free</a></div>
</div>

<form method="post" action="http://www.biocompare.com/nature/search.asp" name="search">
<table width="100%" cellpadding="1" cellspacing="0" border="0" bgcolor="#000000">
<tr><td valign="top">
<table width="100%" cellpadding="0" cellspacing="0" border="0" bgcolor="#ffffff">
<tr><td colspan="3"><img src="/includes/prdct_img/spacer.gif" height="3" width="5" alt=""></td></tr>
<tr><td><img src="/includes/prdct_img/spacer.gif" height="1" width="5" alt=""></td>
<td valign="top"><img src="/includes/prdct_img/logo.gif" width="88" height="16" alt="natureproducts" border="0" /></td>
<td bgcolor="ffffff"><img src="/includes/prdct_img/spacer.gif" height="1" width="5" alt="" /></td></tr>
<tr><td colspan="3"><img src="/includes/prdct_img/spacer.gif" height="2" width="1" alt=""/></td></tr>
<tr><td colspan="3" bgcolor="#000000" valign="top"><img src="/includes/prdct_img/spacer.gif" height="1" width="1" alt="" border="0"></td></tr>
<tr><td colspan="3" bgcolor="#f3f2f2"><img src="/includes/prdct_img/spacer.gif" height="9" width="5" alt=""></td></tr>
<tr><td bgcolor="#f3f2f2"><img src="/includes/prdct_img/spacer.gif" height="1" width="5" alt=""></td>
<td valign="top" bgcolor="#f3f2f2">
<p class="prdnorm">Search buyers guide:<br /><br />
<input type="text" name="search" size="10">
<input type="image" src="/includes/prdct_img/go.gif"></td><td bgcolor="#f3f2f2"><img src="/includes/prdct_img/spacer.gif" height="1" width="5" alt=""></td></tr>
<tr><td colspan="3" bgcolor="#f3f2f2"><img src="/includes/prdct_img/spacer.gif" height="9" width="5" alt=""></td></tr>

</table>
</td></tr></table></form>


<style type="text/css">
</style>
<!-- skyscraper ad -->
<div class="ad-rh skyscraper narrow">
<p>ADVERTISEMENT</p>











		
<div class="adunit-skyscraper">
<script type="text/javascript"><!--//--><![CDATA[//><!--
	document.write ('<ilayer id="layer1" style="visibility:hidden;"></ilayer>');
	document.write ('<nolayer>');
	document.write ('<iframe src="http://ad.doubleclick.net/adi/nmeth.nature.com/article;type=af;artid=nmeth.1179;date=20080120;issue=2;pos=right;sz=120x600;ptile=2;ord=' + ordval + '?" width="120" height="600" marginwidth="0" marginheight="0" frameborder="0" scrolling="no">');
	document.write ('<a href="http://ad.doubleclick.net/jump/nmeth.nature.com/article;abr=!ie4;abr=!ie5;type=af;artid=nmeth.1179;date=20080120;issue=2;pos=right;sz=120x600;ptile=2;ord=' + ordval + '?">');
	document.write ('<img src="http://ad.doubleclick.net/ad/nmeth.nature.com/article;abr=!ie4;abr=!ie5;type=af;artid=nmeth.1179;date=20080120;issue=2;pos=right;sz=120x600;ptile=2;ord=' + ordval + '?" alt="Advertisement" /></a></iframe>');
	document.write ('</nolayer>');
//--><!]]></script>

<noscript>
	<div><a href="http://ad.doubleclick.net/jump/nmeth.nature.com/article;abr=!NN2;tile=1;type=af;artid=nmeth.1179;date=20080120;issue=2;pos=right;sz=120x600;ptile=2;ord=07893195?"><img src="http://ad.doubleclick.net/ad/nmeth.nature.com/article;abr=!NN2;tile=1;type=af;artid=nmeth.1179;date=20080120;issue=2;pos=right;sz=120x600;ptile=2;ord=07893195?" alt="Advertisement" style="border:0" /></a></div>
</noscript>

<script type="text/javascript"><!--//--><![CDATA[//><!--
	document.writeln ('<layer src="http://ad.doubleclick.net/adl/nmeth.nature.com/article;abr=!NN2;type=af;artid=nmeth.1179;date=20080120;issue=2;pos=right;sz=120x600;ptile=2;ord=' + ordval + '?" visibility="hidden" style="max-width:120px; height:600px;" onload="moveToAbsolute(layer1.pageX,layer1.pageY);clip.height=600;clip.width=120;visibility=\'show\';"></layer>');
//--><!]]></script>

</div>


</div>
<!-- end skyscraper ad -->
</td></tr></table></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="0" border="0" width="100%"><tr><td align="center">&nbsp;</td></tr>
<tr><td height="4" align="center" class="footerline"><img alt="" height="4" width="1" src="/images/spacer.gif"></td></tr><tr><td align="center"><table cellpadding="0" cellspacing="0" border="0" width="760"><tr><td valign="top"><img alt="" height="10" width="1" src="/nmeth/images/spacer.gif"></td></tr><tr><td class="norm" valign="top"><em><b>Nature Methods</b></em><br><span class="blacksmltext">
ISSN:&nbsp;1548-7091<br>
EISSN:&nbsp;1548-7105</span></td></tr><tr><td class="blacknorm" width="615" valign="top" align="left">
<a class="footer" href="/nmeth/index.html">Journal home</a> | <a class="footer" href="/nmeth/current_issue/index.html">
Current issue
</a> | <a class="footer" href="/nmeth/archive/index.html">Archive</a> | <a class="footer" href="/nmeth/press_release/index.html">Press releases</a> | </td></tr><tr><td valign="top"><img alt="" height="10" width="1" src="/nmeth/images/spacer.gif"></td></tr></table></td></tr><tr><td height="2" align="center" class="footerline"><img alt="" height="2" width="1" src="/images/spacer.gif"></td></tr><tr><td align="center"><table xmlns="" cellpadding="0" cellspacing="0" border="0" width="760"><tr><td colspan="3" valign="top"><img alt="" height="10" width="1" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td align="right" width="130" valign="top"><a href="http://npg.nature.com/npg/servlet/Content?data=xml/02_welcome.xml&style=xml/02_welcome.xsl"><img border="0" alt="Nature Publishing Group, publisher of Nature, and other science journals and reference works" height="26" width="26" src="/includes/rj_globnavimages/npg_footer_logo.gif"></a></td><td align="right" width="15" valign="top"><img alt="" height="1" width="15" src="/includes/rj_globnavimages/spacer.gif"></td><td class="blacksmltext" width="615" valign="middle"><a href="/common/copyright_statement.html" onClick="window.open('/common/copyright_statement.html','copyright_window','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=no,resizable=no,copyhistory=yes,width=450,height=280,top=380,left=20');return false;" class="copyright">&copy;2008&nbsp;Nature Publishing Group</a> | <a target="_blank" class="privacy" href="http://npg.nature.com/npg/servlet/Content?data=xml/08_privacy.xml&style=xml/08_privacy.xsl">Privacy policy</a></td></tr><tr><td colspan="3" valign="top"><img alt="" height="10" width="1" src="/includes/rj_globnavimages/spacer.gif"></td></tr></table></td></tr></table>
<!--extra code for 468 x 60 ad-->
<script language="JavaScript">
<!--
document.writeln ('<layer SRC="http://ad.doubleclick.net/adl/nmeth.nature.com/article;type=;artid=nmeth.1179;date=20080120;pos=right;' + adinf + 'sz=468x60;ord=' + ordval + '?" visibility="hidden" width=468 height=60 onload="moveToAbsolute(layer1.pageX,layer2.pageY);clip.height=60;clip.width=468;visibility=\'show\';"></layer>');
// -->
</script>
<!--end of extra code--><!-- super body bottom -->
<!-- super javascript -->
															
																			
						<script type="text/javascript" src="/super/scripts/jquery.js"></script>
								<script type="text/javascript" src="/common/scripts/runOnLoad.js"></script>
								<script type="text/javascript" src="/rj/scripts/group.js"></script>
			

<script type="text/javascript"><!--//--><![CDATA[//><!--
(function($) {
	$('a.popup-this').bind('click', function() {
		window.open(this.href);
		return false;
	});
})(jQuery);
//--><!]]></script>



																																																																																																																																																																																																																																								
<!--[if IE 6]>
<![endif]-->
																																																																																																																																																																																																																																																																																																																																																																																																																																																																																



						<!-- webtrends Version: 8.6.0 -->
	<script src="/super/scripts/webtrends/webtrends.js" type="text/javascript"></script>
	<script type="text/javascript"><!--//--><![CDATA[//><!--
		var _tag=new WebTrends();
		_tag.dcsGetId();
	//--><!]]></script>
	<script type="text/javascript"><!--//--><![CDATA[//><!--
		// Add custom parameters here.
		// _tag.DCSext.param_name=param_value;
		_tag.dcsCollect();
	//--><!]]></script>
	<noscript>
		<div><img alt="" id="DCSIMG" width="1" height="1" src="http://statse.webtrendslive.com/dcs0zztfg00000s969s37qoal_2f6z/njs.gif?dcsuri=/nojavascript&amp;WT.js=No&amp;WT.tv=8.6.0" /></div>
	</noscript>
	<script type="text/javascript"><!--//--><![CDATA[//><!--
	(function($) {
		$(function() {
			var getMeta = function(name) {
				return $('meta[name="' + name + '"]').attr('content');
			};
			var applyParams = function(wt) {
				var params = [];
				for (var prop in wt) {
					params.push(prop);
					params.push(wt[prop]);
				}
				_tag.dcsMultiTrack.apply(_tag, params);
			};

			$('a[href$=".pdf"], a[href*=".pdf?"]').bind('click', function() {
				var wt = {};
				wt['DCS.dcssip'] = this.hostname ? this.hostname.split(':')[0] : '';
				wt['DCS.dcsuri'] = this.pathname ? this.pathname.indexOf('/') !== 0 ? '/' + this.pathname : this.pathname : '/';
				wt['DCS.dcsqry'] = this.search ? this.search.substring(1) : '';
				wt['WT.cg_n'] = 'Nature Methods';
				wt['WT.cg_s'] = '';
				wt['WT.art_journal_title'] = getMeta('citation_journal_title');
				wt['WT.art_volume'] = getMeta('citation_volume');
				wt['WT.art_issue'] = getMeta('citation_issue');
				wt['WT.art_title'] = getMeta('citation_title');
				wt['WT.art_doi'] = getMeta('citation_doi');
				wt['WT.art_authors'] = getMeta('citation_authors').replace(/;/g, '%3B').replace(/\s*,\s*/g, ';');
				wt['WT.art_date'] = getMeta('citation_date');
				applyParams(wt);
			});
			
			$('a[href*="/rss"]').each(function() {
				if (this.href.match(/\/rss\/?(\?.*)?$/)) {
					$(this).bind('click', function() {
						var wt = {};
						wt['DCS.dcsuri'] = this.pathname ? this.pathname.indexOf('/') !== 0 ? '/' + this.pathname : this.pathname : '/';
						wt['DCS.dcsqry'] = this.search ? this.search.substring(1) : '';
						wt['WT.ti'] = 'Download:' + (this.hostname ? this.hostname.split(':')[0] : '') + wt['DCS.dcsuri'] + (wt['DCS.dcsqry'] ? '?' + wt['DCS.dcsqry'] : '');
						wt['WT.dl'] = '20';
						applyParams(wt);
					});
				}
			});

			$('div[class*="adunit-"] a').bind('click', function() {
				var classes = $(this).parents('div[class*="adunit-"]:first').attr('class');
				if (classes) {
					var match = /(^|\s)adunit-([^\s]+)/.exec(classes);
					_tag.dcsMultiTrack('WT.ac', (match) ? match[2] : '');
				}
			});
		});	
	})(jQuery);
	//--><!]]></script>
	<!-- end webtrends Version: 8.6.0 -->
</body></html>