<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature"><meta name="citation_title" content="Mapping and sequencing of structural variation from eight human genomes"><meta name="citation_authors" content="Jeffrey M. Kidd, Gregory M. Cooper, William F. Donahue, Hillary S. Hayden, Nick Sampas, Tina Graves, Nancy Hansen, Brian Teague, Can Alkan, Francesca Antonacci, Eric Haugen, Troy Zerr, N. Alice Yamada, Peter Tsang, Tera L. Newman, Eray Tüzün, Ze Cheng, Heather M. Ebling, Nadeem Tusneem, Robert David, Will Gillett, Karen A. Phelps, Molly Weaver, David Saranga, Adrianne Brand, Wei Tao, Erik Gustafson, Kevin McKernan, Lin Chen, Maika Malig, Joshua D. Smith, Joshua M. Korn, Steven A. McCarroll, David A. Altshuler, Daniel A. Peiffer, Michael Dorschner, John Stamatoyannopoulos, David Schwartz, Deborah A. Nickerson, James C. Mullikin, Richard K. Wilson, Laurakay Bruhn, Maynard V. Olson, Rajinder Kaul, Douglas R. Smith, and  Evan E. Eichler"><meta name="citation_date" content="2008 May 1"><meta name="citation_issue" content="7191"><meta name="citation_volume" content="453"><meta name="citation_firstpage" content="56"><meta name="citation_doi" content="10.1038/nature06862"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287"><meta name="citation_pmid" content="18451855"><meta name="DC.Title" content="Mapping and sequencing of structural variation from eight human genomes"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Howard Hughes Medical Institute"><meta name="DC.Contributor" content="Jeffrey M. Kidd"><meta name="DC.Contributor" content="Gregory M. Cooper"><meta name="DC.Contributor" content="William F. Donahue"><meta name="DC.Contributor" content="Hillary S. Hayden"><meta name="DC.Contributor" content="Nick Sampas"><meta name="DC.Contributor" content="Tina Graves"><meta name="DC.Contributor" content="Nancy Hansen"><meta name="DC.Contributor" content="Brian Teague"><meta name="DC.Contributor" content="Can Alkan"><meta name="DC.Contributor" content="Francesca Antonacci"><meta name="DC.Contributor" content="Eric Haugen"><meta name="DC.Contributor" content="Troy Zerr"><meta name="DC.Contributor" content="N. Alice Yamada"><meta name="DC.Contributor" content="Peter Tsang"><meta name="DC.Contributor" content="Tera L. Newman"><meta name="DC.Contributor" content="Eray Tüzün"><meta name="DC.Contributor" content="Ze Cheng"><meta name="DC.Contributor" content="Heather M. Ebling"><meta name="DC.Contributor" content="Nadeem Tusneem"><meta name="DC.Contributor" content="Robert David"><meta name="DC.Contributor" content="Will Gillett"><meta name="DC.Contributor" content="Karen A. Phelps"><meta name="DC.Contributor" content="Molly Weaver"><meta name="DC.Contributor" content="David Saranga"><meta name="DC.Contributor" content="Adrianne Brand"><meta name="DC.Contributor" content="Wei Tao"><meta name="DC.Contributor" content="Erik Gustafson"><meta name="DC.Contributor" content="Kevin McKernan"><meta name="DC.Contributor" content="Lin Chen"><meta name="DC.Contributor" content="Maika Malig"><meta name="DC.Contributor" content="Joshua D. Smith"><meta name="DC.Contributor" content="Joshua M. Korn"><meta name="DC.Contributor" content="Steven A. McCarroll"><meta name="DC.Contributor" content="David A. Altshuler"><meta name="DC.Contributor" content="Daniel A. Peiffer"><meta name="DC.Contributor" content="Michael Dorschner"><meta name="DC.Contributor" content="John Stamatoyannopoulos"><meta name="DC.Contributor" content="David Schwartz"><meta name="DC.Contributor" content="Deborah A. Nickerson"><meta name="DC.Contributor" content="James C. Mullikin"><meta name="DC.Contributor" content="Richard K. Wilson"><meta name="DC.Contributor" content="Laurakay Bruhn"><meta name="DC.Contributor" content="Maynard V. Olson"><meta name="DC.Contributor" content="Rajinder Kaul"><meta name="DC.Contributor" content="Douglas R. Smith"><meta name="DC.Contributor" content="Evan E. Eichler"><meta name="DC.Date" content="2008 May 1"><meta name="DC.Identifier" content="10.1038/nature06862"><meta name="DC.Language" content="en"><link rel="stylesheet" href="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/pmcstatic.css" type="text/css"><link rel="stylesheet" href="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/pmcbase1.css" type="text/css"><link rel="stylesheet" href="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/pmcbody5.css" type="text/css"><link rel="stylesheet" href="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/pmcrefs1.css" type="text/css"><script type="text/javascript" src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/tileshop_pmc.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/utils.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/tileshop_pmc.js"></script><script src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/firebugx.js" type="text/javascript"></script><script src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/tile.js" type="text/javascript"></script><script src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/notify.js" type="text/javascript"></script><script src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/drag_n_drop.js" type="text/javascript"></script><script src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/scale_pmc.js" type="text/javascript"></script><script type="text/javascript" src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/tileshop_data_db.js"></script><script src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/remote_data_provider.js" type="text/javascript"></script><title>Mapping and sequencing of structural variation from eight human genomes</title></head><div FirebugVersion="1.3.3" style="display: none;" id="_firebugConsole"></div><body class="pmc-body"><div style="height: 0px;"><a id="top" name="top"></a></div><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr valign="top"><td colspan="3"><table style="border-collapse: collapse;" border="0" cellpadding="0" cellspacing="0"><tbody><tr><td valign="top" width="145"><img style="border-style: none;" usemap="#pmclogo" alt="pmc logo image" src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/pmc3_logo_v5.gif"><map name="pmclogo"><area shape="rect" coords="65,52,137,67" href="http://www.pubmedcentral.nih.gov/fprender.fcgi" alt="Journal List"><area shape="rect" coords="7,52,52,67" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0H1xBDhMd0zwL2K6XVOf69XHPqb6TW2jUqfY-JQ0O&amp;reftype=other&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?db=pmc" alt="Search"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="http://www.pubmedcentral.nih.gov/"></map></td><td style="padding-left: 4px;" height="75" valign="top"><div><img style="border-style: none;" src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/logo-hhmi.gif" alt="Logo of hhmipa" usemap="#logo-imagemap" align="top"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="HHMI-PA banner" coords="0,0,499,51" href="http://www.hhmi.org/about/research/papp.html" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="rect" alt="about author manuscripts" coords="0,52,249,74" href="http://www.pubmedcentral.nih.gov/about/authorms.html"><area shape="rect" alt="submit a manuscript" coords="250,52,499,74" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=02CpJ0SjfNWwo5xkE1sL0Rc6xt_zY7_b1gqHLKkDg&amp;reftype=publisher&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CN%2FA&amp;TO=N%2FA%7CN%2FA%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.nihms.nih.gov"></map></div></td></tr></tbody></table></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="3" width="100%"><tbody><tr><td></td><td colspan="2"><div style="margin: 8px 0pt; width: 675px; background-color: rgb(232, 232, 192); font-family: arial,sans-serif; font-size: 80%;"><span style="background-color: rgb(210, 105, 30); color: white; font-weight: bold;">&nbsp;Note:&nbsp;</span>
 Performing your original search, <span style="font-weight: bold;">next generation sequencing pipeline</span>, in PubMed Central will retrieve
<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?tool=QuerySuggestion&amp;cmd=search&amp;db=pmc&amp;term=next+generation+sequencing+pipeline" style="font-weight: bold;" class="int-reflink">319 citations</a>.</div><div class="navlink-box navlink-box-gray"><a href="http://www.pubmedcentral.nih.gov/fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0nei9p3bVhYRW8THyFjme1CofLqKtJ23bmdQi2kBO&amp;reftype=other&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CN%2FA&amp;TO=Entrez%7CSearch%7CHHMI%20Author%20Manuscripts&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?db=pmc&amp;cmd=search&amp;term=hhmi%20author%20manuscript%5Bfilter%5D" class="navlink">HHMI Author Manuscripts</a></div></td></tr><tr><td class="sidebar-cell" valign="top" width="145"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/square.gif" alt="&gt;" style="border-style: none;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="#supplementary-material-sec" class="sidefm-supplink">Supplementary Material</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobtype=pdf" class="sidefm-pmclink">PDF (1.3M)</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0RezdgsjjQtDfLlB20GIF6kvyqSsYKiDrOd8H3Av6&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18451855">PubMed record</option><option selected="selected" value="redirect3.cgi?&amp;&amp;auth=0zEnpTaLtg76apU-rHFWf6Wo9uxOvQG49DzPJtQTy&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=18451855">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0n1JXSb_ctZgQw_U4a9DGXqJavoaorz-En3fKbQae&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=18451855">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0lgfs7wob_q6D-ltRNd5Y_GQMuJiMr8w2jnwOp5Sc&amp;reftype=GEO DataSets&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|GEO%20DataSets&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=link&amp;linkname=pmc_gds&amp;from_uid=2424287">GEO DataSets</option><option value="redirect3.cgi?&amp;&amp;auth=00a0aLbbjZ4-Y1swpphdYSciR-ZUJg8ogdxB0RA59&amp;reftype=Genome Project&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Genome%20Project&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=link&amp;linkname=pmc_genomeprj&amp;from_uid=2424287">Genome Project</option></select><br><input class="sidefm-pmart" alt="GO" src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/go.gif" type="image"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0vefER5rrgJ0fPNRGwxiXbzTOry-2XN_GDY99XAMS&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Kidd+JM%5Bauth%5D" class="sidefm-pmclink"> Kidd, J.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Nsn0p6QLbRTFQHLOkzAOTWjWyyM_5u-wTjmos5T6&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Cooper+GM%5Bauth%5D" class="sidefm-pmclink"> Cooper, G.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0tioaWM6-f95nOtNtwQug6AZNRXYlT3jbMoN-HkJB&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Donahue+WF%5Bauth%5D" class="sidefm-pmclink"> Donahue, W.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0BIREeP3HEQ6ErJnKmkZYyCnc1ccd4o6wtGeMoQrB&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Eichler+EE%5Bauth%5D" class="sidefm-pmclink"> Eichler, E.</a></div></div></div></td><td rowspan="106" background="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/wm-hhmi.gif" height="300" width="20"><div style="width: 20px;"></div></td><td class="content-cell" valign="top"><div class="front-matter-section"><table cellpadding="0" cellspacing="0" width="100%"><tbody><tr style="vertical-align: top;"><td><div class="fm-citation">Nature.Author manuscript; available in PMC 2008 June 11.</div><div class="fm-citation">Published in final edited form as:</div><div style="margin-left: 5px;"><div class="fm-vol-iss-date"><a class="ext-reflink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0nEVFt3IkZJ3hEo2D8qH8HXj58nky18jSJj6-zAnK&amp;reftype=publisher&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18451855" target="pmc_ext" onclick="focuswin('pmc_ext')"><span class="citation-abbreviation">Nature. </span><span class="citation-publication-date">2008 May 1; </span><span class="citation-volume">453</span><span class="citation-issue">(7191)</span><span class="citation-flpages">: 56–64. </span></a></div><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1038/nature06862.</span></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC2424287</div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">HHMIMSID: </span><span>HHMIMS49728</span></div></td></tr></tbody></table><div class="fm-copyright"><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/about/copyright.html">Copyright notice</a>  and <a class="int-reflink" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a></div><div class="fm-title">Mapping and sequencing of structural variation from eight human genomes</div><div class="fm-author contrib-group">Jeffrey M. Kidd,<sup>1</sup> Gregory M. Cooper,<sup>1</sup> William F. Donahue,<sup>2</sup> Hillary S. Hayden,<sup>3</sup> Nick Sampas,<sup>4</sup> Tina Graves,<sup>5</sup> Nancy Hansen,<sup>6</sup> Brian Teague,<sup>7</sup> Can Alkan,<sup>1</sup> Francesca Antonacci,<sup>1</sup> Eric Haugen,<sup>3</sup> Troy Zerr,<sup>1</sup> N. Alice Yamada,<sup>4</sup> Peter Tsang,<sup>4</sup> Tera L. Newman,<sup>1</sup> Eray Tüzün,<sup>1</sup> Ze Cheng,<sup>1</sup> Heather M. Ebling,<sup>2</sup> Nadeem Tusneem,<sup>2</sup> Robert David,<sup>2</sup> Will Gillett,<sup>3</sup> Karen A. Phelps,<sup>3</sup> Molly Weaver,<sup>1</sup> David Saranga,<sup>2</sup> Adrianne Brand,<sup>2</sup> Wei Tao,<sup>2</sup> Erik Gustafson,<sup>2</sup> Kevin McKernan,<sup>2</sup> Lin Chen,<sup>1</sup> Maika Malig,<sup>1</sup> Joshua D. Smith,<sup>1</sup> Joshua M. Korn,<sup>8</sup> Steven A. McCarroll,<sup>8</sup> David A. Altshuler,<sup>8</sup> Daniel A. Peiffer,<sup>9</sup> Michael Dorschner,<sup>1</sup> John Stamatoyannopoulos,<sup>1</sup> David Schwartz,<sup>7</sup> Deborah A. Nickerson,<sup>1</sup> James C. Mullikin,<sup>6</sup> Richard K. Wilson,<sup>5</sup> Laurakay Bruhn,<sup>4</sup> Maynard V. Olson,<sup>3</sup> Rajinder Kaul,<sup>3</sup> Douglas R. Smith,<sup>2</sup> and  Evan E. Eichler<sup>1</sup><div class="fm-affl"><sup>1</sup>Department of Genome Sciences and Howard Hughes Medical Institute, University of Washington, Seattle, Washington 98195, USA.</div><div class="fm-affl"><sup>2</sup>Agencourt Bioscience Corporation, Beverly, Massachusetts 01915, USA.</div><div class="fm-affl"><sup>3</sup>Division
of Medical Genetics, Department of Medicine, and University of
Washington Genome Center, University of Washington, Seattle, Washington
98195, USA.</div><div class="fm-affl"><sup>4</sup>Agilent Technologies, Santa Clara, California 95051, USA.</div><div class="fm-affl"><sup>5</sup>Washington University Genome Sequencing Center, School of Medicine, St Louis, Missouri 63108, USA.</div><div class="fm-affl"><sup>6</sup> Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892, USA.</div><div class="fm-affl"><sup>7</sup>Laboratory of Genetics, University of Wisconsin, Madison, Wisconsin 53706, USA.</div><div class="fm-affl"><sup>8</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02114, USA.</div><div class="fm-affl"><sup>9</sup>Illumina, Inc., 9885 Towne Centre Drive, San Diego, California 92121, USA.</div></div><div class="fm-footnote"></div><div class="fm-footnote" id="CR1"><strong>Author Information</strong> Reprints and permissions information is available at <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=00aHN9aXUXfzAmZrb8edAdf-cS0S_dBiO-JzGXAKx&amp;reftype=extlink&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.nature.com/reprints" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">www.nature.com/reprints</a></span>. The authors declare competing financial interests: details accompany the paper on <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ragLGP-FX3Yq5yNXjdjIFv9-gdWHEmrxjyR_aQ7I&amp;reftype=extlink&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.nature.com/nature" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">www.nature.com/nature</a></span>. Correspondence and requests for materials should be addressed to E.E.E. (Email: <span class="e_id471626"><a class="ext-reflink" href="mailto:eee@gs.washington.edu">eee@gs.washington.edu</a></span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id471626", '<a class="ext-reflink" href="' + reverseAndReplaceString('ude.notgnihsaw.sg/ta/eee:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ude.notgnihsaw.sg/ta/eee', '/at/','@') + '</a>')}catch(e){}
                                //--></script>).</div><div class="links-box"><div class="fm-footnote"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> The publisher's final edited version of this article is available at <a class="ext-reflink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ryCF9te-t408j8MTI2_ZLPOHahRifcVSguqFTkzi&amp;reftype=publisher&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18451855" target="pmc_ext" onclick="focuswin('pmc_ext')">Nature</a>.</div><div class="fm-footnote"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> See other articles in PMC that <a class="int-reflink" href="http://www.pubmedcentral.nih.gov/tocrender.fcgi?artid=2424287&amp;action=cited">cite</a> the published article.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/square.gif" alt="&gt;" style="border-style: none;"></span>Abstract</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471879" class="sidefm-pmclink" style="text-transform: none;">Fine-<img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/dot.gif" style="border-style: none;">scale map of human genome structural variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472497" class="sidefm-pmclink" style="text-transform: none;">Novel human euchromatic sequences</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472670" class="sidefm-pmclink" style="text-transform: none;">Sequence resolution</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472908" class="sidefm-pmclink" style="text-transform: none;">Other forms of genetic variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473016" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473190" class="sidefm-pmclink" style="text-transform: none;">METHODS SUMMARY</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473543" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id471676">Abstract</div><div class="section-content"><!--article-meta--><div class="p p-first-last" id="P1"><span id="P1"></span>Genetic
variation among individual humans occurs on many different scales,
ranging from gross alterations in the human karyotype to single
nucleotide changes. Here we explore variation on an intermediate
scale—particularly insertions, deletions and inversions affecting from
a few thousand to a few million base pairs. We employed a clone-based
method to interrogate this intermediate structural variation in eight
individuals of diverse geographic ancestry. Our analysis provides a
comprehensive overview of the normal pattern of structural variation
present in these genomes, refining the location of 1,695 structural
variants. We find that 50% were seen in more than one individual and
that nearly half lay outside regions of the genome previously described
as structurally variant. We discover 525 new insertion sequences that
are not present in the human reference genome and show that many of
these are variable in copy number between individuals. Complete
sequencing of 261 structural variants reveals considerable locus
complexity and provides insights into the different mutational
processes that have shaped the human genome. These data provide the
first high-resolution sequence map of human structural variation—a
standard for genotyping platforms and a prelude to future individual
genome sequencing projects.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471676" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471879" class="sidefm-pmclink" style="text-transform: none;">Fine-<img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/dot.gif" style="border-style: none;">scale map of human genome structural variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472497" class="sidefm-pmclink" style="text-transform: none;">Novel human euchromatic sequences</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472670" class="sidefm-pmclink" style="text-transform: none;">Sequence resolution</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472908" class="sidefm-pmclink" style="text-transform: none;">Other forms of genetic variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473016" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473190" class="sidefm-pmclink" style="text-transform: none;">METHODS SUMMARY</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473543" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id471721">&nbsp;</div><div class="section-content"><div class="p p-first-last" id="P2" cite="cite-id471722" pmids="15286789 15273396 17122850 17160897 16327809 16136132 16482158 18059266 17597777 14769938 17803354"><span id="P2"></span>Human
genetic structural variation, including large (more than 1 kilobase
pair (kbp)) insertions, deletions and inversions of DNA, is common<a href="#R1" rid="R1" class="cite-reflink bibr popnode">1</a>–<a href="#R9" rid="R9" class="cite-reflink bibr popnode">9</a>. These differences are thought to encompass more polymorphic base pairs than single nucleotide differences<a href="#R5" rid="R5" class="cite-reflink bibr popnode">5</a>,<a href="#R6" rid="R6" class="cite-reflink bibr popnode">6</a>,<a href="#R9" rid="R9" class="cite-reflink bibr popnode">9</a>,<a href="#R10" rid="R10" class="cite-reflink bibr popnode">10</a>. The importance of structural variation to human health and common genetic disease has become increasingly apparent<a href="#R11" rid="R11" class="cite-reflink bibr popnode">11</a>–<a href="#R14" rid="R14" class="cite-reflink bibr popnode">14</a>. However, only a small fraction of copy-number variant (CNV) base pairs have been determined at the sequence level<a href="#R15" rid="R15" class="cite-reflink bibr popnode">15</a>.
Most genome-wide approaches for detecting CNVs are indirect, depending
on signal intensity differences to predict regions of variation. They
therefore provide limited positional information and cannot detect
balanced events such as inversions. Because the human genome reference
assembly is now viewed as a patchwork of structurally variant sequence<a href="#R1" rid="R1" class="cite-reflink bibr popnode">1</a>,<a href="#R2" rid="R2" class="cite-reflink bibr popnode">2</a>,
it is expected that sequencing projects of other individuals would
reveal previously uncharacterized human euchromatic sequence, in a
similar manner to comparisons between the Celera and International
Human Genome Project assemblies<a href="#R16" rid="R16" class="cite-reflink bibr popnode">16</a>–<a href="#R18" rid="R18" class="cite-reflink bibr popnode">18</a>.
We implemented an approach to construct clone-based maps of eight human
genomes with the aim of systematically cloning and sequencing
structural variants more than 8 kbp in length. We present a validated,
structural variation map of these eight human genomes of Asian,
European and African ancestry, identify 525 regions of previously
uncharacterized ‘novel sequence’, and provide sequence resolution of
261 selected regions of structural variation in the human genome.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471676" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/square.gif" alt="&gt;" style="border-style: none;"></span>Fine-<img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/dot.gif" style="border-style: none;">scale map of human genome structural variation</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472497" class="sidefm-pmclink" style="text-transform: none;">Novel human euchromatic sequences</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472670" class="sidefm-pmclink" style="text-transform: none;">Sequence resolution</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472908" class="sidefm-pmclink" style="text-transform: none;">Other forms of genetic variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473016" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473190" class="sidefm-pmclink" style="text-transform: none;">METHODS SUMMARY</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473543" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><span id="S1"></span><div class="head1 section-title" style="text-transform: none;" id="id471879"><a id="S1"></a>Fine-scale map of human genome structural variation</div><div class="section-content"><span id="S1"></span><div class="p p-first" id="P3" cite="cite-id471887" pmids="15895083 17495918 16255080 12788976"><span id="P3"></span>We selected eight individuals as part of the first phase of the Human Genome Structural Variation Project<a href="#R19" rid="R19" class="cite-reflink bibr popnode">19</a> (<a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>). This included four individuals of Yoruba Nigerian ethnicity and four individuals of non-African ethnicity<a href="#R20" rid="R20" class="cite-reflink bibr popnode">20</a> (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a> and <a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>). For each individual we constructed a whole genomic library of about 1 million clones by using a fosmid subcloning strategy<a href="#R21" rid="R21" class="cite-reflink bibr popnode">21</a>.
Each library was arrayed and both ends of each clone insert were
sequenced to generate a pair of high-quality end sequences (termed an
end-sequence pair (ESP)<a href="#R22" rid="R22" class="cite-reflink bibr popnode">22</a>).
The overall approach generated a physical clone map for each individual
human genome, flagging regions discrepant by size or orientation on the
basis of the placement of end sequences against the reference assembly (<a href="#SD1" rid="SD1" class="side-supplink supplementary-material">Supplementary Fig. 1</a>)<a href="#R3" rid="R3" class="cite-reflink bibr popnode">3</a>,<a href="#R19" rid="R19" class="cite-reflink bibr popnode">19</a>. Across all eight libraries, we mapped 6.1 million clones to distinct locations against the reference sequence (<a href="#SD3" rid="SD3" class="side-supplink supplementary-material">Supplementary Fig. 2</a>; <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=06ShQNgeYMhc_05Sl1NM-K0aFsXrCw_WixPr8lQGq&amp;reftype=extlink&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://hgsv.washington.edu" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://hgsv.washington.edu</a></span>). Of these, 76,767 were discordant by length and/or orientation (<a href="#SD4" rid="SD4" class="side-supplink supplementary-material">Supplementary Fig. 3</a> and <a href="#SD14" rid="SD14" class="side-supplink supplementary-material">Supplementary Table 1</a>),
indicating potential sites of structural variation. About 0.4% (23,742)
of the ESPs mapped with only one end to the reference assembly despite
the presence of high-quality sequence at the other end (termed one-end
anchored (OEA) clones; <a href="#SD18" rid="SD18" class="side-supplink supplementary-material">Supplementary Table 2</a> and <a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>).</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="T1" name="T1"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/table-icon.gif" class="icon-reflink" style="border: 1px solid ;" alt="Table 1" title="Table 1"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 1</strong></a><div class="figure-table-caption-in-article"><span>Validated sites of structural variation detected by fosmid end sequence pairs</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p" id="P4" cite="cite-id472072" pmids="15895083"><span id="P4"></span>We
undertook three main approaches to validate sites of copy-number
variation. First, we selected 3,371 discordant fosmids corresponding to
sites supported by two or more overlapping fosmids from the same
individual whose apparent insert size deviated from the library mean
insert size. These corresponded to 2,990 non-overlapping sites that are
supported by multiple independent clones<a href="#R3" rid="R3" class="cite-reflink bibr popnode">3</a>.
Using four multiple complete restriction enzyme digests (MCD analysis),
we compared the predicted and expected insert sizes, confirming 1,182
non-redundant sites of copy-number variation (<a href="#SD19" rid="SD19" class="side-supplink supplementary-material">Supplementary Tables 3</a> and <a href="#SD20" rid="SD20" class="side-supplink supplementary-material">4</a>).
As a secondary validation method, we designed two high-density
customized oligonucleotide microarrays targeting a subset of insertion
and deletion regions (<a href="#SD7" rid="SD7" class="side-supplink supplementary-material">Supplementary Fig. 4</a>).
This analysis recovered an additional 194 regions that had a
copy-number difference but were not validated by MCD analysis. Combined
with other experimental methods, we validated a total of 1,471 sites of
copy-number variation (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 1</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a>, <a href="#SD19" rid="SD19" class="side-supplink supplementary-material">Supplementary Tables 3</a> and <a href="#SD20" rid="SD20" class="side-supplink supplementary-material">4</a>, and <a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>).
To assess the heritability of our events, we further intersected
validated deletions with single nucleotide polymorphism (SNP)
genotyping data (Illumina Human1M BeadChip) collected for 125 HapMap
DNAs of African, European and Asian individuals, which included 28
parent–child trios. Although only a subset of the deletion events (<em>n</em> = 130) could be reliably genotyped because of a lack of informative probes (<a href="#SD8" rid="SD8" class="side-supplink supplementary-material">Supplementary Fig. 5</a> and <a href="#SD21" rid="SD21" class="side-supplink supplementary-material">Supplementary Table 5</a>),
the allele frequencies ranged from rare (1%) to common (more than 50%),
were generally consistent with Hardy–Weinberg equilibrium, and more
than 98% of parent–child transmissions were consistent with mendelian
patterns of inheritance (<a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>).</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F1" name="F1"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/picrender_003.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span>Map of structural variation in the human genome</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p" id="P5" cite="cite-id472220" pmids="9140403 15654335 16906162 17360722 15466286"><span id="P5"></span>Inversions
proved more difficult to validate in a high-throughput manner because
the events are balanced and because breakpoints are prone to map in the
largest and most complex regions of segmental duplications<a href="#R23" rid="R23" class="cite-reflink bibr popnode">23</a>–<a href="#R25" rid="R25" class="cite-reflink bibr popnode">25</a>.
We validated 217 inversions by detailed fingerprint analysis and/or
sequence analysis. In addition, we validated seven larger ESP-detected
inversions by interphase and metaphase fluorescence <em>in situ</em> hybridization (<a href="#SD9" rid="SD9" class="side-supplink supplementary-material">Supplementary Fig. 6</a>, and <a href="#SD9" rid="SD9" class="side-supplink supplementary-material">Supplementary Tables 6</a> and <a href="#SD10" rid="SD10" class="side-supplink supplementary-material">7</a>).
This included two previously described events: a roughly 5-million-base
pair (Mbp) inversion on 8p23.1 and a roughly 1-Mbp inversion on
17q21.3. We detected five novel large inversions, including a 1.2-Mbp
inversion on 15q24, a 2.1-Mbp inversion on 15q13, and a 1.7-Mbp
inversion on 17q12. Three of these regions correspond to sites of
recurrent microdeletion associated with human disease, providing
further support for a link between common inversion polymorphisms and
genomic disorders<a href="#R26" rid="R26" class="cite-reflink bibr popnode">26</a>,<a href="#R27" rid="R27" class="cite-reflink bibr popnode">27</a>. Overall, we found a twofold enrichment for inversions mapping to clustered regions of the X chromosome (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 1</a> and <a href="#SD23" rid="SD23" class="side-supplink supplementary-material">Supplementary Table 7</a>), consistent with theoretical predictions of increased inversion content based on unusual inverted repeat structures<a href="#R28" rid="R28" class="cite-reflink bibr popnode">28</a>. These data provide one of the first high-quality inversion maps of the human genome.</div><div class="p" id="P6" cite="cite-id472346" pmids="15286789"><span id="P6"></span>In
total, we validated and refined the location of 1,695 sites of
structural variation across nine diploid human genomes (eight fosmid
libraries plus the original genome examined by the fosmid ESP approach
(G248)) (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 1</a>, <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a> and <a href="#SD10" rid="SD10" class="side-supplink supplementary-material">Supplementary Fig. 7</a>).
This included 747 deletions, 724 insertions and 224 inversions. A large
fraction of the insertion/deletion events (40%) are novel when compared
with previous published reports of CNVs. This is particularly
unexpected, considering that at least 25% of the human genome now shows
some evidence of copy-number variation (The Database of Genomic Variants<a href="#R1" rid="R1" class="cite-reflink bibr popnode">1</a>,
hg17.v2). Many of the events (856, or 50%) were identified in multiple
libraries and probably represent common polymorphisms (more than 5%
frequency) (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 2</a>);
261 (15%) of the sites were observed in five or more individuals,
indicating that the current reference human genome sequence
organization may actually represent a minor allele. At 34 loci, all
nine individuals were inconsistent with the build35 assembly,
identifying the reference allele as rare or as a potential sequence
misassembly.</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F2" name="F2"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/picrender.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span>Frequency distribution</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p p-last" id="P7" cite="cite-id472412" pmids="17122850"><span id="P7"></span>Using the refined set of CNVs, we compared CNV predictions within eight of the same samples analysed in ref. <a href="#R5" rid="R5" class="cite-reflink bibr popnode">5</a> (<a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>).
When we compared the predicted size of intersected sites on the same
eight samples, we found that the bacterial artificial chromosome (BAC)
array comparative genomic hybridization (CGH) CNVs were substantially
(tenfold) larger and showed no correlation with the ESP estimated size (<a href="#SD11" rid="SD11" class="side-supplink supplementary-material">Supplementary Fig. 8</a>).
In contrast, we found extremely strong concordance between the sizes
estimated from the ESP map and the annotations generated by our
targeted high-density array CGH experiments (<a href="#SD11" rid="SD11" class="side-supplink supplementary-material">Supplementary Fig. 8b</a>) and independent predictions on the same eight individuals analysed using the Affymetrix 6.0 platform (<a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a> and <a href="#SD11" rid="SD11" class="side-supplink supplementary-material">Supplementary Fig. 8c</a>). We conclude that the BAC array CGH experiments performed in ref. <a href="#R5" rid="R5" class="cite-reflink bibr popnode">5</a>
had, in some cases, exquisite sensitivity to detect much smaller events
(about 10 kbp) than previously expected. However, our analysis
indicates that the current amount of the reference genome sequence
represented as CNV in these eight genomes has been overestimated.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471676" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471879" class="sidefm-pmclink" style="text-transform: none;">Fine-<img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/dot.gif" style="border-style: none;">scale map of human genome structural variation</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/square.gif" alt="&gt;" style="border-style: none;"></span>Novel human euchromatic sequences</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472670" class="sidefm-pmclink" style="text-transform: none;">Sequence resolution</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472908" class="sidefm-pmclink" style="text-transform: none;">Other forms of genetic variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473016" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473190" class="sidefm-pmclink" style="text-transform: none;">METHODS SUMMARY</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473543" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><span id="S2"></span><div class="head1 section-title" style="text-transform: none;" id="id472497"><a id="S2"></a>Novel human euchromatic sequences</div><div class="section-content"><span id="S2"></span><div class="p p-first" id="P8"><span id="P8"></span>To
identify potentially novel euchromatic sequences not present within the
reference genome, we first identified clusters of clones in which one
end sequence mapped to the human reference assembly but the other end
sequence did not, termed OEA clusters (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 3a</a> and <a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>).
Pooling results from the first seven genomic libraries, we identified
21,556 OEA clones. Next, we assembled the sequence corresponding to all
non-anchored ends by using the TIGR assembler<a href="#R29" rid="R29" class="cite-reflink bibr popnode">29</a>. This procedure generated 1,736 sequence contigs (<em>n</em>
= 4,996 OEA clones) of which 48% (820) had no matches to previously
published human sequence assemblies (minimum 100 base pairs (bp) with
more than 98% sequence identity). By combining these sequence contigs
with the positions of the OEA clusters we identified the map location
of 525 regions of novel sequence insertion.</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F3" name="F3"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/picrender_004.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 3" title="Figure 3"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 3</strong></a><div class="figure-table-caption-in-article"><span>Discovery of novel human sequences that are CNV</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p" id="P9" cite="cite-id472553" pmids="18157130"><span id="P9"></span>We distinguished three categories of novel insertion (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 3a</a> and <a href="#SD12" rid="SD12" class="side-supplink supplementary-material">Supplementary Fig. 9</a>):
214 of the novel insertion loci intersected with regions identified as
insertions with the paired-end sequence approach (see above); 139
putative ‘insertions’ flanked sequence assembly gaps<a href="#R30" rid="R30" class="cite-reflink bibr popnode">30</a>;
and another 172 new sites did not correspond to known gaps or spanned
insertions within the human genome. Among these we identified at least
11 regions where we estimate that the insertions are too large (more
than 40 kbp in length) to be physically spanned by fosmid ESPs.
Examination of these loci in a whole-genome restriction map constructed
by optical mapping (<a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>) on one of the same individuals confirms that the majority (8 of 11) correspond to insertions as large as 130 kbp in length (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 3c</a> and <a href="#SD24" rid="SD24" class="side-supplink supplementary-material">Supplementary Table 8</a>).</div><div class="p p-last" id="P10"><span id="P10"></span>To
assess copy-number variation of these unannotated human sequences, we
designed an oligonucleotide microarray specifically for these 525 loci
and assessed copy-number status by array CGH (<a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>) among the eight genomes tested (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 3b</a>).
Novel sequences not associated with gaps showed the most extensive
variation in copy number. For example, we found that 49% of novel
sequences associated with fosmid ESPs (spanned insertions) showed
evidence of copy-number variation. We note that sequence contigs
mapping to the same novel locus (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 3d</a>)
often showed the same pattern of copy-number variation. Such regions
cannot be genotyped by existing commercial platforms that depend on
sequence in the reference genome. The presence of a mapped clone
ensures that these regions can be sequenced in their entirety (see
below) and incorporated as part of future CNV and SNP genotyping
platforms.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471676" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471879" class="sidefm-pmclink" style="text-transform: none;">Fine-<img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/dot.gif" style="border-style: none;">scale map of human genome structural variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472497" class="sidefm-pmclink" style="text-transform: none;">Novel human euchromatic sequences</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/square.gif" alt="&gt;" style="border-style: none;"></span>Sequence resolution</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472908" class="sidefm-pmclink" style="text-transform: none;">Other forms of genetic variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473016" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473190" class="sidefm-pmclink" style="text-transform: none;">METHODS SUMMARY</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473543" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><span id="S3"></span><div class="head1 section-title" style="text-transform: none;" id="id472670"><a id="S3"></a>Sequence resolution</div><div class="section-content"><span id="S3"></span><div class="p p-first" id="P11" cite="cite-id472678" pmids="17597783"><span id="P11"></span>The
acquisition of high-quality, finished sequence corresponding to the
breakpoints of a rearrangement is the ultimate form of validation<a href="#R31" rid="R31" class="cite-reflink bibr popnode">31</a>. We selected 405 clones with predicted structurally variant haplotypes by ESP analysis for full-length sequencing (<a href="#SD2" rid="SD2" class="side-supplink supplementary-material">Supplementary Fig. 10</a> and <a href="#SD25" rid="SD25" class="side-supplink supplementary-material">Supplementary Table 9</a>),
generating 16 Mbp of alternative haplotype sequence. Sequence
validation confirmed 230 insertion/deletion loci (median 8.1 kbp, mean
15.3 kbp) and 35 inversions (up to 2 Mbp in size). Validation for 63
sequenced clones could not be conclusively resolved, despite the fact
that both fingerprint and ESP analysis confirmed 80% of these
‘ambiguous’ clones as being structurally variant with respect to the
reference genome. Detailed sequence analyses revealed that most of
these contained large, multi-copy tandem repeat sequences, which
confound breakpoint identification and complicate the final sequence
assembly of the insert. Including these ambiguous clones, we estimate
that 84% (341 of 405) of the clones contained structural variants. The
vast majority of the clones that failed to confirm at the sequence
level represented putative insertion events (<a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>) as a result of a slight subcloning preference for ‘short insert’ clones.</div><div class="p" id="P12"><span id="P12"></span>High-quality
finished sequence at the breakpoints allowed us to assess the potential
molecular mechanisms underlying larger structural variation events in
the human genome (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T2" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 2</a>).
Non-allelic homologous recombination between repeated sequences
accounts for 47% (124 of 261) of events assigned a mechanism.
Recombination between segmental duplications is more common than L1 or
Alu-mediated events. Of the inversions, 67% show evidence of large
blocks of sequence homology at the breakpoints, with the remainder
mediated by shorter common repeat sequences. An additional ten events
(4%) involved the expansion or contraction of a variable number of
tandem repeats. Retrotransposition accounted for 15% (40 of 261) of
events, although this is likely to represent a lower bound given that
the detection thresholds exceeded the length of an L1 insertion (6 kbp)
for several of the libraries (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a>).
Analysis of structurally variant sequences found a slight enrichment of
repetitive DNA for both insertion (58.5%) and deletion (60.8%) events,
with 28% of events having a repeat content greater than 90%. Such
events are not resolvable with array-based techniques and will probably
require directed, PCR-based assays for genotyping.</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="T2" name="T2"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T2" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/table-icon.gif" class="icon-reflink" style="border: 1px solid ;" alt="Table 2" title="Table 2"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 2</strong></a><div class="figure-table-caption-in-article"><span></span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p" id="P13" cite="cite-id472773" pmids="17009402 8395942"><span id="P13"></span>We
compared RefSeq gene annotation between the structurally variant
haplotypes and found that 107 distinct gene structures were altered (<a href="#SD15" rid="SD15" class="side-supplink supplementary-material">Supplementary Table 10</a>).
Of these genes, 87% belong to members of a gene family, suggesting
potential functional redundancy. We specifically examined insertion
sequences and found homology for 60 spliced expressed sequence tags and
15 RefSeq gene annotations. Most of these putative gene structures
corresponded to duplicated copies of genes or portions of genes (<em>NAIP</em>, <em>BIRCA1</em>, <em>NBPF11</em>, <em>DNM1</em> and <em>LPA</em>) and/or had homology to genes predicted in either chimpanzee or macaque (<em>ANKRD20A</em> and <em>LOC713531</em>). There are three examples of insertions restricted to coding exons (<em>EPPK1</em>, <em>BAHCC1</em> and <em>MUC6</em>)—events predicted to alter the composition and structure of the encoded transcripts and proteins. In the case of <em>MUC6</em> and <em>LPA</em>, these protein length polymorphisms have been associated with <em>H. pylori</em> infection<a href="#R32" rid="R32" class="cite-reflink bibr popnode">32</a> and risk of coronary heart disease<a href="#R33" rid="R33" class="cite-reflink bibr popnode">33</a>, respectively.</div><div class="p p-last" id="P14"><span id="P14"></span>We sequenced multiple alleles for the <em>SIRPB1</em>
locus and found evidence for recurrent deletion events on different
haplotypes. Sequencing confirmed two distinct deletion alleles having
different breakpoints (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 4</a>)
embedded within segmental duplications. Both deletion alleles seem to
be common and only one of these two results in the loss of an exon,
raising the possibility that the two events have different functional
consequences despite their extensive overlap. The two different alleles
cannot be reliably distinguished by array CGH genotyping because of the
presence of duplicated sequences at the boundary and uncertainty in the
reference sample genotype (<a href="#SD7" rid="SD7" class="side-supplink supplementary-material">Supplementary Fig. 4</a>).
Although we have only begun to survey the sequence organization of a
small fraction of our sites, a preliminary analysis of the SNP content
of sequenced sites suggests that about 24% of the variants predicted in
multiple individuals may be found on different haplotype backgrounds.</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F4" name="F4"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F4" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/picrender_002.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 4" title="Figure 4"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 4</strong></a><div class="figure-table-caption-in-article"><span>Sequence resolution of human structural variation</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471676" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471879" class="sidefm-pmclink" style="text-transform: none;">Fine-<img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/dot.gif" style="border-style: none;">scale map of human genome structural variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472497" class="sidefm-pmclink" style="text-transform: none;">Novel human euchromatic sequences</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472670" class="sidefm-pmclink" style="text-transform: none;">Sequence resolution</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/square.gif" alt="&gt;" style="border-style: none;"></span>Other forms of genetic variation</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473016" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473190" class="sidefm-pmclink" style="text-transform: none;">METHODS SUMMARY</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473543" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><span id="S4"></span><div class="head1 section-title" style="text-transform: none;" id="id472908"><a id="S4"></a>Other forms of genetic variation</div><div class="section-content"><span id="S4"></span><div class="p p-first" id="P15" cite="cite-id472916" pmids="11591649 17571346"><span id="P15"></span>One
of the ancillary benefits of sequence-based detection of structural
variation is the identification and characterization of other forms of
human genetic variation. Because each library represents about 0.3-fold
sequence coverage per individual, the ESP pipeline generated about
three genomic equivalents (8.5 × 10<sup>9</sup> bases) of high-quality sequence data from the eight individuals. We mined the existing 13 million end sequences<a href="#R34" rid="R34" class="cite-reflink bibr popnode">34</a>
and identified 4.0 million non-redundant single nucleotide variants and
796,273 smaller insertion/deletion events (more than 1 bp to less than
100 bp in length); 28% (1.29 million) of the single nucleotide variants
and 75% (597,790) of the insertion/deletion variants (indels) were
novel when compared with dbSNP (build125). Of the eight HapMap
individuals selected in this project, five are common to the ENCODE
resequencing project<a href="#R35" rid="R35" class="cite-reflink bibr popnode">35</a>.
We therefore compared our SNP and indel predictions against those ten
regions resequenced in the same individuals as a measure of SNP/indel
accuracy. On the basis of 1,988 SNP and 120 indel genotypes, we
estimated false positive rates of 3.5% (SNPs) and 10.0% (indels).</div><div class="p p-last" id="P16" cite="cite-id472957" pmids="11237013 16421571"><span id="P16"></span>As
expected, the Yoruba African samples showed 15.3% more single
nucleotide genetic diversity than non-African samples on the autosomes.
The X chromosome shows greater genetic diversity (40%) between African
and non-African samples when compared with the autosomes. Because this
is one of the first random surveys of sequence data from an ethnically
diverse collection of individuals, we also assessed single nucleotide
density within 100-kbp windows across the entire genome, identifying
regions significantly enriched or depleted in single nucleotide
variants (<a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>).
After masking sites of segmental duplication, we identified 15 large
regions of excess nucleotide variation, ranging in size from 500 kbp to
3 Mbp. These include known sites of increased sequence diversity<a href="#R36" rid="R36" class="cite-reflink bibr popnode">36</a>,<a href="#R37" rid="R37" class="cite-reflink bibr popnode">37</a>,
for instance HLA and 8p23, as well as several previously undescribed
regions such as two large (more than 10 Mbp) regions on each arm of
chromosome 16 (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F5" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 5</a>).
The interval on 8p23 also showed the highest concentration of
structural variants validated by our ESP approach (22 distinct
variants). The molecular basis for this regional enrichment of genetic
diversity across human genomes is unknown, but our preliminary data
suggest that structural and single nucleotide variation may correlate.</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F5" name="F5"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F5" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/picrender_005.gif" style="border: 1px solid ;" class="icon-reflink" alt="Figure 5" title="Figure 5"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=figure&amp;id=F5" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 5</strong></a><div class="figure-table-caption-in-article"><span>Regions of enriched SNP density</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471676" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471879" class="sidefm-pmclink" style="text-transform: none;">Fine-<img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/dot.gif" style="border-style: none;">scale map of human genome structural variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472497" class="sidefm-pmclink" style="text-transform: none;">Novel human euchromatic sequences</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472670" class="sidefm-pmclink" style="text-transform: none;">Sequence resolution</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472908" class="sidefm-pmclink" style="text-transform: none;">Other forms of genetic variation</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/square.gif" alt="&gt;" style="border-style: none;"></span>Discussion</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473190" class="sidefm-pmclink" style="text-transform: none;">METHODS SUMMARY</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473543" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><span id="S5"></span><div class="head1 section-title" style="text-transform: none;" id="id473016"><a id="S5"></a>Discussion</div><div class="section-content"><span id="S5"></span><div class="p p-first" id="P17"><span id="P17"></span>We
present a high-resolution integrated map of genetic variation for eight
human genomes. We refine the location of 1,695 sites of structural
variation (more than about 6 kbp in length), identify 525 regions of
novel sequence that harbour highly polymorphic CNVs, and provide
single-base-pair sequence resolution for 261 regions of structural
variation. These events are placed within the context of 4 million SNPs
and 796,273 small indels (1−100 bp in size).</div><div class="p" id="P18"><span id="P18"></span>Our
detailed analysis of eight human genomes provides significant
biological and technological insights into human genetic variation.
First, we have discovered and mapped a large number of novel sequences
not represented in the human reference genome and show that more than
40% are CNV. These sequences range in size from a few kilobase pairs up
to 130 kbp and are randomly distributed, located both within genic and
intergenic regions. Although the sequences represent only a fraction of
the euchromatin (less than 0.1%), these results strongly argue that the
human genome sequence is still incomplete. The role of such sequences
in disease association cannot be determined without <em>de novo</em>
sequencing of additional genomes and the design of new platforms to
genotype these variants specifically on the basis of these ‘new’
sequences.</div><div class="p" id="P19" cite="cite-id473053" pmids="16906162 17360722 17666407 17363630"><span id="P19"></span>Second,
our refined map of structural variation predicts that the current
database of copy-number variation is inflated, which is consistent with
previous studies<a href="#R38" rid="R38" class="cite-reflink bibr popnode">38</a>.
An analysis of the same samples with customized high-resolution
microarrays and two independent commercial platforms shows an excellent
correspondence between ESP-predicted size and commercial SNP platforms
(Affymetrix 6.0 arrays and Illumina Human1M BeadChips). The net effect
is that there are fewer CNV base pairs per haplotype; consequently,
fewer genes and exons are affected. This is an important consideration
in view of the fact that CNV maps and databases based largely on
BAC-based array CGH are being used to exclude disease-causing variation<a href="#R26" rid="R26" class="cite-reflink bibr popnode">26</a>,<a href="#R27" rid="R27" class="cite-reflink bibr popnode">27</a>,<a href="#R39" rid="R39" class="cite-reflink bibr popnode">39</a>.
A comparison of the same eight individuals with the highest-density SNP
commercial platforms reveals that more than 50% of the structural
variants that we have detected cannot be adequately genotyped, although
we note that many more events can be detected than is possible with the
fosmid ESP approach. These data argue for the need for customized CNV
genotyping platforms based on sequence-validated sites of structural
variation.</div><div class="p" id="P20" cite="cite-id473110" pmids="15895083 15918152 17122850 17901297"><span id="P20"></span>Third, our sequence analyses suggest that non-allelic homologous recombination is the predominant mechanism (48%; <a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>)
altering the larger structural variation landscape of the human genome.
This is consistent with several reports confirming that copy-number
variation is enriched fourfold to tenfold for regions of segmental
duplication<a href="#R3" rid="R3" class="cite-reflink bibr popnode">3</a>,<a href="#R4" rid="R4" class="cite-reflink bibr popnode">4</a>,<a href="#R5" rid="R5" class="cite-reflink bibr popnode">5</a>.
However, these findings are in contrast with a recent analysis of two
individuals with next-generation ESP sequencing technology<a href="#R40" rid="R40" class="cite-reflink bibr popnode">40</a>,
which reported that events mediated by non-allelic homologous
recombination were relatively rare. One possibility for this
discrepancy is that shorter reads with lower sequence quality offered
by next-generation sequencing technologies may have less power to map
within duplication and repeat-rich regions of the genome, thereby
missing a large fraction of variation.</div><div class="p p-last" id="P21"><span id="P21"></span>The
establishment of a clone-based framework for each of these eight
genomes provides an important resource for future studies of genetic
variation. The clones provide the ability to recover and integrate all
forms of genetic variation, ranging from SNPs to larger structural
variants within specific haplotypes. The existing end-sequenced clone
map permits novel insert sequences anchored within the genome to be
mapped and sequenced completely, generating complete alternative human
haplotypes. It also permits sequence-based validation of CNV events
predicted with other methods and facilitates the targeted resequencing
of any genomic region of interest. These full insert sequences will be
important in the identification of haplotype-specific ‘tag’ SNPs that
may be used to genotype more complex structural variants indirectly and
to assess more fully the spectrum of human genetic variation. Thus,
these eight genomes can serve as an important benchmark as new genomes
become routinely sequenced with next-generation technologies.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471676" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471879" class="sidefm-pmclink" style="text-transform: none;">Fine-<img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/dot.gif" style="border-style: none;">scale map of human genome structural variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472497" class="sidefm-pmclink" style="text-transform: none;">Novel human euchromatic sequences</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472670" class="sidefm-pmclink" style="text-transform: none;">Sequence resolution</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472908" class="sidefm-pmclink" style="text-transform: none;">Other forms of genetic variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473016" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/square.gif" alt="&gt;" style="border-style: none;"></span>METHODS SUMMARY</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473543" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><span id="S6"></span><div class="head1 section-title" style="text-transform: none;" id="id473190"><a id="S6"></a>METHODS SUMMARY</div><div class="section-content"><span id="S6"></span><div class="sec sec-first"><span id="S7"></span><div class="head2 head-separate">Library construction and ESP analysis</div><div class="p p-first-last" id="P22"><span id="P22"></span>Fosmid libraries (pCC2Fos vector) were constructed<a href="#R21" rid="R21" class="cite-reflink bibr popnode">21</a> from human genomic DNA samples (Coriell Cell Repositories) corresponding to eight HapMap individuals (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2424287&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a>). We sequenced about 1 million clones (900 Mbp) for each genome in the form of high-quality ESPs (<a href="#SD16" rid="SD16" class="side-supplink supplementary-material">Supplementary Table 11</a>) and deposited sequences into the NIH trace repository (<span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0rOtRA1ZLsdBtj8gAdi-FIrxB8HDDuhY0DyiMJQ_b&amp;reftype=extlink&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/Traces" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.ncbi.nlm.nih.gov/Traces</a></span>). All ESPs were mapped to the human genome assembly (build35) with a previously described algorithm<sup>3</sup>.
Map information, including ESP alignments and corresponding clone IDs
of discordant and concordant clones, are available in an interactive
browser format and database (<span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0vCA9FBjtGgGwMh8y6sJl-gxKPTAQny_qiNOZb5gw&amp;reftype=extlink&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://hgsv.washington.edu" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://hgsv.washington.edu</a></span>).</div></div><div class="sec"><span id="S8"></span><div class="head2 head-separate">Validation</div><div class="p p-first-last" id="P23" cite="cite-id473276" pmids="8660999 9144219"><span id="P23"></span>Fosmid clones discordant by size (<em>n</em>
= 3,371 fosmid clones) were subjected to fingerprint analysis using
four multiple complete restriction enzyme digests (MCD analysis) to
confirm insert size and eliminate rearranged clones<a href="#R41" rid="R41" class="cite-reflink bibr popnode">41</a>,<a href="#R42" rid="R42" class="cite-reflink bibr popnode">42</a>.
Two high-density customized oligonucleotide microarrays (Agilent and
NimbleGen) were designed to confirm sites of deletion and insertion
(GEO accessions <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0bM6FdOEwn8H4NlmmUi1pKyJ3Izb6W6XoZi1fvQze&amp;reftype=extlink-entrez-geo&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CGEO%20DataSets&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE10008" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">GSE10008</a></span> and <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0b9cDNpBlhG8NhQYQIqd1dVbH0hOsfq90E2lf55x9&amp;reftype=extlink-entrez-geo&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CGEO%20DataSets&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE10037" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">GSE10037</a></span>).
We developed a new, expectation maximization-based clustering approach
to genotype deletions with the use of data from the Illumina Human1M
BeadChip collected for 125 HapMap DNA samples (<a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>).
We found that more than 98% of the children's genotypes were consistent
with mendelian transmission on the basis of an analysis of 28
parent–child trios.</div></div><div class="sec"><span id="S9"></span><div class="head2 head-separate">Fosmid insert sequencing</div><div class="p p-first-last" id="P24"><span id="P24"></span>We
completely sequenced the inserts of 405 fosmid clones from six genomic
libraries (210 from G248, 31 from ABC7, 39 from ABC8, 21 from ABC9, 98
from ABC10, and 6 from ABC12) with previously described methods. All
sequences have been deposited in GenBank (<a href="#SD25" rid="SD25" class="side-supplink supplementary-material">Supplementary Table 9</a>).</div></div><div class="sec sec-last"><span id="S10"></span><div class="head2 head-separate">SNP/indel analysis</div><div class="p p-first-last" id="P25" cite="cite-id473379" pmids="11591649"><span id="P25"></span>We identified single nucleotide variants using the ssahaSNP software tool<a href="#R34" rid="R34" class="cite-reflink bibr popnode">34</a> and indels (1−100 bp in size) using ssahaSNP in combination with cross_match (<span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0HKt61VcqiO3YBFgO7CLhQ7x268mbUq1SHlFCYQ-d&amp;reftype=extlink&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.phrap.org" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.phrap.org</a></span>; <a href="#SD13" rid="SD13" class="side-supplink supplementary-material">Supplementary Information</a>). SNP and indel variants have been deposited in dbSNP (release 129). </div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145">&nbsp;</td><td class="content-cell"><span id="S11"></span><div class="head1 section-title" style="text-transform: none;" id="id473417"><a id="S11"></a>Acknowledgements</div><div class="section-content"><div class="sec"><div class="p" id="id473425">We
thank the staff from the University of Washington Genome Center and the
Washington University Genome Sequencing Center for technical
assistance. J.M.K. is supported by a National Science Foundation
Graduate Research Fellowship. G.M.C. is supported by a Merck, Jane
Coffin Childs Memorial Fund Postdoctoral Fellowship. This work was
supported by National Institutes of Health grants HG004120 to E.E.E.,
D.A.N. and M.V.O., and 3 U54 HG002043 to M.V.O. E.E.E. is an
Investigator of the Howard Hughes Medical Institute.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145">&nbsp;</td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="footnotes">Footnotes</div><div class="section-content"><div class="fm-footnote" id="FN1"><div class="p"><strong>Supplementary Information</strong> is linked to the online version of the paper at <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Pe6LKskHqz18Tn3rWv-cFO8It42Rlrcu_nYWCkXs&amp;reftype=extlink&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.nature.com/nature" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">www.nature.com/nature</a></span>.</div></div><div class="fm-footnote" id="FN2"><div class="p">Author Contributions</div><div class="p" id="P28"><span id="P28"></span>J.M.K.,
G.M.C., M.V.O, D.A.N, and E.E.E. contributed to the writing of this
paper. The study was coordinated by L.B., M.V.O, R.K., D.R.S., J.M.K.
and E.E.E.;</div><div class="p" id="P29"><span id="P29"></span>A.B., D.R.S., D.Sa., E.G., H.M.E., K.M., N.T., R.D., W.F.D., and W.T. performed library construction and end sequencing.</div><div class="p" id="P30"><span id="P30"></span>E.H., H.S.H., K.A.P., M.V.O., R.K., R.K.W., T.G., and W.G. performed clone insert validation and sequencing.</div><div class="p" id="P31"><span id="P31"></span>C.A., D.A.N., E.T., J.D.S., J.S., L.C., M.D., M.M., M.W., T.L.N. and Z.C. provided technical and analytical support.</div><div class="p" id="P32"><span id="P32"></span>D.A.P., D.A.A., J.M.Ko. and S.A.M. contributed variation data.</div><div class="p" id="P33"><span id="P33"></span>G.M.C., J.M.K., L.B., N.A.Y., N.S., and P.T. designed and analyzed array CGH experiments.</div><div class="p" id="P34"><span id="P34"></span>G.M.C., and T.Z. performed the genotype analysis.</div><div class="p" id="P35"><span id="P35"></span>F.A. performed FISH experiments.</div><div class="p" id="P36"><span id="P36"></span>B.T. and D.S. performed optical mapping experiments.</div><div class="p" id="P37"><span id="P37"></span>E.E.E., J.M.K., and L.C. analyzed sequenced clones.</div><div class="p" id="P38"><span id="P38"></span>J.C.M. and N.H. identified SNPs and indels.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145">&nbsp;</td><td class="content-cell"><span id="SM"></span><div class="head1 section-title" style="text-transform: none;" id="id436437"><a id="supplementary-material-sec"></a><a id="SM"></a>Supplementary Material</div><div class="section-content"><!--back--><div id="SD1">Figure S1<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5a900" id="N0x1cfb0c0N0x2e5a900"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S1.pdf">Click here to view.</a><sup> (15K)</sup></blockquote></div></div><div id="SD10">Figure S7<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5a9c0" id="N0x1cfb0c0N0x2e5a9c0"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S7.pdf">Click here to view.</a><sup> (13K)</sup></blockquote></div></div><div id="SD11">Figure S8<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5aa80" id="N0x1cfb0c0N0x2e5aa80"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S8.pdf">Click here to view.</a><sup> (872K)</sup></blockquote></div></div><div id="SD12">Figure S9<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5ab40" id="N0x1cfb0c0N0x2e5ab40"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S9.ppt">Click here to view.</a><sup> (195K)</sup></blockquote></div></div><div id="SD13">Supp Text<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5ac00" id="N0x1cfb0c0N0x2e5ac00"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Supp_Text.doc">Click here to view.</a><sup> (1.3M)</sup></blockquote></div></div><div id="SD14">Table S1<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5acc0" id="N0x1cfb0c0N0x2e5acc0"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S1.xls">Click here to view.</a><sup> (23K)</sup></blockquote></div></div><div id="SD15">Table S10<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5ad80" id="N0x1cfb0c0N0x2e5ad80"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S10.xls">Click here to view.</a><sup> (26K)</sup></blockquote></div></div><div id="SD16">Table S11<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5ae40" id="N0x1cfb0c0N0x2e5ae40"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S11.xls">Click here to view.</a><sup> (17K)</sup></blockquote></div></div><div id="SD17">Table S12<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5af00" id="N0x1cfb0c0N0x2e5af00"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S12.xls">Click here to view.</a><sup> (79K)</sup></blockquote></div></div><div id="SD18">Table S2<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5afc0" id="N0x1cfb0c0N0x2e5afc0"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S2.xls">Click here to view.</a><sup> (15K)</sup></blockquote></div></div><div id="SD19">Table S3<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b080" id="N0x1cfb0c0N0x2e5b080"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S3.xls">Click here to view.</a><sup> (4.9M)</sup></blockquote></div></div><div id="SD2">Figure S10<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b140" id="N0x1cfb0c0N0x2e5b140"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S10.pdf">Click here to view.</a><sup> (3.4M)</sup></blockquote></div></div><div id="SD20">Table S4<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b200" id="N0x1cfb0c0N0x2e5b200"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S4.xls">Click here to view.</a><sup> (4.0M)</sup></blockquote></div></div><div id="SD21">Table S5<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b2c0" id="N0x1cfb0c0N0x2e5b2c0"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S5.xls">Click here to view.</a><sup> (39K)</sup></blockquote></div></div><div id="SD22">Table S6<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b380" id="N0x1cfb0c0N0x2e5b380"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S6.xls">Click here to view.</a><sup> (308K)</sup></blockquote></div></div><div id="SD23">Table S7<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b440" id="N0x1cfb0c0N0x2e5b440"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S7.xls">Click here to view.</a><sup> (64K)</sup></blockquote></div></div><div id="SD24">Table S8<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b500" id="N0x1cfb0c0N0x2e5b500"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S8.xls">Click here to view.</a><sup> (16K)</sup></blockquote></div></div><div id="SD25">Table S9<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b5c0" id="N0x1cfb0c0N0x2e5b5c0"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Table_S9.xls">Click here to view.</a><sup> (73K)</sup></blockquote></div></div><div id="SD3">Figure S2<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b680" id="N0x1cfb0c0N0x2e5b680"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S2.pdf">Click here to view.</a><sup> (194K)</sup></blockquote></div></div><div id="SD4">Figure S3a<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b740" id="N0x1cfb0c0N0x2e5b740"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S3a.pdf">Click here to view.</a><sup> (4.3M)</sup></blockquote></div></div><div id="SD5">Figure S3b<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b800" id="N0x1cfb0c0N0x2e5b800"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S3b.pdf">Click here to view.</a><sup> (6.0M)</sup></blockquote></div></div><div id="SD6">Figure S3c<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b8c0" id="N0x1cfb0c0N0x2e5b8c0"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S3c.pdf">Click here to view.</a><sup> (3.4M)</sup></blockquote></div></div><div id="SD7">Figure S4<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5b980" id="N0x1cfb0c0N0x2e5b980"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S4.pdf">Click here to view.</a><sup> (387K)</sup></blockquote></div></div><div id="SD8">Figure S5<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5ba40" id="N0x1cfb0c0N0x2e5ba40"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S5.pdf">Click here to view.</a><sup> (228K)</sup></blockquote></div></div><div id="SD9">Figure S6<div class="media"><blockquote><a name="N0x1cfb0c0N0x2e5bb00" id="N0x1cfb0c0N0x2e5bb00"></a><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2424287&amp;blobname=NIHMS49728-supplement-Figure_S6.pdf">Click here to view.</a><sup> (5.0M)</sup></blockquote></div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471676" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471879" class="sidefm-pmclink" style="text-transform: none;">Fine-<img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/dot.gif" style="border-style: none;">scale map of human genome structural variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472497" class="sidefm-pmclink" style="text-transform: none;">Novel human euchromatic sequences</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472670" class="sidefm-pmclink" style="text-transform: none;">Sequence resolution</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472908" class="sidefm-pmclink" style="text-transform: none;">Other forms of genetic variation</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473016" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473190" class="sidefm-pmclink" style="text-transform: none;">METHODS SUMMARY</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Kidd%202008%20Mapping%20and%20sequencing%20of%20structural%20variation%20from%20eight%20human%20genomes_files/square.gif" alt="&gt;" style="border-style: none;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id473543">References</div><div class="section-content"><div class="back-matter-section" id="reference-list"><div class="ref-cit-blk" id="R1"><div class="ref-label">1.</div><div class="ref-cit"><div class="citation">Iafrate AJ, et al.  Detection of large-scale variation in the human genome. <span><span class="ref-journal">Nature Genet. </span>2004;<span class="ref-vol">36</span>:949–951.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0hyQqC3nE_bAPqgCripbK7BOddDtsw82V7KkbRkZb&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15286789">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R2"><div class="ref-label">2.</div><div class="ref-cit"><div class="citation">Sebat J, et al.  Large-scale copy number polymorphism in the human genome. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">305</span>:525–528.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=00qRdVRBpW_mkyGSK2nScRQTpYTRwxQOV4nv5z5pQ&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15273396">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R3"><div class="ref-label">3.</div><div class="ref-cit"><div class="citation">Tuzun E, et al.  Fine-scale structural variation of the human genome. <span><span class="ref-journal">Nature Genet. </span>2005;<span class="ref-vol">37</span>:727–732.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0PuK5v6OT6vEJY3FSS_HAXYaSJ9CfrTfCcWOkMknR&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15895083">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R4"><div class="ref-label">4.</div><div class="ref-cit"><div class="citation">Sharp AJ, et al.  Segmental duplications and copy-number variation in the human genome. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2005;<span class="ref-vol">77</span>:78–88.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0bw-u608jG_1Bt_34evtfTsBZh2CUzImJX0UNJA5W&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15918152">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R5"><div class="ref-label">5.</div><div class="ref-cit"><div class="citation">Redon R, et al.  Global variation in copy number in the human genome. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">444</span>:444–454.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0D51HRPhDDop9X__KZxYI1nvLUPt0CfUdFZWlrQDV&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17122850">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R6"><div class="ref-label">6.</div><div class="ref-cit"><div class="citation">Wong KK, et al.  A comprehensive analysis of common copy-number variations in the human genome. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:91–104.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0j5GBSJv9IPv9tIzTFBOEgX5mEqw3Ciq7OgNqwkYK&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17160897">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R7"><div class="ref-label">7.</div><div class="ref-cit"><div class="citation">Conrad
DF, Andrews TD, Carter NP, Hurles ME, Pritchard JK. A high-resolution
survey of deletion polymorphisms in the human genome. <span><span class="ref-journal">Nature Genet. </span>2006;<span class="ref-vol">38</span>:75–81.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0f7X2U85-X_MrvZoOAjT7sXRQsTmmnzP6PDFTKABS&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16327808">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R8"><div class="ref-label">8.</div><div class="ref-cit"><div class="citation">McCarroll SA, et al.  Common deletion polymorphisms in the human genome. <span><span class="ref-journal">Nature Genet. </span>2006;<span class="ref-vol">38</span>:86–92.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0deBlbq-r1LXXMcrSg5oZWCXO5-mLXW7E2GbudAhw&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16468122">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R9"><div class="ref-label">9.</div><div class="ref-cit"><div class="citation">Hinds DA, Kloek AP, Jen M, Chen X, Frazer KA. Common deletions and SNPs are in linkage disequilibrium in the human genome. <span><span class="ref-journal">Nature Genet. </span>2006;<span class="ref-vol">38</span>:82–85.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0VmWIowdeRv3gbfgJDko9UM6BSwSGksIxQlQbtkQd&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16327809">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R10"><div class="ref-label">10.</div><div class="ref-cit"><div class="citation">Cheng Z, et al.  A genome-wide comparison of recent chimpanzee and human segmental duplications. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:88–93.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0dkRl-hS4RZ4VzcQ4AA2avMOKrxysvv-igqwgn5jx&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16136132">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R11"><div class="ref-label">11.</div><div class="ref-cit"><div class="citation">Aitman TJ, et al.  Copy number polymorphism in Fcgr3 predisposes to glomerulonephritis in rats and humans. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">439</span>:851–855.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0vAgL83duhcVIampBvQEc2mTpTygsTUggdmBvt5fC&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16482158">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R12"><div class="ref-label">12.</div><div class="ref-cit"><div class="citation">Gonzalez E, et al.  The influence of CCL3L1 gene-containing segmental duplications on HIV-1/AIDS susceptibility. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">307</span>:1434–1440.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0HcQzeL416SFyEJWCla2RQ6jkjic0SoKNoMW9EkaX&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15637236">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R13"><div class="ref-label">13.</div><div class="ref-cit"><div class="citation">Fellermann
K, et al. A chromosome 8 gene-cluster polymorphism with low human
β-defensin 2 gene copy number predisposes to Crohn disease of the
colon. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">79</span>:439–448.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0HKL81q7cBnKyGA8cNLBCBKE0Ob9dZmSWCSK7hAsl&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16909382">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R14"><div class="ref-label">14.</div><div class="ref-cit"><div class="citation">Hollox EJ, et al.  Psoriasis is associated with increased β-defensin genomic copy number. <span><span class="ref-journal">Nature Genet. </span>2007;<span class="ref-vol">40</span>:23–25.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0NSMZBmNBq45STjqhTJ2UvOg6pDJ4mP2hsyOqik6h&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18059266">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R15"><div class="ref-label">15.</div><div class="ref-cit"><div class="citation">Cooper GM, Nickerson DA, Eichler EE. Mutational and selective effects on copy-number variants in the human genome. <span><span class="ref-journal">Nature Genet. </span>2007;<span class="ref-vol">39</span>:S22–S29.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0XKRcCibqTbd4gE3MVKpzqwQbWtUpZkq-Yf6nhAJ9&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17597777">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R16"><div class="ref-label">16.</div><div class="ref-cit"><div class="citation">Istrail S, et al.  Whole-genome shotgun assembly and comparison of human genome assemblies. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2004;<span class="ref-vol">101</span>:1916–1921.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0DwMyjtxlmsKiHPKl-ACaZaiTmGzbqKn-e1fjiQAP&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14769938">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R17"><div class="ref-label">17.</div><div class="ref-cit"><div class="citation">Khaja R, et al.  Genome assembly comparison identifies structural variants in the human genome. <span><span class="ref-journal">Nature Genet. </span>2006;<span class="ref-vol">38</span>:1413–1418.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0dKRdVGEWhybXa7g4all9YQIHHlW0aJzC1WK1ek-a&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17115057">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R18"><div class="ref-label">18.</div><div class="ref-cit"><div class="citation">Levy S, et al.  The diploid genome sequence of an individual human. <span><span class="ref-journal">PLoS Biol. </span>2007;<span class="ref-vol">5</span>:e254.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Fk39kHuHSpfxSAYedfl_gQSk4lyfwCdM6MlIQk02&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17803354">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R19"><div class="ref-label">19.</div><div class="ref-cit"><div class="citation">Eichler EE, et al.  Completing the map of human genetic variation. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:161–165.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=00YhkpeX26NiF2EbRYkI3A7uKWKT3GBLy_b8ug5dH&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17495918">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R20"><div class="ref-label">20.</div><div class="ref-cit"><div class="citation">The International HapMap Consortium.  A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0JGyEG5kR9ajM19q8GOxAWtFfp9IFUzC6nySN7kUF&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16255080">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R21"><div class="ref-label">21.</div><div class="ref-cit"><div class="citation">Donahue W, Ebling HM. Fosmid libraries for genomic structural variation detection. <span><span class="ref-journal">Curr. Protocols Hum. Genet. </span>2007;<span class="ref-vol">5</span>:20.1–20.18.</span></div></div></div><div class="ref-cit-blk" id="R22"><div class="ref-label">22.</div><div class="ref-cit"><div class="citation">Volik S, et al.  End-sequence profiling: sequence-based analysis of aberrant genomes. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2003;<span class="ref-vol">100</span>:7696–7701.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0XI6nThi5_Kacfe01n85agFdiP-O0g8pYdlM_AAKV&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12788976">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R23"><div class="ref-label">23.</div><div class="ref-cit"><div class="citation">Small K, Iber J, Warren S. Emerin deletion revals a common X-chromosome inversion mediated by inverted repeats. <span><span class="ref-journal">Nature Genet. </span>1997;<span class="ref-vol">16</span>:96–99.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Vmo8IZEjD0DlTU3mK4Y5ZAMm6mUt0IZ9EB1zfQB-&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9140403">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R24"><div class="ref-label">24.</div><div class="ref-cit"><div class="citation">Giglio
S, et al. Heterozygous submicroscopic inversions involving olfactory
receptor-gene clusters mediate the recurrent t(4;8)(p16;p23)
translocation. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">71</span>:276–285.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=06svZ7Zv3u0lAr9sRaOMa0ooFOw6Omx8UB7KW8kev&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12058347">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R25"><div class="ref-label">25.</div><div class="ref-cit"><div class="citation">Stefansson H, et al.  A common inversion under selection in Europeans. <span><span class="ref-journal">Nature Genet. </span>2005;<span class="ref-vol">37</span>:129–137.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ho0TeLZd-viY4PlwsjbQn5J5pwlGsJe-spMk7AZl&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15654335">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R26"><div class="ref-label">26.</div><div class="ref-cit"><div class="citation">Sharp AJ, et al.  Discovery of previously unidentified genomic disorders from the duplication architecture of the human genome. <span><span class="ref-journal">Nature Genet. </span>2006;<span class="ref-vol">38</span>:1038–1042.</span> &lt;/jrn&gt;.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0VSCRR2gR9alQy5nrvCPXUSrJ0M-cdoivHC_xw5AL&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16906162">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R27"><div class="ref-label">27.</div><div class="ref-cit"><div class="citation">Sharp AJ, et al.  Characterization of a recurrent 15q24 microdeletion syndrome. <span><span class="ref-journal">Hum. Mol. Genet. </span>2007;<span class="ref-vol">16</span>:567–572.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0roieXFnQgOo9C27_ZDhpmWkMs-2ytFWVJVZmaAsC&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17360722">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R28"><div class="ref-label">28.</div><div class="ref-cit"><div class="citation">Warburton
PE, Giordano J, Cheung F, Gelfand Y, Benson G. Inverted repeat
structure of the human genome: the X-chromosome contains a
preponderance of large, highly homologous inverted repeats that contain
testes genes. <span><span class="ref-journal">Genome Res. </span>2004;<span class="ref-vol">14</span>:1861–1869.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0HKOJa5i2xVhAJXqPgf7_xp-h75GOEbictff6vADc&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15466286">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R29"><div class="ref-label">29.</div><div class="ref-cit"><div class="citation">Sutton GG, White O, Adams MD, Kerlavage A. TIGR Assembler: a new tool for assembling large shotgun sequencing projects. <span><span class="ref-journal">Genome Sci. Technol. </span>1995;<span class="ref-vol">1</span>:9–19.</span></div></div></div><div class="ref-cit-blk" id="R30"><div class="ref-label">30.</div><div class="ref-cit"><div class="citation">Bovee D, et al.  Closing gaps in the human genome with fosmid resources generated from multiple individuals. <span><span class="ref-journal">Nature Genet. </span>2008;<span class="ref-vol">40</span>:96–101.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=00SBQzOa8So53Nr0CCATCwDmfXvCn-I4mjnY6tQ6y&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18157130">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R31"><div class="ref-label">31.</div><div class="ref-cit"><div class="citation">Scherer SW, et al.  Challenges and standards in integrating surveys of structural variation. <span><span class="ref-journal">Nature Genet. </span>2007;<span class="ref-vol">39</span>:S7–S15.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0xghynsHi_a8Qzv94ul_EK3Pxe2119p0jZ_DBUkvy&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17597783">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R32"><div class="ref-label">32.</div><div class="ref-cit"><div class="citation">Nguyen TV, et al.  Short mucin 6 alleles are associated with <em>H. pylori</em> infection. <span><span class="ref-journal">World J. Gastroenterol. </span>2006;<span class="ref-vol">12</span>:6021–6025.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0hC20KlqVhZSIVmG1TyPzPTyBBXp0I95GH7Fo85_Y&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17009402">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R33"><div class="ref-label">33.</div><div class="ref-cit"><div class="citation">Lackner C, Cohen JC, Hobbs HH. Molecular definition of the extreme size polymorphism in apolipoprotein(a). <span><span class="ref-journal">Hum. Mol. Genet. </span>1993;<span class="ref-vol">2</span>:933–940.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0JKK6ph-h2z0n91uDsw6w0nUU9mxeGzFRtuzMv5q9&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8395942">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R34"><div class="ref-label">34.</div><div class="ref-cit"><div class="citation">Ning Z, Cox AJ, Mullikin JC. SSAHA: a fast search method for large DNA databases. <span><span class="ref-journal">Genome Res. </span>2001;<span class="ref-vol">11</span>:1725–1729.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0L3O7owZ8yvSMAZZUiRLrCP1d4j2bCRmmzQiLrARM&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11591649">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R35"><div class="ref-label">35.</div><div class="ref-cit"><div class="citation">ENCODE
Project Consortium. Identification and analysis of functional elements
in 1% of the human genome by the ENCODE pilot project. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:799–816.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0bEd67cyV-WBXa3-gcLjSCuOHKaEHNZpxyvsuMAUp&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17571346">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R36"><div class="ref-label">36.</div><div class="ref-cit"><div class="citation">Sachidanandam R, et al.  A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms. <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">409</span>:928–933.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0tkO2EO_JR4frybcfGdA_owWejMfaCl8UqnEOC5oO&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11237013">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R37"><div class="ref-label">37.</div><div class="ref-cit"><div class="citation">Nusbaum C, et al.  DNA sequence and analysis of human chromosome 8. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">439</span>:331–335.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ROkTCDKJ4JQdj3czWoZnp8iCSao0GhY9rtd4hQ5A&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16421571">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R38"><div class="ref-label">38.</div><div class="ref-cit"><div class="citation">de
Smith AJ, et al. Array CGH analysis of copy number variation identifies
1284 new genes variant in healthy white males: implications for
association studies of complex diseases. <span><span class="ref-journal">Hum. Mol. Genet. </span>2007;<span class="ref-vol">16</span>:2783–2794.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Lu0soqXoApod17iiEaw72Rr85K26Iho5W4r7XQqL&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17666407">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R39"><div class="ref-label">39.</div><div class="ref-cit"><div class="citation">Sebat J, et al.  Strong association of <em>de novo</em> copy number mutations with autism. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:445–449.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0HajQ0ZguNXFCtqava5ABxnZU71nPS0WZBcOrFANK&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17363630">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R40"><div class="ref-label">40.</div><div class="ref-cit"><div class="citation">Korbel JO, et al.  Paired-end mapping reveals extensive structural variation in the human genome. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">318</span>:420–426.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=00wPDeP_WdRCuaqI1fD5xjgIgvuweie__REbQiQOm&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17901297">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R41"><div class="ref-label">41.</div><div class="ref-cit"><div class="citation">Gillett
W, et al. Assembly of high-resolution restriction maps based on
multiple complete digests of a redundant set of overlapping clones. <span><span class="ref-journal">Genomics. </span>1996;<span class="ref-vol">33</span>:389–408.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0TSudEYi3PN4eZTKyAFk_stN9rBKYlHt3vdzI1kmK&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8660999">PubMed</a>]</span></div></div></div><div class="ref-cit-blk" id="R42"><div class="ref-label">42.</div><div class="ref-cit"><div class="citation">Wong
GK, Yu J, Thayer EC, Olson MV. Multiple-complete-digest restriction
fragment mapping: generating sequence-ready maps for large-scale DNA
sequencing. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>1997;<span class="ref-vol">94</span>:5225–5230.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0x96VuEPMwL8oCuVa2J3dYkbQV8wwhpy5SM1dOQg9&amp;reftype=pubmed&amp;artid=2424287&amp;article-id=2424287&amp;iid=167608&amp;issue-id=167608&amp;jid=544&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9144219">PubMed</a>]</span></div></div></div></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr>
    <div class="footer-link" id="fp-footer">
      <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.pubmedcentral.nih.gov/">PMC Home</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov/">NCBI</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov/">NIH</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="http://www.pubmedcentral.nih.gov/about/privacypolicy.html">Privacy Policy</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></tbody></table><!--host:ipmc1-->

</body></html>