<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Biochemical Society Transactions"><meta name="citation_title" content="British Yeast Group Meeting 2008: Next-generation sequencing: applications beyond genomes"><meta name="citation_authors" content="Samuel Marguerat, Brian&nbsp;T. Wilhelm, and  Jürg Bähler"><meta name="citation_date" content="2008 October 1"><meta name="citation_issue" content="Pt 5"><meta name="citation_volume" content="36"><meta name="citation_firstpage" content="1091"><meta name="citation_doi" content="10.1042/BST0361091"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889"><meta name="citation_pmid" content="18793195"><meta name="DC.Title" content="British Yeast Group Meeting 2008: Next-generation sequencing: applications beyond genomes"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Portland Press Ltd"><meta name="DC.Contributor" content="Samuel Marguerat"><meta name="DC.Contributor" content="Brian&nbsp;T. Wilhelm"><meta name="DC.Contributor" content="Jürg Bähler"><meta name="DC.Date" content="2008 October 1"><meta name="DC.Identifier" content="10.1042/BST0361091"><meta name="DC.Language" content="en"><link rel="stylesheet" href="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/pmcstatic.css" type="text/css"><link rel="stylesheet" href="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/pmcbase1.css" type="text/css"><link rel="stylesheet" href="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/pmcbody1.css" type="text/css"><link rel="stylesheet" href="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/pmcrefs1.css" type="text/css"><script type="text/javascript" src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/tileshop_pmc.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/utils.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/tileshop_pmc.js"></script><script src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/firebugx.js" type="text/javascript"></script><script src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/tile.js" type="text/javascript"></script><script src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/notify.js" type="text/javascript"></script><script src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/drag_n_drop.js" type="text/javascript"></script><script src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/scale_pmc.js" type="text/javascript"></script><script type="text/javascript" src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/tileshop_data_db.js"></script><script src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/remote_data_provider.js" type="text/javascript"></script><title>British Yeast Group Meeting 2008: Next-generation sequencing: applications beyond genomes</title></head><script style="display: none;" id="_firebugConsoleInjector" type="text/javascript">/* See license.txt for terms of usage */
function _FirebugConsole()
{
    this.init = function()
    {
        var commands = ["log", "debug", "info", "warn", "error", "assert", "dir",
            "dirxml", "trace", "group", "groupEnd", "groupCollapsed",
            "time", "timeEnd", "profile", "profileEnd", "count"];

        // Create console API
        for (var i=0; i<commands.length; i++)
        {
            var command = commands[i];
            this[command] = new Function(
                "return window.console.notifyFirebug(arguments, '" + command + "', 'firebugAppendConsole');");
        }

        // Initialize DOM element for communication betwen the web-page a chrome.
        this.getFirebugElement();
    },

    this.notifyFirebug = function(objs, methodName, eventId)
    {
        var element = this.getFirebugElement();

        var event = document.createEvent("Events");
        event.initEvent(eventId, true, false);

        this.userObjects = [];
        for (var i=0; i<objs.length; i++)
            this.userObjects.push(objs[i]);

        var length = this.userObjects.length;
        element.setAttribute("methodName", methodName);
        element.dispatchEvent(event);

        //dump("FirebugConsole dispatched event "+methodName+"\n");
        var result;
        if (element.getAttribute("retValueType") == "array")
            result = [];

        if (!result && this.userObjects.length == length+1)
            return this.userObjects[length];

        for (var i=length; i<this.userObjects.length && result; i++)
            result.push(this.userObjects[i]);

        return result;
    };

    this.getFirebugElement = function()
    {
        var element = document.getElementById("_firebugConsole");
        if (!element)
        {
            element = document.createElementNS("http://www.w3.org/1999/xhtml","html:div"); // NS for XML/svg
            element.setAttribute("id", "_firebugConsole");
            element.firebugIgnore = true;
            element.setAttribute("style", "display:none");

            var self = this;
            element.addEventListener("firebugCommandLine", function(event)
            {
                var element = event.target;
                var expr = element.getAttribute("expr"); // see commandLine.js
                self.evaluate(expr);
            }, true);

            document.documentElement.appendChild(element);
        }
        return element;
    };

    // ***********************************************************************
    // Console API

    this.__defineGetter__("firebug", function(){
        return this.getFirebugElement().getAttribute("FirebugVersion");
    });

    this.evaluate = function(expr)
    {
        try
        {
            var result = top.eval(expr);
            if (typeof result != "undefined")
                this.notifyFirebug([result], "evaluated", "firebugAppendConsole");
        }
        catch(exc)
        {
            var result = exc;
            result.source = expr;
            this.notifyFirebug([result], "evaluateError", "firebugAppendConsole");
        }
    };
}
//window.dump("============================>>>> Setting _firebug <<<< ====================================\n");
window._firebug =  new _FirebugConsole();
//window.dump("============================>>>> Set _firebug "+window._firebug+" "+window._FirebugConsole+" <<<< ====================================\n");
//for (var p in window.firebug)
//    window.dump(p+"="+window.console[p]+"\n");
window._firebug.init();
</script><div FirebugVersion="1.2.1" style="display: none;" id="_firebugConsole"></div><body class="pmc-body"><a id="top" name="top"></a><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr valign="top"><td colspan="3"><table style="border-collapse: collapse;" border="0" cellpadding="0" cellspacing="0"><tbody><tr><td valign="top" width="145"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/pmc3_logo_v5.gif" usemap="#pmclogo" alt="pmc logo image" border="0"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="http://www.pubmedcentral.nih.gov/fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0R71OuAaGoSZ27Lq1KjzkBwXLpOL2DycyKMQkykru&amp;reftype=other&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="http://www.pubmedcentral.nih.gov/"></map></td><td style="padding-left: 4px;" height="75" valign="top"><div class="banner-generic-logo-background"><div class="banner-journal-publisher-over-image"><div class="banner-journal-name" style="overflow: hidden;">Biochemical Society Transactions</div><div class="banner-publisher-name"><a class="ext-reflink" alt="Publisher URL" href="javascript:%20alert('URL%20is%20not%20defined%20in%20the%20database')" onclick="focuswin('pmc_ext')" target="pmc_ext">Portland Press Ltd.</a></div></div></div></td></tr></tbody></table></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="3" width="100%"><tbody><tr><td></td><td colspan="2"><div style="margin: 8px 0pt; width: 675px; background-color: rgb(232, 232, 192); font-family: arial,sans-serif; font-size: 80%;"><span style="background-color: rgb(210, 105, 30); color: white; font-weight: bold;">&nbsp;Note:&nbsp;</span> Performing your original search, <span style="font-weight: bold;">nature "next generation sequencing</span>, in PubMed Central will retrieve <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?tool=QuerySuggestion&amp;cmd=search&amp;db=pmc&amp;term=nature+%22next+generation+sequencing" style="font-weight: bold;" class="int-reflink">104 citations</a>.</div><div class="navlink-box navlink-box-gray"><a href="http://www.pubmedcentral.nih.gov/fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0-wX76xGPl0fbP9HK1LV7ykZ7kFuNR4abtQMlOQwF&amp;reftype=other&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CN%2FA&amp;TO=Entrez%7CSearch%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=search&amp;term=portland%20press%20opt2pay%5Bfilter%5D" class="navlink">Portland Press Opt2Pay Articles</a></div></td></tr><tr><td class="sidebar-cell" valign="top" width="145"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/square.gif" alt="&gt;" border="0"></span>Full Text</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0Hs9kNtRcUOks-iWFpaDyia0WgOo6AsjBu2nXvQGR&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18793195">PubMed record</option><option selected="selected" value="redirect3.cgi?&amp;&amp;auth=0xEC7dVNDWGnELv5OTXhaSQCMLsjILOSEEIv9Lk0l&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=18793195">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0DGt2LMX6Evc60Uy9rc2chwiX8sqAueHXzZ13kQVG&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=18793195">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0pSM4u_hNmDasQ0SdCxwSJGjROQSg6XyDemqoh5EN&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=2563889&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0fiXU6DjbGRywS5wzuAwNzH2Fcpag6gxxW_TPK5RV&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=2563889">Taxonomy tree</option></select><br><input src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/go.gif" alt="Go" class="sidefm-pmart" type="image"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0L9QQLPqm9N3CIgSnIrm67dBQ43TlJMU653gTr5pZ&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Marguerat+S%5Bauth%5D" class="sidefm-pmclink"> Marguerat, S.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0-MZxX_oCblcBcrZM_t58i8ITXe_IZQBwst6TYky0&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Wilhelm+B%5Bauth%5D" class="sidefm-pmclink"> Wilhelm, B.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0RSTyY9aRFMhYGRaRUz7VnhqETz7V_qVFOYqSh5Xy&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20B&amp;%23x000e4;hler+J%5Bauth%5D" class="sidefm-pmclink"> Bähler, J.</a></div></div></div></td><td rowspan="107" background="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/wm-portlandopen.gif" height="300" width="20"><div style="width: 20px;"></div></td><td class="content-cell" valign="top"><div class="front-matter-section"><table cellpadding="0" cellspacing="0" width="100%"><tbody><tr style="vertical-align: top;"><td><div class="fm-citation"><div><span class="citation-abbreviation">Biochem Soc Trans. </span><span class="citation-publication-date">2008 October 1; </span><span class="citation-volume">36</span><span class="citation-issue">(Pt 5)</span><span class="citation-flpages">: 1091–1096. </span></div><div><span class="fm-vol-iss-date">Published online 2008 September 19. </span><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1042/BST0361091.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC2563889</div></td></tr></tbody></table><div class="fm-copyright"><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/about/copyright.html">Copyright</a>
© 2008 The Author(s) The author(s) has paid for this article to be
freely available under the terms of the Creative Commons Attribution
Non-Commercial Licence (http://creativecommons.org/licenses/by-nc/2.5/)
which permits unrestricted non-commercial use, distribution and
reproduction in any medium, provided the original work is properly
cited.</div><div class="fm-ser-title"><div>British Yeast Group Meeting 2008</div></div><div class="fm-ser-title">Independent
Meeting held at National University of Ireland Maynooth, Maynooth, Co.
Kildare, Ireland, 18–20 March 2008. Organized and Edited by Gary Jones
(National University of Ireland Maynooth, Ireland).</div><div class="fm-title">Next-generation sequencing: applications beyond genomes</div><div class="fm-author contrib-group">Samuel Marguerat,<sup></sup><sup>1</sup> Brian&nbsp;T. Wilhelm,<sup><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/corrauth.gif" alt="corresponding author"></sup><sup></sup><sup>2</sup> and  Jürg Bähler<sup></sup><sup>1,</sup><sup>3</sup><div class="fm-affl">Cancer
Research UK, Fission Yeast Functional Genomics Group, The Wellcome
Trust Sanger Institute, Hinxton, Cambridge CB10 1HH, U.K.</div></div><div class="fm-footnote"><sup><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div class="fm-footnote"></div><div class="fm-footnote" id="fn1"><div class="p"><sup><strong>1</strong></sup><strong>Present address:</strong> Department of Genetics, Evolution and Environment and UCL Cancer Institute, University College London, London WC1E 6BT, U.K.</div></div><div class="fm-footnote" id="fn2"><div class="p"><sup><strong>2</strong></sup><strong>Present address:</strong> IRIC (Institut de Recherche en Immunologie et en Cancérologie), Montreal, QC, Canada, H3C 3J7</div></div><div class="fm-footnote" id="COR1"><sup>3</sup>To whom correspondence should be addressed (email <span class="e_id399358"><a class="ext-reflink" href="mailto:jurg@sanger.ac.uk">jurg@sanger.ac.uk</a></span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id399358", '<a class="ext-reflink" href="' + reverseAndReplaceString('ku.ca.regnas/ta/gruj:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ku.ca.regnas/ta/gruj', '/at/','@') + '</a>')}catch(e){}
                                //--></script>).</div><div class="fm-pubdate">Received April 30, 2008.</div><div class="fm-copyright"><p>This
is an open-access article distributed under the terms of the Creative
Commons Attribution License, which permits unrestricted use,
distribution, and reproduction in any medium, provided the original
work is properly cited.</p></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/square.gif" alt="&gt;" border="0"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411622" class="sidefm-pmclink" style="text-transform: none;">Sequencing as never before</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411699" class="sidefm-pmclink" style="text-transform: none;">From genomes to function</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500123" class="sidefm-pmclink" style="text-transform: none;">Mapping of DNA-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">binding proteins and chromatin</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500294" class="sidefm-pmclink" style="text-transform: none;">Deep sampling of transcriptomes</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500421" class="sidefm-pmclink" style="text-transform: none;">Next-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">generation sequencing compared with microarrays</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500559" class="sidefm-pmclink" style="text-transform: none;">Concluding remarks</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500573" class="sidefm-pmclink" style="text-transform: none;">Note added in proof (received 4 August 2008)</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500611" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id399430">Abstract</div><div class="section-content"><!--article-meta-->The
development of DNA sequencing more than 30&nbsp;years ago has
profoundly impacted biological research. In the last couple of years,
remarkable technological innovations have emerged that allow the direct
and cost-effective sequencing of complex samples at unprecedented scale
and speed. These next-generation technologies make it feasible to
sequence not only static genomes, but also entire transcriptomes
expressed under different conditions. These and other powerful
applications of next-generation sequencing are rapidly revolutionizing
the way genomic studies are carried out. Below, we provide a snapshot
of these exciting new approaches to understanding the properties and
functions of genomes. Given that sequencing-based assays may
increasingly supersede microarray-based assays, we also compare and
contrast data obtained from these distinct approaches.<div class="p"><span class="kwd-label">Keywords: </span><span class="kwd-text">ChIP-Seq, high-throughput sequencing, massively parallel sequencing, microarray, RNA-Seq, transcriptome, yeast</span></div><div class="p"><span class="kwd-label">Abbreviations: </span><span class="kwd-text">ChIP,
chromatin immunoprecipitation; ChIP-on-chip, ChIP using microarrays;
NRSF, neuron-restrictive silencer factor; STAT1, signal transducer and
activator of transcription 1</span></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399430" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/square.gif" alt="&gt;" border="0"></span>Sequencing as never before</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411699" class="sidefm-pmclink" style="text-transform: none;">From genomes to function</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500123" class="sidefm-pmclink" style="text-transform: none;">Mapping of DNA-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">binding proteins and chromatin</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500294" class="sidefm-pmclink" style="text-transform: none;">Deep sampling of transcriptomes</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500421" class="sidefm-pmclink" style="text-transform: none;">Next-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">generation sequencing compared with microarrays</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500559" class="sidefm-pmclink" style="text-transform: none;">Concluding remarks</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500573" class="sidefm-pmclink" style="text-transform: none;">Note added in proof (received 4 August 2008)</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500611" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id411622">Sequencing as never before</div><div class="section-content"><p>Just
when the era of sequencing seemed to have passed its peak,
technological breakthroughs are launching a new dawn with huge
potential and broad applications that are already transforming
biological research. Two pioneering papers reporting new sequencing
developments in 2005 have provided the first glimpse of things to come [<a href="#B1" class="cite-reflink">1</a>,<a href="#B2" class="cite-reflink">2</a>].
The sequencing revolution is currently driven by three commercially
available platforms: 454 (Roche), Genome Analyzer (Illumina/Solexa) and
ABI-SOLiD (Applied Biosystems) [<a href="#B3" class="cite-reflink">3</a>,<a href="#B4" class="cite-reflink">4</a>].
The development of additional platforms is well under way. These new
technologies are based on different principles than the classical
Sanger-based method [<a href="#B5" class="cite-reflink">5</a>], and
they are collectively referred to as either ‘next-generation’
sequencing, ‘high-throughput’ (or even ‘ultrahigh-throughput’)
sequencing, ‘ultra-deep’ sequencing or ‘massively parallel’ sequencing.
(Makes you wonder what terms they will come up with once even more
powerful technologies become available…) These novel technologies apply
distinct principles, resulting in differences in sequence read lengths
and numbers, which may provide distinct advantages and disadvantages
for different applications. All technologies have in common, however,
that they generate sequences on an unprecedented scale, without the
requirement for DNA cloning, and at a fraction of the costs required
for traditional sequencing. These features are the basis for the
current revolution and provide the inspiration to apply sequencing
approaches to biological questions that would not have been
economically or logistically practical before. Next-generation
sequencing should also democratize science in the sense that ambitious
sequencing-based projects can now be tackled by individual laboratories
or institutes, whereas before such projects would only have been
possible in genome centres.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399430" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411622" class="sidefm-pmclink" style="text-transform: none;">Sequencing as never before</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/square.gif" alt="&gt;" border="0"></span>From genomes to function</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500123" class="sidefm-pmclink" style="text-transform: none;">Mapping of DNA-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">binding proteins and chromatin</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500294" class="sidefm-pmclink" style="text-transform: none;">Deep sampling of transcriptomes</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500421" class="sidefm-pmclink" style="text-transform: none;">Next-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">generation sequencing compared with microarrays</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500559" class="sidefm-pmclink" style="text-transform: none;">Concluding remarks</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500573" class="sidefm-pmclink" style="text-transform: none;">Note added in proof (received 4 August 2008)</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500611" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id411699">From genomes to function</div><div class="section-content"><p>Early studies applied next-generation sequencing to sampling microbial diversities in a deep mine and in oceans [<a href="#B6" class="cite-reflink">6</a>–<a href="#B8" class="cite-reflink">8</a>], launching the field of ‘meta-genomics’ where entire biological communities are sequenced, <em>en masse</em>, to survey the variety of all organisms living together in particular ecosystems [<a href="#B9" class="cite-reflink">9</a>–<a href="#B13" class="cite-reflink">13</a>].
Other interesting applications are in the field of ancient DNA
research, where next-generation sequencing has been successfully
applied to analyse genomes of woolly mammoths [<a href="#B14" class="cite-reflink">14</a>] and Neanderthals [<a href="#B15" class="cite-reflink">15</a>,<a href="#B16" class="cite-reflink">16</a>]. Naturally, next-generation sequencing is also used to decode modern genomes, from bacteria [<a href="#B17" class="cite-reflink">17</a>,<a href="#B18" class="cite-reflink">18</a>] and viral isolates [<a href="#B19" class="cite-reflink">19</a>,<a href="#B20" class="cite-reflink">20</a>] to James Watson [<a href="#B21" class="cite-reflink">21</a>].
The latter example illustrates that the power of next-generation
sequencing is increasingly exploited to re-sequence strains and
individuals for which reference genome sequences are available to
sample genomic diversity. Such studies have identified mutations in
bacterial strains [<a href="#B22" class="cite-reflink">22</a>,<a href="#B23" class="cite-reflink">23</a>], polymorphisms in worm [<a href="#B24" class="cite-reflink">24</a>], structural variation in the human genome [<a href="#B25" class="cite-reflink">25</a>] and specific alleles involved in cancer [<a href="#B26" class="cite-reflink">26</a>].</p><p>In
addition to established analyses of genome sequences, next-generation
sequencing is triggering new assays and applications that should
greatly advance our understanding of genome function (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1</a>) [<a href="#B27" class="cite-reflink">27</a>].
The principle behind these alternative applications, which have been
termed ‘sequence census’ methods, is simple: complex DNA or RNA samples
are directly sequenced to determine their content. With reference
genomes available, short sequence reads are sufficient to map their
locations (except for repeated regions), and once mapped, millions of
sequence hits are simply counted to determine their genomic
distribution (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2</a>). This concept is based on previous approaches such as serial analysis of gene expression [<a href="#B28" class="cite-reflink">28</a>] and massively parallel signature sequencing [<a href="#B29" class="cite-reflink">29</a>].
Next-generation sequencing, however, delivers much more information at
affordable costs, and it is easy to implement for a wider range of
applications. Below, we will survey initial studies that analyse genome
function exploiting sequence census methods, which will increasingly
supersede microarray-based approaches (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1</a>).</p><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F1" name="F1"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a target="figure" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/picrender_002.gif" class="icon-reflink" alt="Figure 1" title="Figure 1" border="1"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span>Sequence
census technologies have added a new dimension to the analysis of gene
expression regulation, which has been dominated by hybridization-based
methods</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F2" name="F2"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/picrender.gif" class="icon-reflink" alt="Figure 2" title="Figure 2" border="1"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span>Sequence census data applied to cDNA allow genome-wide measurements of transcript levels (RNA-Seq)</span></div></td></tr></tbody></table></div></div><div style="clear: both;"></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399430" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411622" class="sidefm-pmclink" style="text-transform: none;">Sequencing as never before</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411699" class="sidefm-pmclink" style="text-transform: none;">From genomes to function</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/square.gif" alt="&gt;" border="0"></span>Mapping of DNA-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">binding proteins and chromatin</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500294" class="sidefm-pmclink" style="text-transform: none;">Deep sampling of transcriptomes</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500421" class="sidefm-pmclink" style="text-transform: none;">Next-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">generation sequencing compared with microarrays</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500559" class="sidefm-pmclink" style="text-transform: none;">Concluding remarks</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500573" class="sidefm-pmclink" style="text-transform: none;">Note added in proof (received 4 August 2008)</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500611" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id500123">Mapping of DNA-binding proteins and chromatin</div><div class="section-content"><p>ChIP-on-chip [ChIP (chromatin immunoprecipitation) using microarrays] [<a href="#B30" class="cite-reflink">30</a>,<a href="#B31" class="cite-reflink">31</a>] is a key approach to globally mapping the <em>in vivo</em>
binding sites of various DNA-binding proteins across genomes. Instead
of using the DNA that is precipitated with the protein of interest to
interrogate microarrays, recent studies have directly sequenced this
DNA to analyse the protein-binding sites at high resolution. This
approach, termed ‘ChIP-Seq’, should produce a huge windfall, in
particular for studies in multicellular eukaryotes where whole genome
coverage has generally required the use of several arrays.</p><p>Initial
studies looking at the binding sites of human NRSF (neuron-restrictive
silencer factor) and STAT1 (signal transducer and activator of
transcription 1) [<a href="#B32" class="cite-reflink">32</a>,<a href="#B33" class="cite-reflink">33</a>]
indicate that the resolution of ChIP-Seq is far better than that of
ChIP-on-chip. NRSF, a well-documented zinc-finger repressor that
negatively regulates gene expression of neuronal genes in non-neuronal
cells, has &gt;80 previously validated targets, providing a
well-defined test set to define false-positive and -negative rates [<a href="#B32" class="cite-reflink">32</a>].
The vast majority of previously known target sites have been confirmed
among the ∼2000 targets identified through ChIP-Seq. Moreover, this
analysis has exploited the deep sampling and high resolution of
ChIP-Seq to identify a novel class of genomic NRSF-binding sites,
suggesting the existence of different subclasses of genes regulated by
the same factor [<a href="#B32" class="cite-reflink">32</a>]. Another
ChIP-Seq study has mapped the binding sites of STAT1, a transcription
factor that regulates genes involved in cell differentiation, survival
and proliferation [<a href="#B33" class="cite-reflink">33</a>]. The
dynamic behaviour of STAT1 is of interest as it usually localizes in
the cytoplasm, but translocates to the nucleus on stimulation by an
extracellular signal. As expected, the authors have observed a large
increase in STAT1 binding sites after stimulation of cells with
interferon-γ (from 11000 to 41000), and the results also agree well
with previously published data.</p><p>Approaches to map the genomic
protein-binding sites are not limited to transcription factors. One of
the first papers demonstrating the utility of ChIP-Seq for whole-genome
location analysis [<a href="#B34" class="cite-reflink">34</a>] mapped
the genome-wide sites of 20 histone methylation marks, along with CTCF
(CCCTC-binding factor), the histone variant H2A.Z and RNA polymerase II
in human cells. The unprecedented detail of these data has led to
several valuable conclusions about the association of specific sets of
histone modifications with either active or repressed promoters. Such
comprehensive and predictive patterns can be used not only to confirm
annotated promoters but also to identify new ones [<a href="#B34" class="cite-reflink">34</a>].
Another comprehensive survey of two types&nbsp;of histone modifications
has been reported for pluripotent and lineage-committed mouse cells,
revealing how these modifications change during development [<a href="#B35" class="cite-reflink">35</a>].</p><p>In
another adaptation of array-based methods, next-generation sequencing
has also been applied to map regions with few or no chromatin proteins,
namely DNase-hypersensitive sites in human cell lines to identify
locations with regulatory elements [<a href="#B36" class="cite-reflink">36</a>].
Using this approach, ∼95000 DNase-hypersensitive sites have been
uncovered, a surprising majority (∼80%) of which are not associated
with promoter regions. This finding strongly suggests that the genome
is replete with regions of open DNA, many of which may have
unrecognized roles in genome function [<a href="#B36" class="cite-reflink">36</a>].</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399430" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411622" class="sidefm-pmclink" style="text-transform: none;">Sequencing as never before</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411699" class="sidefm-pmclink" style="text-transform: none;">From genomes to function</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500123" class="sidefm-pmclink" style="text-transform: none;">Mapping of DNA-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">binding proteins and chromatin</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/square.gif" alt="&gt;" border="0"></span>Deep sampling of transcriptomes</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500421" class="sidefm-pmclink" style="text-transform: none;">Next-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">generation sequencing compared with microarrays</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500559" class="sidefm-pmclink" style="text-transform: none;">Concluding remarks</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500573" class="sidefm-pmclink" style="text-transform: none;">Note added in proof (received 4 August 2008)</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500611" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id500294">Deep sampling of transcriptomes</div><div class="section-content"><p>Next-generation
sequencing is also changing the ways in which gene expression is
studied, which is likely to have much future impact. Complex RNA
mixtures can be analysed using sequence census methods, an approach
termed ‘RNA-Seq’. Initial applications include the accelerated
discovery of small RNAs [<a href="#B37" class="cite-reflink">37</a>–<a href="#B40" class="cite-reflink">40</a>]. Other studies have used early RNA-Seq approaches to quantify expression levels using a modified paired-end ditagging method [<a href="#B41" class="cite-reflink">41</a>], to detect rare cardiac mRNAs in mouse by ‘polony multiplex analysis of gene expression’ [<a href="#B42" class="cite-reflink">42</a>,<a href="#B43" class="cite-reflink">43</a>], or to directly sequence cDNAs of human tumour and fly cells [<a href="#B44" class="cite-reflink">44</a>–<a href="#B46" class="cite-reflink">46</a>].
Another RNA-Seq study has moved beyond simply describing the expression
levels of transcripts towards assigning functions to the observed
expression differences [<a href="#B47" class="cite-reflink">47</a>].
Together, such studies pave the way for complete transcriptome
coverage, providing the ultimate resolution to analyse the levels as
well as the structures of both processed and unprocessed transcripts
under different conditions.</p><p>We have recently applied RNA-Seq,
complemented with high-resolution tiling arrays, to obtain a detailed
picture of the fission yeast transcriptome, independently of available
gene annotations, at the best possible resolution [<a href="#B48" class="cite-reflink">48</a>].
The transcriptome has been interrogated under multiple conditions,
including rapid proliferation, meiotic differentiation and
environmental stress, as well as in splicing and exosome mutants, to
analyse the dynamic adaptation of the transcriptional landscape as a
function of environmental, developmental and genetic factors. These
results provide rich, condition-specific information on widespread
transcription, on novel, mostly non-coding transcripts, as well as on
untranslated regions and gene structures, thus improving the existing
genome annotation. Perhaps most interestingly, sequence reads spanning
exon–exon or exon–intron junctions have given a unique and direct
insight into a surprising variability in splicing efficiency across
introns, genes and conditions. This analysis has revealed that splicing
efficiency is largely co-ordinated with transcript levels, and hundreds
of introns show regulated splicing during cellular proliferation or
differentiation. These results suggest a global co-ordination between
splicing efficiency and transcription, which may help to optimize and
streamline gene expression programmes. As elaborated in the next
section, the combined RNA-Seq and array data have also allowed
comparing and contrasting of the relative performance and properties of
sequencing- and hybridization-based approaches.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399430" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411622" class="sidefm-pmclink" style="text-transform: none;">Sequencing as never before</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411699" class="sidefm-pmclink" style="text-transform: none;">From genomes to function</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500123" class="sidefm-pmclink" style="text-transform: none;">Mapping of DNA-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">binding proteins and chromatin</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500294" class="sidefm-pmclink" style="text-transform: none;">Deep sampling of transcriptomes</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/square.gif" alt="&gt;" border="0"></span>Next-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">generation sequencing compared with microarrays</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500559" class="sidefm-pmclink" style="text-transform: none;">Concluding remarks</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500573" class="sidefm-pmclink" style="text-transform: none;">Note added in proof (received 4 August 2008)</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500611" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id500421">Next-generation sequencing compared with microarrays</div><div class="section-content"><p>Currently,
global analysis of gene expression relies largely on
hybridization-based platforms such as microarrays, which are routinely
used for determining relative expression levels or changes in gene
expression between different biological conditions. Unlike
hybridization data, which consist of continuous signals, sequence
census data are made of absolute numbers of reads (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2563889&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2</a>).
The countable, almost digital, nature of these results makes them
highly suitable for the analysis of gene expression levels. Applying
sequence census approaches to cDNAs (RNA-Seq), we can therefore
estimate the relative abundance of given transcripts by counting the
number of times they are hit by sequence reads. Recent studies have
shown that, indeed, scores based on the number of sequence reads
hitting a transcript, or on the average number of hits per base and per
transcript, provide accurate measurements of relative RNA levels [<a href="#B45" class="cite-reflink">45</a>,<a href="#B46" class="cite-reflink">46</a>,<a href="#B48" class="cite-reflink">48</a>].
We have shown that sequencing-based estimates of transcript abundance
are in good agreement with estimates acquired with microarrays,
provided that sequencing depth is sufficient [<a href="#B48" class="cite-reflink">48</a>].
In addition, unlike microarray data, which are affected by the dynamic
range of the scanner, sequence data have a linear dynamic range only
limited by the sequencing depth. This aspect is attractive, because the
dynamic range of different RNAs in a cell is almost certainly larger
than the range provided by microarray scanners. In addition, and unlike
hybridization-based techniques, sequencing-based approaches produce
little or no noise, allowing detection of even very minimally expressed
transcripts [<a href="#B48" class="cite-reflink">48</a>]. In the short
term, however, the costs and amount of data produced make it unlikely
that sequence census approaches will completely replace microarrays as
the routine tool for expression profiling.</p><p>The structure of
eukaryotic transcriptomes has received considerable attention with the
availability of high-density tiling arrays [<a href="#B49" class="cite-reflink">49</a>].
These arrays consist of probes tiled evenly across the genome allowing
characterization of transcript structure without prior knowledge of
genome annotation. Sequence census methods are likely to provide an
exciting new twist in the structural analysis of transcriptomes. Unlike
tiling arrays whose resolution is limited by the number of probes on
the platform, sequencing provides, by default, the best possible
resolution. This feature may prove particularly powerful for dense
genomes with small gene features or for large genomes that would
otherwise require a substantial number of tiling arrays to provide
adequate resolution. In addition, sequencing of the fission yeast
transcriptome has proved sensitive enough to detect widespread
transcription in &gt;90% of the genome, including traces of RNAs that
are not robustly transcribed or rapidly degraded [<a href="#B48" class="cite-reflink">48</a>],
such that very rare isoforms expressed at levels below the detection
threshold of tiling arrays can be identified. However, the sequencing
costs incurred to reach a sufficiently deep coverage of large genomes
remain currently an issue. Fortunately, the costs are likely to
decrease further, allowing a wider and more extensive use of these
technologies.</p><p>Sequence census technologies have been developed to
analyse double-stranded DNA. When it comes to the analysis of
transcriptomes, converting RNA into double-stranded cDNA is required,
which may, in many protocols, result in the loss of strand-specific
information. Given the large extent of overlapping and antisense
transcription reported even in simple eukaryotes, this is clearly an
issue. Moreover, classical protocols for cDNA synthesis do not produce
samples allowing unambiguous detection of overlapping transcription.
Therefore, in addition to classical cDNA analysis, techniques allowing
the specific analysis of the 5′- and 3′-ends of transcripts combined
with sequencing census approaches should prove useful for unambiguously
determining the extent and structure of different transcripts.</p><p>Finally,
a unique feature of sequence census technologies is their ability to
identify, without prior knowledge, transcripts made of sequences that
are not adjacent in the genome but that are connected when they are
expressed. For instance, spliced transcripts can be uniquely detected
through the presence of sequence reads spanning exon–exon junctions.
Such positive evidence for splicing is not available from tiling array
data, and although spliced transcripts could be probed with specially
designed arrays, it would require <em>a priori</em> knowledge of the
splice sites. For these reasons, sequence census technologies will
provide powerful and versatile tools to study post-transcriptional
processing of genetic sequences.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399430" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411622" class="sidefm-pmclink" style="text-transform: none;">Sequencing as never before</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411699" class="sidefm-pmclink" style="text-transform: none;">From genomes to function</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500123" class="sidefm-pmclink" style="text-transform: none;">Mapping of DNA-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">binding proteins and chromatin</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500294" class="sidefm-pmclink" style="text-transform: none;">Deep sampling of transcriptomes</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500421" class="sidefm-pmclink" style="text-transform: none;">Next-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">generation sequencing compared with microarrays</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/square.gif" alt="&gt;" border="0"></span>Concluding remarks</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500573" class="sidefm-pmclink" style="text-transform: none;">Note added in proof (received 4 August 2008)</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500611" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id500559">Concluding remarks</div><div class="section-content"><p>Next-generation
sequencing technologies have had an enormous impact on research within
a short time frame, and this impact appears certain to increase
further, as many institutions are now acquiring these prevailing new
sequencing platforms. Beyond conventional sampling of genome content,
wide-ranging applications are rapidly evolving for next-generation
sequencing. Sequence census methods such as ChIP-Seq and RNA-Seq are
becoming powerful and quantitative approaches to analyse the structures
and functions of both genomes and transcriptomes at maximal resolution.
At this time, the huge amount of data generated by next-generation
sequencing creates an informatics challenge. The establishment of
routine data analysis methods, together with future decreases in
sequencing costs and increases in the numbers and lengths of sequence
reads, will help to unleash the full potential of next-generation
sequencing.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id500573">Note added in proof (received 4 August 2008)</div><div class="section-content"><p>Since submission of this paper, six additional papers have been published reporting various applications of RNA-Seq [<a href="#B50" class="cite-reflink">50</a>–<a href="#B55" class="cite-reflink">55</a>].</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145">&nbsp;</td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id500604">Acknowledgments</div><div class="section-content"><p></p><p>We
thank Josette-Renée Landry, Vera Pancaldi and Falk Schubert for
comments on this paper. S.M. was supported by a Fellowship for Advanced
Researchers from the Swiss National Science Foundation, and B.T.W. by
Sanger postdoctoral and Canadian NSERC (Natural Sciences and
Engineering Research Council) fellowships. Work in our laboratory is
funded by Cancer Research UK grant number C9546/A6517.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399430" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411622" class="sidefm-pmclink" style="text-transform: none;">Sequencing as never before</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id411699" class="sidefm-pmclink" style="text-transform: none;">From genomes to function</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500123" class="sidefm-pmclink" style="text-transform: none;">Mapping of DNA-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">binding proteins and chromatin</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500294" class="sidefm-pmclink" style="text-transform: none;">Deep sampling of transcriptomes</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500421" class="sidefm-pmclink" style="text-transform: none;">Next-<img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/dot.gif" border="0">generation sequencing compared with microarrays</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500559" class="sidefm-pmclink" style="text-transform: none;">Concluding remarks</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id500573" class="sidefm-pmclink" style="text-transform: none;">Note added in proof (received 4 August 2008)</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Marguerat%202008%20Next-generation%20sequencing%20-%20applications%20beyond%20genomes_files/square.gif" alt="&gt;" border="0"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id500611">References</div><div class="section-content"><div class="back-matter-section"><div class="ref-cit-blk" id="B1"><div class="ref-label">1.</div><div class="ref-cit">Margulies,
M.; Egholm, M.; Altman, W.E.; Attiya, S.; Bader, J.S.; Bemben, L.A.;
Berka, J.; Braverman, M.S.; Chen, Y.-J.; Chen, Z., et al. Genome
sequencing in microfabricated high-density picolitre reactors. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:376–380.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0jaV31Hvdk3O1V4km79IaeOxY06VjUTxvDPcrLQxk&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16056220">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B2"><div class="ref-label">2.</div><div class="ref-cit">Shendure,
J.; Porreca, G.J.; Reppas, N.B.; Lin, X.; McCutcheon, J.P.; Rosenbaum,
A.M.; Wang, M.D.; Zhang, K.; Mitra, R.D.; Church, G.M. Accurate
multiplex polony sequencing of an evolved bacterial genome. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">309</span>:1728–1732.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ty0Om1YMFSlRZrDCtnIls7gB8WeVWkmpWa-iiQ7K&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16081699">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B3"><div class="ref-label">3.</div><div class="ref-cit">Rusk, N.; Kiermer, V. Primer: sequencing: the next generation. <span><span class="ref-journal">Nat. Methods. </span>2008;<span class="ref-vol">5</span>:15.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0PSYrxP1aVwsyKVi3FTt93ORGlVpqOZxr_NwhLAcr&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18175411">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B4"><div class="ref-label">4.</div><div class="ref-cit">Schuster, S.C. Next-generation sequencing transforms today's biology. <span><span class="ref-journal">Nat. Methods. </span>2008;<span class="ref-vol">5</span>:16–18.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0V9Qn5eJnwdb62TrgFT_j-98MIrWHaQakap6ij5wG&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18165802">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B5"><div class="ref-label">5.</div><div class="ref-cit">Sanger, F.; Nicklen, S.; Coulson, A.R. DNA sequencing with chain-terminating inhibitors. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>1977;<span class="ref-vol">74</span>:5463–5467.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0XMEqKzfwoH7lAMsrtjdDrbQlYG9HCrdKQQoh9QL7&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/271968">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B6"><div class="ref-label">6.</div><div class="ref-cit">Angly,
F.E.; Felts, B.; Breitbart, M.; Salamon, P.; Edwards, R.A.; Carlson,
C.; Chan, A.M.; Haynes, M.; Kelley, S.; Liu, H., et al. The marine
viromes of four oceanic regions. <span><span class="ref-journal">PLoS Biol. </span>2006;<span class="ref-vol">4</span>:e368.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Ji6P4DAKzyVdCABEJN04vXLI5wb7MzE-7TL7-AMP&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17090214">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B7"><div class="ref-label">7.</div><div class="ref-cit">Edwards,
R.A.; Rodriguez-Brito, B.; Wegley, L.; Haynes, M.; Breitbart, M.;
Peterson, D.M.; Saar, M.O.; Alexander, S.; Alexander, E.C., Jr; Rohwer,
F. Using pyrosequencing to shed light on deep mine microbial ecology. <span><span class="ref-journal">BMC Genomics. </span>2006;<span class="ref-vol">7</span>:57.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0jCU9n-mIZ_HmxDi3PAosCSQZgsJAEmoISChe9kzX&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16549033">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B8"><div class="ref-label">8.</div><div class="ref-cit">Sogin,
M.L.; Morrison, H.G.; Huber, J.A.; Mark&nbsp;Welch, D.; Huse, S.M.;
Neal, P.R.; Arrieta, J.M.; Herndl, G.J. Microbial diversity in the deep
sea and the underexplored ‘rare biosphere’ <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>2006;<span class="ref-vol">103</span>:12115–12120.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0VGoacPeowxVpp_4X9XPdICe5sI0__kTZgaO7k51A&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16880384">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B9"><div class="ref-label">9.</div><div class="ref-cit">Cox-Foster,
D.L.; Conlan, S.; Holmes, E.C.; Palacios, G.; Evans, J.D.; Moran, N.A.;
Quan, P.L.; Briese, T.; Hornig, M.; Geiser, D.M., et al. A metagenomic
survey of microbes in honey bee colony collapse disorder. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">318</span>:283–287.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0zup-MwZEtpnzkYN4L_Ft1b_oFpZb30cqXCcq9kSP&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17823314">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B10"><div class="ref-label">10.</div><div class="ref-cit">Huber,
J.A.; Mark&nbsp;Welch, D.B.; Morrison, H.G.; Huse, S.M.; Neal, P.R.;
Butterfield, D.A.; Sogin, M.L. Microbial population structures in the
deep marine biosphere. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">318</span>:97–100.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0j7thALwigwTlD5ZmAj_8VPo1UymkXo1Fx5NIVA66&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17916733">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B11"><div class="ref-label">11.</div><div class="ref-cit">Tringe, S.G.; Rubin, E.M. Metagenomics: DNA sequencing of environmental samples. <span><span class="ref-journal">Nat. Rev. Genet. </span>2005;<span class="ref-vol">6</span>:805–814.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0tI4O7ZpYDnvvAshN0EDFeWEWVbAttUEfsaL7Q5Ct&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16304596">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B12"><div class="ref-label">12.</div><div class="ref-cit">Warnecke,
F.; Luginbühl, P.; Ivanova, N.; Ghassemian, M.; Richardson, T.H.;
Stege, J.T.; Cayouette, M.; McHardy, A.C.; Djordjevic, G.; Aboushadi,
N., et al. Metagenomic and functional analysis of hindgut microbiota of
a wood-feeding higher termite. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">450</span>:560–565.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Z9K60RHU9lTw4vr1_b7zLYL7kGqjhs7JNYc05kw2&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18033299">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B13"><div class="ref-label">13.</div><div class="ref-cit">Woyke,
T.; Teeling, H.; Ivanova, N.N.; Huntemann, M.; Richter, M.; Gloeckner,
F.O.; Boffelli, D.; Anderson, I.J.; Barry, K.W.; Shapiro, H.J., et al.
Symbiosis insights through metagenomic analysis of a microbial
consortium. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">443</span>:950–955.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0XcZ-6QP044FDci0fC3dLVgXgbPZHfsDi9abE6kuC&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16980956">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B14"><div class="ref-label">14.</div><div class="ref-cit">Poinar,
H.N.; Schwarz, C.; Qi, J.; Shapiro, B.; Macphee, R.D.; Buigues, B.;
Tikhonov, A.; Huson, D.H.; Tomsho, L.P.; Auch, A., et al. Metagenomics
to paleogenomics: large-scale sequencing of mammoth DNA. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">311</span>:392–394.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0VaAZuOjn16TLVwGJ0X9lGwhiQnxMFyLu7Y6XDkVR&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16368896">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B15"><div class="ref-label">15.</div><div class="ref-cit">Green,
R.E.; Krause, J.; Ptak, S.E.; Briggs, A.W.; Ronan, M.T.; Simons, J.F.;
Du, L.; Egholm, M.; Rothberg, J.M.; Paunovic, M.; Paabo, S. Analysis of
one million base pairs of Neanderthal DNA. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">444</span>:330–336.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0R_7XoEyP_cx3advFwd2ntL26mmPCtEFc57rmRkPB&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17108958">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B16"><div class="ref-label">16.</div><div class="ref-cit">Noonan,
J.P.; Coop, G.; Kudaravalli, S.; Smith, D.; Krause, J.; Alessi, J.;
Chen, F.; Platt, D.; Paabo, S.; Pritchard, J.K.; Rubin, E.M. Sequencing
and analysis of Neanderthal genomic DNA. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">314</span>:1113–1118.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=06OmDOHE6ws3o-Fi8Epn7W5Qkmm9XYRTpbKb-7Qkj&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17110569">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B17"><div class="ref-label">17.</div><div class="ref-cit">Hofreuter,
D.; Tsai, J.; Watson, R.O.; Novik, V.; Altman, B.; Benitez, M.; Clark,
C.; Perbost, C.; Jarvie, T.; Du, L.; Galan, J.E. Unique features of a
highly pathogenic <em>Campylobacter jejuni</em> strain. <span><span class="ref-journal">Infect. Immun. </span>2006;<span class="ref-vol">74</span>:4694–4707.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0LOzbX8wXA14fqCvHcm1M-XnrnyhOifb-TRzqOAXm&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16861657">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B18"><div class="ref-label">18.</div><div class="ref-cit">Oh,
J.D.; Kling-Backhed, H.; Giannakis, M.; Xu, J.; Fulton, R.S.; Fulton,
L.A.; Cordum, H.S.; Wang, C.; Elliott, G.; Edwards, J., et al. The
complete genome sequence of a chronic atrophic gastritis <em>Helicobacter pylori</em> strain: evolution during disease progression. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>2006;<span class="ref-vol">103</span>:9999–10004.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0zOVP-ONQVM1hYFKMYoArjTrMWgJRlD5MMrIUSAWA&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16788065">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B19"><div class="ref-label">19.</div><div class="ref-cit">Hoffmann,
C.; Minkah, N.; Leipzig, J.; Wang, G.; Arens, M.Q.; Tebas, P.; Bushman,
F.D. DNA bar coding and pyrosequencing to identify rare HIV drug
resistance mutations. <span><span class="ref-journal">Nucleic Acids Res. </span>2007;<span class="ref-vol">35</span>:e91.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0FUhVhpfZcP4Mle1CQalg-yxY9H1ZUpSu4Zy5Vksv&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17576693">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B20"><div class="ref-label">20.</div><div class="ref-cit">Wang,
C.; Mitsuya, Y.; Gharizadeh, B.; Ronaghi, M.; Shafer, R.W.
Characterization of mutation spectra with ultra-deep pyrosequencing:
application to HIV-1 drug resistance. <span><span class="ref-journal">Genome Res. </span>2007;<span class="ref-vol">17</span>:1195–1201.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=04qJC64j91bkomTJCwe5usiqnBUxDDoJ2MihW5A7M&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17600086">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B21"><div class="ref-label">21.</div><div class="ref-cit">Wheeler,
D.A.; Srinivasan, M.; Egholm, M.; Shen, Y.; Chen, L.; McGuire, A.; He,
W.; Chen, Y.-J.; Makhijani, V.; Roth, G.T., et al. The complete genome
of an individual by massively parallel DNA sequencing. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">452</span>:872–876.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=02stlz3dcHnHM5zI9t5LhlfLDY7iETcrXrwQnW5Kt&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18421352">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B22"><div class="ref-label">22.</div><div class="ref-cit">Andries,
K.; Verhasselt, P.; Guillemont, J.; Gohlmann, H.W.; Neefs, J.M.;
Winkler, H.; Van&nbsp;Gestel, J.; Timmerman, P.; Zhu, M.; Lee, E., et
al. A diarylquinoline drug active on the ATP synthase of <em>Mycobacterium tuberculosis</em>. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">307</span>:223–227.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0BCKVWTwudeuUMU4VXy0iUtZiWwbMbLkdEsD2ykSg&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15591164">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B23"><div class="ref-label">23.</div><div class="ref-cit">Velicer,
G.J.; Raddatz, G.; Keller, H.; Deiss, S.; Lanz, C.; Dinkelacker, I.;
Schuster, S.C. Comprehensive mutation identification in an evolved
bacterial cooperator and its cheating ancestor. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>2006;<span class="ref-vol">103</span>:8107–8112.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0js1n1mnLdsQ5kA-ojwfhRl_QWBm582R5dyBBXk3R&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16707573">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B24"><div class="ref-label">24.</div><div class="ref-cit">Hillier,
L.W.; Marth, G.T.; Quinlan, A.R.; Dooling, D.; Fewell, G.; Barnett, D.;
Fox, P.; Glasscock, J.I.; Hickenbotham, M.; Huang, W., et al.
Whole-genome sequencing and variant discovery in <em>C. elegans</em>. <span><span class="ref-journal">Nat. Methods. </span>2008;<span class="ref-vol">5</span>:183–188.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0JiWM2CCJTNIhpDuTt_cL1cLxJF1FlQO0cV7tm5GI&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18204455">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B25"><div class="ref-label">25.</div><div class="ref-cit">Korbel, J.O. Paired-end mapping reveals extensive structural variation in the human genome. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">318</span>:420–426.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0f1HNSd1V4nRqmszVw17v5w0NG1204lyK2Q9-MkMw&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17901297">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B26"><div class="ref-label">26.</div><div class="ref-cit">Thomas,
R.K.; Baker, A.C.; Debiasi, R.M.; Winckler, W.; Laframboise, T.; Lin,
W.M.; Wang, M.; Feng, W.; Zander, T.; Macconnaill, L.E., et al.
High-throughput oncogene mutation profiling in human cancer. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:347–351.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0HKW9o8UTcEcJRxWVL3N-6Dm30KaXfQCp6AHSQAP3&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17293865">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B27"><div class="ref-label">27.</div><div class="ref-cit">Wold, B.; Myers, R.M. Sequence census methods for functional genomics. <span><span class="ref-journal">Nat. Methods. </span>2008;<span class="ref-vol">5</span>:19–21.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0doK_qC7LELSCF1HdiVDDlEdykW1rs_FgceC_IA_J&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18165803">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B28"><div class="ref-label">28.</div><div class="ref-cit">Velculescu, V.E.; Zhang, L.; Vogelstein, B.; Kinzler, K.W. Serial analysis of gene expression. <span><span class="ref-journal">Science. </span>1995;<span class="ref-vol">270</span>:484–487.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0dqfk6znxk1FBY--hY8E2DlsTsAZJtNKVVZflr5U4&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7570003">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B29"><div class="ref-label">29.</div><div class="ref-cit">Brenner,
S.; Johnson, M.; Bridgham, J.; Golda, G.; Lloyd, D.H.; Johnson, D.;
Luo, S.; McCurdy, S.; Foy, M.; Ewan, M., et al. Gene expression
analysis by massively parallel signature sequencing (MPSS) on microbead
arrays. <span><span class="ref-journal">Nat. Biotechnol. </span>2000;<span class="ref-vol">18</span>:630–634.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0-90_TFErrj2kHAn2m9kAouXAUIwOkPOpHugayQIa&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10835600">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B30"><div class="ref-label">30.</div><div class="ref-cit">Iyer,
V.R.; Horak, C.E.; Scafe, C.S.; Botstein, D.; Snyder, M.; Brown, P.O.
Genomic binding sites of the yeast cell-cycle transcription factors SBF
and MBF. <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">409</span>:533–538.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0vOHtqqzCZd6yxYMtkw08q_LBaatm2PxhePkLj5zz&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11206552">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B31"><div class="ref-label">31.</div><div class="ref-cit">Ren,
B.; Robert, F.; Wyrick, J.J.; Aparicio, O.; Jennings, E.G.; Simon, I.;
Zeitlinger, J.; Schreiber, J.; Hannett, N.; Kanin, E., et al.
Genome-wide location and function of DNA binding proteins. <span><span class="ref-journal">Science. </span>2000;<span class="ref-vol">290</span>:2306–2309.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0X3U4iLfNmupamCMNTHJ43meVHGijW7Uk5mqtEQpI&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11125145">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B32"><div class="ref-label">32.</div><div class="ref-cit">Johnson, D.S.; Mortazavi, A.; Myers, R.M.; Wold, B. Genome-wide mapping of <em>in vivo</em> protein–DNA interactions. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1497–1502.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0zQqpSup-Gp_OwJTyjLlMWZJHXVfYM2kypMBUQkDG&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17540862">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B33"><div class="ref-label">33.</div><div class="ref-cit">Robertson,
G.; Hirst, M.; Bainbridge, M.; Bilenky, M.; Zhao, Y.; Zeng, T.;
Euskirchen, G.; Bernier, B.; Varhol, R.; Delaney, A., et al.
Genome-wide profiles of STAT1 DNA association using chromatin
immunoprecipitation and massively parallel sequencing. <span><span class="ref-journal">Nat. Methods. </span>2007;<span class="ref-vol">4</span>:651–657.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0PmrhBAvKgw6WtuisupKt7b2PmSLIvxaNOv4PCkQL&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17558387">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B34"><div class="ref-label">34.</div><div class="ref-cit">Barski,
A.; Cuddapah, S.; Cui, K.; Roh, T.Y.; Schones, D.E.; Wang, Z.; Wei, G.;
Chepelev, I.; Zhao, K. High-resolution profiling of histone
methylations in the human genome. <span><span class="ref-journal">Cell. </span>2007;<span class="ref-vol">129</span>:823–837.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0J9vToH3hXTZipQkcPpqm-f58wAj2TahADU8XP5l4&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17512414">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B35"><div class="ref-label">35.</div><div class="ref-cit">Mikkelsen,
T.S.; Ku, M.; Jaffe, D.B.; Issac, B.; Lieberman, E.; Giannoukos, G.;
Alvarez, P.; Brockman, W.; Kim, T.K.; Koche, R.P., et al. Genome-wide
maps of chromatin state in pluripotent and lineage-committed cells. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">448</span>:553–560.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0tAC5yAvylKX6dLEv5cobIXuxgnZ4RiEoZunZ-5XR&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17603471">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B36"><div class="ref-label">36.</div><div class="ref-cit">Boyle,
A.P.; Davis, S.; Shulha, H.P.; Meltzer, P.; Margulies, E.H.; Weng, Z.;
Furey, T.S.; Crawford, G.E. High-resolution mapping and
characterization of open chromatin across the genome. <span><span class="ref-journal">Cell. </span>2008;<span class="ref-vol">132</span>:311–322.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0JKT6kdYACKeE0y3IQmanYly_8Cm7itZoIdhX4A_f&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18243105">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B37"><div class="ref-label">37.</div><div class="ref-cit">Lu,
C.; Kulkarni, K.; Souret, F.F.; MuthuValliappan, R.; Tej, S.S.;
Poethig, R.S.; Henderson, I.R.; Jacobsen, S.E.; Wang, W.; Green, P.J.;
Meyers, B.C. MicroRNAs and other small RNAs enriched in the <em>Arabidopsis</em> RNA-dependent RNA polymerase-2 mutant. <span><span class="ref-journal">Genome Res. </span>2006;<span class="ref-vol">16</span>:1276–1288.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0jYTzqujGJAxo4iedQvJoCGISLvSNdUiACjxxT5rA&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16954541">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B38"><div class="ref-label">38.</div><div class="ref-cit">Ruby,
J.G.; Jan, C.; Player, C.; Axtell, M.J.; Lee, W.; Nusbaum, C.; Ge, H.;
Bartel, D.P. Large-scale sequencing reveals 21U-RNAs and additional
microRNAs and endogenous siRNAs in <em>C. elegans</em>. <span><span class="ref-journal">Cell. </span>2006;<span class="ref-vol">127</span>:1193–1207.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0n3AUrsh-PrsF7f_PKKSCN8ccrrFjAAApgJuFLQKc&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17174894">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B39"><div class="ref-label">39.</div><div class="ref-cit">Stark,
A.; Bushati, N.; Jan, C.H.; Kheradpour, P.; Hodges, E.; Brennecke, J.;
Bartel, D.P.; Cohen, S.M.; Kellis, M. A single Hox locus in <em>Drosophila</em> produces functional microRNAs from opposite DNA strands. <span><span class="ref-journal">Genes Dev. </span>2008;<span class="ref-vol">22</span>:8–13.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0LEGQ54ZHD_AQl8WXHcnLSdWS0G9-1ZnB_8Elo5X8&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18172160">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B40"><div class="ref-label">40.</div><div class="ref-cit">Tyler,
D.M.; Okamura, K.; Chung, W.J.; Hagen, J.W.; Berezikov, E.; Hannon,
G.J.; Lai, E.C. Functionally distinct regulatory RNAs generated by
bidirectional transcription and processing of microRNA loci. <span><span class="ref-journal">Genes Dev. </span>2008;<span class="ref-vol">22</span>:26–36.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0lOpnWrV2zIyAbYLjK8wfezAcl1kB6vbINrJ7kA27&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18172163">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B41"><div class="ref-label">41.</div><div class="ref-cit">Ng,
P.; Tan, J.J.; Ooi, H.S.; Lee, Y.L.; Chiu, K.P.; Fullwood, M.J.;
Srinivasan, K.G.; Perbost, C.; Du, L.; Sung, W.K., et al. Multiplex
sequencing of paired-end ditags (MS-PET): a strategy for the
ultra-high-throughput analysis of transcriptomes and genomes. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:e84.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0pq_9WsjDvtPmxBpxGK2CuMEpy_SqxPaHtX8j8ki5&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16840528">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B42"><div class="ref-label">42.</div><div class="ref-cit">Kim,
J.B.; Porreca, G.J.; Song, L.; Greenway, S.C.; Gorham, J.M.; Church,
G.M.; Seidman, C.E.; Seidman, J.G. Polony multiplex analysis of gene
expression (PMAGE) in mouse hypertrophic cardiomyopathy. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1481–1484.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=04YRq2ZdpxKVEAKPZaB8toWZOsALU5L4YJPCXUkst&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17556586">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B43"><div class="ref-label">43.</div><div class="ref-cit">Velculescu, V.E.; Kinzler, K.W. Gene expression analysis goes digital. <span><span class="ref-journal">Nat. Biotechnol. </span>2007;<span class="ref-vol">25</span>:878–880.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0-yolISx0unNoszDTUMt8a6hVc6DT1a9L2AbXOAPy&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17687366">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B44"><div class="ref-label">44.</div><div class="ref-cit">Bainbridge,
M.N.; Warren, R.L.; Hirst, M.; Romanuik, T.; Zeng, T.; Go, A.; Delaney,
A.; Griffith, M.; Hickenbotham, M.; Magrini, V., et al. Analysis of the
prostate cancer cell line LNCaP transcriptome using a
sequencing-by-synthesis approach. <span><span class="ref-journal">BMC Genomics. </span>2006;<span class="ref-vol">7</span>:246.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0le7fwu59ZF5asjuU5AJyBq0oJN1u2INfXCzT1AIs&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17010196">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B45"><div class="ref-label">45.</div><div class="ref-cit">Sugarbaker,
D.J.; Richards, W.G.; Gordon, G.J.; Dong, L.; De&nbsp;Rienzo, A.;
Maulik, G.; Glickman, J.N.; Chirieac, L.R.; Hartman, M.L.; Taillon,
B.E., et al. Transcriptome sequencing of malignant pleural mesothelioma
tumors. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A. </span>2008;<span class="ref-vol">105</span>:3521–3526.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ZocBGT50NxODKk5ECcay7HB90gsw7WHK903MW5al&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18303113">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B46"><div class="ref-label">46.</div><div class="ref-cit">Torres, T.T.; Metta, M.; Ottenwalder, B.; Schlotterer, C. Gene expression profiling by massively parallel sequencing. <span><span class="ref-journal">Genome Res. </span>2008;<span class="ref-vol">18</span>:172–177.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=08OBml9EIZf61Yu7NPg9fiS2N1aOtCIq3od9Glk40&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18032722">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B47"><div class="ref-label">47.</div><div class="ref-cit">Toth,
A.L.; Varala, K.; Newman, T.C.; Miguez, F.E.; Hutchison, S.K.;
Willoughby, D.A.; Simons, J.F.; Egholm, M.; Hunt, J.H.; Hudson, M.E.;
Robinson, G.E. Wasp gene expression supports an evolutionary link
between maternal behavior and eusociality. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">318</span>:441–444.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0Ng-PFQd8RDDkz30fOFJ__hFu18H2HG6dekwd_AAO&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17901299">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B48"><div class="ref-label">48.</div><div class="ref-cit">Wilhelm,
B.T.; Marguerat, S.; Watt, S.; Schubert, F.; Wood, V.; Goodhead, I.;
Penkett, C.J.; Rogers, J.; Bähler, J. Dynamic repertoire of a
eukaryotic transcriptome surveyed at single nucleotide resolution. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">453</span>:1239–1243.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0zI9874du_43ASnR7P236bTd8xeaIvPUmkBVGykjL&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18488015">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B49"><div class="ref-label">49.</div><div class="ref-cit">Kapranov, P.; Willingham, A.T.; Gingeras, T.R. Genome-wide transcription and the implications for genomic organization. <span><span class="ref-journal">Nat. Rev. Genet. </span>2007;<span class="ref-vol">8</span>:413–423.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0pCxVgA1pxKk2-lBo6sB6OLEkbwMJFyZT4ZX4ZkNp&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17486121">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B50"><div class="ref-label">50.</div><div class="ref-cit">Nagalakshmi,
U.; Wang, Z.; Waern, K.; Shou, C.; Raha, D.; Gerstein, M.; Snyder, M.
The transcriptional landscape of the yeast genome defined by RNA
sequencing. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">320</span>:1344–1349.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0xOsnT2ff3tx7eTZtQkpIZmra7D7VbUatZ-W1wkug&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18451266">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B51"><div class="ref-label">51.</div><div class="ref-cit">Lister,
R.; O'Malley, R.C.; Tonti-Filippini, J.; Gregory, B.D.; Berry, C.C.;
Millar, A.H.; Ecker, J.R. Highly integrated single-base resolution maps
of the epigenome in <em>Arabidopsis</em>. <span><span class="ref-journal">Cell. </span>2008;<span class="ref-vol">133</span>:523–536.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=08KPu9w3cwHN2Dl-Ad8-4V_ZYGNjGDPKzubEO0Q2X&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18423832">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B52"><div class="ref-label">52.</div><div class="ref-cit">Mortazavi, A.; Williams, B.A.; McCue, K.; Schaeffer, L.; Wold, B. Mapping and quantifying mammalian transcriptomes by RNA-Seq. <span><span class="ref-journal">Nat. Methods. </span>2008;<span class="ref-vol">5</span>:621–628.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0xqx6FKzKtgJxsRfVaBEzYsWCf-TcOJlorbhjEA3F&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18516045">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B53"><div class="ref-label">53.</div><div class="ref-cit">Cloonan,
N.; Forrest, A.R.; Kolle, G.; Gardiner, B.B.; Faulkner, G.J.; Brown,
M.K.; Taylor, D.F.; Steptoe, A.L.; Wani, S.; Bethel, G., et al. Stem
cell transcriptome profiling via massive-scale mRNA sequencing. <span><span class="ref-journal">Nat. Methods. </span>2008;<span class="ref-vol">5</span>:613–619.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0fEEhXbRuhQadZMcuYyI4j4ETqSEP_qzVob7PyAm_&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18516046">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B54"><div class="ref-label">54.</div><div class="ref-cit">Morin,
R.; Bainbridge, M.; Fejes, A.; Hirst, M.; Krzywinski, M.; Pugh, T.;
McDonald, H.; Varhol, R.; Jones, S.; Marra, M. Profiling the HeLa S3
transcriptome using randomly primed cDNA and massively parallel
short-read sequencing. <span><span class="ref-journal">Bio Techniques. </span>2008;<span class="ref-vol">45</span>:81–94.</span></div></div><div class="ref-cit-blk" id="B55"><div class="ref-label">55.</div><div class="ref-cit">Sultan,
M.; Schulz, M.H.; Richard, H.; Magen, A.; Klingenholff, A.; Scherf, M.;
Seifert, M.; Borodina, T.; Soldatov, A.; Parkhomchuk, D., et al. A
global view of gene activity and alternative splicing by deep
sequencing of the human transcriptome. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">321</span>:956–960.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0bsdbSy1HU3n9Ia95_Lz-UppyeO91fukmwcBxOQ7B&amp;reftype=pubmed&amp;artid=2563889&amp;article-id=2563889&amp;iid=170999&amp;issue-id=170999&amp;jid=671&amp;journal-id=671&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18599741">PubMed</a>]</span></div></div></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">Portland Press Opt2Pay</span> are provided here courtesy of <br><b>Portland Press Ltd</b></div><div class="footer-link" id="fp-footer"><div><a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>&nbsp;|&nbsp;<a class="link" href="http://www.pubmedcentral.nih.gov/">PMC Home</a>&nbsp;|&nbsp;<a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a></div><div><a class="link" href="http://www.ncbi.nlm.nih.gov/">NCBI</a>&nbsp;|&nbsp;<a class="link" href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></div><div><a class="link" href="http://www.nih.gov/">NIH</a>&nbsp;|&nbsp;<a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a></div><div><a class="link" href="http://www.pubmedcentral.nih.gov/about/privacypolicy.html">Privacy Policy</a>&nbsp;|&nbsp;<a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>&nbsp;|&nbsp;<a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a></div></div></div></td></tr></tbody></table><!--host:ipmc3-->

</body></html>