<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xml:lang="en" xmlns="http://www.w3.org/1999/xhtml" lang="en"><head>

   
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
      <title>Emerging technologies in DNA sequencing  —  Genome Research </title>
      <meta name="DC.Format" content="text/html">
      <meta name="DC.Language" content="en">
      <meta content="Emerging technologies in DNA sequencing" name="DC.Title">
      <meta content="10.1101/gr.3770505" name="DC.Identifier">
      <meta content="2005-12-01" name="DC.Date">
      <meta content="Cold Spring Harbor Laboratory Press" name="DC.Publisher">
      <meta content="Michael L. Metzker" name="DC.Contributor">
      <meta content="Genome Research" name="citation_journal_title">
      <meta content="Cold Spring Harbor Lab" name="citation_publisher">
      <meta content="1088-9051" name="citation_issn">
      <meta content="1549-5469" name="citation_issn">
      <meta content="Metzker, Michael L." name="citation_authors">
      <meta content="Emerging technologies in DNA sequencing" name="citation_title">
      <meta content="12/01/2005" name="citation_date">
      <meta content="15" name="citation_volume">
      <meta content="12" name="citation_issue">
      <meta content="1767" name="citation_firstpage">
      <meta content="15/12/1767" name="citation_id">
      <meta content="genome;15/12/1767" name="citation_mjid">
      <meta content="10.1101/gr.3770505" name="citation_doi">
      <meta content="http://genome.cshlp.org/content/15/12/1767.abstract" name="citation_abstract_html_url">
      <meta content="http://genome.cshlp.org/content/15/12/1767.full" name="citation_fulltext_html_url">
      <meta content="http://genome.cshlp.org/content/15/12/1767.full.pdf+html" name="citation_pdf_url">
      <meta content="16339375" name="citation_pmid">
      <meta name="robots" content="noarchive,nofollow">
      <link rel="stylesheet" type="text/css" media="all" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-global.css">
      <link rel="stylesheet" type="text/css" media="all" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/main.css">
      <link rel="stylesheet" type="text/css" media="all" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-cshljnls-global.css">
      <link rel="stylesheet" type="text/css" media="all" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-journal-global.css">
      <link rel="stylesheet" type="text/css" media="print" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-print.css">
      <link rel="stylesheet" type="text/css" media="print" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-cshljnls-print.css">
      <link rel="stylesheet" type="text/css" media="all" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-page-content.css">
      <link rel="stylesheet" type="text/css" media="all" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-cshljnls-page-content.css">
      <link rel="stylesheet" type="text/css" media="all" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-journal-page-content.css">
      <link rel="stylesheet" type="text/css" media="all" href="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/custom-global.css"><script type="text/javascript">var callbackToken='61389958F1D374C';</script><script type="text/javascript" src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/jquery-min.js"></script><script type="text/javascript" src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-shared.js"></script><script type="text/javascript" src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-design1.js"></script><script type="text/javascript" src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/cshl-shared.js"></script><script type="text/javascript" src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/hw-content.js"></script><script type="text/javascript" src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/custom-global.js"></script></head><div methodName="log" FirebugVersion="1.3.0" style="display: none;" id="_firebugConsole"></div><body>
      <div class="hw-gen-page pagetype-content" id="pageid-content">
         <div id="header">
            <a href="http://www.cshlpress.com/"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/society_logo.gif" alt="Publisher Logo" id="main-logo" height="83" width="74"></a><a href="http://genome.cshlp.org/"><img title="Genome Research" alt="Genome Research" src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/journal_logo_standalone.gif" height="83" width="313"></a>
            
            <p id="skip-link">
               <a href="#content-block">Skip to main page content</a>
               
            </p>
            
            <ul class="button-list header-buttons">
               <li class="first"><a href="http://genome.cshlp.org/" title="HOME"><span>HOME</span></a></li>
               <li><a href="http://genome.cshlp.org/site/misc/about.xhtml" title="ABOUT"><span>ABOUT</span></a></li>
               <li><a href="http://genome.cshlp.org/content" title="ARCHIVE"><span>ARCHIVE</span></a></li>
               <li><a href="http://submit.genome.org/" title="SUBMIT"><span>SUBMIT</span></a></li>
               <li><a href="http://genome.cshlp.org/site/subscriptions/" title="SUBSCRIBE"><span>SUBSCRIBE</span></a></li>
               <li><a href="http://genome.cshlp.org/site/misc/adinfo.xhtml" title="ADVERTISE"><span>ADVERTISE</span></a></li>
               <li><a href="http://genome.cshlp.org/site/misc/ifora.xhtml" title="AUTHOR INFO"><span>AUTHOR INFO</span></a></li>
               <li><a href="http://genome.cshlp.org/cgi/feedback" title="CONTACT"><span>CONTACT</span></a></li>
               <li class="last"><a href="http://genome.cshlp.org/help" title="HELP"><span>HELP</span></a></li>
            </ul>
            
            
            <div class="header-qs">
               
               <form class="searchbox" action="/search" method="get">
                  <div>
                     	<label for="header-qs-input" id="header-qs-search-label">Search for Keyword:</label>
                     	<input style="color: rgb(160, 160, 160);" value="" title="Search Genome Research" name="fulltext" id="header-qs-input" type="text"><input name="submit" value="yes" type="hidden"><a href="#" id="hdr-qs-search-a"><label for="header-qs-search" id="header-qs-search-label">GO</label></a><input value="GO" alt="Link: Go" id="header-qs-search-go" src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/go.gif" type="image">
                     
                  </div>
                  <div class="adv-search-link"><a href="http://genome.cshlp.org/search">Advanced Search</a></div>
               </form>
               
               
            </div>
            
            <div class="header-ac-elements">
               
               <div class="inst-branding">
                  
               </div>
               
               <div id="authstring" class="suppress-header-login">
                  
                  <ul>
                     <li class="subscr-ref">Institution: UNIV OF MIAMI SCH OF MED</li>
                     <li><a href="http://genome.cshlp.org/login?uri=/content/15/12/1767.full">
                           Sign In via User Name/Password
                           </a></li>
                  </ul>           
                  
               </div>                                                                                                                         
               
               <div id="hdr-login" class="suppress-header-login"></div>
               
            </div>
            
            <div class="banner-ads">
               
               <ul>
                  <li><a href="http://genome.cshlp.org/cgi/adclick/?ad=19295&amp;adclick=true&amp;url=http%3A%2F%2Fwww.affymetrix.com%2Fgenechip%2Fwtexpression" target="_blank"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/Affymetrix%2520WT_468x60_1023.gif" alt="'Affymetrix(r) Exon and Gene Arrays'" border="0"></a></li>
               </ul>
               
            </div>
            
            <div class="bar">
               
               <div class="bar-inner"></div>
               
            </div>
            
         </div>
         <div style="height: auto;" id="content-block">
            <div class="article fulltext-view">
               <h1 id="article-title-1">Emerging technologies in DNA sequencing</h1>
               <div class="contributors">
                  <ol class="contributor-list" id="contrib-group-1">
                     <li id="contrib-1"><span class="name"><a class="name-search" href="http://genome.cshlp.org/search?author1=Michael+L.+Metzker&amp;sortspec=date&amp;submit=Submit">Michael L. Metzker</a></span></li>
                  </ol><p class="affiliation-list-reveal"><a href="#" class="view-more">+</a>Author Affiliations</p>
                  <ol class="affiliation-list hideaffil">
                     <li class="aff"><a id="aff-1" name="aff-1"></a><address><em>Human Genome Sequencing Center and Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas
                              77030, USA</em></address>
                     </li>
                  </ol>
               </div>
               <div class="section abstract" id="abstract-1">
                  <div class="section-nav">
                     <div class="nav-placeholder">&nbsp;</div><a href="#sec-1" title="Next Section" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Abstract</h2>
                  
                  <p id="p-1">Demand
for DNA sequence information has never been greater, yet current Sanger
technology is too costly, time consuming, and labor intensive to meet
this ongoing demand. Applications span numerous research interests,
including sequence variation studies, comparative genomics and
evolution, forensics, and diagnostic and applied therapeutics. Several
emerging technologies show promise of delivering next-generation
solutions for fast and affordable genome sequencing. In this review
article, the DNA polymerase-dependent strategies of Sanger sequencing,
single nucleotide addition, and cyclic reversible termination are
discussed to highlight recent advances and potential challenges these
technologies face in their development for ultrafast DNA sequencing. </p>
               </div>
               <div class="section" id="sec-1">
                  <div class="section-nav"><a href="#abstract-1" title="Abstract" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-2" title="Sanger sequencing: State-of-the-art technology" class="next-section-link"><span>Next Section</span></a></div>
                  
                  
                  <p id="p-2">More than just a mapping and sequencing endeavor, the Human Genome Project (HGP) has altered the mindset and approach to many
                     basic and applied research efforts. Early skepticism and controversy (<a class="xref-" href="#ref-43" id="xref-ref-43-1">Koshland 1989</a>; <a class="xref-" href="#ref-57" id="xref-ref-57-1">Luria et al. 1989</a>; <a class="xref-" href="#ref-82" id="xref-ref-82-1">Roberts 1989b</a>; <a class="xref-" href="#ref-26" id="xref-ref-26-1">Fox et al. 1990</a>) were soon laid to rest by well-developed strategies (<a class="xref-" href="#ref-81" id="xref-ref-81-1">Roberts 1989a</a>; <a class="xref-" href="#ref-14" id="xref-ref-14-1">Collins and Galas 1993</a>; <a class="xref-" href="#ref-16" id="xref-ref-16-1">Collins et al. 1998</a>)
that led to the successful execution of mankind's largest biology
project. At the core of the HGP was technology development that
advanced the pace of sequencing a mammalian-size genome from years to
months. Along the way, numerous strategies emerged that hold promise
for rapid, efficient, and inexpensive delivery of DNA sequence
information. For the HGP, a brute-force approach was adopted for
completing the job by coupling the core technologies of Sanger
sequencing and fluorescence detection. The completion of the sequencing
phase could not have been accomplished without major innovations in
recombinant protein engineering, fluorescent dye development, capillary
electrophoresis, automation, robotics, informatics, and process
management. The result was completion of a high-quality, reference
sequence of the human genome in April, 2003 (<a class="xref-" href="#ref-17" id="xref-ref-17-1">Collins et al. 2003</a>),
marking the 50-year anniversary of the discovery of the double-helix
structure. For many outside the genome community, that heroic milestone
signaled the end of this international scientific project, but for the
rest of us, it only marked the beginning of things to come. </p>
                  
                  <p id="p-3">The
need for sequencing has never been greater than it is today, with
applications spanning diverse research sectors including comparative
genomics and evolution, forensics, epidemiology, and applied medicine
for diagnostics and therapeutics. Arguably, the strongest rationale for
ongoing sequencing is the quest for identification and interpretation
of human sequence variation as it relates to health and disease. The
most common form of variation is the single nucleotide polymorphism
(SNP). Although two unrelated people share, on average, 99.9% sequence
identity (i.e., one difference in a thousand base pairs), the average
occurrence of an SNP in the general population is once every few
hundred base pairs. As such, more than nine million unique SNPs have
been cataloged in the public database, dbSNP (<a class="xref-" href="#ref-18" id="xref-ref-18-1">Crawford and Nickerson 2005</a>), with many more expected to be found in large-scale resequencing efforts.
                  </p>
                  
                  <p id="p-4">A
great deal of attention has been focused on common SNPs with a minor
allele frequency &gt;5% and their potential role in common disease (<a class="xref-" href="#ref-45" id="xref-ref-45-1">Lander 1996</a>; <a class="xref-" href="#ref-80" id="xref-ref-80-1">Risch and Merikangas 1996</a>; <a class="xref-" href="#ref-15" id="xref-ref-15-1">Collins et al. 1997</a>). Recent, large-scale genotyping efforts of these common SNPs have shown that much of the human genome can be parsed into
                     common haplotype blocks (<a class="xref-" href="#ref-20" id="xref-ref-20-1">Daly et al. 2001</a>; <a class="xref-" href="#ref-75" id="xref-ref-75-1">Patil et al. 2001</a>; <a class="xref-" href="#ref-27" id="xref-ref-27-1">Gabriel et al. 2002</a>). The International HapMap Consortium (<a class="xref-" href="#ref-37" id="xref-ref-37-1">2003</a>)
was formed to characterize common patterns of sequence variation by
determining allele frequencies and the degree of association between
SNPs among geographically distinct groups, leading to the
identification of “tagSNPs” for genome-wide, disease-based association
studies. With this method of characterization, however, rare
SNPs/haplotypes may be overlooked, as highlighted by Liu et al. (<a class="xref-" href="#ref-56" id="xref-ref-56-1">2005</a>), who described an association of rare variants/haplotypes with osteoporosis.
                  </p>
                  
                  <p id="p-5">A
shift in large-scale strategies from genotyping to resequencing is
currently taking place to explore the significance of less-common SNPs
to human biology and disease. The “re” in this approach is the
sequencing of additional genomes related to a reference genome for de
novo SNP discovery and comparative genomics application. The ENCODE
Project Consortium (<a class="xref-" href="#ref-21" id="xref-ref-21-1">2004</a>)
has described significant efforts toward resequencing megabase-sized
blocks of the human genome. Consequently, genome centers are now
diverting at least 10%-20% of their resources, which currently
translates to ∼5% capacity, to resequencing hundreds to thousands of
gene regions. This increase in momentum for high-throughput
resequencing will greatly facilitate studies to determine the genetic
basis of susceptibility to common disease, cancer biology, and disease
association in model and nonmodel organisms. </p>
                  
                  <p id="p-6">Current sequencing technologies are too expensive, labor intensive, and time consuming for broad application in human sequence
                     variation studies. Genome center cost is calculated on the basis of dollars per 1000 Q<sub>20</sub> bases (defined below) and can be generally divided into the categories of instrumentation, personnel, reagents and materials,
                     and overhead expenses. Currently, these centers are operating at less than one dollar per 1000 Q<sub>20</sub>
bases, with at least 50% of the cost resulting from DNA sequencing
instrumentation alone. Developments in novel detection methods,
miniaturization in instrumentation, microfluidic separation
technologies, and an increase in the number of assays per run will most
likely have the biggest impact on reducing cost. It should be
emphasized, however, that new sequencing strategies will be needed to
use these high-throughput platforms effectively. In September, 2004,
the National Human Genome Research Institute (NHGRI) initiated two new
programs aimed at bringing the cost of whole-genome sequencing down to
$100,000 (<a class="xref-" href="#ref-111" id="xref-ref-111-1">http://grants.nih.gov/grants/guide/rfa-files/RFA-HG-04-002.html</a>), with the eventual goal being $1000 (<a class="xref-" href="#ref-112" id="xref-ref-112-1">http://grants.nih.gov/grants/guide/rfa-files/RFA-HG-04-003.html</a>).
                  </p>
                  
                  <p id="p-7">Numerous
strategies and platforms for ultrafast DNA sequencing currently under
development include sequencing-by-hybridization (SBH), nanopore
sequencing, and sequencing-by-synthesis (SBS), the latter of which
encompasses many different DNA polymerase-dependent strategies. Use of
the term SBS has become increasingly ambiguous in the literature;
therefore, I propose a classification of DNA polymerase-dependent
strategies into three major categories: <em>Sanger sequencing, single nucleotide addition</em> (<em>SNA</em>), and <em>cyclic reversible termination</em> (<em>CRT</em>) (<a class="xref-" href="#boxed-text-1" id="xref-boxed-text-1-1">Text Box 1</a>).
In this review, I will focus only on DNA polymerase-dependent
strategies, which represent the broadest area of research and
development. For the SNA and CRT strategies, I will emphasize the
chemistry in an effort to illustrate the advantages and challenges of
these methods. Because of the competitive nature of technology
development, the exchange of scientific ideas is often thwarted, as
many companies do not readily publish results. Although this review
will highlight recent advances reported in the literature, readers are
directed to the Web sites of companies who are active in the sequencing
field (<a class="xref-" href="#T1" id="xref-table-wrap-1-1">Table 1</a>). A recent review by Shendure et al. (<a class="xref-" href="#ref-96" id="xref-ref-96-1">2004</a>)
provides a comprehensive overview of SBH and nanopore sequencing
technologies. Important issues surrounding whole-genome sequencing,
such as ownership, consent, privacy, and legal, ethical, and social
implications, will not be addressed here (<a class="xref-" href="#ref-25" id="xref-ref-25-1">Foster and Sharp 2002</a>; <a class="xref-" href="#ref-83" id="xref-ref-83-1">Robertson 2003</a>; <a class="xref-" href="#ref-6" id="xref-ref-6-1">Bonham et al. 2005</a>).
                  </p>
                  
                  <div class="table" id="T1">
                     <div class="table-inline">
                        <div class="callout"><span>View this table:</span><ul class="callout-links">
                              <li><a href="http://genome.cshlp.org/content/15/12/1767/T1.expansion.html">In this window</a></li>
                              <li><a target="_blank" class="in-nw-vis" href="http://genome.cshlp.org/content/15/12/1767/T1.expansion.html">In a new window</a></li>
                           </ul>
                        </div>
                     </div>
                     <div class="table-caption"><span class="table-label"><strong>Table 1.</strong></span> 
                        
                        <p id="p-9" class="first-child">Companies involved in DNA sequencing technology development</p><br class="caption-clear"></div>
                  </div>
               </div>
               <div class="section" id="sec-2">
                  <div class="section-nav"><a href="#sec-1" title="Previoius Section" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-3" title="Sanger sequencing: Recent advances" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Sanger sequencing: State-of-the-art technology</h2>
                  
                  <p id="p-10">The Sanger method is a mixed-mode process involving synthesis of a complementary DNA template using natural 2′-deoxynucleotides
                     (dNTPs) and termination of synthesis using 2′,3′-dideoxynucleotides (ddNTPs) by DNA polymerase (<a class="xref-" href="#ref-92" id="xref-ref-92-1">Sanger et al. 1977</a>).
Balanced appropriately, competition between synthesis and termination
processes results in the generation of a set of nested fragments, which
differ in nucleoside monophosphate units. The ratio of dNTP/ddNTP in
the sequencing reaction determines the frequency of chain termination,
and hence the distribution of lengths of terminated chains. The nested
fragments are then separated by their size using high-resolution gel
electrophoresis and analyzed to reveal the DNA sequence. Advancements
in fluorescence detection (<a class="xref-" href="#ref-100" id="xref-ref-100-1">Smith et al. 1986</a>; <a class="xref-" href="#ref-78" id="xref-ref-78-1">Prober et al. 1987</a>), enzymology (Tabor and Richardson <a class="xref-" href="#ref-102" id="xref-ref-102-1">1989</a>, <a class="xref-" href="#ref-103" id="xref-ref-103-1">1995</a>), fluorescent dyes (<a class="xref-" href="#ref-39" id="xref-ref-39-1">Ju et al. 1995</a>; <a class="xref-" href="#ref-64" id="xref-ref-64-1">Metzker et al. 1996</a>; <a class="xref-" href="#ref-49" id="xref-ref-49-1">Lee et al. 1997</a>), dynamic-coating polymers and their derivatives (<a class="xref-" href="#ref-88" id="xref-ref-88-1">Ruiz-Martinez et al. 1993</a>; <a class="xref-" href="#ref-11" id="xref-ref-11-1">Carrilho et al. 1996</a>; Madabhushi et al. <a class="xref-" href="#ref-59" id="xref-ref-59-1">1996</a>, <a class="xref-" href="#ref-60" id="xref-ref-60-1">1999</a>; <a class="xref-" href="#ref-58" id="xref-ref-58-1">Madabhushi 1998</a>; <a class="xref-" href="#ref-90" id="xref-ref-90-1">Salas-Solano et al. 1998</a>; Guttman <a class="xref-" href="#ref-29" id="xref-ref-29-1">2002a</a>, <a class="xref-" href="#ref-30" id="xref-ref-30-1">2002b</a>), and capillary array electrophoresis (CAE) (<a class="xref-" href="#ref-104" id="xref-ref-104-1">Takahashi et al. 1994</a>; <a class="xref-" href="#ref-42" id="xref-ref-42-1">Kheterpal et al. 1996</a>) have helped to define current DNA sequencing platforms.
                  </p>
                  
                  <p id="p-11">For
automated Sanger sequencing, either the primer or the terminating ddNTP
is tagged with a specific fluorescent dye (e.g., ddATP is labeled with
the green dye). As these dye-labeled fragments pass through the
detection region, fluorophores are excited by the laser in the DNA
sequencer, producing fluorescence emissions of four different colors.
The determination of the color is the underlying method for assigning a
base call, and the order of the fluorescent fragments reveals the DNA
sequence. The “raw” fluorescence signals, however, must be transformed.
Removal of cross-talk, correction for dye mobility alterations, and
normalization of emission intensities must be performed before readable
DNA sequence information can be obtained (<a class="xref-" href="#ref-101" id="xref-ref-101-1">Smith et al. 1987</a>). Base-calling and error probability assignment (<a class="xref-" href="#ref-23" id="xref-ref-23-1">Ewing and Green 1998</a>; <a class="xref-" href="#ref-24" id="xref-ref-24-1">Ewing et al. 1998</a>) applications are then used to call the DNA sequence and assess the accuracy of the call. A Phred<sub>20</sub> or Q<sub>20</sub> score, equivalent to an error probability of 1% for a given base call, is considered a high-quality base and serves as the
                     commodity standard throughout the sequencing community.
                  </p>
                  
                  <p id="p-12">
                     </p><div class="boxed-text" id="boxed-text-1">
                        
                        <p id="p-13"><strong>Text Box 1. DNA polymerase-dependent strategies</strong></p>
                        
                        <p id="p-14">In
the broadest sense, all methods involving a DNA polymerase could be
considered a SBS approach, if synthesis alone was the defining process.
The defining element of these DNA polymerase-dependent methods,
however, is not really synthesis at all but rather the means by which
DNA synthesis terminates. From this point of view, the DNA sequencing
approaches highlighted here have been organized according to their
termination strategies. <em>Sanger sequencing</em> and “dideoxy” sequencing are frequently used as synonymous terms.
                        </p>
                        
                        <p id="p-15">These
unnatural ddNTP terminators replace the OH with an H at the 3′-position
of the deoxyribose molecule and irreversibly terminate DNA polymerase
activity, unless the nucleotide is removed by the process of
phosphorolysis. This process is mediated by high concentrations of
pyrophosphate or ATP and is a major cause of “drop-outs” in DNA
sequence data. </p>
                        
                        <p id="p-16"><em>Single nucleotide addition</em>
(SNA) methods such as pyrosequencing use limiting amounts of individual
natural dNTPs to cause DNA synthesis to pause, which, unlike the Sanger
method, can be resumed with the addition of natural nucleotides.
Limiting the amount of a given dNTP is required to minimize
misincorporation effects observed at higher concentrations. A major
drawback with the SNA approach is the incomplete extension through
homopolymer repeats. </p>
                        
                        <p id="p-17"><em>Cyclic reversible termination</em>
(CRT) uses reversible terminators containing a protecting group
attached to the nucleotide that terminates DNA synthesis. For the
reversible terminator, removal of the protecting group restores the
natural nucleotide substrate, allowing subsequent addition of
reversible terminating nucleotides. One example of a reversible
terminator is a 3′-<em>O</em>-protected nucleotide (<a class="xref-" href="#F4" id="xref-fig-4-1">Fig. 4B</a>),
although protecting groups can be attached to other sites on the
nucleotide as well. This step-wise base addition approach, which cycles
between coupling and deprotection, mimics many of the steps of
automated DNA synthesis of oligonucleotides. </p>
                     </div>
                  
                  
                  <p id="p-18">High-throughput DNA sequencing is conducted primarily at large genome centers that continue to refine the sequencing process
                     and strive for Q<sub>20</sub>
bases at lower cost. For example, the Baylor College of Medicine Human
Genome Sequencing Center (BCM-HGSC) produces approximately four million
sequencing reactions per month (R.A. Gibbs, pers. comm.). The current
production efficiency or pass rate is approximately 89% (after removal
of failed reactions, vector sequences, etc.), with sequencing reads
averaging 805 Q<sub>20</sub>
bases in length. These metrics translate into the equivalent of
sequencing one mammalian-size genome per month. Redundancy is required
to improve the base-calling accuracy and contiguity of assembled
genomes, resulting in the generation of six times the genome size in Q<sub>20</sub> bases for
production of a draft-quality sequence. Thus, delivery of a
mammalian-size, draft-quality sequence requires approximately six
months and $12 million. Ongoing advances in new technologies will be
critical to meet the goal of rapid, genome-scale sequencing for the
price of $100,000 and, ultimately, $1000 per genome. </p>
               </div>
               <div class="section" id="sec-3">
                  <div class="section-nav"><a href="#sec-2" title="Sanger sequencing: State-of-the-art technology" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-5" title="Fluorescence detection" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Sanger sequencing: Recent advances</h2>
                  <h3>Microfluidic separation platforms</h3>
                  
                  <p id="p-19">Technology
development remains active for the fluorescence-based Sanger approach
with emphasis on producing faster and cheaper sequencing reads. One key
area of research is the application of microfluidic separation devices
to DNA sequencing. These microfluidic devices can be fabricated using a
variety of substrate materials, with several molecular biology
processes integrated onto a single device (e.g., lab-on-a-chip). A
number of reviews have been devoted to microfluidic devices (<a class="xref-" href="#ref-5" id="xref-ref-5-1">Becker and Gartner 2000</a>; <a class="xref-" href="#ref-10" id="xref-ref-10-1">Carrilho 2000</a>; <a class="xref-" href="#ref-62" id="xref-ref-62-1">McDonald et al. 2000</a>; <a class="xref-" href="#ref-79" id="xref-ref-79-1">Quake and Scherer 2000</a>; <a class="xref-" href="#ref-7" id="xref-ref-7-1">Boone et al. 2002</a>; <a class="xref-" href="#ref-72" id="xref-ref-72-1">Paegel et al. 2003</a>; <a class="xref-" href="#ref-40" id="xref-ref-40-1">Kan et al. 2004</a>), recent advances of which I will highlight as they relate to DNA sequencing. These miniature devices have several advantages
                     over CAE, including improved sample injection and faster separation times.
                  </p>
                  
                  <p id="p-20">The
separation principles of microfabricated devices are similar to those
of conventional CAE, however, their injection methods are very
different. With CAE, the sample is introduced by electrokinetic
injection into the capillary. The injection time, which defines the
length of the sample plug, is typically short and allows only a minute
fraction of the sample to be analyzed. A further drawback is that data
quality is compromised with increasing impurities in the sample and an
intrinsic bias in favor of shorter DNA fragments over longer ones.
Microfluidic devices, on the other hand, are less susceptible to these
injection problems because the sample is introduced via a channel
network by a variety of process strategies (<a class="xref-" href="#ref-108" id="xref-ref-108-1">Zhang and Manz 2001</a>). Although early microfabricated chips employed a “T”-injector design (<a class="xref-" href="#ref-31" id="xref-ref-31-1">Harrison et al. 1992</a>), the cross-T design (<a class="xref-" href="#ref-32" id="xref-ref-32-1">Harrison et al. 1993</a>) is widely used today because of its superior sample control (<a class="xref-" href="#F1" id="xref-fig-1-1">Fig. 1A</a>). The narrow width of the injector affords greater control in selection of sample plug size, which contributes to higher
                     resolution separations with shorter separation lengths compared with CAE.
                  </p>
                  
                  <p id="p-21">Most
microfabricated devices use borofloat glass or fused silica substrates,
which have the advantages of (1) high-quality optical properties, (2)
good thermal conductivity, (3) well-documented surface chemistry, and
(4) effective translation of capillary innovations. Woolley and Mathies
(<a class="xref-" href="#ref-107" id="xref-ref-107-1">1995</a>)
demonstrated the first application of DNA sequencing using a
microfabricated glass device in 1995, reporting single-base resolution
using their four-color scanner technology. Data quality and
read-lengths have improved significantly since then, because of an
increase in the effective separation lengths with run times of 30
minutes or less (<a class="xref-" href="#T2" id="xref-table-wrap-2-1">Table 2</a>) (<a class="xref-" href="#ref-107" id="xref-ref-107-2">Woolley and Mathies 1995</a>; <a class="xref-" href="#ref-54" id="xref-ref-54-1">Liu et al. 1999</a>; <a class="xref-" href="#ref-93" id="xref-ref-93-1">Schmalzing et al. 1999</a>; <a class="xref-" href="#ref-3" id="xref-ref-3-1">Backhouse et al. 2000</a>; <a class="xref-" href="#ref-44" id="xref-ref-44-1">Koutny et al. 2000</a>; <a class="xref-" href="#ref-55" id="xref-ref-55-1">Liu et al. 2000</a>; <a class="xref-" href="#ref-91" id="xref-ref-91-1">Salas-Solano et al. 2000</a>; <a class="xref-" href="#ref-98" id="xref-ref-98-1">Simpson et al. 2000</a>; <a class="xref-" href="#ref-7" id="xref-ref-7-2">Boone et al. 2002</a>; <a class="xref-" href="#ref-71" id="xref-ref-71-1">Paegel et al. 2002</a>; <a class="xref-" href="#ref-97" id="xref-ref-97-1">Shi and Anderson 2003</a>). For example, Liu et al. (<a class="xref-" href="#ref-54" id="xref-ref-54-2">1999</a>) reported 99.4% accuracy over 500 bases in 20 minutes, with an increase in separation length from 3.5 cm to 6.5 cm. More
                     recent developments by Boone et al. (<a class="xref-" href="#ref-7" id="xref-ref-7-3">2002</a>) and Shi and Anderson (<a class="xref-" href="#ref-97" id="xref-ref-97-2">2003</a>) have shown the first DNA sequencing applications on plastic chips (<a class="xref-" href="#T2" id="xref-table-wrap-2-2">Table 2</a>).
These chips can be fabricated with high geometric aspect ratios (i.e.,
deep and narrow channels) at significantly lower cost. Deep and narrow
channel structures have the advantages of improved electrophoretic
resolution (i.e., longer read-length) and better detection sensitivity.
</p>
                  
                  <div class="table" id="T2">
                     <div class="table-inline">
                        <div class="callout"><span>View this table:</span><ul class="callout-links">
                              <li><a href="http://genome.cshlp.org/content/15/12/1767/T2.expansion.html">In this window</a></li>
                              <li><a target="_blank" class="in-nw-vis" href="http://genome.cshlp.org/content/15/12/1767/T2.expansion.html">In a new window</a></li>
                           </ul>
                        </div>
                     </div>
                     <div class="table-caption"><span class="table-label"><strong>Table 2.</strong></span> 
                        
                        <p id="p-23" class="first-child">Summary of microfabricated devices for DNA sequencing applications</p><br class="caption-clear"></div>
                  </div>
                  
                  <div id="F1" class="fig pos-float">
                     <div class="fig-inline"><a href="http://genome.cshlp.org/content/15/12/1767/F1.expansion.html"><img alt="Figure 1." src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/F1.gif"></a><div class="callout"><span>View larger version:</span><ul class="callout-links">
                              <li><a href="http://genome.cshlp.org/content/15/12/1767/F1.expansion.html">In this page</a></li>
                              <li><a target="_blank" class="in-nw-vis" href="http://genome.cshlp.org/content/15/12/1767/F1.expansion.html">In a new window</a></li>
                           </ul>
                           <ul class="fig-services">
                              <li class="ppt-link"><a href="http://genome.cshlp.org/powerpoint/15/12/1767/F1">Download as PowerPoint Slide</a></li>
                           </ul>
                        </div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 1.</strong></span> 
                        
                        <p id="p-26" class="first-child">Microfabricated technologies. (<em>A</em>) Examples of a T-injector and cross-T injector layout. (<em>B</em>) Expanded view of the sample injector and pinched turn. (<em>C</em>) Schematic of the 96 channels in a radial chip design. (<em>B,C</em>) Reprinted with permission from National Academy of Sciences, U.S.A. © 2002, <a class="xref-" href="#ref-71" id="xref-ref-71-3">Paegel et al. 2002</a>.
                        </p><br class="caption-clear"></div>
                  </div>
                  
                  <p id="p-27">While
single-channel devices are useful for demonstrating feasibility, the
construction of multiple channel arrays is essential for
high-throughput DNA sequencing. A summary of DNA sequence metrics from
several microfabricated multiple channel array devices is presented in <a class="xref-" href="#T2" id="xref-table-wrap-2-3">Table 2</a>. While Backhouse et al. (<a class="xref-" href="#ref-3" id="xref-ref-3-3">2000</a>) and Koutny et al. (<a class="xref-" href="#ref-44" id="xref-ref-44-3">2000</a>)
reported improved read-lengths by increasing the effective separation
lengths to 46.5 cm and 40 cm, respectively, these microfabricated
channels were constructed on glass plates ≥50 cm in length, which is
out of line with current efforts to miniaturize devices. One approach
to circumvent this dilemma has been the introduction of turns along the
length of the separation channel. Early studies, however, reported
lower separation efficiency in channel turns due to band broadening (<a class="xref-" href="#ref-38" id="xref-ref-38-1">Jacobson et al. 1994</a>) and differential field strength effects (<a class="xref-" href="#ref-19" id="xref-ref-19-1">Culbertson et al. 1998</a>). Paegel et al. (<a class="xref-" href="#ref-70" id="xref-ref-70-1">2000</a>) introduced a “pinched-turn” design (<a class="xref-" href="#F1" id="xref-fig-1-2">Fig. 1B</a>) with an effective separation length of 15.9 cm on a 15-cm-diameter silica disc, which has been multiplexed into a 96-channel
                     radial device (<a class="xref-" href="#F1" id="xref-fig-1-3">Fig. 1C</a>) showing tremendous potential for increasing throughput in DNA sequencing applications (<a class="xref-" href="#ref-71" id="xref-ref-71-4">Paegel et al. 2002</a>). Most of the data shown in <a class="xref-" href="#T2" id="xref-table-wrap-2-4">Table 2</a>,
however, were derived using the standard M13mp18 vector as the
sequencing template, and similar performance is not typically observed
under the same conditions with “real-world” samples such as those from
genome center production lines. </p>
               </div>
               <div class="section" id="sec-5">
                  <div class="section-nav"><a href="#sec-3" title="Sanger sequencing: Recent advances" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-6" title="SNA methodologies" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Fluorescence detection</h2>
                  
                  <p id="p-28">The most widely used detection method for four-color DNA sequencing was initially described almost 20 years ago (<a class="xref-" href="#ref-100" id="xref-ref-100-2">Smith et al. 1986</a>; <a class="xref-" href="#ref-78" id="xref-ref-78-2">Prober et al. 1987</a>).
This method is based on resolution of the emission signal from a
dye-labeled nucleotide into color, with subsequent assignment in the
DNA sequence. While successful for the sequencing of numerous higher
and lower eukaryotic and prokaryotic genomes, these four-color systems
have several disadvantages, including inefficient excitation of the
fluorescent dyes, significant spectral overlap, and inefficient
collection of the emission signals. The issue of inefficient excitation
has been partially addressed by the use of fluorescence resonance
energy-transfer (FRET) dyes (<a class="xref-" href="#ref-39" id="xref-ref-39-2">Ju et al. 1995</a>; <a class="xref-" href="#ref-64" id="xref-ref-64-2">Metzker et al. 1996</a>; <a class="xref-" href="#ref-49" id="xref-ref-49-2">Lee et al. 1997</a>).
At present, FRET dye-labeled ddNTP terminators are widely used
throughout the sequencing community. The resulting improvements in
acceptor dye signal intensities, however, are suboptimal compared with
those of single dyes excited at their absorption maxima by the
appropriate laser source. </p>
                  
                  <p id="p-29">To overcome these deficiencies, some investigators have proposed strategies using additional properties such as fluorescence
                     life-time (<a class="xref-" href="#ref-68" id="xref-ref-68-1">Nunnally et al. 1997</a>; <a class="xref-" href="#ref-53" id="xref-ref-53-1">Lieberwirth et al. 1998</a>; <a class="xref-" href="#ref-47" id="xref-ref-47-1">Lassiter et al. 2000</a>; Zhu et al. <a class="xref-" href="#ref-109" id="xref-ref-109-1">2003</a>, <a class="xref-" href="#ref-110" id="xref-ref-110-1">2004</a>) and radio frequency (RF) modulation (<a class="xref-" href="#ref-1" id="xref-ref-1-1">Alaverdian et al. 2002</a>).
For DNA sequencing applications, fluorescence life-time measurements
have been described using pulsed lasers with high repetition rates
(picosecond time-scale) with detection in the photon-counting mode.
Soper and colleagues have recently demonstrated a combined approach of
emission wavelength and fluorescence life-time measurements, with the
potential to increase the number of fluorescent components in DNA
sequencing assays (Zhu et al. <a class="xref-" href="#ref-109" id="xref-ref-109-2">2003</a>, <a class="xref-" href="#ref-110" id="xref-ref-110-2">2004</a>). Alaverdian et al. (<a class="xref-" href="#ref-1" id="xref-ref-1-2">2002</a>)
proposed using four continuous wave (CW) mode lasers, which are
modulated at different RFs. To estimate the fluorescence signal for
each dye, however, the resulting emission intensity pattern must be
demodulated, which introduces a significant computational load for each
capillary signal channel. Coupled with repetition rates on the order of
≥100 Hz, the RF method does not appear to be compatible with
conventional CCD technology, limiting its scalability for detection of
high-density capillary arrays. </p>
                  
                  <p id="p-30">Recently, Lewis et al. (<a class="xref-" href="#ref-51" id="xref-ref-51-1">2005</a>)
described a simple but effective method for multifluorescence
discrimination called pulsed multiline excitation (PME). The underlying
principle of this four-laser system is the correlation of sequential
laser pulses with detector response (<a class="xref-" href="#F2" id="xref-fig-2-1">Fig. 2A</a>).
Advantages of PME are such that (1) absorption maxima for the four
fluorescent dyes are matched to the excitation sources yielding maximum
signal intensities, (2) temporal separation of the laser pulses and
expansion of the dye set across the visible spectrum eliminate
cross-talk between the dyes, and (3) collection of emission signals is
improved by eliminating the requirement for dispersing elements (prisms
or gratings) in color separation. In other words, PME measures
multicomponent fluorescence assays in a color-blind manner. To
demonstrate these advantages, Lewis et al. (<a class="xref-" href="#ref-51" id="xref-ref-51-2">2005</a>) applied the PME technology to capillary electrophoresis for DNA sequencing. <a class="xref-" href="#F2" id="xref-fig-2-2">Figure 2B</a> shows the unprocessed signals from the four PME laser waveforms for a portion of the PCR amplicon for the <em>TCF1</em> (formerly known as <em>HNF1A</em>) exon 10. Transformation of the data into unambiguous sequence data (<a class="xref-" href="#F2" id="xref-fig-2-3">Fig. 2C</a>)
is accomplished by applying only dye mobility correction software,
eliminating the need for cross-talk and signal normalization software
transformation. The PME technology holds promise for real-time field
applications for DNA sequencing. </p>
               </div>
               <div class="section" id="sec-6">
                  <div class="section-nav"><a href="#sec-5" title="Fluorescence detection" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-8" title="Other single addition dNTP strategies" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>SNA methodologies</h2>
                  <h3>Pyrosequencing</h3>
                  
                  <p id="p-31">Arguably the most successful non-Sanger method developed to date is pyrosequencing, first described in the literature by Hyman
                     (<a class="xref-" href="#ref-36" id="xref-ref-36-1">1988</a>). Pyrosequencing is a nonfluorescence technique that measures the release of inorganic pyrophosphate, which is proportionally
                     converted into visible light by a series of enzymatic reactions (Ronaghi et al. <a class="xref-" href="#ref-86" id="xref-ref-86-1">1996</a>, <a class="xref-" href="#ref-87" id="xref-ref-87-1">1998</a>).
Unlike other sequencing approaches that use 3′-modified dNTPs to
terminate DNA synthesis, the pyrosequencing assay manipulates DNA
polymerase by single addition of dNTPs in limiting amounts. Upon
addition of the complementary dNTP, DNA polymerase extends the primer
and pauses when it encounters a noncomplementary base. DNA synthesis is
reinitiated following the addition of the next complementary dNTP in
the dispensing cycle. The light generated by the enzymatic cascade is
recorded as a series of peaks called a pyrogram, which corresponds to
the order of complementary dNTPs incorporated and reveals the
underlying DNA sequence. Applications for pyrosequencing have been
reviewed by Ronaghi (<a class="xref-" href="#ref-85" id="xref-ref-85-1">2001</a>) and Langaee and Ronaghi (<a class="xref-" href="#ref-46" id="xref-ref-46-1">2005</a>).
                  </p>
                  
                  <div id="F2" class="fig pos-float">
                     <div class="fig-inline"><a href="http://genome.cshlp.org/content/15/12/1767/F2.expansion.html"><img alt="Figure 2." src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/F2.gif"></a><div class="callout"><span>View larger version:</span><ul class="callout-links">
                              <li><a href="http://genome.cshlp.org/content/15/12/1767/F2.expansion.html">In this page</a></li>
                              <li><a target="_blank" class="in-nw-vis" href="http://genome.cshlp.org/content/15/12/1767/F2.expansion.html">In a new window</a></li>
                           </ul>
                           <ul class="fig-services">
                              <li class="ppt-link"><a href="http://genome.cshlp.org/powerpoint/15/12/1767/F2">Download as PowerPoint Slide</a></li>
                           </ul>
                        </div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 2.</strong></span> 
                        
                        <p id="p-33" class="first-child">(<em>A</em>)
Illustration of the PME technology. Here, each laser operates in a CW
mode with mechanical shutters pulsing the different excitation beams in
sequential order. The single coaxial PME beam interrogates the
fluorescently labeled DNA fragments, which are separated by capillary
gel electrophoresis. Scattered laser light is rejected via specific
long-pass or wavelength notch filters, with pulsed emission signals
from the dye-labeled DNA fragments being detected by the
photomultiplier tube (PMT) without use of any dispersing elements. (<em>B</em>) Unprocessed fluorescence data obtained during the electrophoretic run for the <em>TCF1</em>
exon 10 gene region using PME dye-primers. Blue, green, black, and red
traces are AF-405, BODIPY-FL, 6-ROX, and Cy5.5 dye-primers terminated
with ddCTP, ddATP, ddGTP, and ddTTP respectively. (<em>C</em>) Transformation of the raw trace data derived from the experiment described in <em>B</em>
into readable, DNA sequence data using mobility software correction.
Reprinted with permission from National Academy of Sciences, U.S.A. ©
2005, <a class="xref-" href="#ref-51" id="xref-ref-51-3">Lewis et al. 2005</a>.
                        </p><br class="caption-clear"></div>
                  </div>
                  
                  <p id="p-34">Although
elegant in design, the pyrosequencing approach has several limitations.
For example, sequence reads are typically fewer than 100 bases in
length, which has application in sequence tag identification such as
serial analysis of gene expression (SAGE) (<a class="xref-" href="#ref-105" id="xref-ref-105-1">Velculescu et al. 1995</a>), mini-sequencing for known SNPs, and mapping related genomes to a reference sequence, but limited application for whole-genome
                     sequencing. Recent reports describe the use of single-stranded binding protein (<a class="xref-" href="#ref-84" id="xref-ref-84-1">Ronaghi 2000</a>) and the isomeric Sp form of the dATPαS nucleotide (<a class="xref-" href="#ref-28" id="xref-ref-28-1">Gharizadeha et al. 2002</a>),
which may improve read-lengths up to 100 bases in routine settings.
Secondly, homopolymer repeats greater than five nucleotides cannot be
quantitatively measured. This is attributed to incomplete extension by
DNA polymerase, which results from limiting the dNTP concentration to
minimize nucleotide misincorporation effects. It has been suggested
that re-addition of the same dNTP may be performed to ensure complete
polymerization (<a class="xref-" href="#ref-85" id="xref-ref-85-2">Ronaghi 2001</a>),
although its practicality for high-throughput sequencing is unclear.
Finally, the dispensing order of dNTPs determines the pyrogram profile,
which must be carefully designed to avoid asynchronistic extensions of
heterozygous sequences. </p>
                  
                  <p id="p-35">For a given dispensing order, approximately one half of all heterozygous sequences will result in asynchronistic extensions
                     past the variable site. A survey of heterozygous variants detected by direct DNA sequencing of the <em>TCF1</em>
gene revealed that 16 of 37 SNPs would result in nonsynchronistic
extension after the heterozygous base (data not shown). If one allele
extends past the heterozygous base position before the other and
advances to the next nucleotide cycle, the nonsynchronicity becomes
permanent. An illustration of the effect of dispensing order on
asynchronistic extension is shown in <a class="xref-" href="#F3" id="xref-fig-3-1">Figure 3A</a>. This observation is further highlighted by Entz et al. (<a class="xref-" href="#ref-22" id="xref-ref-22-1">2005</a>) with the identification of more than 40 unique dispensing orders for the accurate typing of <em>HLA-DQB1</em> and <em>HLA-DRB1</em>
alleles. Pyrosequencing may, therefore, be suited for pattern matching
of known SNP profiles, while its application for de novo SNP discovery
is less certain. Not surprisingly, base-calling for de novo SNPs is
problematic and still performed manually (<a class="xref-" href="#ref-46" id="xref-ref-46-2">Langaee and Ronaghi 2005</a>).
                  </p>
                  
                  <p id="p-36">The
454 Corporation has recently introduced a whole-genome sequencing
strategy by integrating pyrosequencing with their PicoTiterPlate (PTP)
platform, which has been shown to amplify and image approximately
300,000 PCR templates captured on Sepharose beads (<a class="xref-" href="#ref-48" id="xref-ref-48-1">Leamon et al. 2003</a>).
The PTP is manufactured by anisotropic etching of a fiber optic
faceplate with a well diameter of approximately 40 μm. The 454 group
has developed a solution-based emulsion strategy to create
microreactors for clonal amplification of single DNA molecules and
attachment to these beads. One advantage of the clonal amplification
strategy is that it addresses the dependence issue of dispensing order
for sequencing of heterozygous bases discussed above. Following an
enrichment step, DNA positive beads are loaded into individual PTP
wells, which contain additional beads coupled with the necessary
enzymes to perform the pyrosequencing chemistry (<a class="xref-" href="#ref-61" id="xref-ref-61-1">Margulies et al. 2005</a>). Recently, the company announced its first complete genome sequencing of a recombinant adenoviral construct and the shotgun
                     sequencing of the <em>Mycoplasma genitalium</em> genome.
                  </p>
                  
                  <p id="p-37">The
assembly of non-Sanger sequencing data will represent new challenges
because the input read will differ in length, quantity, and quality.
The complexity of the genome under analysis may also prove more
difficult for assemblies compared with Sanger data, even when the
offset is higher coverage of shorter reads. Chaisson et al. (<a class="xref-" href="#ref-12" id="xref-ref-12-1">2004</a>) recently performed a simulated assembly study (short, error-free reads sampled at 30× coverage) using genome sequences from
                     adenovirus, two mouse BACs, and two bacteria: <em>Campylobacter jejuni</em>, which contains very few repeat sequences (<a class="xref-" href="#ref-74" id="xref-ref-74-1">Parkhill et al. 2000b</a>), and <em>Neisseria meningitidis</em>, which contains several hundred repetitive elements (<a class="xref-" href="#ref-73" id="xref-ref-73-1">Parkhill et al. 2000a</a>). Compared with Sanger data, Chaisson et al. (<a class="xref-" href="#ref-12" id="xref-ref-12-2">2004</a>)
found that the read-length was inversely proportional to the number of
contigs in the assembly (i.e., longer reads gave fewer contigs).
Increasing genome complexity, on the other hand, directly increases the
number of contigs. Here, they found that 95% of the genome was
contained within 9-10 contigs for the BAC clones, and the number of
contigs increased from 21 to 344 for <em>C. jejuni</em> and <em>N. meningitides</em>
genome sequences, respectively. Observed errors for real sequence data
will undoubtedly decrease assembly performance for short reads. Thus,
the success of the non-Sanger strategies for whole-genome sequencing
applications will be highly dependent on the degree of its complexity,
which appears to traverse all three phylogenetic domains. </p>
                  
                  <div id="F3" class="fig pos-float">
                     <div class="fig-inline"><a href="http://genome.cshlp.org/content/15/12/1767/F3.expansion.html"><img alt="Figure 3." src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/F3.gif"></a><div class="callout"><span>View larger version:</span><ul class="callout-links">
                              <li><a href="http://genome.cshlp.org/content/15/12/1767/F3.expansion.html">In this page</a></li>
                              <li><a target="_blank" class="in-nw-vis" href="http://genome.cshlp.org/content/15/12/1767/F3.expansion.html">In a new window</a></li>
                           </ul>
                           <ul class="fig-services">
                              <li class="ppt-link"><a href="http://genome.cshlp.org/powerpoint/15/12/1767/F3">Download as PowerPoint Slide</a></li>
                           </ul>
                        </div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 3.</strong></span> 
                        
                        <p id="p-39" class="first-child">SNA technologies. (<em>A</em>) Simulated effects of two different dNTP dispensing orders on the outcome of the pyrogram profile. (<em>B</em>) The photocleavage reaction of a fluorescently labeled dNTP coupled with a photocleavable linker.
                        </p><br class="caption-clear"></div>
                  </div>
               </div>
               <div class="section" id="sec-8">
                  <div class="section-nav"><a href="#sec-6" title="SNA methodologies" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-9" title="CRT" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Other single addition dNTP strategies</h2>
                  
                  <p id="p-40">Methods other than pyrophosphate detection can be used to monitor single dNTP additions. For example, Braslavsky et al. (<a class="xref-" href="#ref-8" id="xref-ref-8-1">2003</a>)
used the technique of single-pair FRET (spFRET) to determine the order
of nonconsecutive nucleotide additions. With this single molecule
approach, Cy3-labeled-UTP was initially incorporated into the primer
strand, serving as the donor dye. Subsequent incorporation of a
complementary Cy5-labeled-UTP or Cy5-labeled-dCTP substrate resulted in
the spFRET signal. Following photobleaching of the Cy5 dye, the natural
nucleotides dATP and dGTP were added to increase the nucleotide
distance between subsequent Cy5-labeled dNTP additions, which would
otherwise have resulted in a significant reduction in incorporation
efficiencies due to steric hindrance effects. For the DNA template
sequence, written 3′-<strong><span class="underline">A</span></strong>TC<strong><span class="underline">G</span></strong>TC<strong><span class="underline">A</span></strong>TC<strong><span class="underline">G</span></strong>-5′ for convenience, the read-out would be the fingerprint sequence of 5′-UCUC. Levene et al. (<a class="xref-" href="#ref-50" id="xref-ref-50-1">2003</a>)
have recently described a zero-mode waveguide approach to
single-molecule detection of R110-labeled-dCTP and
coumarin-labeled-dCTP incorporation events by DNA polymerase. </p>
                  
                  <p id="p-41">Taking advantage of the steric effects observed in consecutively incorporated dye-labeled dNTPs, Mitra et al. (<a class="xref-" href="#ref-67" id="xref-ref-67-1">2003</a>) introduced fluorescently labeled dNTPs, which contained cleavable linkers, to remove the bulky fluorescent group following
                     incorporation by DNA polymerase. This method, called <em>fluorescent in situ sequencing</em>
(FISSEQ), used linkers containing either a disulfide bridge, which is
efficiently cleaved with a reducing agent, or a photocleavable group (<a class="xref-" href="#F3" id="xref-fig-3-2">Fig. 3B</a>). Using the polony technology (<a class="xref-" href="#ref-66" id="xref-ref-66-1">Mitra and Church 1999</a>),
Church and colleagues elegantly demonstrated the addition of single
Cy5-SS-dNTPs followed by dye cleavage for accurate DNA sequencing of
several templates. The presence of a fluorescence signal corresponding
to the dispersing order of the Cy5-SS-dNTPs revealed the DNA sequence.
Although read-lengths up to eight bases were demonstrated, several
miscalls were reported. One such call resulted from nucleotide
read-through. That is, consecutive incorporations of dye-labeled dNTPs
can occur (e.g., the sequence 5′-CAGCC was read as 5′-CAGC), presumably
with different efficiencies that are dependent on the local DNA
sequence context. A second error occurred as a result of a single
nucleotide insertion (e.g., the sequence 5′-ATGT was read as 5′-AGTGT).
Although more difficult to interpret, it is possible that the residual
linker structure, remaining on the nucleobases following dye cleavage,
could alter nucleotide specificity and incorporation efficiency of
subsequent incoming dNTPs in a sequence-dependent manner. More
recently, Seo et al. (<a class="xref-" href="#ref-94" id="xref-ref-94-1">2004</a>, <a class="xref-" href="#ref-95" id="xref-ref-95-1">2005</a>) described a similar strategy using four different dye-labeled dNTPs with photocleavable linkers (<a class="xref-" href="#F3" id="xref-fig-3-3">Fig. 3B</a>) and reported read-lengths of 12 bases. A key advantage of the four-color approach is that all four dNTPs can be assayed
                     simultaneously, although both reports demonstrated use of the single dNTP addition method.
                  </p>
                  
                  <p id="p-42">Kartalov and Quake (<a class="xref-" href="#ref-41" id="xref-ref-41-1">2004</a>)
proposed a different approach to overcome the steric effects of
consecutive dye-labeled bases by use of single-addition,
same-nucleobase mixtures (e.g., dCTP/TAMRA-labeled ddCTP) as a method
for DNA sequencing. The nucleobase mixture strategy serves the dual
purpose of dye-labeling for fluorescence detection (reporter phase) and
ongoing DNA synthesis of the complementary nucleotide (extension
phase). The dNTP and dye-labeled ddNTP concentrations are balanced
appropriately so that only a fraction of the primer strands incorporate
the dye-labeled ddNTP. The presence of a fluorescence signal reveals
the complementary nucleotide in the DNA sequence, but reporters are
eliminated from subsequent dNTP additions. With each nucleotide
addition, signal loss is inversely proportional to the increased
accumulation of termination products. The fluorescence is then quenched
by photobleaching before the next nucleobase mixture is dispensed to
repeat the process. Configured in a microfluidic device, the average
read-length for the mixed nucleobase addition scheme was three bases,
which can be partially attributed to signal loss with subsequent base
additions. The accuracy of the method is highly dependent on the
reporter phase mimicking the extension phase. For example, a simple
homopolymer repeat of two bases will be under-called in the DNA
sequence, as the reporter phase will reflect a single base addition
while the extension phase will incorporate two bases. </p>
               </div>
               <div class="section" id="sec-9">
                  <div class="section-nav"><a href="#sec-8" title="Other single addition dNTP strategies" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-10" title="Conclusions" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>CRT</h2>
                  
                  <p id="p-43">While
CRT technology represents tremendous potential for whole-genome
sequencing, this strategy still faces significant challenges in its
implementation. The CRT cycle is comprised of three steps: <em>incorporation, imaging</em>, and <em>deprotection</em>, as illustrated in <a class="xref-" href="#F4" id="xref-fig-4-2">Figure 4A</a>.
The advantages of CRT over Sanger are (1) elimination of gel
electrophoresis and (2) formatting of the CRT assay in a highly
parallel fashion. Its advantages over pyrosequencing are that (1) all
four bases are present during the incorporation phase, (2) step-wise
control allows for single-base additions through homopolymer repeats,
and (3) synchronistic extensions are maintained past heterozygous
bases. An additional advantage is that unlike the pyrosequencing assay,
which must be contained within a defined reaction well, the CRT assay
can be performed on a number of highly parallel platforms, such as
high-density oligonucleotide arrays (<a class="xref-" href="#ref-76" id="xref-ref-76-1">Pease et al. 1994</a>; <a class="xref-" href="#ref-2" id="xref-ref-2-1">Albert et al. 2003</a>), PTP arrays, (<a class="xref-" href="#ref-48" id="xref-ref-48-2">Leamon et al. 2003</a>), polony arrays (<a class="xref-" href="#ref-66" id="xref-ref-66-2">Mitra and Church 1999</a>), or random dispersion of single molecules. Albert et al. (<a class="xref-" href="#ref-2" id="xref-ref-2-2">2003</a>)
have demonstrated the 5′→3′ synthesis of oligonucleotide on a
high-density array and the application of incorporation of dye-labeled
ddNTPs by DNA polymerase. These advantages of the CRT technology could
represent significant improvements in speed, throughput, and accuracy
over Sanger and SNA approaches. </p>
                  
                  <p id="p-44">At
the center of the CRT chemistry is the reversible terminator. Ideally,
these terminators should exhibit fast and efficient deprotection
kinetics, efficient incorporation kinetics by DNA polymerase, and
labels with desired characteristics, such as fluorophores with good
fluorescence properties. Of the challenges associated with CRT for
high-throughput genome sequencing, creating these reversible
terminators with the desired properties and identifying DNA polymerases
that recognize these substrates with high affinities are the most
demanding aspects of the technology. The latter point is exemplified by
the presence of competing natural nucleotides, which can readily cause
asynchronistic base extensions (<a class="xref-" href="#ref-65" id="xref-ref-65-1">Metzker et al. 1998</a>). The first examples of reversible terminators using commercially available DNA polymerases were reported by Canard and Sarfati
                     (<a class="xref-" href="#ref-9" id="xref-ref-9-1">1994</a>) and Metzker et al. (<a class="xref-" href="#ref-63" id="xref-ref-63-1">1994</a>).
                  </p>
                  
                  <div id="F4" class="fig pos-float">
                     <div class="fig-inline"><a href="http://genome.cshlp.org/content/15/12/1767/F4.expansion.html"><img alt="Figure 4." src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/F4.gif"></a><div class="callout"><span>View larger version:</span><ul class="callout-links">
                              <li><a href="http://genome.cshlp.org/content/15/12/1767/F4.expansion.html">In this page</a></li>
                              <li><a target="_blank" class="in-nw-vis" href="http://genome.cshlp.org/content/15/12/1767/F4.expansion.html">In a new window</a></li>
                           </ul>
                           <ul class="fig-services">
                              <li class="ppt-link"><a href="http://genome.cshlp.org/powerpoint/15/12/1767/F4">Download as PowerPoint Slide</a></li>
                           </ul>
                        </div>
                     </div>
                     <div class="fig-caption"><span class="fig-label"><strong>Figure 4.</strong></span> 
                        
                        <p id="p-46" class="first-child">CRT technologies. (<em>A</em>) The CRT cycle. (<em>B</em>) The photocleavage reaction of a 3′-<em>O</em>-2-nitrobenzyl-nucleoside. (<em>C</em>) Effect of cycle efficiency on CRT read-length. (<em>D</em>)
Kinetic study of protocleavage reaction for single substituted (2-SSNB)
and double substituted (2-dsNB) 2-nitrobenzyl thymidine analogs.
Percentage thymidine (%Thy) was calculated according to the equation:
%Thy = A<sub>Thy</sub>/(A<sub>Thy</sub> + A<sub>s2NB</sub>), where A<sub>Thy</sub> and A<sub>s2NB</sub>
are the integrated peak areas from RP-HPLC analysis for thymidine and
substituted 2-nitrobenzyl thymidine analogs, respectively. </p><br class="caption-clear"></div>
                  </div>
                  
                  <p id="p-47">For
CRT terminators to function properly, the protecting group must be
efficiently cleaved under mild conditions while coupled to the primer.
Removal of the protecting group generally involves either treatment
with strong acid or base, catalytic or chemical reduction, or a
combination of these methods. Unfortunately, these conditions may
chemically perturb the DNA polymerase, nucleotides,
oligonucleotide-primed template, or the solid support. Use of
photocleavable protecting groups is an attractive alternative to
rigorous chemical treatment and can be employed in a noninvasive
manner. Of the various photocleavable protecting groups (<a class="xref-" href="#ref-77" id="xref-ref-77-1">Pillai 1980</a>),
the light-sensitive 2-nitrobenzyl group has been widely used. For
example, it has been applied to natural nucleotides (Metzker et al. <a class="xref-" href="#ref-63" id="xref-ref-63-2">1994</a>, <a class="xref-" href="#ref-65" id="xref-ref-65-2">1998</a>), to the linker structure coupling a fluorescent dye to nucleobases (<a class="xref-" href="#ref-52" id="xref-ref-52-1">Li et al. 2003</a>; <a class="xref-" href="#ref-67" id="xref-ref-67-2">Mitra et al. 2003</a>), and to other nucleic acid structures as well (<a class="xref-" href="#ref-69" id="xref-ref-69-1">Ohtsuka et al. 1974</a>; <a class="xref-" href="#ref-76" id="xref-ref-76-2">Pease et al. 1994</a>; <a class="xref-" href="#ref-13" id="xref-ref-13-1">Chaulk and MacMillan 1998</a>; <a class="xref-" href="#ref-99" id="xref-ref-99-1">Singh-Gasson et al. 1999</a>). Under appropriate deprotection conditions (e.g., ultraviolet light &gt;300 nm), the 2-nitrobenzyl group can be efficiently
                     cleaved (<a class="xref-" href="#F4" id="xref-fig-4-3">Fig. 4B</a>) without affecting either the pyrimidine or purine bases (<a class="xref-" href="#ref-4" id="xref-ref-4-1">Bartholomew and Broom 1975</a>; <a class="xref-" href="#ref-76" id="xref-ref-76-3">Pease et al. 1994</a>).
                  </p>
                  
                  <p id="p-48">Other protecting groups have been described for reversible terminators as well. For example, Metzker et al. (<a class="xref-" href="#ref-63" id="xref-ref-63-3">1994</a>) first described the synthesis and incorporation of a 3′-<em>O</em>-allyl-dATP by DNA polymerase, with the <em>O</em>-allyl group being removed using the well-known palladium (Pd) catalyst chemistry (Hayakawa et al. <a class="xref-" href="#ref-33" id="xref-ref-33-1">1986</a>, <a class="xref-" href="#ref-34" id="xref-ref-34-1">1993</a>; <a class="xref-" href="#ref-35" id="xref-ref-35-1">Honda et al. 1997</a>). Recently, Ruparel et al. (<a class="xref-" href="#ref-89" id="xref-ref-89-1">2005</a>) reported the synthesis of the first fluorescently labeled 3′-<em>O</em>-allyl-dNTPs. These unique reversible terminators require dual deprotection steps using UV light to cleave the fluorophore
                     from the nucleotide (<a class="xref-" href="#F3" id="xref-fig-3-4">Fig. 3B</a>), and the Pd catalyst reaction to restore the natural 3′-OH substrate. At this year's <em>Advances in Genome Biology and Technology/Automation in Mapping and Sequencing</em> meeting, Solexa reported on a similar CRT chemistry with a sequence read-length of approximately 20 bases (<a class="xref-" href="#ref-113" id="xref-ref-113-1">http://www.agbt.org</a>) and recently reported the complete sequencing of the ϕχ174 genome (<a class="xref-" href="#ref-114" id="xref-ref-114-2">http://www.solexa.com</a>).
                  </p>
                  
                  <p id="p-49">Earlier
concerns regarding short read-lengths and assemblies for SNA strategies
will prove relevant to CRT as well. To overcome this issue, research
efforts in CRT technology development will continue to focus on the
cycle efficiency. The CRT read-length is governed by the overall cycle
efficiency, which is highly dependent on the product of deprotection
and incorporation efficiencies. For example, if one considers the
conservative loss of 50% signal as the assay's end-point, the
read-length is a function of the cycle efficiency (C<em><sub>eff</sub></em>) (<a class="xref-" href="#F4" id="xref-fig-4-4">Fig. 4C</a>). Here, a read-length of only seven bases will be achieved with an overall cycle efficiency of 90% and can be increased beyond
                     100 bases in length by improving cycle efficiency to &gt;99%. <a class="xref-" href="#F4" id="xref-fig-4-5">Figure 4D</a>
illustrates the effect that chemical modifications of the 2-nitrobenzyl
ring system have on deprotection efficiency and thymidine production
(V.A. Litosh, W. Wu, B. Stupi, and M. Metzker, unpubl.). Thus, recent
improvements in chemical engineering of reversible terminators are
important developments for CRT as an emerging technology for DNA
sequencing applications. </p>
               </div>
               <div class="section" id="sec-10">
                  <div class="section-nav"><a href="#sec-9" title="CRT" class="prev-section-link"><span>Previous Section</span></a><a href="#ack-1" title="Acknowledgments" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Conclusions</h2>
                  
                  <p id="p-50">Recent
developments in DNA polymerase-dependent strategies highlight the
central role these methods play in determination of the overall success
of the sequencing assay. Although the standards for current Sanger
technology have set the mark for emerging SNA and CRT technologies,
these measures have evolved over several decades and from numerous
research laboratories. The integration of additional technologies will
be key for development of robust DNA sequencing platforms, including
instrumentation, microfluidics, robotics, automation, software control,
data acquisition, and informatics. </p>
                  
                  <p id="p-51">Beyond
the integrated instrumentation built around the chemistry, the method
by which genomes are sequenced will be important. Most strategies
described in this review will employ the random approach of
whole-genome shotgun sequencing and assembly (<a class="xref-" href="#ref-106" id="xref-ref-106-1">Weber and Myers 1997</a>),
including resequencing efforts for human sequence variation studies.
While the random approach has the advantage of simplicity, it will
require a tremendous number of sequence reads (i.e., a minimum of 900
million, 100-base reads will be needed to achieve a 30× assembly for a
mammalian-size genome) to produce comprehensive sequence data for
comparative studies between genomes. A directed approach, which targets
specific regions across the genome, can effectively reduce genome size
and complexity and, therefore, the number of sequencing reads needed to
produce these comprehensive data sets. One example of a directed
strategy for human resequencing could be the application of the CRT
method to 5′→3′ synthesized high-density oligonucleotide arrays (<a class="xref-" href="#ref-2" id="xref-ref-2-3">Albert et al. 2003</a>)
by relying on the reference sequence as anchor points along the genome.
The careful selection of unique and functional priming sites would
represent an oligonucleotide tiling path across the genome. Priming CRT
reactions from these anchor points and sequencing to adjacent priming
sites would provide contiguous coverage of the targeted regions of
interest. CRT reads could then be aligned to the known positions along
the reference genome in a straightforward manner. This approach could
also be used for mapping sequence reads to related genomes for
comparative genomics studies. Alignment of random reads could be
performed using conventional assembly algorithms, guided by the
reference sequence, to produce contiguous DNA sequence information. </p>
                  
                  <p id="p-52">Although
in its infancy, the potential for these emerging sequencing strategies
to deliver next-generation technologies looks promising. Improvements
in speed, efficiency, throughput, and sensitivity will all contribute
to a reduction in cost over the next several years. The timing of these
strategies coincides with an increasing demand for resequencing
capacity, which will provide valuable insight into the role of specific
sequence variation with common disease. Integration of
multidisciplinary technologies will translate into practical and
affordable sequencing devices capable of whole-genome analyses.
Application of genome sequence information to health benefits could
revolutionize disease prevention measures, early disease interventions,
and make the possibility of personalized therapies routine. </p>
               </div>
               <div class="section ack" id="ack-1">
                  <div class="section-nav"><a href="#sec-10" title="Conclusions" class="prev-section-link"><span>Previous Section</span></a><a href="#fn-group-1" title="Footnotes" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Acknowledgments</h2>
                  
                  <p id="p-55">I
am extremely grateful to Richard A. Gibbs, Donna M. Muzny, and Sherry
Metzker for critical review of the manuscript; Steven A. Soper for
technical discussion; and NHGRI for their support from grants R01
HG003573, R41 HG003072, R41 HG003265, and R21 HG002443. </p>
               </div>
               <div class="section fn-group" id="fn-group-1">
                  <div class="section-nav"><a href="#ack-1" title="Acknowledgments" class="prev-section-link"><span>Previous Section</span></a><a href="#ref-list-1" title="References" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Footnotes</h2>
                  <ul>
                     <li class="fn" id="fn-2">
                        <p id="p-53"><strong>E-mail <span class="em-link"><a href="mailto:mmetzker@bcm.tmc.edu">mmetzker@bcm.tmc.edu</a></span>; fax (713) 798-5741.</strong></p>
                     </li>
                     <li class="fn" id="fn-3">
                        <p id="p-54">Article and publication are at http://www.genome.org/cgi/doi/10.1101/gr.3770505.</p>
                     </li>
                     <li class="copyright-statement" id="copyright-statement-1">Cold Spring Harbor Laboratory Press</li>
                  </ul>
               </div>
               <div class="section ref-list" id="ref-list-1">
                  <div class="section-nav"><a href="#fn-group-1" title="Footnotes" class="prev-section-link"><span>Previous Section</span></a><a href="#ref-list-2" title="Web site references" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>References</h2>
                  <ol class="cit-list ref-use-labels">
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.1">
                           <div class="cit-metadata"><cite>Alaverdian, L., Alaverdian, S., Bilenko, O., Bogdanov, I., Filippova, E., Gavrilov, D., Gorbovitski, B., Gouzman, M., Gudkov,
                                 G., Domratchev, S., et al. <span class="cit-pub-date">2002</span>. A family of novel DNA sequencing instruments based on single-photon detection. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">23</span>: <span class="cit-fpage">2804</span>-2817.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/1522-2683%28200208%2923:16%3C2804::AID-ELPS2804%3E3.0.CO;2-9&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=12210185&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.2">
                           <div class="cit-metadata"><cite>Albert, T.J., Norton, J., Ott, M., Richmond, T., Nuwaysir, K., Nuwaysir, E.F., Stengele, K.-P., and Green, R.D. <span class="cit-pub-date">2003</span>. Light-directed 5′→3′ synthesis of complex oligonucleotide microarrays. <span class="cit-source">Nucleic Acids Res.</span> <span class="cit-vol">31</span>: <span class="cit-fpage">e35</span>.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=31/7/e35">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.3">
                           <div class="cit-metadata"><cite>Backhouse, C., Caamano, M., Oaks, F., Nordman, E., Carrillo, A., Johnson, B., and Bay, S. <span class="cit-pub-date">2000</span>. DNA sequencing in a monolithic microchannel device. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">21</span>: <span class="cit-fpage">150</span>-156.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/%28SICI%291522-2683%2820000101%2921:1%3C150::AID-ELPS150%3E3.0.CO;2-5&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=10634481&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.4">
                           <div class="cit-metadata"><cite>Bartholomew, D.G. and Broom, A.D. <span class="cit-pub-date">1975</span>. One-step chemical synthesis of ribonucleosides bearing a photolabile ether protecting group. <span class="cit-source">J. Chem. Soc. Chem. Commun.</span> <strong>Issue 2:</strong> <span class="cit-fpage">38</span>.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.5">
                           <div class="cit-metadata"><cite>Becker, H. and Gartner, C. <span class="cit-pub-date">2000</span>. Polymer microfabrication methods for microfluidic analytical applications. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">21</span>: <span class="cit-fpage">12</span>-26.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/%28SICI%291522-2683%2820000101%2921:1%3C12::AID-ELPS12%3E3.0.CO;2-7&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=10634467&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.6">
                           <div class="cit-metadata"><cite>Bonham, V.L., Warshauer-Baker, E., and Collins, F.S. <span class="cit-pub-date">2005</span>. Race and ethnicity in the genome era: The complexity of the constructs. <span class="cit-source">Am. Psychol.</span> <span class="cit-vol">60</span>: <span class="cit-fpage">9</span>-15.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1037/0003-066X.60.1.9&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=15641917&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.7">
                           <div class="cit-metadata"><cite>Boone, T., Fan, Z., Hooper, H., Ricco, A., Tan, H., and Williams, S. <span class="cit-pub-date">2002</span>. Plastic advances microfluidic devices. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">74</span>: <span class="cit-fpage">78A</span>-86A.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=11838703&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.8">
                           <div class="cit-metadata"><cite>Braslavsky, I., Hebert, B., Kartalov, E., and Quake, S.R. <span class="cit-pub-date">2003</span>. Sequence information can be obtained from single DNA molecules. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">100</span>: <span class="cit-fpage">3960</span>-3964.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=100/7/3960">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.9">
                           <div class="cit-metadata"><cite>Canard, B. and Sarfati, R. <span class="cit-pub-date">1994</span>. DNA polymerase fluorescent substrates with reversible 3′-tags. <span class="cit-source">Gene</span> <span class="cit-vol">148</span>: <span class="cit-fpage">1</span>-6.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1016/0378-1119%2894%2990226-7&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=7523248&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=A1994PM49200001&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.10">
                           <div class="cit-metadata"><cite>Carrilho, E. <span class="cit-pub-date">2000</span>. DNA sequencing by capillary array electrophoresis and microfabricated array systems. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">21</span>: <span class="cit-fpage">55</span>-65.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/%28SICI%291522-2683%2820000101%2921:1%3C55::AID-ELPS55%3E3.0.CO;2-I&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=10634470&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000085065800006&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.11">
                           <div class="cit-metadata"><cite>Carrilho, E., Ruiz-Martinez, M.C., Berka, J., Smirnov, I., Goetzinger, W., Miller, A.W., Brady, D., and Karger, B.L. <span class="cit-pub-date">1996</span>. Rapid DNA sequencing of more than 1000 bases per run by capillary electrophoresis using replaceable linear polyacrylamide
                                 solutions. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">68</span>: <span class="cit-fpage">3305</span>-3313.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=8843133&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.12">
                           <div class="cit-metadata"><cite>Chaisson, M., Pevzner, P., and Tang, H. <span class="cit-pub-date">2004</span>. Fragment assembly with short reads. <span class="cit-source">Bioinformatics</span> <span class="cit-vol">20</span>: <span class="cit-fpage">2067</span>-2074.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=20/13/2067">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.13">
                           <div class="cit-metadata"><cite>Chaulk, S. and MacMillan, A. <span class="cit-pub-date">1998</span>. Caged RNA: Photo-control of a ribozyme reaction. <span class="cit-source">Nucleic Acids Res.</span> <span class="cit-vol">26</span>: <span class="cit-fpage">3173</span>-3178.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=26/13/3173">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.14">
                           <div class="cit-metadata"><cite>Collins, F. and Galas, D. <span class="cit-pub-date">1993</span>. A new five-year plan for the U.S. Human Genome Project. <span class="cit-source">Science</span> <span class="cit-vol">262</span>: <span class="cit-fpage">43</span>-46.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=PDF&amp;journalCode=sci&amp;resid=262/5130/43">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.15">
                           <div class="cit-metadata"><cite>Collins, F.S., Guyer, M.S., and Chakravarti, A. <span class="cit-pub-date">1997</span>. Variations on a theme: Cataloging human DNA sequence variation. <span class="cit-source">Science</span> <span class="cit-vol">278</span>: <span class="cit-fpage">1580</span>-1581.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=FULL&amp;journalCode=sci&amp;resid=278/5343/1580">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.16">
                           <div class="cit-metadata"><cite>Collins, F.S., Patrinos, A., Jordon, E., Chakravarti, A., Gesteland, R., Walters, L., and Members of the DOE and NIH Planning
                                 Groups. <span class="cit-pub-date">1998</span>. New goals for the U.S. Human Genome Project: 1998-2003. <span class="cit-source">Science</span> <span class="cit-vol">282</span>: <span class="cit-fpage">682</span>-689.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=282/5389/682">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.17">
                           <div class="cit-metadata"><cite>Collins, F.S., Green, E.D., Guttmacher, A.E., and Guyer, M.S. <span class="cit-pub-date">2003</span>. A vision for the future of genomics research. <span class="cit-source">Nature</span> <span class="cit-vol">422</span>: <span class="cit-fpage">835</span>-847.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1038/nature01626&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=12695777&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.18">
                           <div class="cit-metadata"><cite>Crawford, D.C. and Nickerson, D.A. <span class="cit-pub-date">2005</span>. Definition and clinical importance of haplotypes. <span class="cit-source">Annu. Rev. Med.</span> <span class="cit-vol">56</span>: <span class="cit-fpage">303</span>-320.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1146/annurev.med.56.082103.104540&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=15660514&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000227504100017&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.19">
                           <div class="cit-metadata"><cite>Culbertson, C.T., Jacobson, S.C., and Ramsey, J.M. <span class="cit-pub-date">1998</span>. Dispersion sources for compact geometries on microchips. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">70</span>: <span class="cit-fpage">3781</span>-3789.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1021/ac9804487&amp;link_type=DOI">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.20">
                           <div class="cit-metadata"><cite>Daly, M.J., Rioux, J.D., Schaffner, S.F., Hudson, T.J., and Lander, E.S. <span class="cit-pub-date">2001</span>. High-resolution haplotype structure in the human genome. <span class="cit-source">Nat. Genet.</span> <span class="cit-vol">29</span>: <span class="cit-fpage">229</span>-237.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1038/ng1001-229&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=11586305&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000171374100033&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.21">
                           <div class="cit-metadata"><cite>The ENCODE Project Consortium. <span class="cit-pub-date">2004</span>. The ENCODE (ENCyclopedia Of DNA Elements) Project. <span class="cit-source">Science</span> <span class="cit-vol">306</span>: <span class="cit-fpage">636</span>-640.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=306/5696/636">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.22">
                           <div class="cit-metadata"><cite>Entz, P., Toliat, M.R., Hampe, J., Valentonyte, R., Jenisch, S., Nürnberg, P., and Nagy, M. <span class="cit-pub-date">2005</span>. New strategies for efficient typing of HLA class-II loci DQB1 and DRB1 by using pyrosequencing. <span class="cit-source">Tissue Antigens</span> <span class="cit-vol">65</span>: <span class="cit-fpage">67</span>-80.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1111/j.1399-0039.2005.00345.x&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=15663743&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.23">
                           <div class="cit-metadata"><cite>Ewing, B. and Green, P. <span class="cit-pub-date">1998</span>. Base-calling of automated sequencer traces using Phred. II. Error probabilities. <span class="cit-source">Genome Res.</span> <span class="cit-vol">8</span>: <span class="cit-fpage">186</span>-194.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=8/3/186">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.24">
                           <div class="cit-metadata"><cite>Ewing, B., Hillier, L., Wendl, M.C., and Green, P. <span class="cit-pub-date">1998</span>. Base-calling of automated sequencer traces using Phred. I. Accuracy assessment. <span class="cit-source">Genome Res.</span> <span class="cit-vol">8</span>: <span class="cit-fpage">175</span>-185.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=8/3/175">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.25">
                           <div class="cit-metadata"><cite>Foster, M.W. and Sharp, R.R. <span class="cit-pub-date">2002</span>. Race, ethnicity, and genomics: Social classifications as proxies of biological heterogeneity. <span class="cit-source">Genome Res.</span> <span class="cit-vol">12</span>: <span class="cit-fpage">844</span>-850.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=12/6/844">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.26">
                           <div class="cit-metadata"><cite>Fox, M.S., Magasanik, B., Signer, E.R., Solomon, F., Gellert, M.F., Haber, J.E., Daniel, J., Koshland, E., and Muschel, L.H.
                                 <span class="cit-pub-date">1990</span>. The Genome Project: Pro and con. <span class="cit-source">Science</span> <span class="cit-vol">247</span>: <span class="cit-fpage">270</span>.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=PDF&amp;journalCode=sci&amp;resid=247/4940/270">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.27">
                           <div class="cit-metadata"><cite>Gabriel, S.B., Schaffner, S.F., Nguyen, H., Moore, J.M., Roy, J., Blumenstiel, B., Higgins, J., DeFelice, M., Lochner, A.,
                                 Faggart, M., et al. <span class="cit-pub-date">2002</span>. The structure of haplotype blocks in the human genome. <span class="cit-source">Science</span> <span class="cit-vol">296</span>: <span class="cit-fpage">2225</span>-2229.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=296/5576/2225">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.28">
                           <div class="cit-metadata"><cite>Gharizadeha, B., Nordströma, T., Ahmadiana, A., Ronaghi, M., and Nyrén, P. <span class="cit-pub-date">2002</span>. Long-read pyrosequencing using pure 2′-deoxyadenosine-5′-<em>O</em>′-(1-thiotriphosphate) Sp-isomer. <span class="cit-source">Anal. Biochem.</span> <span class="cit-vol">301</span>: <span class="cit-fpage">82</span>-90.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1006/abio.2001.5494&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=11811970&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000173661300011&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.29">
                           <div class="cit-metadata"><cite>Guttman, A. <span class="cit-pub-date">2002a</span>. <span class="cit-source">Capillary electrophoresis using replaceable gels</span>. U.S. patent no. RE37,606.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.30">
                           <div class="cit-metadata"><cite>———. <span class="cit-pub-date">2002b</span>. <span class="cit-source">Capillary electrophoresis using replaceable gels</span>. U.S. patent no. RE37,941.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.31">
                           <div class="cit-metadata"><cite>Harrison, D.J., Manz, A., Fan, Z., Luedi, H., and Widmer, H.M. <span class="cit-pub-date">1992</span>. Capillary electrophoresis and sample injection systems integrated on a planar glass chip. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">64</span>: <span class="cit-fpage">1926</span>-1932.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1021/ac00041a030&amp;link_type=DOI">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.32">
                           <div class="cit-metadata"><cite>Harrison, D.J., Fluri, K., Seiler, K., Fan, Z., Effenhauser, C.S., and Manz, A. <span class="cit-pub-date">1993</span>. Micromachining a miniaturized capillary electrophoresis-based chemical analysis system on a chip. <span class="cit-source">Science</span> <span class="cit-vol">261</span>: <span class="cit-fpage">895</span>-897.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.33">
                           <div class="cit-metadata"><cite>Hayakawa, Y., Kato, H., Uchiyama, M., Kajino, H., and Noyori, R. <span class="cit-pub-date">1986</span>. Allyloxycarbonyl group: A versatile blocking group for nucleotide synthesis. <span class="cit-source">J. Org. Chem.</span> <span class="cit-vol">51</span>: <span class="cit-fpage">2400</span>-2402.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1021/jo00362a052&amp;link_type=DOI">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.34">
                           <div class="cit-metadata"><cite>Hayakawa, Y., Hirose, M., and Noyori, R. <span class="cit-pub-date">1993</span>. <em>O</em>-Allyl protection of guanine and thymine residues in oligodeoxyribonucleotides. <span class="cit-source">J. Org. Chem.</span> <span class="cit-vol">58</span>: <span class="cit-fpage">5551</span>-5555.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1021/jo00072a050&amp;link_type=DOI">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.35">
                           <div class="cit-metadata"><cite>Honda, M., Morita, H., and Nagakura, I. <span class="cit-pub-date">1997</span>. Deprotection of allyl groups with sulfinic acids and palladium catalyst. <span class="cit-source">J. Org. Chem.</span> <span class="cit-vol">62</span>: <span class="cit-fpage">8932</span>-8936.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1021/jo971194c&amp;link_type=DOI">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference in text" id="ref-36">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.36">
                           <div class="cit-metadata"><cite>Hyman, E.D. <span class="cit-pub-date">1988</span>. A new method of sequencing DNA. <span class="cit-source">Anal. Biochem.</span> <span class="cit-vol">174</span>: <span class="cit-fpage">423</span>-436.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1016/0003-2697%2888%2990041-3&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=2853582&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=A1988Q905300007&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.37">
                           <div class="cit-metadata"><cite>The International HapMap Consortium. <span class="cit-pub-date">2003</span>. The International HapMap Project. <span class="cit-source">Nature</span> <span class="cit-vol">426</span>: <span class="cit-fpage">789</span>-796.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1038/nature02168&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=14685227&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.38">
                           <div class="cit-metadata"><cite>Jacobson, S.C., Hergenroder, R., Koutny, L.B., Warmack, R.J., and Ramsey, J.M. <span class="cit-pub-date">1994</span>. Effects of injection schemes and column geometry on the performance of microchip electrophoresis devices. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">66</span>: <span class="cit-fpage">1107</span>-1113.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1021/ac00079a028&amp;link_type=DOI">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.39">
                           <div class="cit-metadata"><cite>Ju, J., Ruan, C., Fuller, C., Glazer, A., and Mathies, R. <span class="cit-pub-date">1995</span>. Fluorescence energy transfer dye-labeled primers for DNA sequencing and analysis. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">92</span>: <span class="cit-fpage">4347</span>-4351.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=92/10/4347">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.40">
                           <div class="cit-metadata"><cite>Kan, C.-W., Fredlake, C.P., Doherty, E.A.S., and Barron, A.E. <span class="cit-pub-date">2004</span>. DNA sequencing and genotyping in miniaturized electrophoresis systems. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">25</span>: <span class="cit-fpage">3564</span>-3588.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/elps.200406161&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=15565709&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000225679000008&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.41">
                           <div class="cit-metadata"><cite>Kartalov, E.P. and Quake, S.R. <span class="cit-pub-date">2004</span>. Microfluidic device reads up to four consecutive base pairs in DNA sequencing-by-synthesis. <span class="cit-source">Nucleic Acids Res.</span> <span class="cit-vol">32</span>: <span class="cit-fpage">2873</span>-2879.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=32/9/2873">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference in text" id="ref-42">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.42">
                           <div class="cit-metadata"><cite>Kheterpal, I., Scherer, J., Clark, S., Radhakrishnan, A., Ju, J., Ginther, C., Sensabaugh, G.F., and Mathies, R.A. <span class="cit-pub-date">1996</span>. DNA sequencing using a four-color confocal fluorescence capillary array scanner. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">17</span>: <span class="cit-fpage">1852</span>-1859.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/elps.1150171209&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=9034766&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=A1996WE80800008&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.43">
                           <div class="cit-metadata"><cite>Koshland, D.E. <span class="cit-pub-date">1989</span>. Sequences and consequences of the human genome. <span class="cit-source">Science</span> <span class="cit-vol">246</span>: <span class="cit-fpage">189</span>.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=PDF&amp;journalCode=sci&amp;resid=246/4927/189">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference in text" id="ref-44">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.44">
                           <div class="cit-metadata"><cite>Koutny, L., Schmalzing, D., Salas-Solano, O., El-Difrawy, S., Adourian, A., Buonocore, S., Abbey, K., McEwan, P., Matsudaira,
                                 P., and Ehrlich, D. <span class="cit-pub-date">2000</span>. Eight hundred-base sequencing in a microfabricated electrophoretic device. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">72</span>: <span class="cit-fpage">3388</span>-3391.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10939418&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.45">
                           <div class="cit-metadata"><cite>Lander, E.S. <span class="cit-pub-date">1996</span>. The new genomics: Global views of biology. <span class="cit-source">Science</span> <span class="cit-vol">274</span>: <span class="cit-fpage">536</span>-539.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=FULL&amp;journalCode=sci&amp;resid=274/5287/536">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference in text" id="ref-46">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.46">
                           <div class="cit-metadata"><cite>Langaee, T. and Ronaghi, M. <span class="cit-pub-date">2005</span>. Genetic variation analyses by pyrosequencing. <span class="cit-source">Mutat. Res.</span> <span class="cit-vol">573</span>: <span class="cit-fpage">96</span>-102.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=15829240&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference in text" id="ref-47">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.47">
                           <div class="cit-metadata"><cite>Lassiter, S.J., Stryjewski, W., Benjamin, J., Legendre, L., Erdmann, R., Wahl, M., Wurm, J., Peterson, R., Middendorf, L.,
                                 and Soper, S.A. <span class="cit-pub-date">2000</span>. Time-resolved fluorescence imaging of slab gels for lifetime base-calling in DNA sequencing applications. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">72</span>: <span class="cit-fpage">5373</span>-5382.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=11080890&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference in text" id="ref-48">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.48">
                           <div class="cit-metadata"><cite>Leamon, J.H., Lee, W.L., Tartaro, K.R., Lanza, J.R., Sarkis, G.J., deWinter, A.D., Berka, J., Weiner, M., Rothberg, J.M.,
                                 and Lohman, K.L. <span class="cit-pub-date">2003</span>. A massively parallel PicoTiterPlate™ based platform for discrete picoliter-scale polymerase chain reactions. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">24</span>: <span class="cit-fpage">3769</span>-3777.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/elps.200305646&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=14613204&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000186858700027&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference in text" id="ref-49">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.49">
                           <div class="cit-metadata"><cite>Lee, L., Spurgeon, S., Heiner, C., Benson, S., Rosenblum, B., Menchen, S., Graham, R., Constantinescu, A., Upadhya, K., and
                                 Cassel, J. <span class="cit-pub-date">1997</span>. New energy transfer dyes for DNA sequencing. <span class="cit-source">Nucleic Acids Res.</span> <span class="cit-vol">25</span>: <span class="cit-fpage">2816</span>-2822.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=25/14/2816">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference in text" id="ref-50">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.50">
                           <div class="cit-metadata"><cite>Levene, M.J., Korlach, J., Turner, S.W., Foquet, M., Craighead, H.G., and Webb, W.W. <span class="cit-pub-date">2003</span>. Zero-mode waveguides for single-molecule analysis at high concentrations. <span class="cit-source">Science</span> <span class="cit-vol">299</span>: <span class="cit-fpage">682</span>-686.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=299/5607/682">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference in text" id="ref-51">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.51">
                           <div class="cit-metadata"><cite>Lewis, E.K., Haaland, W.C., Nguyen, F., Heller, D.A., Allen, M.J., MacGregor, R.R., Berger, C.S., Willingham, B., Burns, L.A.,
                                 Scott, G.B.I., et al. <span class="cit-pub-date">2005</span>. Color-blind fluorescence detection for four-color DNA sequencing. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">102</span>: <span class="cit-fpage">5346</span>-5351.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=102/15/5346">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference in text" id="ref-52">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.52">
                           <div class="cit-metadata"><cite>Li, Z., Bai, X., Ruparel, H., Kim, S., Turro, N.J., and Ju, J. <span class="cit-pub-date">2003</span>. A photocleavable fluorescent nucleotide for DNA sequencing and analysis. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">100</span>: <span class="cit-fpage">414</span>-419.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=100/2/414">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference in text" id="ref-53">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.53">
                           <div class="cit-metadata"><cite>Lieberwirth, U., Arden-Jacob, J., Drexhage, K.H., Herten, D.P., Muller, R., Neumann, M., Schulz, A., Siebert, S., Sagner,
                                 G., Klingel, S., et al. <span class="cit-pub-date">1998</span>. Multiplex dye DNA sequencing in capillary gel electrophoresis by diode laser-based time-resolved fluorescence detection.
                                 <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">70</span>: <span class="cit-fpage">4771</span>-4779.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=9844573&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference in text" id="ref-54">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.54">
                           <div class="cit-metadata"><cite>Liu, S., Shi, Y., Ja, W., and Mathies, R.A. <span class="cit-pub-date">1999</span>. Optimization of high-speed DNA sequencing on microfabricated capillary electrophoresis channels. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">71</span>: <span class="cit-fpage">566</span>-573.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=9989377&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference in text" id="ref-55">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.55">
                           <div class="cit-metadata"><cite>Liu, S., Ren, H., Gao, Q., Roach, D.J., Loder Jr., R.T., Armstrong, T.M., Mao, Q., Blaga, I., Barker, D.L., and Jovanovich,
                                 S.B. <span class="cit-pub-date">2000</span>. Automated parallel DNA sequencing on multiple channel microchips. <span class="cit-source">Proc. Natl. Acad. Sci..</span> <span class="cit-vol">97</span>: <span class="cit-fpage">5369</span>-5374.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=97/10/5369">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference in text" id="ref-56">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.56">
                           <div class="cit-metadata"><cite>Liu, P.-Y., Zhang, Y.-Y., Lu, Y., Long, J.-R., Shen, H., Zhao, L.-J., Xu, F.-H., Xiao, P., Xiong, D.-H., Liu, Y.-J., et al.
                                 <span class="cit-pub-date">2005</span>. A survey of haplotype variants at several disease candidate genes: The importance of rare variants for complex diseases.
                                 <span class="cit-source">J. Med. Genet.</span> <span class="cit-vol">42</span>: <span class="cit-fpage">221</span>-227.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=jmedgenet&amp;resid=42/3/221">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference in text" id="ref-57">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.57">
                           <div class="cit-metadata"><cite>Luria, S.E., Cooper, D.M., and Berkowitz, A. <span class="cit-pub-date">1989</span>. Human Genome Project. <span class="cit-source">Science</span> <span class="cit-vol">246</span>: <span class="cit-fpage">873</span>-874.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=PDF&amp;journalCode=sci&amp;resid=246/4932/873">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference in text" id="ref-58">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.58">
                           <div class="cit-metadata"><cite>Madabhushi, R.S. <span class="cit-pub-date">1998</span>. Separation of 4-color DNA sequencing extension products in noncovalently coated capillaries using low viscosity polymer
                                 solutions. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">19</span>: <span class="cit-fpage">224</span>-230.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/elps.1150190215&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=9548284&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000072433100014&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference in text" id="ref-59">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.59">
                           <div class="cit-metadata"><cite>Madabhushi, R.S., Menchen, S.M., Efcavitch, J.W., and Grossman, P.D. <span class="cit-pub-date">1996</span>. <span class="cit-source">Polymers for separation of biomolecules by capillary electrophoresis</span>. U.S. patent no. 5,567,292.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference in text" id="ref-60">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.60">
                           <div class="cit-metadata"><cite>———. <span class="cit-pub-date">1999</span>. <span class="cit-source">Polymers for separation of biomolecules by capillary electrophoresis</span>. U.S. patent no. 5,916,426.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference in text" id="ref-61">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.61">
                           <div class="cit-metadata"><cite>Margulies, M., Egholm, M., Altman, W.E., Attiya, S., Bader, J.S., Bemben, L.A., Berka, J., Braverman, M.S., Chen, Y.-J., Chen,
                                 Z., et al. <span class="cit-pub-date">2005</span>. Genome sequencing in microfabricated high-density picolitre reactors. <span class="cit-source">Nature</span> <span class="cit-vol">437</span>: <span class="cit-fpage">376</span>-380.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=16056220&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference in text" id="ref-62">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.62">
                           <div class="cit-metadata"><cite>McDonald, J.C., Duffy, D.C., Anderson, J.R., Chiu, D.T., Wu, H., Schueller, O.J.A., and Whitesides, G.M. <span class="cit-pub-date">2000</span>. Fabrication of microfluidic systems in poly(dimethylsiloxane). <span class="cit-source">Electrophoresis</span> <span class="cit-vol">21</span>: <span class="cit-fpage">27</span>-40.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/%28SICI%291522-2683%2820000101%2921:1%3C27::AID-ELPS27%3E3.0.CO;2-C&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=10634468&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000085065800004&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference in text" id="ref-63">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.63">
                           <div class="cit-metadata"><cite>Metzker, M.L., Raghavachari, R., Richards, S., Jacutin, S.E., Civitello, A., Burgess, K., and Gibbs, R.A. <span class="cit-pub-date">1994</span>. Termination of DNA synthesis by novel 3′-modified deoxyribonucleoside triphosphates. <span class="cit-source">Nucleic Acids Res.</span> <span class="cit-vol">22</span>: <span class="cit-fpage">4259</span>-4267.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=22/20/4259">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference in text" id="ref-64">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.64">
                           <div class="cit-metadata"><cite>Metzker, M.L., Lu, J., and Gibbs, R.A. <span class="cit-pub-date">1996</span>. Electrophoretically uniform fluorescent dyes for automated DNA sequencing. <span class="cit-source">Science</span> <span class="cit-vol">271</span>: <span class="cit-fpage">1420</span>-1422.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=271/5254/1420">Abstract</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference in text" id="ref-65">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.65">
                           <div class="cit-metadata"><cite>Metzker, M.L., Raghavachari, R., Burgess, K., and Gibbs, R.A. <span class="cit-pub-date">1998</span>. Elimination of residual natural nucleotides from 3′-<em>O</em>-modified-dNTP syntheses by enzymatic Mop-Up. <span class="cit-source">BioTechniques</span> <span class="cit-vol">25</span>: <span class="cit-fpage">814</span>-817.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=9821582&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000076896100011&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference in text" id="ref-66">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.66">
                           <div class="cit-metadata"><cite>Mitra, R. and Church, G. <span class="cit-pub-date">1999</span>. In situ localized amplification and contact replication of many individual DNA molecules. <span class="cit-source">Nucleic Acids Res.</span> <span class="cit-vol">27</span>: <span class="cit-fpage">e34</span>.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=27/24/e34">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference in text" id="ref-67">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.67">
                           <div class="cit-metadata"><cite>Mitra, R.D., Shendure, J., Olejnik, J., Edyta-Krzymanska-Olejnik, and Church, G.M. <span class="cit-pub-date">2003</span>. Fluorescent in situ sequencing on polymerase colonies. <span class="cit-source">Anal. Biochem.</span> <span class="cit-vol">320</span>: <span class="cit-fpage">55</span>-65.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1016/S0003-2697%2803%2900291-4&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=12895469&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000184794200006&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference in text" id="ref-68">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.68">
                           <div class="cit-metadata"><cite>Nunnally, B.K., He, H., Li, L.-C., Tucker, S.A., and McGown, L.B. <span class="cit-pub-date">1997</span>. Characterization of visible dyes for four-decay fluorescence detection in DNA sequencing. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">69</span>: <span class="cit-fpage">2392</span>-2397.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=9212706&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference in text" id="ref-69">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.69">
                           <div class="cit-metadata"><cite>Ohtsuka, E., Tanaka, S., and Ikehara, M. <span class="cit-pub-date">1974</span>. Studies on transfer ribonucleic acids and related compounds. IX(1) Ribooligonucleotide synthesis using a photosensitive
                                 <em>o</em>-nitrobenzyl protection at the 2′-hydroxyl group. <span class="cit-source">Nucleic Acids Res.</span> <span class="cit-vol">1</span>: <span class="cit-fpage">1351</span>-1357.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=1/10/1351">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference in text" id="ref-70">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.70">
                           <div class="cit-metadata"><cite>Paegel, B.M., Hutt, L.D., Simpson, P.C., and Mathies, R.A. <span class="cit-pub-date">2000</span>. Turn geometry for minimizing band broadening in microfabricated capillary electrophoresis channels. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">70</span>: <span class="cit-fpage">3030</span>-3037.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1021/ac000054r&amp;link_type=DOI">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference in text" id="ref-71">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.71">
                           <div class="cit-metadata"><cite>Paegel, B.M., Emrich, C.A., Wedemayer, G.J., Scherer, J.R., and Mathies, R.A. <span class="cit-pub-date">2002</span>. High throughput DNA sequencing with a microfabricated 96-lane capillary array electrophoresis bioprocessor. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">99</span>: <span class="cit-fpage">574</span>-579.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=99/2/574">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference in text" id="ref-72">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.72">
                           <div class="cit-metadata"><cite>Paegel, B.M., Blazej, R.G., and Mathies, R.A. <span class="cit-pub-date">2003</span>. Microfluidic devices for DNA sequencing: Sample preparation and electrophoretic analysis. <span class="cit-source">Curr. Opin. Biotechnol.</span> <span class="cit-vol">14</span>: <span class="cit-fpage">42</span>-50.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1016/S0958-1669%2802%2900004-6&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=12566001&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000181006300007&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference in text" id="ref-73">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.73">
                           <div class="cit-metadata"><cite>Parkhill,
J., Achtman, M., James, K.D., Bentley, S.D., Churcher, C., Klee, S.R.,
Morelli, G., Basham, D., Brown, D., Chillingworth, T., et al. <span class="cit-pub-date">2000a</span>. Complete DNA sequence of a serogroup A strain of <em>Neisseria meningitidis</em> Z2491. <span class="cit-source">Nature</span> <span class="cit-vol">404</span>: <span class="cit-fpage">502</span>-506.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1038/35006655&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=10761919&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference in text" id="ref-74">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.74">
                           <div class="cit-metadata"><cite>Parkhill, J., Wren, B.W., Mungall, K., Ketley, J.M., Churcher, C., Basham, D., Chillingworth, T., Davies, R.M., Feltwell,
                                 T., Holroyd, S., et al. <span class="cit-pub-date">2000b</span>. The genome sequence of the food-borne pathogen <em>Campylobacter jejuni</em> reveals hypervariable sequences. <span class="cit-source">Nature</span> <span class="cit-vol">403</span>: <span class="cit-fpage">665</span>-668.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1038/35001088&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=10688204&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference in text" id="ref-75">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.75">
                           <div class="cit-metadata"><cite>Patil, N., Berno, A.J., Hinds, D.A., Barrett, W.A., Doshi, J.M., Hacker, C.R., Kautzer, C.R., Lee, D.H., Marjoribanks, C.,
                                 McDonough, D.P., et al. <span class="cit-pub-date">2001</span>. Blocks of limited haplotype diversity revealed by high-resolution scanning of human chromosome 21. <span class="cit-source">Science</span> <span class="cit-vol">294</span>: <span class="cit-fpage">1719</span>-1723.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=294/5547/1719">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference in text" id="ref-76">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.76">
                           <div class="cit-metadata"><cite>Pease, A.C., Solas, D., Sullivan, E.J., Cronin, M.T., Holmes, C.P., and Fodor, S.P.A. <span class="cit-pub-date">1994</span>. Light-generated oligonucleotide arrays for rapid DNA sequence analysis. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">91</span>: <span class="cit-fpage">5022</span>-5026.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=91/11/5022">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference in text" id="ref-77">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.77">
                           <div class="cit-metadata"><cite>Pillai, V.N.R. <span class="cit-pub-date">1980</span>. Photoremovable protecting groups in organic synthesis. <span class="cit-source">Synthesis</span> <strong>Issue 2:</strong> <span class="cit-fpage">1</span>-26.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference in text" id="ref-78">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.78">
                           <div class="cit-metadata"><cite>Prober, J., Trainor, G., Dam, R., Hobbs, F., Robertson, C., Zagursky, R., Cocuzza, A., Jensen, M., and Baumeister, K. <span class="cit-pub-date">1987</span>. A system for rapid DNA sequencing with fluorescent chain-terminating dideoxynucleotides. <span class="cit-source">Science</span> <span class="cit-vol">238</span>: <span class="cit-fpage">336</span>-341.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=238/4825/336">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference in text" id="ref-79">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.79">
                           <div class="cit-metadata"><cite>Quake, S. and Scherer, A. <span class="cit-pub-date">2000</span>. From micro- to nanofabrication with soft materials. <span class="cit-source">Science</span> <span class="cit-vol">290</span>: <span class="cit-fpage">1536</span>-1540.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=290/5496/1536">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference in text" id="ref-80">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.80">
                           <div class="cit-metadata"><cite>Risch, N. and Merikangas, K. <span class="cit-pub-date">1996</span>. The future of genetic studies of complex human diseases. <span class="cit-source">Science</span> <span class="cit-vol">273</span>: <span class="cit-fpage">1516</span>-1517.</cite></div>
                           <div class="cit-extra"><a href="">CANCERLIT</a><a href="http://genome.cshlp.org/external-ref?access_num=8801636&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=A1996VG59700028&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference in text" id="ref-81">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.81">
                           <div class="cit-metadata"><cite>Roberts, L. <span class="cit-pub-date">1989a</span>. New game plan for genome mapping. <span class="cit-source">Science</span> <span class="cit-vol">245</span>: <span class="cit-fpage">1438</span>-1440.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=PDF&amp;journalCode=sci&amp;resid=245/4925/1438">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference in text" id="ref-82">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.82">
                           <div class="cit-metadata"><cite>———. <span class="cit-pub-date">1989b</span>. Watson versus Japan. <span class="cit-source">Science</span> <span class="cit-vol">246</span>: <span class="cit-fpage">576</span>-578.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=PDF&amp;journalCode=sci&amp;resid=246/4930/576">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference in text" id="ref-83">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.83">
                           <div class="cit-metadata"><cite>Robertson, J.A. <span class="cit-pub-date">2003</span>. The $1000 genome: Ethical and legal issues in whole genome sequencing of individuals. <span class="cit-source">Am. J. Bioeth.</span> <span class="cit-vol">3</span>: <span class="cit-fpage">W-IF1</span>.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference in text" id="ref-84">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.84">
                           <div class="cit-metadata"><cite>Ronaghi, M. <span class="cit-pub-date">2000</span>. Improved performance of pyrosequencing using single-stranded DNA-binding protein. <span class="cit-source">Anal. Biochem.</span> <span class="cit-vol">286</span>: <span class="cit-fpage">282</span>-288.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1006/abio.2000.4808&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=11067751&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000165345700014&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference in text" id="ref-85">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.85">
                           <div class="cit-metadata"><cite>———. <span class="cit-pub-date">2001</span>. Pyrosequencing sheds light on DNA sequencing. <span class="cit-source">Genome Res.</span> <span class="cit-vol">11</span>: <span class="cit-fpage">3</span>-11.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=11/1/3">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference in text" id="ref-86">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.86">
                           <div class="cit-metadata"><cite>Ronaghi, M., Karamohamed, S., Pettersson, B., Uhlén, M., and Nyrén, P. <span class="cit-pub-date">1996</span>. Real-time DNA sequencing using detection of pyrophosphate release. <span class="cit-source">Anal. Biochem.</span> <span class="cit-vol">242</span>: <span class="cit-fpage">84</span>-89.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1006/abio.1996.0432&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=8923969&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=A1996VT72900014&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference in text" id="ref-87">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.87">
                           <div class="cit-metadata"><cite>Ronaghi, M., Uhlén, M., and Nyrén, P. <span class="cit-pub-date">1998</span>. A sequencing method based on real-time pyrophosphate. <span class="cit-source">Science</span> <span class="cit-vol">281</span>: <span class="cit-fpage">363</span>, 365.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=FULL&amp;journalCode=sci&amp;resid=281/5375/363">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference in text" id="ref-88">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.88">
                           <div class="cit-metadata"><cite>Ruiz-Martinez, M.C., Berka, J., Belenkii, A., Foret, F., Miller, A.W., and Karger, B.L. <span class="cit-pub-date">1993</span>. DNA sequencing by capillary electrophoresis with replaceable linear polyacrylamide and laser-induced fluorescence detection.
                                 <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">65</span>: <span class="cit-fpage">2851</span>-2858.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=8250265&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference in text" id="ref-89">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.89">
                           <div class="cit-metadata"><cite>Ruparel, H., Bi, L., Li, Z., Bai, X., Kim, D.H., Turro, N.J., and Ju, J. <span class="cit-pub-date">2005</span>. Design and synthesis of a 3′-<em>O</em>-allyl photocleavable fluorescent nucleotide as a reversible terminator for DNA sequencing by synthesis. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">102</span>: <span class="cit-fpage">5932</span>-5937.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=102/17/5932">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference in text" id="ref-90">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.90">
                           <div class="cit-metadata"><cite>Salas-Solano, O., Carrilho, E., Kotler, L., Miller, A.W., Goetzinger, W., Sosic, Z., and Karger, B.L. <span class="cit-pub-date">1998</span>. Routine DNA Sequencing of 1000 Bases in Less Than One Hour by Capillary Electrophoresis with Replaceable Linear Polyacrylamide
                                 Solutions. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">70</span>: <span class="cit-fpage">3996</span>-4003.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=9784744&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference in text" id="ref-91">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.91">
                           <div class="cit-metadata"><cite>Salas-Solano, O., Schmalzing, D., Koutny, L., Buonocore, S., Adourian, A., Matsudaira, P., and Ehrlich, D. <span class="cit-pub-date">2000</span>. Optimization of high-performance DNA sequencing on short microfabricated electrophoretic devices. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">72</span>: <span class="cit-fpage">3129</span>-3137.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10939377&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference in text" id="ref-92">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.92">
                           <div class="cit-metadata"><cite>Sanger, F., Nicklen, S., and Coulson, A.R. <span class="cit-pub-date">1977</span>. DNA sequencing with chain-terminating inhibitors. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">74</span>: <span class="cit-fpage">5463</span>-5467.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=74/12/5463">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference in text" id="ref-93">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.93">
                           <div class="cit-metadata"><cite>Schmalzing, D., Tsao, N., Koutny, L., Chisholm, D., Srivastava, A., Adourian, A., Linton, L., McEwan, P., Matsudaira, P.,
                                 and Ehrlich. D. <span class="cit-pub-date">1999</span>. Toward real-world sequencing by microdevice electrophoresis. <span class="cit-source">Genome Res.</span> <span class="cit-vol">9</span>: <span class="cit-fpage">853</span>-858.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=9/9/853">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference in text" id="ref-94">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.94">
                           <div class="cit-metadata"><cite>Seo, T.S., Bai, X., Ruparel, H., Li, Z., Turro, N.J., and Ju, J. <span class="cit-pub-date">2004</span>. Photocleavable fluorescent nucleotides for DNA sequencing on a chip constructed by site-specific coupling chemistry. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">101</span>: <span class="cit-fpage">5488</span>-5493.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=101/15/5488">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference in text" id="ref-95">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.95">
                           <div class="cit-metadata"><cite>Seo, T.S., Bai, X., Kim, D.H., Meng, Q., Shi, S., Ruparel, H., Li, Z., Turro, N.J., and Ju, J. <span class="cit-pub-date">2005</span>. Four-color DNA sequencing by synthesis on a chip using photocleavable fluorescent nucleotides. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">102</span>: <span class="cit-fpage">5926</span>-5931.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=102/17/5926">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference in text" id="ref-96">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.96">
                           <div class="cit-metadata"><cite>Shendure, J., Mitra, R.D., Varma, C., and Church, G.M. <span class="cit-pub-date">2004</span>. Advanced sequencing technologies: Methods and goals. <span class="cit-source">Nat. Rev. Genet.</span> <span class="cit-vol">5</span>: <span class="cit-fpage">335</span>-344.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=15143316&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000221139100011&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference in text" id="ref-97">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.97">
                           <div class="cit-metadata"><cite>Shi, Y. and Anderson, R.C. <span class="cit-pub-date">2003</span>. High-resolution single-stranded DNA analysis on 4.5 cm plastic electrophoretic microchannels. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">24</span>: <span class="cit-fpage">3371</span>-3377.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/elps.200305553&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=14595684&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000186457900010&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference in text" id="ref-98">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.98">
                           <div class="cit-metadata"><cite>Simpson, J.W., Ruiz-Martinez, M.C., Mulhern, G.T., Berka, J., Latimer, D.R., Ball, J.A., Rothberg, J.M., and Went, G.T. <span class="cit-pub-date">2000</span>. Transmission imaging spectrograph and microfabricated channel system for DNA analysis. <span class="cit-source">Electrophoresis</span> <span class="cit-vol">21</span>: <span class="cit-fpage">135</span>-149.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1002/%28SICI%291522-2683%2820000101%2921:1%3C135::AID-ELPS135%3E3.0.CO;2-U&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=10634480&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference in text" id="ref-99">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.99">
                           <div class="cit-metadata"><cite>Singh-Gasson, S., Green, R.D., Yue, Y., Nelson, C., Blattner, F., Sussman, M.R., and Cerrina, F. <span class="cit-pub-date">1999</span>. Maskless fabrication of light-directed oligonucleotide microarrays using a digital micromirror array. <span class="cit-source">Nat. Biotechnol.</span> <span class="cit-vol">17</span>: <span class="cit-fpage">974</span>-978.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1038/13664&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=10504697&amp;link_type=MED">Medline</a><a href="http://genome.cshlp.org/external-ref?access_num=000083015700022&amp;link_type=ISI">ISI</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference in text" id="ref-100">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.100">
                           <div class="cit-metadata"><cite>Smith, L., Sanders, J., Kaiser, R., Hughes, P., Dodd, C., Connell, C., Heiner, C., Kent, S., and Hood, L. <span class="cit-pub-date">1986</span>. Fluorescence detection in automated DNA sequence analysis. <span class="cit-source">Nature</span> <span class="cit-vol">321</span>: <span class="cit-fpage">674</span>-679.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1038/321674a0&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=3713851&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference in text" id="ref-101">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.101">
                           <div class="cit-metadata"><cite>Smith, L.M., Kaiser, R.J., Sanders, J.Z., and Hood, L.E. <span class="cit-pub-date">1987</span>. The synthesis and use of fluorescent oligonucleotides in DNA sequence analysis. <span class="cit-source">Methods Enzymol.</span> <span class="cit-vol">155</span>: <span class="cit-fpage">260</span>-301.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=3431463&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference in text" id="ref-102">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.102">
                           <div class="cit-metadata"><cite>Tabor, S. and Richardson, C.C. <span class="cit-pub-date">1989</span>. Effect of manganese ions on the incorporation of dideoxynucleotides by bacteriophage T7 DNA polymerase and <em>Escherichia coli</em> DNA polymerase I. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">86</span>: <span class="cit-fpage">4076</span>-4080.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=86/11/4076">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference in text" id="ref-103">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.103">
                           <div class="cit-metadata"><cite>———. <span class="cit-pub-date">1995</span>. A single residue in DNA polymerases of the <em>Escherichia coli</em> DNA polymerase I family is critical for distinguishing between deoxy- and dideoxyribonucleotides. <span class="cit-source">Proc. Natl. Acad. Sci.</span> <span class="cit-vol">92</span>: <span class="cit-fpage">6339</span>-6343.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=92/14/6339">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference in text" id="ref-104">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.104">
                           <div class="cit-metadata"><cite>Takahashi, S., Murakami, K., Anazawa, T., and Kambara, H. <span class="cit-pub-date">1994</span>. Multiple sheath-flow gel capillary-array electrophoresis for multicolor fluorescent DNA detection. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">66</span>: <span class="cit-fpage">1021</span>-1026.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1021/ac00079a016&amp;link_type=DOI">CrossRef</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference in text" id="ref-105">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.105">
                           <div class="cit-metadata"><cite>Velculescu, V.E., Zhang, L., Vogelstein, B., and Kinzler, K.W. <span class="cit-pub-date">1995</span>. Serial analysis of gene expression. <span class="cit-source">Science</span> <span class="cit-vol">270</span>: <span class="cit-fpage">484</span>-487.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=270/5235/484">Abstract/FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference in text" id="ref-106">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.106">
                           <div class="cit-metadata"><cite>Weber, J.L. and Myers, E.W. <span class="cit-pub-date">1997</span>. Human whole-genome shotgun sequencing. <span class="cit-source">Genome Res.</span> <span class="cit-vol">7</span>: <span class="cit-fpage">401</span>-409.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/cgi/ijlink?linkType=FULL&amp;journalCode=genome&amp;resid=7/5/401">FREE Full Text</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference in text" id="ref-107">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.107">
                           <div class="cit-metadata"><cite>Woolley, A.T. and Mathies, R.A. <span class="cit-pub-date">1995</span>. Ultra-high-speed DNA sequencing using capillary electrophoresis chips. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">67</span>: <span class="cit-fpage">3676</span>-3680.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=8644919&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference in text" id="ref-108">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.108">
                           <div class="cit-metadata"><cite>Zhang, C.-X. and Manz, A. <span class="cit-pub-date">2001</span>. Narrow sample channel injectors for capillary electrophoresis on microchips. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">73</span>: <span class="cit-fpage">2656</span>-2662.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=11403313&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference in text" id="ref-109">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.109">
                           <div class="cit-metadata"><cite>Zhu, L., Stryjewski, W., Lassiter, S., and Soper, S.A. <span class="cit-pub-date">2003</span>. Fluorescence multiplexing with time-resolved and spectral discrimination using a near-IR detector. <span class="cit-source">Anal. Chem.</span> <span class="cit-vol">75</span>: <span class="cit-fpage">2280</span>-2291.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=12918968&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference in text" id="ref-110">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.110">
                           <div class="cit-metadata"><cite>Zhu, L., Stryjewski, W.J., and Soper, S.A. <span class="cit-pub-date">2004</span>. Multiplexed fluorescence detection in microfabricated devices with both time-resolved and spectral-discrimination capabilities
                                 using near-infrared fluorescence. <span class="cit-source">Anal. Biochem.</span> <span class="cit-vol">330</span>: <span class="cit-fpage">206</span>-218.</cite></div>
                           <div class="cit-extra"><a href="http://genome.cshlp.org/external-ref?access_num=10.1016/j.ab.2004.03.047&amp;link_type=DOI">CrossRef</a><a href="http://genome.cshlp.org/external-ref?access_num=15203326&amp;link_type=MED">Medline</a></div>
                        </div>
                     </li>
                  </ol>
               </div>
               <div class="section ref-list" id="ref-list-2">
                  <div class="section-nav"><a href="#ref-list-1" title="References" class="prev-section-link"><span>Previous Section</span></a><div class="nav-placeholder">&nbsp;</div>
                  </div>
                  <h2>Web site references</h2>
                  <ol class="cit-list ref-use-labels">
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference in text" id="ref-111">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.111">
                           <div class="cit-metadata"><cite><a xmlns:l="http://schema.highwire.org/Linking" href="pending:yes" l:ref-type="url" l:ref="http://grants.nih.gov/grants/guide/rfa-files/RFA-HG-04-002.html">http://grants.nih.gov/grants/guide/rfa-files/RFA-HG-04-002.html</a>; RFA-HG-04-002. <span class="cit-pub-date">2004</span>. $100,000 genome RFA.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference in text" id="ref-112">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.112">
                           <div class="cit-metadata"><cite><a xmlns:l="http://schema.highwire.org/Linking" href="pending:yes" l:ref-type="url" l:ref="http://grants.nih.gov/grants/guide/rfa-files/RFA-HG-04-003.html">http://grants.nih.gov/grants/guide/rfa-files/RFA-HG-04-003.html</a>; RFA-HG-04-003. <span class="cit-pub-date">2004</span>. $1000 genome RFA.</cite></div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference in text" id="ref-113">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.113">
                           <div class="cit-metadata unstructured"><a xmlns:l="http://schema.highwire.org/Linking" href="pending:yes" l:ref-type="url" l:ref="http://www.agbt.org">http://www.agbt.org</a>; Home page for the Advances in Genome Biology and Technology meeting.
                           </div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference in text" id="ref-114">↵</a><div class="cit ref-cit ref-" id="cit-15.12.1767.114">
                           <div class="cit-metadata unstructured"><a xmlns:l="http://schema.highwire.org/Linking" href="pending:yes" l:ref-type="url" l:ref="http://www.solexa.com">http://www.solexa.com</a>; Home page for Solexa, Inc.
                           </div>
                           <div class="cit-extra"></div>
                        </div>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="social-bookmarking">
               
               <ul class="social-bookmark-links">
                  <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=CITEULIKE"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/citeulike.gif" alt="Add to CiteULike" title="CiteULike"><span class="soc-bm-link-text">CiteULike</span></a></li>
                  <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=CONNOTEA"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/connotea.gif" alt="Add to Connotea" title="Connotea"><span class="soc-bm-link-text">Connotea</span></a></li>
                  <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=DEL_ICIO_US"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/delicious.gif" alt="Add to Del.icio.us" title="Del.icio.us"><span class="soc-bm-link-text">Del.icio.us</span></a></li>
                  <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=DIGG"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/digg.gif" alt="Add to Digg" title="Digg"><span class="soc-bm-link-text">Digg</span></a></li>
                  <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=REDDIT"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/reddit.gif" alt="Add to Reddit" title="Reddit"><span class="soc-bm-link-text">Reddit</span></a></li>
                  <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=TECHNORATI"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/technorati.gif" alt="Add to Technorati" title="Technorati"><span class="soc-bm-link-text">Technorati</span></a></li>
               </ul>
               
               <p class="social-bookmarking-help"><a href="http://genome.cshlp.org/help/social_bookmarks.dtl">What's this?</a></p>
               
            </div><span id="related-urls"></span><div id="content-end-sb" class="sb-div"></div>
         <div xmlns="http://www.w3.org/1999/xhtml" id="cited-by"><h2>Articles citing this article</h2><ul class="cited-by-list"><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Real-Time DNA Sequencing from Single Polymerase Molecules</span><ul class="cit-auth-list"><li><span class="cit-auth">J. Eid</span>, </li><li><span class="cit-auth">A. Fehr</span>, </li><li><span class="cit-auth">J. Gray</span>, </li><li><span class="cit-auth">K. Luong</span>, </li><li><span class="cit-auth">J. Lyle</span>, </li><li><span class="cit-auth">G. Otto</span>, </li><li><span class="cit-auth">P. Peluso</span>, </li><li><span class="cit-auth">D. Rank</span>, </li><li><span class="cit-auth">P. Baybayan</span>, </li><li><span class="cit-auth">B. Bettman</span>, </li><li><span class="cit-auth">A. Bibillo</span>, </li><li><span class="cit-auth">K. Bjornson</span>, </li><li><span class="cit-auth">B. Chaudhuri</span>, </li><li><span class="cit-auth">F. Christians</span>, </li><li><span class="cit-auth">R. Cicero</span>, </li><li><span class="cit-auth">S. Clark</span>, </li><li><span class="cit-auth">R. Dalal</span>, </li><li><span class="cit-auth">A. deWinter</span>, </li><li><span class="cit-auth">J. Dixon</span>, </li><li><span class="cit-auth">M. Foquet</span>, </li><li><span class="cit-auth">A. Gaertner</span>, </li><li><span class="cit-auth">P. Hardenbol</span>, </li><li><span class="cit-auth">C. Heiner</span>, </li><li><span class="cit-auth">K. Hester</span>, </li><li><span class="cit-auth">D. Holden</span>, </li><li><span class="cit-auth">G. Kearns</span>, </li><li><span class="cit-auth">X. Kong</span>, </li><li><span class="cit-auth">R. Kuse</span>, </li><li><span class="cit-auth">Y. Lacroix</span>, </li><li><span class="cit-auth">S. Lin</span>, </li><li><span class="cit-auth">P. Lundquist</span>, </li><li><span class="cit-auth">C. Ma</span>, </li><li><span class="cit-auth">P. Marks</span>, </li><li><span class="cit-auth">M. Maxham</span>, </li><li><span class="cit-auth">D. Murphy</span>, </li><li><span class="cit-auth">I. Park</span>, </li><li><span class="cit-auth">T. Pham</span>, </li><li><span class="cit-auth">M. Phillips</span>, </li><li><span class="cit-auth">J. Roy</span>, </li><li><span class="cit-auth">R. Sebra</span>, </li><li><span class="cit-auth">G. Shen</span>, </li><li><span class="cit-auth">J. Sorenson</span>, </li><li><span class="cit-auth">A. Tomaney</span>, </li><li><span class="cit-auth">K. Travers</span>, </li><li><span class="cit-auth">M. Trulson</span>, </li><li><span class="cit-auth">J. Vieceli</span>, </li><li><span class="cit-auth">J. Wegener</span>, </li><li><span class="cit-auth">D. Wu</span>, </li><li><span class="cit-auth">A. Yang</span>, </li><li><span class="cit-auth">D. Zaccarin</span>, </li><li><span class="cit-auth">P. Zhao</span>, </li><li><span class="cit-auth">F. Zhong</span>, </li><li><span class="cit-auth">J. Korlach</span>, </li><li>and <span class="cit-auth">S. Turner</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Science">Science</abbr> <span class="cit-pub-dt">January 2, 2009</span> <span class="cit-vol">323</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">133</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">138</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://www.sciencemag.org/cgi/content/abstract/323/5910/133" rel="abstract">Abstract</a></li><li><a href="http://www.sciencemag.org/cgi/content/full/323/5910/133" rel="full-text">Full Text</a></li><li><a href="http://www.sciencemag.org/cgi/reprint/323/5910/133" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Sequencing by Cyclic Ligation and Cleavage (CycLiC) directly on a microarray captured template</span><ul class="cit-auth-list"><li><span class="cit-auth">K. U. Mir</span>, </li><li><span class="cit-auth">H. Qi</span>, </li><li><span class="cit-auth">O. Salata</span>, </li><li>and <span class="cit-auth">G. Scozzafava</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">January 1, 2009</span> <span class="cit-vol">37</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">e5</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/37/1/e5" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/37/1/e5" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/37/1/e5" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Sequencing by Cyclic Ligation and Cleavage (CycLiC) directly on a microarray captured template</span><ul class="cit-auth-list"><li><span class="cit-auth">K. U. Mir</span>, </li><li><span class="cit-auth">H. Qi</span>, </li><li><span class="cit-auth">O. Salata</span>, </li><li>and <span class="cit-auth">G. Scozzafava</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">November 16, 2008</span> <span class="cit-vol">0</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">gkn906v1</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">gkn906</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/gkn906v1" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/gkn906v1" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/gkn906v1" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Nested Patch PCR enables highly multiplexed mutation discovery in candidate genes</span><ul class="cit-auth-list"><li><span class="cit-auth">K. E. Varley</span></li><li>and <span class="cit-auth">R. D. Mitra</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Genome Research">Genome Res</abbr> <span class="cit-pub-dt">November 1, 2008</span> <span class="cit-vol">18</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">1844</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">1850</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://genome.cshlp.org/cgi/content/abstract/18/11/1844" rel="abstract">Abstract</a></li><li><a href="http://genome.cshlp.org/cgi/content/full/18/11/1844" rel="full-text">Full Text</a></li><li><a href="http://genome.cshlp.org/cgi/reprint/18/11/1844" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">An artificial processivity clamp made with streptavidin facilitates oriented attachment of polymerase-DNA complexes to surfaces</span><ul class="cit-auth-list"><li><span class="cit-auth">J. G. K. Williams</span>, </li><li><span class="cit-auth">D. L. Steffens</span>, </li><li><span class="cit-auth">J. P. Anderson</span>, </li><li><span class="cit-auth">T. M. Urlacher</span>, </li><li><span class="cit-auth">D. T. Lamb</span>, </li><li><span class="cit-auth">D. L. Grone</span>, </li><li>and <span class="cit-auth">J. C. Egelhoff</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">October 1, 2008</span> <span class="cit-vol">36</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">e121</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/36/18/e121" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/36/18/e121" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/36/18/e121" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Fine Expression Profiling of Full-length Transcripts using a Size-unbiased cDNA Library Prepared with the Vector-capping Method</span><ul class="cit-auth-list"><li><span class="cit-auth">M. Oshikawa</span>, </li><li><span class="cit-auth">Y. Sugai</span>, </li><li><span class="cit-auth">R. Usami</span>, </li><li><span class="cit-auth">K. Ohtoko</span>, </li><li><span class="cit-auth">S. Toyama</span>, </li><li>and <span class="cit-auth">S. Kato</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="DNA Research">DNA Res</abbr> <span class="cit-pub-dt">June 1, 2008</span> <span class="cit-vol">15</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">123</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">136</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://dnaresearch.oxfordjourn/cgi/content/abstract/15/3/123" rel="abstract">Abstract</a></li><li><a href="http://dnaresearch.oxfordjourn/cgi/content/full/15/3/123" rel="full-text">Full Text</a></li><li><a href="http://dnaresearch.oxfordjourn/cgi/reprint/15/3/123" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Velvet: Algorithms for de novo short read assembly using de Bruijn graphs</span><ul class="cit-auth-list"><li><span class="cit-auth">D. R. Zerbino</span></li><li>and <span class="cit-auth">E. Birney</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Genome Research">Genome Res</abbr> <span class="cit-pub-dt">May 1, 2008</span> <span class="cit-vol">18</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">821</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">829</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://genome.cshlp.org/cgi/content/abstract/18/5/821" rel="abstract">Abstract</a></li><li><a href="http://genome.cshlp.org/cgi/content/full/18/5/821" rel="full-text">Full Text</a></li><li><a href="http://genome.cshlp.org/cgi/reprint/18/5/821" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">A
new class of cleavable fluorescent nucleotides: synthesis and
optimization as reversible terminators for DNA sequencing by synthesis</span><ul class="cit-auth-list"><li><span class="cit-auth">G. Turcatti</span>, </li><li><span class="cit-auth">A. Romieu</span>, </li><li><span class="cit-auth">M. Fedurco</span>, </li><li>and <span class="cit-auth">A.-P. Tairi</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">March 27, 2008</span> <span class="cit-vol">36</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">e25</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/36/4/e25" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/36/4/e25" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/36/4/e25" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Short read fragment assembly of bacterial genomes</span><ul class="cit-auth-list"><li><span class="cit-auth">M. J. Chaisson</span></li><li>and <span class="cit-auth">P. A. Pevzner</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Genome Research">Genome Res</abbr> <span class="cit-pub-dt">February 1, 2008</span> <span class="cit-vol">18</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">324</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">330</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://genome.cshlp.org/cgi/content/abstract/18/2/324" rel="abstract">Abstract</a></li><li><a href="http://genome.cshlp.org/cgi/content/full/18/2/324" rel="full-text">Full Text</a></li><li><a href="http://genome.cshlp.org/cgi/reprint/18/2/324" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Termination of DNA synthesis by N6-alkylated, not 3'-O-alkylated, photocleavable 2'-deoxyadenosine triphosphates</span><ul class="cit-auth-list"><li><span class="cit-auth">W. Wu</span>, </li><li><span class="cit-auth">B. P. Stupi</span>, </li><li><span class="cit-auth">V. A. Litosh</span>, </li><li><span class="cit-auth">D. Mansouri</span>, </li><li><span class="cit-auth">D. Farley</span>, </li><li><span class="cit-auth">S. Morris</span>, </li><li><span class="cit-auth">S. Metzker</span>, </li><li>and <span class="cit-auth">M. L. Metzker</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">October 8, 2007</span> <span class="cit-vol">35</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">6339</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">6349</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/35/19/6339" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/35/19/6339" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/35/19/6339" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">DNA sequencing: bench to bedside and beyond</span><ul class="cit-auth-list"><li><span class="cit-auth">C. A. Hutchison</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">September 25, 2007</span> <span class="cit-vol">35</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">6227</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">6237</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/35/18/6227" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/35/18/6227" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/35/18/6227" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Termination of DNA synthesis by N6-alkylated, not 3'-O-alkylated, photocleavable 2'-deoxyadenosine triphosphates</span><ul class="cit-auth-list"><li><span class="cit-auth">W. Wu</span>, </li><li><span class="cit-auth">B. P. Stupi</span>, </li><li><span class="cit-auth">V. A. Litosh</span>, </li><li><span class="cit-auth">D. Mansouri</span>, </li><li><span class="cit-auth">D. Farley</span>, </li><li><span class="cit-auth">S. Morris</span>, </li><li><span class="cit-auth">S. Metzker</span>, </li><li>and <span class="cit-auth">M. L. Metzker</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">September 18, 2007</span> <span class="cit-vol">0</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">gkm689v1</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">gkm689</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/gkm689v1" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/gkm689v1" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/gkm689v1" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">DNA sequencing: bench to bedside and beyond</span><ul class="cit-auth-list"><li><span class="cit-auth">C. A. Hutchison</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">September 12, 2007</span> <span class="cit-vol">0</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">gkm688v1</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">gkm688</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/gkm688v1" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/gkm688v1" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/gkm688v1" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Joint
Estimates of Quantitative Trait Locus Effect and Frequency Using
Synthetic Recombinant Populations of Drosophila melanogaster</span><ul class="cit-auth-list"><li><span class="cit-auth">S. J. Macdonald</span></li><li>and <span class="cit-auth">A. D. Long</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Genetics">Genetics</abbr> <span class="cit-pub-dt">June 1, 2007</span> <span class="cit-vol">176</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">1261</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">1281</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://www.genetics.org/cgi/content/abstract/176/2/1261" rel="abstract">Abstract</a></li><li><a href="http://www.genetics.org/cgi/content/full/176/2/1261" rel="full-text">Full Text</a></li><li><a href="http://www.genetics.org/cgi/reprint/176/2/1261" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">PEDANT genome database: 10 years online</span><ul class="cit-auth-list"><li><span class="cit-auth">M. L. Riley</span>, </li><li><span class="cit-auth">T. Schmidt</span>, </li><li><span class="cit-auth">I. I. Artamonova</span>, </li><li><span class="cit-auth">C. Wagner</span>, </li><li><span class="cit-auth">A. Volz</span>, </li><li><span class="cit-auth">K. Heumann</span>, </li><li><span class="cit-auth">H.-W. Mewes</span>, </li><li>and <span class="cit-auth">D. Frishman</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">January 12, 2007</span> <span class="cit-vol">35</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">D354</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">D357</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/35/suppl_1/D354" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/35/suppl_1/D354" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/35/suppl_1/D354" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Four-color DNA sequencing by synthesis using cleavable fluorescent nucleotide reversible terminators</span><ul class="cit-auth-list"><li><span class="cit-auth">J. Ju</span>, </li><li><span class="cit-auth">D. H. Kim</span>, </li><li><span class="cit-auth">L. Bi</span>, </li><li><span class="cit-auth">Q. Meng</span>, </li><li><span class="cit-auth">X. Bai</span>, </li><li><span class="cit-auth">Z. Li</span>, </li><li><span class="cit-auth">X. Li</span>, </li><li><span class="cit-auth">M. S. Marma</span>, </li><li><span class="cit-auth">S. Shi</span>, </li><li><span class="cit-auth">J. Wu</span>, </li><li><span class="cit-auth">J. R. Edwards</span>, </li><li><span class="cit-auth">A. Romu</span>, </li><li>and <span class="cit-auth">N. J. Turro</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="PNAS">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-pub-dt">December 26, 2006</span> <span class="cit-vol">103</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">19635</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">19640</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://www.pnas.org/cgi/content/abstract/103/52/19635" rel="abstract">Abstract</a></li><li><a href="http://www.pnas.org/cgi/content/full/103/52/19635" rel="full-text">Full Text</a></li><li><a href="http://www.pnas.org/cgi/reprint/103/52/19635" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">PEDANT genome database: 10 years online</span><ul class="cit-auth-list"><li><span class="cit-auth">M. L. Riley</span>, </li><li><span class="cit-auth">T. Schmidt</span>, </li><li><span class="cit-auth">I. I. Artamonova</span>, </li><li><span class="cit-auth">C. Wagner</span>, </li><li><span class="cit-auth">A. Volz</span>, </li><li><span class="cit-auth">K. Heumann</span>, </li><li><span class="cit-auth">H.-W. Mewes</span>, </li><li>and <span class="cit-auth">D. Frishman</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">December 5, 2006</span> <span class="cit-vol">0</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">gkl1005v1</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">D5</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/gkl1005v1" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/gkl1005v1" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/gkl1005v1" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Robust analysis of 5'-transcript ends (5'-RATE): a novel technique for transcriptome analysis and genome annotation</span><ul class="cit-auth-list"><li><span class="cit-auth">M. Gowda</span>, </li><li><span class="cit-auth">H. Li</span>, </li><li><span class="cit-auth">J. Alessi</span>, </li><li><span class="cit-auth">F. Chen</span>, </li><li><span class="cit-auth">R. Pratt</span>, </li><li>and <span class="cit-auth">G.-L. Wang</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">November 14, 2006</span> <span class="cit-vol">34</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">e126</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/34/19/e126" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/34/19/e126" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/34/19/e126" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Capturing genomic signatures of DNA sequence variation using a standard anonymous microarray platform</span><ul class="cit-auth-list"><li><span class="cit-auth">C. H. Cannon</span>, </li><li><span class="cit-auth">C. S. Kua</span>, </li><li><span class="cit-auth">E. K. Lobenhofer</span>, </li><li>and <span class="cit-auth">P. Hurban</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">October 6, 2006</span> <span class="cit-vol">34</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">e121</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/34/18/e121" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/34/18/e121" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/34/18/e121" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Robust analysis of 5'-transcript ends (5'-RATE): a novel technique for transcriptome analysis and genome annotation</span><ul class="cit-auth-list"><li><span class="cit-auth">M. Gowda</span>, </li><li><span class="cit-auth">H. Li</span>, </li><li><span class="cit-auth">J. Alessi</span>, </li><li><span class="cit-auth">F. Chen</span>, </li><li><span class="cit-auth">R. Pratt</span>, </li><li>and <span class="cit-auth">G.-L. Wang</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">September 29, 2006</span> <span class="cit-vol">0</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">gkl522v1</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">e9</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/gkl522v1" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/gkl522v1" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/gkl522v1" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Capturing genomic signatures of DNA sequence variation using a standard anonymous microarray platform</span><ul class="cit-auth-list"><li><span class="cit-auth">C. H. Cannon</span>, </li><li><span class="cit-auth">C. S. Kua</span>, </li><li><span class="cit-auth">E. K. Lobenhofer</span>, </li><li>and <span class="cit-auth">P. Hurban</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">September 25, 2006</span> <span class="cit-vol">0</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">gkl478v1</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">e10</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/gkl478v1" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/gkl478v1" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/gkl478v1" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Systematic identification of pseudogenes through whole genome expression evidence profiling</span><ul class="cit-auth-list"><li><span class="cit-auth">A. Yao</span>, </li><li><span class="cit-auth">R. Charlab</span>, </li><li>and <span class="cit-auth">P. Li</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="Nucleic Acids Research">Nucleic Acids Res</abbr> <span class="cit-pub-dt">September 11, 2006</span> <span class="cit-vol">34</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">4477</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">4485</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://nar.oxfordjournals.org/cgi/content/abstract/34/16/4477" rel="abstract">Abstract</a></li><li><a href="http://nar.oxfordjournals.org/cgi/content/full/34/16/4477" rel="full-text">Full Text</a></li><li><a href="http://nar.oxfordjournals.org/cgi/reprint/34/16/4477" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li><li class="cit"><div class="cit-metadata"><span class="cit-title cit-first-element">Comparative sequencing of plant genomes: choices to make.</span><ul class="cit-auth-list"><li><span class="cit-auth">S. Jackson</span>, </li><li><span class="cit-auth">S. Rounsley</span>, </li><li>and <span class="cit-auth">M. Purugganan</span></li></ul><cite><abbr class="cit-jnl-abbrev" title="The Plant Cell Online">Plant Cell</abbr> <span class="cit-pub-dt">May 1, 2006</span> <span class="cit-vol">18</span><span class="cit-sep cit-sep-vol">:</span><span class="cit-pages"><span class="cit-pages-fpage">1100</span><span class="cit-sep cit-sep-page-range">-</span><span class="cit-pages-lpage">1104</span></span></cite></div><div class="cit-extra"><ul class="cit-views"><li><a href="http://www.plantcell.org/cgi/content/full/18/5/1100" rel="full-text">Full Text</a></li><li><a href="http://www.plantcell.org/cgi/reprint/18/5/1100" rel="full-text.pdf">Full Text (PDF)</a></li></ul></div></li></ul></div></div>
         <div style="height: 23568px;" id="col-2">
            
            <div class="article-nav">
               <a href="http://genome.cshlp.org/content/15/12/1759.short" title="Previous article">« Previous</a><span class="article-nav-sep"> | </span><a href="http://genome.cshlp.org/content/15/12/1777.short" title="Next article">Next Article »</a><span class="toc-link"><a href="http://genome.cshlp.org/content/15/12.toc" title="Table of Contents">Table of Contents</a></span>
               
            </div>
            <div class="content-box" id="article-cb-main">
               <div class="cb-contents">
                  <h3 class="cb-contents-header">This Article</h3>
                  <div class="cb-section cb-slug">
                     <ol>
                        <li>
                           <div id="slugline">
                              
                              
                              	    
                              
                              doi:
                              <span title="10.1101/gr.3770505" class="slug-doi">10.1101/gr.3770505
                                 </span>
                              <cite>
                                 	    <abbr title="Genome Research" class="slug-jnl-abbrev">
                                    Genome Res.</abbr><span class="slug-pub-date">
                                    2005. 
                                    </span>
                                 	    <span class="slug-vol">
                                    15: 
                                    </span><span class="slug-pages">
                                    1767-1776
                                    </span>
                                 </cite>
                              
                              <a href="http://genome.cshlp.org/site/misc/terms.xhtml"><span class="copyright-statement">Copyright © 2005, by Cold Spring Harbor Laboratory Press</span></a>
                              
                           </div>
                        </li>
                     </ol>
                  </div>
                  <div class="cb-section" id="cb-art-links">
                     <ol>
                        <li><a href="http://genome.cshlp.org/content/15/12/1767.abstract" rel="view-abstract">Abstract</a><span class="free">Free</span></li>
                        <li class="notice"><span class="variant-indicator">» <span>Full Text</span></span><span class="free">Free</span></li>
                        <li class="notice"><a href="http://genome.cshlp.org/content/15/12/1767.full.pdf+html" rel="view-full-text.pdf">Full Text (PDF)</a><span class="free">Free</span></li>
                     </ol>
                  </div>
                  <div class="cb-section collapsible" id="cb-art-cat">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">-</span> Article Category</a></h4>
                     <ol>
                        <li>
                           <ul class="subject-headings">
                              <li><a class="tocsection-search" href="http://genome.cshlp.org/search?tocsectionid=Perspectives&amp;sortspec=date&amp;submit=Submit">PERSPECTIVE</a></li>
                           </ul>
                        </li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="cb-art-svcs">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> Services</a></h4>
                     <ol style="display: none;">
                        <li><a href="http://genome.cshlp.org/email?gca=genome;15/12/1767&amp;current-view-path=/content/15/12/1767.full">
                              Email this article to a colleague</a></li>
                        <li><a href="http://genome.cshlp.org/cgi/alerts/ctalert?alertType=citedby&amp;addAlert=cited_by&amp;saveAlert=no&amp;cited_by_criteria_resid=genome;15/12/1767&amp;return-type=article&amp;return_url=http://genome.cshlp.org/content/15/12/1767.full">Alert me when this article is cited</a></li>
                        <li><a href="http://genome.cshlp.org/cgi/alerts/ctalert?alertType=correction&amp;addAlert=correction&amp;saveAlert=no&amp;correction_criteria_value=15/12/1767&amp;return-type=article&amp;return_url=http://genome.cshlp.org/content/15/12/1767.full">Alert me if a correction is posted</a></li>
                        <li><a href="http://genome.cshlp.org/search?qbe=genome;15/12/1767&amp;citation=Metzker%2015%20%2812%29:%201767&amp;submit=yes">Similar articles in this journal</a></li>
                        <li class="nodata"><div id="cb-loaded-isi-related-none">No ISI related articles</div></li>
                        <li><a href="http://genome.cshlp.org/external-ref?access_num=16339375&amp;link_type=MED_NBRS">Similar articles in PubMed</a></li>
                        <li><a href="http://genome.cshlp.org/citmgr?gca=genome;15/12/1767">Download to citation manager</a></li>
                        <li><a href="http://genome.cshlp.org/external-ref?link_type=PERMISSIONDIRECT"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/permissions.gif"></a></li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="cb-art-cit">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> Citing Articles</a></h4>
                     <ol style="display: none;">
                        <li><a href="#cited-by">View citing article information</a></li>
                        <li class="nodata"><div id="cb-loaded-isi-cited-none">No ISI citing articles</div></li>
                        <li><a href="http://genome.cshlp.org/external-ref?access_num=http://genome.cshlp.org/cgi/content/abstract/15/12/1767&amp;link_type=GOOGLESCHOLAR">Citing articles via Google Scholar</a></li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="cb-art-gs">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> Google Scholar</a></h4>
                     <ol style="display: none;">
                        <li><a href="http://scholar.google.com/scholar?q=author%3AMetzker%20author%3AM.L.">Articles by  Metzker, M. L.</a></li>
                        <li><a href="http://genome.cshlp.org/external-ref?access_num=http://genome.cshlp.org/cgi/content/abstract/15/12/1767&amp;link_type=GOOGLESCHOLARRELATED">Search for related content</a></li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="cb-art-pm">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> PubMed</a></h4>
                     <ol style="display: none;">
                        <li><a href="http://genome.cshlp.org/external-ref?access_num=16339375&amp;link_type=PUBMED">PubMed citation</a></li>
                        <li><a href="http://genome.cshlp.org/external-ref?access_num=Metzker%20ML&amp;link_type=AUTHORSEARCH">Articles by  Metzker, M. L.</a></li>
                        <li>
                           <div id="cb-entrez-links-placeholder"><ul xmlns="http://www.w3.org/1999/xhtml">
   <li>Pubmed/NCBI databases
      <ul>
         <li><a href="http://genome.cshlp.org/cgi/external_ref?id=pubmed_pccompound_mesh&amp;type=acheck&amp;target=pccompound&amp;pmid=16339375&amp;link_type=ENTREZLINKS">Compound via MeSH</a></li>
         <li><a href="http://genome.cshlp.org/cgi/external_ref?id=pubmed_pcsubstance_mesh&amp;type=acheck&amp;target=pcsubstance&amp;pmid=16339375&amp;link_type=ENTREZLINKS">Substance via MeSH</a></li>
      </ul>
   </li>
</ul></div>
                        </li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="cb-art-soc">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> Social Bookmarking</a></h4>
                     <ol style="display: none;">
                        <li>
                           <div class="social-bookmarking">
                              
                              <ul class="social-bookmark-links">
                                 <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=CITEULIKE"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/citeulike.gif" alt="Add to CiteULike" title="CiteULike"><span class="soc-bm-link-text">CiteULike</span></a></li>
                                 <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=CONNOTEA"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/connotea.gif" alt="Add to Connotea" title="Connotea"><span class="soc-bm-link-text">Connotea</span></a></li>
                                 <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=DEL_ICIO_US"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/delicious.gif" alt="Add to Del.icio.us" title="Del.icio.us"><span class="soc-bm-link-text">Del.icio.us</span></a></li>
                                 <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=DIGG"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/digg.gif" alt="Add to Digg" title="Digg"><span class="soc-bm-link-text">Digg</span></a></li>
                                 <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=REDDIT"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/reddit.gif" alt="Add to Reddit" title="Reddit"><span class="soc-bm-link-text">Reddit</span></a></li>
                                 <li><a href="http://genome.cshlp.org/external-ref?tag_url=http://genome.cshlp.org/cgi/content/long/15/12/1767&amp;title=+--+Metzker%2015%20%2812%29%3A%201767+--+&amp;doi=10.1101/gr.3770505&amp;link_type=TECHNORATI"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/technorati.gif" alt="Add to Technorati" title="Technorati"><span class="soc-bm-link-text">Technorati</span></a></li>
                              </ul>
                              
                              <p class="social-bookmarking-help"><a href="http://genome.cshlp.org/help/social_bookmarks.dtl">What's this?</a></p>
                              
                           </div>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
            
            
            
            <div id="col-2-widget-placeholder"></div>
            
            <ul class="tower-ads">
               <li class="no-ad tower_col_2"><span></span></li>
            </ul>
            <div class="content-box" id="article-dyn-nav">
               <div class="cb-contents">
                  <h3 class="cb-contents-header">Navigate This Article</h3>
                  <div class="cb-section" id="cb-art-nav">
                     <ol>
                        <li><a href="#content-block">Top</a></li>
                        <li><a href="#abstract-1">Abstract</a></li>
                        <li><a href="#sec-1"></a></li>
                        <li><a href="#sec-2">Sanger sequencing: State-of-the-art technology</a></li>
                        <li><a href="#sec-3">Sanger sequencing: Recent advances</a></li>
                        <li><a href="#sec-5">Fluorescence detection</a></li>
                        <li><a href="#sec-6">SNA methodologies</a></li>
                        <li><a href="#sec-8">Other single addition dNTP strategies</a></li>
                        <li><a href="#sec-9">CRT</a></li>
                        <li><a href="#sec-10">Conclusions</a></li>
                        <li><a href="#ack-1">Acknowledgments</a></li>
                        <li><a href="#fn-group-1">Footnotes</a></li>
                        <li><a href="#ref-list-1">References</a></li>
                        <li><a href="#ref-list-2">Web site references</a></li>
                     </ol>
                  </div>
               </div>
            </div>
            
         </div>
         <div style="height: 23568px;" id="col-3">
            
            <div class="content-box" id="sidebar-current-issue">
               <div class="cb-contents">
                  <h3 class="cb-contents-header">Current Issue</h3>
                  <div class="cb-section">
                     <ol>
                        <li><span><a href="http://genome.cshlp.org/content/current" rel="current-issue">January 2009, 19 (1)</a></span></li>
                     </ol>
                  </div>
                  <div class="cb-section">
                     <ol>
                        <li>
                           <div class="current-issue"><a href="http://genome.cshlp.org/content/current" rel="current-issue"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/1.gif" alt="Current Issue" height="89" width="67"></a></div>
                        </li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="sidebar-cur-issue-coverlines">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> From the Cover</a></h4>
                     <ul style="display: none;">
                        <li><a href="http://genome.cshlp.org/content/19/1/1">
                              Allele-specific Expression and Methylation in Acute Lymphoblastic Leukemia
                              </a></li>
                        <li><a href="http://genome.cshlp.org/content/19/1/12">
                              Genomic Prophage Islands in Cystic Fibrosis Lung Pathogen P. aeruginosa
                              </a></li>
                        <li><a href="http://genome.cshlp.org/content/19/1/42">
                              MITE-derived siRNAs in Solanaceae
                              </a></li>
                     </ul>
                  </div>
                  <div class="cb-section sidebar-etoc-link">
                     <ol>
                        <li><a href="http://genome.cshlp.org/cgi/alerts/etoc">Alert me to new issues of <span class="journal-name">
                                 Genome Research
                                 </span></a></li>
                     </ol>
                  </div>
               </div>
            </div>
            <div id="sidebar-global-nav">
               
               
               <ul class="button-list pub-links">
                  <li class="first"><a href="http://genome.cshlp.org/content/early/recent" title="Advance Online Articles "><span>Advance Online Articles </span></a></li>
                  <li><a target="_blank" href="http://submit.genome.org/" rel="external-nw" title="Submit a Manuscript [opens in a new window]"><span>Submit a Manuscript</span></a></li>
                  <li><a href="http://genome.cshlp.org/site/press/Announcements.xhtml" title="GR in the News "><span>GR in the News </span></a></li>
                  <li><a href="http://genome.cshlp.org/site/misc/about.xhtml" title="Editorial Board"><span>Editorial Board</span></a></li>
                  <li><a href="http://genome.cshlp.org/site/misc/alertland.xhtml" title="E-mail Alerts &amp; RSS Feeds"><span>E-mail Alerts &amp; RSS Feeds</span></a></li>
                  <li><a href="http://www.cshlpress.com/library/?hwinfo" title="Recommend to Your Library"><span>Recommend to Your Library</span></a></li>
                  <li class="last"><a href="http://genome.cshlp.org/site/misc/classifieds.xhtml" title="Job Opportunities"><span>Job Opportunities</span></a></li>
               </ul>
               
            </div>
            
            
            <ul class="tower-ads">
               <li><a href="http://genome.cshlp.org/cgi/adclick/?ad=19517&amp;adclick=true&amp;url=http%3A%2F%2Fwww.science.smith.edu%2Fneb" target="_blank"><img src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/NEB.bmp" alt="(advertisement)" border="0"></a></li>
            </ul>
            
            <div id="col-3-widget-placeholder"></div>
            
            
         </div>
         <div id="footer">
            
            <div class="bar">
               	  
               <div class="footer-group footer-col-left">
                  	    
                  <ul class="button-list footer-buttons">
                     <li class="first"><a href="http://genome.cshlp.org/" title="Home"><span>Home</span></a></li>
                     <li><a href="http://genome.cshlp.org/site/misc/about.xhtml" title="About"><span>About</span></a></li>
                     <li><a href="http://genome.cshlp.org/content" title="Archive"><span>Archive</span></a></li>
                     <li><a href="http://submit.genome.org/" title="Submit"><span>Submit</span></a></li>
                     <li><a href="http://genome.cshlp.org/site/subscriptions/" title="Subscribe"><span>Subscribe</span></a></li>
                     <li><a href="http://genome.cshlp.org/site/misc/adinfo.xhtml" title="Advertise"><span>Advertise</span></a></li>
                     <li><a href="http://genome.cshlp.org/site/misc/ifora.xhtml" title="Author Info"><span>Author Info</span></a></li>
                     <li><a href="http://genome.cshlp.org/cgi/feedback" title="Contact"><span>Contact</span></a></li>
                     <li class="last"><a href="http://genome.cshlp.org/help" title="Help"><span>Help</span></a></li>
                  </ul>
                  	    
                  <p class="copyright"><a href="http://www.cshlpress.com/">Copyright © 2009 by Cold Spring Harbor Laboratory Press</a></p>
                  	  
               </div>
               	  
               <div class="footer-group footer-col-right">
                  	    
                  <ul class="issns">
                     <li><span>Print ISSN: </span><span class="issn">1088-9051</span></li>
                     <li><span>Online ISSN: </span><span class="issn">1549-5469</span></li>
                  </ul>
                  	  
               </div>
               	
            </div>
            
            
            
            <div class="block-2 sb-div"></div>
            <script type="text/javascript">
        var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
        document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
        </script><script src="Metzker%202005%20Emerging%20technologies%20in%20DNA%20sequencing_files/ga.js" type="text/javascript"></script><script type="text/javascript">
        var pageTracker = _gat._getTracker("UA-743586-2");
        pageTracker._trackPageview();
        </script>
            </div>
      </div>
   </body></html>