<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "">
<!-- saved from url=(0037)http://genomebiology.com/2007/8/1/102 -->
<HTML xmlns:m="http://www.w3.org/1998/Math/MathML"><HEAD><META http-equiv="Content-Type" content="text/html; charset=UTF-8"><TITLE>Genome Biology | Full text | Genome re-annotation: a wiki solution?</TITLE>

<META name="citation_pdf_url" content="http://genomebiology.com/content/pdf/gb-2007-8-1-102.pdf">

<META name="citation_abstract_html_url" content="http://genomebiology.com/2007/8/1/102/abstract/">
<META name="citation_fulltext_html_url" content="http://genomebiology.com/2007/8/1/102">

<SCRIPT src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/viewingoptions.js" type="text/javascript"></SCRIPT>
<SCRIPT type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</SCRIPT><SCRIPT src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/ga.js" type="text/javascript"></SCRIPT>
<SCRIPT type="text/javascript">
var pageTracker = _gat._getTracker("UA-72326-7");
pageTracker._setLocalRemoteServerMode(); pageTracker._initData();
pageTracker._trackPageview();
</SCRIPT>
<LINK rel="stylesheet" type="text/css" href="./Salzberg 2007 Genome re-annotation  a wiki solution_files/screen.css" title="Default Styles"><!--[if gte IE 7]><link rel="stylesheet" type="text/css" href="/stylesheets/ie7.css" /><![endif]--><!--[if lte IE 6]><link rel="stylesheet" type="text/css" href="/stylesheets/ie56.css" /><![endif]-->

<META name="dc.title" content="Genome re-annotation: a wiki solution?">
<META name="dc.creator" content="Steven Salzberg">
<META name="dc.type" content="Opinion">
<META name="dc.source" content="Genome Biology 2007 8:102">
<META name="dc.date" content="2007-2-1">
<META name="dc.identifier" content="10.1186/gb-2007-8-1-102">
<META name="dc.publisher" content="BioMed Central Ltd">
<META name="dc.format" content="text/html">
<META name="dc.language" content="en">

<META name="prism.publicationName" content="Genome Biology">
<META name="prism.issn" content="1465-6906">
<META name="prism.publicationDate" content="2007-2-1">
<META name="prism.volume" content="8">
<META name="prism.number" content="1">
<META name="prism.section" content="Opinion">
<META name="prism.startingPage" content="102">
<META name="prism.copyright" content="2007 Salzberg">


<META name="citation_journal_title" content="Genome Biology">
<META name="citation_publisher" content="BioMed Central Ltd">
<META name="citation_authors" content="Steven Salzberg">
<META name="citation_title" content="Genome re-annotation: a wiki solution?">
<META name="citation_volume" content="8">
<META name="citation_issue" content="1">
<META name="citation_firstpage" content="102">
<META name="citation_doi" content="10.1186/gb-2007-8-1-102">

<LINK rel="stylesheet" type="text/css" href="./Salzberg 2007 Genome re-annotation  a wiki solution_files/screen(1).css" title="Default Styles">

<LINK rel="alternate" type="application/pdf" title="PDF" href="http://genomebiology.com/content/pdf/gb-2007-8-1-102.pdf">
<LINK rel="alternate" type="text/xml" title="XML version" href="http://genomebiology.com/content/xml/gb-2007-8-1-102.xml">

</HEAD><BODY link="##003399" vlink="#3366ff" alink="#6699ff" class="bgimage"><CENTER><DIV id="outerframe"><DIV class="skip"><A href="http://genomebiology.com/2007/8/1/102#content">Skip to main content</A></DIV><DIV id="newad"><TABLE cellpadding="0" cellspacing="0" border="0" align="center"><TBODY><TR><TD align="center" class="topad">

<CENTER>

<!------ OAS SETUP begin ------>
<SCRIPT language="JavaScript">
<!--
OAS_version = 10;
function OAS_NORMAL(pos)
{
   document.write('<a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/genomebiology.com/article/genomebiol/1384651778@Top,Bottom!' + pos + '" target=_top>');
   document.write('<img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/genomebiology.com/article/genomebiol/1384651778@Top,Bottom!' + pos + '" border=0></a>');
}
//-->
</SCRIPT>
<SCRIPT language="JavaScript1.1">
<!--
OAS_version = 11;
if (navigator.userAgent.indexOf('Mozilla/3') > 1)
   OAS_version = 10;
if (navigator.userAgent.indexOf('Netscape/7.0') > 1)
   OAS_version = 10;
if (OAS_version >= 11)
   document.write('<scr' + 'ipt language="JavaScript1.1" src="http://oas.biomedcentral.com/RealMedia/ads/adstream_mjx.ads/genomebiology.com/article/genomebiol/1384651778@Top,Bottom"></scr' + 'ipt>');
//-->
</SCRIPT><SCRIPT language="JavaScript1.1" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/1384651778@Top,Bottom"></SCRIPT>
<SCRIPT language="JavaScript">
<!--
document.write('');
function OAS_AD(pos)
{
   if (OAS_version >= 11)
      OAS_RICH(pos);
   else
      OAS_NORMAL(pos);
}
//-->
</SCRIPT>
<!------ OAS SETUP end ------>
<DIV id="dropdown" onmouseover="testMouse()" onmouseout="testMouse()">
<!------ OAS AD 'Top' begin ------>
<SCRIPT language="JavaScript">
<!--
OAS_AD('Top');
//-->
</SCRIPT><A href="http://oas.biomedcentral.com/5c/genomebiology.com/article/genomebiol/494220354/Top/biomed/house_contra_keystone09_10_Genom/KS_Genetics_728x90.gif/38316162393639363439653533343330" target="_blank"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/KS_Genetics_728x90.gif" width="728" height="90" alt="" border="0"></A><NOSCRIPT></NOSCRIPT></DIV></CENTER></TD><TD valign="top"><TABLE height="95"><TBODY><TR><TD style="vertical-align:middle"><A href="http://www.biomedcentral.com/info/mediainformation/"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/advhere.gif" border="0" alt=""></A></TD></TR></TBODY></TABLE>
<!------ OAS AD end ------>


</TD></TR></TBODY></TABLE></DIV><DIV id="innerframe"><TABLE width="100&percnt;" cellpadding="5" cellspacing="0" border="0"><TBODY><TR valign="bottom"><TD width="50"><A href="http://genomebiology.com/"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/headsquare.gif" border="0" alt="" hspace="3"></A></TD><TD id="topnav" align="right"><STRONG>Welcome </STRONG>University of Miami (<A href="http://genomebiology.com/registration/technical.asp?process=default">Log on / register</A>)<BR><A href="http://www.biomedcentral.com/">BioMed Central home</A> | <A href="http://www.biomedcentral.com/browse/journals/">Journals A-Z</A> | <A href="http://genomebiology.com/feedback.asp?referedPage=%2Frender%2Frender%2Easp%3Farx%5Fid%3D2007%2D8%2D1%2D102%26type%3D%26item%3D%26pubtype%3Dcore%26" onclick="popup(this.href,&#39;Feedback&#39;,650,500); return false;">Feedback</A> | <A href="http://genomebiology.com/support/">Support</A> | <A href="http://genomebiology.com/my/">My details</A></TD></TR></TBODY></TABLE><TABLE width="100&percnt;" cellpadding="0" cellspacing="0" border="0"><TBODY><TR valign="TOP"><TD bgcolor="#333333" colspan="3"><DIV id="TopBar"><FONT color="#ffffff">&nbsp;<A href="http://genomebiology.com/home/" class="xtoplinks">home</A> | <A href="http://genomebiology.com/commentsection/" class="xtoplinks">comment</A> | <A href="http://genomebiology.com/reviewsection/" class="xtoplinks">reviews</A>  | <A href="http://genomebiology.com/reports/" class="xtoplinks">reports</A>  | <A href="http://genomebiology.com/preprint/" class="xtoplinks">deposited research</A>  | <A href="http://genomebiology.com/researchsection/" class="xtoplinks">refereed research</A> | <A href="http://genomebiology.com/interactions/" class="xtoplinks">interactions</A> | <A href="http://genomebiology.com/supplements/" class="xtoplinks">supplements</A> | <A href="http://genomebiology.com/search" class="xtoplinks">search</A> | <A href="http://genomebiology.com/information/" class="xtoplinks">information</A> | <A href="http://genomebiology.com/my" class="xtoplinks">my journal</A> </FONT></DIV></TD></TR></TBODY></TABLE><TABLE width="160" height="14" border="0" cellpadding="3" cellspacing="1" bgcolor="#000000"><TBODY><TR><TD bgcolor="#333333" width="14" height="14"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/whitearrow.gif" width="14" height="14" border="0" alt="" align="absmiddle"></TD><TD bgcolor="#cccccc"><DIV class="midSans"><FONT color="#000000">.comment</FONT></DIV></TD></TR></TBODY></TABLE><TABLE width="100&percnt;" cellpadding="10" cellspacing="1" border="0"><TBODY><TR><TD valign="top"><TABLE align="right"><TBODY><TR valign="top"><TD><DIV id="SectionNav" class="SectionNav"><A href="http://genomebiology.com/researchnews/default.asp?Detail=less" class="SectionNav">research news</A></DIV></TD><TD class="spacer">&nbsp;|&nbsp;</TD><TD><DIV id="SectionNav" class="SectionNav"><A href="http://genomebiology.com/editorial" class="SectionNav">editorial</A></DIV></TD><TD class="spacer">&nbsp;|&nbsp;</TD><TD><DIV id="SectionNav" class="SectionNav"><A href="http://genomebiology.com/comment/" class="SectionNav">comment</A></DIV></TD><TD class="spacer">&nbsp;|&nbsp;</TD><TD><DIV id="SectionNav" class="SectionNav"><A href="http://genomebiology.com/opinion/" class="SectionNav">opinion</A><BR><A href="http://genomebiology.com/toc.asp?sort=Opinion&sortby=date&date=12-2008" class="SectionNav">sort by date</A><BR><A href="http://genomebiology.com/toc.asp?sort=Opinion&sortby=subject" class="SectionNav">sort by subject</A><BR><A href="http://genomebiology.com/toc.asp?sort=Opinion&sortby=all" class="SectionNav"> list all</A></DIV></TD><TD class="spacer">&nbsp;|&nbsp;</TD><TD><DIV id="SectionNav" class="SectionNav"><A href="http://genomebiology.com/onlinesubmission/" class="SectionNav">upload!</A></DIV></TD></TR></TBODY></TABLE><BR><BR></TD></TR><TR></TBODY></TABLE><DIV id="article"><SCRIPT src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/fulltext.js" type="text/javascript"></SCRIPT><TABLE cellpadding="0" cellspacing="0" width="100&percnt;"><TBODY valign="top"><TR><TD id="leftcolumn"><UL id="outline" style="position: static; left: 120px; ">
<LI><A href="http://genomebiology.com/2007/8/1/102#">Top</A></LI>
<LI><A href="http://genomebiology.com/2007/8/1/102#IDAEVDJE">Abstract</A></LI>
<LI><A href="http://genomebiology.com/2007/8/1/102#IDAEWDJE">Opinion</A></LI>
<LI class="long"><A href="http://genomebiology.com/2007/8/1/102#IDA5WDJE" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="What is genome annotation?">What is genom...</A></LI>
<LI class="long"><A href="http://genomebiology.com/2007/8/1/102#IDALRDJE" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="Partial and draft genomes">Partial and d...</A></LI>
<LI class="long"><A href="http://genomebiology.com/2007/8/1/102#IDANRDJE" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="The role of GenBank">The role of G...</A></LI>
<LI class="long"><A href="http://genomebiology.com/2007/8/1/102#IDAXSDJE" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="Some inconvenient truths">Some inconven...</A></LI>
<LI class="long"><A href="http://genomebiology.com/2007/8/1/102#IDAIQNCK" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="Possible solutions">Possible solu...</A></LI>
<LI class="long"><A href="http://genomebiology.com/2007/8/1/102#IDAOQNCK" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="A wiki solution?">A wiki soluti...</A></LI>
<LI class="lowest"><A href="http://genomebiology.com/2007/8/1/102#refs">References</A></LI>
</UL>
<TABLE width="130"><TBODY><TR><TD>&nbsp;</TD></TR></TBODY></TABLE></TD><TD id="content"><DIV id="navbox">
<TABLE cellpadding="5" cellspacing="2">
<TBODY><TR>
<TD><A href="http://genomebiology.com/home"><STRONG>Genome Biology</STRONG></A><BR><A href="http://genomebiology.com/2007/8">Volume 8</A><BR><A href="http://genomebiology.com/2007/8/1">Issue 1</A></TD>
</TR><TR><TD><STRONG>Viewing options:</STRONG><UL>
<LI>&nbsp;<A href="http://genomebiology.com/2007/8/1/102/abstract/">Abstract</A></LI>
<LI>&nbsp;<STRONG>Full text</STRONG></LI>
<LI>&nbsp;<A href="http://genomebiology.com/content/pdf/gb-2007-8-1-102.pdf">PDF </A>
		(706KB)
				</LI>
</UL><STRONG>Associated material:</STRONG><UL><LI>&nbsp;<A href="http://genomebiology.com/2007/8/1/102/comments">Readers&#39; comments</A></LI><LI>&nbsp;<A href="http://www.biomedcentral.com/pubmed/17274839" onclick="largepopup(this.href,&#39;pubmed&#39;);return false">PubMed record</A></LI></UL><STRONG>Related literature:</STRONG><UL><LI>&nbsp;Articles citing this article<DIV><A href="http://www.biomedcentral.com/search/citedby/17274839">on BioMed Central</A></DIV><DIV><A href="http://scholar.google.com/scholar?hl=en&lr=&safe=off&c2coff=1&q=link%3Ahttp://genomebiology.com/2007/8/1/102&btnG=Search" onclick="largepopup(this.href,&#39;on_google&#39;);return false">on Google Scholar</A></DIV><DIV><A href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=BMC_sci&SrcApp=BMC_sci&DestLinkType=FullRecord&DestApp=WOS&KeyUT=000244821100002" onclick="largepopup(this.href,&#39;on_isi&#39;);return false">on&nbsp;ISI&nbsp;Web&nbsp;of&nbsp;Science</A></DIV><DIV><A href="http://www.biomedcentral.com/search/citedby/pmc/17274839">on PubMed Central</A></DIV></LI>
<LI>&nbsp;Other articles by authors<BR><DIV><A href="javascript://" onclick="toggleauthors(&#39;authg&#39;,&#39;authbuttong&#39;); return false"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/expand2.gif" id="authbuttong" alt="">on Google Scholar
				</A></DIV>
<DIV id="authg" style="display: none; "><A href="http://scholar.google.com/scholar?q=author%3A%22SL+Salzberg%22">Salzberg SL</A></DIV>
<DIV><A href="javascript://" onclick="toggleauthors(&#39;authpm&#39;,&#39;authbuttonpm&#39;); return false"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/expand2.gif" id="authbuttonpm" alt="">on PubMed
				</A></DIV>
<DIV id="authpm" style="display: none; "><A href="http://genomebiology.com/search/results.asp?db=pm&terms=Salzberg_SL&field=AU" onclick="largepopup(this.href,&#39;pubmed&#39;,800,600);return false">Salzberg SL</A></DIV>
</LI>
<LI>&nbsp;Related articles/pages<BR><DIV><A href="http://www.google.co.uk/search?hl=en&q=related:http://genomebiology.com/2007/8/1/102">on Google</A></DIV>
<DIV><A href="http://scholar.google.com/scholar?q=related:http://genomebiology.com/2007/8/1/102">on Google Scholar</A></DIV>
<DIV><A href="http://genomebiology.com/pubmed/related/17274839">on PubMed</A></DIV>
</LI></UL><STRONG>Tools:</STRONG><UL><LI>&nbsp;<A href="http://genomebiology.com/2007/8/1/102/citation">Download citation(s)</A></LI><LI>&nbsp;<A href="http://www.biomedcentral.com/content/download/xml/gb-2007-8-1-102.xml">Download XML</A></LI><LI>&nbsp;<A href="http://genomebiology.com/2007/8/1/102/email">Email to a friend</A></LI><LI>&nbsp;<A href="http://www.biomedcentral.com/info/about/reprintsandperm">Order reprints</A></LI><LI>&nbsp;<A href="http://genomebiology.com/2007/8/1/102/postcomment">Post a comment</A></LI>
<LI>&nbsp;<A href="http://genomebiology.com/alerts/">Sign up for article alerts</A></LI></UL><SCRIPT type="text/javascript">toggleauthors('authpm','authbuttonpm'); toggleauthors('authg','authbuttong'); toggleauthors('authos','authbuttonos'); toggleauthors('autha','authbuttona');</SCRIPT><STRONG>Post to:</STRONG><BR><UL class="iconlist">
<LI><A href="http://www.citeulike.org/posturl?url=http://genomebiology.com/2007/8/1/102" onclick="largepopup(this.href,&#39;citeulike_popup_post&#39;,800,600);return false"><IMG alt="" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/citeulink.gif"></A>&nbsp;<A href="http://www.citeulike.org/posturl?url=http://genomebiology.com/2007/8/1/102" onclick="largepopup(this.href,&#39;citeulike_popup_post&#39;,800,600);return false">Citeulike				
				</A></LI>
<LI><A onclick="largepopup(this.href,&#39;Connotea&#39;,800,600);return false" href="http://www.connotea.org/addpopup?continue=confirm&uri=http://dx.doi.org/doi:10.1186/gb-2007-8-1-102"><IMG alt="" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/connotea.gif"></A>&nbsp;<A href="http://www.connotea.org/addpopup?continue=confirm&uri=http://dx.doi.org/doi:10.1186/gb-2007-8-1-102" onclick="largepopup(this.href,&#39;Connotea&#39;,800,600);return false">Connotea
					</A></LI>
<LI><A href="http://del.icio.us/post?url=http://genomebiology.com/2007/8/1/102&title=Genome%20re-annotation:%20a%20wiki%20solution?" onclick="largepopup(this.href,&#39;Del.icio.us&#39;,800,600);return false"><IMG alt="" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/delicious.gif"></A>&nbsp;<A href="http://del.icio.us/post?url=http://genomebiology.com/2007/8/1/102&title=Genome%20re-annotation:%20a%20wiki%20solution?" onclick="largepopup(this.href,&#39;Del.icio.us&#39;,800,600);return false">Del.icio.us</A></LI>
<LI><A href="http://digg.com/submit?phase=2&url=http://genomebiology.com/2007/8/1/102&title=Genome%20re-annotation:%20a%20wiki%20solution?&topic=general_sciences" onclick="largepopup(this.href,&#39;Digg&#39;,800,600);return false"><IMG alt="" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/digg.gif"></A>&nbsp;<A href="http://digg.com/submit?phase=2&url=http://genomebiology.com/2007/8/1/102&title=Genome%20re-annotation:%20a%20wiki%20solution?&topic=general_sciences" onclick="largepopup(this.href,&#39;Digg&#39;,800,600);return false">Digg
				</A></LI>
<LI><A href="http://www.facebook.com/sharer.php?u=http://genomebiology.com/2007/8/1/102" onclick="largepopup(this.href,&#39;facebook&#39;,800,600);return false"><IMG alt="" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/facebook.gif"></A>&nbsp;<A href="http://www.facebook.com/sharer.php?u=http://genomebiology.com/2007/8/1/102" onclick="largepopup(this.href,&#39;facebook&#39;,800,600);return false">Facebook</A></LI>
</UL></TD></TR></TBODY></TABLE>
</DIV><A name="maincontent"></A><DIV id="topmatter"><A name="text"></A><A href="http://www.biomedcentral.com/info/about/mostviewed/"><IMG class="access" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/highlyaccessed-large.gif" width="124" height="24" alt="Highly Access"></A><!-- <rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns="http://purl.org/rss/1.0/">
<item rdf:about="http://genomebiology.com/2007/8/1/102">
<title>Genome re-annotation: a wiki solution?</title>
<dc:title>Genome re-annotation: a wiki solution?</dc:title>
<dc:creator>Salzberg, Steven L</dc:creator>
<dc:identifier>info:doi/10.1186/gb-2007-8-1-102</dc:identifier>
<dc:identifier>info:pmid/17274839</dc:identifier>
<dc:source>Genome Biology 2007, 8:102</dc:source>
<dc:date>2007-02-01</dc:date>
<prism:publicationName>Genome Biology</prism:publicationName>
<prism:publicationDate>2007-02-01</prism:publicationDate>
<prism:volume>8</prism:volume>
<prism:number>1</prism:number>
<prism:section>Opinion</prism:section>
<prism:startingPage>102</prism:startingPage>
</item>
</rdf:RDF> --><SPAN class="articletype">Opinion</SPAN><H1>Genome re-annotation: a wiki solution?</H1><DIV class="singleins"><P class="authors"><STRONG>Steven L Salzberg</STRONG> <A href="http://genomebiology.com/logon/logon.asp?msg=ce"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/email-ca.gif" title="Corresponding author" alt="email"></A></P><P>Center for Bioinformatics and Computational Biology and Department of Computer Science, 3125 Biomolecular Sciences Building, University of Maryland, College Park, MD 20742, USA</P></DIV><P><SPAN class="pseudotabau"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/email.gif" border="0"> author email</SPAN><SPAN class="pseudotabau"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/email-ca(1).gif"> corresponding author email</SPAN></P><P><EM>Genome Biology</EM> 2007,
			 <STRONG>8</STRONG><STRONG>:</STRONG>102<SPAN class="pseudotab">doi:10.1186/gb-2007-8-1-102</SPAN></P><P><STRONG>Subject areas: </STRONG>Bioinformatics, Genome studies, Microbiology and parasitology, Model organisms, Methods</P>
<P>The electronic version of this article is the complete one and can be found online at: <A href="./Salzberg 2007 Genome re-annotation  a wiki solution_files/Salzberg 2007 Genome re-annotation  a wiki solution.htm">http://genomebiology.com/2007/8/1/102</A></P>
<TABLE cellpadding="0" cellspacing="0">
<TBODY>
<TR>
<TD>Published:</TD>
<TD>1&nbsp;February&nbsp;2007</TD>
</TR>
</TBODY>
</TABLE>
<P>
						&copy;
						 2007 BioMed Central Ltd</P></DIV><A name="IDAEVDJE"></A><H3>Abstract</H3><P>The annotation of most genomes becomes outdated over time, owing in part to our ever-improving knowledge of genomes and in part to improvements in bioinformatics software. Unfortunately, annotation is rarely if ever updated and resources to support routine reannotation are scarce. Wiki software, which would allow many scientists to edit each genome&#39;s annotation, offers one possible solution.</P><A name="IDAEWDJE"></A><H3>Opinion</H3><P>So you think that gene you just retrieved from GenBank <A name="IDAKWDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B1">1</A>] is correct? Are you certain? If it is a eukaryotic gene, and especially if it is from an unfinished genome, there is a pretty good chance that the amino acid sequence is wrong. And depending on when the genome was sequenced and annotated, there is a chance that the description of its function is wrong too.</P>
<P>Large-scale genome sequencing has revolutionized biology over the past ten years, generating vast amounts of new information that has radically transformed our understanding of hundreds of species, including ourselves. Sequencing centers continue to churn out new DNA sequences for a fantastic variety of species, covering more and more of the tree of life. Along with these sequences, the centers also produce genome annotation, which includes the locations and descriptions of all identifiable genes. These gene lists are the first pictures we get of what&#39;s inside a newly sequenced genome, and they can reveal key insights into what makes an organism distinctive. Sometimes the gene lists themselves are part of the story; for example, when the human genome was published <A name="IDARWDJE"></A><A name="IDAUWDJE"></A>[<A onclick="LoadInParent(&#39;#B2&#39;); return false;" href="http://genomebiology.com/2007/8/1/102#B2">2</A>,<A href="http://genomebiology.com/2007/8/1/102#B3">3</A>], the headline was that humans have &#39;only&#39; 25,000 genes, in contrast to earlier estimates of 100,000 or more. For many microbial species, the genome helps us to understand how the organism can accomplish something particularly difficult, such as how <EM>Deinococcus radiodurans </EM>(to cite just one of many examples) can withstand exposure to radiation levels far in excess of what a human could tolerate <A name="IDA1WDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B4">4</A>]. With each new human pathogen, the gene list helps us determine how the organism infects humans, how it causes sickness and (sometimes) how it becomes resistant to antibiotics. For these and other reasons, the accuracy of the gene list is tremendously important.</P><A name="IDA5WDJE"></A><H3>What is genome annotation?</H3><P>Before addressing the problems with annotation, I will first summarize how it is done. The process of sequencing and annotating the DNA of a bacterial species has become highly automated in recent years, but the major steps are quite similar to what was done for the very first bacterial genome, <EM>Haemophilus influenzae</EM>, in 1995 <A name="IDAHXDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B5">5</A>].</P>
<P>Figure <A name="IDANXDJE"></A><A onclick="popup(&#39;/2007/8/1/102/figure/F1&#39;,&#39;&#39;,800,470); return false;" href="http://genomebiology.com/2007/8/1/102/figure/F1">1</A> shows an outline of the main steps of the whole-genome shotgun sequencing and annotation process for a bacterial genome. Similar procedures - for both sequencing and annotation - are followed for much larger genomes, including the human genome, although the details vary. The laboratory steps have not changed greatly since <EM>H. influenzae</EM>: they begin with DNA purification, followed by shearing the DNA into countless small fragments (the &#39;shotgun&#39; step). These fragments are then cloned and sequenced from both ends and assembled, usually resulting in a set of contiguous DNA sequences (contigs) joined together into larger scaffolds. The annotation pipeline can be applied immediately to these contigs, but in projects where the genome will be finished, the annotation software is usually run later, when the gaps between contigs have been filled in.</P><DIV class="figs"><DIV class="fig"><P><A onclick="popup(&#39;/2007/8/1/102/figure/F1&#39;,&#39;F1&#39;,800,470); return false;" href="http://genomebiology.com/2007/8/1/102/figure/F1"><IMG align="top" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/gb-2007-8-1-102-1.gif" alt="thumbnail" class="thumbnail"><STRONG>Figure 1.</STRONG></A> Overview of sequencing and annotation for a whole-genome shotgun project, for example, sequencing a bacterial genome. First <STRONG>(a)</STRONG>, genomic DNA is purified, broken into short fragments and cloned into <EM>E. coli</EM>. The cloned fragments are then sequenced from both ends on an automated sequencing machine. The resulting sequences (shown in <STRONG>(b) </STRONG>as they appear on the sequencing machine display) are then assembled using a complex software program that identifies overlaps into <STRONG>(c) </STRONG>large, contiguous sequences representing the chromosomes from the original DNA. Gaps are filled until the genome is complete. <STRONG>(d) </STRONG>Annotation begins with the execution of several gene-finding programs, such as Glimmer, which identifies protein-coding genes, tRNAScan, which identifies tRNAs, and other programs for other genome features. <STRONG>(e) </STRONG>These initial predictions are used as the basis for BLAST searches against large protein databases, which identify related proteins based on sequence similarity. Translated (Blastx) searches are then used to scan the databases to detect any proteins that match the DNA regions in between predicted genes. Customized annotation programs are used to decide what name and function to assign to each protein, leading to <STRONG>(f) </STRONG>the final annotated genome.</P></DIV></DIV><P>Most annotation pipelines are considerably more complex than shown in Figure <A name="IDAUQDJE"></A><A onclick="popup(&#39;/2007/8/1/102/figure/F1&#39;,&#39;&#39;,800,470); return false;" href="http://genomebiology.com/2007/8/1/102/figure/F1">1</A>, but they share the same outline. First a gene finder (such as, for bacteria, Glimmer <A name="IDAZQDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B6">6</A>] or GeneMark <A name="IDA1QDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B7">7</A>]) is run over the genome, producing a set of predicted protein-coding genes. These programs are very accurate, though not perfect. They are far more accurate than eukaryotic gene finders, however, primarily because the problem is far more difficult in eukaryotic genomes. In either case, the next step in the pipeline is to take the set of predictions and search them against one or more protein databases using BLAST <A name="IDADRDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B8">8</A>], HMMer <A name="IDAHRDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B9">9</A>] or other programs. For each gene that has a significant match, the BLAST output can be used to assign a name and function to the protein. The accuracy of this step depends not only on the annotation software, but also on the quality of the annotations already in the database. For genes with no match, the pipeline might keep them and label them as &#39;hypothetical&#39;, or it might discard them based on criteria as simple as minimum length.</P>
<P>Annotation pipelines also run separate searches for tRNA and rRNA genes, and they may include other components as well. The pipeline software will usually take extra steps to find any genes missed by earlier steps; typically this involves running a translated search, aligning all six possible translations of the unannotated sections to a database.</P><A name="IDALRDJE"></A><H3>Partial and draft genomes</H3><P>Finishing a genome - sequencing every remaining nucleotide of every chromosome and creating a gap-free assembly - is considerably slower and more expensive than the high-throughput shotgun-sequencing phase. As a result, a growing number of genomes are being released in &#39;draft&#39; form and will remain in this form indefinitely. These include many bacteria and the majority of eukaryotic genomes. (In fact, only a handful of eukaryotic genomes, such as those of <EM>Saccharomyces cerevisiae </EM>and <EM>Caenorhabditis elegans </EM>but not including the human, are truly finished.)</P>
<P>The effect of draft genomes upon annotation is considerable: many genes will &#39;run off&#39; the end of contigs or appear on two or more separate contigs. This in turn complicates the subsequent steps of annotation and is likely to lead to additional errors in assigning gene function. For example, a gene fragment is liable to match a small protein domain, and functions based on a single domain hit are not reliable. A gene that is split across two contigs might be annotated twice. Draft sequences also have much higher sequencing error rates, which can introduce erroneous stop codons in the middle of genes or improperly merge adjacent but distinct genes.</P><A name="IDANRDJE"></A><H3>The role of GenBank</H3><P>Once a genome - draft or complete - is annotated, the DNA sequence along with the annotation is normally deposited in GenBank. Countless researchers rely on GenBank <A name="IDA3RDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B1">1</A>], EMBL <A name="IDACSDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B10">10</A>] and DDBJ <A name="IDAESDJE"></A>[<A href="http://genomebiology.com/2007/8/1/102#B11">11</A>] (which mirror one another) as their primary source for genome annotation, and for a good reason: these databases are the world&#39;s largest public repositories of genome information. GenBank now contains over 65 billion base pairs (Gbp) of sequence, up from just 2 Gbp in 1998 and 10 Gbp in 2000, and it continues to grow at an astonishing rate. If you want to find a gene, GenBank should definitely be your first stop. Yet I frequently hear claims within the bioinformatics community that the &#39;GenBank annotation&#39; of a particular genome is fraught with problems, and that the speaker can fix them.</P>
<P>Is the GenBank annotation perfect? Of course not. How good it is, though, depends on many variables, and the consumer of GenBank data would be wise to be aware of them (in other words, <EM>caveat emptor</EM>). The first and most important point to understand is that GenBank is not simply a database; it is also a library. A scientist who submits a sequence to GenBank is the owner of that sequence and is listed on the &#39;author&#39; line in the GenBank entry. Just as with any article published in a journal, the author (and only the author) has the right to submit an erratum. Because GenBank is an electronic library, an erratum is really an update: new sequences or annotations replace the old ones, although GenBank keeps a record of the changes so that the original entry can still be retrieved if necessary. This notion of GenBank as a library (or an electronic journal) is frequently misunderstood, especially when a scientist discovers an annotation error. Even if the error is overwhelmingly obvious, the custodians of GenBank cannot simply fix it, any more than the editor of a journal can correct one of the papers published in that journal. Another way to think of this is to recognize that a &#39;GenBank annotation&#39; is not &#39;GenBank&#39;s&#39; annotation, but rather the annotation of whoever deposited the sequence in the first place.</P>
<P>When confronted with this problem, some scientists react by suggesting that GenBank (and DDBJ and EMBL) should allow scientists to fix errors that they find. But this would quickly destroy the archival function of GenBank, as original entries would be erased over time. It would also violate the agreement that GenBank has with all its submitters that their entries belong to them and can only be changed by them. This agreement has been crucial in GenBank&#39;s near-universal acceptance by the genomics community as the central resource for DNA sequences. The idea of allowing others to alter GenBank annotation also immediately begs the question of who should be permitted to make such alterations.</P>
<P>This leaves us with a problem: users go to GenBank expecting to find the authoritative annotation for a genome, and what they find might be far less than that. Most genome annotation deposited in GenBank remains static for years, and many annotations have never been changed since their initial publication. Nonetheless, many scientists assume that GenBank annotation is kept up to date, and they are surprised to hear that it is not.</P>
<P>For example, 479 genes in the <EM>H. influenzae </EM>Rd genome are currently listed as hypothetical proteins. Of these, 217 have at least one extremely strong BLAST hit to another species (<EM>E</EM>-value &lt; 10-100), which means they should at least be called &#39;conserved hypothetical&#39; proteins. And 40 of these have matches to a gene with an assigned function, meaning that a re-annotation would result in these genes having a more meaningful name than &#39;hypothetical protein&#39;.</P><A name="IDAXSDJE"></A><H3>Some inconvenient truths</H3><P>Even considering all of the issues above, one might reasonably expect that as protein databases have grown, annotation has improved and that recently annotated genomes (at least) will be of the highest quality. This is not quite true. What is true is that a BLAST search of a protein that is run today will yield far more results than it would have five or ten years ago, and these results in turn should lead to better annotation. Not all software is equally good, however, and the annotation pipelines vary considerably in their quality. There is also wide variation in the skills and experience of those operating the pipelines. Further complicating matters, some genomes are subjected to careful curation and review, whereas others receive only automated annotation. In the early days of sequencing, the sequencing teams included experts on the biology of each genome, and their manual curation dramatically improved the annotation of those species. Today that is no longer true: high-throughput sequencing centers are large, efficient factories with unique expertise in the methods necessary for sequencing, but they sometimes have very little expertise on the biology of the species they are sequencing. The inconvenient truth is that, as a result of these factors and others, some genomes are poorly annotated even today.</P>
<P>There are several ways in which genome annotation can be erroneous. The first and most fundamental is simply that the gene models may be wrong. Although bacterial gene-finding systems <A name="IDA2SDJE"></A><A name="IDACTDJE"></A>[<A onclick="LoadInParent(&#39;#B6&#39;); return false;" href="http://genomebiology.com/2007/8/1/102#B6">6</A>,<A href="http://genomebiology.com/2007/8/1/102#B7">7</A>] are highly accurate, finding 98-99&percnt; of protein-coding genes in most species, they still occasionally miss genes. Their accuracy at placing the start site is a bit lower, probably closer to 90&percnt;, which is excellent but far from the perfect accuracy that some might expect. In the past, the accuracy of (bacterial) start-site prediction was closer to 80&percnt;, and many of the genomes in GenBank were predicted with earlier versions of gene finders. Note that all these accuracy figures are much lower for eukaryotic annotation. Some annotation pipelines include algorithms to adjust start sites, which can be done by looking closely at the boundaries of alignments to homologous proteins.</P>
<P>False positives represent another type of erroneous annotation: when the prediction of a gene-finding program does not match any previously known protein, the annotators (or the annotation pipeline software) must decide whether or not to include that prediction in the gene list. Over the years, annotation groups have used a variety of rules to make this decision, and they have inevitably included thousands of false predictions in the publicly available genome annotation. These predictions are mostly harmless unless they result in effort being expended trying to verify them. In some cases, too, they might &#39;hide&#39; functional RNA genes or true genes in a different reading frame from that of the false prediction.</P>
<P>Perhaps the biggest problem with genome annotation is erroneous and inconsistent naming of genes. Much of this is due to the simple fact that our knowledge of genes has improved but the annotation has remained static. Thus a gene labeled &#39;hypothetical protein&#39; a few years ago might now have a known function. A second problem is what&#39;s known as transitive catastrophe: the phenomenon whereby a name is transferred from one gene to another on the basis of sequence similarity (usually from a BLAST search) but where the original name is incorrect. As more genomes are annotated, and more BLAST searches are run, the name gets transferred to other proteins, and the original source of the name quickly becomes lost. It is well known in the genomics community that thousands of such transitive errors have propagated through sequence databases, and efforts are under way to try to clean up some of the mess. In the meantime, though, many genes remain incorrectly annotated.</P>
<P>Let us consider just one example, selected more or less at random from the bacterium <EM>H. influenzae </EM>Rd <A name="IDADQNCK"></A>[<A href="http://genomebiology.com/2007/8/1/102#B4">4</A>]. The gene <EM>fdxH </EM>encodes formate dehydrogenase, <SPAN class="entity">&beta;</SPAN> subunit, GenBank accession number NP438180. When the genome was sequenced in 1995, this gene (encoding a 312 amino acid protein) was similar to very few other genes; even the orthologous <EM>Escherichia coli </EM>gene was not yet sequenced. It is very difficult today to reconstruct what the best BLAST hit was back then, but today there are 197 highly significant BLAST hits to 123 distinct species. Thus, it is pretty clear that this gene today should be well-annotated because of the multitude of highly similar proteins. Yet if we look at the list of matching proteins, we find a variety of names given, including not only the name found on NP438180 itself, but also: formate dehydrogenase-O <SPAN class="entity">&beta;</SPAN> subunit; formate dehydro-genase, nitrate-inducible, iron-sulfur subunit; HybA protein; formate dehydrogenase-N, Fe-S <SPAN class="entity">&beta;</SPAN> subunit, nitrate-inducible; hypothetical protein PaerPA_01004979; hypothetical protein Bpse11_03005113; 4Fe-4S ferredoxin, iron-sulfur binding; and Twin-arginine translocation pathway signal. Some of these names seem to be synonymous, but others clearly are not. To decide properly among them, we need to look at the source of each annotation and at the species to which it is attached.</P><A name="IDAIQNCK"></A><H3>Possible solutions</H3><P>So if we can&#39;t always trust GenBank, what can we do? Clearly we cannot just ignore it. The scientific community must have a resource that contains the genes from all the species that have been sequenced. For the past 25 years, GenBank, EMBL and DDBJ have been enormously successful at providing these data. The pace of sequencing has changed the rules of the game, however: sequencing centers are pouring out genomes, annotating them rapidly and moving on. An archive of these annotations may be useful, but a static archive is insufficient.</P>
<P>One part of the solution is obvious: annotation must be regularly re-computed using the latest databases and software. For a small number of model organisms, this is already happening, but these species represent a tiny proportion of all known genes. Simply re-running an automated pipeline on all genomes is not sufficient, though, because that would over-write many of the carefully curated, manually annotated genes that have been produced in the past. Unfortunately, there is no standard label attached to such genes, so there is no way for an automated pipeline to know that they should be trusted. Therefore, we also need to launch an effort to start identifying those genes that are well annotated and, beyond that, to start recording the evidence used to annotate each gene.</P>
<P>Another solution is to create a new, expanded database that can display all the alternative annotations for any locus in a genome. If this were available, then scientists could be provided with links from any gene to alternative or overlapping gene predictions as well as alternative gene names. Along with each annotation could be a link to the evidence supporting it; for example, the date of a BLAST search or a citation to experiments contained in a journal article.</P><A name="IDAOQNCK"></A><H3>A wiki solution?</H3><P>Various members of the genomics community have considered these and other solutions, but so far none have emerged as the standard. Several new databases have been developed with alternative genome annotation, or with re-annotation (for example, the TIGR Comprehensive Microbial Resource <A name="IDAVQNCK"></A>[<A href="http://genomebiology.com/2007/8/1/102#B12">12</A>]), but none of them has attracted nearly as much web traffic as GenBank or the other databases at NCBI <A name="IDA3QNCK"></A>[<A href="http://genomebiology.com/2007/8/1/102#B13">13</A>]. The difficulties in changing this system are many: first, for example, there are some genomes for which GenBank is still the best source, and second, if another, better source of annotation exists, how is someone to discover it?</P>
<P>A relatively new model of sharing expertise through the Internet might offer a solution. This model is the &#39;wiki&#39;: a shared resource that anyone can edit. This open-editing framework for websites and data was first introduced in 1995, and it was initially viewed with skepticism by many in the Internet community, who argued that wiki-based websites would be filled with unreliable, inaccurate information. But the success of the online encyclopedia Wikipedia <A name="IDAERNCK"></A>[<A href="http://genomebiology.com/2007/8/1/102#B14">14</A>] has demonstrated that, despite the skeptics, a wiki site can be accurate, up-to-date and incredibly useful. Genome annotation has many of the same features of an encyclopedia: the information required to produce it is broad-based and the expertise is scattered around the scientific community in a very wide range of laboratories, most of whom are not connected to genome projects. I therefore propose that a &#39;genome wiki&#39; might provide just the solution we need for genome annotation. A wiki would allow the community of experts to work out the best name for each gene, to indicate uncertainty where appropriate and to discuss alternative annotations. Although wikis will not (and should not) supplant well-curated model-organism databases, for the majority of species they might represent our best chance for creating accurate, up-to-date genome annotation.</P>
<P>Whether or not a genome wiki emerges, we will probably need an archival repository of annotation for many years to come. The international database consortium represented by GenBank, EMBL and DDBJ has served that purpose remarkably well for a long time and will continue to do so. Despite this success, the genomics community needs an accurate, continually updated source of genome annotation for every species, and we can hope that a solution to this problem will emerge in the near future.</P><A name="refs"></A><H3>References</H3><OL id="references">
<LI id="B1">
<P><A name="B1"></A><STRONG> GenBank </STRONG>
		[<A target="_blank" href="http://www.ncbi.nlm.nih.gov/Genbank/">http://www.ncbi.nlm.nih.gov/Genbank/</A>]
			
		 <A href="http://www.webcitation.org/query.php?url=http://www.ncbi.nlm.nih.gov/Genbank/&refdoi=10.1186/gb-2007-8-1-102" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</A></P><P> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B1" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B1&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B2">
<P><A name="B2"></A>The International Human Genome Sequencing Consortium: <STRONG> Initial sequencing and analysis of the human genome. </STRONG></P><P><EM>Nature</EM> 2001,  <STRONG>409</STRONG><STRONG>:</STRONG>860-921. <A target="_blank" href="http://genomebiology.com/pubmed/11237011">PubMed Abstract</A> | <A target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&cmd=prlinks&retmode=ref&id=11237011">Publisher Full Text</A> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B2" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B2&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B3">
<P><A name="B3"></A> Venter JC,  Adams MD,  Myers EW,  Li PW,  Mural RJ,  Sutton GG,  Smith HO,  Yandell M,  Evans CA,  Holt RA, <EM>et al</EM>.: <STRONG> The sequence of the human genome. </STRONG></P><P><EM>Science</EM> 2001,  <STRONG>291</STRONG><STRONG>:</STRONG>1304-1351. <A target="_blank" href="http://genomebiology.com/pubmed/11181995">PubMed Abstract</A> | <A target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&cmd=prlinks&retmode=ref&id=11181995">Publisher Full Text</A> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B3" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B3&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B4">
<P><A name="B4"></A> White O,  Eisen JA,  Heidelberg JF,  Hickey EK,  Peterson JD,  Dodson RJ,  Haft DH,  Gwinn ML,  Nelson WC,  Richardson DL, <EM>et al</EM>.: <STRONG> Genome sequence of the radioresistant bacterium <EM>Deinococcus radiodurans </EM>R1. </STRONG></P><P><EM>Science</EM> 1999,  <STRONG>286</STRONG><STRONG>:</STRONG>1571-1577. <A target="_blank" href="http://genomebiology.com/pubmed/10567266">PubMed Abstract</A> | <A target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&cmd=prlinks&retmode=ref&id=10567266">Publisher Full Text</A> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B4" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B4&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B5">
<P><A name="B5"></A> Fleischmann RD,  Adams MD,  White O,  Clayton RA,  Kirkness EF,  Kerlavage AR,  Bult CJ,  Tomb JF,  Dougherty BA,  Merrick JM, <EM>et al</EM>.: <STRONG> Whole-genome random sequencing and assembly of <EM>Haemophilus influenzae </EM>Rd. </STRONG></P><P><EM>Science</EM> 1995,  <STRONG>269</STRONG><STRONG>:</STRONG>496-512. <A target="_blank" href="http://genomebiology.com/pubmed/7542800">PubMed Abstract</A> | <A target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&cmd=prlinks&retmode=ref&id=7542800">Publisher Full Text</A> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B5" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B5&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B6">
<P><A name="B6"></A> Delcher AL,  Harmon D,  Kasif S,  White O,  Salzberg SL: <STRONG> Improved microbial gene identification with GLIMMER. </STRONG></P><P><EM>Nucleic Acids Res</EM> 1999,  <STRONG>27</STRONG><STRONG>:</STRONG>4636-4641. <A target="_blank" href="http://genomebiology.com/pubmed/10556321">PubMed Abstract</A> | <A target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&cmd=prlinks&retmode=ref&id=10556321">Publisher Full Text</A> | <A target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=10556321">PubMed Central Full Text</A> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B6" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B6&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B7">
<P><A name="B7"></A> Lukashin AV,  Borodovsky M: <STRONG> GeneMark.hmm: new solutions for gene finding. </STRONG></P><P><EM>Nucleic Acids Res</EM> 1998,  <STRONG>26</STRONG><STRONG>:</STRONG>1107-1115. <A target="_blank" href="http://genomebiology.com/pubmed/9461475">PubMed Abstract</A> | <A target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&cmd=prlinks&retmode=ref&id=9461475">Publisher Full Text</A> | <A target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=9461475">PubMed Central Full Text</A> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B7" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B7&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B8">
<P><A name="B8"></A> Altschul SF,  Madden TL,  Schaffer AA,  Zhang J,  Zhang Z,  Miller W,  Lipman DJ: <STRONG> Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. </STRONG></P><P><EM>Nucleic Acids Res</EM> 1997,  <STRONG>25</STRONG><STRONG>:</STRONG>3389-3402. <A target="_blank" href="http://genomebiology.com/pubmed/9254694">PubMed Abstract</A> | <A target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&cmd=prlinks&retmode=ref&id=9254694">Publisher Full Text</A> | <A target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=9254694">PubMed Central Full Text</A> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B8" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B8&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B9">
<P><A name="B9"></A> Eddy SR: <STRONG> Profile hidden Markov models. </STRONG></P><P><EM>Bioinformatics</EM> 1998,  <STRONG>14</STRONG><STRONG>:</STRONG>755-763. <A target="_blank" href="http://genomebiology.com/pubmed/9918945">PubMed Abstract</A> | <A target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&cmd=prlinks&retmode=ref&id=9918945">Publisher Full Text</A> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B9" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B9&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B10">
<P><A name="B10"></A><STRONG> EMBL Nucleotide Sequence Database </STRONG>
		[<A target="_blank" href="http://www.ebi.ac.uk/embl/">http://www.ebi.ac.uk/embl/</A>]
			
		 <A href="http://www.webcitation.org/query.php?url=http://www.ebi.ac.uk/embl/&refdoi=10.1186/gb-2007-8-1-102" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</A></P><P> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B10" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B10&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B11">
<P><A name="B11"></A><STRONG> DNA Data Bank of Japan </STRONG>
		[<A target="_blank" href="http://www.ddbj.nig.ac.jp/">http://www.ddbj.nig.ac.jp/</A>]
			
		 <A href="http://www.webcitation.org/query.php?url=http://www.ddbj.nig.ac.jp/&refdoi=10.1186/gb-2007-8-1-102" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</A></P><P> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B11" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B11&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B12">
<P><A name="B12"></A> Peterson JD,  Umayam LA,  Dickinson T,  Hickey EK,  White O: <STRONG> The Comprehensive Microbial Resource. </STRONG></P><P><EM>Nucleic Acids Res</EM> 2001,  <STRONG>29</STRONG><STRONG>:</STRONG>123-125. <A target="_blank" href="http://genomebiology.com/pubmed/11125067">PubMed Abstract</A> | <A target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&cmd=prlinks&retmode=ref&id=11125067">Publisher Full Text</A> | <A target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=11125067">PubMed Central Full Text</A> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B12" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B12&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B13">
<P><A name="B13"></A><STRONG> National Center for Biotechnology Information </STRONG>
		[<A target="_blank" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</A>]
			
		 <A href="http://www.webcitation.org/query.php?url=http://www.ncbi.nlm.nih.gov/&refdoi=10.1186/gb-2007-8-1-102" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</A></P><P> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B13" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B13&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
<LI id="B14">
<P><A name="B14"></A><STRONG> Wikipedia </STRONG>
		[<A target="_blank" href="http://www.wikipedia.org/">http://www.wikipedia.org</A>]
			
		 <A href="http://www.webcitation.org/query.php?url=http://www.wikipedia.org&refdoi=10.1186/gb-2007-8-1-102" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</A></P><P> <A href="http://genomebiology.com/sfx_links.asp?ui=gb-2007-8-1-102&bibl=B14" onclick="popup(&#39;/sfx_links.asp?ui=gb-2007-8-1-102&amp;bibl=B14&#39;,&#39;SFXMenu&#39;,&#39;460&#39;,&#39;420&#39;); return false;"><IMG src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/sfx_links.asp" alt="OpenURL"></A></P><P class="totext"><SCRIPT type="text/javascript">totext()</SCRIPT><A href="javascript:turn();">Return to text</A></P>
<P>
</LI>
</OL>
<H4 class="bottommessage"><A href="http://genomebiology.com/2007/8/1/102/postcomment">Have something to say? Post a comment on this article!</A></H4><SCRIPT type="text/javascript">initializeOutline();</SCRIPT></TD></TR></TBODY></TABLE></DIV><HR size="1" noshade="" color="#cccccc">
<TABLE width="100&percnt;" cellpadding="3"><TBODY><TR valign="middle"><TD align="center">


<CENTER>

<!------ OAS AD 'Bottom' begin ------>
<SCRIPT language="JavaScript">
<!--
OAS_AD('Bottom');
//-->
</SCRIPT><SCRIPT type="text/javascript"><!--
google_ad_client = "pub-8287243681099367";
/* 728x90, created 3/5/09 */
google_ad_slot = "2716523086";
google_ad_width = 728;
google_ad_height = 90;
//-->
</SCRIPT>
<SCRIPT type="text/javascript" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/show_ads.js">
</SCRIPT><SCRIPT src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/expansion_embed.js"></SCRIPT><SCRIPT src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/test_domain.js"></SCRIPT><SCRIPT>window.google_render_ad();</SCRIPT><INS style="display:inline-table;border:none;height:90px;margin:0;padding:0;position:relative;visibility:visible;width:728px"><INS style="display:block;border:none;height:90px;margin:0;padding:0;position:relative;visibility:visible;width:728px"><IFRAME allowtransparency="true" frameborder="0" height="90" hspace="0" id="google_ads_frame1" marginheight="0" marginwidth="0" name="google_ads_frame" scrolling="no" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/ads.htm" style="left:0;position:absolute;top:0" vspace="0" width="728"></IFRAME></INS></INS><NOSCRIPT></NOSCRIPT>
<!------ OAS AD end ------>
</CENTER>

</TD></TR>
<TR><TD align="center"><DIV class="smalltext">
<A href="http://genomebiology.com/info/terms">Terms and Conditions</A> 
<A href="http://genomebiology.com/info/privacy">Privacy statement</A> 
<A href="http://www.biomedcentral.com/info/mediainformation/">Information for advertisers</A> 
<A href="http://www.biomedcentral.com/info/about/bmcjobs">Jobs at BMC</A> 
<A href="mailto:info@genomebiology.com">Contact us</A> 
</DIV>
</TD></TR></TBODY></TABLE>
<TABLE width="100&percnt;" cellpadding="0" cellspacing="0" border="0" class="footer2t">
<TBODY><TR>
<TD bgcolor="#333333" colspan="3" align="center">
&copy; 1999-2009 BioMed Central Ltd unless otherwise stated.
 Part of <A href="http://www.springer.com/">Springer Science+Business Media</A>.
<SCRIPT src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/urchin.js" type="text/javascript">
</SCRIPT>
<SCRIPT type="text/javascript">_uacct = 'UA-72326-7'; urchinTracker();
</SCRIPT>
</TD>
</TR>
</TBODY></TABLE>
</DIV>
</DIV>
</CENTER><IFRAME style="position: absolute; top: -10px; left: 10px; width: 1px; height: 1px; " name="googlexpc_K1A0FR5jkF_msg" id="googlexpc_K1A0FR5jkF_msg" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/robots.htm"></IFRAME><IFRAME style="position: absolute; top: -10px; left: 10px; width: 1px; height: 1px; " name="googlexpc_K1A0FR5jkF_ack" id="googlexpc_K1A0FR5jkF_ack" src="./Salzberg 2007 Genome re-annotation  a wiki solution_files/robots.htm"></IFRAME>


</BODY></HTML>