<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN">
<html xmlns:m="http://www.w3.org/1998/Math/MathML"><head><title>BioMed Central | Full text | Pegasys: software for executing and integrating analyses of biological sequences</title><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

<meta name="citation_pdf_url" content="http://www.biomedcentral.com/content/pdf/1471-2105-5-40.pdf">

<meta name="citation_abstract_html_url" content="http://www.biomedcentral.com/1471-2105/5/40/abstract/">
<meta name="citation_fulltext_html_url" content="http://www.biomedcentral.com/1471-2105/5/40">

<script src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/viewingoptions.js" type="text/javascript"></script><link rel="search" type="application/opensearchdescription+xml" title="BioMed Central Search" href="http://www.biomedcentral.com/download/toolbar/firefox/bmc.xml">
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script><script src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/ga.js" type="text/javascript"></script>
<script type="text/javascript">
var pageTracker = _gat._getTracker("UA-72326-1");
pageTracker._setLocalRemoteServerMode(); pageTracker._initData();
pageTracker._trackPageview();
</script>
<link rel="stylesheet" type="text/css" href="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/screen.css" title="Default Styles"><!--[if gte IE 7]><link rel="stylesheet" type="text/css" href="/stylesheets/ie7.css" /><![endif]--><!--[if lte IE 6]><link rel="stylesheet" type="text/css" href="/stylesheets/ie56.css" /><![endif]-->

<meta name="dc.title" content="Pegasys: software for executing and integrating analyses of biological sequences">
<meta name="dc.creator" content="Sohrab Shah">
<meta name="dc.creator" content="David He">
<meta name="dc.creator" content="Jessica Sawkins">
<meta name="dc.creator" content="Jeffrey Druce">
<meta name="dc.creator" content="Gerald Quon">
<meta name="dc.creator" content="Drew Lett">
<meta name="dc.creator" content="Grace Zheng">
<meta name="dc.creator" content="Tao Xu">
<meta name="dc.creator" content="BF Francis Ouellette">
<meta name="dc.type" content="Software">
<meta name="dc.source" content="BMC Bioinformatics 2004 5:40">
<meta name="dc.date" content="2004-4-19">
<meta name="dc.identifier" content="10.1186/1471-2105-5-40">
<meta name="dc.publisher" content="BioMed Central Ltd">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/">
<meta name="dc.format" content="text/html">
<meta name="dc.language" content="en">

<meta name="prism.publicationName" content="BMC Bioinformatics">
<meta name="prism.issn" content="1471-2105">
<meta name="prism.publicationDate" content="2004-4-19">
<meta name="prism.volume" content="5">
<meta name="prism.number" content="1">
<meta name="prism.section" content="Software">
<meta name="prism.startingPage" content="40">
<meta name="prism.copyright" content="2004 Shah et al; ">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

<meta name="citation_journal_title" content="BMC Bioinformatics">
<meta name="citation_publisher" content="BioMed Central Ltd">
<meta name="citation_authors" content="Sohrab Shah, David He, Jessica Sawkins, Jeffrey Druce, Gerald Quon, Drew Lett, Grace Zheng, Tao Xu, BF Francis Ouellette">
<meta name="citation_title" content="Pegasys: software for executing and integrating analyses of biological sequences">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="40">
<meta name="citation_doi" content="10.1186/1471-2105-5-40">

<link rel="stylesheet" type="text/css" href="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/screen_002.css" title="Default Styles">

<link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/content/pdf/1471-2105-5-40.pdf">
<link rel="alternate" type="text/xml" title="XML version" href="http://www.biomedcentral.com/content/xml/1471-2105-5-40.xml">

</head><div FirebugVersion="1.3.3" style="display: none;" id="_firebugConsole"></div><body class="bgimage" alink="#6699ff" link="#003399" vlink="#3366cc"><center><div id="outerframe"><div class="skip"><a href="#content">Skip to main content</a></div><div style="z-index: 1;" id="newad"><table align="center" border="0" cellpadding="0" cellspacing="0"><tbody><tr><td class="topad" align="center">

<center>

<!------ OAS SETUP begin ------>
<script language="JavaScript">
<!--
OAS_version = 10;
function OAS_NORMAL(pos)
{
   document.write('<a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1570519025@Top,Bottom,Left1,Bottom1!' + pos + '" target=_top>');
   document.write('<img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1570519025@Top,Bottom,Left1,Bottom1!' + pos + '" border=0></a>');
}
//-->
</script>
<script language="JavaScript1.1">
<!--
OAS_version = 11;
if (navigator.userAgent.indexOf('Mozilla/3') > 1)
   OAS_version = 10;
if (navigator.userAgent.indexOf('Netscape/7.0') > 1)
   OAS_version = 10;
if (OAS_version >= 11)
   document.write('<scr' + 'ipt language="JavaScript1.1" src="http://oas.biomedcentral.com/RealMedia/ads/adstream_mjx.ads/www.biomedcentral.com/article/bmcbioinformatics/1570519025@Top,Bottom,Left1,Bottom1"></scr' + 'ipt>');
//-->
</script><script language="JavaScript1.1" src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1570519025TopBottomLeft1Bottom1.js"></script>
<script language="JavaScript">
<!--
document.write('');
function OAS_AD(pos)
{
   if (OAS_version >= 11)
      OAS_RICH(pos);
   else
      OAS_NORMAL(pos);
}
//-->
</script>
<!------ OAS SETUP end ------>
<div id="dropdown" onmouseover="testMouse()" onmouseout="testMouse()">
<!------ OAS AD 'Top' begin ------>
<script language="JavaScript">
<!--
OAS_AD('Top');
//-->
</script><!-- 
Support: http://oas.biomedcentral.com#biomed/Alma-Biomed-Lead-3#Alma-Biomed-Lead-3#Alma-Biomed-Lead-3.html#5b135#1239788142#437#S#Top#www.biomedcentral.com/article/bmcbioinformatics##
--><script id="extFlashTop1" type="text/javascript" src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/TFSMflashobject.js"></script>
<script language="JavaScript">
<!--
if(!document.body)
document.write("<html><body>");
OASd = document;
var plug = false;
var flashVersion = -1;
var minFlashVersion = 6;
if(navigator.plugins != null && navigator.plugins.length > 0){flashVersion =(navigator.plugins["Shockwave Flash 2.0"] || navigator.plugins["Shockwave Flash"]) ? navigator.plugins["Shockwave Flash" +(navigator.plugins["Shockwave Flash 2.0"] ? " 2.0" : "")].description.split(" ")[2].split(".")[0] : -1;
plug = flashVersion >= minFlashVersion;}
else if(navigator.userAgent.toLowerCase().indexOf("webtv/2.6") != -1){flashVersion = 4;plug = flashVersion >= minFlashVersion;}
else if(navigator.userAgent.toLowerCase().indexOf("webtv/2.5") != -1){flashVersion = 3;plug = flashVersion >= minFlashVersion;}
else if(navigator.userAgent.toLowerCase().indexOf("webtv") != -1){flashVersion = 2;plug = flashVersion >= minFlashVersion;}
else if((navigator.appVersion.indexOf("MSIE") != -1) &&(navigator.appVersion.toLowerCase().indexOf("win") != -1) &&(navigator.userAgent.indexOf("Opera") == -1)){var oasobj;
var exc;
try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.7");
flashVersion = oasobj.GetVariable("$version");} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.6");
version = "WIN 6,0,21,0"; 
oasobj.AllowScriptAccess = "always";
flashVersion = oasobj.GetVariable("$version");} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.3");
flashVersion = oasobj.GetVariable("$version");} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.3");
flashVersion = "WIN 3,0,18,0";} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash");
flashVersion = "WIN 2,0,0,11";} catch(exc){flashVersion = -1;}}}}}
plug =(flashVersion != -1)? flashVersion.split(" ")[1].split(",")[0] >= minFlashVersion : false;}

if(plug)
{
document.write("<div id=\"FinContentTop1\"></div>");
function loadFlashTop1(){
	if(navigator.userAgent.indexOf("MSIE") != -1 && navigator.userAgent.indexOf("Opera") == -1){
	
		if (extFlashTop1.readyState == "complete")
  		{
	   		FlashObject("http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/novoseek_728x90_05.swf?clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1260255323/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?", "OAS_AD_Top", "width=728 height=90", "opaque", "clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" ,"6", "FinContentTop1");
			extFlashTop1.onreadystatechange = "";
		}
	
		extFlashTop1.onreadystatechange = loadFlashTop1;
	}
	else
	{
           OASfp=" Menu=FALSE swModifyReport=TRUE width=728 height=90 ";
           OASd.write("<object id=\"techsource_Top\" classid=\"clsid:D27CDB6E-AE6D-11cf-96B8-444553540000\" codebase=\"http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,0,0\""+OASfp+">");
           OASd.write("<param name=src value=\"http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/novoseek_728x90_05.swf\"><param name=allowScriptAccess value=always><param name=quality value=autohigh><param name=loop value=true><param name=play value=true><param name=menu value=false><param name=wmode value=opaque><param name=FlashVars value=\"clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1260255323/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?\">");
           OASd.write("<embed src=\"http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/novoseek_728x90_05.swf\""+OASfp+"pluginspage=\"http://www.macromedia.com/shockwave\" type=\"application/x-shockwave-flash\" width=728 height=90 PLAY=true LOOP=true QUALITY=autohigh WMODE=opaque FlashVars=\"clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1260255323/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?\" allowScriptAccess=always>");
           OASd.write("</embed></object>");
        }

}
	loadFlashTop1();
}
else
{
OASd.write("<a href=\"http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1260255323/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?\" target=\"_blank\"><img src=\"http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/banner_alternativo3.gif\" width=728 height=90 border=0 alt=\"\"></a>");}
if(!document.body)
document.write("</body></html>");
//-->
</script><div id="FinContentTop1"></div><object id="techsource_Top" classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,0,0" menu="FALSE" swmodifyreport="TRUE" height="90" width="728"><param name="src" value="http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/novoseek_728x90_05.swf"><param name="allowScriptAccess" value="always"><param name="quality" value="autohigh"><param name="loop" value="true"><param name="play" value="true"><param name="menu" value="false"><param name="wmode" value="opaque"><param name="FlashVars" value="clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1260255323/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?"><embed src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/novoseek_728x90_05.swf" menu="FALSE" swmodifyreport="TRUE" pluginspage="http://www.macromedia.com/shockwave" type="application/x-shockwave-flash" play="true" loop="true" quality="autohigh" wmode="opaque" flashvars="clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1260255323/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?" allowscriptaccess="always" height="90" width="728"></object><noscript>
<a
href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1570519025@Top,Bottom,Left1,Bottom1!Top"
target=_top>
<img
src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1570519025@Top,Bottom,Left1,Bottom1!Top"
border=0></a>
</noscript></div></center></td><td valign="top"><table height="95"><tbody><tr><td style="vertical-align: middle;"><a href="http://www.biomedcentral.com/info/mediainformation/"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/advhere.gif" alt="" border="0"></a></td></tr></tbody></table>
<!------ OAS AD end ------>


</td></tr></tbody></table></div><div id="innerframe"><table border="0" cellpadding="5" cellspacing="0" width="100%"><tbody><tr valign="bottom"><td width="50"><a href="http://www.biomedcentral.com/bmcbioinformatics"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/logo.gif" alt="" border="0" hspace="3"></a></td><td align="left"><a href="http://www.biomedcentral.com/info/about/faq?name=impactfactor"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1002.gif" alt="" border="0" hspace="3"></a></td><td id="topnav" align="right"><a href="http://www.biomedcentral.com/registration/">Log on / register</a><br><a href="http://www.biomedcentral.com/support/feedback/" onclick="popup(this.href,'Feedback',650,500); return false;">Feedback</a> | <a href="http://www.biomedcentral.com/support/">Support</a> | <a href="http://www.biomedcentral.com/my/">My details</a></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr valign="top"><td colspan="3" bgcolor="#003399"><script type="text/javascript"><!--
document.write("<div id='TopBar'><font color='#ffffff'>&nbsp;<a href='/' class='xtoplinks'><b> home </b></a>|<a href='/browse/journals/' class='xtoplinks'><b> journals A-Z </b></a>|<a href='/browse/bysubject/' class='xtoplinks'><b> subject areas </b></a>|<a href='/search/' class='xtoplinks'><b> advanced search </b></a>|<a href='/info/authors/' class='xtoplinks'><b> authors </b></a>|<a href='/info/about/peerreview/' class='xtoplinks'><b> reviewers </b></a>|<a href='/info/libraries' class='xtoplinks'><b> libraries </b></a>|<a href='http://careers.the-scientist.com/' class='xtoplinks'><b> jobs </b></a>|<a href='/info/' class='xtoplinks'><b> about </b></a>|<a href='/my/' class='xtoplinks'><b> my BioMed Central </b></a></font></div>");//--></script><div id="TopBar"><font color="#ffffff">&nbsp;<a href="http://www.biomedcentral.com/" class="xtoplinks"><b> home </b></a>|<a href="http://www.biomedcentral.com/browse/journals/" class="xtoplinks"><b> journals A-Z </b></a>|<a href="http://www.biomedcentral.com/browse/bysubject/" class="xtoplinks"><b> subject areas </b></a>|<a href="http://www.biomedcentral.com/search/" class="xtoplinks"><b> advanced search </b></a>|<a href="http://www.biomedcentral.com/info/authors/" class="xtoplinks"><b> authors </b></a>|<a href="http://www.biomedcentral.com/info/about/peerreview/" class="xtoplinks"><b> reviewers </b></a>|<a href="http://www.biomedcentral.com/info/libraries" class="xtoplinks"><b> libraries </b></a>|<a href="http://careers.the-scientist.com/" class="xtoplinks"><b> jobs </b></a>|<a href="http://www.biomedcentral.com/info/" class="xtoplinks"><b> about </b></a>|<a href="http://www.biomedcentral.com/my/" class="xtoplinks"><b> my BioMed Central </b></a></font></div><noscript>&nbsp;<a
href='/'><span class='tinytext' style='color:#ffffff'><B> home
</b></span></A>|<a href='/browse/journals/'><span class='tinytext'
style='color:#ffffff'><B> journals A-Z </b></span></A>|<a
href='/browse/bysubject/'><span class='tinytext'
style='color:#ffffff'><B> subject areas </b></span></A>|<a
href='/search/'><span class='tinytext' style='color:#ffffff'><B>
advanced search </b></span></A>|<a href='/info/authors/'><span
class='tinytext' style='color:#ffffff'><B> authors </b></span></A>|<a
href='/info/about/peerreview/'><span class='tinytext'
style='color:#ffffff'><B> reviewers </b></span></A>|<a
href='/info/libraries'><span class='tinytext' style='color:#ffffff'><B>
libraries </b></span></A>|<a
href='http://careers.the-scientist.com/'><span class='tinytext'
style='color:#ffffff'><B> jobs </b></span></A>|<a href='/info/'><span
class='tinytext' style='color:#ffffff'><B> about </b></span></A>|<a
href='/my/'><span class='tinytext' style='color:#ffffff'><B> my BioMed
Central </b></span></A></noscript></td></tr></tbody></table><div id="article"><script src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/fulltext.js" type="text/javascript"></script><table cellpadding="0" cellspacing="0" width="100%"><tbody valign="top"><tr><td id="leftcolumn"><ul style="position: static;" id="outline">
<li><a href="#">Top</a></li>
<li><a href="#IDAORATG">Abstract</a></li>
<li><a href="#IDAESATG">Background</a></li><li><a href="#IDALVATG">Implementation</a></li><li><a href="#IDAVEBTG">Results and discussion</a></li><li><a href="#IDATHBTG">Conclusions</a></li><li><a href="#IDA1HBTG">Availability and requirements</a></li><li><a href="#IDAGIBTG">Authors' contributions</a></li><li><a href="#IDAMIBTG">Acknowledgments</a></li>
<li class="lowest"><a href="#refs">References</a></li>
</ul>
<table width="130"><tbody><tr><td>&nbsp;</td></tr></tbody></table></td><td id="content"><div id="navbox">
<table cellpadding="5" cellspacing="2">
<tbody><tr>
<td><a href="http://www.biomedcentral.com/bmcbioinformatics"><strong>BMC Bioinformatics</strong></a><br><a href="http://www.biomedcentral.com/bmcbioinformatics/5">Volume 5</a><br></td>
</tr><tr><td><strong>Viewing options:</strong><ul>
<li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/5/40/abstract/">Abstract</a></li>
<li>&nbsp;<strong>Full text</strong></li>
<li>&nbsp;<a href="http://www.biomedcentral.com/content/pdf/1471-2105-5-40.pdf">PDF </a>
		(589KB)
				</li>
</ul><strong>Associated material:</strong><ul><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/5/40/comments">Readers' comments</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/pubmed/15096276" onclick="largepopup(this.href,'pubmed');return false">PubMed record</a></li></ul><strong>Related literature:</strong><ul><li>&nbsp;Articles citing this article<div><a href="http://www.biomedcentral.com/search/citedby/15096276">on BioMed Central</a></div><div><a href="http://scholar.google.com/scholar?hl=en&amp;lr=&amp;safe=off&amp;c2coff=1&amp;q=link%3Ahttp://www.biomedcentral.com/1471-2105/5/40&amp;btnG=Search" onclick="largepopup(this.href,'on_google');return false">on Google Scholar</a></div><div><a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=BMC_sci&amp;SrcApp=BMC_sci&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000221698100002" onclick="largepopup(this.href,'on_isi');return false">on&nbsp;ISI&nbsp;Web&nbsp;of&nbsp;Science</a></div><div><a href="http://www.biomedcentral.com/search/citedby/pmc/15096276">on PubMed Central</a></div></li>
<li>&nbsp;Other articles by authors<br><div><a href="javascript://" onclick="toggleauthors('authg','authbuttong'); return false"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/expand2.gif" id="authbuttong" alt="">on Google Scholar
				</a></div>
<div style="display: none;" id="authg"><a href="http://scholar.google.com/scholar?q=author%3A%22SP+Shah%22">Shah SP</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22DY+He%22">He DY</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22JN+Sawkins%22">Sawkins JN</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22JC+Druce%22">Druce JC</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22G+Quon%22">Quon G</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22D+Lett%22">Lett D</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22GX+Zheng%22">Zheng GX</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22T+Xu%22">Xu T</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22BF+Ouellette%22">Ouellette BFF</a></div>
<div><a href="javascript://" onclick="toggleauthors('authpm','authbuttonpm'); return false"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/expand2.gif" id="authbuttonpm" alt="">on PubMed
				</a></div>
<div style="display: none;" id="authpm"><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Shah_SP&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Shah SP</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=He_DY&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">He DY</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Sawkins_JN&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Sawkins JN</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Druce_JC&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Druce JC</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Quon_G&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Quon G</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Lett_D&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Lett D</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Zheng_GX&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Zheng GX</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Xu_T&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Xu T</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Ouellette_BF&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Ouellette BFF</a></div>
</li>
<li>&nbsp;Related articles/pages<br><div><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2105/5/40">on Google</a></div>
<div><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2105/5/40">on Google Scholar</a></div>
<div><a href="http://www.biomedcentral.com/pubmed/related/15096276">on PubMed</a></div>
</li></ul><strong>Tools:</strong><ul><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/5/40/citation">Download citation(s)</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/content/download/xml/1471-2105-5-40.xml">Download XML</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/5/40/email">Email to a friend</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/info/about/reprintsandperm">Order reprints</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/5/40/postcomment">Post a comment</a></li>
<li>&nbsp;<a href="http://www.biomedcentral.com/bmcbioinformatics/alerts/">Sign up for article alerts</a></li></ul><script type="text/javascript">toggleauthors('authpm','authbuttonpm'); toggleauthors('authg','authbuttong'); toggleauthors('authos','authbuttonos'); toggleauthors('autha','authbuttona');</script><strong>Post to:</strong><br><ul class="iconlist">
<li><a href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2105/5/40" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false"><img alt="" src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/citeulink.gif"></a>&nbsp;<a href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2105/5/40" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false">Citeulike				
				</a></li>
<li><a onclick="largepopup(this.href,'Connotea',800,600);return false" href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1471-2105-5-40"><img alt="" src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/connotea.gif"></a>&nbsp;<a href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1471-2105-5-40" onclick="largepopup(this.href,'Connotea',800,600);return false">Connotea
					</a></li>
<li><a href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2105/5/40&amp;title=Pegasys:%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences" onclick="largepopup(this.href,'Del.icio.us',800,600);return false"><img alt="" src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/delicious.gif"></a>&nbsp;<a href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2105/5/40&amp;title=Pegasys:%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences" onclick="largepopup(this.href,'Del.icio.us',800,600);return false">Del.icio.us</a></li>
<li><a href="http://digg.com/submit?phase=2&amp;url=http://www.biomedcentral.com/1471-2105/5/40&amp;title=Pegasys:%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences&amp;topic=general_sciences" onclick="largepopup(this.href,'Digg',800,600);return false"><img alt="" src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/digg.gif"></a>&nbsp;<a href="http://digg.com/submit?phase=2&amp;url=http://www.biomedcentral.com/1471-2105/5/40&amp;title=Pegasys:%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences&amp;topic=general_sciences" onclick="largepopup(this.href,'Digg',800,600);return false">Digg
				</a></li>
<li><a href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2105/5/40" onclick="largepopup(this.href,'facebook',800,600);return false"><img alt="" src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/facebook.gif"></a>&nbsp;<a href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2105/5/40" onclick="largepopup(this.href,'facebook',800,600);return false">Facebook</a></li>
</ul></td></tr></tbody></table>
</div><a name="maincontent"></a><div id="topmatter"><a name="text"></a><a href="http://www.biomedcentral.com/info/about/openaccess/"><img class="access" src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/openaccess-large.gif" alt="Open Access" height="24" width="89"></a><a href="http://www.biomedcentral.com/info/about/mostviewed/"><img class="access" src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/highlyaccessed-large.gif" alt="Highly Access" height="24" width="124"></a><!-- <rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns="http://purl.org/rss/1.0/">
<cc:Work rdf:about="http://www.biomedcentral.com/1471-2105/5/40">
<cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/>
</cc:Work>
<cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/">
<cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
<cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Notice"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/>
<cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/>
</cc:License>
<item rdf:about="http://www.biomedcentral.com/1471-2105/5/40">
<title>Pegasys: software for executing and integrating analyses of biological sequences</title>
<dc:title>Pegasys: software for executing and integrating analyses of biological sequences</dc:title>
<dc:creator>Shah, Sohrab P</dc:creator>
<dc:creator>He, David YM</dc:creator>
<dc:creator>Sawkins, Jessica N</dc:creator>
<dc:creator>Druce, Jeffrey C</dc:creator>
<dc:creator>Quon, Gerald</dc:creator>
<dc:creator>Lett, Drew</dc:creator>
<dc:creator>Zheng, Grace XY</dc:creator>
<dc:creator>Xu, Tao</dc:creator>
<dc:creator>Ouellette, BF Francis</dc:creator>
<dc:identifier>info:doi/10.1186/1471-2105-5-40</dc:identifier>
<dc:identifier>info:pmid/15096276</dc:identifier>
<dc:source>BMC Bioinformatics 2004, 5:40</dc:source>
<dc:date>2004-04-19</dc:date>
<prism:publicationName>BMC Bioinformatics</prism:publicationName>
<prism:publicationDate>2004-04-19</prism:publicationDate>
<prism:volume>5</prism:volume>
<prism:number>1</prism:number>
<prism:section>Software</prism:section>
<prism:startingPage>40</prism:startingPage>
</item>
</rdf:RDF> --><span class="articletype">Software</span><h1>Pegasys: software for executing and integrating analyses of biological sequences</h1><div class="singleins"><p class="authors"><strong>Sohrab P Shah</strong> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email_002.gif" alt="email" title="Email"></a>, <strong>David YM He</strong> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email_002.gif" alt="email" title="Email"></a>, <strong>Jessica N Sawkins</strong> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email_002.gif" alt="email" title="Email"></a>, <strong>Jeffrey C Druce</strong> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email_002.gif" alt="email" title="Email"></a>, <strong>Gerald Quon</strong> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email_002.gif" alt="email" title="Email"></a>, <strong>Drew Lett</strong> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email_002.gif" alt="email" title="Email"></a>, <strong>Grace XY Zheng</strong> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email_002.gif" alt="email" title="Email"></a>, <strong>Tao Xu</strong> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email_002.gif" alt="email" title="Email"></a> and <strong>BF Francis Ouellette</strong> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email-ca.gif" title="Corresponding author" alt="email"></a></p><p>UBC Bioinformatics Centre, University of British Columbia, Vancouver, British Columbia, Canada</p></div><p><span class="pseudotabau"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email.gif" border="0"> author email</span><span class="pseudotabau"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/email-ca_002.gif"> corresponding author email</span></p><p><em>BMC Bioinformatics</em> 2004,
			 <strong>5</strong><strong>:</strong>40<span class="pseudotab">doi:10.1186/1471-2105-5-40</span></p><p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2105/5/40">http://www.biomedcentral.com/1471-2105/5/40</a></p>
<table cellpadding="0" cellspacing="0">
<tbody>
<tr>
<td>Received:</td>
<td>27&nbsp;February&nbsp;2004</td>
</tr>
<tr>
<td>Accepted:</td>
<td>19&nbsp;April&nbsp;2004</td>
</tr>
<tr>
<td>Published:</td>
<td>19&nbsp;April&nbsp;2004</td>
</tr>
</tbody>
</table>
<p> © 2004 Shah et al; licensee BioMed Central Ltd. This is an Open
Access article: verbatim copying and redistribution of this article are
permitted in all media for any purpose, provided this notice is
preserved along with the article's original URL.</p></div><a name="IDAORATG"></a><h3>Abstract</h3>
<h4>Background</h4>
<p>We present Pegasys – a flexible, modular and customizable software
system that facilitates the execution and data integration from
heterogeneous biological sequence analysis tools.</p>
<h4>Results</h4>
<p>The Pegasys system includes numerous tools for pair-wise and multiple sequence alignment, <em>ab initio </em>gene
prediction, RNA gene detection, masking repetitive sequences in genomic
DNA as well as filters for database formatting and processing raw
output from various analysis tools. We introduce a novel data structure
for creating workflows of sequence analyses and a unified data model to
store its results. The software allows users to dynamically create
analysis workflows at run-time by manipulating a graphical user
interface. All non-serial dependent analyses are executed in parallel
on a compute cluster for efficiency of data generation. The uniform
data model and backend relational database management system of Pegasys
allow for results of heterogeneous programs included in the workflow to
be integrated and exported into General Feature Format for further
analyses in GFF-dependent tools, or GAME XML for import into the Apollo
genome editor. The modularity of the design allows for new tools to be
added to the system with little programmer overhead. The database
application programming interface allows programmatic access to the
data stored in the backend through SQL queries.</p>
<h4>Conclusions</h4>
<p>The Pegasys system enables biologists and bioinformaticians to
create and manage sequence analysis workflows. The software is released
under the Open Source GNU General Public License. All source code and
documentation is available for download at <a href="http://bioinformatics.ubc.ca/pegasys/">http://bioinformatics.ubc.ca/pegasys/</a> <a href="http://www.webcitation.org/query.php?url=http://bioinformatics.ubc.ca/pegasys/&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>.</p><a name="IDAESATG"></a><h3>Background</h3>
<h4>Pipelines for biological sequence analysis</h4>
<p>Large scale sequence analysis is a complex task that involves the
integration of results from numerous computational tools. For
high-throughput data analysis, these tools must be tied together in a
coordinated system that can automate the execution of a set of analyses
in sequence or in parallel. To this end, a diverse array of software
systems for biological sequence analysis have emerged in recent years.
For example, the Ensembl pipeline <a name="IDANSATG"></a>[<a href="#B1">1</a>] automates the annotation of several eukaryotic genomes, Mungall <em>et al </em><a name="IDATSATG"></a>[<a href="#B2">2</a>] have created a robust pipeline for annotation and analysis of the Drosophila genome, GenDB <a name="IDAYSATG"></a>[<a href="#B3">3</a>] is used as an annotation system for several prokaryotic genomes and Yuan <em>et al </em><a name="IDA4SATG"></a>[<a href="#B4">4</a>]
have published resources for annotating the rice and other plant
genomes. These pipelines are extensive in their scope, are
well-designed and meet their objectives. In surveying these and other
systems, we have identified three critical areas that are essential for
building on the design of existing biological sequence analysis
pipelines:</p>
<p>• There is a need for flexible architecture so that one software
system can be used to analyse different data sets that may require
different analysis tools.</p>
<p>• A system needs to allow for the inclusion of new tools in a
modular fashion so the software architecture does not have to change
with the addition of new tools.</p>
<p>• A system should provide the framework to facilitate data
integration of analysis results from different tools that were computed
on the same input.</p>
<h4>The need for flexible architecture</h4><p>The systems outlined
above differ substantially from each other in their design and
application, but share common attributes. The diversity is naturally
reflective of the varied computational tasks that biologists working on
different projects need to perform in order to analyse their data. A
researcher working on bacteria will need different tools for her
analyses than someone working on mouse. The specificity driven by the
needs of a research project makes it impossible to use a pipeline
designed for a particular data set for analysis of another data set
that has inherent differences such as the organism from which it was
generated. As a result, numerous software pipelines have been created,
many of which perform similar analyses (such as genome annotation) but
on different data. For example, the concept of constructing a pipeline
or 'workflows' of data processing are common to nearly all
high-throughput sequence analysis projects. This shared concept
provides an opportunity to harness the commonality in software so that
a new system need not be designed for every new project.</p>
<h4>Incorporating new tools into existing frameworks</h4><p>The
bioinformatics community is faced with a challenging and dynamic
environment where new computational tools and data sets for sequence
analysis are constantly being generated. Capitalizing on algorithmic
and computational advances is critical to discovering more about the
data being analysed. For a system that has a rigid pipeline that is
'hard coded', it may require a significant programming investment to
incorporate a new tool. This may discourage biologists from integrating
a new tool on the basis of logistics, rather than on the basis of
scientific applicability. Therefore, a system should provide a
framework that is designed for flexibility and extensibility.</p>
<h4>Facilitating data integration</h4><p>Genome annotation requires data integration. For example <em>ab initio </em>prediction of gene structures on genomic sequence can be greatly enhanced by using supporting sequence similarity searches <a name="IDAVTATG"></a><a name="IDAYTATG"></a><a name="IDA1TATG"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>-<a href="#B7">7</a>].
Concordance between different methodologies lends stronger support and
gives more compelling evidence to an algorithm or a person trying to
infer true biological features from computationally derived features <a name="IDAAUATG"></a>[<a href="#B8">8</a>].
It follows that any analysis pipeline or system should provide a design
that facilitates integration of heterogeneous sources of data.</p>
<h4>The Pegasys biological sequence analysis system</h4><p>To meet the
challenges outlined above we have designed and implemented Pegasys: a
flexible, modular and customizable framework for biological sequence
analysis. The software is implemented in the Java programming language
and is Open Source, released under the GNU General Public License. The
features of Pegasys allow it to be used on a wide variety of tasks and
data. Analysis modules for pair-wise and multiple sequence alignment, <em>ab initio </em>gene
prediction, masking of repetitive elements, prediction of RNA sequences
and eukaryotic splice site predictors have been developed. A new set of
analyses is performed by first creating a new 'workflow'. We define a
workflow as a set of analyses a biologist wishes to perform on a single
sequence or set of sequences. Each workflow has the following
qualities: a) the analyses can be linked together such that output from
one analysis can be used as input to a subsequent analysis, b) analyses
can accept outputs from more than one analysis as input, and c)
analyses that are not serially dependent can be executed in parallel.</p>
<p>Analysis tools in the Pegasys system are wrapped in modules that can
easily be plugged into the system. The backend database system provides
a data model that abstracts the concept of a computational feature and
captures data from all the different analysis tools in the same
framework. We have implemented data adaptors that can export
computational results in General Feature Format <a name="IDAOUATG"></a>[<a href="#B9">9</a>] and Genome Annotation Markup Elements (GAME) XML <a name="IDATUATG"></a>[<a href="#B10">10</a>] for import into the Apollo genome editor <a name="IDAYUATG"></a>[<a href="#B11">11</a>].
For simple workflows where data integration is not applicable, for
example one analysis on an input sequence, raw, untransformed output
from the analysis can also be retrieved.</p>
<p>The system is fronted by a graphical user interface that allows
users to create workflows at run-time and have them executed on the
Pegasys server. The GUI also allows users to save their workflows for
repeat execution on different input, or using different reagents.</p>
<p>To demonstrate the utility of Pegasys in widely different
bioinformatics tasks, we present three use cases of the system: a
single application workflow, a workflow designed for formatting a
database for BLAST <a name="IDAAVATG"></a><a name="IDADVATG"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>,<a href="#B13">13</a>]
and searching the newly formatted database, and finally a workflow
designed for genome annotation of eukaryotic genomic sequence.</p>
<p>We are releasing this work with the intention that a wide variety of
sequence analyses in the bioinformatics research community will be
enabled. Full details of the availability, support and documentation of
Pegasys can be found at <a href="http://bioinformatics.ubc.ca/pegasys/">http://bioinformatics.ubc.ca/pegasys/</a> <a href="http://www.webcitation.org/query.php?url=http://bioinformatics.ubc.ca/pegasys/&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>.</p><a name="IDALVATG"></a><h3>Implementation</h3>
<p>The design of the Pegasys system is guided by three main principles:
modularity, flexibility and data integration. With these principles in
mind, we designed Pegasys with the following architecture.</p>
<h4>Architecture and data flow</h4>
<p>The architecture of the system has a layered topology that uses a
client/server model. The client has a graphical user interface (see
Figure <a name="IDAUVATG"></a><a onclick="popup('/1471-2105/5/40/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F4">4</a>)
for the creation of workflows. Once a workflow is created, it is sent
to the server where it is executed. The server is made up of separate
layers for job scheduling, execution, database interaction, and
adaptors. The connectivity between layers is shown in Figure <a name="IDAYVATG"></a><a onclick="popup('/1471-2105/5/40/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F1">1</a>.
The application layer converts the work flow rendered in XML into a
directed acyclic graph (DAG) of analyses in memory. While traversing
the DAG, the application schedules all of the analyses on a distributed
compute cluster and facilitates the flow of data so that a particular
node's program is only executed once all of its inputs are ready (i.e.
all of the 'parent' analyses are complete). As each analysis completes,
the results are inserted into the backend database layer. Complete
reports and computational features of a sequence are inserted into
relational tables. Sophisticated queries on the data, in which results
from selected programs can be integrated together over a portion or all
of the input sequence, can then be run to compile data for output. The
data is exported from the system via the adaptor layer in various
formats (currently GFF, GAME XML and raw output from each analysis tool
are supported) for human interpretation or for import into other
applications such as viewing tools (DAS <a name="IDA3VATG"></a>[<a href="#B14">14</a>]), editing tools (Apollo <a name="IDACWATG"></a>[<a href="#B11">11</a>]) or statistical analysis tools such as R <a name="IDAHWATG"></a>[<a href="#B15">15</a>].</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/5/40/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F1"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1471-2105-5-40-1.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 1.</strong></a> <strong>Diagram showing the client/server model and layering of the Pegasys architecture. </strong>Arrows
between the layers indicate a transfer of data. The workflow created by
manipulating the GUI in the client is sent as a Pegasys DAG XML file to
the server. The application layer then processes the XML file, and
sends jobs to the job scheduling layer. The analyses are then executed
and the results are stored in the database. The adaptor layer takes
results stored in the <strong>PegasysResultSet </strong>data structure
in memory in the application layer and can create output in GFF or GAME
XML format. This file is then returned to the GUI where it can be
digested by the user or input into a visualization tool.</p>
</div></div><h4>The Pegasys data structure</h4>
<p>The core data structure of the Pegasys system is a DAG <em>G(V, E)</em>, consisting of a set of nodes <em>V </em>and a set of edges connecting the nodes <em>E </em>(see Figure <a name="IDAEXATG"></a><a onclick="popup('/1471-2105/5/40/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F2">2</a>).
The DAG data structure models a workflow created by a user of the
Pegasys system. A node can take one of three forms: a) an input
sequence or b) an individual run of a program in the system or c) an
output node. An edge (<em>v</em>1, <em>v</em>2) where <em>v</em>1 and <em>v</em>2 are nodes in <em>V </em>links data flow between <em>v</em>1 and <em>v</em>2. An edge represents a serial dependency, indicating that the input of <em>v</em>2 is tied to the output of <em>v</em>1. We refer to this relationship as a parent-child relationship: node <em>v</em>2 is a child of node <em>v</em>1 and node <em>v</em>1 is the parent of node <em>v</em>2. The edge ensures that the output format from <em>v</em>1 is consistent with the input format of <em>v</em>2.
A node in the DAG can have more than one parent and therefore can have
heterogeneous input from multiple sources. The edges in the graph are
directional and can only connect two nodes that are executed one after
another. The graph therefore has a chronological axis: the child nodes
are executed after their parent nodes have completed.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/5/40/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F2"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1471-2105-5-40-2.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 2.</strong></a> <strong>Diagram showing an abstract representation of a Pegasys DAG. </strong>A): Consider <em>v</em>1: this could be an input sequence that is used by two sequence analysis programs <em>v</em>2 and <em>v</em>3. <em>v</em>4 is dependent on the output of both <em>v</em>2 and <em>v</em>3 and therefore cannot execute until <em>v</em>2 and <em>v</em>3 have completed. In this diagram, <em>v</em>2 and <em>v</em>3 will be executed in parallel as will <em>v</em>4 and <em>v</em>5. B): DAG in the case where <em>v</em>3 produces two instances of the expected output to <em>v</em>5. The sub-DAG rooted at <em>v</em>5 replicates itself (<em>v</em>5<em>a </em>and <em>v</em>5<em>b</em>) for each instance of its input. All of the new sub-DAGs are executed in parallel.</p>
</div></div><p>The DAG is created dynamically at run time as the user
manipulates the GUI (see The Graphical User Interface section). The
user can create workflows using any combination of the available
programs in Pegasys by dragging/dropping and linking graphical icons
that represent sequence analysis tools on a canvas together with edges
in much the same way that one would use drawing tool software to create
a flow diagram. Each program icon can be clicked to open a dialogue box
that can take inputs for parameters that are supported by that
particular program. Once all of the parameters for all the nodes have
been filled in, the information for each node and their relationships
to each other are compiled into a structured XML file. This file is
then used as input to the Pegasys server that executes the analyses in
parallel (described in the Architecture and Data Flow section) or can
be saved for later editing or distribution. During the execution of the
DAG, the data structure can adjust itself to accommodate outputs
generated from the nodes. Consider the edge (<em>v</em>3, <em>v</em>5) depicted in Figure <a name="IDAB0ATG"></a><a onclick="popup('/1471-2105/5/40/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F2">2</a> that connects an <em>ab initio </em>gene prediction program <em>v</em>3 with a sequence alignment program <em>v</em>5. In <em>v</em>5, the user wishes to search the coding regions from the output of <em>v</em>3 against a protein database. <em>v</em>5 cannot know how many genes will be predicted from <em>v</em>3 before <em>v</em>3 has terminated. Once <em>v</em>3 has terminated however, <em>v</em>5 will replicate itself for each 'output unit' generated from <em>v</em>3 (see Figure <a name="IDA10ATG"></a><a onclick="popup('/1471-2105/5/40/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F2">2B</a>). In this case, <em>v</em>5 replicates itself for each of the coding regions and the DAG executes each 'copy' of <em>v</em>5
in parallel. This built-in elasticity confers maximum parallel
execution of analyses and therefore more efficient execution of the
computations in the DAG.</p>
<h4>The Program module</h4>
<p>The <strong>Program </strong>module is the fundamental unit of the
nodes of the aforementioned DAG in the application layer of the server
and is a real instance of a node <em>v </em>∈ <em>V</em>. <strong>'Program' </strong>is
an object oriented class that abstracts the concept of a Unix program
that is natively compiled. Unix programs generally have a set of input
command line parameters and output that is sent to the standard output,
standard error or an output file. The <strong>Program </strong>class
has a data structure to store a program's command line arguments and
parameters. It contains methods for setting the path to the program's
location on the system, executing the program and capturing its output
from a file, standard error and standard output streams. To abstract a
sequence analysis program, we created a <strong>PegasysProgram </strong>class that extends <strong>Program </strong>by adding an input sequence attribute and a <strong>PegasysResultSet </strong>to store the results of the analysis. The <strong>ProgramResultSet </strong>is
a hierarchical, recursive data structure that allows storage of nested
analysis results. For example a BLAST output has a list of similar
sequences that each in turn has a list of high scoring pairs. Similarly
Genscan produces output that contains a list of predicted genes, each
of which could have a promoter, a list of exons and a poly-A signal. <strong>PegasysResultSet </strong>captures the hierarchical nature of these results.</p>
<p>For each sequence specific analysis tool in Pegasys, we created a class that extends <strong>PegasysProgram</strong>. Each of these classes implement their own methods that load the particular output of the program and parse it into their <strong>PegasysResultSet</strong>.
For example, the locations of computational evidences such as predicted
exons from a gene finding tool, or a high scoring pair from an
alignment algorithm are parsed along with a statistic and/or score when
available. This architecture generalises a computational feature so
that programmatically, results from different analysis programs can be
treated equally. As mentioned earlier, this allows the user to output
results from different programs in a unified format such as GFF, or
GAME XML. In addition, it facilitates querying for all computational
evidence computed on a segment of sequence that may be of interest to
the biologist.</p>
<p>Creating a new <strong>PegasysProgram </strong>derivative involves
writing a parser for the particular application that can extract data
that is amenable to being loaded into a <strong>PegasysResultSet</strong>. The system, at the time of this writing has <strong>PegasysPrograms </strong>for RepeatMasker <a name="IDAL2ATG"></a>[<a href="#B16">16</a>], BLAST (blastn, blastp, blastx, tblastn, tblastx) <a name="IDAQ2ATG"></a><a name="IDAT2ATG"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>,<a href="#B13">13</a>], WU BLAST <a name="IDAY2ATG"></a>[<a href="#B17">17</a>], the EMBOSS <a name="IDA32ATG"></a>[<a href="#B18">18</a>] implementation of Smith-Waterman <a name="IDAC3ATG"></a>[<a href="#B19">19</a>], Genscan <a name="IDAH3ATG"></a>[<a href="#B20">20</a>], HMMgene <a name="IDAM3ATG"></a>[<a href="#B21">21</a>], Mlagan <a name="IDAR3ATG"></a>[<a href="#B22">22</a>], Sim4 <a name="IDAW3ATG"></a>[<a href="#B23">23</a>], TrnaScan-SE <a name="IDA13ATG"></a>[<a href="#B24">24</a>] and GeneSplicer <a name="IDAA4ATG"></a>[<a href="#B25">25</a>].</p>
<h4>The database</h4>
<p>The backend database of the Pegasys system was created with the goal
of maximizing information capture during execution of a workflow. The
database tracks all parameters used for the invocations of analysis
programs, all input sequences, and all output generated by computation.</p>
<h4>The Pegasys schema</h4><p>The Pegasys schema has three main tables:
'sequence' which stores the input sequences, 'program_run' which stores
the information about an individual program's process on the system and
'pegasys_result' which stores the locations of computational features
on the input sequence. Peripheral to the three core tables are
seventeen meta tables that store information about the data in the core
tables. The full schema is presented in Figure <a name="IDAN4ATG"></a><a onclick="popup('/1471-2105/5/40/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F3">3</a>.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/5/40/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F3"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1471-2105-5-40-3.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 3.</strong></a> <strong>Diagram showing the relations of the Pegasys database model. </strong>There
are three core tables to the database: sequence (shown in blue),
program_run (shown in orange) and pegasys_result (shown in yellow). The
meta tables for each of the three core tables are colour coded to match
the corresponding core table. Foreign keys are indicated with 'FK' and
indexed fields are marked with T.</p>
</div></div><p>The 'program_run' table is designed to store all
information on an invocation of an analysis tool in order to facilitate
reprocessing of results without having to recompute an analysis and can
also aid in diagnosing problems that are bound to occur in the system.
'program_run' stores the class that invoked the process, the raw
unprocessed output of the program, the start and end time of the
process and the exit status of the process. In addition, all command
line arguments used to invoke the program are stored in support tables
to 'program_run' in the structured tables 'argument', 'parameter', and
'flag'. Entries into 'program_run' can be grouped into batches for
selective retrieval of analysis results.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/5/40/figure/F4','F4',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F4"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1471-2105-5-40-4.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 4.</strong></a> <strong>Screenshot of the Pegasys GUI showing the three pane design. </strong>The
visible pane is the canvas pane which allows the user to create a
workflow by clicking and dragging icons corresponding to the programs
available to the system. The icons can be connected to each other
through edges. The parameters used for the execution of each program
can be set by double clicking the icon and filling in the dialogue box
that appears (see Figure <a name="IDAI5ATG"></a><a onclick="popup('/1471-2105/5/40/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F5">5</a>). Expected inputs and outputs for the edge can be set by double clicking the edge and filling in the dialogue (see Figure <a name="IDAM5ATG"></a><a onclick="popup('/1471-2105/5/40/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F6">6</a>).
This workflow will run RepeatMasker on the sequence specified in the
File node and write the results to a text file whose path is specified
in the text output node. The RepeatMasker analysis itself is run on the
compute server and the results are communicated back to the client.</p>
</div></div><p>The 'sequence' table stores the raw sequence string
itself, a unique hash code for the sequence string generated by the
java.lang.String.hashCode() function, an identifier for the sequence
(by default the GenBank accession.version number) and a description of
the sequence (by default the NCBI definition line of the FASTA file).
This table does not store meta data about the sequence, rather it is
meant to store unique sequences used for computation. The system
assumes additional information on the sequence is stored elsewhere. The
uniqueness is enforced by ensuring all sequences have distinct hash
codes, description and identifiers.Support tables for sequence have
been created to enable the analysis of sub-sequences of a larger input
sequence. The subsequence relationship to the sequence is stored in the
'subseq' and 'seq_has_subseq' relations. These tables are useful for
'sliding window' analyses or when focusing in on small regions of
interest of a larger input sequence.</p>
<p>The 'pegasys_result' table stores the results of the computations.
It has attributes for a computational evidence type, a database reagent
(if the result is from similarity searches or uses a particular model
in <em>ab initio </em>predictions), the strand, start and end
positions of the computational feature, a score and a statistic for the
computational feature and a free-text description of the feature. If
available, the strand, start and end position on the target sequence of
an alignment are also recorded. To support hierarchical computational
evidences, the table has a 'parent_id' that is a self-referential
foreign key. This enables relating a particular row entry in the table
to another row in the table. Theoretically, the table supports infinite
nesting of hierarchical data types, although in practice results are no
more than 2 levels deep.</p>
<p>The support tables for 'pegasys_result' allow cross-referencing of
ids. For example, the system models the concept of linking out an
identifier from the result of a database search so that the full
sequence and meta data of that sequence can be easily retrieved. This
cross-referencing of a 'pegasys_result' to an identifier is stored in
the 'result_has_xref' relation. The type of identifier is labeled by a
controlled vocabulary so that one can query on a particular type of
cross-reference (such as accession number) as well as add a new type of
cross-reference to the system. Additional support tables to
'pegasys_result' are: 'database-format', 'database_reagent' and
'evidence-type'. Each of these tables stores controlled nomenclature
that is referenced by 'pegasys_result'. The 'database-format' contains
values such as blast, fasta, and genscan for BLAST formatted, FASTA
formatted and Genscan training model respectively. The
'database_reagent' table stores the names and descriptions of sequence
databases and statistical models that are used in the analysis, so that
a user can query the Pegasys database for results from a particular
database reagent. This structure also allows adding new database
reagents into the system seamlessly. The 'evidence-type' table stores
an ontology of computational evidence types, for example 'blastn_hit'
or 'genscan_exon'. For each program that is part of the Pegasys system,
the computational evidence(s) that it outputs must be recorded in the
'evidence-type' table prior to its use.</p>
<h4>Database API</h4><p>To communicate programmatically with the database, we have created a modular application programming interface (API). The <strong>PegasysDB </strong>class
contains public methods for insertion and retrieval of sequences,
analysis results and sets of results (from different programs) on a
particular sequence. Application developers that wish to access data
from a Pegasys database can use these high-level methods to rapidly
store and access data in a straightforward manner without having to
study the underlying schema of the database. The database API uses the
PostgreSQL JDBC driver and so is backend relational database management
system (RDBMS) independent.</p>
<h4>Adaptors</h4>
<p>We have implemented several adaptors for exporting data from a <strong>PegasysProgram </strong>or set of <strong>PegasysPrograms </strong>that contain analysis results. The derived <strong>PegasysAdaptor </strong>classes
all implement a print method to output data in a specific format. We
currently have derived PegasysAdaptor classes for GAME XML for import
into Apollo <a name="IDAOABTG"></a>[<a href="#B11">11</a>] and GFF <a name="IDATABTG"></a>[<a href="#B9">9</a>] which can be imported into numerous tools and servers such as the Distributed Annotation System <a name="IDAYABTG"></a>[<a href="#B14">14</a>] (DAS) and Gbrowse <a name="IDA3ABTG"></a>[<a href="#B26">26</a>].
The adaptor architecture is extensible and easily allows the
development and inclusion of new adaptors for additional formats. The <strong>PegasysAdaptor </strong>classes
serve as an important bridge from the Pegasys data structure to other
well-used standards and permits interoperability between data computed
using Pegasys and many other bioinformatics tools and databases.</p>
<h4>Parallelism</h4>
<p>Our local installation of Pegasys runs on a 28 CPU distributed
memory compute cluster that runs the OpenPBS parallel batch server <a name="IDAJBBTG"></a>[<a href="#B27">27</a>].
We have implemented 'serial' parallelism into the system meaning that
each application is a serial process, but many serial processes can be
run in parallel. It is important to note that this is distinct from
parallelism where a single application is itself implemented using a
message passing library that can use many distributed processors in a
compute cluster environment. To enable serial parallelism, we
implemented a Runnable thread class in the Pegasys application layer
that can navigate a command line argument of a <strong>PegasysProgram</strong>,
and create a script at runtime that is used to submit a job to a PBS
job queue. To monitor job progress, we implemented a Java server called
<strong>QstatServer</strong>, that registers each job sent to the PBS job queue. The <strong>QstatServer </strong>maintains
a hash table of jobs in the queue and informs the Pegasys application
layer when a particular job has terminated. This architecture enables
the Pegasys application server to execute jobs in sequence or in
parallel according to the structure of the DAG that was sent by the
client.</p>
<h4>Pegasys and Java</h4>
<p>The Pegasys system is implemented in the Java programming language.
Java offers robust data typing that facilitates object-oriented
programming in its truest form. The principles and advantages of
object-oriented design are well documented in the software engineering
literature (see <a name="IDAZBBTG"></a>[<a href="#B28">28</a>]). Java
is becoming widely adopted in the bioinformatics software domain. For
example, the Ensembl database has a Java API to programmatically access
genome annotations <a name="IDA4BBTG"></a>[<a href="#B29">29</a>]. The Biojava toolkit <a name="IDADCBTG"></a>[<a href="#B30">30</a>] is an extensive set of packages written in Java for sequence manipulation, analysis and processing. The Apollo genome editor <a name="IDAICBTG"></a>[<a href="#B11">11</a>],
that we use with Pegasys, allows biologists and bioinformaticians to
edit and create annotations in a sophisticated GUI and is written in
Java. We have integrated the Biojava toolkit into Pegasys for
manipulation of sequence files as well as parsing of BLAST output.
Using Java also allows us to make use of the JDBC library for database
connectivity that facilitates standard database interactions
independent of the RDBMS engine. To enable parallelism, we made use of
the robust Thread and Runnable classes that allow development of
multi-threaded programs.</p>
<p>We have designed Pegasys in a layered architecture that consists of
independent Java packages that can easily be imported into any external
Java application that wishes to make use of them. These packages are
well described in the Pegasys user manual, available at: <a href="http://bioinformatics.ubc.ca/pegasys/">http://bioinformatics.ubc.ca/pegasys/</a> <a href="http://www.webcitation.org/query.php?url=http://bioinformatics.ubc.ca/pegasys/&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>.
Implementing Pegasys in Java has brought the system strength and
robustness that would not have been attainable with using a scripting
language. Pegasys provides a Java alternative to existing Perl-based
sequence analysis systems such as GenDB <a name="IDARCBTG"></a>[<a href="#B3">3</a>] and BioPipe <a name="IDAWCBTG"></a>[<a href="#B31">31</a>].</p>
<h4>The Graphical User Interface</h4>
<p>The Pegasys graphical user interface (GUI) is designed for ease of
use while maximizing functionality. When the client is started, the
user sees a simple three pane design (see Figure <a name="IDA5CBTG"></a><a onclick="popup('/1471-2105/5/40/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F4">4</a>).
On left of the screen is a list of programs (the 'Tool Box') available
to the user. The list is retrieved from the server as an XML
configuration file when the client starts, ensuring all the programs
that are available to the user from the client are available on the
server. The canvas for drawing the workflow is on the upper right side
of the screen, and on the bottom of the screen there is a console to
view feedback from the client program.</p>
<p>The structure of the workflow the user creates on the canvas mirrors
the structure of the DAG (see The Pegasys data structure section). The
nodes of this DAG can either be input files, output files, or a
program, while the edges that connect the nodes manage the flow of
input and output information. For example, the Genscan program node can
produce many types of outputs, a list of nucleotide FASTAs of predicted
transcripts, or a list of amino acid FASTAs of the protein products. If
a user connects a BLASTP node to this Genscan node, then the edge
between these two nodes can be used to get the list of amino acid
FASTAs from the Genscan node as input for the BLASTP node.</p>
<p>During the creation of the workflow, the user can modify the
parameters of the analysis programs by double-clicking a node. This
opens a Node Properties dialogue. An example for BLAST is pictured in
Figure <a name="IDAGDBTG"></a><a onclick="popup('/1471-2105/5/40/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F5">5</a>.
The input/output types for each edge must be set during the creation of
the workflow. This is done through the Edge Properties dialogue (see
Figure <a name="IDAKDBTG"></a><a onclick="popup('/1471-2105/5/40/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F6">6</a>).</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/5/40/figure/F5','F5',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F5"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1471-2105-5-40-5.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 5.</strong></a> <strong>Screenshot of the Node Properties dialogue window where users can input parameters for the analysis programs. </strong>There
are three columns – the name of the parameter, its current value and a
check box to indicate if this parameter is enabled. Disabled parameters
will be excluded from the DAG XML, and consequently from the actual
command that is executed on the server. All default values are set in
the ProgramList.xml file that the server reads on startup.</p>
</div>
<div class="fig"><p></p><p><a onclick="popup('/1471-2105/5/40/figure/F6','F6',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F6"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1471-2105-5-40-6.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 6.</strong></a> <strong>Screenshot of the Edge Properties dialogue window where users set the inputs and outputs of an edge. </strong>The
input/output values are selected with drop-down select bars so users
can only select input/output types that are available to the two nodes.
Incompatible input/output types for an edge are not allowed by the GUI
and the user is alerted to the error. The input/output lists for each
node are set in the ProgramList.xml file that the server reads on
startup.</p>
</div></div><p>When the user has finished creating the workflow, it can
be saved as an XML file representing the DAG. This XML file stores all
the parameters for the nodes and edges that have been set by the user
during the creation of the DAG. This file can be kept on the local hard
drive and retrieved for later modification or distribution, or sent to
the server to be executed on the compute cluster. The saved DAG can
also be sent to the server using the command-line Java client for
high-throughput, or automated processing. When the processing is
complete, the results are sent back to the GUI client to be saved as
text files.</p>
<p>To ensure that the user's workflow is syntactically correct, the
Pegasys client validates the workflow in real time. As the user draw
nodes and edges, they are validated for correctness based on their
requirements. For example, if a Program Node has a required parameter
that is not filled in, the Pegasys client will display that node with a
red 'X' beside it. Once this required parameter is filled in, the red
'X' will turn into a green tick mark, indicating the correctness of
this node. Invalid edges are displayed in red, while correct ones are
displayed in black. Typically, edges will be invalid if the 'output'
and 'input' values of the edges are not set or do not match. If the
workflow has a red edge or a node marked with a red 'X', the Pegasys
client will not allow the user to send the workflow to the server and
will output a warning to the 'Console' area.</p>
<p>The GUI component of the Pegasys system is implemented in C++, using QT graphical libraries <a name="IDALEBTG"></a>[<a href="#B32">32</a>].
The QT libraries offer a "write once compile anywhere" approach.
Because the QT components are natively compiled for its target
operating system, GUI components written in C++/QT have a more native
look and feel and give fast response times to the user. In addition,
C++/QT can be compiled on all the major operating systems, giving it
nearly the same level of portability as Java and facilitating the
distribution of the Pegasys GUI client for most platforms.</p>
<h4>XML configuration files</h4><p>Communication between the client and
server is mediated through XML files. There are three key XML files in
the Pegasys client. The first XML file, the Pegasys configuration file
(PegasysConfig.xml), keeps track of the system settings for default
output directories on the server, queuing time for the scheduler,
location of Pegasys Java jar files, and database information. This file
also contains the path to the second XML file – the program list file
which list all of the programs and their associated parameters that are
currently available on the Pegasys server (ProgramList.xml). This file
needs to be updated whenever a new module is added to the server, or
the parameters of an existing module are changed. It is kept on the
server and is transmitted to the client every time it starts up to
inform the users of the available programs on the server and their
associated parameters.</p>
<p>The third XML file is the textual representation of the workflow.
This file is generated by saving the workflow using the client. It can
be sent to the server where it is parsed and then executed, or it can
be re-opened at a later time for further modification. For each node on
the canvas, its parameters, flags, and coordinates on the canvas are
recorded in the DAG XML file. Edges have their start and end nodes
recorded.</p>
<p>Communication via XML is one of the standard ways of disseminating
information on the Internet. Both Java for the backend and QT for the
client have ready-made parsers for XML. This allowed us to rapidly
build the software components that exchange information between the
client and the server.</p><a name="IDAVEBTG"></a><h3>Results and discussion</h3>
<p>To illustrate the flexibility of Pegasys for diverse analyses, we
chose three workflows to demonstrate as use cases for the system. The
simplest workflow takes an input sequence, runs a single analysis on
this sequence and saves the unprocessed results.</p>
<p>Figure <a name="IDA1EBTG"></a><a onclick="popup('/1471-2105/5/40/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F4">4</a>
shows an example of detecting repeats in a genomic sequence using
RepeatMasker. In this example, the unprocessed results are written to a
text file. This example is almost as if RepeatMasker were run locally
on the command line, except that all information about the parameters
used, the input sequence and the results are logged to the Pegasys
database.</p>
<p>Figure <a name="IDABFBTG"></a><a onclick="popup('/1471-2105/5/40/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F7">7</a>
shows a workflow that has two inputs. The first is a FASTA-formatted
nucleotide sequence file. This file is used as input to 'formatdb' – an
application that transforms FASTA-formatted databases into a format
that can be used by BLAST. The second input is a query sequence that
will be used to search the newly formatted database using BLAST. The
results of the search are outputted in a GFF-formatted text file.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/5/40/figure/F7','F7',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F7"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1471-2105-5-40-7.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 7.</strong></a> <strong>Workflow showing a BLAST pipeline. </strong>A
FASTA formatted database is to be formatted for BLAST using 'formatdb'.
A query sequence is then searched against this new database using
BLAST. The results are written to a text file in GFF format.</p>
</div></div><p>Figure <a name="IDATFBTG"></a><a onclick="popup('/1471-2105/5/40/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F8">8</a>
shows a workflow that would be suitable for annotation of eukaryotic
genomic sequence. The output of this workflow would serve as the input
for an annotation tool like Apollo. The DAG branches after the input
sequence File node into a sub-DAG of analyses that work on the input as
is and a sub-DAG that analyzes the input sequence that is masked for
repeats with RepeatMasker. The unmasked sequence is analysed for tRNAs
using tRNAscan-SE, and for protein coding genes using <em>ab initio </em>gene
predictors Genscan and HmmGene. The masked sequence is searched against
a database of curated proteins using BLASTX and against a database
compiled from ESTs, full-length cDNAs and mRNA sequences
(dbTranscript). The results from the latter search are further
processed by an application (bt2fasta) that filters all hits based on
taxonomy (in this case the user-inputted NCBI taxonid of the source
organism of the input sequence) and retrieves their full sequences.
This results in an organism-specific database of FASTA formatted
sequences consisting of the BLASTN against dbTranscript hits. The
unmasked input sequence is then used as input to Sim4, which in turn
aligns the input sequence to the entries in the organism specific
database. Results for all analyses are then integrated into a GAME XML
file for further interpretation using Apollo. The Pegasys XML DAG file
that includes the parameters for all programs is available for download
at <a href="http://bioinformatics.ubc.ca/pegasys/">http://bioinformatics.ubc.ca/pegasys/</a> <a href="http://www.webcitation.org/query.php?url=http://bioinformatics.ubc.ca/pegasys/&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/5/40/figure/F8','F8',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/5/40/figure/F8"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/1471-2105-5-40-8.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 8.</strong></a> <strong>Workflow for genome annotation. </strong>This workflow executes <em>ab initio </em>gene
prediction, tRNA detection, repeat detection, sequence similarity
searching against protein and transcript databases and alignments of
transcripts to genomic sequence. Results for all of these analyses are
integrated into a single GAME XML output file that can be inputted into
Apollo, where a user can create annotations on the original input
sequence.</p>
</div></div><p>These use cases provide good examples of how Pegasys can
be used in sequence-based bioinformatics analyses. The system itself is
by no means limited to these examples. In theory any Unix program or
script can be incorporated into the system and Pegasys could be used
for workflows for systems administration, or other high-level scripting.</p>
<h4>Comparison with other systems</h4>
<p>As mentioned above, there are other systems that are similar to Pegasys in philosophy and approach. The DiscoveryNet platform <a name="IDAQGBTG"></a>[<a href="#B33">33</a>]
is a system that integrates bioinformatics tools based on Grid
computing technologies. This system is a 'middleware' system that can
be used to create workflows of annotation tools. Pegasys differs from
the DiscoveryNet approach in two major ways. First, Pegasys provides a
rigorously defined data model for storing computational features that
is mapped by a relational backend database. The use case for
DiscoveryNet describes output in the form of text-based flat files.
Storing the data in a database allows it to be mined using SQL for
selective sub-sets of computational evidence and gives the user more
control over what they are interpreting. Second, the Pegasys system is
designed to create workflows on the fly using the GUI and XML. The
DiscoveryNet genome annotation workflow was programmed and any new
workflow would also require programming investment. DiscoveryNet uses
the concept of web-services and distributed computing. The architecture
of Pegasys is extensible to web service based analyses. We plan on
adding the capability of making remote calls to application servers and
being able to integrate their analysis results into the Pegasys
framework. This would give Pegasys the utmost flexibility and
extensibility by combining the power of locally installed applications
with remote web services.</p>
<p>The Biopipe framework <a name="IDAXGBTG"></a>[<a href="#B31">31</a>]
describes a framework for protocol-based bioinformatics. The protocols
are developed with the goal of creating reproducibility of results from
computational analyses. This idea complements Pegasys quite well and we
envisage using Pegasys to encode protocols by creating workflow
standards generated from the Pegasys GUI for specific types of analyses
(e.g. genome annotation or mass spectrometry peptide fragment
identification) that we can distribute to the Pegasys user community.
This will facilitate cross-comparison of results from similar
bioinformatics experiments performed on data sources in different
research labs, or by colleagues working in the same lab. In addition,
Pegasys can be used to compare results of different protocols designed
to address similar scientific problems.</p>
<h4>Future directions</h4>
<p>The work described in this paper has led us to consider many new
challenges for future work on Pegasys. While the specifications, the
data model and the software are mature enough to be used in a research
setting, there remain many features and enhancements to the system that
we are implementing in on-going work. We are adding new modules to
Pegasys for distribution to the community. We are implementing Pegasys
modules for the Infernal package that is driving the Rfam repository of
families of functional RNAs <a name="IDABHBTG"></a>[<a href="#B34">34</a>].
Our genome annotation work to date has focused largely on eukaryotic
systems, and we have therefore devoted most of our development time to
applications tuned for eukaryotic animal analysis. We are adding
modules for prokaryotic analysis (e.g. Glimmer <a name="IDAGHBTG"></a><a name="IDAJHBTG"></a>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>,<a href="#B36">36</a>]) and plants (Eugene <a name="IDAOHBTG"></a>[<a href="#B37">37</a>]) to complement the current tools in Pegasys.</p>
<p>From a software perspective, we hope to make Pegasys inter-operable
and compliant with additional existing Open Source bioinformatics
standards and specifications, namely BioSQL and Chado to allow data
computed with Pegasys to be used in other systems that employ and
interact with these specifications.</p><a name="IDATHBTG"></a><h3>Conclusions</h3>
<p>We have created a robust, modular, flexible software system for the
execution and integration of heterogeneous biological sequence
analyses. Pegasys can execute and integrate results from <em>ab initio </em>gene
prediction, pair-wise and multiple sequence alignments, RNA gene
detection and masking of repetitive sequences to greatly enhance and
automate several levels of the biological sequence analysis process.
The GUI allows users to create workflows of analyses by dragging and
dropping icons on a canvas and joining processes together by connecting
them with graphical 'edges'. Each analysis is highly configurable and
users are presented with the option to change all parameters that are
supported by the underlying program. Data integration is facilitated
through the creation of a data model to represent computational
evidence which is in turn implemented in a robust backend relational
database management system. The database API provides programmatic
access to the results through high-level methods that implement SQL
queries on the data. The Pegasys system is currently driving numerous
diverse sequence analysis projects and can be easily configured for
others.</p>
<p>Implemented in Java, the backend of Pegasys is inter-operable with a
growing number of bioinformatics tools developed in Java. Pegasys can
output text files in standard formats that can then be imported into
other tools for subsequent analysis or viewing. We are continually
adding to Pegasys through the development of additional modules and
methods of data integration. The flexibility, customization, modularity
and data integration capabilities of Pegasys make it an attractive
system to use in any high throughput sequence analysis endeavour. We
are releasing the source code of Pegasys under the GNU General Public
License with the hope that the bioinformatics community worldwide will
make use of our efforts and in turn contribute improvements in the
spirit of Open Source.</p><a name="IDA1HBTG"></a><h3>Availability and requirements</h3>
<p>Pegasys is available at <a href="http://bioinformatics.ubc.ca/pegasys/">http://bioinformatics.ubc.ca/pegasys/</a> <a href="http://www.webcitation.org/query.php?url=http://bioinformatics.ubc.ca/pegasys/&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>
and is distributed under the GNU General Public License. Pegasys is
designed to run on Unix based systems. Please consult the user manual
(available with the distribution) for detailed installation and
configuration instructions. The Pegasys server is written in Java and
has the following dependencies: Java 1.3.1 or higher, PostgreSQL 7.3.*,
JDBC driver for PostgreSQL 7.3.* and BioJava 1.2*. We have tested
Pegasys on a distributed memory cluster (recommended) running OpenPBS
2.3.16 to administer the job scheduling. In theory an SMP system
running OpenPBS should work, but this has not been tested. The system's
analysis programs include the following: NCBI BLAST 2.2.3, WU BLAST
2.0, EMBOSS 2.7.1 (for Smith-Waterman implementation only), tRNAscan-SE
1.23, the LAGAN toolkit 1.2, Sim4, Genscan 1.0, HMMgene 1.1, MaskerAid
(2001-11-08) and GeneSplicer. All of the analysis tools are freely
available to academics. For details please consult the Pegasys manual
available with the distribution. The server has successfully been
deployed and tested on a 28 CPU Linux cluster running RedHat 7.3.</p>
<p>The client is written in C++ and requires the QT libraries version
3.11, and gcc version 3.2.2. The client has been tested on Linux
Mandrake9.x, Solaris 8, Mac OSX, Windows98/NT/ME/XP.</p><a name="IDAGIBTG"></a><h3>Authors' contributions</h3>
<p>SS was the lead architect of the system and contributed to the
design and implementation and wrote most of this manuscript. DH was the
principal developer and contributed to the design and implementation of
the server and the GUI. JS contributed to the design of the project and
provided requirements to the developers who were designing the system.
GQ, GZ, JD, DL and TX all participated in the implementation of various
components of the system. BFFO conceived of the project, guided its
development, and edited this manuscript.</p><a name="IDAMIBTG"></a><h3>Acknowledgments</h3>
<p>BFFO would like to acknowledge GenomeBC for funding this project. DL
is supported by the CIHR/MSFHR Strategic Training Program in
Bioinformatics <a href="http://bioinformatics.bcgsc.ca/">http://bioinformatics.bcgsc.ca</a>.
TX is supported by CIHR grant #MOP-53259. We wish to thank Stefanie
Butland, Joanne Fox and Yong Huang for critical reviews of this
manuscript. We also thank Miroslav Hatas and Graeme Campbell for
systems and software installation and maintenance for the Pegasys
server.</p><a name="refs"></a><h3>References</h3>
<ol id="references">
<li id="B1"><a name="B1"></a> Hubbard T, Barker D, Birney E, Cameron G,
Chen Y, Clark L, Cox T, Cuff J, Curwen V, Down T, Durbin R, Eyras E,
Gilbert J, Hammond M, Huminiecki L, Kasprzyk A, Lehvaslaiho H, Lijnzaad
P, Melsopp C, Mongin E, Pettett R, Pocock M, Potter S, Rust A, Schmidt
E, Searle S, Slater G, Smith J, Spooner W, Stabenau A, Stalker J,
Stupka E, Ureta-Vidal A, Vastrik I, Clamp M: <strong> The Ensembl genome database project. </strong><p><em>Nucleic Acids Res</em> 2002,  <strong>30</strong><strong>:</strong>38-41. <a target="_blank" href="http://www.biomedcentral.com/pubmed/11752248">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11752248">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B1" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B1','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B2"><a name="B2"></a> Mungall CJ, Misra S, Berman BP, Carlson
J, Frise E, Harris N, Marshall B, Shu S, Kaminker JS, Prochnik SE,
Smith CD, Smith E, Tupy JL, Wiel C, Rubin GM, Lewis SE: <strong> An integrated computational pipeline and database to support whole-genome sequence annotation. </strong><p><em>Genome Biol</em> 2002.,  <strong>3</strong>(12)<strong>:</strong> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B2" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B2','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p>RESEARCH0081. Epub 2002 Dec 23. Review</p>
<p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B3"><a name="B3"></a> Meyer F,  Goesmann A,  McHardy A,  Bartels D,  Bekel T,  Clausen J,  Kalinowski J,  Linke B,  Rupp O,  Giegerich R,  Pühler A: <strong> GenDB – an open source genome annotation system for prokaryote genomes. </strong><p><em>Nucleic Acids Res</em> 2003,  <strong>31</strong>(8)<strong>:</strong>2187-2195. <a target="_blank" href="http://www.biomedcentral.com/pubmed/12682369">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12682369">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B3" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B3','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B4"><a name="B4"></a> Yuan Q,  Ouyang S,  Liu J,  Suh B,  Cheung F,  Sultana R,  Lee D,  Quackenbush J,  Buell C: <strong> The TIGR rice genome annotation resource: annotating the rice genome and creating resources for plant biologists. </strong><p><em>Nucleic Acids Res</em> 2003,  <strong>31</strong><strong>:</strong>229-233. <a target="_blank" href="http://www.biomedcentral.com/pubmed/12519988">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12519988">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B4" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B4','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B5"><a name="B5"></a> Korf I,  Flicek P,  Duan D,  Brent MR: <strong> Integrating genomic homology into gene structure prediction. </strong><p><em>Bioinformatics</em> 2001,  <strong>17</strong><strong>:</strong>S140-S148. <a target="_blank" href="http://www.biomedcentral.com/pubmed/11473003">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11473003">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B5" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B5','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p>Suppl 1</p>
<p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B6"><a name="B6"></a> Mathé C,  Déhais P,  Pavy N,  Rombauts S,  Van Montagu M,  Rouzé P: <strong> Gene prediction and gene classes in Arabidopsis thaliana. </strong><p><em>J Biotechnol</em> 2000,  <strong>78</strong>(3)<strong>:</strong>293-299. <a target="_blank" href="http://www.biomedcentral.com/pubmed/10751690">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10751690">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B6" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B6','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B7"><a name="B7"></a> Yeh R,  Lim L,  Burge C: <strong> Computational inference of homologous gene structures in the human genome. </strong><p><em>Genome Res</em> 2001,  <strong>11</strong>(5)<strong>:</strong>803-816. <a target="_blank" href="http://www.biomedcentral.com/pubmed/11337476">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11337476">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B7" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B7','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B8"><a name="B8"></a> Rogic S,  Ouellette B,  Mackworth A: <strong> Improving gene recognition accuracy by combining predictions from two gene-finding programs. </strong><p><em>Bioinformatics</em> 2002,  <strong>18</strong>(8)<strong>:</strong>1034-1045. <a target="_blank" href="http://www.biomedcentral.com/pubmed/12176826">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12176826">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B8" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B8','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B9"><a name="B9"></a><strong> General Feature Format</strong>
		[<a target="_blank" href="http://www.sanger.ac.uk/Software/formats/GFF/index.shtml">http://www.sanger.ac.uk/Software/formats/GFF/index.shtml</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.sanger.ac.uk/Software/formats/GFF/index.shtml&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></li>
<li id="B10"><a name="B10"></a><strong> GAME XML DTD</strong>
		[<a target="_blank" href="http://flybase.bio.indiana.edu/annot/gamexml.dtd.txt">http://flybase.bio.indiana.edu/annot/gamexml.dtd.txt</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://flybase.bio.indiana.edu/annot/gamexml.dtd.txt&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></li>
<li id="B11"><a name="B11"></a> Lewis SE, Searle SM, Harris N, Gibson
M, Lyer V, Richter J, Wiel C, Bayraktaroglir L, Birney E, Crosby MA,
Kaminker JS, Matthews BB, Prochnik SE, Smithy CD, Tupy JL, Rubin GM,
Misra S, Mungall CJ, Clamp ME: <strong> Apollo: a sequence annotation editor. </strong><p><em>Genome Biol</em> 2002.,  <strong>3</strong>(12)<strong>:</strong> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B11" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B11','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p>RESEARCH0082. Epub 2002 Dec 23. Review.</p>
<p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B12"><a name="B12"></a> Altschul S,  Gish W,  Miller W,  Myers E,  Lipman D: <strong> Basic local alignment search tool. </strong><p><em>J Mol Biol</em> 1990,  <strong>215</strong>(3)<strong>:</strong>403-410. <a target="_blank" href="http://www.biomedcentral.com/pubmed/2231712">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2231712">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B12" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B12','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B13"><a name="B13"></a> Altschul S,  Madden T,  Schäffer A,  Zhang J,  Zhang Z,  Miller W,  Lipman D: <strong> Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. </strong><p><em>Nucleic Acids Res</em> 1997,  <strong>25</strong>(17)<strong>:</strong>3389-3402. <a target="_blank" href="http://www.biomedcentral.com/pubmed/9254694">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9254694">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B13" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B13','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B14"><a name="B14"></a> Dowell R,  Jokerst R,  Day A,  Eddy S,  Stein L: <strong> The distributed annotation system. </strong><p><em>BMC Bioinformatics</em> 2001,  <strong>2</strong><strong>:</strong>7-7. <a target="_blank" href="http://www.biomedcentral.com/pubmed/11667947">PubMed Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-2-7">BioMed Central Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B14" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B14','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B15"><a name="B15"></a>R Development Core Team: <strong> R: A language and environment for statistical computing. </strong>
		[<a target="_blank" href="http://www.r-project.org/">http://www.R-project.org</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.R-project.org&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a><p><em>R Foundation for Statistical Computing, Vienna, Austria</em> 2003. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B15" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B15','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p>[ISBN 3-900051-00-3]</p>
<p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B16"><a name="B16"></a> Bedell J,  Korf I,  Gish W: <strong> Masker Aid: a performance enhancement to RepeatMasker. </strong><p><em>Bioinformatics</em> 2000,  <strong>16</strong>(11)<strong>:</strong>1040-1041. <a target="_blank" href="http://www.biomedcentral.com/pubmed/11159316">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11159316">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B16" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B16','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B17"><a name="B17"></a> Gish W: <strong> WU BLAST 2.0. </strong>
		[<a target="_blank" href="http://blast.wustl.edu/blast/README.html">http://blast.wustl.edu/blast/README.html</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://blast.wustl.edu/blast/README.html&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></li>
<li id="B18"><a name="B18"></a> Rice P,  Longden I,  Bleasby A: <strong> EMBOSS: the European Molecular Biology Open Software Suite. </strong><p><em>Trends Genet</em> 2000,  <strong>16</strong>(6)<strong>:</strong>276-277. <a target="_blank" href="http://www.biomedcentral.com/pubmed/10827456">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10827456">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B18" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B18','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B19"><a name="B19"></a> Smith T,  Waterman M: <strong> Identification of common molecular subsequences. </strong><p><em>J Mol Biol</em> 1981,  <strong>147</strong><strong>:</strong>195-197. <a target="_blank" href="http://www.biomedcentral.com/pubmed/7265238">PubMed Abstract</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B19" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B19','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B20"><a name="B20"></a> Burge C,  Karlin S: <strong> Prediction of complete gene structures in human genomic DNA. </strong><p><em>J Mol Biol</em> 1997,  <strong>268</strong><strong>:</strong>78-94. <a target="_blank" href="http://www.biomedcentral.com/pubmed/9149143">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9149143">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B20" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B20','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B21"><a name="B21"></a> Krogh A: <strong> Two methods for improving performance of an HMM and their application for gene finding. </strong><p><em>Proc Int Conf Intell Syst Mol Biol</em> 1997,  <strong>5</strong><strong>:</strong>179-186. <a target="_blank" href="http://www.biomedcentral.com/pubmed/9322033">PubMed Abstract</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B21" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B21','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B22"><a name="B22"></a> Brudno M,  Do C,  Cooper G,  Kim M,  Davydov E,  Green E,  Sidow A,  Batzoglou S: <strong> LAGAN and Multi-LAGAN: efficient tools for large-scale multiple alignment of genomic DNA. </strong><p><em>Genome Res</em> 2003,  <strong>13</strong>(4)<strong>:</strong>721-731. <a target="_blank" href="http://www.biomedcentral.com/pubmed/12654723">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12654723">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B22" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B22','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B23"><a name="B23"></a> Florea L,  Hartzell G,  Zhang Z,  Rubin G,  Miller W: <strong> A computer program for aligning a cDNA sequence with a genomic DNA sequence. </strong><p><em>Genome Res</em> 1998,  <strong>8</strong>(9)<strong>:</strong>967-974. <a target="_blank" href="http://www.biomedcentral.com/pubmed/9750195">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9750195">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B23" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B23','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B24"><a name="B24"></a> Lowe T,  Eddy S: <strong> tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. </strong><p><em>Nucleic Acids Res</em> 1997,  <strong>25</strong>(5)<strong>:</strong>955-964. <a target="_blank" href="http://www.biomedcentral.com/pubmed/9023104">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9023104">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B24" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B24','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B25"><a name="B25"></a> Pertea M,  Lin X,  Salzberg S: <strong> GeneSplicer: a new computational method for splice site prediction. </strong><p><em>Nucleic Acids Res</em> 2001,  <strong>29</strong>(5)<strong>:</strong>1185-1190. <a target="_blank" href="http://www.biomedcentral.com/pubmed/11222768">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11222768">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B25" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B25','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B26"><a name="B26"></a> Stein L,  Mungall C,  Shu S,  Gaudy M,  Mangone M,  Day A,  Nickerson E,  Stajich J,  Harris T,  Arva A,  Lewis S: <strong> The generic genome browser: a building block for a model organism system database. </strong><p><em>Genome Res</em> 2002,  <strong>12</strong>(10)<strong>:</strong>1599-1610. <a target="_blank" href="http://www.biomedcentral.com/pubmed/12368253">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12368253">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B26" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B26','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B27"><a name="B27"></a><strong> OpenPBS</strong>
		[<a target="_blank" href="http://www.openpbs.org/">http://www.openpbs.org</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.openpbs.org&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></li>
<li id="B28"><a name="B28"></a> Booch G: <p><em>Object-oriented Analysis and Design with Applications</em> The Benjamin/Cummings Publishing Company  1994. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B28" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B28','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B29"><a name="B29"></a><strong> Ensj</strong>
		[<a target="_blank" href="http://www.ensembl.org/java/">http://www.ensembl.org/java/</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.ensembl.org/java/&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></li>
<li id="B30"><a name="B30"></a><strong> BioJava.org</strong>
		[<a target="_blank" href="http://www.biojava.org/">http://www.biojava.org</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.biojava.org&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></li>
<li id="B31"><a name="B31"></a> Hoon S,  Ratnapu K,  Chia J,  Kumarasamy B,  Juguang X,  Clamp M,  Stabenau A,  Potter S,  Clarke L,  Stupka E: <strong> Biopipe: a flexible framework for protocol-based bioinformatics analysis. </strong><p><em>Genome Res</em> 2003,  <strong>13</strong>(8)<strong>:</strong>1904-1915. <a target="_blank" href="http://www.biomedcentral.com/pubmed/12869579">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12869579">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B31" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B31','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B32"><a name="B32"></a><strong> Trolltech – Qt Overview</strong>
		[<a target="_blank" href="http://www.trolltech.com/products/qt/index.html">http://www.trolltech.com/products/qt/index.html</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.trolltech.com/products/qt/index.html&amp;refdoi=10.1186/1471-2105-5-40" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></li>
<li id="B33"><a name="B33"></a> Rowe A,  Kalaitzopoulos D,  Osmond M,  Ghanem M,  Guo Y: <strong> The discovery net system for high throughput bioinformatics. </strong><p><em>Bioinformatics</em> 2003,  <strong>19</strong>(Suppl 1)<strong>:</strong>225-225.  <a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btg1031">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B33" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B33','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B34"><a name="B34"></a> Griffiths-Jones S,  Bateman A,  Marshall M,  Khanna A,  Eddy S: <strong> Rfam: an RNA family database. </strong><p><em>Nucleic Acids Res</em> 2003,  <strong>31</strong><strong>:</strong>439-441. <a target="_blank" href="http://www.biomedcentral.com/pubmed/12520045">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12520045">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B34" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B34','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B35"><a name="B35"></a> Delcher A,  Harmon D,  Kasif S,  White O,  Salzberg S: <strong> Improved microbial gene identification with GLIMMER. </strong><p><em>Nucleic Acids Res</em> 1999,  <strong>27</strong>(23)<strong>:</strong>4636-4641. <a target="_blank" href="http://www.biomedcentral.com/pubmed/10556321">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10556321">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B35" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B35','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B36"><a name="B36"></a> Salzberg S,  Delcher A,  Kasif S,  White O: <strong> Microbial gene identification using interpolated Markov models. </strong><p><em>Nucleic Acids Res</em> 1998,  <strong>26</strong>(2)<strong>:</strong>544-548. <a target="_blank" href="http://www.biomedcentral.com/pubmed/9421513">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9421513">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B36" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B36','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
<li id="B37"><a name="B37"></a> Schiex T,  A M,  P R: <strong> EUGENE: An Eukaryotic Gene Finder That Combines Several Sources of Evidence. </strong><p><em>In JOBIM</em> 2000, 111-125.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B37" onclick="popup('/sfx_links.asp?ui=1471-2105-5-40&amp;bibl=B37','SFXMenu','460','420'); return false;"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>
</li>
</ol>
<h4 class="bottommessage"><a href="http://www.biomedcentral.com/1471-2105/5/40/postcomment">Have something to say? Post a comment on this article!</a></h4><script type="text/javascript">initializeOutline();</script></td></tr></tbody></table></div>

<center>

<!------ OAS AD 'Bottom1' begin ------>
<script language="JavaScript">
<!--
OAS_AD('Bottom1');
//-->
</script><a href="http://oas.biomedcentral.com/5c/www.biomedcentral.com/article/bmcbioinformatics/2001365529/Bottom1/default/empty.gif/34386665393030393439386238646230" target="_blank"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/empty.gif" alt="" border="0" height="2" width="2"></a><noscript>
<a
href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1570519025@Top,Bottom,Left1,Bottom1!Bottom1"
target=_top>
<img
src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1570519025@Top,Bottom,Left1,Bottom1!Bottom1"
border=0></a>
</noscript>
<!------ OAS AD end ------>
</center>

<hr color="#cccccc" noshade="noshade" size="1">
<table cellpadding="3" width="100%"><tbody><tr valign="middle"><td align="center">


<center>

<!------ OAS AD 'Bottom' begin ------>
<script language="JavaScript">
<!--
OAS_AD('Bottom');
//-->
</script><a href="http://oas.biomedcentral.com/5c/www.biomedcentral.com/article/bmcbioinformatics/1541802639/Bottom/biomed/biomarkworldc-contra-ban/BMC_biomarker_468x60.JPG/34386665393030393439386238646230" target="_blank"><img src="Shah%202004%20Pegasys%20-%20software%20for%20executing%20and%20integrating%20analyses%20of%20biological%20sequences_files/BMC_biomarker_468x60.JPG" alt="" border="0"></a><noscript><a
href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1570519025@Top,Bottom,Left1,Bottom1!Bottom"
target=_top>
<img
src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1570519025@Top,Bottom,Left1,Bottom1!Bottom"
border=0></a>
</noscript>
<!------ OAS AD end ------>
</center>

</td></tr>
<tr><td align="center"><div class="smalltext">
<a href="http://www.biomedcentral.com/info/about/tandc">Terms and Conditions</a> 
<a href="http://www.biomedcentral.com/info/about/privacy">Privacy statement</a> 
<a href="http://www.biomedcentral.com/info/mediainformation/">Information for advertisers</a> 
<a href="http://www.biomedcentral.com/info/about/bmcjobs">Jobs at BMC</a> 
<a href="mailto:info@biomedcentral.com">Contact us</a> 
</div>
</td></tr></tbody></table>
<table class="footer2t" border="0" cellpadding="0" cellspacing="0" width="100%">
<tbody><tr>
<td colspan="3" align="center" bgcolor="#003399">
© 1999-2009 BioMed Central Ltd unless otherwise stated.
 Part of <a href="http://www.springer.com/">Springer Science+Business Media</a>.
</td>
</tr>
</tbody></table>
</div>
</div>
</center>
</body></html>