<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xml:lang="en" xmlns="http://www.w3.org/1999/xhtml" lang="en"><head>                
































	
		
						
																																																																																																
																					
																					
 														 		
					






























 






        
               
      







        
                
 	



 










	
				

        
        
 

 





 
 
 


 
 
  


	







<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<meta name="robots" content="noarchive">

<meta name="keywords" content="Nature, science, science news, biology, physics, genetics, astronomy, astrophysics, quantum physics, evolution, evolutionary biology, geophysics, climate change, earth science, materials science, interdisciplinary science, science policy, medicine, systems biology, genomics, transcriptomics, palaeobiology, ecology, molecular biology, cancer, immunology, pharmacology, development, developmental biology, structural biology, biochemistry, bioinformatics, computational biology, nanotechnology, proteomics, metabolomics, biotechnology, drug discovery, environmental science, life, marine biology, medical research, neuroscience, neurobiology, functional genomics, molecular interactions, RNA, DNA, cell cycle, signal transduction, cell signalling">
<meta name="description" content="Nature - the world's best science and medicine on your desktop">
<link rel="shortcut icon" href="http://www.nature.com/favicon.ico">
<link rel="home" href="http://www.nature.com/nbt/" title="home">
<link rel="copyright" href="http://www.nature.com/common/info/copyright_statement.html" title="copyright">
<link rel="help" title="help" href="http://www.nature.com/help/">
	

		<title>Next-generation DNA sequencing : Article : Nature Biotechnology</title>
	

	
	

	
																																	<style type="text/css" media="screen, projection, print"><!--/*--><![CDATA[/*><!--*/
	/* high pass filter to hide @imports from browsers with poor CSS1 parsing - also hide from mac IE \*/
	@import "/common/style/null.css?\"\{";
	@import "/common/includes/header/03/style/header.css";	@import "/common/includes/footer/03/style/footer.css";
		    @import "/common/style/layout.css";
		    @import "/common/style/layout_wide_banner.css";
		    @import "/common/style/main.css";
		    @import "/common/style/header_footer_smallfonts.css";
		    @import "/rj/style/group.css";
		    @import "/nbt/new/style/site.css";
		
	 		@import "/common/style/restrict_width/966px.css";
						#hdr .login-nav {background-image: url(/nbt/images/journal_header_v3.jpg);}
				
/* end hide from mac ie */
/*]]>*/--></style>
<link rel="stylesheet" type="text/css" href="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/print.css" media="print" title="Print Stylesheet">
	

	<link title="schema(PRISM)" rel="schema.prism" href="http://prismstandard.org/namespaces/1.2/basic/">
																																														<meta name="prism.copyright" content="© 2008 Nature Publishing Group">
			<meta name="prism.rightsAgent" content="permissions@nature.com">
			<meta name="prism.publicationName" content="">
			<meta name="prism.issn" content="${footerJournalISSN}">
			<meta name="prism.eIssn" content="${footerJournalEISSN}">
			<meta name="prism.publicationDate" content="2008-10-09">
			<meta name="prism.startingPage" content="1135">
			<meta name="prism.endingPage" content="1145">
	
	<link title="schema(DC)" rel="schema.dc" href="http://purl.org/dc/elements/1.1/">
																																																			<meta name="dc.publisher" content="Nature Publishing Group">
			<meta name="dc.language" content="en">
			<meta name="dc.rights" content="© 2008 Nature Publishing Group">
			<meta name="dc.title" content="Next-generation DNA sequencing">
			<meta name="dc.creator" content="Jay Shendure">
			<meta name="dc.creator" content="Hanlee Ji">
			<meta name="dc.identifier" content="doi:10.1038/nbt1486">
			<meta name="dc.source" content=", Published online: 09 October 2008; | doi:10.1038/nbt1486">
			<meta name="dc.date" content="2008-10-09">
	
																																																		<meta name="citation_journal_title" content="">
			<meta name="citation_publisher" content="Nature Publishing Group">
			<meta name="citation_authors" content="Jay Shendure, Hanlee Ji">
			<meta name="citation_title" content="Next-generation DNA sequencing">
			<meta name="citation_date" content="2008-10-09">
			<meta name="citation_volume" content="26">
			<meta name="citation_issue" content="10">
			<meta name="citation_firstpage" content="1135">
			<meta name="citation_doi" content="doi:10.1038/nbt1486">
	
										<meta name="WT.cg_n" content="Nature Biotechnology 2008-10 26">
	
<script type="text/javascript"><!--//--><![CDATA[//><!--
//--><!]]></script>



																																																		
<!--[if lte IE 7]>
<style type="text/css" media="screen, projection">
	@import "/common/style/null.css?\"\{";
	@import "/common/includes/header/03/style/ie-specific/lte-ie-7.css";
	@import "/rj/style/ie-specific/lte-ie-7.css";
</style>
<![endif]-->
																																																																												
<!--[if IE 7]>
<style type="text/css" media="screen, projection">
	@import "/common/includes/header/03/style/ie-specific/ie-7.css";
	@import "/common/style/ie-specific/ie-7.css";
	@import "/rj/style/ie-specific/ie-7.css";
</style>
<![endif]-->
																																																																																																																
<!--[if IE 6]>
<script type="text/javascript" src="/common/scripts/ie-specific/ie-6.js"></script>
<style type="text/css" media="screen, projection">
	@import "/common/includes/header/03/style/ie-specific/ie-6.css";
	@import "/common/includes/footer/03/style/ie-specific/ie-6.css";
	@import "/common/style/ie-specific/ie-6.css";
	@import "/rj/style/ie-specific/ie-6.css";
	@import "/nbt/new/style/ie-specific/ie-6.css";
</style>
<![endif]-->
</head><body id="article" class="www-nature-com-nbt">			<div id="constrain"><div class="constrain">


<div id="hdr">
			<div class="publications-search" id="top">
			<div class="publications">
				<ul>
														<li class="first jumplink"><a href="#content">Jump&nbsp;to&nbsp;main&nbsp;content</a></li>
										<li class="jumplink"><a href="#journalnav">Jump&nbsp;to&nbsp;navigation</a></li>
																		<li class="nature-link"><a href="http://www.nature.com/">nature.com<span class="hidden"> homepage</span></a></li>
																	<li class="pub1"><a href="http://www.nature.com/siteindex/">Publications&nbsp;A-Z&nbsp;index</a></li><li><a href="http://www.nature.com/browse/">Browse&nbsp;by&nbsp;subject</a></li>								
																																																																																																																	</ul>
			</div>	
		</div>
	
	<span class="cleardiv"><!-- --></span>

			<div class="leaderboard-links">
									
																																							
			
							<div class="user-services">
																					<ul class="list">
																						<li class="us-link li1"><a href="http://www.nature.com/myaccount/">My account</a></li><li class="us-link li2"><a href="http://mts-nbt.nature.com/cgi-bin/main.plex" title="submit manuscript">Submit manuscript</a></li>															<li class="us-link li3"><a href="http://www.nature.com/register/" title="Register with Nature.com">Register</a></li><li class="us-link li4"><a href="https://secure.nature.com/subscribe/nbt" title="Subscribe to Nature Biotechnology">Subscribe</a></li>													</ul>
																<span class="cleardiv"><!-- --></span>
								</div>
						
							<!-- leaderboard ad -->
				<div style="overflow: visible;" class="leaderboard">
										<div style="overflow: visible;" class="padder">
																								

				






<script type="text/javascript"><!--//--><![CDATA[//><!--
	var include = '';

	var wp = '';
	if (wp != '') include += 'wp='+wp;
	include += ';';

	var rf = '';
	if (rf != '') include += "rf="+rf;
	include += ';';
	
	var sp = '';
	if (sp != '') include += "sp="+sp;
	include += ";";
	
	var ps = '';
	if (ps != '') include += "ps="+ps;
	include += ";";
	
	var adinf = "lg=no;";

	if(include != ";;;;")
	{
		adinf = include;
	}

	var time = new Date();
	ordval= (time.getTime());
	
		ord=Math.random()*10000000000000000;
//--><!]]></script>

	








		
<div class="adunit-leaderboard">
<script type="text/javascript"><!--//--><![CDATA[//><!--
		document.write('<script type="text/javascript" src="http://ad.doubleclick.net/adj/biotech.nature.com/article;abr=!webtv;type=re;artid=nbt1486;date=20081009;issue=10;subco=;chaco=;keywd=;njdis=;njreg=;crstg=;pos=top;sz=728x90;ptile=1;ord=' + ord + '?" ><\/script>');

	if ((!document.images && navigator.userAgent.indexOf('Mozilla/2.') >= 0) || (navigator.userAgent.indexOf("WebTV") >= 0)) {
		document.write('<a href="http://ad.doubleclick.net/jump/biotech.nature.com/article;keywd=;sz=728x90;ord=' + ord + '?">');
		document.write('<img src="http://ad.doubleclick.net/ad/biotech.nature.com/article;keywd=;sz=728x90;ord=' + ord + '?" width="728" height="90" border="0"></a>');
	}
//--><!]]></script><script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/article.js"></script><!-- Template Id = 3879 Template Name = Banner Creative (NEW NPG Flash) -->
<!-- Created by Matt Peskett - 11-10-2006 clickTAG instead if clickTag. -->
<!-- Copyright 2002 DoubleClick Inc., All rights reserved. -->
<script src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/flashwrite_1_2.js"></script>
<object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" id="FLASH_AD" height="90" width="728"><param style="" name="movie" value="http://m1.2mdn.net/1615871/728x90_nature_v3.swf?clickTAG=http%3A//ad.doubleclick.net/click%253Bh%3Dv8/379f/3/0/%252a/u%253B209712161%253B0-0%253B0%253B17389285%253B3454-728/90%253B29523645/29541524/1%253B%253B%257Esscs%253D%253fhttp%3A//www.master-biotech.ie.edu/index.php%3F%20continente%3D2%26ns_campaign%3D_probetas_arista_banner_728x90_ros_pr_mbm_int_20081200_nuria_en%26ns_mchannel%3Dbanner%26ns_source%3Dnature%26ns_linkname%3D_pr_mbm_int_20081200_nuria_en%26ns_fee%3D1"><param style="" name="quality" value="high"><param style="" name="bgcolor" value="#"><param style="" name="wmode" value="opaque"><embed src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/728x90_nature_v3.swf" quality="high" wmode="opaque" swliveconnect="TRUE" bgcolor="#" type="application/x-shockwave-flash" height="90" width="728"></object><noembed style="">&lt;A
TARGET="blank"
HREF="http://ad.doubleclick.net/click%3Bh=v8/379f/3/0/%2a/u%3B209712161%3B0-0%3B0%3B17389285%3B3454-728/90%3B29523645/29541524/1%3B%3B%7Esscs%3D%3fhttp://www.master-biotech.ie.edu/index.php?
continente=2&amp;ns_campaign=_probetas_arista_banner_728x90_ros_pr_mbm_int_20081200_nuria_en&amp;ns_mchannel=banner&amp;ns_source=nature&amp;ns_linkname=_pr_mbm_int_20081200_nuria_en&amp;ns_fee=1"&gt;&lt;IMG
SRC="http://m1.2mdn.net/1615871/728x90_2.gif" BORDER=0&gt;&lt;/A&gt;</noembed><noscript style=""><A
TARGET="blank"
HREF="http://ad.doubleclick.net/click%3Bh=v8/379f/3/0/%2a/u%3B209712161%3B0-0%3B0%3B17389285%3B3454-728/90%3B29523645/29541524/1%3B%3B%7Esscs%3D%3fhttp://www.master-biotech.ie.edu/index.php?
continente=2&ns_campaign=_probetas_arista_banner_728x90_ros_pr_mbm_int_20081200_nuria_en&ns_mchannel=banner&ns_source=nature&ns_linkname=_pr_mbm_int_20081200_nuria_en&ns_fee=1"><IMG
SRC="http://m1.2mdn.net/1615871/728x90_2.gif" BORDER=0></A></noscript>
<noscript style=""> <div><a
href="http://ad.doubleclick.net/jump/biotech.nature.com/article;abr=!NN2;tile=1;type=re;artid=nbt1486;date=20081009;issue=10;subco=;chaco=;keywd=;njdis=;njreg=;crstg=;pos=top;sz=728x90;ptile=1;ord=123456789?"><img
src="http://ad.doubleclick.net/ad/biotech.nature.com/article;abr=!NN2;tile=1;type=re;artid=nbt1486;date=20081009;issue=10;subco=;chaco=;keywd=;njdis=;njreg=;crstg=;pos=top;sz=728x90;ptile=1;ord=123456789?"
alt="Advertisement" style="border:0" /></a></div>
</noscript>

</div>


					</div>
				</div>
				<!-- end leaderboard ad -->
					</div>
		
		<span class="cleardiv"><!-- --></span>
		
	<div class="login-nav">
		
						
					
						
						
											<div class="logon">
					<p><a href="http://www.nature.com/foxtrot/svc/login" class="login" title="login to Nature.com">Login</a></p>
									</div>
													
														
		<a href="http://www.nature.com/nbt/" class="journal-header" title="Nature Biotechnology homepage"><img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/journal_header_v3.jpg" alt="Nature Biotechnology homepage" style="display: block;" class="journal-header-image"></a>		
				
				<form class="search" action="/search/executeSearch?" method="get">
			<div>
				<label for="searchtext">Search</label>
								<select name="sp-q-9">
										<option value="NBT">This journal</option>
										<option value="">All of nature.com</option>
									</select>
								<input id="searchtext" name="sp-q" type="text">
				<input name="sp-c" value="10" type="hidden">
								<input name="sp-x-9" value="cat" type="hidden">
								<input name="sp-s" value="0" type="hidden">
				<input name="submit" class="button" value="go" title="search now" type="submit">
								<a href="http://www.nature.com/search/adv_search?sp-a=sp1001702d&amp;sp-x-1=ujournal&amp;sp-q-1=Nature%20Biotechnology">Advanced&nbsp;search</a>
						
								<input name="sp-a" value="sp1001702d" type="hidden">
								<input name="sp-sfvl-field" value="subject|ujournal" type="hidden">
								<input name="sp-x-1" value="ujournal" type="hidden">
								<input name="sp-p-1" value="phrase" type="hidden">
								<input name="sp-p" value="all" type="hidden">
							</div>
		</form>
		
							<span class="cleardiv"><!-- --></span>
			</div>
</div>


		<div id="breadcrumb"><div><a href="http://www.nature.com/nbt/index.html">Journal home</a><span class="divider"> &gt; </span>
				 <a href="http://www.nature.com/nbt/archive/">Archive</a><span class="divider"> &gt; </span><a href="http://www.nature.com/nbt/journal/v26/n10/index.html#re">Research</a><span class="divider"> &gt; </span><a href="http://www.nature.com/nbt/journal/v26/n10/index.html#rv">Review</a><span class="divider"> &gt; </span><span class="thisitem">Full Text</span></div></div><div id="content-journalnav"><div id="content"><h1 class="page-header">Review</h1>
<!-- Start Focus section -->
	<!-- Focus content -->
	<div id="toc">


	<h4 class="focus-title">Focus on Next-Generation Sequencing</h4>

	<div class="issue foc">

	<div class="issue-links" style="background: transparent url(/nbt/focus/sequencing/images/focus_bg_1008.jpg) no-repeat scroll left top; -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;">
<dl class="issue">
		<dt>Focus issue: </dt>
		<dd><a href="http://www.nature.com/nbt/journal/v26/n10/index.html">October<span class="hidden"></span> 2008</a> Volume <span class="journalnumber">26</span>, <abbr title="number">No</abbr> 10</dd>
		</dl>


	<ul class="focsup-anchor"><li class="right"><ul><li class="out"><a href="http://www.nature.com/nbt/journal/v26/n10/full/nbt1008-1113">Commentary</a></li><li class="out"><a href="http://www.nature.com/nbt/journal/v26/n10/index.html#pe">Perspectives</a></li><li class="out"><a href="http://www.nature.com/nbt/journal/v26/n10/index.html#rv">Reviews</a></li><li class="out"><a href="http://www.nature.com/nbt/focus/sequencing/sponsors.html">Sponsors</a></li></ul>
		</li><li class="left"><ul><li class="out"><a href="http://www.nature.com/nbt/focus/sequencing/index.html">Contents</a></li><li class="out"><a href="http://www.nature.com/nbt/journal/v26/n10/full/nbt1008-1099.html">Editorial</a></li><li class="out"><a href="http://www.nature.com/nbt/journal/v26/n10/full/nbt1008-1100.html">Profile</a></li><li class="out"><a href="http://www.nature.com/nbt/journal/v26/n10/index.html#nf">News Features</a></li></ul></li></ul>	


	</div>	

<span class="cleardiv"><!-- --></span>
	</div>
	<!-- end Focus box -->


</div>

<!-- end Focus section --><hr class="separator">
<p id="cite"><span class="journalname">Nature Biotechnology</span> <span class="b">26</span>, 1135 - 1145 (2008) <br>Published online: 9 October 2008 | <span class="doi"><abbr title="Digital Object Identifier">doi</abbr>:10.1038/nbt1486</span></p><p id="errorcor"></p><h2 id="atl">Next-generation DNA sequencing</h2><p id="aug">Jay Shendure<sup><a title="affiliated with " href="#a1">1</a></sup>
						&amp;
			Hanlee Ji<sup><a title="affiliated with " href="#a2">2</a></sup></p><p class="caff"></p><hr class="separator">
<div id="abs"><h3 class="hidden">Abstract</h3><p class="lead">DNA
sequence represents a single format onto which a broad range of
biological phenomena can be projected for high-throughput data
collection. Over the past three years, massively parallel DNA
sequencing platforms have become widely available, reducing the cost of
DNA sequencing by over two orders of magnitude, and democratizing the
field by putting the sequencing capacity of a major genome center in
the hands of individual investigators. These new technologies are
rapidly evolving, and near-term challenges include the development of
robust protocols for generating sequencing libraries, building
effective new approaches to data-analysis, and often a rethinking of
experimental design. Next-generation DNA sequencing has the potential
to dramatically accelerate biological and biomedical research, by
enabling the comprehensive analysis of genomes, transcriptomes and
interactomes to become inexpensive, routine and widespread, rather than
requiring significant production-scale efforts.</p></div><hr class="separator">
<div id="articlebody"><h3 class="hidden">Introduction</h3><p class="norm">The field of DNA sequencing technology development has a rich and diverse history<sup><a href="#B1">1, </a></sup><sup><a href="#B2">2</a></sup>. However, the overwhelming majority of DNA sequence production to date has relied on some version of the Sanger biochemistry<sup><a href="#B3">3</a></sup>.
Over the past five years, the incentive for developing entirely new
strategies for DNA sequencing has emerged on at least four levels,
undeniably reinvigorating this field (for a review, see ref. <a href="#B4">4</a>).
First, in the wake of the Human Genome Project, there are few remaining
avenues of optimization through which significant reductions in the
cost of conventional DNA sequencing can be achieved. Second, the
potential utility of short-read sequencing has been tremendously
strengthened by the availability of whole genome assemblies for <i>Homo sapiens</i>
and all major model organisms, as these effectively provide a reference
against which short reads can be mapped. Third, a growing variety of
molecular methods have been developed, whereby a broad range of
biological phenomena can be assessed by high-throughput DNA sequencing
(e.g., genetic variation, RNA expression, protein-DNA interactions and
chromosome conformation). And fourth, general progress in technology
across disparate fields, including microscopy, surface chemistry,
nucleotide biochemistry, polymerase engineering, computation, data
storage and others, have made alternative strategies for DNA sequencing
increasingly practical to realize.</p><p class="norm">Here, we review
the current crop of next-generation DNA sequencing platforms: how they
work, their relative strengths and limitations, and current and
emerging applications. We briefly discuss related developments in this
field, such as new software tools and front-end methods for isolating
arbitrary genomic subsets. We emphasize that the DNA sequencing
technology field has become a quickly moving target, and we can at best
provide a snapshot of this particular moment.</p><div id="Sanger-sequencing"><h4 class="norm">Sanger sequencing</h4><p class="follows-h4">Since
the early 1990s, DNA sequence production has almost exclusively been
carried out with capillary-based, semi-automated implementations of the
Sanger biochemistry<sup><a href="#B3">3, </a></sup><sup><a href="#B5">5, </a></sup><sup><a href="#B6">6</a></sup> (<a href="#f1">Fig. 1a</a>). In high-throughput production pipelines, DNA to be sequenced is prepared by one of two approaches: first, for shotgun <i>de novo</i> sequencing, randomly fragmented DNA is cloned into a high-copy-number plasmid, which is then used to transform <i>Escherichia coli;</i>
or second, for targeted resequencing, PCR amplification is carried out
with primers that flank the target. The output of both approaches is an
amplified template, either as many 'clonal' copies of a single plasmid
insert present within a spatially isolated bacterial colony that can be
picked, or as many PCR amplicons present within a single reaction
volume. The sequencing biochemistry takes place in a 'cycle sequencing'
reaction, in which cycles of template denaturation, primer annealing
and primer extension are performed. The primer is complementary to
known sequence immediately flanking the region of interest. Each round
of primer extension is stochastically terminated by the incorporation
of fluorescently labeled dideoxynucleotides (ddNTPs). In the resulting
mixture of end-labeled extension products, the label on the terminating
ddNTP of any given fragment corresponds to the nucleotide identity of
its terminal position. Sequence is determined by high-resolution
electrophoretic separation of the single-stranded, end-labeled
extension products in a capillary-based polymer gel. Laser excitation
of fluorescent labels as fragments of discreet lengths exit the
capillary, coupled to four-color detection of emission spectra,
provides the readout that is represented in a Sanger sequencing
'trace'. Software translates these traces into DNA sequence, while also
generating error probabilities for each base-call<sup><a href="#B7">7, </a></sup><sup><a href="#B8">8</a></sup>.
The approach that is taken for subsequent analysis&#8212;for example, genome
assembly or variant identification&#8212;depends on precisely what is being
sequenced and why. Simultaneous electrophoresis in 96 or 384
independent capillaries provides a limited level of parallelization.</p><div id="f1" class="figure-table"><div class="padding clearfix"><h3 class="norm"><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_F1.html">
Figure 1:&nbsp;Work flow of conventional versus second-generation sequencing.</a></h3><div class="image"><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_F1.html"><img alt="Figure 1 : Work flow of conventional versus second-generation sequencing." class="thumb" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/nbt1486-F1.gif"></a></div><div class="text"><p class="">(<b>a</b>)
With high-throughput shotgun Sanger sequencing, genomic DNA is
fragmented, then cloned to a plasmid vector and used to transform <i>E. coli.</i>
For each sequencing reaction, a single bacterial colony is picked and
plasmid DNA isolated. Each cycle sequencing reaction takes place within
a microliter-scale volume, generating a ladder of ddNTP-terminated,
dye-labeled products, which are subjected to high-resolution
electrophoretic separation within one of 96 or 384 capillaries in one
run of a sequencing instrument. As fluorescently labeled fragments of
discrete sizes pass a detector, the four-channel emission spectrum is
used to generate a sequencing trace. (<b>b</b>) In shotgun sequencing
with cyclic-array methods, common adaptors are ligated to fragmented
genomic DNA, which is then subjected to one of several protocols that
results in an array of millions of spatially immobilized PCR colonies
or 'polonies'<sup><a href="#B15">15</a></sup>. Each polony consists of
many copies of a single shotgun library fragment. As all polonies are
tethered to a planar array, a single microliter-scale reagent volume
(e.g., for primer hybridization and then for enzymatic extension
reactions) can be applied to manipulate all array features in parallel.
Similarly, imaging-based detection of fluorescent labels incorporated
with each extension can be used to acquire sequencing data on all
features in parallel. Successive iterations of enzymatic interrogation
and imaging are used to build up a contiguous sequencing read for each
array feature.</p><a class="full" href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_F1.html">Full size image
	(57 <abbr title="Kilobyte">KB</abbr>)
</a></div><span class="cleardiv"><!-- --></span>
</div></div><br class="clear"><p class="norm">After three decades of gradual improvement, the Sanger biochemistry can be applied to achieve read-lengths of up to <img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_003.gif" style="border: 0pt none ; vertical-align: baseline;" alt="approx">1,000
bp, and per-base 'raw' accuracies as high as 99.999%. In the context of
high-throughput shotgun genomic sequencing, Sanger sequencing costs on
the order of $0.50 per kilobase.</p></div><div id="Secondgeneration-DNA-sequencing"><h4 class="norm">Second-generation DNA sequencing</h4><p class="follows-h4">Alternative strategies for DNA sequencing can be grouped into several categories (as discussed previously in ref. <a href="#B4">4</a>). These include (i) microelectrophoretic methods<sup><a href="#B9">9</a></sup> (<a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_BX1.html">Box 1</a>), (ii) sequencing by hybridization<sup><a href="#B10">10</a></sup> (<a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_BX2.html">Box 2</a>), (iii) real-time observation of single molecules<sup><a href="#B11">11, </a></sup><sup><a href="#B12">12</a></sup> (<a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_BX3.html">Box 3</a>) and (iv) cyclic-array sequencing (J.S. <i>et al</i>.<sup><a href="#B13">13</a></sup> and ref. <a href="#B14">14</a>).
Here, we use 'second-generation' in reference to the various
implementations of cyclic-array sequencing that have recently been
realized in a commercial product (e.g., 454 sequencing (used in the 454
Genome Sequencers, Roche Applied Science; Basel), Solexa technology
(used in the Illumina (San Diego) Genome Analyzer), the SOLiD platform
(Applied Biosystems; Foster City, CA, USA), the Polonator
(Dover/Harvard) and the HeliScope Single Molecule Sequencer technology
(Helicos; Cambridge, MA, USA). The concept of cyclic-array sequencing
can be summarized as the sequencing of a dense array of DNA features by
iterative cycles of enzymatic manipulation and imaging-based data
collection<sup><a href="#B15">15</a></sup> (Shendure and colleagues<sup><a href="#B16">16</a></sup>).
Two reports in 2005 described the first integrated implementations of
cyclic-array strategies that were both practical and cost-competitive
with conventional sequencing (J.S. <i>et al</i>.<sup><a href="#B13">13</a></sup> and ref. <a href="#B14">14</a>), and other groups have quickly followed<sup><a href="#B17">17, </a></sup><sup><a href="#B18">18</a></sup>.</p><p><!-- 300x250 ad -->
</p><div class="ad ad300x250">
<p>ADVERTISEMENT</p>











		
<div class="adunit-leaderboard">
<script type="text/javascript"><!--//--><![CDATA[//><!--
		document.write('<script type="text/javascript" src="http://ad.doubleclick.net/adj/biotech.nature.com/article;abr=!webtv;type=re;artid=nbt1486;date=20081009;issue=10;subco=;chaco=;keywd=;njdis=;njreg=;crstg=;pos=left;sz=300x250;ptile=2;ord=' + ord + '?" ><\/script>');

	if ((!document.images && navigator.userAgent.indexOf('Mozilla/2.') >= 0) || (navigator.userAgent.indexOf("WebTV") >= 0)) {
		document.write('<a href="http://ad.doubleclick.net/jump/biotech.nature.com/article;keywd=;sz=300x250;ord=' + ord + '?">');
		document.write('<img src="http://ad.doubleclick.net/ad/biotech.nature.com/article;keywd=;sz=300x250;ord=' + ord + '?" width="300" height="250" border="0"></a>');
	}
//--><!]]></script><script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/article_002.js"></script><!-- Template Id = 3879 Template Name = Banner Creative (NEW NPG Flash) -->
<!-- Created by Matt Peskett - 11-10-2006 clickTAG instead if clickTag. -->
<!-- Copyright 2002 DoubleClick Inc., All rights reserved. -->
<script src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/flashwrite_1_2.js"></script>
<object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" id="FLASH_AD" height="90" width="728"><param name="movie" value="http://m1.2mdn.net/27743/epadvan_banner_300x250.swf?clickTAG=http%3A//ad.doubleclick.net/click%253Bh%3Dv8/379f/3/0/%252a/s%253B209779163%253B0-0%253B0%253B17389285%253B4307-300/250%253B29360923/29378802/1%253B%253B%257Esscs%253D%253fhttp%3A//de.sitestat.com/eppendorf/com/s%3Fnature.advantage_II_2008.banner%26ns_type%3Dclickin"><param name="quality" value="high"><param name="bgcolor" value="#"><param name="wmode" value="opaque"><embed src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/epadvan_banner_300x250.swf" quality="high" wmode="opaque" swliveconnect="TRUE" bgcolor="#" type="application/x-shockwave-flash" height="90" width="728"></object><noembed>&lt;A
TARGET="blank"
HREF="http://ad.doubleclick.net/click%3Bh=v8/379f/3/0/%2a/s%3B209779163%3B0-0%3B0%3B17389285%3B4307-300/250%3B29360923/29378802/1%3B%3B%7Esscs%3D%3fhttp://de.sitestat.com/eppendorf/com/s?nature.advantage_II_2008.banner&amp;ns_type=clickin"&gt;&lt;IMG
SRC="http://m1.2mdn.net/27743/epadvan_2008_2_300x250_banner.gif"
BORDER=0&gt;&lt;/A&gt;</noembed><noscript><A TARGET="blank"
HREF="http://ad.doubleclick.net/click%3Bh=v8/379f/3/0/%2a/s%3B209779163%3B0-0%3B0%3B17389285%3B4307-300/250%3B29360923/29378802/1%3B%3B%7Esscs%3D%3fhttp://de.sitestat.com/eppendorf/com/s?nature.advantage_II_2008.banner&ns_type=clickin"><IMG
SRC="http://m1.2mdn.net/27743/epadvan_2008_2_300x250_banner.gif"
BORDER=0></A></noscript>
<noscript> <div><a
href="http://ad.doubleclick.net/jump/biotech.nature.com/article;abr=!NN2;tile=1;type=re;artid=nbt1486;date=20081009;issue=10;subco=;chaco=;keywd=;njdis=;njreg=;crstg=;pos=left;sz=300x250;ptile=2;ord=123456789?"><img
src="http://ad.doubleclick.net/ad/biotech.nature.com/article;abr=!NN2;tile=1;type=re;artid=nbt1486;date=20081009;issue=10;subco=;chaco=;keywd=;njdis=;njreg=;crstg=;pos=left;sz=300x250;ptile=2;ord=123456789?"
alt="Advertisement" style="border:0" /></a></div>
</noscript>

</div>


</div>
<!-- end 300x250 ad --><p class="norm">Although these platforms are
quite diverse in sequencing biochemistry as well as in how the array is
generated, their work flows are conceptually similar (<a href="#f1">Fig. 1b</a>). Library preparation is accomplished by random fragmentation of DNA, followed by <i>in vitro</i>
ligation of common adaptor sequences. Alternative protocols can be used
to generate jumping libraries of mate-paired tags with controllable
distance distributions<sup><a href="#B13">13, </a></sup><sup><a href="#B19">19</a></sup>.
The generation of clonally clustered amplicons to serve as sequencing
features can be achieved by several approaches, including <i>in situ</i> polonies<sup><a href="#B15">15</a></sup>, emulsion PCR<sup><a href="#B20">20</a></sup> or bridge PCR<sup><a href="#B21">21, </a></sup><sup><a href="#B22">22</a></sup> (<a href="#f2">Fig. 2</a>).
What is common to these methods is that PCR amplicons derived from any
given single library molecule end up spatially clustered, either to a
single location on a planar substrate (<i>in situ</i> polonies, bridge
PCR), or to the surface of micron-scale beads, which can be recovered
and arrayed (emulsion PCR). The sequencing process itself consists of
alternating cycles of enzyme-driven biochemistry and imaging-based data
acquisition (<a href="#f3">Fig. 3</a>). The platforms that are
discussed here all rely on sequencing by synthesis, that is, serial
extension of primed templates, but the enzyme driving the synthesis can
be either a polymerase<sup><a href="#B16">16, </a></sup><sup><a href="#B23">23</a></sup> or a ligase<sup><a href="#B13">13, </a></sup><sup><a href="#B24">24</a></sup>.
Data are acquired by imaging of the full array at each cycle (e.g., of
fluorescently labeled nucleotides incorporated by a polymerase).</p><div id="f2" class="figure-table"><div class="padding clearfix"><h3 class="norm"><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_F2.html">
Figure 2:&nbsp;Clonal amplification of sequencing features.</a></h3><div class="image"><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_F2.html"><img alt="Figure 2 : Clonal amplification of sequencing features." class="thumb" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/nbt1486-F2.gif"></a></div><div class="text"><p class="">(<b>a</b>) The 454, the Polonator and SOLiD platforms rely on emulsion PCR<sup><a href="#B20">20</a></sup> to amplify clonal sequencing features. In brief, an <i>in vitro</i>&#8211;constructed
adaptor-flanked shotgun library (shown as gold and turquoise adaptors
flanking unique inserts) is PCR amplified (that is, multi-template PCR,
not multiplex PCR, as only a single primer pair is used, corresponding
to the gold and turquoise adaptors) in the context of a water-in-oil
emulsion. One of the PCR primers is tethered to the surface
(5'-attached) of micron-scale beads that are also included in the
reaction. A low template concentration results in most bead-containing
compartments having either zero or one template molecule present. In
productive emulsion compartments (where both a bead and template
molecule is present), PCR amplicons are captured to the surface of the
bead. After breaking the emulsion, beads bearing amplification products
can be selectively enriched. Each clonally amplified bead will bear on
its surface PCR products corresponding to amplification of a single
molecule from the template library. (<b>b</b>) The Solexa technology relies on bridge PCR<sup><a href="#B21">21, </a></sup><sup><a href="#B22">22</a></sup> (aka 'cluster PCR') to amplify clonal sequencing features. In brief, an <i>in vitro</i>&#8211;constructed
adaptor-flanked shotgun library is PCR amplified, but both primers
densely coat the surface of a solid substrate, attached at their 5'
ends by a flexible linker. As a consequence, amplification products
originating from any given member of the template library remain
locally tethered near the point of origin. At the conclusion of the
PCR, each clonal cluster contains <img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_003.gif" style="border: 0pt none ; vertical-align: baseline;" alt="approx">1,000
copies of a single member of the template library. Accurate measurement
of the concentration of the template library is critical to maximize
the cluster density while simultaneously avoiding overcrowding.</p><a class="full" href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_F2.html">Full size image
	(38 <abbr title="Kilobyte">KB</abbr>)
</a></div><span class="cleardiv"><!-- --></span>
</div></div><br class="clear"><div id="f3" class="figure-table"><div class="padding clearfix"><h3 class="norm"><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_F3.html">
Figure 3:&nbsp;Strategies for cyclic array sequencing.</a></h3><div class="image"><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_F3.html"><img alt="Figure 3 : Strategies for cyclic array sequencing." class="thumb" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/nbt1486-F3.gif"></a></div><div class="text"><p class="">(<b>a</b>) With the 454 platform, clonally amplified 28-<img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_002.gif" style="border: 0pt none ; vertical-align: baseline;" alt="mu">m
beads generated by emulsion PCR serve as sequencing features and are
randomly deposited to a microfabricated array of picoliter-scale wells.
With pyrosequencing, each cycle consists of the introduction of a
single nucleotide species, followed by addition of substrate
(luciferin, adenosine 5'-phosphosulphate) to drive light production at
wells where polymerase-driven incorporation of that nucleotide took
place. This is followed by an apyrase wash to remove unincorporated
nucleotide. Image from Margulies <i>et al</i>. (2005)<sup><a href="#B14">14</a></sup>. (<b>b</b>)
With the Solexa technology, a dense array of clonally amplified
sequencing features is generated directly on a surface by bridge PCR
(aka cluster PCR). Each sequencing cycle includes the simultaneous
addition of a mixture of four modified deoxynucleotide species, each
bearing one of four fluorescent labels and a reversibly terminating
moiety at the 3' hydroxyl position. A modified DNA polymerase drives
synchronous extension of primed sequencing features. This is followed
by imaging in four channels and then cleavage of both the fluorescent
labels and the terminating moiety. (<b>c</b>) With the SOLiD and the Polonator platforms, clonally amplified 1-<img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_002.gif" style="border: 0pt none ; vertical-align: baseline;" alt="mu">m beads are used to generate a disordered, dense array of sequencing features<sup><a href="#B13">13</a></sup>. Sequencing is performed with a ligase, rather than a polymerase<sup><a href="#B13">13, </a></sup><sup><a href="#B24">24, </a></sup><sup><a href="#B26">26, </a></sup><sup><a href="#B27">27, </a></sup><sup><a href="#B28">28</a></sup>.
With SOLiD, each sequencing cycle introduces a partially degenerate
population of fluorescently labeled octamers. The population is
structured such that the label correlates with the identity of the
central 2 bp in the octamer (the correlation with 2 bp, rather than 1
bp, is the basis of two-base encoding)<sup><a href="#B26">26</a></sup>.
After ligation and imaging in four channels, the labeled portion of the
octamer (that is, 'zzz') is cleaved via a modified linkage between
bases 5 and 6, leaving a free end for another cycle of ligation.
Several such cycles will iteratively interrogate an evenly spaced,
discontiguous set of bases. The system is then reset (by denaturation
of the extended primer), and the process is repeated with a different
offset (e.g., a primer set back from the original position by one or
several bases) such that a different set of discontiguous bases is
interrogated on the next round of serial ligations. (<b>d</b>) With the
HeliScope platform, single nucleic acid molecules are sequenced
directly, that is, there is no clonal amplification step required.
Poly-A&#8211;tailed template molecules are captured by hybridization to
surface-tethered poly-T oligomers to yield a disordered array of primed
single-molecule sequencing templates. Templates are labeled with Cy3,
such that imaging can identify the subset of array coordinates where a
sequencing read is expected. Each cycle consists of the
polymerase-driven incorporation of a single species of fluorescently
labeled nucleotide at a subset of templates, followed by fluorescence
imaging of the full array and chemical cleavage of the label. Image
from Braslavsky <i>et al</i>. (2003)<sup><a href="#B30">30</a></sup>.</p><a class="full" href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_F3.html">Full size image
	(117 <abbr title="Kilobyte">KB</abbr>)
</a></div><span class="cleardiv"><!-- --></span>
</div></div><br class="clear"><p class="norm">Global advantages of second-generation or cyclic-array strategies, relative to Sanger sequencing, include the following: (i) <i>in vitro</i> construction of a sequencing library, followed by <i>in vitro</i>
clonal amplification to generate sequencing features, circumvents
several bottlenecks that restrict the parallelism of conventional
sequencing (that is, transformation of <i>E. coli</i> and colony
picking). (ii) Array-based sequencing enables a much higher degree of
parallelism than conventional capillary-based sequencing. As the
effective size of sequencing features can be on the order of 1 <img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_002.gif" style="border: 0pt none ; vertical-align: baseline;" alt="mu">m,
hundreds of millions of sequencing reads can potentially be obtained in
parallel by rastered imaging of a reasonably sized surface area. (iii)
Because array features are immobilized to a planar surface, they can be
enzymatically manipulated by a single reagent volume. Although
microliter-scale reagent volumes are used in practice, these are
essentially amortized over the full set of sequencing features on the
array, dropping the effective reagent volume per feature to the scale
of picoliters or femtoliters. Collectively, these differences translate
into dramatically lower costs for DNA sequence production.</p><p class="norm">The
advantages of second-generation DNA sequencing are currently offset by
several disadvantages. The most prominent of these include read-length
(for all of the new platforms, read-lengths are currently much shorter
than conventional sequencing) and raw accuracy (on average, base-calls
generated by the new platforms are at least tenfold less accurate than
base-calls generated by Sanger sequencing). Although these limitations
create important algorithmic challenges for the immediate future, we
should bear in mind that these technologies will continue to improve
with respect to these parameters, much as conventional sequencing
progressed gradually over three decades to reach its current level of
technical performance.</p><p class="norm"><b>454 pyrosequencing.</b> The 454 system was the first next-generation sequencing platform available as a commercial product<sup><a href="#B14">14</a></sup>.
In this approach, libraries may be constructed by any method that gives
rise to a mixture of short, adaptor-flanked fragments. Clonal
sequencing features are generated by emulsion PCR<sup><a href="#B20">20</a></sup>, with amplicons captured to the surface of 28-<img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_002.gif" style="border: 0pt none ; vertical-align: baseline;" alt="mu">m beads (<a href="#f2">Fig. 2a</a>).
After breaking the emulsion, beads are treated with denaturant to
remove untethered strands, and then subjected to a hybridization-based
enrichment for amplicon-bearing beads (that is, those that were present
in an emulsion compartment supporting a productive PCR reaction). A
sequencing primer is hybridized to the universal adaptor at the
appropriate position and orientation, that is, immediately adjacent to
the start of unknown sequence.</p><p class="norm">Sequencing is performed by the pyrosequencing method<sup><a href="#B25">25</a></sup> (<a href="#f3">Fig. 3a</a>). In brief, the amplicon-bearing beads are preincubated with <i>Bacillus stearothermophilus</i> (<i>Bst</i>)
polymerase and single-stranded binding protein, and then deposited on
to a microfabricated array of picoliter-scale wells (with dimensions
such that only one bead will fit per well) to render this biochemistry
compatible with array-based sequencing. Smaller beads are also added,
bearing immobilized enzymes also required for pyrosequencing (ATP
sulfurylase and luciferase). During the sequencing, one side of the
semi-ordered array functions as a flow cell for introducing and
removing sequencing reagents, whereas the other side is bonded to a
fiber-optic bundle for CCD (charge-coupled device)-based signal
detection. At each of several hundred cycles, a single species of
unlabeled nucleotide is introduced. On templates where this results in
an incorporation event, pyrophosphate is released. Via ATP sulfurylase
and luciferase, incorporation events immediately drive the generation
of a burst of light, which is detected by the CCD as corresponding to
the array coordinates of specific wells. In contrast with other
platforms, therefore, the sequencing by synthesis must be monitored
'live (that is, the camera does not move relative to the array). Across
multiple cycles (e.g., A-G-C-T-A-G-C-T...), the pattern of detected
incorporation events reveals the sequence of templates represented by
individual beads. Like the HeliScope (discussed below), the sequencing
is 'asynchronous' in that some features may get ahead or behind other
features depending on their sequence relative to the order of base
addition.</p><p class="norm">A major limitation of the 454 technology
relates to homopolymers (that is, consecutive instances of the same
base, such as AAA or GGG). Because there is no terminating moiety
preventing multiple consecutive incorporations at a given cycle, the
length of all homopolymers must be inferred from the signal intensity.
This is prone to a greater error rate than the discrimination of
incorporation versus nonincorporation. As a consequence, the dominant
error type for the 454 platform is insertion-deletion, rather than
substitution. Relative to other next-generation platforms, the key
advantage of the 454 platform is read-length. For example, the 454 FLX
instrument generates <img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_003.gif" style="border: 0pt none ; vertical-align: baseline;" alt="approx">400,000
reads per instrument-run at lengths of 200 to 300 bp. Currently, the
per-base cost of sequencing with the 454 platform is much greater than
that of other platforms (e.g., SOLiD and Solexa) but it may be the
method of choice for certain applications where long read-lengths are
critical (e.g., <i>de novo</i> assembly and metagenomics).</p><p class="norm"><b>Illumina Genome Analyzer.</b> Commonly referred to as 'the Solexa', this platform has its origins in work by Turcatti and colleages<sup><a href="#B22">22, </a></sup><sup><a href="#B23">23</a></sup>
and the merger of four companies&#8212;Solexa (Essex, UK), Lynx Therapeutics
(Hayward, CA, USA), Manteia Predictive Medicine (Coinsins, Switzerland)
and Illumina. Libraries can be constructed by any method that gives
rise to a mixture of adaptor-flanked fragments up to several hundred
base-pairs (bp) in length. Amplified sequencing features are generated
by bridge PCR<sup><a href="#B21">21, </a></sup><sup><a href="#B22">22</a></sup> (<a href="#f2">Fig. 2b</a>).
In this approach, both forward and reverse PCR primers are tethered to
a solid substrate by a flexible linker, such that all amplicons arising
from any single template molecule during the amplification remain
immobilized and clustered to a single physical location on an array. On
the Illumina platform, the bridge PCR is somewhat unconventional in
relying on alternating cycles of extension with <i>Bst</i> polymerase and denaturation with formamide. The resulting 'clusters' each consist of <img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_003.gif" style="border: 0pt none ; vertical-align: baseline;" alt="approx">1,000
clonal amplicons. Several million clusters can be amplified to
distinguishable locations within each of eight independent 'lanes' that
are on a single flow-cell (such that eight independent libraries can be
sequenced in parallel during the same instrument run). After cluster
generation, the amplicons are single stranded (linearization) and a
sequencing primer is hybridized to a universal sequence flanking the
region of interest. Each cycle of sequence interrogation consists of
single-base extension with a modified DNA polymerase and a mixture of
four nucleotides (<a href="#f3">Fig. 3b</a>). These nucleotides are
modified in two ways. They are 'reversible terminators', in that a
chemically cleavable moiety at the 3' hydroxyl position allows only a
single-base incorporation to occur in each cycle; and one of four
fluorescent labels, also chemically cleavable, corresponds to the
identity of each nucleotide<sup><a href="#B23">23</a></sup>. After
single-base extension and acquisition of images in four channels,
chemical cleavage of both groups sets up for the next cycle.
Read-lengths up to 36 bp are currently routine; longer reads are
possible but may incur a higher error rate.</p><p class="norm">Read-lengths
are limited by multiple factors that cause signal decay and dephasing,
such as incomplete cleavage of fluorescent labels or terminating
moieties. The dominant error type is substitution, rather than
insertions or deletions (and homopolymers are certainly less of an
issue than with other platforms such as 454). Average raw error-rates
are on the order of 1&#8211;1.5%, but higher accuracy bases with error rates
of 0.1% or less can be identified through quality metrics associated
with each base-call. As with other systems, modifications have recently
enabled mate-paired reads; for example, each sequencing feature
yielding 2 <img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph.gif" style="border: 0pt none ; vertical-align: middle;" alt="times"> 36 bp independent reads derived from each end of a given library molecule several hundred bases in length.</p><p class="norm"><b>AB SOLiD.</b> This platform has its origins in the system described by J.S. and colleagues<sup><a href="#B13">13</a></sup> in 2005 and in work by McKernan and colleagues<sup><a href="#B26">26</a></sup>
at Agencourt Personal Genomics (Beverly, MA, USA) (acquired by Applied
Biosystems (Foster City, CA, USA) in 2006). Libraries may be
constructed by any method that gives rise to a mixture of short,
adaptor-flanked fragments, though much effort with this system has been
put into protocols for mate-paired tag libraries with controllable and
highly flexible distance distributions<sup><a href="#B13">13, </a></sup><sup><a href="#B19">19</a></sup>. Clonal sequencing features are generated by emulsion PCR, with amplicons captured to the surface of 1-<img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_002.gif" style="border: 0pt none ; vertical-align: baseline;" alt="mu">M paramagnetic beads<sup><a href="#B20">20</a></sup> (<a href="#f2">Fig. 2a</a>).
After breaking the emulsion, beads bearing amplification products are
selectively recovered, and then immobilized to a solid planar substrate
to generate a dense, disordered array. Sequencing by synthesis is
driven by a DNA ligase<sup><a href="#B13">13, </a></sup><sup><a href="#B24">24, </a></sup><sup><a href="#B26">26, </a></sup><sup><a href="#B27">27, </a></sup><sup><a href="#B28">28</a></sup>,
rather than a polymerase. A universal primer complementary to adaptor
sequence is hybridized to the array of amplicon-bearing beads. Each
cycle of sequencing involves the ligation of a degenerate population of
fluorescently labeled octamers (<a href="#f3">Fig. 3c</a>). The octamer
mixture is structured, in that the identity of specific position(s)
within the octamer (e.g., base 5) correlate with the identity of the
fluorescent label. After ligation, images are acquired in four
channels, effectively collecting data for the same base positions
across all template-bearing beads. Then, the octamer is chemically
cleaved between positions 5 and 6, removing the fluorescent label.
Progressive rounds of octamer ligation enable sequencing of every 5<sup>th</sup>
base (e.g., bases 5, 10, 15, 20). Upon completing several such cycles,
the extended primer is denatured to reset the system. Subsequent
iterations of this process can be directed at a different set of
positions (e.g., bases 4, 9, 14, 19) either by using a primer that is
set back one or more bases from the adaptor-insert junction, or by
using different mixtures of octamers where a different position (e.g.,
base 2) is correlated with the label. An additional feature of this
platform involves the use of two-base encoding, which is an
error-correction scheme in which two adjacent bases, rather than a
single base, are correlated with the label<sup><a href="#B26">26</a></sup>.
Each base position is then queried twice (once as the first base, and
once as the second base, in a set of 2 bp interrogated on a given
cycle) such that miscalls can be more readily identified.</p><p class="norm">A related system to the SOLiD is the Polonator, also based in part on the system developed by J.S. and the Church group<sup><a href="#B13">13</a></sup>
at Harvard. This platform also uses sequencing features generated by
emulsion PCR and sequencing by ligation. The cost of the instrument,
however, is substantially lower than that of other second-generation
sequencing instruments. Additionally, the instrument is open source and
programmable, potentially enabling user innovation (e.g., the use of
alternative biochemistries). The current read-lengths, however, may be
significantly limiting.</p><p class="norm">An additional disadvantage,
common to 454, SOLiD and the Polonator, is that emulsion PCR can be
cumbersome and technically challenging. On the other hand, it is
possible that sequencing on a high-density array of very small (1 <img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_002.gif" style="border: 0pt none ; vertical-align: baseline;" alt="mu">m)
beads (with sequencing by ligation, polymerase extension, or another
biochemistry) may represent the most straightforward opportunity to
achieve extremely high data densities, simply because 1-<img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_002.gif" style="border: 0pt none ; vertical-align: baseline;" alt="mu">m
beads physically exclude one another at a spacing that is on the order
of the diffraction limit. Furthermore, high-resolution ordering of 1-<img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/glyph_002.gif" style="border: 0pt none ; vertical-align: baseline;" alt="mu">m bead arrays, as recently described<sup><a href="#B29">29</a></sup>, may enable the limit of one pixel per sequencing feature to be closely approached.</p><p class="norm"><b>HeliScope.</b> The Helicos sequencer<sup><a href="#B18">18</a></sup>, based on work by Quake's group<sup><a href="#B30">30</a></sup>,
also relies on cyclic interrogation of a dense array of sequencing
features. However, a unique aspect of this platform is that no clonal
amplification is required. Instead, a highly sensitive fluorescence
detection system is used to directly interrogate single DNA molecules
via sequencing by synthesis. Template libraries, prepared by random
fragmentation and poly-A tailing (that is, no PCR amplification), are
captured by hybridization to surface-tethered poly-T oligomers to yield
a disordered array of primed single-molecule sequencing templates. At
each cycle, DNA polymerase and a single species of fluorescently
labeled nucleotide are added, resulting in template-dependent extension
of the surface-immobilized primer-template duplexes (<a href="#f3">Fig. 3d</a>).
After acquisition of images tiling the full array, chemical cleavage
and release of the fluorescent label permits the subsequent cycle of
extension and imaging. As described in a recent report<sup><a href="#B18">18</a></sup>,
several hundred cycles of single-base extension (that is, A, G, C, T,
A, G, C, T...) yield average read-lengths of 25 bp or greater. Notable
aspects of this system include the following. First, like the 454
platform, the sequencing is asynchronous, as some strands will fall
ahead or behind others in a sequence-dependent manner. Chance also
plays a role, as some templates may simply fail to incorporate on a
given cycle despite having the appropriate base at the next position.
However, because these are single molecules, dephasing is not an issue,
and such events do not in and of themselves lead to errors.</p><p class="norm">Second,
no terminating moiety is present on the labeled nucleotides. As with
the 454 system, therefore, homopolymer runs are an important issue.
However, because single molecules are being sequenced, the problem can
be mitigated by limiting the rate of incorporation events.
Additionally, Harris <i>et al</i>.<sup><a href="#B18">18</a></sup>
noted that consecutive incorporations of labeled nucleotide at
homopolymers produced a quenching interaction that enabled the authors
to infer the discreet number of incorporations (e.g., A versus AA
versus AAA).</p><p class="norm">Third, the raw sequencing accuracy can
be substantially improved by a two-pass strategy in which the array of
single-molecule templates (here with adaptors at both ends) is
sequenced as described above, and then fully copied. As the newly
synthesized strand is surface-tethered, the original template can be
removed by denaturing. Sequencing primed from the distal adaptor then
yields a second sequence for the same template, obtained in the
opposite orientation. Positions that are concordant between the two
reads have <i>phred</i>-like quality scores approaching 30 (refs. <a href="#B8">8</a>,<a href="#B18">18</a>).</p><p class="norm">And
finally, largely secondary to the incorporation of contaminating,
unlabeled or nonemitting bases, the dominant error type is deletion
(2&#8211;7% error rate with one pass; 0.2&#8211;1% with two passes). However,
substitution error rates are substantially lower (0.01&#8211;1% with one
pass). With two passes, the per-base raw substitution error rate
(approaching 0.001%) may currently be the lowest of all the
second-generation platforms.</p></div><div id="Advantages-and-disadvantages-of-different-approaches"><h4 class="norm">Advantages and disadvantages of different approaches</h4><p class="follows-h4">In
terms of costs, limitations and practical aspects of implementation,
clear differences between conventional sequencing and the
second-generation platforms determine which general strategy represents
the best option for any given project. The applications of conventional
sequencing (that is, Sanger) have grown diverse, and for small-scale
projects in the kilobase-to-megabase range, this will likely remain the
technology of choice for the immediate future. This is a consequence of
its greater 'granularity' (that is, the ability to efficiently operate
at either small or large production scales) relative to the new
technologies. Even so, it is clear that despite limitations relative to
Sanger sequencing (e.g., in terms of read-length and accuracy),
large-scale projects will quickly come to depend entirely on
next-generation sequencing. As an example of the advantages of the new
platforms, consider that large-scale resequencing studies for
identifying germline variation or somatic mutations have relied on
Sanger-based resequencing approaches, that in turn are reliant on
one-at-a-time PCR amplification of each targeted region<sup><a href="#B31">31, </a></sup><sup><a href="#B32">32</a></sup>.
In this context, the requirements of a Sanger sequencing approach
include major costs beyond just reagents. These include robotic support
of reagents, processing of multiple samples in 96- or 384-well formats,
maintenance of capillary-based sequencers, extensive bioinformatics
infrastructure to handle the flow of data and dedicated support staff
to maintain complicated equipment. In a recent informal survey we
conducted of the overall cost to conventionally sequence 100 genes from
100 samples, assuming each gene has an average of 10 exons, quoted
estimates from noncommercial genome centers and commercial sequence
service providers ranged from $300,000 to over $1,000,000. Clearly,
this cost is beyond the range of most individual laboratories. In
addition to reducing the per-base cost of sequencing by several orders
of magnitude, second-generation instruments have fewer infrastructure
requirements; instead, the principle challenge is downstream data
management.</p><p class="norm">There are important differences among the second-generation platforms themselves (<a href="#t1">Table 1</a>) that may result in advantages with respect to specific applications (<a href="#t2">Table 2</a>). Some applications (e.g., resequencing) may be more tolerant of short read-lengths than others (e.g., <i>de novo</i>
assembly). For applications relying on tag counting (e.g.,
quantification of protein-DNA interactions), one would actually prefer
a given amount of sequencing to be split into as many reads as possible
(above some minimum length that allows placement to a reference). The
overall accuracy as well as the specific error distributions of
individual technologies (e.g., the rate of insertion-deletion versus
substitution errors; the propensity for systematic consensus errors)
may also be highly relevant. Mate-paired reads, useful in <i>de novo</i>
assembly and for mapping structural variants, for example, are now
available with all of the second-generation platforms, but the extent
to which the distance distribution with which the read pairs are
separated can be controlled or varied may be an important factor.
Finally, of course, the cost of sequencing varies greatly between the
second-generation platforms, and as consumers, we hope for more
competition between vendors than was the case with conventional
sequencing in the past decade. Comparisons of 'per-base' costs can be
helpful but occasionally misleading, as, for example, more accurate
bases may be worth more than less accurate bases.</p><div id="t1" class="figure-table clearfix"><div class="padding clearfix"><h5 class="norm"><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_T1.html">
Table 1:
			Second-generation DNA sequencing technologies</a></h5><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_T1.html"><img alt="Table 1 - Second-generation DNA sequencing technologies" class="thumb" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/table_thumb.gif"></a><div class="text"><br><a class="full" href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_T1.html">Full table</a></div><span class="cleardiv"><!-- --></span>
</div></div> <br class="clear"> <div id="t2" class="figure-table clearfix"><div class="padding clearfix"><h5 class="norm"><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_T2.html">
Table 2:
			Applications of next-generation sequencing</a></h5><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_T2.html"><img alt="Table 2 - Applications of next-generation sequencing" class="thumb" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/table_thumb.gif"></a><div class="text"><br><a class="full" href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_T2.html">Full table</a></div><span class="cleardiv"><!-- --></span>
</div></div> <br class="clear"> </div><div id="Software-and-standards-for-nextgeneration-sequencing-data"><h4 class="norm">Software and standards for next-generation sequencing data</h4><p class="follows-h4">The
diversity and rapid evolution of next-generation sequencing technology
is posing challenges for bioinformatics in areas including sequence
quality scoring, alignment, assembly and data release. These are
discussed in more detail below.</p><p class="norm"><b>Sequence quality.</b>
The topic of sequence quality scoring has become an area of intense
interest, given the relatively low quality of raw data from the new
sequencing platforms, and the various context-dependent error
distributions associated with different sequencing by synthesis
biochemistries. As second-generation sequencing matures and the range
of biological and clinical problems to which it is applied expands, it
will be critical to have clear metrics in place for data quality,
reliability, reproducibility and biological relevance. Given the
nascent state of the field, there is an opportunity to establish an
early consensus of standardized benchmarks for comparing current and
newly introduced platforms, and, it is hoped, avoid the dilemmas of
data comparison that have occurred in the past with multi-vendor
genomic technologies, such as microarrays. Multiple applications will
benefit from standardized quality metrics. This might include metrics
to quantify the general quality of <i>de novo</i> sequence assemblies;
metrics for the confidence associated with individual read alignments
to a reference; metrics for confidence in raw and consensus base-calls
for improved polymorphism and mutation discovery; and general quality
control and assurance metrics for large-scale sequencing projects.</p><p class="norm">Examples
of areas that should be systematically evaluated in assessing data
quality with new sequencing platforms include: (i) technical
reproducibility; (ii) distribution of estimated accuracies for raw
base-calls (e.g., <i>phred</i>-like scores); (iii) systematic error
patterns in raw or consensus sequence data; (iv) bias and skewing of
true ratios in tag-counting applications. Quality data should be
included by default along with sequences, ideally in a simple,
standardized format. All of the current vendors are adopting quality
metrics in their data reporting, and consumers of these technologies
need to be able to make cross-comparisons among data sets, as well as
to potentially integrate data from multiple platforms while exploiting
their individual advantages (e.g., mixed-technology genome assemblies).
Current resequencing approaches for polymorphism and mutation discovery
from short-read data sets sometimes rely on majority voting schemes,
which are easy to implement but prone to error and false discovery.
More appropriate schemes for single-nucleotide polymorphism (SNP)
detection, such as those developed by Brockman <i>et al</i>.<sup><a href="#B33">33</a></sup> and Quinlan <i>et al</i>.<sup><a href="#B34">34</a></sup>,
estimate the error probabilities associated with individual base-calls
(based on data quality and context) and use this information in making
consensus calls. We anticipate that much as <i>phred</i> and related
tools did for Sanger sequencing, the development of 'third party'
algorithms will substantially improve base-calling and polymorphism
detection with all of the new platforms.</p><p class="norm"><b>Software and bioinformatics tools for data analysis.</b>
Even at this early stage of commercial availability, a variety of
software tools are available for analyzing next-generation sequencing
data (<a href="#t3">Table 3</a>). Their functions fit into several
general categories, including: (i) alignment of sequence reads to a
reference; (ii) base-calling and/or polymorphism detection; (iii) <i>de novo</i> assembly, from paired or unpaired reads; and (iv) genome browsing and annotation.</p><div id="t3" class="figure-table clearfix"><div class="padding clearfix"><h5 class="norm"><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_T3.html">
Table 3:
			Bioinformatics tools for short-read sequencing</a></h5><a href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_T3.html"><img alt="Table 3 - Bioinformatics tools for short-read sequencing" class="thumb" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/table_thumb.gif"></a><div class="text"><br><a class="full" href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_T3.html">Full table</a></div><span class="cleardiv"><!-- --></span>
</div></div> <br class="clear"> <p class="norm">Alignment and assembly
represent particularly interesting problems. Whereas alignment
solutions like BLAST or BLAT are largely adequate for long reads such
as those generated by conventional sequencing, these are unlikely to be
the best algorithms for handling short-read sequence data. An
increasing number of alignment tools have been developed specifically
for rapid alignment of large sets of short reads, while allowing for
mismatches and/or gaps. Some of these tools take advantage of
well-established alignment algorithms, such as Smith-Waterman, but
there also has been significant innovation in developing new algorithms
specifically tailored for short reads. For example, SOAP, a software
package for efficient gapped or ungapped alignment, uses a
memory-intensive seed and look-up table algorithm to accelerate
alignment, while allowing iterative trimming of the 3' end of reads
(usually associated with a higher error rate)<sup><a href="#B35">35</a></sup>.
Other approaches used to accelerate processing include 'bit encoding'
to compress sequence data into a computationally more manageable and
efficient format<sup><a href="#B35">35, </a></sup><sup><a href="#B36">36</a></sup>.
Alignment software is increasingly taking into account the estimated
quality of the underlying data in generating read-placements, as is the
case with MAQ<sup><a href="#B37">37</a></sup>, an alignment and variation discovery tool that works with data from either Solexa or SOLiD data, and SHRiMP (<a href="http://compbio.cs.toronto.edu/shrimp/">http://compbio.cs.toronto.edu/shrimp/</a>),
which includes a novel "color-space to letter-space" Smith-Waterman
algorithm compatible with two base&#8211;encoded sequence data from the SOLiD
platform.</p><p class="norm">As with alignment algorithms, the short
read-lengths, relatively lower accuracies and quantity of data
associated with the new technologies make <i>de novo</i> assembly into
a challenging problem once again. Several assembly tools have recently
been adapted or independently developed for generating assemblies from
short, unpaired sequencing reads<sup><a href="#B38">38, </a></sup><sup><a href="#B39">39, </a></sup><sup><a href="#B40">40</a></sup>.
Mate-paired reads, now possible with all of the major platforms, are
anticipated to have a major impact on the overall success of <i>de novo</i> assembly with short reads, and several algorithms have been already developed that take advantage of these<sup><a href="#B38">38, </a></sup><sup><a href="#B39">39, </a></sup><sup><a href="#B40">40, </a></sup><sup><a href="#B41">41</a></sup>.</p><p class="norm"><b>Guidelines for publication, release and archiving.</b>
There are active ongoing discussions regarding the establishment of
consensus guidelines for reporting and archiving short-read sequence
data, similar to the MIAME (Minimum Information About a Microarray
Experiment) guidelines for microarray experiments<sup><a href="#B42">42</a></sup> (<a href="http://uhts.lbl.gov/">http://uhts.lbl.gov/</a>).
These early efforts have proposed that metadata include annotation of
the biological or clinical nature of the data, key experimental details
such as sample properties and treatments, specific format styles for
sequence reads among multiple samples, policies for release and
publication of sequence data and data processing methods. Public
archiving of next-generation sequence data is another area that will
require substantial effort given the sheer quantity of data being
generated. The National Center for Biotechnology Information at the
National Institutes of Health (Bethesda, MD) has recently established a
Short Read Archive (SRA), and is defining submission formats for data
sets corresponding to major next-generation sequencing platforms<sup><a href="#B43">43</a></sup>.
The SRA is designed to integrate with the Entrez system and to include
metadata, such as experimental annotation and parameters from actual
instrument runs. Current efforts include the development of online
tools for efficiently searching the archive and for data visualization;
these are expected to appear in the coming year<sup><a href="#B43">43</a></sup>.</p></div><div id="Applications-of-nextgeneration-sequencing"><h4 class="norm">Applications of next-generation sequencing</h4><p class="follows-h4">The
past several years have seen an accelerating flurry of publications in
which next-generation sequencing is applied for a variety of goals (<a href="#t2">Table 2</a>). Important applications include: (i) full-genome resequencing or more targeted discovery of mutations or polymorphisms (<a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_BX4.html">Box 4</a>);
(ii) mapping of structural rearrangements, which may include copy
number variation, balanced translocation breakpoints and chromosomal
inversions; (iii) 'RNA-Seq', analogous to expressed sequence tags (EST)
or serial analysis of gene expression (SAGE), where shotgun libraries
derived from mRNA or small RNAs are deeply sequenced; the counts
corresponding to individual species can be used for quantification over
a broad dynamic range, and the sequences themselves can be used for
annotation (e.g., splice junctions and transcript boundaries); (iv)
large-scale analysis of DNA methylation, by deep sequencing of
bisulfite-treated DNA; (v) 'ChIP-Seq', or genome-wide mapping of
DNA-protein interactions, by deep sequencing of DNA fragments pulled
down by chromatin immunoprecipitation. Over the next few years, the
list of applications will undoubtedly grow, as will the sophistication
with which existing applications are carried out.</p></div><div id="Conclusions"><h4 class="norm">Conclusions</h4><p class="follows-h4">Over
the past several years, next-generation DNA sequencing technologies
have catapulted to prominence, with increasingly widespread adoption of
several platforms that individually implement different flavors of
massively parallel cyclic-array sequencing. Common characteristics
extend beyond the technologies themselves, to the quantity and quality
of data that are generated, such that all raise a similar set of new
challenges for experimental design, data analysis and interpretation.
The reduction in the costs of DNA sequencing by several orders of
magnitude is democratizing the extent to which individual investigators
can pursue projects at a scale previously accessible only to major
genome centers. The dramatic increase in interest in this area is also
evident in the number of groups that are now working on sequencing
methods to supplant even the new technologies discussed here. Given the
current state of flux, it is difficult to peer even a few years into
the future, but we anticipate that next-generation sequencing
technologies will become as widespread, commoditized and routine as
microarray technology has become over the past decade. Also analogous
to microarrays, we also expect that the challenges will quickly shift
from mastering of the technologies themselves to the question of how
best to go about extracting biologically meaningful or clinically
useful insights from a very large amount of data.</p></div><br><br></div><div id="acknowledgements"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3 class="page">Acknowledgments</h3><p class="norm before-inline">We thank G. Church and G. Porreca for helpful comments on the manuscript.</p><h4 class="inline">Competing interests statement: </h4><p class="follows-h5">The authors declare

			&nbsp;<a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_audecl.html">competing financial interests</a>.</p></div><div id="References"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3 class="page">References</h3><ol class="norm"><li id="B1"><a name="B1"><!-- . --></a>Hutchison, C.A., <span class="atl"> III. DNA sequencing: bench to bedside and beyond</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">35</span>, 6227&#8211;6237 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/gkm688" class="reftxt" title="III. DNA sequencing: bench to bedside and beyond">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17855400&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - III. DNA sequencing: bench to bedside and beyond">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXht1ejsLzF&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=6b3717c23886d8fa2a5d59feae00d1f8" class="reftxt" title="Article on ChemPort - III. DNA sequencing: bench to bedside and beyond">ChemPort</a>&nbsp;|</li><li id="B2"><a name="B2"><!-- . --></a>Sanger, F. <span class="atl"> Sequences, sequences, and sequences</span>. <span class="journalname">Annu. Rev. Biochem.</span> <span class="journalnumber">57</span>, 1&#8211;28 (<span class="cite-month-year">1988</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1146/annurev.biochem.57.1.1" class="reftxt" title="Sequences, sequences, and sequences">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2460023&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Sequences, sequences, and sequences">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1988P116200001&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Sequences, sequences, and sequences">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaL1cXkslalsr8%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=f150b425ecf4064845b2e8574bfe2b54" class="reftxt" title="Article on ChemPort - Sequences, sequences, and sequences">ChemPort</a>&nbsp;|</li><li id="B3"><a name="B3"><!-- . --></a>Sanger, F. <i>et al</i>. <span class="atl"> Nucleotide sequence of bacteriophage phi X174 DNA</span>. <span class="journalname">Nature</span> <span class="journalnumber">265</span>, 687&#8211;695 (<span class="cite-month-year">1977</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/265687a0" class="reftxt" title="Article on Article - Nucleotide sequence of bacteriophage phi X174 DNA">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=870828&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Nucleotide sequence of bacteriophage phi X174 DNA">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1977CW61200017&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Nucleotide sequence of bacteriophage phi X174 DNA">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaE2sXksV2js7s%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=bb666ccb011b5e9b4a90d3689f1df54b" class="reftxt" title="Article on ChemPort - Nucleotide sequence of bacteriophage phi X174 DNA">ChemPort</a>&nbsp;|</li><li id="B4"><a name="B4"><!-- . --></a>Shendure, J., Mitra, R.D., Varma, C. &amp; Church, G.M. <span class="atl"> Advanced sequencing technologies: methods and goals</span>. <span class="journalname">Nat. Rev. Genet.</span> <span class="journalnumber">5</span>, 335&#8211;344 (<span class="cite-month-year">2004</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nrg1325" class="reftxt" title="Article on Article - Advanced sequencing technologies: methods and goals">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15143316&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Advanced sequencing technologies: methods and goals">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000221139100011&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Advanced sequencing technologies: methods and goals">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXjsl2iurc%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=ebbf472f064f8069380f98a09a0655b2" class="reftxt" title="Article on ChemPort - Advanced sequencing technologies: methods and goals">ChemPort</a>&nbsp;|</li><li id="B5"><a name="B5"><!-- . --></a>Swerdlow, H., Wu, S.L., Harke, H. &amp; Dovichi, N.J. <span class="atl"> Capillary gel electrophoresis for DNA sequencing. Laser-induced fluorescence detection with the sheath flow cuvette</span>. <span class="journalname">J. Chromatogr.</span> <span class="journalnumber">516</span>, 61&#8211;67 (<span class="cite-month-year">1990</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1016/S0021-9673%2801%2990204-3" class="reftxt" title="Capillary gel electrophoresis for DNA sequencing. Laser-induced fluorescence detection with the sheath flow cuvette">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2286629&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Capillary gel electrophoresis for DNA sequencing. Laser-induced fluorescence detection with the sheath flow cuvette">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1990EA11800007&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Capillary gel electrophoresis for DNA sequencing. Laser-induced fluorescence detection with the sheath flow cuvette">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3MXjslWnsQ%3D%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=0f72dbc189631fecc143243d772ae707" class="reftxt" title="Article on ChemPort - Capillary gel electrophoresis for DNA sequencing. Laser-induced fluorescence detection with the sheath flow cuvette">ChemPort</a>&nbsp;|</li><li id="B6"><a name="B6"><!-- . --></a>Hunkapiller, T., Kaiser, R.J., Koop, B.F. &amp; Hood, L. <span class="atl"> Large-scale and automated DNA sequence determination</span>. <span class="journalname">Science</span> <span class="journalnumber">254</span>, 59&#8211;67 (<span class="cite-month-year">1991</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1925562" class="reftxt" title="Large-scale and automated DNA sequence determination">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1925562&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Large-scale and automated DNA sequence determination">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1991GH60500028&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Large-scale and automated DNA sequence determination">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK38XhtlCl&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=bdd3bd1f9ff0d1ca26c6adad05d444a0" class="reftxt" title="Article on ChemPort - Large-scale and automated DNA sequence determination">ChemPort</a>&nbsp;|</li><li id="B7"><a name="B7"><!-- . --></a>Ewing, B. &amp; Green, P. <span class="atl"> Base-calling of automated sequencer traces using phred. II. Error probabilities</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">8</span>, 186&#8211;194 (<span class="cite-month-year">1998</span>).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9521922&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Base-calling of automated sequencer traces using phred. II. Error probabilities">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000072838200007&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Base-calling of automated sequencer traces using phred. II. Error probabilities">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1cXitlWlu7g%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=c92aa0146e17734ba405e826f82f45c2" class="reftxt" title="Article on ChemPort - Base-calling of automated sequencer traces using phred. II. Error probabilities">ChemPort</a>&nbsp;|</li><li id="B8"><a name="B8"><!-- . --></a>Ewing, B., Hillier, L., Wendl, M.C. &amp; Green, P. <span class="atl"> Base-calling of automated sequencer traces using phred. I. Accuracy assessment</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">8</span>, 175&#8211;185 (<span class="cite-month-year">1998</span>).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9521921&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Base-calling of automated sequencer traces using phred. I. Accuracy assessment">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000072838200006&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Base-calling of automated sequencer traces using phred. I. Accuracy assessment">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1cXitlWlu78%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=5d0ee69b206c3c872755a3419043f809" class="reftxt" title="Article on ChemPort - Base-calling of automated sequencer traces using phred. I. Accuracy assessment">ChemPort</a>&nbsp;|</li><li id="B9"><a name="B9"><!-- . --></a>Blazej, R.G., Kumaresan, P. &amp; Mathies, R.A. <span class="atl"> Microfabricated bioprocessor for integrated nanoliter-scale Sanger DNA sequencing</span>. <span class="journalname">Proc. Natl. Acad. Sci. USA</span> <span class="journalnumber">103</span>, 7240&#8211;7245 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1073/pnas.0602476103" class="reftxt" title="Microfabricated bioprocessor for integrated nanoliter-scale Sanger DNA sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16648246&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Microfabricated bioprocessor for integrated nanoliter-scale Sanger DNA sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XkslOktbc%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=7d189937465524af8bb3237f5f7de678" class="reftxt" title="Article on ChemPort - Microfabricated bioprocessor for integrated nanoliter-scale Sanger DNA sequencing">ChemPort</a>&nbsp;|</li><li id="B10"><a name="B10"><!-- . --></a>Gresham, D., Dunham, M.J. &amp; Botstein, D. <span class="atl"> Comparing whole genomes using DNA microarrays</span>. <span class="journalname">Nat. Rev. Genet.</span> <span class="journalnumber">9</span>, 291&#8211;302 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nrg2335" class="reftxt" title="Article on Article - Comparing whole genomes using DNA microarrays">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18347592&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Comparing whole genomes using DNA microarrays">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXjtlGntbg%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=06ff115c9413b21cb5b5788a91e92a41" class="reftxt" title="Article on ChemPort - Comparing whole genomes using DNA microarrays">ChemPort</a>&nbsp;|</li><li id="B11"><a name="B11"><!-- . --></a>Soni, G.V. &amp; Meller, A. <span class="atl"> Progress toward ultrafast DNA sequencing using solid-state nanopores</span>. <span class="journalname">Clin. Chem.</span> <span class="journalnumber">53</span>, 1996&#8211;2001 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1373/clinchem.2007.091231" class="reftxt" title="Progress toward ultrafast DNA sequencing using solid-state nanopores">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17890440&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Progress toward ultrafast DNA sequencing using solid-state nanopores">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXht12jtbnM&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=f7287907667b19f99580aa8292baf345" class="reftxt" title="Article on ChemPort - Progress toward ultrafast DNA sequencing using solid-state nanopores">ChemPort</a>&nbsp;|</li><li id="B12"><a name="B12"><!-- . --></a>Healy, K. <span class="atl"> Nanopore-based single-molecule DNA analysis</span>. <span class="journalname">Nanomed.</span> <span class="journalnumber">2</span>, 459&#8211;481 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.2217/17435889.2.4.459" class="reftxt" title="Nanopore-based single-molecule DNA analysis">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17716132&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Nanopore-based single-molecule DNA analysis">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXpsVaqurk%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=d799d0239a7380763a30cd26312db355" class="reftxt" title="Article on ChemPort - Nanopore-based single-molecule DNA analysis">ChemPort</a>&nbsp;|</li><li id="B13"><a name="B13"><!-- . --></a>Shendure, J. <i>et al</i>. <span class="atl"> Accurate multiplex polony sequencing of an evolved bacterial genome</span>. <span class="journalname">Science</span> <span class="journalnumber">309</span>, 1728&#8211;1732 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1117389" class="reftxt" title="Accurate multiplex polony sequencing of an evolved bacterial genome">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16081699&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Accurate multiplex polony sequencing of an evolved bacterial genome">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000231836700046&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Accurate multiplex polony sequencing of an evolved bacterial genome">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXpvFCis70%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=32c64f69d2b757ec952b262d7a22efed" class="reftxt" title="Article on ChemPort - Accurate multiplex polony sequencing of an evolved bacterial genome">ChemPort</a>&nbsp;|</li><li id="B14"><a name="B14"><!-- . --></a>Margulies, M. <i>et al</i>. <span class="atl"> Genome sequencing in microfabricated high-density picolitre reactors</span>. <span class="journalname">Nature</span> <span class="journalnumber">437</span>, 376&#8211;380 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nature03959" class="reftxt" title="Article on Article - Genome sequencing in microfabricated high-density picolitre reactors">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16056220&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Genome sequencing in microfabricated high-density picolitre reactors">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000231849100045&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Genome sequencing in microfabricated high-density picolitre reactors">ISI</a>&nbsp;|</li><li id="B15"><a name="B15"><!-- . --></a>Mitra, R.D. &amp; Church, G.M. <span class="atl"> In situ localized amplification and contact replication of many individual DNA molecules</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">27</span>, e34 (<span class="cite-month-year">1999</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/27.24.e34" class="reftxt" title="In situ localized amplification and contact replication of many individual DNA molecules">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10572186&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - In situ localized amplification and contact replication of many individual DNA molecules">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:STN:280:DC%2BD3c%2Fkt1Gitg%3D%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=1cfb3811c6e88c9ab43214bfea9cd8e6" class="reftxt" title="Article on ChemPort - In situ localized amplification and contact replication of many individual DNA molecules">ChemPort</a>&nbsp;|</li><li id="B16"><a name="B16"><!-- . --></a>Mitra, R.D., Shendure, J., Olejnik, J., Edyta Krzymanska, O. &amp; Church, G.M. <span class="atl"> Fluorescent in situ sequencing on polymerase colonies</span>. <span class="journalname">Anal. Biochem.</span> <span class="journalnumber">320</span>, 55&#8211;65 (<span class="cite-month-year">2003</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1016/S0003-2697%2803%2900291-4" class="reftxt" title="Fluorescent in situ sequencing on polymerase colonies">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12895469&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Fluorescent in situ sequencing on polymerase colonies">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000184794200006&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Fluorescent in situ sequencing on polymerase colonies">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXlvVCksb4%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=210aedacc19fb4b1795a43e1b1577c34" class="reftxt" title="Article on ChemPort - Fluorescent in situ sequencing on polymerase colonies">ChemPort</a>&nbsp;|</li><li id="B17"><a name="B17"><!-- . --></a>Bentley, D.R. <span class="atl"> Whole-genome re-sequencing</span>. <span class="journalname">Curr. Opin. Genet. Dev.</span> <span class="journalnumber">16</span>, 545&#8211;552 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1016/j.gde.2006.10.009" class="reftxt" title="Whole-genome re-sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17055251&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Whole-genome re-sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000242647400002&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Whole-genome re-sequencing">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28Xht1Cmt7jN&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=ba35d7ccda90f041e489632ec49b8f61" class="reftxt" title="Article on ChemPort - Whole-genome re-sequencing">ChemPort</a>&nbsp;|</li><li id="B18"><a name="B18"><!-- . --></a>Harris, T.D. <i>et al</i>. <span class="atl"> Single-molecule DNA sequencing of a viral genome</span>. <span class="journalname">Science</span> <span class="journalnumber">320</span>, 106&#8211;109 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1150427" class="reftxt" title="Single-molecule DNA sequencing of a viral genome">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18388294&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Single-molecule DNA sequencing of a viral genome">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXktVKhsbc%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=77d9e0e9c73adada7fe7bb93018b1aad" class="reftxt" title="Article on ChemPort - Single-molecule DNA sequencing of a viral genome">ChemPort</a>&nbsp;|</li><li id="B19"><a name="B19"><!-- . --></a>Ng, P. <i>et al</i>. <span class="atl"> Gene identification signature (GIS) analysis for transcriptome characterization and genome annotation</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">2</span>, 105&#8211;111 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth733" class="reftxt" title="Article on Article - Gene identification signature (GIS) analysis for transcriptome characterization and genome annotation">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15782207&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Gene identification signature (GIS) analysis for transcriptome characterization and genome annotation">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000226754100011&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Gene identification signature (GIS) analysis for transcriptome characterization and genome annotation">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXisVGhsrw%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=233ebb13b05dc27f9f13b2c87052628d" class="reftxt" title="Article on ChemPort - Gene identification signature (GIS) analysis for transcriptome characterization and genome annotation">ChemPort</a>&nbsp;|</li><li id="B20"><a name="B20"><!-- . --></a>Dressman, D., Yan, H., Traverso, G., Kinzler, K.W. &amp; Vogelstein, B. <span class="atl"> Transforming single DNA molecules into fluorescent magnetic particles for detection and enumeration of genetic variations</span>. <span class="journalname">Proc. Natl. Acad. Sci. USA</span> <span class="journalnumber">100</span>, 8817&#8211;8822 (<span class="cite-month-year">2003</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1073/pnas.1133470100" class="reftxt" title="Transforming single DNA molecules into fluorescent magnetic particles for detection and enumeration of genetic variations">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12857956&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Transforming single DNA molecules into fluorescent magnetic particles for detection and enumeration of genetic variations">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXlvVyit7Y%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=81092666d5646f2eb2e48dc09ee0f970" class="reftxt" title="Article on ChemPort - Transforming single DNA molecules into fluorescent magnetic particles for detection and enumeration of genetic variations">ChemPort</a>&nbsp;|</li><li id="B21"><a name="B21"><!-- . --></a>Adessi, C. <i>et al</i>. <span class="atl"> Solid phase DNA amplification: characterisation of primer attachment and amplification mechanisms</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">28</span>, e87 (<span class="cite-month-year">2000</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/28.20.e87" class="reftxt" title="Solid phase DNA amplification: characterisation of primer attachment and amplification mechanisms">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11024189&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Solid phase DNA amplification: characterisation of primer attachment and amplification mechanisms">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:STN:280:DC%2BD3cvnvVarsg%3D%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=2cfc685dfceee88961a2dfe21bf2a65e" class="reftxt" title="Article on ChemPort - Solid phase DNA amplification: characterisation of primer attachment and amplification mechanisms">ChemPort</a>&nbsp;|</li><li id="B22"><a name="B22"><!-- . --></a>Fedurco, M., Romieu, A., Williams, S., Lawrence, I. &amp; Turcatti, G. <span class="atl"> BTA, a novel reagent for DNA attachment on glass and efficient generation of solid-phase amplified DNA colonies</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">34</span>, e22 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/gnj023" class="reftxt" title="BTA, a novel reagent for DNA attachment on glass and efficient generation of solid-phase amplified DNA colonies">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16473845&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - BTA, a novel reagent for DNA attachment on glass and efficient generation of solid-phase amplified DNA colonies">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28Xhs1Srsrw%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=1e25879b2dedd73513fe5aa5e5f5efe9" class="reftxt" title="Article on ChemPort - BTA, a novel reagent for DNA attachment on glass and efficient generation of solid-phase amplified DNA colonies">ChemPort</a>&nbsp;|</li><li id="B23"><a name="B23"><!-- . --></a>Turcatti, G., Romieu, A., Fedurco, M. &amp; Tairi, A.P. <span class="atl">
A new class of cleavable fluorescent nucleotides: synthesis and
optimization as reversible terminators for DNA sequencing by synthesis</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">36</span>, e25 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/gkn021" class="reftxt" title="A new class of cleavable fluorescent nucleotides: synthesis and optimization as reversible terminators for DNA sequencing by synthesis">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18263613&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - A new class of cleavable fluorescent nucleotides: synthesis and optimization as reversible terminators for DNA sequencing by synthesis">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXislKhsrc%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=7ad02fef8945527ed8e3792e7b8962a4" class="reftxt" title="Article on ChemPort - A new class of cleavable fluorescent nucleotides: synthesis and optimization as reversible terminators for DNA sequencing by synthesis">ChemPort</a>&nbsp;|</li><li id="B24"><a name="B24"><!-- . --></a>Brenner, S. <i>et al</i>. <span class="atl"> Gene expression analysis by massively parallel signature sequencing (MPSS) on microbead arrays</span>. <span class="journalname">Nat. Biotechnol.</span> <span class="journalnumber">18</span>, 630&#8211;634 (<span class="cite-month-year">2000</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/76469" class="reftxt" title="Article on Article - Gene expression analysis by massively parallel signature sequencing (MPSS) on microbead arrays">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10835600&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Gene expression analysis by massively parallel signature sequencing (MPSS) on microbead arrays">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000087663200022&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Gene expression analysis by massively parallel signature sequencing (MPSS) on microbead arrays">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3cXkt12gur8%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=d9faa36a16935016a71aaebb8954fa61" class="reftxt" title="Article on ChemPort - Gene expression analysis by massively parallel signature sequencing (MPSS) on microbead arrays">ChemPort</a>&nbsp;|</li><li id="B25"><a name="B25"><!-- . --></a>Ronaghi, M., Karamohamed, S., Pettersson, B., Uhlen, M. &amp; Nyren, P. <span class="atl"> Real-time DNA sequencing using detection of pyrophosphate release</span>. <span class="journalname">Anal. Biochem.</span> <span class="journalnumber">242</span>, 84&#8211;89 (<span class="cite-month-year">1996</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1006/abio.1996.0432" class="reftxt" title="Real-time DNA sequencing using detection of pyrophosphate release">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8923969&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Real-time DNA sequencing using detection of pyrophosphate release">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1996VT72900014&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Real-time DNA sequencing using detection of pyrophosphate release">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK28XmvVemurY%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=e077062d067795c72ab6a0a017f9a40c" class="reftxt" title="Article on ChemPort - Real-time DNA sequencing using detection of pyrophosphate release">ChemPort</a>&nbsp;|</li><li id="B26"><a name="B26"><!-- . --></a>McKernan, K., Blanchard, A., Kotler, L. &amp; Costa, G. <span class="atl"> Reagents, methods, and libraries for bead-based sequencing</span>. US patent application 20080003571 (<span class="cite-month-year">2006</span>).</li><li id="B27"><a name="B27"><!-- . --></a>Housby, J.N. &amp; Southern, E.M. <span class="atl"> Fidelity of DNA ligation: a novel experimental approach based on the polymerisation of libraries of oligonucleotides</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">26</span>, 4259&#8211;4266 (<span class="cite-month-year">1998</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/26.18.4259" class="reftxt" title="Fidelity of DNA ligation: a novel experimental approach based on the polymerisation of libraries of oligonucleotides">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9722647&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Fidelity of DNA ligation: a novel experimental approach based on the polymerisation of libraries of oligonucleotides">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000076004500022&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Fidelity of DNA ligation: a novel experimental approach based on the polymerisation of libraries of oligonucleotides">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1cXmsVygu78%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=904ea304b72d969e6461b52aac7d5fb6" class="reftxt" title="Article on ChemPort - Fidelity of DNA ligation: a novel experimental approach based on the polymerisation of libraries of oligonucleotides">ChemPort</a>&nbsp;|</li><li id="B28"><a name="B28"><!-- . --></a>Macevicz, S.C. <span class="atl"> DNA sequencing by parallel oligonucleotide extensions</span>. US patent 5750341 (<span class="cite-month-year">1998</span>).</li><li id="B29"><a name="B29"><!-- . --></a>Barbee, K.D. &amp; Huang, X. <span class="atl"> Magnetic assembly of high-density DNA arrays for genomic analyses</span>. <span class="journalname">Anal. Chem.</span> <span class="journalnumber">80</span>, 2149&#8211;2154 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1021/ac702192y" class="reftxt" title="Magnetic assembly of high-density DNA arrays for genomic analyses">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18260655&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Magnetic assembly of high-density DNA arrays for genomic analyses">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhs1Grtr4%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=4969102c08354756fc61fc920b1d8763" class="reftxt" title="Article on ChemPort - Magnetic assembly of high-density DNA arrays for genomic analyses">ChemPort</a>&nbsp;|</li><li id="B30"><a name="B30"><!-- . --></a>Braslavsky, I., Hebert, B., Kartalov, E. &amp; Quake, S.R. <span class="atl"> Sequence information can be obtained from single DNA molecules</span>. <span class="journalname">Proc. Natl. Acad. Sci. USA</span> <span class="journalnumber">100</span>, 3960&#8211;3964 (<span class="cite-month-year">2003</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1073/pnas.0230489100" class="reftxt" title="Sequence information can be obtained from single DNA molecules">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12651960&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Sequence information can be obtained from single DNA molecules">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXivFSqtb8%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=d875f8c942951a8b8f7c54920193e223" class="reftxt" title="Article on ChemPort - Sequence information can be obtained from single DNA molecules">ChemPort</a>&nbsp;|</li><li id="B31"><a name="B31"><!-- . --></a>Sjoblom, T. <i>et al</i>. <span class="atl"> The consensus coding sequences of human breast and colorectal cancers</span>. <span class="journalname">Science</span> <span class="journalnumber">314</span>, 268&#8211;274 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1133427" class="reftxt" title="The consensus coding sequences of human breast and colorectal cancers">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16959974&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - The consensus coding sequences of human breast and colorectal cancers">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000241194100033&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - The consensus coding sequences of human breast and colorectal cancers">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XhtVOgtrjM&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=bcd96422022324847e40a4f0d06a3297" class="reftxt" title="Article on ChemPort - The consensus coding sequences of human breast and colorectal cancers">ChemPort</a>&nbsp;|</li><li id="B32"><a name="B32"><!-- . --></a>Wood, L.D. <i>et al</i>. <span class="atl"> The genomic landscapes of human breast and colorectal cancers</span>. <span class="journalname">Science</span> <span class="journalnumber">318</span>, 1108&#8211;1113 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1145720" class="reftxt" title="The genomic landscapes of human breast and colorectal cancers">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17932254&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - The genomic landscapes of human breast and colorectal cancers">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXht1ykurrP&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=c3ccca78c77a62b4fb99cce7f9d98362" class="reftxt" title="Article on ChemPort - The genomic landscapes of human breast and colorectal cancers">ChemPort</a>&nbsp;|</li><li id="B33"><a name="B33"><!-- . --></a>Brockman, W. <i>et al</i>. <span class="atl"> Quality scores and SNP detection in sequencing-by-synthesis systems</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">18</span>, 763&#8211;770 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.070227.107" class="reftxt" title="Quality scores and SNP detection in sequencing-by-synthesis systems">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18212088&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Quality scores and SNP detection in sequencing-by-synthesis systems">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXlslChsL4%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=f0e1c0aa1c9e22c11aeacea386378e70" class="reftxt" title="Article on ChemPort - Quality scores and SNP detection in sequencing-by-synthesis systems">ChemPort</a>&nbsp;|</li><li id="B34"><a name="B34"><!-- . --></a>Quinlan, A.R., Stewart, D.A., Stromberg, M.P. &amp; Marth, G.T. <span class="atl"> Pyrobayes: an improved base caller for SNP discovery in pyrosequences</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">5</span>, 179&#8211;181 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth.1172" class="reftxt" title="Article on Article - Pyrobayes: an improved base caller for SNP discovery in pyrosequences">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18193056&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Pyrobayes: an improved base caller for SNP discovery in pyrosequences">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhsV2ksLo%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=36e7aae71ad89c19a52e2f3b72a3512d" class="reftxt" title="Article on ChemPort - Pyrobayes: an improved base caller for SNP discovery in pyrosequences">ChemPort</a>&nbsp;|</li><li id="B35"><a name="B35"><!-- . --></a>Li, R., Li, Y., Kristiansen, K. &amp; Wang, J. <span class="atl"> SOAP: short oligonucleotide alignment program</span>. <span class="journalname">Bioinformatics</span> <span class="journalnumber">24</span>, 713&#8211;714 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/bioinformatics/btn025" class="reftxt" title="SOAP: short oligonucleotide alignment program">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18227114&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - SOAP: short oligonucleotide alignment program">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXislKlt7Y%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=999803351532e8091bfbcdd53b50e36e" class="reftxt" title="Article on ChemPort - SOAP: short oligonucleotide alignment program">ChemPort</a>&nbsp;|</li><li id="B36"><a name="B36"><!-- . --></a>Ning, Z., Cox, A.J. &amp; Mullikin, J.C. <span class="atl"> SSAHA: a fast search method for large DNA databases</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">11</span>, 1725&#8211;1729 (<span class="cite-month-year">2001</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/GR.194201" class="reftxt" title="SSAHA: a fast search method for large DNA databases">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11591649&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - SSAHA: a fast search method for large DNA databases">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000171456000014&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - SSAHA: a fast search method for large DNA databases">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3MXnvFCmtLw%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=d3c9d866e928f8b5b841332dff069bec" class="reftxt" title="Article on ChemPort - SSAHA: a fast search method for large DNA databases">ChemPort</a>&nbsp;|</li><li id="B37"><a name="B37"><!-- . --></a>Li, H., Ruan, J. &amp; Durbin, R. <span class="atl"> Mapping short DNA sequencing reads and calling variants using mapping quality scores</span>. <span class="journalname">Genome Res.</span> published online, doi:doi: 10.1101/gr.078212.108 (19 August <span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.078212.108" class="reftxt" title="Mapping short DNA sequencing reads and calling variants using mapping quality scores">Article</a>&nbsp;|</li><li id="B38"><a name="B38"><!-- . --></a>Butler, J. <i>et al</i>. <span class="atl"> ALLPATHS: de novo assembly of whole-genome shotgun microreads</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">18</span>, 810&#8211;820 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.7337908" class="reftxt" title="ALLPATHS: de novo assembly of whole-genome shotgun microreads">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18340039&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - ALLPATHS: de novo assembly of whole-genome shotgun microreads">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXlslChsLs%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=c93c3dfb269d5f3b812f86ae57aed24f" class="reftxt" title="Article on ChemPort - ALLPATHS: de novo assembly of whole-genome shotgun microreads">ChemPort</a>&nbsp;|</li><li id="B39"><a name="B39"><!-- . --></a>Sundquist, A., Ronaghi, M., Tang, H., Pevzner, P. &amp; Batzoglou, S. <span class="atl"> Whole-genome sequencing and assembly with high-throughput, short-read technologies</span>. <span class="journalname">PLoS ONE</span> <span class="journalnumber">2</span>, e484 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1371/journal.pone.0000484" class="reftxt" title="Whole-genome sequencing and assembly with high-throughput, short-read technologies">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17534434&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Whole-genome sequencing and assembly with high-throughput, short-read technologies">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXmvFyrsrc%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=5b225e47a67bdec54892da33a933b098" class="reftxt" title="Article on ChemPort - Whole-genome sequencing and assembly with high-throughput, short-read technologies">ChemPort</a>&nbsp;|</li><li id="B40"><a name="B40"><!-- . --></a>Warren, R.L., Sutton, G.G., Jones, S.J. &amp; Holt, R.A. <span class="atl"> Assembling millions of short DNA sequences using SSAKE</span>. <span class="journalname">Bioinformatics</span> <span class="journalnumber">23</span>, 500&#8211;501 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/bioinformatics/btl629" class="reftxt" title="Assembling millions of short DNA sequences using SSAKE">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17158514&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Assembling millions of short DNA sequences using SSAKE">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXhvFWltLg%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=3a6151f8406fea9d3628227a6d2284d5" class="reftxt" title="Article on ChemPort - Assembling millions of short DNA sequences using SSAKE">ChemPort</a>&nbsp;|</li><li id="B41"><a name="B41"><!-- . --></a>Zerbino, D.R. &amp; Birney, E. <span class="atl"> Velvet: algorithms for de novo short read assembly using de Bruijn graphs</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">18</span>, 821&#8211;829 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.074492.107" class="reftxt" title="Velvet: algorithms for de novo short read assembly using de Bruijn graphs">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18349386&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Velvet: algorithms for de novo short read assembly using de Bruijn graphs">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXlslChsLg%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=295ebc48721b05bf0133786a9ca51e92" class="reftxt" title="Article on ChemPort - Velvet: algorithms for de novo short read assembly using de Bruijn graphs">ChemPort</a>&nbsp;|</li><li id="B42"><a name="B42"><!-- . --></a>Brazma, A. <i>et al</i>. <span class="atl"> Minimum information about a microarray experiment (MIAME)-toward standards for microarray data</span>. <span class="journalname">Nat. Genet.</span> <span class="journalnumber">29</span>, 365&#8211;371 (<span class="cite-month-year">2001</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/ng1201-365" class="reftxt" title="Article on Article - Minimum information about a microarray experiment (MIAME)-toward standards for microarray data">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11726920&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Minimum information about a microarray experiment (MIAME)-toward standards for microarray data">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000172507500006&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Minimum information about a microarray experiment (MIAME)-toward standards for microarray data">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3MXovFamurw%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=24c70e2635454a868001443c09653cd0" class="reftxt" title="Article on ChemPort - Minimum information about a microarray experiment (MIAME)-toward standards for microarray data">ChemPort</a>&nbsp;|</li><li id="B43"><a name="B43"><!-- . --></a>Wheeler, D.L. <i>et al</i>. <span class="atl"> Database resources of the National Center for Biotechnology Information</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">36</span>, D13&#8211;D21 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/gkm1000" class="reftxt" title="Database resources of the National Center for Biotechnology Information">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18045790&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Database resources of the National Center for Biotechnology Information">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhtVSqtL8%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=0edb7078ef3ccfde172e656475702f41" class="reftxt" title="Article on ChemPort - Database resources of the National Center for Biotechnology Information">ChemPort</a>&nbsp;|</li><li id="B44"><a name="B44"><!-- . --></a>Wheeler, D.A. <i>et al</i>. <span class="atl"> The complete genome of an individual by massively parallel DNA sequencing</span>. <span class="journalname">Nature</span> <span class="journalnumber">452</span>, 872&#8211;876 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nature06884" class="reftxt" title="Article on Article - The complete genome of an individual by massively parallel DNA sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18421352&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - The complete genome of an individual by massively parallel DNA sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXkvVSisL0%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=2e0da445cccf6486cc2b7261ab0e5e33" class="reftxt" title="Article on ChemPort - The complete genome of an individual by massively parallel DNA sequencing">ChemPort</a>&nbsp;|</li><li id="B45"><a name="B45"><!-- . --></a>Van Tassell, C.P. <i>et al</i>. <span class="atl"> SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">5</span>, 247&#8211;252 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth.1185" class="reftxt" title="Article on Article - SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18297082&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXisFOmtLk%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=e9dfc9313fef09254e07d06a62e581fd" class="reftxt" title="Article on ChemPort - SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries">ChemPort</a>&nbsp;|</li><li id="B46"><a name="B46"><!-- . --></a>Dahl, F. <i>et al</i>. <span class="atl"> Multigene amplification and massively parallel sequencing for cancer mutation discovery</span>. <span class="journalname">Proc. Natl. Acad. Sci. USA</span> <span class="journalnumber">104</span>, 9387&#8211;9392 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1073/pnas.0702165104" class="reftxt" title="Multigene amplification and massively parallel sequencing for cancer mutation discovery">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17517648&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Multigene amplification and massively parallel sequencing for cancer mutation discovery">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXmtlegsr0%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=7dd74427997722688cdbbc253d06bb8b" class="reftxt" title="Article on ChemPort - Multigene amplification and massively parallel sequencing for cancer mutation discovery">ChemPort</a>&nbsp;|</li><li id="B47"><a name="B47"><!-- . --></a>Fredriksson, S. <i>et al</i>. <span class="atl"> Multiplex amplification of all coding sequences within 10 cancer genes by Gene-Collector</span>. <span class="journalname">Nucleic Acids Res.</span> <span class="journalnumber">35</span>, e47 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/nar/gkm078" class="reftxt" title="Multiplex amplification of all coding sequences within 10 cancer genes by Gene-Collector">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17317684&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Multiplex amplification of all coding sequences within 10 cancer genes by Gene-Collector">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXmsVWgtbw%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=11469797d8916637a8230c3a3f20fd71" class="reftxt" title="Article on ChemPort - Multiplex amplification of all coding sequences within 10 cancer genes by Gene-Collector">ChemPort</a>&nbsp;|</li><li id="B48"><a name="B48"><!-- . --></a>Porreca, G.J. <i>et al</i>. <span class="atl"> Multiplex amplification of large sets of human exons</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">4</span>, 931&#8211;936 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth1110" class="reftxt" title="Article on Article - Multiplex amplification of large sets of human exons">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17934468&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Multiplex amplification of large sets of human exons">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXht1aqtbfJ&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=080ad45e96d3cf071ad5c9a33be687b6" class="reftxt" title="Article on ChemPort - Multiplex amplification of large sets of human exons">ChemPort</a>&nbsp;|</li><li id="B49"><a name="B49"><!-- . --></a>Bashiardes, S. <i>et al</i>. <span class="atl"> Direct genomic selection</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">2</span>, 63&#8211;69 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth0105-63" class="reftxt" title="Article on Article - Direct genomic selection">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16152676&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Direct genomic selection">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000226754000018&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Direct genomic selection">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXisVKmsbo%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=9d67dcda423efd2d8828285888c27fd1" class="reftxt" title="Article on ChemPort - Direct genomic selection">ChemPort</a>&nbsp;|</li><li id="B50"><a name="B50"><!-- . --></a>Albert, T.J. <i>et al</i>. <span class="atl"> Direct selection of human genomic loci by microarray hybridization</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">4</span>, 903&#8211;905 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth1111" class="reftxt" title="Article on Article - Direct selection of human genomic loci by microarray hybridization">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17934467&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Direct selection of human genomic loci by microarray hybridization">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXht1aqtbfL&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=3c16fc4669beafe9698be7f70c769079" class="reftxt" title="Article on ChemPort - Direct selection of human genomic loci by microarray hybridization">ChemPort</a>&nbsp;|</li><li id="B51"><a name="B51"><!-- . --></a>Hodges, E. <i>et al</i>. <span class="atl"> Genome-wide <i>in situ</i> exon capture for selective resequencing</span>. <span class="journalname">Nat. Genet.</span> <span class="journalnumber">39</span>, 1522&#8211;1527 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/ng.2007.42" class="reftxt" title="Article on Article - Genome-wide in situ exon capture for selective resequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17982454&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Genome-wide in situ exon capture for selective resequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXhtlKmtL3P&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=896db8fafb8b465016d777bd04482bca" class="reftxt" title="Article on ChemPort - Genome-wide in situ exon capture for selective resequencing">ChemPort</a>&nbsp;|</li><li id="B52"><a name="B52"><!-- . --></a>Okou, D.T. <i>et al</i>. <span class="atl"> Microarray-based genomic selection for high-throughput resequencing</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">4</span>, 907&#8211;909 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth1109" class="reftxt" title="Article on Article - Microarray-based genomic selection for high-throughput resequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17934469&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Microarray-based genomic selection for high-throughput resequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXht1aqur7N&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=5bc738c74437dc182d174d242500e997" class="reftxt" title="Article on ChemPort - Microarray-based genomic selection for high-throughput resequencing">ChemPort</a>&nbsp;|</li><li id="B53"><a name="B53"><!-- . --></a>Campbell, P.J. <i>et al</i>. <span class="atl"> Identification of somatically acquired rearrangements in cancer using genome-wide massively parallel paired-end sequencing</span>. <span class="journalname">Nat. Genet.</span> <span class="journalnumber">40</span>, 722&#8211;729 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/ng.128" class="reftxt" title="Article on Article - Identification of somatically acquired rearrangements in cancer using genome-wide massively parallel paired-end sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18438408&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Identification of somatically acquired rearrangements in cancer using genome-wide massively parallel paired-end sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXmsVejsL4%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=d9447c71edc8158344a46635f60e1db2" class="reftxt" title="Article on ChemPort - Identification of somatically acquired rearrangements in cancer using genome-wide massively parallel paired-end sequencing">ChemPort</a>&nbsp;|</li><li id="B54"><a name="B54"><!-- . --></a>Chen, W. <i>et al</i>. <span class="atl"> Mapping translocation breakpoints by next-generation sequencing</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">18</span>, 1143&#8211;1149 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.076166.108" class="reftxt" title="Mapping translocation breakpoints by next-generation sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18326688&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Mapping translocation breakpoints by next-generation sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXotlKmtrs%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=41fe157c34305ce6b070ba6d7dc8233a" class="reftxt" title="Article on ChemPort - Mapping translocation breakpoints by next-generation sequencing">ChemPort</a>&nbsp;|</li><li id="B55"><a name="B55"><!-- . --></a>Cox-Foster, D.L. <i>et al</i>. <span class="atl"> A metagenomic survey of microbes in honey bee colony collapse disorder</span>. <span class="journalname">Science</span> <span class="journalnumber">318</span>, 283&#8211;287 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1146498" class="reftxt" title="A metagenomic survey of microbes in honey bee colony collapse disorder">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17823314&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - A metagenomic survey of microbes in honey bee colony collapse disorder">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXhtFCjt7vM&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=8847d93d9aee1913e1f9cf4f6b4b04d9" class="reftxt" title="Article on ChemPort - A metagenomic survey of microbes in honey bee colony collapse disorder">ChemPort</a>&nbsp;|</li><li id="B56"><a name="B56"><!-- . --></a>Wilhelm, B.T. <i>et al</i>. <span class="atl"> Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution</span>. <span class="journalname">Nature</span> <span class="journalnumber">453</span>, 1239&#8211;1243 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1038/nature07002" class="reftxt" title="Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18488015&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXnslGksLs%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=3f263df4fd88904f7b199641a4fc88c8" class="reftxt" title="Article on ChemPort - Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution">ChemPort</a>&nbsp;|</li><li id="B57"><a name="B57"><!-- . --></a>Sugarbaker, D.J. <i>et al</i>. <span class="atl"> Transcriptome sequencing of malignant pleural mesothelioma tumors</span>. <span class="journalname">Proc. Natl. Acad. Sci. USA</span> <span class="journalnumber">105</span>, 3521&#8211;3526 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1073/pnas.0712399105" class="reftxt" title="Transcriptome sequencing of malignant pleural mesothelioma tumors">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18303113&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Transcriptome sequencing of malignant pleural mesothelioma tumors">PubMed</a>&nbsp;|</li><li id="B58"><a name="B58"><!-- . --></a>Nagalakshmi, U. <i>et al</i>. <span class="atl"> The transcriptional landscape of the yeast genome defined by RNA sequencing</span>. <span class="journalname">Science</span> <span class="journalnumber">320</span>, 1344&#8211;1349 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1158441" class="reftxt" title="The transcriptional landscape of the yeast genome defined by RNA sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18451266&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - The transcriptional landscape of the yeast genome defined by RNA sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXmslWgtbY%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=d60d7be7d24c1ab1b42e670bfecb41de" class="reftxt" title="Article on ChemPort - The transcriptional landscape of the yeast genome defined by RNA sequencing">ChemPort</a>&nbsp;|</li><li id="B59"><a name="B59"><!-- . --></a>Mortazavi, A., Williams, B.A., McCue, K., Schaeffer, L. &amp; Wold, B. <span class="atl"> Mapping and quantifying mammalian transcriptomes by RNA-Seq</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">5</span>, 621&#8211;628 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth.1226" class="reftxt" title="Article on Article - Mapping and quantifying mammalian transcriptomes by RNA-Seq">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18516045&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Mapping and quantifying mammalian transcriptomes by RNA-Seq">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXnslyqs7k%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=1457c1e7b5cc56b644fba82e66399fa7" class="reftxt" title="Article on ChemPort - Mapping and quantifying mammalian transcriptomes by RNA-Seq">ChemPort</a>&nbsp;|</li><li id="B60"><a name="B60"><!-- . --></a>Lister, R. <i>et al</i>. <span class="atl"> Highly integrated single-base resolution maps of the epigenome in <i>Arabidopsis</i></span>. <span class="journalname">Cell</span> <span class="journalnumber">133</span>, 523&#8211;536 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1016/j.cell.2008.03.029" class="reftxt" title="Highly integrated single-base resolution maps of the epigenome in Arabidopsis">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18423832&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Highly integrated single-base resolution maps of the epigenome in Arabidopsis">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXmt1Smur8%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=1da0f9d8364fdbab725c51a219f819e0" class="reftxt" title="Article on ChemPort - Highly integrated single-base resolution maps of the epigenome in Arabidopsis">ChemPort</a>&nbsp;|</li><li id="B61"><a name="B61"><!-- . --></a>Kim, J.B. <i>et al</i>. <span class="atl"> Polony multiplex analysis of gene expression (PMAGE) in mouse hypertrophic cardiomyopathy</span>. <span class="journalname">Science</span> <span class="journalnumber">316</span>, 1481&#8211;1484 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1137325" class="reftxt" title="Polony multiplex analysis of gene expression (PMAGE) in mouse hypertrophic cardiomyopathy">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17556586&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Polony multiplex analysis of gene expression (PMAGE) in mouse hypertrophic cardiomyopathy">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000247066400044&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Polony multiplex analysis of gene expression (PMAGE) in mouse hypertrophic cardiomyopathy">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXmtFSjtr8%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=8e75c795dc71519401a8028202918467" class="reftxt" title="Article on ChemPort - Polony multiplex analysis of gene expression (PMAGE) in mouse hypertrophic cardiomyopathy">ChemPort</a>&nbsp;|</li><li id="B62"><a name="B62"><!-- . --></a>Cloonan, N. <i>et al</i>. <span class="atl"> Stem cell transcriptome profiling via massive-scale mRNA sequencing</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">5</span>, 613&#8211;619 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth.1223" class="reftxt" title="Article on Article - Stem cell transcriptome profiling via massive-scale mRNA sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18516046&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Stem cell transcriptome profiling via massive-scale mRNA sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXnslyqs78%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=e8fda2d6d27836b83d3ebeb06e355985" class="reftxt" title="Article on ChemPort - Stem cell transcriptome profiling via massive-scale mRNA sequencing">ChemPort</a>&nbsp;|</li><li id="B63"><a name="B63"><!-- . --></a>Bainbridge, M.N. <i>et al</i>. <span class="atl"> Analysis of the prostate cancer cell line LNCaP transcriptome using a sequencing-by-synthesis approach</span>. <span class="journalname">BMC Genomics</span> <span class="journalnumber">7</span>, 246 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1186/1471-2164-7-246" class="reftxt" title="Analysis of the prostate cancer cell line LNCaP transcriptome using a sequencing-by-synthesis approach">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17010196&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Analysis of the prostate cancer cell line LNCaP transcriptome using a sequencing-by-synthesis approach">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XhtV2itLnI&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=3f182873034e86ce07733163f49f0259" class="reftxt" title="Article on ChemPort - Analysis of the prostate cancer cell line LNCaP transcriptome using a sequencing-by-synthesis approach">ChemPort</a>&nbsp;|</li><li id="B64"><a name="B64"><!-- . --></a>Morin, R.D. <i>et al</i>. <span class="atl"> Application of massively parallel sequencing to microRNA profiling and discovery in human embryonic stem cells</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">18</span>, 610&#8211;621 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.7179508" class="reftxt" title="Application of massively parallel sequencing to microRNA profiling and discovery in human embryonic stem cells">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18285502&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Application of massively parallel sequencing to microRNA profiling and discovery in human embryonic stem cells">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXks1alsLk%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=8cebc50286cac8adec07f99e608615cc" class="reftxt" title="Article on ChemPort - Application of massively parallel sequencing to microRNA profiling and discovery in human embryonic stem cells">ChemPort</a>&nbsp;|</li><li id="B65"><a name="B65"><!-- . --></a>Korshunova, Y. <i>et al</i>. <span class="atl">
Massively parallel bisulphite pyrosequencing reveals the molecular
complexity of breast cancer-associated cytosine-methylation patterns
obtained from tissue and serum DNA</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">18</span>, 19&#8211;29 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.6883307" class="reftxt" title="Massively parallel bisulphite pyrosequencing reveals the molecular complexity of breast cancer-associated cytosine-methylation patterns obtained from tissue and serum DNA">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18032725&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Massively parallel bisulphite pyrosequencing reveals the molecular complexity of breast cancer-associated cytosine-methylation patterns obtained from tissue and serum DNA">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXnvFyn&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=81160c3847c6882a4195e92bff693566" class="reftxt" title="Article on ChemPort - Massively parallel bisulphite pyrosequencing reveals the molecular complexity of breast cancer-associated cytosine-methylation patterns obtained from tissue and serum DNA">ChemPort</a>&nbsp;|</li><li id="B66"><a name="B66"><!-- . --></a>Ordway, J.M. <i>et al</i>. <span class="atl"> Identification of novel high-frequency DNA methylation changes in breast cancer</span>. <span class="journalname">PLoS ONE</span> <span class="journalnumber">2</span>, e1314 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1371/journal.pone.0001314" class="reftxt" title="Identification of novel high-frequency DNA methylation changes in breast cancer">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18091988&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Identification of novel high-frequency DNA methylation changes in breast cancer">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhtVSmur0%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=d8f132dd8cd95b6736e54941cbd87ffd" class="reftxt" title="Article on ChemPort - Identification of novel high-frequency DNA methylation changes in breast cancer">ChemPort</a>&nbsp;|</li><li id="B67"><a name="B67"><!-- . --></a>Johnson, D.S., Mortazavi, A., Myers, R.M. &amp; Wold, B. <span class="atl"> Genome-wide mapping of in vivo protein-DNA interactions</span>. <span class="journalname">Science</span> <span class="journalnumber">316</span>, 1497&#8211;1502 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1141319" class="reftxt" title="Genome-wide mapping of in vivo protein-DNA interactions">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17540862&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Genome-wide mapping of in vivo protein-DNA interactions">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000247066400049&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Genome-wide mapping of in vivo protein-DNA interactions">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXmtFSjtrg%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=9e716177516ec1c55109cc6315394229" class="reftxt" title="Article on ChemPort - Genome-wide mapping of in vivo protein-DNA interactions">ChemPort</a>&nbsp;|</li><li id="B68"><a name="B68"><!-- . --></a>Robertson, G. <i>et al</i>. <span class="atl"> Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">4</span>, 651&#8211;657 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth1068" class="reftxt" title="Article on Article - Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17558387&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXot12mt74%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=b65a0bedb4fa344d710e6128fcfef132" class="reftxt" title="Article on ChemPort - Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing">ChemPort</a>&nbsp;|</li><li id="B69"><a name="B69"><!-- . --></a>Schones, D.E. <i>et al</i>. <span class="atl"> Dynamic regulation of nucleosome positioning in the human genome</span>. <span class="journalname">Cell</span> <span class="journalnumber">132</span>, 887&#8211;898 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1016/j.cell.2008.02.022" class="reftxt" title="Dynamic regulation of nucleosome positioning in the human genome">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18329373&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Dynamic regulation of nucleosome positioning in the human genome">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXjs1ait7s%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=c9d59a0574c91e3de1d45d16fa94f5d5" class="reftxt" title="Article on ChemPort - Dynamic regulation of nucleosome positioning in the human genome">ChemPort</a>&nbsp;|</li><li id="B70"><a name="B70"><!-- . --></a>Wold, B. &amp; Myers, R.M. <span class="atl"> Sequence census methods for functional genomics</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">5</span>, 19&#8211;21 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth1157" class="reftxt" title="Article on Article - Sequence census methods for functional genomics">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18165803&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Sequence census methods for functional genomics">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXht1Slug%3D%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=a628e11c917fc9c635f16af79c829623" class="reftxt" title="Article on ChemPort - Sequence census methods for functional genomics">ChemPort</a>&nbsp;|</li><li id="B71"><a name="B71"><!-- . --></a>Meyer, M., Stenzel, U. &amp; Hofreiter, M. <span class="atl"> Parallel tagged sequencing on the 454 platform</span>. <span class="journalname">Nat. Protocols</span> <span class="journalnumber">3</span>, 267&#8211;278 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1038/nprot.2007.520" class="reftxt" title="Parallel tagged sequencing on the 454 platform">Article</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhvVyls74%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=8790e91e097a9897f965e669594d5990" class="reftxt" title="Article on ChemPort - Parallel tagged sequencing on the 454 platform">ChemPort</a>&nbsp;|</li><li id="B72"><a name="B72"><!-- . --></a>Slater, G.S. &amp; Birney, E. <span class="atl"> Automated generation of heuristics for biological sequence comparison</span>. <span class="journalname">BMC Bioinformatics</span> <span class="journalnumber">6</span>, 31 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1186/1471-2105-6-31" class="reftxt" title="Automated generation of heuristics for biological sequence comparison">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15713233&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Automated generation of heuristics for biological sequence comparison">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXisVymur0%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=2f6e945e310eeb312795f70555444ce4" class="reftxt" title="Article on ChemPort - Automated generation of heuristics for biological sequence comparison">ChemPort</a>&nbsp;|</li><li id="B73"><a name="B73"><!-- . --></a>Smith, A.D., Xuan, Z. &amp; Zhang, M.Q. <span class="atl"> Using quality scores and longer reads improves accuracy of Solexa read mapping</span>. <span class="journalname">BMC Bioinformatics</span> <span class="journalnumber">9</span>, 128 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1186/1471-2105-9-128" class="reftxt" title="Using quality scores and longer reads improves accuracy of Solexa read mapping">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18307793&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Using quality scores and longer reads improves accuracy of Solexa read mapping">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXkt1SjtL8%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=b33c1325bea550978f3de97af4c0c982" class="reftxt" title="Article on ChemPort - Using quality scores and longer reads improves accuracy of Solexa read mapping">ChemPort</a>&nbsp;|</li><li id="B74"><a name="B74"><!-- . --></a>Hernandez, D., Francois, P., Farinelli, L., Osteras, M. &amp; Schrenzel, J. <span class="atl"> De novo bacterial genome sequencing: Millions of very short reads assembled on a desktop computer</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">18</span>, 802&#8211;809 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.072033.107" class="reftxt" title="De novo bacterial genome sequencing: Millions of very short reads assembled on a desktop computer">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18332092&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - De novo bacterial genome sequencing: Millions of very short reads assembled on a desktop computer">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXlslChsLo%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=88ce6596fff0200ae387d6e442dfcb98" class="reftxt" title="Article on ChemPort - De novo bacterial genome sequencing: Millions of very short reads assembled on a desktop computer">ChemPort</a>&nbsp;|</li><li id="B75"><a name="B75"><!-- . --></a>Chaisson, M.J. &amp; Pevzner, P.A. <span class="atl"> Short read fragment assembly of bacterial genomes</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">18</span>, 324&#8211;330 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.7088808" class="reftxt" title="Short read fragment assembly of bacterial genomes">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18083777&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Short read fragment assembly of bacterial genomes">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhvVSrtro%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=2e1801d9807eb23a478eb805fb90cf2e" class="reftxt" title="Article on ChemPort - Short read fragment assembly of bacterial genomes">ChemPort</a>&nbsp;|</li><li id="B76"><a name="B76"><!-- . --></a>Dohm, J.C., Lottaz, C., Borodina, T. &amp; Himmelbauer, H. <span class="atl"> SHARCGS, a fast and highly accurate short-read assembly algorithm for de novo genomic sequencing</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">17</span>, 1697&#8211;1706 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1101/gr.6435207" class="reftxt" title="SHARCGS, a fast and highly accurate short-read assembly algorithm for de novo genomic sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17908823&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - SHARCGS, a fast and highly accurate short-read assembly algorithm for de novo genomic sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXhtlSlu7fN&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=a16b4b65468c20a627989e08ff6cb6d8" class="reftxt" title="Article on ChemPort - SHARCGS, a fast and highly accurate short-read assembly algorithm for de novo genomic sequencing">ChemPort</a>&nbsp;|</li><li id="B77"><a name="B77"><!-- . --></a>Jeck, W.R. <i>et al</i>. <span class="atl"> Links extending assembly of short DNA sequences to handle error</span>. <span class="journalname">Bioinformatics</span> <span class="journalnumber">23</span>, 2942&#8211;2944 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1093/bioinformatics/btm451" class="reftxt" title="Links extending assembly of short DNA sequences to handle error">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17893086&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Links extending assembly of short DNA sequences to handle error">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXhtlaqsL%2FO&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=d61eec8dd21f5d8740641936a6e29842" class="reftxt" title="Article on ChemPort - Links extending assembly of short DNA sequences to handle error">ChemPort</a>&nbsp;|</li><li id="B78"><a name="B78"><!-- . --></a>Paegel, B.M., Blazej, R.G. &amp; Mathies, R.A. <span class="atl"> Microfluidic devices for DNA sequencing: sample preparation and electrophoretic analysis</span>. <span class="journalname">Curr. Opin. Biotechnol.</span> <span class="journalnumber">14</span>, 42&#8211;50 (<span class="cite-month-year">2003</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1016/S0958-1669%2802%2900004-6" class="reftxt" title="Microfluidic devices for DNA sequencing: sample preparation and electrophoretic analysis">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12566001&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Microfluidic devices for DNA sequencing: sample preparation and electrophoretic analysis">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000181006300007&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Microfluidic devices for DNA sequencing: sample preparation and electrophoretic analysis">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXps1Wguw%3D%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=3bbb74fc52f8750c36a327bb0919864b" class="reftxt" title="Article on ChemPort - Microfluidic devices for DNA sequencing: sample preparation and electrophoretic analysis">ChemPort</a>&nbsp;|</li><li id="B79"><a name="B79"><!-- . --></a>Hong, J.W. &amp; Quake, S.R. <span class="atl"> Integrated nanoliter systems</span>. <span class="journalname">Nat. Biotechnol.</span> <span class="journalnumber">21</span>, 1179&#8211;1183 (<span class="cite-month-year">2003</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nbt871" class="reftxt" title="Article on Article - Integrated nanoliter systems">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=14520403&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Integrated nanoliter systems">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000185647200029&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Integrated nanoliter systems">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXns1Clsbk%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=657bc55b515ba232d464ab2f97688335" class="reftxt" title="Article on ChemPort - Integrated nanoliter systems">ChemPort</a>&nbsp;|</li><li id="B80"><a name="B80"><!-- . --></a>Emrich, C.A., Tian, H., Medintz, I.L. &amp; Mathies, R.A. <span class="atl"> Microfabricated 384-lane capillary array electrophoresis bioanalyzer for ultrahigh-throughput genetic analysis</span>. <span class="journalname">Anal. Chem.</span> <span class="journalnumber">74</span>, 5076&#8211;5083 (<span class="cite-month-year">2002</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1021/ac020236g" class="reftxt" title="Microfabricated 384-lane capillary array electrophoresis bioanalyzer for ultrahigh-throughput genetic analysis">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12380833&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Microfabricated 384-lane capillary array electrophoresis bioanalyzer for ultrahigh-throughput genetic analysis">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000178418100039&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Microfabricated 384-lane capillary array electrophoresis bioanalyzer for ultrahigh-throughput genetic analysis">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD38XmsVCntr0%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=90bcc7e3cddab81a42d6bfdb0e672c7b" class="reftxt" title="Article on ChemPort - Microfabricated 384-lane capillary array electrophoresis bioanalyzer for ultrahigh-throughput genetic analysis">ChemPort</a>&nbsp;|</li><li id="B81"><a name="B81"><!-- . --></a>Toriello, N.M., Liu, C.N., Blazej, R.G., Thaitrong, N. &amp; Mathies, R.A. <span class="atl">
Integrated affinity capture, purification, and capillary
electrophoresis microdevice for quantitative double-stranded DNA
analysis</span>. <span class="journalname">Anal. Chem.</span> <span class="journalnumber">79</span>, 8549&#8211;8556 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1021/ac0712547" class="reftxt" title="Integrated affinity capture, purification, and capillary electrophoresis microdevice for quantitative double-stranded DNA analysis">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17929900&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Integrated affinity capture, purification, and capillary electrophoresis microdevice for quantitative double-stranded DNA analysis">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXhtFGisrnE&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=188ea0b8aad29a767c2d0cfd78f6535e" class="reftxt" title="Article on ChemPort - Integrated affinity capture, purification, and capillary electrophoresis microdevice for quantitative double-stranded DNA analysis">ChemPort</a>&nbsp;|</li><li id="B82"><a name="B82"><!-- . --></a>Blazej, R.G., Kumaresan, P., Cronier, S.A. &amp; Mathies, R.A. <span class="atl"> Inline injection microdevice for attomole-scale sanger DNA sequencing</span>. <span class="journalname">Anal. Chem.</span> <span class="journalnumber">79</span>, 4499&#8211;4506 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1021/ac070126f" class="reftxt" title="Inline injection microdevice for attomole-scale sanger DNA sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17497827&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Inline injection microdevice for attomole-scale sanger DNA sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXltFequ7w%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=d61d0f4e8e2956d275ffcadf33eba7ab" class="reftxt" title="Article on ChemPort - Inline injection microdevice for attomole-scale sanger DNA sequencing">ChemPort</a>&nbsp;|</li><li id="B83"><a name="B83"><!-- . --></a>Hong, J.W., Studer, V., Hang, G., Anderson, W.F. &amp; Quake, S.R. <span class="atl"> A nanoliter-scale nucleic acid processor with parallel architecture</span>. <span class="journalname">Nat. Biotechnol.</span> <span class="journalnumber">22</span>, 435&#8211;439 (<span class="cite-month-year">2004</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nbt951" class="reftxt" title="Article on Article - A nanoliter-scale nucleic acid processor with parallel architecture">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15024389&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - A nanoliter-scale nucleic acid processor with parallel architecture">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXis1yjtrY%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=9a15e132fa1fe3f8944379414e673d82" class="reftxt" title="Article on ChemPort - A nanoliter-scale nucleic acid processor with parallel architecture">ChemPort</a>&nbsp;|</li><li id="B84"><a name="B84"><!-- . --></a>Patil, N. <i>et al</i>. <span class="atl"> Blocks of limited haplotype diversity revealed by high-resolution scanning of human chromosome 21</span>. <span class="journalname">Science</span> <span class="journalnumber">294</span>, 1719&#8211;1723 (<span class="cite-month-year">2001</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1065573" class="reftxt" title="Blocks of limited haplotype diversity revealed by high-resolution scanning of human chromosome 21">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11721056&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Blocks of limited haplotype diversity revealed by high-resolution scanning of human chromosome 21">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000172307400045&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Blocks of limited haplotype diversity revealed by high-resolution scanning of human chromosome 21">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3MXoslCrsLY%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=fac6bdcddbc61d1397be2a1158241fd8" class="reftxt" title="Article on ChemPort - Blocks of limited haplotype diversity revealed by high-resolution scanning of human chromosome 21">ChemPort</a>&nbsp;|</li><li id="B85"><a name="B85"><!-- . --></a>Frazer, K.A. <i>et al</i>. <span class="atl"> A sequence-based variation map of 8.27 million SNPs in inbred mouse strains</span>. <span class="journalname">Nature</span> <span class="journalnumber">448</span>, 1050&#8211;1053 (<span class="cite-month-year">2007</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1038/nature06067" class="reftxt" title="A sequence-based variation map of 8.27 million SNPs in inbred mouse strains">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17660834&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - A sequence-based variation map of 8.27 million SNPs in inbred mouse strains">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXps12nsLw%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=76979658466be444b1fabe2c74f6eaa8" class="reftxt" title="Article on ChemPort - A sequence-based variation map of 8.27 million SNPs in inbred mouse strains">ChemPort</a>&nbsp;|</li><li id="B86"><a name="B86"><!-- . --></a>Gresham, D. <i>et al</i>. <span class="atl"> Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray</span>. <span class="journalname">Science</span> <span class="journalnumber">311</span>, 1932&#8211;1936 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1123726" class="reftxt" title="Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16527929&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000236387800047&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XivFSmt7s%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=ddee0ae7a0c679be5aaa3e4dcd99debc" class="reftxt" title="Article on ChemPort - Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray">ChemPort</a>&nbsp;|</li><li id="B87"><a name="B87"><!-- . --></a>Albert, T.J. <i>et al</i>. <span class="atl"> Mutation discovery in bacterial genomes: metronidazole resistance in <i>Helicobacter pylori.</i></span> <span class="journalname">Nat. Methods</span> <span class="journalnumber">2</span>, 951&#8211;953 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nmeth805" class="reftxt" title="Article on Article - Mutation discovery in bacterial genomes: metronidazole resistance in Helicobacter pylori.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16299480&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Mutation discovery in bacterial genomes: metronidazole resistance in Helicobacter pylori.">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXht1aksr3O&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=15af932167cf6818fe492364d6204d7f" class="reftxt" title="Article on ChemPort - Mutation discovery in bacterial genomes: metronidazole resistance in Helicobacter pylori.">ChemPort</a>&nbsp;|</li><li id="B88"><a name="B88"><!-- . --></a>Drmanac, S. <i>et al</i>. <span class="atl"> Accurate sequencing by hybridization for DNA diagnostics and individual genomics</span>. <span class="journalname">Nat. Biotechnol.</span> <span class="journalnumber">16</span>, 54&#8211;58 (<span class="cite-month-year">1998</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nbt0198-54" class="reftxt" title="Article on Article - Accurate sequencing by hybridization for DNA diagnostics and individual genomics">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9447594&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Accurate sequencing by hybridization for DNA diagnostics and individual genomics">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000071306000029&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Accurate sequencing by hybridization for DNA diagnostics and individual genomics">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:STN:280:DyaK1c7gvVymtw%3D%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=eae5f13abf88546f4eb3e29fb36a2fe1" class="reftxt" title="Article on ChemPort - Accurate sequencing by hybridization for DNA diagnostics and individual genomics">ChemPort</a>&nbsp;|</li><li id="B89"><a name="B89"><!-- . --></a>Drmanac, R., Labat, I., Brukner, I. &amp; Crkvenjakov, R. <span class="atl"> Sequencing of megabase plus DNA by hybridization: theory of the method</span>. <span class="journalname">Genomics</span> <span class="journalnumber">4</span>, 114&#8211;128 (<span class="cite-month-year">1989</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1016/0888-7543%2889%2990290-5" class="reftxt" title="Sequencing of megabase plus DNA by hybridization: theory of the method">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2737674&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Sequencing of megabase plus DNA by hybridization: theory of the method">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1989R935900001&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Sequencing of megabase plus DNA by hybridization: theory of the method">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaL1MXhs1Onu7k%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=868ca1637f28bccf6c22294281f80859" class="reftxt" title="Article on ChemPort - Sequencing of megabase plus DNA by hybridization: theory of the method">ChemPort</a>&nbsp;|</li><li id="B90"><a name="B90"><!-- . --></a>Lizardi, P.M. <i>et al</i>. <span class="atl"> Mutation detection and single-molecule counting using isothermal rolling-circle amplification</span>. <span class="journalname">Nat. Genet.</span> <span class="journalnumber">19</span>, 225&#8211;232 (<span class="cite-month-year">1998</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/898" class="reftxt" title="Article on Article - Mutation detection and single-molecule counting using isothermal rolling-circle amplification">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9662393&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Mutation detection and single-molecule counting using isothermal rolling-circle amplification">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000074565900016&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Mutation detection and single-molecule counting using isothermal rolling-circle amplification">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1cXls1Sns7g%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=4464c0b0df8729bddd19fbda9d12964a" class="reftxt" title="Article on ChemPort - Mutation detection and single-molecule counting using isothermal rolling-circle amplification">ChemPort</a>&nbsp;|</li><li id="B91"><a name="B91"><!-- . --></a>Pihlak, A. <i>et al</i>. <span class="atl"> Rapid genome sequencing with short universal tiling probes</span>. <span class="journalname">Nat. Biotechnol.</span> <span class="journalnumber">26</span>, 676&#8211;684 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://www.nature.com/doifinder/10.1038/nbt1405" class="reftxt" title="Article on Article - Rapid genome sequencing with short universal tiling probes">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18500336&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Rapid genome sequencing with short universal tiling probes">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXmvVCisrw%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=5e7720526ddcf016553cdc69b153071a" class="reftxt" title="Article on ChemPort - Rapid genome sequencing with short universal tiling probes">ChemPort</a>&nbsp;|</li><li id="B92"><a name="B92"><!-- . --></a>Deamer, D.W. &amp; Akeson, M. <span class="atl"> Nanopores and nucleic acids: prospects for ultrarapid sequencing</span>. <span class="journalname">Trends Biotechnol.</span> <span class="journalnumber">18</span>, 147&#8211;151 (<span class="cite-month-year">2000</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1016/S0167-7799%2800%2901426-8" class="reftxt" title="Nanopores and nucleic acids: prospects for ultrarapid sequencing">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10740260&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Nanopores and nucleic acids: prospects for ultrarapid sequencing">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000086186400003&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Nanopores and nucleic acids: prospects for ultrarapid sequencing">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3cXit1SqsLw%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=b7b8b2708e0afc6f7125d0db74be5887" class="reftxt" title="Article on ChemPort - Nanopores and nucleic acids: prospects for ultrarapid sequencing">ChemPort</a>&nbsp;|</li><li id="B93"><a name="B93"><!-- . --></a>Meller, A., Nivon, L., Brandin, E., Golovchenko, J. &amp; Branton, D. <span class="atl"> Rapid nanopore discrimination between single polynucleotide molecules</span>. <span class="journalname">Proc. Natl. Acad. Sci. USA</span> <span class="journalnumber">97</span>, 1079&#8211;1084 (<span class="cite-month-year">2000</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1073/pnas.97.3.1079" class="reftxt" title="Rapid nanopore discrimination between single polynucleotide molecules">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10655487&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Rapid nanopore discrimination between single polynucleotide molecules">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3cXpvFegug%3D%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=f27731c3cd024d571960d5bafde232b6" class="reftxt" title="Article on ChemPort - Rapid nanopore discrimination between single polynucleotide molecules">ChemPort</a>&nbsp;|</li><li id="B94"><a name="B94"><!-- . --></a>Cockroft, S.L., Chu, J., Amorin, M. &amp; Ghadiri, M.R. <span class="atl"> A single-molecule nanopore device detects DNA polymerase activity with single-nucleotide resolution</span>. <span class="journalname">J. Am. Chem. Soc.</span> <span class="journalnumber">130</span>, 818&#8211;820 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1021/ja077082c" class="reftxt" title="A single-molecule nanopore device detects DNA polymerase activity with single-nucleotide resolution">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18166054&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - A single-molecule nanopore device detects DNA polymerase activity with single-nucleotide resolution">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhtlyhsQ%3D%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=5b628ac4b73aa369e78d34cc34cc70b9" class="reftxt" title="Article on ChemPort - A single-molecule nanopore device detects DNA polymerase activity with single-nucleotide resolution">ChemPort</a>&nbsp;|</li><li id="B95"><a name="B95"><!-- . --></a>Levene, M.J. <i>et al</i>. <span class="atl"> Zero-mode waveguides for single-molecule analysis at high concentrations</span>. <span class="journalname">Science</span> <span class="journalnumber">299</span>, 682&#8211;686 (<span class="cite-month-year">2003</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1126/science.1079700" class="reftxt" title="Zero-mode waveguides for single-molecule analysis at high concentrations">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12560545&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Zero-mode waveguides for single-molecule analysis at high concentrations">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000180687700039&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Zero-mode waveguides for single-molecule analysis at high concentrations">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXntFWisg%3D%3D&amp;pissn=1087-0156&amp;pyear=2008&amp;md5=7b45097a9dc0a956eab00918d15110e6" class="reftxt" title="Article on ChemPort - Zero-mode waveguides for single-molecule analysis at high concentrations">ChemPort</a>&nbsp;|</li><li id="B96"><a name="B96"><!-- . --></a>Lundquist, P.M. <i>et al</i>. <span class="atl"> Parallel confocal detection of single molecules in real time</span>. <span class="journalname">Opt. Lett.</span> <span class="journalnumber">33</span>, 1026&#8211;1028 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1364/OL.33.001026" class="reftxt" title="Parallel confocal detection of single molecules in real time">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18451975&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Parallel confocal detection of single molecules in real time">PubMed</a>&nbsp;|</li><li id="B97"><a name="B97"><!-- . --></a>Korlach, J. <i>et al</i>. <span class="atl">
Selective aluminum passivation for targeted immobilization of single
DNA polymerase molecules in zero-mode waveguide nanostructures</span>. <span class="journalname">Proc. Natl. Acad. Sci. USA</span> <span class="journalnumber">105</span>, 1176&#8211;1181 (<span class="cite-month-year">2008</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org/10.1073/pnas.0710982105" class="reftxt" title="Selective aluminum passivation for targeted immobilization of single DNA polymerase molecules in zero-mode waveguide nanostructures">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18216253&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Selective aluminum passivation for targeted immobilization of single DNA polymerase molecules in zero-mode waveguide nanostructures">PubMed</a>&nbsp;|</li></ol></div><div id="affiliations-notes" class="aff-bottom">
			<ol class="decimal"><li id="a1">Department
of Genome Sciences, University of Washington, Foege Building S-250, Box
355065, 1705 NE Pacific St., Seattle, Washington 98195-5065, USA.</li><li id="a2">Stanford
Genome Technology Center and Division of Oncology, Dept. of Medicine,
Stanford University School of Medicine, CCSR 3215, 269 Campus Drive,
Stanford, California 94305, USA.</li></ol><p class="caff">Correspondence to: Jay Shendure<sup><a title="affiliated with " href="#a1">1</a></sup> e-mail:&nbsp;<a href="mailto:shendure@u.washington.edu">shendure@u.washington.edu</a></p><p class="caff">Correspondence to: Hanlee Ji<sup><a title="affiliated with " href="#a2">2</a></sup> e-mail:&nbsp;<a href="mailto:genomics_ji@stanford.edu">genomics_ji@stanford.edu</a></p></div><!-- generated by Nature-branded standard table template --><div id="more-like-this"> <h3>MORE ARTICLES LIKE THIS</h3> <p class="explain">These links to content published by NPG are automatically generated.</p><div class="wrapper"><h4 class="area">NEWS AND VIEWS</h4><a href="http://www.nature.com/doifinder/10.1038/nbt0608-649" class="article">Next-generation sequencing-by-hybridization</a><p class="journal">Nature Biotechnology <span>News and Views (01 Jun 2008)</span></p><a href="http://www.nature.com/doifinder/10.1038/437326a" class="article">Genomics Massively parallel sequencing</a><p class="journal">Nature <span>News and Views (15 Sep 2005)</span></p><a href="http://sherlock.nature.com/arls/Query?src_jrnl=Nature%20Biotechnology&amp;src_vol=26&amp;query=http://xml.nature.com/nbt/journal/v26/n10/xml/nbt1486.xml&amp;type=RE&amp;src_pp=1135-1145&amp;prod=NBT&amp;src_date=2008&amp;src_title=Next-generation%20DNA%20sequencing&amp;showall=news" class="allmatches" target="_new">See all 4 matches for News And Views</a></div><div class="wrapper"><h4 class="area">RESEARCH</h4><a href="http://www.nature.com/doifinder/10.1038/sj.emboj.7601742" class="article">BOB.1/OBF.1 controls the balance of TH1 and TH2 immune responses</a><p class="journal">The EMBO Journal <span>Article (11 Jul 2007)</span></p><a href="http://www.nature.com/doifinder/10.1038/nbt1485" class="article">The development and impact of 454 sequencing</a><p class="journal">Nature Biotechnology <span>Research (01 Oct 2008)</span></p><a href="http://sherlock.nature.com/arls/Query?src_jrnl=Nature%20Biotechnology&amp;src_vol=26&amp;query=http://xml.nature.com/nbt/journal/v26/n10/xml/nbt1486.xml&amp;type=RE&amp;src_pp=1135-1145&amp;prod=NBT&amp;src_date=2008&amp;src_title=Next-generation%20DNA%20sequencing&amp;showall=research" class="allmatches" target="_new">See all 6 matches for Research</a></div></div></div>
						<!-- value from issueHighlight directive to variable --><div id="journalnav"><h1 class="hidden">Main navigation</h1><!-- whitespace removed to deal with IE5.0 PC bug. -->
<map id="main-journal-nav" class="journal-nav">
<h2 class="nav-header">Journal content</h2>
<ul>
	<li><a href="http://www.nature.com/nbt/">Journal home</a></li><li><a href="http://www.nature.com/nbt/journal/vaop/ncurrent/">Advance online publication</a>
			</li>
		<li><a href="http://www.nature.com/nbt/current_issue/">Current issue</a></li><li class="selected"><a href="http://www.nature.com/nbt/archive/">Archive</a></li><li><a href="http://www.nature.com/nbt/conferences/">Conferences</a></li><li><a href="http://www.nature.com/nbt/focsup_index.html">Focuses and Supplements</a></li><li><a href="http://www.nature.com/nbt/press_release/">Press releases</a></li>
</ul>
</map><!-- whitespace removed to deal with IE5.0 PC bug. -->
<map id="journal-information" class="journal-nav">
<h2 class="nav-header">Journal information</h2>
<ul>
<li><a href="http://www.nature.com/nbt/authors/index.html">Guide to authors</a>
		
			</li>
	<li><a href="http://mts-nbt.nature.com/" class="submission">Online submission</a>
	</li><li><a href="http://www.nature.com/reprints/index.html" class="permissions">Permissions</a></li>
	<li><a href="http://www.nature.com/authors/editorial_policies/peer_review.html">For referees</a></li>
	<li><a href="http://www.nature.com/nbt/journal/v22/n10/index.html">Free online issue</a></li>
	<li><a href="http://www.nature.com/nbt/contact/index.html">Contact the journal</a></li>
	<li><a href="https://secure.nature.com/subscribe/nbt">Subscribe</a></li>
	<li><a href="http://www.nature.com/nbt/site/index.html">About this site</a></li>
</ul>
</map>

<map id="npg-services" class="journal-nav npg-nav">
<h2 class="nav-header"><abbr title="nature publishing group">NPG</abbr> services</h2>
<ul>
	<li><a href="http://www.nature.com/authors/index.html">Authors &amp; Referees</a></li>
	<li><a href="http://www.nature.com/advertising/index.html">Advertising</a></li>
	<li><a href="http://npg.nature.com/npg/forms/03_current.jsp">work@<abbr title="Nature Publishing Group">npg</abbr></a></li>
	<li><a href="http://www.naturereprints.com/">nature<span class="hidden"> </span>reprints</a></li>
	<li><a href="http://www.nature.com/libraries/index.html">For librarians</a></li>
</ul>
</map><map id="npg-resources" class="journal-nav npg-nav">
	<h2 class="npg-resources"><abbr title="Nature Publishing Group">NPG</abbr> resources</h2>
	<ul class="npg-resources">
		<li><a href="http://www.scibx.com/"><b>New!</b> SciBX</a></li>
		<li><a href="http://www.nature.com/bioent/">Bioentrepreneur</a></li>
		<li><a href="http://www.nature.com/biopartnering/">Biopartnering</a></li>
		<li><a href="http://www.nature.com/news/">Nature News</a></li>
		<li><a href="http://www.nature.com/nrd/">Nature Reviews Drug Discovery</a></li>
		<li><a href="http://www.nature.com/nm/">Nature Medicine</a></li>
		<li><a href="http://www.nature.com/ng/">Nature Genetics</a></li>
		<li><a href="http://www.nature.com/nrg/">Nature Reviews Genetics</a></li>
		<li><a href="http://www.nature.com/nmeth/">Nature Methods</a></li>
		<li><a href="http://www.nature.com/nchembio/">Nature Chemical Biology</a></li>
		<li><a href="http://www.nature.com/stemcells/">Nature Reports Stem Cells</a></li>
		<li><a href="http://www.nature.com/clpt/">Clinical Pharmacology &amp; Therapeutics</a></li>
		<li><a href="http://www.nature.com/nature/">Nature</a></li>
		<li><a href="http://www.nature.com/natureconferences/">Nature Conferences</a></li>
		<li><a href="http://www.nature.com/rnai/index.html">RNAi Gateway</a></li>
	</ul>
</map> 
<div id="global-subject-areas">
	<h2><span><abbr title="Nature Publishing Group">NPG</abbr> Journals</span></h2>	
	<h3 class="view">by Subject Area</h3>
		<ul class="category-list">
		<li><h4>Chemistry</h4>
            <ul>
                <li><a href="http://www.nature.com/chemistry/index.html">Chemistry</a></li>
                <li><a href="http://www.nature.com/drugdisc/index.html">Drug discovery</a></li>
		<li><a href="http://www.nature.com/biotech/index.html">Biotechnology</a></li>
		<li><a href="http://www.nature.com/materials/index.html">Materials</a></li>
		<li><a href="http://www.nature.com/protometh/index.html">Methods &amp; Protocols</a></li>
            </ul>
        </li>        <li><h4>Clinical Practice &amp; Research</h4>
            <ul>
                <li><a href="http://www.nature.com/cancer/index.html">Cancer</a></li>
                <li><a href="http://www.nature.com/cardio/index.html">Cardiovascular medicine</a></li>
                <li><a href="http://www.nature.com/dentistry/index.html">Dentistry</a></li>
                <li><a href="http://www.nature.com/endocrinology/index.html" title="">Endocrinology</a></li>
                <li><a href="http://www.nature.com/gastrohep/index.html">Gastroenterology &amp; Hepatology</a></li>
		<li><a href="http://www.nature.com/protometh/index.html">Methods &amp; Protocols</a></li>
                <li><a href="http://www.nature.com/pathology/index.html">Pathology &amp; Pathobiology</a></li>
                <li><a href="http://www.nature.com/urology/index.html">Urology</a></li>

            </ul>
        </li>        <li><h4>Earth &amp; Environment</h4>
            <ul>
                <li><a href="http://www.nature.com/earthsciences/index.html">Earth sciences</a></li>
                <li><a href="http://www.nature.com/evoeco/index.html">Evolution &amp; Ecology</a></li>
	
            </ul>
        </li>        <li><h4>Life sciences</h4>
            <ul>
		<li><a href="http://www.nature.com/biotech/index.html">Biotechnology</a></li>
                <li><a href="http://www.nature.com/cancer/index.html">Cancer</a></li>
                <li><a href="http://www.nature.com/development/index.html">Development</a></li>
                <li><a href="http://www.nature.com/drugdisc/index.html">Drug discovery</a></li>
                <li><a href="http://www.nature.com/evoeco/index.html">Evolution &amp; Ecology</a></li>
                <li><a href="http://www.nature.com/genetics/index.html">Genetics</a></li>
                <li><a href="http://www.nature.com/immuno/index.html">Immunology</a></li>
                <li><a href="http://www.nature.com/medicalresearch/index.html">Medical research</a></li>
				<li><a href="http://www.nature.com/protometh/index.html">Methods &amp; Protocols</a></li>
                <li><a href="http://www.nature.com/micro/index.html">Microbiology</a></li>
                <li><a href="http://www.nature.com/molcellbio/index.html">Molecular cell biology</a></li>
                <li><a href="http://www.nature.com/neurosci/index.html">Neuroscience</a></li>
                <li><a href="http://www.nature.com/pharma/index.html">Pharmacology</a></li>
                <li><a href="http://www.nature.com/sysbio/index.html">Systems biology</a></li>
            </ul>
        </li>        <li><h4>Physical sciences</h4>
            <ul>
				<li><a href="http://www.nature.com/physics/index.html">Physics</a></li>
				<li><a href="http://www.nature.com/materials/index.html">Materials</a></li>
            </ul>
        </li>	</ul>
	
	<h3 class="view"><a href="http://www.nature.com/siteindex/index.html" title="View A to Z list of Nature Publishing Group journals">by A - Z Index</a></h3>
</div></div></div><div id="extranav"><hr class="separator">
<h1 class="hidden">Extra navigation</h1><!--<div class="ad-rh marketing">
	<div>
		<h2>Subscribe to <span class="journal-name">Nature Biotechnology</span></h2>
				<p class="fake-button"><a href="https://secure.nature.com/subscribe/nbt">Subscribe<span class="hidden"> to Nature Biotechnology</span></a></p>
	</div> 
</div>-->
<div class="ad-rh marketing advert">
	<div class="advertise-button"><a href="http://ad.doubleclick.net/clk;203935506;4756428;m?http://www.scibx.com/scibx/index.html"><img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/scibx_trial.gif" alt="SciBX - Science-Business eXchange - New from NPG and BioCentury Free Trial"></a></div>
</div>
<div id="articlenav"><h2 class="nav-header"><span class="hidden">ARTICLE NAVIGATION - </span>This issue</h2><ul class="nav-list"><li><a class="toc" href="http://www.nature.com/nbt/journal/v26/n10">Table of contents<span class="hidden"> for this issue</span></a></li><li><a title="previous article in this issue" class="prev" href="http://www.nature.com/nbt/journal/v26/n10/full/nbt1494.html">Previous article</a></li><li><a title="next article in this issue" class="next" href="http://www.nature.com/nbt/journal/v26/n10/full/nbt.1495.html">Next article</a></li></ul>
	


		

	



		

		

		
	





<h2 class="nav-header">Article tools</h2>
<ul class="nav-list"><li><a href="http://www.nature.com/nbt/journal/v26/n10/pdf/nbt1486.pdf" class="download-pdf">Download PDF</a></li><li><a href="http://www.nature.com/foxtrot/svc/mailform?file=/nbt/journal/v26/n10/full/nbt1486.html" class="sendtofriend">Send to a friend</a></li><li class="export"><a href="http://www.nature.com/nbt/journal/v26/n10/ris/nbt1486.ris" class="export">Export citation</a></li><li class="export"><a href="http://www.nature.com/nbt/journal/v26/n10/ris/nbt1486refs.ris" class="export">Export references</a></li><li class="permissions"><a class="permissions" href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPG&amp;publication=Nature%20Biotechnology&amp;title=Next-generation%20DNA%20sequencing&amp;author=Jay%20Shendure%20and%20Hanlee%20Ji&amp;contentID=10.1038/nbt1486&amp;publicationDate=10/09/2008&amp;volumeNum=26&amp;issueNum=10&amp;numPages=11&amp;pageNumbers=pp1135-1145">Rights and permissions</a></li><li><a class="commercial-reprints" href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPGR&amp;publication=Nature%20Biotechnology&amp;title=Next-generation%20DNA%20sequencing&amp;author=Jay%20Shendure%20and%20Hanlee%20Ji%20&amp;contentID=10.1038/nbt1486&amp;publicationDate=10/09/2008&amp;volumeNum=26&amp;issueNum=10&amp;numPages=11&amp;pageNumbers=pp1135-1145&amp;orderBeanReset=true"> Order commercial reprints</a></li><li><a href="http://www.connotea.org/add?uri=http://www.nature.com/nbt/journal/v26/n10/full/nbt1486.html" class="save">Save this link</a></li></ul>
<!-- end articlenav - main --><h2 class="nav-header">Article navigation</h2><ul class="nav-list border"><li><a class="page" href="#abs">Abstract</a></li><li><a class="figures" href="http://www.nature.com/nbt/journal/v26/n10/fig_tab/nbt1486_ft.html">Figures &amp; tables</a></li><li><a class="page" href="#acknowledgements">Acknowledgments</a></li><li><a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_audecl.html">Competing financial interests</a></li><li><a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_BX1.html">Box 1</a></li><li><a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_BX2.html">Box 2</a></li><li><a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_BX3.html">Box 3</a></li><li><a href="http://www.nature.com/nbt/journal/v26/n10/box/nbt1486_BX4.html">Box 4</a></li><li><a class="page" href="#References">References</a></li></ul></div><div class="extra liteblue"><h2 class="purp_back">Search PubMed for</h2><ul><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Shendure+J">Jay Shendure</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ji+H">Hanlee Ji</a></li></ul></div>				
<div class="box" id="natjob">
<h2 class="natjob"><a href="http://www.nature.com/naturejobs/"><span class="first">nature</span><span class="hidden"> </span><span class="second">jobs</span></a></h2>

	<ul class="natjob">

	<li><h3 class="first"><a href="http://nature.com/naturejobs/science/jobs/81198">Endowed Chair in Glaucoma Research : Toronto, Canada</a></h3>
	<ul>
	<li>University Health Network</li>
	<li>Toronto, Canada</li>
	</ul>
	</li>

			<li><h3><a href="http://nature.com/naturejobs/science/jobs/81367">Manager / Dy Manager - Bioinformatics</a></h3>
		<ul>
		<li>Vimta Labs Limited</li>
		<li>Hyderabad, Andhra Pradesh 500 051 India</li>
		</ul>
		</li>
	</ul>


<div class="natjobmore"><a href="http://www.nature.com/naturejobs/science/jobs">More science jobs</a></div>
<div class="natjobmore"><a href="http://www.nature.com/naturejobs/science/jobs/new">Post a job for free</a></div>
</div>
<div id="natprod" class="box"><form method="post" action="http://www.biocompare.com/nature/search.asp"><h2 class="natprod"><a href="http://www.biocompare.com/nature/"><span class="first">nature</span><span class="hidden"> </span><span class="second">products</span></a></h2><div class="natprodmore"><label for="buyerguide" class="hidden">enter buyers guide search text here</label><input id="buyerguide" name="search" type="text"><br><input class="button" value="search buyers guide" type="submit"></div></form></div>				<!-- skyscraper ad -->
<div class="ad-rh skyscraper wide">
<p>ADVERTISEMENT</p>











		
<div class="adunit-leaderboard">
<script type="text/javascript"><!--//--><![CDATA[//><!--
		document.write('<script type="text/javascript" src="http://ad.doubleclick.net/adj/biotech.nature.com/article;abr=!webtv;type=re;artid=nbt1486;date=20081009;issue=10;subco=;chaco=;keywd=;njdis=;njreg=;crstg=;pos=right;sz=160x600;ptile=3;ord=' + ord + '?" ><\/script>');

	if ((!document.images && navigator.userAgent.indexOf('Mozilla/2.') >= 0) || (navigator.userAgent.indexOf("WebTV") >= 0)) {
		document.write('<a href="http://ad.doubleclick.net/jump/biotech.nature.com/article;keywd=;sz=160x600;ord=' + ord + '?">');
		document.write('<img src="http://ad.doubleclick.net/ad/biotech.nature.com/article;keywd=;sz=160x600;ord=' + ord + '?" width="160" height="600" border="0"></a>');
	}
//--><!]]></script><script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/article_003.js"></script><a target="blank" href="http://ad.doubleclick.net/click;h=v8/379f/0/0/%2a/e;208817344;0-0;0;17389285;2321-160/600;28869846/28887725/3;;%7Esscs=%3fhttp://www.aeonclad.com"><img src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/1-AeonCladNEW160x6002008.gif" alt="Click here to find out more!" border="0"></a>
<noscript> <div><a
href="http://ad.doubleclick.net/jump/biotech.nature.com/article;abr=!NN2;tile=1;type=re;artid=nbt1486;date=20081009;issue=10;subco=;chaco=;keywd=;njdis=;njreg=;crstg=;pos=right;sz=160x600;ptile=3;ord=123456789?"><img
src="http://ad.doubleclick.net/ad/biotech.nature.com/article;abr=!NN2;tile=1;type=re;artid=nbt1486;date=20081009;issue=10;subco=;chaco=;keywd=;njdis=;njreg=;crstg=;pos=right;sz=160x600;ptile=3;ord=123456789?"
alt="Advertisement" style="border:0" /></a></div>
</noscript>

</div>


</div>
<!-- end skyscraper ad -->	</div>	

	



<div class="cleardiv"><!-- --></div>


		
			<p class="back-to-top hidden"><a href="#top" title="Return to the top of this page">Top</a></p>
	
	<div id="ftr" class="footer">
					<ul id="footer-journal">
				<li class="journal-name">Nature Biotechnology</li>				<li class="issn"><abbr title="International Standard Serial Number">ISSN</abbr><span class="hidden">:</span> 1087-0156</li>				<li class="eissn"><abbr title="Electronic International Standard Serial Number">EISSN</abbr><span class="hidden">:</span> 1546-1696</li>											</ul>
			<span class="cleardiv"><!-- --></span>
				
		<div class="footer-inner">
			
		<div class="baseline-wrapper">
													<ul class="footer-links baseline first">
									<li><a href="http://www.nature.com/npg/">About <abbr title="Nature Publishing Group">NPG</abbr></a></li>
														<li><a href="http://www.nature.com/npg_/contact/">Contact <abbr title="Nature Publishing Group">NPG</abbr></a></li>
														<li><a href="http://www.nature.com/webfeeds/"><abbr title="RDF Site Summary">RSS</abbr> web feeds</a></li>
														<li><a href="http://www.nature.com/help/">Help</a></li>
													</ul>
							<ul class="footer-links baseline">
									<li><a href="http://www.nature.com/info/privacy.html">Privacy policy</a></li>
														<li><a href="http://www.nature.com/info/legal_notice.html">Legal notice</a></li>
														<li><a href="http://www.nature.com/info/accessibility_statement.html">Accessibility statement</a></li>
													</ul>
							<ul class="footer-links baseline last">
									<li><a href="http://www.nature.com/news/">Nature News</a></li>
														<li><a href="http://www.nature.com/naturejobs/">Nature<span class="hidden"> </span>jobs</a></li>
														<li><a href="http://www.natureasia.com/">Nature Asia</a></li>
														<li><a href="http://www.nature.com/scitable/">Nature Education</a></li>
													</ul>
					
						<form style="height: 60px;" method="get" action="/search/executeSearch" class="search-form baseline">
				<fieldset>
					<label for="footer-search">Search:</label>
					<input name="sp-q" id="footer-search" value="" class="text" type="text">
					<input name="sp-c" value="10" type="hidden">
					<input name="sp-x-9" value="cat" type="hidden">
					<input name="sp-s" value="0" type="hidden">
					<input name="sp-a" value="sp1001702d" type="hidden">
					<input name="sp-sfvl-field" value="subject|ujournal" type="hidden">
					<input name="sp-x-1" value="ujournal" type="hidden">
					<input name="sp-p-1" value="phrase" type="hidden">
					<input name="search" value="go" class="button" type="submit">
				</fieldset>
			</form>
					</div>
		
		<span class="cleardiv"><!-- --></span>
		
					<div id="footer-copyright">
				<ul class="logo">
					<li>			<a href="http://www.nature.com/info/copyright_statement.html" title="" class="class java.util.HashMap">© 2008 Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.</a>
	</li>					<li>partner of <a href="http://www.nature.com/info/partners.html">AGORA, HINARI, OARE, INASP, CrossRef and COUNTER</a></li>				</ul>
			</div>
				</div>
	</div>

	</div></div><!-- close constrain divs -->

																																																
						<script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/jquery.js"></script>
								<script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/global_002.js"></script>
								<script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/runOnLoad.js"></script>
								<script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/group.js"></script>
														<script type="text/javascript"><!--//--><![CDATA[//><!--
				/* Callback function for "Articles citing this one on Scopus"
   We tried using (function $() {   .....  }) (jQuery); but this delayed loading this script so that the 
   calling script executed first, and could not find this function. */
var scopus = {
	receive: function(data){
		// get the article search total results count (should be 1)
		var articleResultCount=data.OK.TotalResults;
		if(articleResultCount > 0) {
			// get the count of citing articles and output to page
			var citedArticleCount = data.OK.Results[0].citedbycount;
			if(citedArticleCount > 0) {
				// get the url to the "cited by" results page
				var citationLink = data.OK.Results[0].inwardurl;
				// create the HTML snippet <li> to add
				var snippetLIElement = '<li class="cited"><a class="cited" href="' + citationLink + '">Scopus lists ' + citedArticleCount + ' article' + ((citedArticleCount > 1)?'s':'') + ' citing this article</a></li>';
				// add the new <li> a) after a <li class="cited"> if it exists; b) otherwise, before <li class="export">
				var citedLiElements = jQuery("li.cited");
				if(jQuery(citedLiElements).length == 1) {
					// insert the <li> after the "CrossRef citing" element
					jQuery(citedLiElements[0]).after(snippetLIElement);
				} else if (jQuery("li.export").length > 0) {
					// insert the <li> before the "Export references" element
					jQuery("li.export:last").after(snippetLIElement);
				} else if (jQuery("li.save").length > 0) {
					jQuery("li.save").before(snippetLIElement);
				}
			}
		}
	}
}

			//--><!]]></script>
								<script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/search.url"></script>
								<script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/baseline.js"></script>
								<script type="text/javascript" src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/global.js"></script>
			

<script type="text/javascript"><!--//--><![CDATA[//><!--
(function($) {
	$('a.popup-this').bind('click', function() {
		window.open(this.href);
		return false;
	});
})(jQuery);
//--><!]]></script>
<!--[if IE 6]>
<![endif]--><!-- webtrends Version: 8.6.0 -->
	 <script src="Shendure%20&amp;%20Ji%202008%20Next-generation%20DNA%20sequencing_files/webtrends.js" type="text/javascript"></script>
	<script type="text/javascript"><!--//--><![CDATA[//><!--
		var _tag=new WebTrends();
		_tag.dcsGetId();
	//--><!]]></script>
	<script type="text/javascript"><!--//--><![CDATA[//><!--
		// Add custom parameters here.
		// _tag.DCSext.param_name=param_value;
		_tag.dcsCollect();
	//--><!]]></script>
	<noscript>
<div><img alt="" id="DCSIMG" width="1" height="1"
src="http://statse.webtrendslive.com/dcs0zztfg00000s969s37qoal_2f6z/njs.gif?dcsuri=/nojavascript&amp;WT.js=No&amp;WT.tv=8.6.0"
/></div> </noscript>
	<script type="text/javascript"><!--//--><![CDATA[//><!--
	(function($) {
		$(function() {
			var getMeta = function(name) {
				return $('meta[name="' + name + '"]').attr('content');
			};
			var applyParams = function(wt) {
				var params = [];
				for (var prop in wt) {
					params.push(prop);
					params.push(wt[prop]);
				}
				_tag.dcsMultiTrack.apply(_tag, params);
			};

			$('a[href$=".pdf"], a[href*=".pdf?"]').bind('click', function() {
				var wt = {};
				wt['DCS.dcssip'] = this.hostname ? this.hostname.split(':')[0] : '';
				wt['DCS.dcsuri'] = this.pathname ? this.pathname.indexOf('/') !== 0 ? '/' + this.pathname : this.pathname : '/';
				wt['DCS.dcsqry'] = this.search ? this.search.substring(1) : '';
				wt['WT.cg_n'] = '2008-10 26';
				wt['WT.cg_s'] = '';
				wt['WT.art_journal_title'] = getMeta('citation_journal_title');
				wt['WT.art_volume'] = getMeta('citation_volume');
				wt['WT.art_issue'] = getMeta('citation_issue');
				wt['WT.art_title'] = getMeta('citation_title');
				wt['WT.art_doi'] = getMeta('citation_doi');
				wt['WT.art_authors'] = getMeta('citation_authors').replace(/;/g, '%3B').replace(/\s*,\s*/g, ';');
				wt['WT.art_date'] = getMeta('citation_date');
				applyParams(wt);
			});
			
			$('a[href*="/rss"]').each(function() {
				if (this.href.match(/\/rss\/?(\?.*)?$/)) {
					$(this).bind('click', function() {
						var wt = {};
						wt['DCS.dcsuri'] = this.pathname ? this.pathname.indexOf('/') !== 0 ? '/' + this.pathname : this.pathname : '/';
						wt['DCS.dcsqry'] = this.search ? this.search.substring(1) : '';
						wt['WT.ti'] = 'Download:' + (this.hostname ? this.hostname.split(':')[0] : '') + wt['DCS.dcsuri'] + (wt['DCS.dcsqry'] ? '?' + wt['DCS.dcsqry'] : '');
						wt['WT.dl'] = '20';
						applyParams(wt);
					});
				}
			});

			$('div[class*="adunit-"] a').bind('click', function() {
				var classes = $(this).parents('div[class*="adunit-"]:first').attr('class');
				if (classes) {
					var match = /(^|\s)adunit-([^\s]+)/.exec(classes);
					_tag.dcsMultiTrack('WT.ac', (match) ? match[2] : '');
				}
			});
		});	
	})(jQuery);
	//--><!]]></script>
	<!-- end webtrends Version: 8.6.0 -->
	</body></html>