<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN">
<html xmlns:m="http://www.w3.org/1998/Math/MathML"><head><title>BioMed Central | Full text | Wildfire: distributed, Grid-enabled workflow construction and execution</title><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

<meta name="citation_pdf_url" content="http://www.biomedcentral.com/content/pdf/1471-2105-6-69.pdf">

<meta name="citation_abstract_html_url" content="http://www.biomedcentral.com/1471-2105/6/69/abstract/">
<meta name="citation_fulltext_html_url" content="http://www.biomedcentral.com/1471-2105/6/69">

<script src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/viewingoptions.js" type="text/javascript"></script><link rel="search" type="application/opensearchdescription+xml" title="BioMed Central Search" href="http://www.biomedcentral.com/download/toolbar/firefox/bmc.xml">
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script><script src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/ga.js" type="text/javascript"></script>
<script type="text/javascript">
var pageTracker = _gat._getTracker("UA-72326-1");
pageTracker._setLocalRemoteServerMode(); pageTracker._initData();
pageTracker._trackPageview();
</script>
<link rel="stylesheet" type="text/css" href="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/screen.css" title="Default Styles"><!--[if gte IE 7]><link rel="stylesheet" type="text/css" href="/stylesheets/ie7.css" /><![endif]--><!--[if lte IE 6]><link rel="stylesheet" type="text/css" href="/stylesheets/ie56.css" /><![endif]-->

<meta name="dc.title" content="Wildfire: distributed, Grid-enabled workflow construction and execution">
<meta name="dc.creator" content="Francis Tang">
<meta name="dc.creator" content="Ching Lian Chua">
<meta name="dc.creator" content="Liang-Yoong Ho">
<meta name="dc.creator" content="Yun Ping Lim">
<meta name="dc.creator" content="Praveen Issac">
<meta name="dc.creator" content="Arun Krishnan">
<meta name="dc.type" content="Software">
<meta name="dc.source" content="BMC Bioinformatics 2005 6:69">
<meta name="dc.date" content="2005-3-24">
<meta name="dc.identifier" content="10.1186/1471-2105-6-69">
<meta name="dc.publisher" content="BioMed Central Ltd">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/">
<meta name="dc.format" content="text/html">
<meta name="dc.language" content="en">

<meta name="prism.publicationName" content="BMC Bioinformatics">
<meta name="prism.issn" content="1471-2105">
<meta name="prism.publicationDate" content="2005-3-24">
<meta name="prism.volume" content="6">
<meta name="prism.number" content="1">
<meta name="prism.section" content="Software">
<meta name="prism.startingPage" content="69">
<meta name="prism.copyright" content="2005 Tang et al; ">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

<meta name="citation_journal_title" content="BMC Bioinformatics">
<meta name="citation_publisher" content="BioMed Central Ltd">
<meta name="citation_authors" content="Francis Tang, Ching Lian Chua, Liang-Yoong Ho, Yun Ping Lim, Praveen Issac, Arun Krishnan">
<meta name="citation_title" content="Wildfire: distributed, Grid-enabled workflow construction and execution">
<meta name="citation_volume" content="6">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="69">
<meta name="citation_doi" content="10.1186/1471-2105-6-69">

<link rel="stylesheet" type="text/css" href="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/screen_002.css" title="Default Styles">

<link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/content/pdf/1471-2105-6-69.pdf">
<link rel="alternate" type="text/xml" title="XML version" href="http://www.biomedcentral.com/content/xml/1471-2105-6-69.xml">

</head><div FirebugVersion="1.3.3" style="display: none;" id="_firebugConsole"></div><body class="bgimage" alink="#6699ff" link="#003399" vlink="#3366cc"><center><div id="outerframe"><div class="skip"><a href="#content">Skip to main content</a></div><div id="newad"><table align="center" border="0" cellpadding="0" cellspacing="0"><tbody><tr><td class="topad" align="center">

<center>

<!------ OAS SETUP begin ------>
<script language="JavaScript">
<!--
OAS_version = 10;
function OAS_NORMAL(pos)
{
   document.write('<a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1691841604@Top,Bottom,Left1,Bottom1!' + pos + '" target=_top>');
   document.write('<img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1691841604@Top,Bottom,Left1,Bottom1!' + pos + '" border=0></a>');
}
//-->
</script>
<script language="JavaScript1.1">
<!--
OAS_version = 11;
if (navigator.userAgent.indexOf('Mozilla/3') > 1)
   OAS_version = 10;
if (navigator.userAgent.indexOf('Netscape/7.0') > 1)
   OAS_version = 10;
if (OAS_version >= 11)
   document.write('<scr' + 'ipt language="JavaScript1.1" src="http://oas.biomedcentral.com/RealMedia/ads/adstream_mjx.ads/www.biomedcentral.com/article/bmcbioinformatics/1691841604@Top,Bottom,Left1,Bottom1"></scr' + 'ipt>');
//-->
</script><script language="JavaScript1.1" src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/1691841604TopBottomLeft1Bottom1.js"></script>
<script language="JavaScript">
<!--
document.write('');
function OAS_AD(pos)
{
   if (OAS_version >= 11)
      OAS_RICH(pos);
   else
      OAS_NORMAL(pos);
}
//-->
</script>
<!------ OAS SETUP end ------>
<div id="dropdown" onmouseover="testMouse()" onmouseout="testMouse()">
<!------ OAS AD 'Top' begin ------>
<script language="JavaScript">
<!--
OAS_AD('Top');
//-->
</script><!-- 
Support: http://oas.biomedcentral.com#biomed/Alma-Biomed-Lead-3#Alma-Biomed-Lead-3#Alma-Biomed-Lead-3.html#5b135#1239788142#437#S#Top#www.biomedcentral.com/article/bmcbioinformatics##
--><script id="extFlashTop1" type="text/javascript" src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/TFSMflashobject.js"></script>
<script language="JavaScript">
<!--
if(!document.body)
document.write("<html><body>");
OASd = document;
var plug = false;
var flashVersion = -1;
var minFlashVersion = 6;
if(navigator.plugins != null && navigator.plugins.length > 0){flashVersion =(navigator.plugins["Shockwave Flash 2.0"] || navigator.plugins["Shockwave Flash"]) ? navigator.plugins["Shockwave Flash" +(navigator.plugins["Shockwave Flash 2.0"] ? " 2.0" : "")].description.split(" ")[2].split(".")[0] : -1;
plug = flashVersion >= minFlashVersion;}
else if(navigator.userAgent.toLowerCase().indexOf("webtv/2.6") != -1){flashVersion = 4;plug = flashVersion >= minFlashVersion;}
else if(navigator.userAgent.toLowerCase().indexOf("webtv/2.5") != -1){flashVersion = 3;plug = flashVersion >= minFlashVersion;}
else if(navigator.userAgent.toLowerCase().indexOf("webtv") != -1){flashVersion = 2;plug = flashVersion >= minFlashVersion;}
else if((navigator.appVersion.indexOf("MSIE") != -1) &&(navigator.appVersion.toLowerCase().indexOf("win") != -1) &&(navigator.userAgent.indexOf("Opera") == -1)){var oasobj;
var exc;
try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.7");
flashVersion = oasobj.GetVariable("$version");} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.6");
version = "WIN 6,0,21,0"; 
oasobj.AllowScriptAccess = "always";
flashVersion = oasobj.GetVariable("$version");} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.3");
flashVersion = oasobj.GetVariable("$version");} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash.3");
flashVersion = "WIN 3,0,18,0";} catch(exc){try{oasobj = new ActiveXObject("ShockwaveFlash.ShockwaveFlash");
flashVersion = "WIN 2,0,0,11";} catch(exc){flashVersion = -1;}}}}}
plug =(flashVersion != -1)? flashVersion.split(" ")[1].split(",")[0] >= minFlashVersion : false;}

if(plug)
{
document.write("<div id=\"FinContentTop1\"></div>");
function loadFlashTop1(){
	if(navigator.userAgent.indexOf("MSIE") != -1 && navigator.userAgent.indexOf("Opera") == -1){
	
		if (extFlashTop1.readyState == "complete")
  		{
	   		FlashObject("http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/novoseek_728x90_05.swf?clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1552637163/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?", "OAS_AD_Top", "width=728 height=90", "opaque", "clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" ,"6", "FinContentTop1");
			extFlashTop1.onreadystatechange = "";
		}
	
		extFlashTop1.onreadystatechange = loadFlashTop1;
	}
	else
	{
           OASfp=" Menu=FALSE swModifyReport=TRUE width=728 height=90 ";
           OASd.write("<object id=\"techsource_Top\" classid=\"clsid:D27CDB6E-AE6D-11cf-96B8-444553540000\" codebase=\"http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,0,0\""+OASfp+">");
           OASd.write("<param name=src value=\"http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/novoseek_728x90_05.swf\"><param name=allowScriptAccess value=always><param name=quality value=autohigh><param name=loop value=true><param name=play value=true><param name=menu value=false><param name=wmode value=opaque><param name=FlashVars value=\"clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1552637163/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?\">");
           OASd.write("<embed src=\"http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/novoseek_728x90_05.swf\""+OASfp+"pluginspage=\"http://www.macromedia.com/shockwave\" type=\"application/x-shockwave-flash\" width=728 height=90 PLAY=true LOOP=true QUALITY=autohigh WMODE=opaque FlashVars=\"clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1552637163/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?\" allowScriptAccess=always>");
           OASd.write("</embed></object>");
        }

}
	loadFlashTop1();
}
else
{
OASd.write("<a href=\"http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1552637163/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?\" target=\"_blank\"><img src=\"http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/banner_alternativo3.gif\" width=728 height=90 border=0 alt=\"\"></a>");}
if(!document.body)
document.write("</body></html>");
//-->
</script><div id="FinContentTop1"></div><object id="techsource_Top" classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,0,0" menu="FALSE" swmodifyreport="TRUE" height="90" width="728"><param name="src" value="http://realmedia-a592.d4p.net/6/592/1130/0001/oas-eu.247realmedia.com/RealMedia/ads/Creatives/biomed/Alma-Biomed-Lead-3/novoseek_728x90_05.swf"><param name="allowScriptAccess" value="always"><param name="quality" value="autohigh"><param name="loop" value="true"><param name="play" value="true"><param name="menu" value="false"><param name="wmode" value="opaque"><param name="FlashVars" value="clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1552637163/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?"><embed src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/novoseek_728x90_05.swf" menu="FALSE" swmodifyreport="TRUE" pluginspage="http://www.macromedia.com/shockwave" type="application/x-shockwave-flash" play="true" loop="true" quality="autohigh" wmode="opaque" flashvars="clickTag=http://oas.biomedcentral.com/RealMedia/ads/click_lx.ads/www.biomedcentral.com/article/bmcbioinformatics/1552637163/Top/biomed/Alma-Biomed-Lead-3/Alma-Biomed-Lead-3.html/34386665393030393439386238646230?" allowscriptaccess="always" height="90" width="728"></object><noscript>
<a
href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1691841604@Top,Bottom,Left1,Bottom1!Top"
target=_top>
<img
src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1691841604@Top,Bottom,Left1,Bottom1!Top"
border=0></a>
</noscript></div></center></td><td valign="top"><table height="95"><tbody><tr><td style="vertical-align: middle;"><a href="http://www.biomedcentral.com/info/mediainformation/"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/advhere.gif" alt="" border="0"></a></td></tr></tbody></table>
<!------ OAS AD end ------>


</td></tr></tbody></table></div><div id="innerframe"><table border="0" cellpadding="5" cellspacing="0" width="100%"><tbody><tr valign="bottom"><td width="50"><a href="http://www.biomedcentral.com/bmcbioinformatics"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/logo.gif" alt="" border="0" hspace="3"></a></td><td align="left"><a href="http://www.biomedcentral.com/info/about/faq?name=impactfactor"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/1002.gif" alt="" border="0" hspace="3"></a></td><td id="topnav" align="right"><strong>Welcome </strong>University of Miami (<a href="http://www.biomedcentral.com/registration/">Log on / register</a>)<br><a href="http://www.biomedcentral.com/support/feedback/" onclick="popup(this.href,'Feedback',650,500); return false;">Feedback</a> | <a href="http://www.biomedcentral.com/support/">Support</a> | <a href="http://www.biomedcentral.com/my/">My details</a></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr valign="top"><td colspan="3" bgcolor="#003399"><script type="text/javascript"><!--
document.write("<div id='TopBar'><font color='#ffffff'>&nbsp;<a href='/' class='xtoplinks'><b> home </b></a>|<a href='/browse/journals/' class='xtoplinks'><b> journals A-Z </b></a>|<a href='/browse/bysubject/' class='xtoplinks'><b> subject areas </b></a>|<a href='/search/' class='xtoplinks'><b> advanced search </b></a>|<a href='/info/authors/' class='xtoplinks'><b> authors </b></a>|<a href='/info/about/peerreview/' class='xtoplinks'><b> reviewers </b></a>|<a href='/info/libraries' class='xtoplinks'><b> libraries </b></a>|<a href='http://careers.the-scientist.com/' class='xtoplinks'><b> jobs </b></a>|<a href='/info/' class='xtoplinks'><b> about </b></a>|<a href='/my/' class='xtoplinks'><b> my BioMed Central </b></a></font></div>");//--></script><div id="TopBar"><font color="#ffffff">&nbsp;<a href="http://www.biomedcentral.com/" class="xtoplinks"><b> home </b></a>|<a href="http://www.biomedcentral.com/browse/journals/" class="xtoplinks"><b> journals A-Z </b></a>|<a href="http://www.biomedcentral.com/browse/bysubject/" class="xtoplinks"><b> subject areas </b></a>|<a href="http://www.biomedcentral.com/search/" class="xtoplinks"><b> advanced search </b></a>|<a href="http://www.biomedcentral.com/info/authors/" class="xtoplinks"><b> authors </b></a>|<a href="http://www.biomedcentral.com/info/about/peerreview/" class="xtoplinks"><b> reviewers </b></a>|<a href="http://www.biomedcentral.com/info/libraries" class="xtoplinks"><b> libraries </b></a>|<a href="http://careers.the-scientist.com/" class="xtoplinks"><b> jobs </b></a>|<a href="http://www.biomedcentral.com/info/" class="xtoplinks"><b> about </b></a>|<a href="http://www.biomedcentral.com/my/" class="xtoplinks"><b> my BioMed Central </b></a></font></div><noscript>&nbsp;<a
href='/'><span class='tinytext' style='color:#ffffff'><B> home
</b></span></A>|<a href='/browse/journals/'><span class='tinytext'
style='color:#ffffff'><B> journals A-Z </b></span></A>|<a
href='/browse/bysubject/'><span class='tinytext'
style='color:#ffffff'><B> subject areas </b></span></A>|<a
href='/search/'><span class='tinytext' style='color:#ffffff'><B>
advanced search </b></span></A>|<a href='/info/authors/'><span
class='tinytext' style='color:#ffffff'><B> authors </b></span></A>|<a
href='/info/about/peerreview/'><span class='tinytext'
style='color:#ffffff'><B> reviewers </b></span></A>|<a
href='/info/libraries'><span class='tinytext' style='color:#ffffff'><B>
libraries </b></span></A>|<a
href='http://careers.the-scientist.com/'><span class='tinytext'
style='color:#ffffff'><B> jobs </b></span></A>|<a href='/info/'><span
class='tinytext' style='color:#ffffff'><B> about </b></span></A>|<a
href='/my/'><span class='tinytext' style='color:#ffffff'><B> my BioMed
Central </b></span></A></noscript></td></tr></tbody></table><div id="article"><script src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/fulltext.js" type="text/javascript"></script><table cellpadding="0" cellspacing="0" width="100%"><tbody valign="top"><tr><td id="leftcolumn"><ul style="position: static;" id="outline">
<li><a href="#">Top</a></li>
<li><a href="#IDALWETS">Abstract</a></li>
<li><a href="#IDA1WETS">Background</a></li><li><a href="#IDARIH0Z">Implementation</a></li><li><a href="#IDA2MH0Z">Results</a></li><li><a href="#IDAZYH0Z">Discussion</a></li><li><a href="#IDAIZH0Z">Availability and requirements</a></li><li><a href="#IDA10H0Z">Authors' contributions</a></li><li><a href="#IDAB1H0Z">Acknowledgements</a></li>
<li class="lowest"><a href="#refs">References</a></li>
</ul>
<table width="130"><tbody><tr><td>&nbsp;</td></tr></tbody></table></td><td id="content"><div id="navbox">
<table cellpadding="5" cellspacing="2">
<tbody><tr>
<td><a href="http://www.biomedcentral.com/bmcbioinformatics"><strong>BMC Bioinformatics</strong></a><br><a href="http://www.biomedcentral.com/bmcbioinformatics/6">Volume 6</a><br></td>
</tr><tr><td><strong>Viewing options:</strong><ul>
<li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/6/69/abstract/">Abstract</a></li>
<li>&nbsp;<strong>Full text</strong></li>
<li>&nbsp;<a href="http://www.biomedcentral.com/content/pdf/1471-2105-6-69.pdf">PDF </a>
		(613KB)
				</li>
</ul><strong>Associated material:</strong><ul><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/6/69/comments">Readers' comments</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/pubmed/15788106" onclick="largepopup(this.href,'pubmed');return false">PubMed record</a></li></ul><strong>Related literature:</strong><ul><li>&nbsp;Articles citing this article<div><a href="http://www.biomedcentral.com/search/citedby/15788106">on BioMed Central</a></div><div><a href="http://scholar.google.com/scholar?hl=en&amp;lr=&amp;safe=off&amp;c2coff=1&amp;q=link%3Ahttp://www.biomedcentral.com/1471-2105/6/69&amp;btnG=Search" onclick="largepopup(this.href,'on_google');return false">on Google Scholar</a></div><div><a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=BMC_sci&amp;SrcApp=BMC_sci&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000228346900001" onclick="largepopup(this.href,'on_isi');return false">on&nbsp;ISI&nbsp;Web&nbsp;of&nbsp;Science</a></div><div><a href="http://www.biomedcentral.com/search/citedby/pmc/15788106">on PubMed Central</a></div></li>
<li>&nbsp;Other articles by authors<br><div><a href="javascript://" onclick="toggleauthors('authg','authbuttong'); return false"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/expand2.gif" id="authbuttong" alt="">on Google Scholar
				</a></div>
<div style="display: none;" id="authg"><a href="http://scholar.google.com/scholar?q=author%3A%22F+Tang%22">Tang F</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22CL+Chua%22">Chua CL</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22LY+Ho%22">Ho LY</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22YP+Lim%22">Lim YP</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22P+Issac%22">Issac P</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22A+Krishnan%22">Krishnan A</a></div>
<div><a href="javascript://" onclick="toggleauthors('authpm','authbuttonpm'); return false"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/expand2.gif" id="authbuttonpm" alt="">on PubMed
				</a></div>
<div style="display: none;" id="authpm"><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Tang_F&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Tang F</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Chua_CL&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Chua CL</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Ho_LY&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Ho LY</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Lim_YP&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Lim YP</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Issac_P&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Issac P</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Krishnan_A&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Krishnan A</a></div>
</li>
<li>&nbsp;Related articles/pages<br><div><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2105/6/69">on Google</a></div>
<div><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2105/6/69">on Google Scholar</a></div>
<div><a href="http://www.biomedcentral.com/pubmed/related/15788106">on PubMed</a></div>
</li></ul><strong>Tools:</strong><ul><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/6/69/citation">Download citation(s)</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/content/download/xml/1471-2105-6-69.xml">Download XML</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/6/69/email">Email to a friend</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/info/about/reprintsandperm">Order reprints</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/6/69/postcomment">Post a comment</a></li>
<li>&nbsp;<a href="http://www.biomedcentral.com/bmcbioinformatics/alerts/">Sign up for article alerts</a></li></ul><script type="text/javascript">toggleauthors('authpm','authbuttonpm'); toggleauthors('authg','authbuttong'); toggleauthors('authos','authbuttonos'); toggleauthors('autha','authbuttona');</script><strong>Post to:</strong><br><ul class="iconlist">
<li><a href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2105/6/69" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false"><img alt="" src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/citeulink.gif"></a>&nbsp;<a href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2105/6/69" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false">Citeulike				
				</a></li>
<li><a onclick="largepopup(this.href,'Connotea',800,600);return false" href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1471-2105-6-69"><img alt="" src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/connotea.gif"></a>&nbsp;<a href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1471-2105-6-69" onclick="largepopup(this.href,'Connotea',800,600);return false">Connotea
					</a></li>
<li><a href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2105/6/69&amp;title=Wildfire:%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution" onclick="largepopup(this.href,'Del.icio.us',800,600);return false"><img alt="" src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/delicious.gif"></a>&nbsp;<a href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2105/6/69&amp;title=Wildfire:%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution" onclick="largepopup(this.href,'Del.icio.us',800,600);return false">Del.icio.us</a></li>
<li><a href="http://digg.com/submit?phase=2&amp;url=http://www.biomedcentral.com/1471-2105/6/69&amp;title=Wildfire:%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution&amp;topic=general_sciences" onclick="largepopup(this.href,'Digg',800,600);return false"><img alt="" src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/digg.gif"></a>&nbsp;<a href="http://digg.com/submit?phase=2&amp;url=http://www.biomedcentral.com/1471-2105/6/69&amp;title=Wildfire:%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution&amp;topic=general_sciences" onclick="largepopup(this.href,'Digg',800,600);return false">Digg
				</a></li>
<li><a href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2105/6/69" onclick="largepopup(this.href,'facebook',800,600);return false"><img alt="" src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/facebook.gif"></a>&nbsp;<a href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2105/6/69" onclick="largepopup(this.href,'facebook',800,600);return false">Facebook</a></li>
</ul></td></tr></tbody></table>
</div><a name="maincontent"></a><div id="topmatter"><a name="text"></a><a href="http://www.biomedcentral.com/info/about/openaccess/"><img class="access" src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/openaccess-large.gif" alt="Open Access" height="24" width="89"></a><a href="http://www.biomedcentral.com/info/about/mostviewed/"><img class="access" src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/highlyaccessed-large.gif" alt="Highly Access" height="24" width="124"></a><!-- <rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns="http://purl.org/rss/1.0/">
<cc:Work rdf:about="http://www.biomedcentral.com/1471-2105/6/69">
<cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/>
</cc:Work>
<cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/">
<cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
<cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Notice"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/>
<cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/>
</cc:License>
<item rdf:about="http://www.biomedcentral.com/1471-2105/6/69">
<title>Wildfire: distributed, Grid-enabled workflow construction and execution</title>
<dc:title>Wildfire: distributed, Grid-enabled workflow construction and execution</dc:title>
<dc:creator>Tang, Francis</dc:creator>
<dc:creator>Chua, Ching Lian</dc:creator>
<dc:creator>Ho, Liang-Yoong</dc:creator>
<dc:creator>Lim, Yun Ping</dc:creator>
<dc:creator>Issac, Praveen</dc:creator>
<dc:creator>Krishnan, Arun</dc:creator>
<dc:identifier>info:doi/10.1186/1471-2105-6-69</dc:identifier>
<dc:identifier>info:pmid/15788106</dc:identifier>
<dc:source>BMC Bioinformatics 2005, 6:69</dc:source>
<dc:date>2005-03-24</dc:date>
<prism:publicationName>BMC Bioinformatics</prism:publicationName>
<prism:publicationDate>2005-03-24</prism:publicationDate>
<prism:volume>6</prism:volume>
<prism:number>1</prism:number>
<prism:section>Software</prism:section>
<prism:startingPage>69</prism:startingPage>
</item>
</rdf:RDF> --><span class="articletype">Software</span><h1>Wildfire: distributed, Grid-enabled workflow construction and execution</h1><div class="multipleins"><p class="authors"><strong>Francis Tang</strong><sup>1</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/email_002.gif" alt="email" title="Email"></a>, <strong>Ching Lian Chua</strong><sup>2</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/email_002.gif" alt="email" title="Email"></a>, <strong>Liang-Yoong Ho</strong><sup>1</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/email_002.gif" alt="email" title="Email"></a>, <strong>Yun Ping Lim</strong><sup>3</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/email_002.gif" alt="email" title="Email"></a>, <strong>Praveen Issac</strong><sup>4</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/email_002.gif" alt="email" title="Email"></a> and <strong>Arun Krishnan</strong><sup>1</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/email-ca.gif" title="Corresponding author" alt="email"></a></p><p><sup>1</sup>Information Science Research, Bioinformatics Institute, 30 Biopolis Street, #07-01, Matrix, 138671 Singapore</p><p><sup>2</sup>Operations Research Lab, DSO National Laboratories, 20 Science Park Drive, 118230 Singapore</p><p><sup>3</sup>Singapore Biomedical Computing Resource, Bioinformatics Institute, 30 Biopolis Street, #07-01, Matrix, 138671 Singapore</p><p><sup>4</sup>Global Software Group, Motorola Electronics Pte Ltd, 12 Ang Mo Kio St. 64, Ang Mo Kio Industrial Park 3, 569088 Singapore</p></div><p><span class="pseudotabau"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/email.gif" border="0"> author email</span><span class="pseudotabau"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/email-ca_002.gif"> corresponding author email</span></p><p><em>BMC Bioinformatics</em> 2005,
			 <strong>6</strong><strong>:</strong>69<span class="pseudotab">doi:10.1186/1471-2105-6-69</span></p><p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2105/6/69">http://www.biomedcentral.com/1471-2105/6/69</a></p>
<table cellpadding="0" cellspacing="0">
<tbody>
<tr>
<td>Received:</td>
<td>22&nbsp;November&nbsp;2004</td>
</tr>
<tr>
<td>Accepted:</td>
<td>24&nbsp;March&nbsp;2005</td>
</tr>
<tr>
<td>Published:</td>
<td>24&nbsp;March&nbsp;2005</td>
</tr>
</tbody>
</table>
<p>
						©
						 2005 Tang et al; licensee BioMed Central Ltd.<br>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><a name="IDALWETS"></a><h3>Abstract</h3>
<h4>Background</h4>
<p>We observe two trends in bioinformatics: (i) analyses are increasing
in complexity, often requiring several applications to be run as a
workflow; and (ii) multiple CPU clusters and Grids are available to
more scientists. The traditional solution to the problem of running
workflows across multiple CPUs required programming, often in a
scripting language such as perl. Programming places such solutions
beyond the reach of many bioinformatics consumers.</p>
<h4>Results</h4>
<p>We present Wildfire, a graphical user interface for constructing and
running workflows. Wildfire borrows user interface features from
Jemboss and adds a drag-and-drop interface allowing the user to compose
EMBOSS (and other) programs into workflows. For execution, Wildfire
uses GEL, the underlying workflow execution engine, which can exploit
available parallelism on multiple CPU machines including Beowulf-class
clusters and Grids.</p>
<h4>Conclusion</h4>
<p>Wildfire simplifies the tasks of constructing and executing bioinformatics workflows.</p><a name="IDA1WETS"></a><h3>Background</h3>
<p>Seemingly small steps in usability of bioinformatics applications
have, perhaps, been the most important to the bioinformatics consumer.
Suites such as Accelrys SeqWeb and EMBOSS/Jemboss <a name="IDABXETS"></a><a name="IDAEXETS"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>,<a href="#B2">2</a>],
through consistent user interface elements, have narrowed the usability
gap and made individual applications accessible to the non-specialist
bioinformatician.</p>
<p>Bioinformatics analysis is becoming more complex, often requiring
several applications to be run in combination in a workflow.
Beowulf-class clusters have made supercomputing affordable, allowing us
to execute workflows faster, including some which would previously have
been infeasible. Traditionally, building such workflows required
programming expertise, often in scripting languages such as perl. The
usability gap in bioinformatics has now moved from individual
applications to both construction and execution of workflows.</p>
<p>Existing efforts in narrowing this gap include Jemboss <a name="IDAMXETS"></a>[<a href="#B2">2</a>], Taverna/Freefluo <a name="IDARXETS"></a>[<a href="#B3">3</a>], ICENI <a name="IDAWXETS"></a>[<a href="#B4">4</a>] and Biopipe <a name="IDA1XETS"></a>[<a href="#B5">5</a>].
Jemboss, though it does not support workflows directly, addresses
usability of bioinformatics applications by providing a graphical user
interface to EMBOSS. The user interface replaces the command-line
options of the EMBOSS applications with interface elements such as
check boxes, drop-down lists and text boxes. This simplifies the
applications for users unfamiliar with command-line interfaces; even
for command-line enthusiasts, it simplifies learning of new
applications, which might only be used occasionally, since the
interface is consistent across applications. Jemboss can run the EMBOSS
application on the same machine ("Stand-alone mode") or remotely using
a SOAP protocol.</p>
<p>Taverna, by default, constructs workflows which use Web-Services for
components. The interface requires the user to connect together output
and input ports of components to build a workflow. Taverna relies on
Soaplab <a name="IDAFIH0Z"></a>[<a href="#B6">6</a>] to convert the
EMBOSS command-line applications into Web-Services. However, Soaplab
appears to have lost the help text annotations of the different input
fields which is characteristic of other EMBOSS interfaces. ICENI is
also a service-oriented workflow framework and has a Netbeans-based
user interface.</p>
<p>Biopipe is a workflow framework which also allows for execution of
workflows across clusters. However, Biopipe only allows for pipelines,
not more general workflows with iterative loops; in particular, the
particle swarm optimisation example presented later cannot be
implemented in Biopipe. Also, Biopipe currently does not have a
user-friendly interface for building pipelines.</p>
<p>In the next section, we introduce Wildfire which provides an
integrated environment for construction and execution of workflows. It
provides an intuitive interface based on a drawing analogy and, like
Jemboss, presents program options using graphical user interface
elements; thus Wildfire hides the precise syntax of scripting languages
and command-line options from the user. Jemboss can run several
independent processes in the background, but it has no dependency
handling facility, whereas Wildfire allows the user to compose
applications into a workflow. We illustrate by presenting some examples
in the Results section. In contrast to Taverna and ICENI, Wildfire
works directly with program executables, rather than Web- or
Grid-Services. For execution, it uses GEL (Grid Execution Language <a name="IDANIH0Z"></a>[<a href="#B7">7</a>])
which can run the workflow over the compute nodes of a cluster, similar
to Biopipe. However, GEL can also run executables directly, or on the
Grid. Thus, Wildfire and GEL bring supercomputing power to the
bioinformatician.</p><a name="IDARIH0Z"></a><h3>Implementation</h3>
<p>Wildfire allows the user to visually construct workflows. For
execution, Wildfire exports the workflow as a GEL script, and then
calls a GEL interpretor to execute it. The GEL interpretor can either
run on the same machine as Wildfire, or on a remote compute server.
Figure <a name="IDAWIH0Z"></a><a onclick="popup('/1471-2105/6/69/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F1">1</a> summarises the interaction between Wildfire and GEL.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/6/69/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F1"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/1471-2105-6-69-1.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 1.</strong></a> <strong>Relationship between Wildfire and GEL</strong>.
Relationship between Wildfire and GEL. Wildfire is an interactive
application which allows users to construct workflows using a drawing
analogy. Wildfire executes the workflow by exporting it as a GEL script
which is executed using a suitable GEL interpretor. There are GEL
interpretors for execution on (i) the Grid, using Condor, (ii) a
cluster, using LSF, PBS or SGE, and (iii) the same machine, which could
be a laptop, desktop or multi-processor server.</p>
</div></div><p>Wildfire is implemented in Java, and has been tested on
Windows and Linux platforms. On a Linux platform, the user can run
workflows directly on the same machine: ideal for developing and
testing small examples on a laptop, while reserving the multi-processor
servers and clusters for running the workflow on real data.</p>
<p>We next describe the two main activities enabled by Wildfire: <em>construction </em>and <em>execution </em>of workflows.</p>
<h4>Workflow construction</h4>
<p>When constructing workflows, the user does not need to work directly
with the syntax of scripting languages such as GEL or perl. Rather, the
user is presented with a graphical workflow canvas. On the canvas, a
workflow component can be (i) an <em>atomic component</em>, (ii) a
subworkflow or (iii) a loop (both parallel and sequential). An atomic
component approximately corresponds to an EMBOSS application; in
particular, each atomic component has an ACD (Ajax Command Definition <a name="IDAVJH0Z"></a>[<a href="#B8">8</a>])
description of its parameters and options. The user can select the
atomic components from a customisable list of templates, which by
default includes all the EMBOSS 2.8.0 applications (see Availability
and requirements section). Components are visually rendered on the
canvas as yellow rectangles labelled with the component name (e.g.
EMBOSS program name), and a unique numerical identifier which can be
used distinguish instances from the same component template. Sequential
dependencies between components are created by drawing an arrow between
them. By default, components not linked by arrows are assumed to be
independent (and so can be executed in parallel).</p>
<p>Double clicking on an atomic component in the workflow will bring up a properties window resembling that of Jemboss (see Fig. <a name="IDA1JH0Z"></a><a onclick="popup('/1471-2105/6/69/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F2">2</a>). Wildfire uses Jemboss code to parse the ACD <a name="IDAAKH0Z"></a>[<a href="#B8">8</a>]
description of the application to construct the form and provides
default values where defined. These forms simplify configuration
options by replacing the command-line flags and switches with graphical
user interface elements such as drop-down menus. Help text annotations
for the input fields save the user the effort of looking up UNIX man-
or EMBOSS tfm-pages.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/6/69/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F2"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/1471-2105-6-69-2.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 2.</strong></a> <strong>Elements of the Wildfire interface</strong>.
Elements of the Wildfire interface. The main window in the background
shows the workflow canvas. Its left panel lists the atomic workflow
components; this list is preconfigured with all EMBOSS applications and
can be customised. The foreground window shows the properties form for
the blastall program.</p>
</div></div><p>Wildfire extends the Jemboss interface by allowing the
user to use expressions (similar to spreadsheet formulae) in the text
fields. For example, in Fig. <a name="IDASKH0Z"></a><a onclick="popup('/1471-2105/6/69/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F2">2</a>,
the query file for blastall is = $flie. The first letter is an equals
symbol (=) and indicates that this is not a literal string, but an
expression. The remainder is the expression meaning "the value of
variable $flie". Here the value of $flie is determined by the pforeach
container, as shown in the background window, and denotes a parallel
composition of blastall instances with $file set to the different files
matching *_dice*.fna. The output file is</p>
<p>= $file . ".out"</p>
<p>which is an expression meaning "the value of variable $flie with .out appended". Another example of an expression is</p>
<p>= $f % ".fasta"</p>
<p>which means "the value of variable $f without the .fasta extension". The % and . operators can be mixed, for example</p>
<p>= $f % ".fasta" . ".pep"</p>
<p>which replaces the .fasta extension with .pep.</p>
<p>In addition, the user can add his/her own command-line programs to
the list of atomic components by providing a description of its
command-line options using an extended ACD syntax. The Wildfire user
interface has a facility to help the user write ACD files for new
atomic components. The interface shields the user from the complex ACD
syntax.</p>
<p>Other than defining the dependencies between components and the
invocation arguments, the user can place input files required by the
workflow in subdirectories within the workflow directory. Wildfire can
instruct GEL to copy files from these subdirectories into the working
directory before a component is executed. Any instance can specify
input files, thus allowing for files to be staged-in in a just-in-time
manner. However, a common workflow pattern is one which specifies all
input files to copied only by the first instance in the workflow.</p>
<h4>Workflow execution</h4>
<p>For execution, Wildfire exports a programmatic description of the workflow, in a scripting language called GEL <a name="IDAELH0Z"></a>[<a href="#B7">7</a>],
which is passed to a GEL interpretor for execution. GEL is a scripting
language with parallel constructs characterising common parallel
workflow execution patterns. It is designed to be a generic parallel
scripting language which can be executed on different types of
homogeneous and heterogeneous parallel hardware such as shared-memory
SMP servers, clusters with a shared disk image, and Grids without a
shared disk image. There currently exist interpretors that can run GEL
scripts on SMP servers, clusters with Platform LSF, PBS or Sun
GridEngine (SGE), and on Condor Grids <a name="IDAJLH0Z"></a><a name="IDAMLH0Z"></a>[<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a>,<a href="#B10">10</a>]. GEL is similar to APST <a name="IDARLH0Z"></a>[<a href="#B11">11</a>], NIMROD <a name="IDAWLH0Z"></a>[<a href="#B12">12</a>] and DAGMan (part of Condor) but also allows for cyclic dependencies between jobs. The reader is referred to <a name="IDA1LH0Z"></a>[<a href="#B7">7</a>] for a more thorough description of GEL.</p>
<p>When developing small workflows, the user can run the workflow on
the same machine (see Availability and requirements section). In this
way, Wildfire can be used as a stand-alone application without access
to the network.</p>
<p>Alternatively, the user can choose to send the workflow to a remote
server and run it there. In this case, Wildfire uses the secure shell
(SSH) protocol to send the necessary files over to, and then run the
GEL interpretor on the remote server (see Fig. <a name="IDACMH0Z"></a><a onclick="popup('/1471-2105/6/69/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F3">3</a>).
The GEL interpretor can execute the atomic components directly if the
server has multiple processors. If the server is a cluster, then GEL
can submit the atomic components as jobs to the queue manager. In
either case, the GEL interpretor will try to use multiple processors
where possible. Remote server execution is useful for workflows with
large data sets since GEL will make use of multiple processors. It is
also useful if the atomic components are not installed on the local
machine.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/6/69/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F3"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/1471-2105-6-69-3.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 3.</strong></a> <strong>Remote execution of workflows</strong>.
Remote execution of workflows. In the case of large workflows or when
the applications are not available on the client machine, it is
possible to execute the workflow remotely. In this case, Wildfire uses
the secure shell (SSH) protocol to send the files to the remote machine
and start execution. The remote machine can be a cluster or
multiprocessor server.</p>
</div></div><p>Wildfire and GEL do not require super-user privileges to
install: they can be installed in the "home" directory. For the
client-server mode of operation, only an SSH service on the server is
required; there is no need to configure other services such as SOAP
over HTTP or Web-/Grid-Services, and the firewall is only required to
allow incoming SSH connections. Most modern UNIX-style configurations
already provide an SSH service.</p>
<p>Wildfire can also use GEL to break up the workflow and run parts of
it concurrently on different supercomputers using Condor. (Note: GEL
1.0 uses the Globus <a name="IDAWMH0Z"></a>[<a href="#B13">13</a>]
protocols to provide Grid support. GEL 2.0 uses Condor for Grid
execution and future support for Globus Grids will be via Condor-G.)
This is useful for very large workflows which require as many compute
resources as possible. In practice, it is more useful when not all
components are available on any one machine, for example, because of
licence availability.</p>
<p>Wildfire monitors execution of the individual atomic components and
feedback is provided via annotations on the canvas which are updated in
real-time.</p>
<p>The exported GEL script can also be run directly using an
interpretor via the command line. This allows a workflow to be run in
batch mode independently of Wildfire, and is useful for very
long-running workflows or those that have to be run repeatedly.</p><a name="IDA2MH0Z"></a><h3>Results</h3>
<p>We describe three applications of Wildfire. In the first
application, we construct a workflow for analysis of human
tissue-specific transcripts by comparing them against the known exons.
This example shows how Wildfire can make use of the parallel
capabilities of supercomputers. The second example considers a particle
swarm optimisation algorithm implemented as a workflow, and shows that
Wildfire can express workflows requiring iteration. The last example
cross validates an allergenicity prediction algorithm. The number of
parallel processes in this workflow can only be determined at run-time.</p>
<h4>Tissue-specific Gene Expression Analysis</h4>
<p>To study tissue-specific gene expression in humans, we compare the
known exons against a database of 16,385 transcripts obtained from the
Mammalian Gene Collection. Since the human genome contains many exons,
the extraction process is time consuming, but it is easily
parallelised. The standard organisation of the 24 chromosomes into
separate files provides a natural partitioning of the exon extraction
problem: we extract the exons from each chromosome in parallel. To
further increase the granularity of the problem and so exploit more
parallelism, we break up each of the 24 files of exons into five
smaller files, resulting in a total of 120 files. We blast each of
these smaller files against the database of transcripts.</p>
<p>The workflow as constructed in Wildfire is shown in Fig. <a name="IDAGNH0Z"></a><a onclick="popup('/1471-2105/6/69/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F4">4</a>, and its implementation without Wildfire is described in detail in <a name="IDALNH0Z"></a>[<a href="#B14">14</a>].
The atomic component exonx is a program developed in-house to extract
and store exons from a genbank file in fasta format; dice is a perl
script used to break up a fasta file into smaller pieces. The noop.sh
component is required to instruct GEL to copy the input files into the
working directory. (The initial copying of input files will likely be
an implicit feature of workflows in future versions of Wildfire, and so
the explicit noop. sh component will no longer be necessary.) The
remaining components (GNU gunzip, NCBI BLAST formatdb and blastall) are
standard applications which we have incorporated as atomic components
using our ACD editor.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/6/69/figure/F4','F4',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F4"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/1471-2105-6-69-4.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 4.</strong></a> <strong>Tissue-specific gene expression analysis</strong>.
Tissue-specific gene expression analysis. We initially start with 24
compressed genbank files, one for each human chromosome. We decompress
(gunzip) the genbank file, extract (exonx) from it all the exons into
one fasta file and then break up (dice) this fasta file into 5 smaller
fasta files; we do this for each chromosome in parallel. At the same
time, we decompress the file of transcripts and use formatdb to format
it for use as a BLAST database. Finally, we blast the exons against the
transcript database in parallel. Note that components are rendered as
rectangles and the bottom half shows a unique numerical identifier;
this can be used to distinguish components derived from the same
template, e.g. gunzip:2 and gunzip:6.</p>
</div></div><p>The whole workflow takes less than 6000 seconds to run
on a 128 CPU Pentium III cluster, whereas a sequential version of the
same workflow required almost nine times longer. Profiling of the
workflow shows that breaking up the 24 files of exons more evenly would
significantly improve performance. Since all programs are run via the
scheduler on the cluster, the workflow follows whatever scheduling
policies are configured at the component-level. Hence, the workflow in
its current form is already efficient from the point of view of
resource use.</p>
<h4>Swarm optimisation</h4>
<p>Real-life optimisation problems are often intractable and heuristics
are the only choice for finding near optimal solutions. Particle Swarm
Optimisation <a name="IDACOH0Z"></a>[<a href="#B15">15</a>] is such a
heuristic based on simulation of information exchange between leaders
and followers observed in, for example, bird flocking.</p>
<p>The algorithm simulates individuals flying through the search space.
On each iteration, the individuals are separated into a set of leaders
and a set of followers, based on their fitness. The followers use the
locations of the leaders to change their flying direction, i.e. search
velocity. The location of each individual is computed based on its
current location and flying direction. The new location is used to rank
the fitness of individuals and subsequently the leader and follower
sets.</p>
<p>Note that efficient Swarm Optimisation implementations exhibit both (1) <em>iteration </em>and (2) <em>parallelism: </em>successive
generations must be simulated until a termination condition is met, and
simulation of each generation entails simulation of many independent
individuals. Therefore, a workflow tool suitable for implementing such
algorithms must support iteration and parallelism.</p>
<p>The workflow in Fig. <a name="IDAPOH0Z"></a><a onclick="popup('/1471-2105/6/69/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F5">5</a> is a simplified implementation of a swarm algorithm by Ray et al. <a name="IDAUOH0Z"></a>[<a href="#B16">16</a>]
implemented as a workflow. The algorithm is applied to a parameter
estimation problem for a biochemical pathway model consisting of 36
unknowns and eight ordinary differential equations <a name="IDAZOH0Z"></a>[<a href="#B17">17</a>].
Components init1, eval1 and init2 are used to initialise and rank the
individuals. Component test determines whether the workflow should
terminate and extract collects together the results on termination of
the simulation. Component eva12 is used to evaluate the fitness of an
individual; note the outer parallel loop evaluates the fitness of each
follower. The remaining components are used to select the leaders and
followers.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/6/69/figure/F5','F5',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F5"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/1471-2105-6-69-5.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 5.</strong></a> <strong>Swarm optimisation example</strong>.
Swarm optimisation example. Component eval1 is executed four times in
parallel within a parallel for loop. The circle denotes a while loop,
with test as loop guard: if test returns false, then we follow the
bottom branch to extract, otherwise we follow the right branch and test
test again after reassign. Component eval2 is executed in parallel once
for each file matching follower_sol*.</p>
</div></div><p>Component test depends on both init2 and reassign; on
the first iteration, test can start executing only after init2 has
terminated, and on subsequent iterations, only after reassign has
terminated. Since reassign itself hereditarily depends on test (i.e.
reassign depends on eval2 which depends on classify which itself
depends on test) we see there is a cyclic dependency. The while loop in
GEL allows such dependencies and so is crucial for this workflow.</p>
<h4>Allergenicity prediction</h4>
<p>Allergenicity prediction is the process of determining whether a new
protein sequence is an allergen or not. Proteins known to induce
allergic responses have been documented in allergen databases. One
approach to allergenicity prediction is to determine, automatically,
motifs from sequences in such a database, and then search for these
motifs in the query sequences.</p>
<p>The objective of the workflow in Fig. <a name="IDAQPH0Z"></a><a onclick="popup('/1471-2105/6/69/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F6">6</a> is to test the accuracy of the approach described above, where protein sequence motifs are identified using an algorithm <a name="IDAVPH0Z"></a>[<a href="#B18">18</a>] based on wavelet analysis. From a group of 817 sequences, known to be allergens, we take a <em>learning </em>set
consisting of a randomly selected subset covering 90% to be used for
identification of motifs. The remaining 10% are used as query sequences
for allegenicity prediction. We use the predictions to assess the
accuracy of this approach.</p><div class="figs"><div class="fig"><p></p><p><a onclick="popup('/1471-2105/6/69/figure/F6','F6',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/6/69/figure/F6"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/1471-2105-6-69-6.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 6.</strong></a> <strong>An allergenicity prediction workflow</strong>.
The allergenicity prediction workflow from [23] constructed in
Wildfire. The components in this workflow are all custom applications,
or custom scripts calling standard applications. Components format,
group, join_scr and process_scr are administrative programs which
translate and convert files from one format to another. Component
rscript uses the R-project to cluster the amino acid sequences from a
database of known allergens. For each cluster, we align its sequences
and use a wavelet algorithm to predict motifs. The resulting motifs are
used to construct HMM profiles using hmmbuild. Finally, we use these
profiles with hmmpfam to predict allergenic sequences. Components
join_scr and process_scr collate and summarise the results.</p>
</div></div><p>Initially, we use ClustalW to generate the pairwise
global alignment distances between the protein sequences in the
learning set. We then use these distances to cluster the protein
sequences by partitioning around medoids using the R project <a name="IDANYH0Z"></a>[<a href="#B19">19</a>].
We use ClustalW again to align each cluster of protein sequences and we
use the wavelet analysis algorithm on each aligned cluster to identify
motifs in the protein sequences. For each identified motif, we build an
HMM profile <a name="IDASYH0Z"></a><a name="IDAVYH0Z"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a href="#B21">21</a>]
in parallel. Note that the number of motifs is not know a priori; it
can only be known at run-time. The HMM profiles are then used to search
for the motifs in each query sequence, and thus predict whether it is
an allergen or not. The accuracy of the predictions is computed to
assess the effectiveness of this approach.</p><a name="IDAZYH0Z"></a><h3>Discussion</h3>
<p>Bioinformaticians can use Wildfire as an integrated environment to <em>construct </em>and to <em>execute </em>workflows.
The graphical user interface elements ease workflow construction by
hiding the syntax of scripting languages. The constructed workflows can
be executed across multiple processors (i) in the same server, (ii) in
a cluster, or (iii) across several supercomputers across the Grid.</p>
<p>Wildfire is preconfigured to allow applications from EMBOSS to be
used as workflow components. The user can add his own applications to
be used as components by creating the necessary ACD files. Wildfire
also hides this syntax by providing a wizard for creating and editing
ACD files.</p>
<p>In a typical scenario, the user develops a workflow by visually
constructing and executing small examples on his desktop or laptop.
When the workflow is ready, the user can run it on real data on a
shared resource such as a compute cluster running a scheduler. GEL
workflows, such as those constructed in Wildfire, run well on shared
resources since the components of the workflow are run through the
scheduler. This gives the scheduler more opportunities to schedule the
components with respect to whatever policies are configured: for
example, a <em>fair share </em>policy would allow jobs from other users to run even when the workflow would otherwise monopolise the whole cluster.</p>
<p>We have described three applications of Wildfire in three different
fields, and welcome readers to try Wildfire for themselves and solicit
recommendations for improvements.</p><a name="IDAIZH0Z"></a><h3>Availability and requirements</h3>
<h4>Wildfire</h4>
<p>Wildfire is run on the client computer and allows the user to
visually construct workflows. Wildfire invokes GEL (see below) to
execute workflows. EMBOSS 2.8.0 <a name="IDARZH0Z"></a>[<a href="#B22">22</a>] is required (2.9.0 is not yet supported) to run EMBOSS workflows.</p>
<p><strong>Availability:</strong><a href="http://www.biomedcentral.com/1471-2105/6/%C2%A0http://wildfire.bii.a-star.edu.sg">&nbsp;http://wildfire.bii.a-star.edu.sg</a> <a href="http://www.webcitation.org/query.php?url=%C2%A0http://wildfire.bii.a-star.edu.sg&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p>
<p><strong>Operating systems:</strong> Platform Independent (tested on Windows, i386 Linux)</p>
<p><strong>Programming Language:</strong> Java</p>
<p><strong>Other requirements:</strong> Java 1.4.2, GEL</p>
<p><strong>Licence:</strong> GPL</p>
<p>Currently, stand-alone mode is not available for Windows.</p>
<h4>GEL</h4>
<p>GEL is an interpretor which executes GEL scripts generated by
Wildfire. Currently only UNIX platforms are supported. GEL supports
LSF, SGE and PBS clusters, SMP servers and Condor-based Grids.</p>
<p><strong>Availability:</strong>&nbsp;<a href="http://wildfire.bii.a-star.edu.sg/">http://wildfire.bii.a-star.edu.sg</a> <a href="http://www.webcitation.org/query.php?url=http://wildfire.bii.a-star.edu.sg&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p>
<p><strong>Operating systems:</strong> GNU-style UNIX (tested on i386 Linux, ia64 Linux, Spare Solaris, Alpha Tru64)</p>
<p><strong>Programming Language:</strong> Java</p>
<p><strong>Other requirements:</strong> Java 1.4.2, bash</p>
<p><strong>Licence:</strong> free for non commercial use, see <a href="http:///">http://</a> <a href="http://www.webcitation.org/query.php?url=http://&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a><a href="http://wildfire.bii.a-star.edu.sg/">http://wildfire.bii.a-star.edu.sg</a> <a href="http://www.webcitation.org/query.php?url=http://wildfire.bii.a-star.edu.sg&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><a name="IDA10H0Z"></a><h3>Authors' contributions</h3>
<p>FT lead and coordinated the software engineering aspects of the
project, and drafted this manuscript. FT and CCL co-designed Wildfire
and GEL. CCL and HLY programmed, tested and debugged the software. LYP
designed the tissue-specific transcript analysis workflow. PI and AK
designed the allergen prediction workflow. AK participated in the
concept, design and testing of the software and contributed to
successive revisions of this manuscript. All authors read and approved
the manuscript.</p><a name="IDAB1H0Z"></a><h3>Acknowledgements</h3>
<p>The authors would like to thank Tan Chee Meng and Prof. Tapabrata
Ray for their contribution to the Particle Swarm Example, and various
anonymous referees for improvements in this manuscript.</p><a name="refs"></a><h3>References</h3>
<ol id="references">
<li id="B1">
<p><a name="B1"></a> Rice P,  Longden I,  Bleasby A: <strong> EMBOSS: The European Molecular Biology Open Software Suite. </strong></p><p><em>Trends in Genetics</em> 2000,  <strong>16</strong><strong>:</strong>276-277. <a target="_blank" href="http://www.biomedcentral.com/pubmed/10827456">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10827456">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B1" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B1','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B2">
<p><a name="B2"></a> Carver T,  Bleasby A: <strong> The design of Jemboss: a graphical user interface to EMBOSS. </strong></p><p><em>Bioinformatics </em> 2003,  <strong>19</strong>(14)<strong>:</strong>1837-1843. <a target="_blank" href="http://www.biomedcentral.com/pubmed/14512356">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14512356">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B2" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B2','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B3">
<p><a name="B3"></a> Oinn T,  Addis M,  Ferris J,  Marvin D,  Greenwood M,  Carver T,  Pocock MR,  Wipat A,  Li P: <strong> Taverna: A tool for the composition and enactment of bioinformatics workflows. </strong></p><p><em>Bioinformatics</em> 2004,  <strong>20</strong>(7)<strong>:</strong>3045-3054. <a target="_blank" href="http://www.biomedcentral.com/pubmed/15201187">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15201187">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B3" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B3','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B4">
<p><a name="B4"></a> Furmento N,  Lee W,  Mayer A,  Newhouse S,  Darlington J: <strong> ICENI: An Open Grid Service Architecture Implemented with Jini. </strong></p><p><em>SuperComputing 2002</em> 2002. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B4" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B4','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B5">
<p><a name="B5"></a> Hoon S,  Ratnapu KK,  Chia JM,  Kumarasamy B,  Juguang X,  Clamp M,  Stabenau A,  Potter S,  Clarke L,  Stupka E: <strong> Biopipe: a flexible framework for protocol-based bioinformatics analysis. </strong></p><p><em>Genome Res</em> 2003,  <strong>13</strong><strong>:</strong>1904-1915. <a target="_blank" href="http://www.biomedcentral.com/pubmed/12869579">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12869579">Publisher Full Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12869579">PubMed Central Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B5" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B5','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B6">
<p><a name="B6"></a> Senger M,  Rice P,  Oinn T: <strong> Soaplab – a unified Sesame door to analysis tools. </strong>
		[<a target="_blank" href="http://industry.ebi.ac.uk/soaplab/">http://industry.ebi.ac.uk/soaplab/</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://industry.ebi.ac.uk/soaplab/&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p>In <em>Proceedings, UK e-Science, All Hands Meeting</em> Edited by: Cox SJ.  2003, 509-513.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B6" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B6','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B7">
<p><a name="B7"></a> Chua Ching Lian ,  Tang F,  Issac P,  Krishnan A: <strong> GEL: Grid Execution Language. </strong></p><p><em>J Parallel and Distributed Computing</em>, in press.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B7" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B7','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B8">
<p><a name="B8"></a> de Boer T: <strong> AJAX Command Definition (ACD files). </strong>
		[<a target="_blank" href="http://www.rfcgr.mrc.ac.uk/Software/EMBOSS/Acd/">http://www.rfcgr.mrc.ac.uk/Software/EMBOSS/Acd/</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.rfcgr.mrc.ac.uk/Software/EMBOSS/Acd/&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B8" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B8','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B9">
<p><a name="B9"></a> Litzkow MJ,  Livny M,  Mutka MW: <strong> Condor: A Hunter of Idle Workstations. </strong>
		[<a target="_blank" href="http://www.cs.wisc.edu/condor/publications.html">http://www.cs.wisc.edu/condor/publications.html</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.cs.wisc.edu/condor/publications.html&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p><em>Proceedings of 8th International Conference on Distributed Computing Systems</em> 1988, 104-111.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B9" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B9','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B10">
<p><a name="B10"></a>Condor Team: <strong> Condor Home Page. </strong>
		[<a target="_blank" href="http://www.cs.wisc.edu/condor/">http://www.cs.wisc.edu/condor/</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.cs.wisc.edu/condor/&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> 2002. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B10" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B10','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B11">
<p><a name="B11"></a> Berman FD,  Wolski R,  Figueira S,  Schopf J,  Shao G: <strong> Application-Level Scheduling on Distributed Heterogeneous Networks. </strong></p><p><em>Proceedings of Supercomputing 1996</em> 1996. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B11" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B11','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B12">
<p><a name="B12"></a> Abramson D,  Sosic R,  Giddy J,  Hall B: <strong> Nimrod: A Tool for Performing Parameterised Simulations Using Distributed Workstations. </strong></p><p><em>HPDC</em> 1995, 112-121.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B12" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B12','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B13">
<p><a name="B13"></a> Foster I,  Kesselman C: <strong> Globus: A Metacomputing Infrastructure Toolkit. </strong></p><p><em>The International Journal of Supercomputer Applications and High Performance Computing</em> 1997,  <strong>11</strong>(2)<strong>:</strong>115-128.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B13" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B13','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B14">
<p><a name="B14"></a> Chua CL,  Tang F,  Lim YP,  Ho LY,  Krishnan A: <strong> Implementing a Bioinformatics Workflow in a Parallel and Distributed Environment. </strong></p><p><em>Parallel and Distributed Computing: Applications and Technologies, of LNCS, Springer</em> 2005,  <strong>3320</strong><strong>:</strong>1-4.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B14" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B14','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B15">
<p><a name="B15"></a> Eberhart RC,  Kennedy J: <strong> A New Optmizer using Particle Swarm Theory. </strong></p><p><em>Sixth International Symposium on Micro Machine and Human Science, IEEE Service Center</em> 1995, 39-43.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B15" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B15','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B16">
<p><a name="B16"></a> Ray T,  Tai K,  Seow K: <strong> An Evolutionary Algorithm for Constrained Optimization. </strong></p><p><em>Proceedings of the Genetic and Evolutionary Computation Conference, Morgan Kaufmann</em> 2000, 771-777.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B16" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B16','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B17">
<p><a name="B17"></a> Moles CG,  Mendes P,  Banga JR: <strong> Parameter Estimation in Biochemical Pathways: A Comparison of Global Optimization Methods. </strong></p><p><em>Genome Research</em> 2003.,  <strong>13</strong><strong>:</strong> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B17" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B17','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B18">
<p><a name="B18"></a> Krishnan A,  Li KB,  Issac P: <strong> Rapid detection of conserved regions in protein sequences using wavelets. </strong></p><p><em>In Silico Biology</em> 2004,  <strong>4</strong>(2)<strong>:</strong>133-48. <a target="_blank" href="http://www.biomedcentral.com/pubmed/15107019">PubMed Abstract</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B18" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B18','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B19">
<p><a name="B19"></a><strong> R Language Definition </strong>
		[<a target="_blank" href="http://www.r-project.org/">http://www.r-project.org</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://www.r-project.org&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B19" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B19','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B20">
<p><a name="B20"></a> Durbin R,  Eddy S,  Krogh A,  Mitchison G: </p><p><em>Biological sequence analysis. CUP</em>. 1998. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B20" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B20','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B21">
<p><a name="B21"></a> Eddy S: <strong> HMMER User's Guide. </strong>
		[<a target="_blank" href="http://hmmer.wustl.edu/">http://hmmer.wustl.edu</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://hmmer.wustl.edu&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B21" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B21','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B22">
<p><a name="B22"></a><strong> EMBOSS </strong>
		[<a target="_blank" href="http://emboss.sourceforge.net/">http://emboss.sourceforge.net</a>]
			
		 <a href="http://www.webcitation.org/query.php?url=http://emboss.sourceforge.net&amp;refdoi=10.1186/1471-2105-6-69" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B22" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B22','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
<li id="B23">
<p><a name="B23"></a> Li KB,  Issac P,  Krishnan A: <strong> Predicting allergenic proteins using wavelet transform. </strong></p><p><em>Bioinformatics </em> 2004,  <strong>20</strong><strong>:</strong>2572-8. <a target="_blank" href="http://www.biomedcentral.com/pubmed/15117757">PubMed Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15117757">Publisher Full Text</a> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B23" onclick="popup('/sfx_links.asp?ui=1471-2105-6-69&amp;bibl=B23','SFXMenu','460','420'); return false;"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/sfx_links.gif" alt="OpenURL"></a></p><p class="totext"><script type="text/javascript">totext()</script><a href="javascript:turn();">Return to text</a></p>

</li>
</ol>
<h4 class="bottommessage"><a href="http://www.biomedcentral.com/1471-2105/6/69/postcomment">Have something to say? Post a comment on this article!</a></h4><script type="text/javascript">initializeOutline();</script></td></tr></tbody></table></div>

<center>

<!------ OAS AD 'Bottom1' begin ------>
<script language="JavaScript">
<!--
OAS_AD('Bottom1');
//-->
</script><a href="http://oas.biomedcentral.com/5c/www.biomedcentral.com/article/bmcbioinformatics/2054976570/Bottom1/default/empty.gif/34386665393030393439386238646230" target="_blank"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/empty.gif" alt="" border="0" height="2" width="2"></a><noscript>
<a
href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1691841604@Top,Bottom,Left1,Bottom1!Bottom1"
target=_top>
<img
src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1691841604@Top,Bottom,Left1,Bottom1!Bottom1"
border=0></a>
</noscript>
<!------ OAS AD end ------>
</center>

<hr color="#cccccc" noshade="noshade" size="1">
<table cellpadding="3" width="100%"><tbody><tr valign="middle"><td align="center">


<center>

<!------ OAS AD 'Bottom' begin ------>
<script language="JavaScript">
<!--
OAS_AD('Bottom');
//-->
</script><a href="http://oas.biomedcentral.com/5c/www.biomedcentral.com/article/bmcbioinformatics/718777788/Bottom/biomed/biomarkworldc-contra-ban/BMC_biomarker_468x60.JPG/34386665393030393439386238646230" target="_blank"><img src="Tang%202005%20Wildfire%20-%20distributed,%20Grid-enabled%20workflow%20construction%20and%20execution_files/BMC_biomarker_468x60.JPG" alt="" border="0"></a><noscript><a
href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1691841604@Top,Bottom,Left1,Bottom1!Bottom"
target=_top>
<img
src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/www.biomedcentral.com/article/bmcbioinformatics/1691841604@Top,Bottom,Left1,Bottom1!Bottom"
border=0></a>
</noscript>
<!------ OAS AD end ------>
</center>

</td></tr>
<tr><td align="center"><div class="smalltext">
<a href="http://www.biomedcentral.com/info/about/tandc">Terms and Conditions</a> 
<a href="http://www.biomedcentral.com/info/about/privacy">Privacy statement</a> 
<a href="http://www.biomedcentral.com/info/mediainformation/">Information for advertisers</a> 
<a href="http://www.biomedcentral.com/info/about/bmcjobs">Jobs at BMC</a> 
<a href="mailto:info@biomedcentral.com">Contact us</a> 
</div>
</td></tr></tbody></table>
<table class="footer2t" border="0" cellpadding="0" cellspacing="0" width="100%">
<tbody><tr>
<td colspan="3" align="center" bgcolor="#003399">
© 1999-2009 BioMed Central Ltd unless otherwise stated.
 Part of <a href="http://www.springer.com/">Springer Science+Business Media</a>.
</td>
</tr>
</tbody></table>
</div>
</div>
</center>
</body></html>