<html><head>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">







 
 
  







<title>
	
	Next-Generation Sequencing: From Basic Research to Diagnostics -- Voelkerding et al. 55 (4): 641 -- Clinical Chemistry</title>


	<script language="JavaScript">
	<!--
	function startTarget(windowname,wid,hei) {
	
	var dotpos = windowname.indexOf(".");
	if (dotpos > -1)
	{
	var tempwn = windowname.substring(0,dotpos) +
		windowname.substring(dotpos + 1, windowname.length);
	windowname = tempwn;
	}
	var sizestring = 'width=' + wid + ',height=' + hei;
	window.open('',windowname,'scrollbars,resizable,' + sizestring + '\'');
	}
	// -->
	</script>




	<!-- has inhead tag -->
	
		 
	 
	<script language="JavaScript">
	<!--
	if ( top != self )
	{
		top.location.href = unescape(window.location.pathname);
	}
	//-->
	</script>
	
		
	
		<style type="text/css"><!--
table.content_box_outer_table {
	padding-left:10px;
	padding-bottom:5px;
	padding-top:0px;
	padding-right:0px;
	}

table.content_box_outer_table_in_sidebar_frame {
	padding-left:10px;
	padding-bottom:0px;
	padding-top:10px;
	padding-right:10px;
	}

table.content_box_inner_table {
	background-color:#f0f0f0;
	font-family:Verdana,Arial,Helvetica,sans-serif;
	font-size:65%;
	padding-top:0px;
	padding-left:0px;
	padding-right:0px;
	padding-bottom:5px;
	}

table.content_box_pdfinframes {
	padding-top:0px;
	padding-left:0px;
	padding-right:0px;
	padding-bottom:4px;
	}

td.content_box_title_highlight {
	font-family:"Trebuchet MS",Verdana,Arial,Helvetica,sans-serif;
	font-style:italic;
	color:#ffffff;
	background-color:#CC0000;
	font-size:120%;
	font-weight:bold;
	text-align:center;
	padding:1px;
	vertical-align:middle;
	}

td.content_box_title {
	font-family:"Trebuchet MS",Verdana,Arial,Helvetica,sans-serif;
	font-style:italic;
	color:#ffffff;
	background-color:#828282;
	font-size:120%;
	font-weight:bold;
	padding:1px;
	text-align:center;
	vertical-align:middle;
	}

td.content_box_space_between_sections {
	padding-top:2px;
	}

td.content_box_arrow {
	text-align:right;
	padding-left:5px;
	padding-right:0px;
	padding-top:0px;
	padding-bottom:3px;
	}

td.content_box_item {
	text-align:left;
	vertical-align:top;
	padding-left:3px;
	padding-right:5px;
	padding-top:0px;
	padding-bottom:3px;
	}

td.content_box_pdfinframes_citation_cell {
	background-color:#ffffff;
	border:1px #000000 solid;
	padding:3px;
	font-family:Verdana,Arial,Helvetica,sans-serif;
	font-size:65%;
	text-align:left;
	}


font.openaccess {
	font-weight:bold;
	padding-left:3px;
	padding-right:3px;
	background-color:#ffffff;
	}
font.openaccess_open {
	color:#000000;
	}
font.openaccess_access {
	color:#cc0000;
	}
font.openaccess_oa {
	}
.content_box_openaccess_title {
	text-align:center;
	white-space:nowrap;
	background-color:#BABABA;
	}




div.referrer-based-search-line {
	font-family: arial,sans-serif;
	font-size: 0.75em;
	background-color: #009918;
	padding: 3px 25px;
	margin: 0px 4px 5px 0px;
}

div.referrer-based-search-line .search-terms{
	font-weight: bold;
}

div.referrer-based-search-line .collection-name{
	font-style: italic;
}

@media screen,print,handheld
{
    .notonscreen { display: none; }
}
--></style>

	        
<style type="text/css"><!--
	@media screen,print,handheld
{
    .notonscreen { display: none; }
}
--></style>
<!-- ac in the head available -->

 

		
		
	    <script language="javascript">
		<!-- 
	function popwin(url,wid,hei,fromtop,fromleft) {
		var winPref = "channelmode=no,toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=auto,resizable=no,width=" + wid + ",height=" + hei + ",top=" + fromtop + ",left=" + fromleft;
		window.open(url, "popwin", winPref);
	}
	// -->
	</script>
        <script type="text/javascript" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/xmlhttprequest.js"></script>
<script type="text/javascript" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/utility.js"></script>
<script type="text/javascript" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/callback.js"></script>

	
		
	

<meta name="robots" content="nofollow">



<meta name="citation_journal_title" content="Clinical Chemistry">
<meta name="citation_publisher" content="Am Assoc Clin Chem">

<meta name="citation_authors" content="Voelkerding, Karl V.; Dames, Shale A.; Durtschi, Jacob D.">
<meta name="citation_title" content="Next-Generation Sequencing: From Basic Research to Diagnostics">
<meta name="citation_date" content="04/01/2009">
<meta name="citation_volume" content="55">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="641">
<meta name="citation_id" content="55/4/641">
<meta name="citation_mjid" content="clinchem;55/4/641">
<meta name="citation_doi" content="10.1373/clinchem.2008.112789">
<meta name="citation_abstract_html_url" content="http://www.clinchem.org/cgi/content/abstract/55/4/641">
<meta name="citation_fulltext_html_url" content="http://www.clinchem.org/cgi/content/full/55/4/641">
<meta name="citation_pdf_url" content="http://www.clinchem.org/cgi/reprint/55/4/641.pdf">
<meta name="citation_pmid" content="19246620">

<meta name="dc.Contributor" content="Voelkerding, Karl V.">
<meta name="dc.Contributor" content="Dames, Shale A.">
<meta name="dc.Contributor" content="Durtschi, Jacob D.">
<meta name="dc.Title" content="Next-Generation Sequencing: From Basic Research to Diagnostics">
<meta name="dc.Identifier" content="10.1373/clinchem.2008.112789">
<meta name="dc.Date" content="04/01/2009">

<meta name="ROBOTS" content="NOARCHIVE"> 

</head><div FirebugVersion="1.3.3" style="display: none;" id="_firebugConsole"></div><body bgcolor="#ffffff">
	



<a name="top"><!-- null --></a>


	
	<table border="0" cellpadding="0" cellspacing="0">
	<tbody><tr>
		<td nowrap="nowrap" valign="bottom">
			<a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/"><img alt="Clinical Chemistry" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/title.gif" border="0" height="34" width="220"></a>
			<a href="http://www.clinchem.org/subscriptions/etoc.shtml"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/etoc-ad1.gif" alt="Email Content Delivery" border="1" height="40" width="220"></a>
			
	<table cellpadding="2" cellspacing="2" width="450">
	<tbody><tr>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/"><font color="#ffffff" face="arial,helvetica" size="-2">HOME</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/help/"><font color="#ffffff" face="arial,helvetica" size="-2">HELP</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/cgi/feedback"><font color="#ffffff" face="arial,helvetica" size="-2">FEEDBACK</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/subscriptions/"><font color="#ffffff" face="arial,helvetica" size="-2">SUBSCRIPTIONS</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/contents-by-date.0.shtml"><font color="#ffffff" face="arial,helvetica" size="-2">ARCHIVE</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/search.dtl"><font color="#ffffff" face="arial,helvetica" size="-2">SEARCH</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/content/vol55/issue4/"><font color="#ffffff" face="arial,helvetica" size="-2">TABLE OF CONTENTS</font></a></td>
	</tr>
	</tbody></table>
	</td>
		<td><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="5" width="5"></td>
		<td valign="bottom">
	
	<table bgcolor="#000000" border="0" cellpadding="0" cellspacing="1"><tbody><tr>
	<form method="get" name="search" action="/cgi/search"></form><input name="andorexactfulltext" value="and" type="hidden"><input name="resourcetype" value="1" type="hidden"><input name="disp_type" value="" type="hidden"><input name="sortspec" value="relevance" type="hidden"><td align="center" valign="middle"> 
		<table bgcolor="#dbdbdb" border="0" cellpadding="1" cellspacing="0"> 
		<tbody><tr>
			<td bgcolor="#cc0000"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="2" width="2"></td>
			<td bgcolor="#cc0000" nowrap="nowrap" valign="middle"><strong><font color="#ffffff" face="verdana,arial,helvetica,sans-serif" size="-2"><em>QUICK</em> SEARCH:&nbsp;</font></strong></td>
			<td align="right" bgcolor="#cc0000" nowrap="nowrap" valign="middle"><a style="text-decoration: none; color: rgb(255, 255, 255);" href="http://www.clinchem.org/search.dtl"><font color="#ffffff" face="verdana,arial,helvetica" size="-2">&nbsp;[advanced]</font></a></td>
			<td bgcolor="#cc0000"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="2" width="2"></td>
		</tr><tr>
			<td bgcolor="#dbdbdb"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="2" width="2"></td>
			<td colspan="2" bgcolor="#dbdbdb">
			<table border="0" cellpadding="0" cellspacing="0"> 
			<tbody><tr>
				<td align="center" valign="bottom"><input style="background: rgb(255, 255, 255) none repeat scroll 0% 0%; text-decoration: none; font-family: verdana,arial,helvetica; font-size: 10px; font-weight: bold; color: rgb(0, 0, 0); -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;" value="Go" type="submit"></td>
				<th colspan="2" align="left"><font color="#000000" face="verdana,arial,helvetica,sans-serif" size="-2"><label for="qsintdefsauthor1">Author:</label><br></font>
					<input name="author1" id="qsintdefsauthor1" size="9" maxlength="80" value="" style="font-family: verdana,arial,helvetica; font-size: 10px;" type="text"></th>
				<th colspan="3" align="left"><font color="#000000" face="verdana,arial,helvetica,sans-serif" size="-2"><label for="qsintdefsfulltext">Keyword(s):</label><br></font>
					<input name="fulltext" id="qsintdefsfulltext" size="16" maxlength="80" value="" style="font-family: verdana,arial,helvetica; font-size: 10px;" type="text"></th>
			</tr><tr>
				<th><font color="#000000" face="verdana,arial,helvetica,sans-serif" size="-2"><label for="qsintdefspubdate_year">Year:</label>&nbsp;</font></th>
				<td><input name="pubdate_year" id="qsintdefspubdate_year" size="4" maxlength="4" value="" style="font-family: verdana,arial,helvetica; font-size: 10px;" type="text"></td>
				<th align="right"><font color="#000000" face="verdana,arial,helvetica,sans-serif" size="-2"><label for="qsintdefsvolume">Vol:</label>&nbsp;</font></th>
				<td><input name="volume" id="qsintdefsvolume" size="4" maxlength="10" value="" style="font-family: verdana,arial,helvetica; font-size: 10px;" type="text"></td>
				<th><font color="#000000" face="verdana,arial,helvetica,sans-serif" size="-2"><label for="qsintdefsfirstpage">Page:</label>&nbsp;</font></th>
				<td><input name="firstpage" id="qsintdefsfirstpage" size="4" maxlength="10" value="" style="font-family: verdana,arial,helvetica; font-size: 10px;" type="text"></td>
			</tr> 
			</tbody></table> </td>
			<td bgcolor="#dbdbdb"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="2" width="2"></td>
		</tr>
		</tbody></table></td></tr></tbody></table> 
	 
	</td>
	</tr><tr>
		<td colspan="3"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="2" width="2"><br><font face="verdana,arial,helvetica" size="-2"><!-- default (new sl) generic inst auth  --><strong>Institution: UNIV OF MIAMI</strong> | <a target="_top" href="http://www.clinchem.org/cgi/login?uri=%2Fcgi%2Fcontent%2Ffull%2F55%2F4%2F641">Sign In via User Name/Password</a></font><!-- debug:  --><hr align="left" color="#000000" noshade="noshade" size="1"></td>
	</tr>
	</tbody></table>











 
	














	






	
		

<b>
  
	
	





        
                
                        <font color="#cc0000" size="-1"><a target="_top" href="http://www.clinchem.org/"><i>Clinical Chemistry</i></a> 55: 641-658, 2009.
			
                        
                        
                
                        
                         First published February 26, 2009; 10.1373/clinchem.2008.112789
                        
                
<br></font>
        

</b>

	 
 






 



 
	 
		  
			
	
		<table class="content_box_outer_table" align="right">
		
		
			<tbody><tr>
			<td>
		
	

<!-- beginning of inner table -->
	<table class="content_box_inner_table">
<!-- citation -->
	
		




	

	

	
		
			
<tbody><tr><td class="content_box_title_highlight" colspan="2">This Article</td></tr>

	

	<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr>
				
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				
				<strong><a href="http://www.clinchem.org/cgi/content/abstract/55/4/641">
				Abstract</a></strong>
				 					<img alt="Freely available" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/free.gif" border="0" height="7" width="32">
				
	
				</td></tr>
			
		
		
	

	
	
	

	
	
	


	
	
	

	
	
	

	
	
	
	
	
	
	
	
	

	
		
		
	
	
		
	

	

	
		
			
		
	

	

	
	 

	
	 

		
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
		<strong>
		
		
		
			
				
				
					<a href="http://www.clinchem.org/cgi/reprint/55/4/641">
		<strong>Full Text</strong> (PDF)</a>


				
			
		
		</strong>
		</td></tr>
	
	

	
	
	
	
	

	
	
	
	
	

	
	
	

	
	




	

	
		
	

	
		
	

	
		

	
		
		

			


	

			

			
		
	

			
	  
	

			

			
			

			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/content/full/clinchem.2008.112789/DC1">
				HTML Page - index.htslp</a></strong>
			</td></tr>
			
			

			
	
	

	
		
	









	
		
	
		
		
			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item"><strong>
			All Versions of this Article:<br>
		
			<a href="http://www.clinchem.org/cgi/content/short/clinchem.2008.112789v1">
					clinchem.2008.112789v1</a>
			
		<br>
	

	
		
		
			
					55/4/641
				
		&nbsp;&nbsp;&nbsp;<font color="#cc0000"><em>most recent</em></font></strong></td></tr>
	

	





	
	
	

	
		
	

	 

	
	
		  
    

	
	
	
	







		
	
		
			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/alerts/ctalert?alertType=citedby&amp;addAlert=cited_by&amp;saveAlert=no&amp;cited_by_criteria_resid=clinchem;55/4/641&amp;return_type=article&amp;return_url=http%3A%2F%2Fwww.clinchem.org%2Fcgi%2Fcontent%2Ffull%2F55%2F4%2F641">
				Alert me when this article is cited</a></strong>
			</td></tr>
	

		
		

		
		

		
		

			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/alerts/ctalert?alertType=correction&amp;addAlert=correction&amp;saveAlert=no&amp;correction_criteria_value=55/4/641&amp;return_type=article&amp;return_url=http%3A%2F%2Fwww.clinchem.org%2Fcgi%2Fcontent%2Ffull%2F55%2F4%2F641">
				Alert me if a correction is posted</a></strong>
			</td></tr>
		
























			
		
			
			
			
				
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
					<strong><a rel="nofollow" href="http://www.clinchem.org/cgi/citemap?id=clinchem;55/4/641">
					Citation Map</a></strong>
				</td></tr>
		
	

			
			
<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr><tr><td class="content_box_title" colspan="2">Services</td></tr>

	

	<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr>
			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/mailafriend?url=http%3A%2F%2Fwww.clinchem.org%2Fcgi%2Fcontent%2Ffull%2F55%2F4%2F641&amp;title=Next-Generation+Sequencing%3A+From+Basic+Research+to+Diagnostics">
				Email this article to a friend</a></strong>
			</td></tr>
			
			


	

	

	

		
		



			
			
			
		
	

		
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
		<strong>
		<a href="http://www.clinchem.org/cgi/search?qbe=clinchem;clinchem.2008.112789&amp;journalcode=clinchem&amp;minscore=5000">
		Similar articles in this journal</a>
		</strong>
		</td></tr>

		
		

			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/external_ref?access_num=19246620&amp;link_type=MED_NBRS">
				Similar articles in PubMed</a></strong>
			</td></tr>
		
	


		



			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/alerts/etoc">
				Alert me to new issues of the journal</a></strong>
			</td></tr>
		

		
		
	

		
		
		

			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
			<strong><a href="http://www.clinchem.org/cgi/citmgr?gca=clinchem;55/4/641">
			Download to citation manager</a></strong>
			</td></tr>
		
		
	


	
		
	

	



  			
		
		
		
	


	

	




	


			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/external_ref?&amp;link_type=PERMISSIONDIRECT" target="_new">
				<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/permissions.gif" alt="reprints &amp; permissions" border="0"></a></strong>
			</td></tr>	
			





		

		

	
	
	
		


	

	

	


	





	


	
		 
	

		
		

	

	
	















	
		
			
				
				
<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr><tr><td class="content_box_title" colspan="2">Google Scholar</td></tr>

	

	<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr>
						
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
							<strong><a target="_blank" href="http://scholar.google.com/scholar?q=%22author%3AK.%20V.+author%3AVoelkerding%22">
							Articles by Voelkerding, K. V.</a></strong>
						</td></tr>
				
				
			
				
			
				
				

						
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
							<strong><a target="_blank" href="http://scholar.google.com/scholar?q=%22author%3AJ.%20D.+author%3ADurtschi%22">
							Articles by Durtschi, J. D.</a></strong>
						</td></tr>
				
				
			
		
		
	
	
	
	

		
			
				
			
		
	










	





	
		
<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr><tr><td class="content_box_title" colspan="2">PubMed</td></tr>

	

	<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr>
			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/external_ref?access_num=19246620&amp;link_type=PUBMED">
				PubMed Citation</a></strong>
			</td></tr>
		
	

	
		
		
			
				

						
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
							<strong><a target="_blank" href="http://www.clinchem.org/cgi/external_ref?access_num=Voelkerding+KV&amp;link_type=AUTHORSEARCH">
							Articles by Voelkerding, K. V.</a></strong>
						</td></tr>
				
			
				

						
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
							<strong><a target="_blank" href="http://www.clinchem.org/cgi/external_ref?access_num=Durtschi+JD&amp;link_type=AUTHORSEARCH">
							Articles by Durtschi, J. D.</a></strong>
						</td></tr>
				
			
		

		
	

	















	
	


	












	


































	
		

	

		
			
		
	 
		











	

	






































	</tbody></table>
</td></tr></tbody></table> 


		
	



 







 





	
	
	
	
	
	
    
	
	
	
	
		<font size="-1">
(<em>Clinical Chemistry.</em> 2009;55:641-658.)<br>
© 2009 American Association for Clinical Chemistry, Inc.
</font><br>
<table cellpadding="0" cellspacing="0"><tbody><tr><td><hr noshade="noshade" size="1"></td></tr><tr><td><h3>Reviews</h3></td></tr></tbody></table>

<h2>
Next-Generation Sequencing: From Basic Research to Diagnostics</h2>
<strong>
<nobr>Karl V. Voelkerding<sup>1</sup><sup>,2</sup><sup>,a</sup></nobr>, 
<nobr>Shale A. Dames<sup>1</sup><a name="RFN2"></a><sup>,<a href="#FN2">2</a></sup></nobr> and 
<nobr>Jacob D. Durtschi<sup>1</sup><a name="RFN2"></a><sup>,<a href="#FN2">2</a></sup></nobr>
</strong><br>
<p><font size="-1">
<sup>1</sup> ARUP Institute for Experimental and Clinical Pathology, Salt Lake City, Utah; <sup>2</sup> 
Department of Pathology, University of Utah, Salt Lake City, Utah.
</font></p><p>

</p><p><font size="-1"><sup>a</sup>Address correspondence to this
author at: ARUP Laboratories, 500 Chipeta Way, Salt Lake City, Utah
84108. Fax (801) 584-5207; e-mail <span id="em0"><a href="mailto:voelkek@aruplab.com">voelkek@aruplab.com</a></span><script type="text/javascript"><!--
 var u = "voelkek", d = "aruplab.com"; document.getElementById("em0").innerHTML = '<a href="mailto:' + u + '@' + d + '">' + u + '@' + d + '<\/a>'//--></script>.</font></p><p>

<a name="ABS"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
Abstract
</font></th></tr></tbody></table>
<table align="right" border="1" cellpadding="5"><tbody><tr><th align="left"><font size="-1">
<a href="#top"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Top<br></a>
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/dot.gif" border="0" height="9" hspace="5" width="11"><font color="#464c53">Abstract</font><br>
<a href="#BDY"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Introduction<br></a>
<a href="#SEC1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Fundamentals of NGS Platforms<br></a>
<a href="#SEC2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">The Impact of NGS...<br></a>
<a href="#SEC3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">NGS Data Analysis<br></a>
<a href="#SEC4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">A Clinical Future for...<br></a>
<a href="#SEC5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Technologies on the Horizon<br></a>
<a href="#SEC6"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Conclusions<br></a>
<a href="#BIBL"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">References<br></a>
</font></th></tr></tbody></table>
&nbsp;<br>


Background: For the past 30 years, the Sanger method has been<sup> </sup>the dominant approach and gold standard for DNA sequencing.<sup> </sup>The commercial launch of the first massively parallel pyrosequencing<sup> </sup>platform in 2005 ushered in the new era of high-throughput genomic<sup> </sup>analysis now referred to as next-generation sequencing (NGS).<sup> </sup></p><p>

Content: This review describes fundamental principles of commercially<sup> </sup>available NGS platforms. Although the platforms differ in their<sup> </sup>engineering configurations and sequencing chemistries, they<sup> </sup>share a technical paradigm in that sequencing of spatially separated,<sup> </sup>clonally amplified DNA templates or single DNA molecules is<sup> </sup>performed in a flow cell in a massively parallel manner. Through<sup> </sup>iterative cycles of polymerase-mediated nucleotide extensions<sup> </sup>or, in one approach, through successive oligonucleotide ligations,<sup> </sup>sequence outputs in the range of hundreds of megabases to gigabases<sup> </sup>are now obtained routinely. Highlighted in this review are the<sup> </sup>impact of NGS on basic research, bioinformatics considerations,<sup> </sup>and translation of this technology into clinical diagnostics.<sup> </sup>Also presented is a view into future technologies, including<sup> </sup>real-time single-molecule DNA sequencing and nanopore-based<sup> </sup>sequencing.<sup> </sup></p><p>

Summary: In the relatively short time frame since 2005, NGS<sup> </sup>has fundamentally altered genomics research and allowed investigators<sup> </sup>to conduct experiments that were previously not technically<sup> </sup>feasible or affordable. The various technologies that constitute<sup> </sup>this new paradigm continue to evolve, and further improvements<sup> </sup>in technology robustness and process streamlining will pave<sup> </sup>the path for translation into clinical diagnostics.<sup> </sup></p><p>




<a name="BDY"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
Introduction
</font></th></tr></tbody></table>
<table align="right" border="1" cellpadding="5"><tbody><tr><th align="left"><font size="-1">
<a href="#top"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Top<br></a>
<a href="#ABS"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Abstract<br></a>
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/dot.gif" border="0" height="9" hspace="5" width="11"><font color="#464c53">Introduction</font><br>
<a href="#SEC1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Fundamentals of NGS Platforms<br></a>
<a href="#SEC2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">The Impact of NGS...<br></a>
<a href="#SEC3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">NGS Data Analysis<br></a>
<a href="#SEC4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">A Clinical Future for...<br></a>
<a href="#SEC5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Technologies on the Horizon<br></a>
<a href="#SEC6"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Conclusions<br></a>
<a href="#BIBL"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">References<br></a>
</font></th></tr></tbody></table>
&nbsp;<br>

In 1977, 2 landmark articles describing methods for DNA sequencing<sup> </sup>were published. Allan Maxam and Walter Gilbert reported an approach<sup> </sup>in which terminally labeled DNA fragments were subjected to<sup> </sup>base-specific chemical cleavage and the reaction products were<sup> </sup>separated by gel electrophoresis(<a href="#R1">1</a>). In an alternative approach,<sup> </sup>Frederick Sanger and colleagues described the use of chain-terminating<sup> </sup>dideoxynucleotide analogs that caused base-specific termination<sup> </sup>of primed DNA synthesis(<a href="#R2">2</a>). Refinement and commercialization<sup> </sup>of the latter method led to its broad dissemination throughout<sup> </sup>the research community and, ultimately, into clinical diagnostics.<sup> </sup>In an industrial, high-throughput configuration, Sanger technology<sup> </sup>was used in the sequencing of the first human genome, which<sup> </sup>was completed in 2003 through the Human Genome Project, a 13-year<sup> </sup>effort with an estimated cost of $2.7 billion. In 2008, by comparison,<sup> </sup>a human genome was sequenced over a 5-month period for approximately<sup> </sup>$1.5 million(<a href="#R3">3</a>). The latter accomplishment highlights the capabilities<sup> </sup>of the rapidly evolving field of "next-generation" sequencing<sup> </sup>(NGS)<a name="RFN1"></a><sup><a href="#FN1">1</a></sup>
 technologies that have emerged during the past 5 years.<sup> </sup>Currently, 5 NGS platforms are commercially available, with<sup> </sup>additional platforms on the horizon. To add to this pace, the<sup> </sup>US National Human Genome Research Institute (NHGRI) announced<sup> </sup>funding in August 2008 for a series of projects as part of its<sup> </sup>Revolutionary Genome Sequencing Technologies program, which<sup> </sup>has as its goal the sequencing of a human genome for $1000 or<sup> </sup>less (<a href="http://www.genome.gov/27527585">http://www.genome.gov/27527585</a>). This review describes<sup> </sup>NGS technologies, reviews their impact on basic research, and<sup> </sup>explores how they have the translational potential to substantially<sup> </sup>impact molecular diagnostics.<sup> </sup></p><p>

<a name="SEC1"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
Fundamentals of NGS Platforms
</font></th></tr></tbody></table>
<table align="right" border="1" cellpadding="5"><tbody><tr><th align="left"><font size="-1">
<a href="#top"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Top<br></a>
<a href="#ABS"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Abstract<br></a>
<a href="#BDY"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Introduction<br></a>
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/dot.gif" border="0" height="9" hspace="5" width="11"><font color="#464c53">Fundamentals of NGS Platforms</font><br>
<a href="#SEC2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">The Impact of NGS...<br></a>
<a href="#SEC3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">NGS Data Analysis<br></a>
<a href="#SEC4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">A Clinical Future for...<br></a>
<a href="#SEC5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Technologies on the Horizon<br></a>
<a href="#SEC6"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Conclusions<br></a>
<a href="#BIBL"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">References<br></a>
</font></th></tr></tbody></table>
&nbsp;<br>

NGS platforms share a common technological feature&#8212;massively<sup> </sup>parallel sequencing of clonally amplified or single DNA molecules<sup> </sup>that are spatially separated in a flow cell. This design is<sup> </sup>a paradigm shift from that of Sanger sequencing, which is based<sup> </sup>on the electrophoretic separation of chain-termination products<sup> </sup>produced in individual sequencing reactions. In NGS, sequencing<sup> </sup>is performed by repeated cycles of polymerase-mediated nucleotide<sup> </sup>extensions or, in one format, by iterative cycles of oligonucleotide<sup> </sup>ligation. As a massively parallel process, NGS generates hundreds<sup> </sup>of megabases to gigabases of nucleotide-sequence output in a<sup> </sup>single instrument run, depending on the platform. These platforms<sup> </sup>are reviewed next.<sup> </sup></p><p>

<strong>roche/454 life sciences</strong><br>

The 454 technology (<a href="http://www.454.com/">http://www.454.com</a>) is derived from the<sup> </sup>technological convergence of pyrosequencing and emulsion PCR.<sup> </sup>In 1993, Nyren et al. described a sequencing approach based<sup> </sup>on chemiluminescent detection of pyrophosphate released during<sup> </sup>polymerase-mediated deoxynucleoside triphosphate (dNTP) incorporation(<a href="#R4">4</a>).<sup> </sup>Refinement by Ronaghi et al. served as the foundation for the<sup> </sup>commercial development of pyrosequencing(<a href="#R5">5</a>)(<a href="#R6">6</a>). On a separate<sup> </sup>front, Tawfik and Griffiths described single-molecule PCR in<sup> </sup>microcompartments consisting of water-in-oil emulsions(<a href="#R7">7</a>). In<sup> </sup>2000, Jonathan Rothberg founded 454 Life Sciences, which developed<sup> </sup>the first commercially available NGS platform, the GS 20, launched<sup> </sup>in 2005. Combining single-molecule emulsion PCR with pyrosequencing,<sup> </sup>Margulies and colleagues at 454 Life Sciences performed shotgun<sup> </sup>sequencing of the entire 580 069 bp of the <i>Mycoplasma genitalia</i><sup> </sup>genome at 96% coverage and 99.96% accuracy in a single GS 20<sup> </sup>run(<a href="#R8">8</a>). In 2007, Roche Applied Science acquired 454 Life Sciences<sup> </sup>and introduced the second version of the 454 instrument, the<sup> </sup>GS FLX. Sharing the same core technology as the GS 20, the GS<sup> </sup>FLX flow cell is referred to as a "picotiter well" plate, which<sup> </sup>is made from a fused fiber-optic bundle. In its newest configuration,<sup> </sup>approximately 3.4 <font face="arial,helvetica">x</font> 10<sup>6</sup> picoliter-scale sequencing-reaction<sup> </sup>wells are etched into the plate surface, and the well walls<sup> </sup>have a metal coating to improve signal-to-noise discrimination.<sup> </sup>For sequencing (Fig. 1<a href="#F1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-down.gif" border="1" height="7" width="8"> </a>
), a library of template DNA is prepared<sup> </sup>by fragmentation via nebulization or sonication. Fragments several<sup> </sup>hundred base pairs in length are end-repaired and ligated to<sup> </sup>adapter oligonucleotides. The library is then diluted to single-molecule<sup> </sup>concentration, denatured, and hybridized to individual beads<sup> </sup>containing sequences complementary to adapter oligonucleotides.<sup> </sup>The beads are compartmentalized into water-in-oil microvesicles,<sup> </sup>where clonal expansion of single DNA molecules bound to the<sup> </sup>beads occurs during emulsion PCR. After amplification, the emulsion<sup> </sup>is disrupted, and the beads containing clonally amplified template<sup> </sup>DNA are enriched. The beads are again separated by limiting<sup> </sup>dilution, deposited into individual picotiter-plate wells, and<sup> </sup>combined with sequencing enzymes. Loaded into the GS FLX, the<sup> </sup>picotiter plate functions as a flow cell wherein iterative pyrosequencing<sup> </sup>is performed by successive flow addition of the 4 dNTPs. A nucleotide-incorporation<sup> </sup>event in a well containing clonally amplified template produces<sup> </sup>pyrophosphate release and picotiter-plate well&#8211;localized<sup> </sup>luminescence, which is transmitted through the fiber-optic plate<sup> </sup>and recorded on a charge-coupled device camera. With the flow<sup> </sup>of each dNTP reagent, wells are imaged, analyzed for their signal-to-noise<sup> </sup>ratio, filtered according to quality criteria, and subsequently<sup> </sup>algorithmically translated into a linear sequence output. With<sup> </sup>the newest chemistry, termed "Titanium," a single GS FLX run<sup> </sup>generates approximately 1 <font face="arial,helvetica">x</font> 10<sup>6</sup> sequence reads, with read lengths<sup> </sup>of <img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/ge.gif" alt="&#8805;" border="0">400 bases yielding up to 500 million base pairs (Mb) of sequence.<sup> </sup>A recognized strength of the 454 technology is the longer read<sup> </sup>length, which facilitates de novo assembly of genomes(<a href="#R9">9</a>). An<sup> </sup>outstanding concern has been the accurate determination of homopolymers<sup> </sup>&gt;3&#8211;4 bases in length. A 6-base homopolymer should theoretically<sup> </sup>yield twice the luminescence of a 3-base homopolymer. Operationally,<sup> </sup>this luminescence yield varies, and estimates of homopolymer<sup> </sup>length are less accurate with increasing length(<a href="#R8">8</a>)(<a href="#R10">10</a>). 454<sup> </sup>has reported that the metal coating of the walls of picotiter<sup> </sup>wells mentioned above improves the accuracy of homopolymer determination.<sup> </sup>Sequence coverage depth and accuracy for the 454 technology<sup> </sup>is discussed below in the NGS Data Analysis section.<sup> </sup></p><p>

<a name="F1"><!-- null --></a>
<br clear="all"></p><center><table cellpadding="0" cellspacing="0" width="95%"><tbody><tr bgcolor="#e1e1e1"><td><table cellpadding="2" cellspacing="2">
<tbody><tr bgcolor="#e1e1e1"><td align="center" bgcolor="#ffffff" valign="top">
<a href="http://www.clinchem.org/cgi/content/full/55/4/641/F1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/zcy0040992150001.gif" alt="Figure 1" border="2" height="200" hspace="10" vspace="5" width="130"></a><br>

<strong>View larger version</strong> (29K):<br>
<nobr><a href="http://www.clinchem.org/cgi/content/full/55/4/641/F1">[in this window]</a><br>
<a href="http://www.clinchem.org/cgi/content-nw/full/55/4/641/F1" onclick="startTarget('F1', 437, 640); this.href='/cgi/content-nw/full/55/4/641/F1'" onmouseover="window.status='View figure in a separate window'; return true" target="F1">[in a new window]</a><br>
<br>&nbsp;</nobr>
</td><td align="left" valign="top">
<b>Figure 1.</b> <b>Roche 454 GS FLX sequencing.</b><p>
Template DNA is fragmented, end-repaired, ligated to adapters, and
clonally amplified by emulsion PCR. After amplification, the beads are
deposited into picotiter-plate wells with sequencing enzymes. The
picotiter plate functions as a flow cell where iterative pyrosequencing
is performed. A nucleotide-incorporation event results in pyrophosphate
(PP<sub>i</sub>) release and well-localized luminescence. APS, adenosine 5'-phosphosulfate.</p><p>


</p></td></tr></tbody></table>
</td></tr></tbody></table></center><p>


<strong>illumina/solexa</strong><br>

In 1997, British chemists Shankar Balasubramanian and David<sup> </sup>Klenerman conceptualized an approach for sequencing single DNA<sup> </sup>molecules attached to microspheres. They founded Solexa in 1998,<sup> </sup>and their goal during early development of sequencing single<sup> </sup>DNA molecules was not achieved, requiring a shift toward sequencing<sup> </sup>clonally amplified templates. By 2006, the Solexa Genome Analyzer,<sup> </sup>the first "short read" sequencing platform, was commercially<sup> </sup>launched. Acquired by Illumina (<a href="http://www.illumina.com/">http://www.Illumina.com</a>) in<sup> </sup>2006, the Genome Analyzer uses a flow cell consisting of an<sup> </sup>optically transparent slide with 8 individual lanes on the surfaces<sup> </sup>of which are bound oligonucleotide anchors (Fig. 2<a href="#F2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-down.gif" border="1" height="7" width="8"> </a>
). Template<sup> </sup>DNA is fragmented into lengths of several hundred base pairs<sup> </sup>and end-repaired to generate 5'-phosphorylated blunt ends. The<sup> </sup>polymerase activity of Klenow fragment is used to add a single<sup> </sup>A base to the 3' end of the blunt phosphorylated DNA fragments.<sup> </sup>This addition prepares the DNA fragments for ligation to oligonucleotide<sup> </sup>adapters, which have an overhang of a single T base at their<sup> </sup>3' end to increase ligation efficiency. The adapter oligonucleotides<sup> </sup>are complementary to the flow-cell anchors. Under limiting-dilution<sup> </sup>conditions, adapter-modified, single-stranded template DNA is<sup> </sup>added to the flow cell and immobilized by hybridization to the<sup> </sup>anchors. In contrast to emulsion PCR, DNA templates are amplified<sup> </sup>in the flow cell by "bridge" amplification, which relies on<sup> </sup>captured DNA strands "arching" over and hybridizing to an adjacent<sup> </sup>anchor oligonucleotide. Multiple amplification cycles convert<sup> </sup>the single-molecule DNA template to a clonally amplified arching<sup> </sup>"cluster," with each cluster containing approximately 1000 clonal<sup> </sup>molecules. Approximately 50 <font face="arial,helvetica">x</font> 10<sup>6</sup> separate clusters can be generated<sup> </sup>per flow cell. For sequencing, the clusters are denatured, and<sup> </sup>a subsequent chemical cleavage reaction and wash leave only<sup> </sup>forward strands for single-end sequencing. Sequencing of the<sup> </sup>forward strands is initiated by hybridizing a primer complementary<sup> </sup>to the adapter sequences, which is followed by addition of polymerase<sup> </sup>and a mixture of 4 differently colored fluorescent reversible<sup> </sup>dye terminators. The terminators are incorporated according<sup> </sup>to sequence complementarity in each strand in a clonal cluster.<sup> </sup>After incorporation, excess reagents are washed away, the clusters<sup> </sup>are optically interrogated, and the fluorescence is recorded.<sup> </sup>With successive chemical steps, the reversible dye terminators<sup> </sup>are unblocked, the fluorescent labels are cleaved and washed<sup> </sup>away, and the next sequencing cycle is performed. This iterative,<sup> </sup>sequencing-by-synthesis process requires approximately 2.5 days<sup> </sup>to generate read lengths of 36 bases. With 50 <font face="arial,helvetica">x</font> 10<sup>6</sup> clusters<sup> </sup>per flow cell, the overall sequence output is &gt;1 billion<sup> </sup>base pairs (Gb) per analytical run(<a href="#R11">11</a>). The newest platform,<sup> </sup>the Genome Analyzer II, has optical modifications enabling analysis<sup> </sup>of higher cluster densities. Coupled with ongoing improvements<sup> </sup>in sequencing chemistry and projected read lengths of 50-plus<sup> </sup>bases, further increases in output should be realized. Illumina<sup> </sup>and other NGS technologies have devised strategies to sequence<sup> </sup>both ends of template molecules. Such "paired-end" sequencing<sup> </sup>provides positional information that facilitates alignment and<sup> </sup>assembly, especially for short reads(<a href="#R12">12</a>)(<a href="#R13">13</a>). A technical concern<sup> </sup>of Illumina sequencing is that base-call accuracy decreases<sup> </sup>with increasing read length(<a href="#R14">14</a>). This phenomenon is primarily<sup> </sup>due to "dephasing noise." During a given sequencing cycle, nucleotides<sup> </sup>can be under- or overincorporated, or block removal can fail.<sup> </sup>With successive cycles, these aberrations accumulate to produce<sup> </sup>a heterogeneous population in a cluster of strands of varying<sup> </sup>lengths. This heterogeneity decreases signal purity and reduces<sup> </sup>precision in base calling, especially at the 3' ends of reads.<sup> </sup>Modifications in sequencing chemistry and algorithms for data-image<sup> </sup>analysis and interpretation are being pursued to mitigate dephasing(<a href="#R15">15</a>).<sup> </sup>Investigators at the Wellcome Trust Sanger Institute, who have<sup> </sup>extensive experience with the Illumina platform, have published<sup> </sup>a series of technical improvements for library preparation,<sup> </sup>including methods for increasing the reproducibility of fragmentation<sup> </sup>by adaptive focused acoustic wave sonication, enhanced efficiency<sup> </sup>of adapter ligation by use of an alternate ligase, and reducing<sup> </sup>the G+C bias that has been observed in Illumina reads via a<sup> </sup>modified gel-extraction protocol(<a href="#R16">16</a>).<sup> </sup></p><p>

<a name="F2"><!-- null --></a>
<br clear="all"></p><center><table cellpadding="0" cellspacing="0" width="95%"><tbody><tr bgcolor="#e1e1e1"><td><table cellpadding="2" cellspacing="2">
<tbody><tr bgcolor="#e1e1e1"><td align="center" bgcolor="#ffffff" valign="top">
<a href="http://www.clinchem.org/cgi/content/full/55/4/641/F2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/zcy0040992150002.gif" alt="Figure 2" border="2" height="200" hspace="10" vspace="5" width="130"></a><br>

<strong>View larger version</strong> (27K):<br>
<nobr><a href="http://www.clinchem.org/cgi/content/full/55/4/641/F2">[in this window]</a><br>
<a href="http://www.clinchem.org/cgi/content-nw/full/55/4/641/F2" onclick="startTarget('F2', 436, 640); this.href='/cgi/content-nw/full/55/4/641/F2'" onmouseover="window.status='View figure in a separate window'; return true" target="F2">[in a new window]</a><br>
<br>&nbsp;</nobr>
</td><td align="left" valign="top">
<b>Figure 2.</b> <b>Illumina Genome Analyzer sequencing.</b><p>
Adapter-modified, single-stranded DNA is added to the flow cell and
immobilized by hybridization. Bridge amplification generates clonally
amplified clusters. Clusters are denatured and cleaved; sequencing is
initiated with addition of primer, polymerase (POL) and 4 reversible
dye terminators. Postincorporation fluorescence is recorded. The fluor
and block are removed before the next synthesis cycle.</p><p>


</p></td></tr></tbody></table>
</td></tr></tbody></table></center><p>


<strong>applied biosystems/solid</strong><br>

The SOLiD (Supported Oligonucleotide Ligation and Detection)<sup> </sup>System 2.0 platform, which is distributed by Applied Biosystems<sup> </sup>(<a href="http://www.solid.appliedbiosystems.com/">http://www.solid.appliedbiosystems.com</a>), is a short-read sequencing<sup> </sup>technology based on ligation. This approach was developed in<sup> </sup>the laboratory of George Church and reported in 2005 along with<sup> </sup>the resequencing of the <i>Escherichia coli</i> genome(<a href="#R17">17</a>). Applied<sup> </sup>Biosystems refined the technology and released the SOLiD instrumentation<sup> </sup>in 2007. Sample preparation shares similarities with the 454<sup> </sup>technology in that DNA fragments are ligated to oligonucleotide<sup> </sup>adapters, attached to beads, and clonally amplified by emulsion<sup> </sup>PCR. Beads with clonally amplified template are immobilized<sup> </sup>onto a derivitized-glass flow-cell surface, and sequencing is<sup> </sup>begun by annealing a primer oligonucleotide complementary to<sup> </sup>the adapter at the adapter&#8211;template junction (Fig. 3<a href="#F3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-down.gif" border="1" height="7" width="8"> </a>
).<sup> </sup>Instead of providing a 3' hydroxyl group for polymerase-mediated<sup> </sup>extension, the primer is oriented to provide a 5' phosphate<sup> </sup>group for ligation to interrogation probes during the first<sup> </sup>"ligation sequencing" step. Each interrogation probe is an octamer,<sup> </sup>which consists of (in the 3'-to-5' direction) 2 probe-specific<sup> </sup>bases followed by 6 degenerate bases with one of 4 fluorescent<sup> </sup>labels linked to the 5' end. The 2 probe-specific bases consist<sup> </sup>of one of 16 possible 2-base combinations (for example TT, GT,<sup> </sup>and so forth). In the first ligation-sequencing step, thermostable<sup> </sup>ligase and interrogation probes representing the 16 possible<sup> </sup>2-base combinations are present. The probes compete for annealing<sup> </sup>to the template sequences immediately adjacent to the primer.<sup> </sup>After annealing, a ligation step is performed, followed by wash<sup> </sup>removal of unbound probe. Fluorescence signals are optically<sup> </sup>collected before cleavage of the ligated probes, and a wash<sup> </sup>is performed to remove the fluor and regenerate the 5' phosphate<sup> </sup>group. In the subsequent sequencing steps, interrogation probes<sup> </sup>are ligated to the 5' phosphate group of the preceding pentamer.<sup> </sup>Seven cycles of ligation, referred to as a "round," are performed<sup> </sup>to extend the first primer. The synthesized strand is then denatured,<sup> </sup>and a new sequencing primer offset by 1 base in the adapter<sup> </sup>sequence (n &#8211; 1) is annealed. Five rounds total are performed,<sup> </sup>each time with a new primer with a successive offset (n &#8211;<sup> </sup>2, n &#8211; 3, and so on). By this approach, each template<sup> </sup>nucleotide is sequenced twice. A 6-day instrument run generates<sup> </sup>sequence read lengths of 35 bases. Sequence is inferred by interpreting<sup> </sup>the ligation results for the 16 possible 2 base&#8211;combination<sup> </sup>interrogation probes. With the use of offset primers, several<sup> </sup>bases of the adapter are sequenced. This information provides<sup> </sup>a sequence reference starting point that is used in conjunction<sup> </sup>with the color space&#8211;coding scheme to algorithmically<sup> </sup>deconvolute the downstream template sequence (see Fig. 1 in<sup> </sup>the Data Supplement that accompanies the online version of this<sup> </sup>review at <a href="http://www.clinchem.org/content/vol55/issue4">http://www.clinchem.org/content/vol55/issue4</a>). Placing<sup> </sup>2 flow-cell slides in the instrument per analytical run produces<sup> </sup>a combined output of 4 Gb of sequence or greater. Unextended<sup> </sup>strands are capped before the ligation to mitigate signal deterioration<sup> </sup>due to dephasing. Capping coupled with high-fidelity ligation<sup> </sup>chemistry and interrogation of each nucleotide base twice during<sup> </sup>independent ligation cycles yields a company-reported sequence<sup> </sup>consensus accuracy of 99.9% for a known target at a 15-fold<sup> </sup>sequence coverage over sequence reads of 25 nucleotides. On<sup> </sup>an independent track, the Church laboratory has collaborated<sup> </sup>with Danaher Motion and Dover Systems to develop and introduce<sup> </sup>an alternative sequencing-by-ligation platform, the Polonator<sup> </sup>G.007 (<a href="http://www.polonator.org/">http://www.polonator.org</a>). Table 1<a href="#T1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-down.gif" border="1" height="7" width="8"> </a>
 summarizes GS FLX,<sup> </sup>Genome Analyzer, and SOLiD platform features.<sup> </sup></p><p>

<a name="F3"><!-- null --></a>
<br clear="all"></p><center><table cellpadding="0" cellspacing="0" width="95%"><tbody><tr bgcolor="#e1e1e1"><td><table cellpadding="2" cellspacing="2">
<tbody><tr bgcolor="#e1e1e1"><td align="center" bgcolor="#ffffff" valign="top">
<a href="http://www.clinchem.org/cgi/content/full/55/4/641/F3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/zcy0040992150003.gif" alt="Figure 3" border="2" height="200" hspace="10" vspace="5" width="148"></a><br>

<strong>View larger version</strong> (27K):<br>
<nobr><a href="http://www.clinchem.org/cgi/content/full/55/4/641/F3">[in this window]</a><br>
<a href="http://www.clinchem.org/cgi/content-nw/full/55/4/641/F3" onclick="startTarget('F3', 476, 640); this.href='/cgi/content-nw/full/55/4/641/F3'" onmouseover="window.status='View figure in a separate window'; return true" target="F3">[in a new window]</a><br>
<br>&nbsp;</nobr>
</td><td align="left" valign="top">
<b>Figure 3.</b> <b>Applied Biosystems SOLiD sequencing by ligation.</b><p>
Top: SOLiD color-space coding. Each interrogation probe is an octamer,
which consists of (3'-to-5' direction) 2 probe-specific bases followed
by 6 degenerate bases (nnnzzz) with one of 4 fluorescent labels linked
to the 5' end. The 2 probe-specific bases consist of one of 16 possible
2-base combinations. Bottom: (A), The P1 adapter and template with
annealed primer (n) is interrogated by probes representing the 16
possible 2-base combinations. In this example, the 2 specific bases
complementary to the template are AT. (B), After annealing and ligation
of the probe, fluorescence is recorded before cleavage of the last 3
degenerate probe bases. The 5' end of the cleaved probe is
phosphorylated (not shown) before the second sequencing step. (C),
Annealing and ligation of the next probe. (D), Complete extension of
primer (n) through the first round consisting of 7 cycles of ligation.
(E), The product extended from primer (n) is denatured from the
adapter/template, and the second round of sequencing is performed with
primer (n &#8211; 1). With the use of progressively offset primers, in this
example (n &#8211; 1), adapter bases are sequenced, and this known sequence
is used in conjunction with the color-space coding for determining the
template sequence by deconvolution (see Fig. 1<a href="#F1"><img alt="Up" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-up.gif" border="1" height="7" width="8">
</a>
 in the online Data Supplement). In this technology, template bases are interrogated twice.</p><p>


</p></td></tr></tbody></table>
</td></tr></tbody></table></center><p>


<a name="T1"><!-- null --></a>
<br clear="all"></p><center><table cellpadding="0" cellspacing="0" width="95%"><tbody><tr bgcolor="#e1e1e1"><td><table cellpadding="2" cellspacing="2">
<tbody><tr bgcolor="#e1e1e1"><td align="center" bgcolor="#ffffff" valign="top">
<strong>View this table:</strong><br>
<nobr><a href="http://www.clinchem.org/cgi/content/full/55/4/641/T1">[in this window]</a><br>
<a href="http://www.clinchem.org/cgi/content-nw/full/55/4/641/T1" onclick="startTarget('T1', 500, 400); this.href='/cgi/content-nw/full/55/4/641/T1'" onmouseover="window.status='View figure in a separate window'; return true" target="T1">[in a new window]</a><br>
<br>&nbsp;</nobr>
</td><td align="left" valign="top">
<b>Table 1.</b> Comparison of NGS platforms.<p>


</p></td></tr></tbody></table>
</td></tr></tbody></table></center><p>


<strong>helicos biosciences and single-molecule sequencing</strong><br>

The first single-molecule sequencing platform, the HeliScope,<sup> </sup>is now available from Helicos BioSciences (<a href="http://www.helicosbio.com/">http://www.helicosbio.com</a>)<sup> </sup>with a company-reported sequence output of 1 Gb/day. This technology<sup> </sup>stems from the work of Braslavsky et al., published in 2003(<a href="#R18">18</a>).<sup> </sup>Having obviated clonal amplification of template, the method<sup> </sup>involves fragmenting sample DNA and polyadenylation at the 3'<sup> </sup>end, with the final adenosine fluorescently labeled. Denatured<sup> </sup>polyadenylated strands are hybridized to poly(dT) oligonucleotides<sup> </sup>immobilized on a flow-cell surface at a capture density of up<sup> </sup>to 100 <font face="arial,helvetica">x</font> 10<sup>6</sup> template strands per square centimeter. After the<sup> </sup>positional coordinates of the captured strands are recorded<sup> </sup>by a charge-coupled device camera, the label is cleaved and<sup> </sup>washed away before sequencing. For sequencing, polymerase and<sup> </sup>one of 4 Cy5-labeled dNTPs are added to the flow cell, which<sup> </sup>is imaged to determine incorporation into individual strands.<sup> </sup>After label cleavage and washing, the process is repeated with<sup> </sup>the next Cy5-labeled dNTP. Each sequencing cycle, which consists<sup> </sup>of the successive addition of polymerase and each of the 4 labeled<sup> </sup>dNTPs, is termed a "quad." The number of sequencing quads performed<sup> </sup>is approximately 25&#8211;30, with read lengths of up to 45&#8211;50<sup> </sup>bases having been achieved. The Helicos platform was used to<sup> </sup>sequence the 6407-base genome of bacteriophage M13(<a href="#R19">19</a>). This<sup> </sup>study demonstrated both the potential and important technical<sup> </sup>issues that may be relevant to all single-molecule sequencing<sup> </sup>methods that are based on sequencing by synthesis. First, sequencing<sup> </sup>accuracy was appreciably improved when template molecules were<sup> </sup>sequenced twice ("2-pass" sequencing). Second, the accuracy<sup> </sup>of sequencing homopolymers was compromised by the polymerase<sup> </sup>adding additional bases of the same identity in a homopolymeric<sup> </sup>stretch in a given dNTP addition. Helicos has since developed<sup> </sup>proprietary labeled dNTPs, termed "virtual terminators," which<sup> </sup>the company reports reduce polymerase processivity so that only<sup> </sup>single bases are added, improving the accuracy of homopolymer<sup> </sup>sequencing. Interestingly, the percentage of strands in which<sup> </sup>longer read lengths can be achieved (e.g., 50 nucleotides) is<sup> </sup>substantially lower than that obtained with shorter (e.g., 25<sup> </sup>nucleotides) read-length sequencing, possibly reflecting secondary<sup> </sup>structures (e.g., hairpins) assumed by the template molecules.<sup> </sup></p><p>

<a name="SEC2"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
The Impact of NGS on Basic Research
</font></th></tr></tbody></table>
<table align="right" border="1" cellpadding="5"><tbody><tr><th align="left"><font size="-1">
<a href="#top"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Top<br></a>
<a href="#ABS"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Abstract<br></a>
<a href="#BDY"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Introduction<br></a>
<a href="#SEC1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Fundamentals of NGS Platforms<br></a>
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/dot.gif" border="0" height="9" hspace="5" width="11"><font color="#464c53">The Impact of NGS...</font><br>
<a href="#SEC3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">NGS Data Analysis<br></a>
<a href="#SEC4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">A Clinical Future for...<br></a>
<a href="#SEC5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Technologies on the Horizon<br></a>
<a href="#SEC6"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Conclusions<br></a>
<a href="#BIBL"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">References<br></a>
</font></th></tr></tbody></table>
&nbsp;<br>

In the short 4 years since the first commercial platform became<sup> </sup>available, NGS has markedly accelerated multiple areas of genomics<sup> </sup>research, enabling experiments that previously were not technically<sup> </sup>feasible or affordable. We describe major applications of NGS<sup> </sup>and then review the analysis of NGS data.<sup> </sup></p><p>

<strong>genomic analysis</strong><br>

The high-throughput capacity of NGS has been leveraged to sequence<sup> </sup>entire genomes, from microbes to humans(<a href="#R3">3</a>)(<a href="#R8">8</a>)(<a href="#R9">9</a>)(<a href="#R11">11</a>)(<a href="#R20">20</a>)(<a href="#R21">21</a>)(<a href="#R22">22</a>)(<a href="#R23">23</a>)(<a href="#R24">24</a>),<sup> </sup>including the recent sequencing of the genome of cytogenetically<sup> </sup>normal acute myeloid leukemia cells, which identified novel,<sup> </sup>tumor-specific gene mutations(<a href="#R25">25</a>). The longer read lengths of<sup> </sup>the 454 technology, compared with the Illumina and SOLiD short-read<sup> </sup>technologies, facilitate the assembly of genomes in the absence<sup> </sup>of a reference genome (i.e., de novo assembly). For resequencing,<sup> </sup>both long- and short-read technologies have been used successfully.<sup> </sup>In one comparative study, the 454, Illumina, and SOLiD technologies<sup> </sup>all accurately detected single-nucleotide variations when coverage<sup> </sup>depth was <img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/ge.gif" alt="&#8805;" border="0">15-fold per allele(<a href="#R20">20</a>) (the critical issue of coverage<sup> </sup>depth is discussed further in the NGS Data Analysis section).<sup> </sup>The 454 read lengths provide nucleotide haplotype information<sup> </sup>over a range of several hundred base pairs and are predicted<sup> </sup>to be better suited for detecting larger insertions and deletions<sup> </sup>and for producing alignments in areas containing repetitive<sup> </sup>sequences. Further studies are needed to compare technology<sup> </sup>performance for detecting insertions and deletions. Each platform<sup> </sup>has an optional strategy for sequencing both ends of DNA libraries<sup> </sup>(paired-end sequencing). In addition to effectively doubling<sup> </sup>sequence output, knowing that reads are associated with each<sup> </sup>other on a given fragment augments alignment and assembly, especially<sup> </sup>for short reads. Paired-end sequencing has been used to map<sup> </sup>genomic structural variation, including deletions, insertions,<sup> </sup>and rearrangements(<a href="#R12">12</a>)(<a href="#R13">13</a>)(<a href="#R26">26</a>)(<a href="#R27">27</a>). The ability to sequence<sup> </sup>complete human genomes at a substantially reduced cost with<sup> </sup>NGS has energized an international effort to sequence thousands<sup> </sup>of human genomes over the next decade (<a href="http://www.1000genomes.org/">http://www.1000genomes.org</a>),<sup> </sup>which will lead to the characterization and cataloging of human<sup> </sup>genetic variation at an unprecedented level.<sup> </sup></p><p>

<strong>targeted genomic resequencing</strong><br>

Sequencing of genomic subregions and gene sets is being used<sup> </sup>to identify polymorphisms and mutations in genes implicated<sup> </sup>in cancer and in regions of the human genome that linkage and<sup> </sup>whole-genome association studies have implicated in disease(<a href="#R28">28</a>)(<a href="#R29">29</a>).<sup> </sup>Especially in the latter setting, regions of interest can be<sup> </sup>hundreds of kb&#8217;s to several Mb in size. To best use NGS<sup> </sup>for sequencing such candidate regions, several genomic-enrichment<sup> </sup>steps, both traditional and novel, are being incorporated into<sup> </sup>overall experimental designs. Overlapping long-range PCR amplicons<sup> </sup>(approximately 5&#8211;10 kb) can be used for up to several<sup> </sup>hundred kb&#8217;s, but this approach is not practical for larger<sup> </sup>genomic regions. More recently, enrichment has been achieved<sup> </sup>by hybridizing fragmented, denatured human genomic DNA to oligonucleotide<sup> </sup>capture probes complementary to the region of interest and subsequently<sup> </sup>eluting the enriched DNA(<a href="#R30">30</a>)(<a href="#R31">31</a>)(<a href="#R32">32</a>)(<a href="#R33">33</a>). Capture probes can<sup> </sup>be immobilized on a solid surface (Roche NimbleGen, <a href="http://www.nimblegen.com/">http://www.nimblegen.com</a>;<sup> </sup>Agilent Technologies, <a href="http://www.agilent.com/">http://www.agilent.com</a>; and Febit, <a href="http://www.febit.com/">http://www.febit.com</a>)<sup> </sup>or used in solution (Agilent). Current NimbleGen arrays contain<sup> </sup>350 000 oligonucleotides of 60&#8211;90 bp in length that are<sup> </sup>typically spaced 5&#8211;20 nucleotides apart, with oligonucleotides<sup> </sup>complementary to repetitive regions being excluded. For enrichment,<sup> </sup>5&#8211;20 µg of genomic DNA is fragmented and ligated<sup> </sup>to oligonucleotide linkers containing universal PCR priming<sup> </sup>sites. This material is denatured, hybridized to an array for<sup> </sup>3 days, and eluted, with the enriched DNA amplified by the PCR<sup> </sup>before NGS library preparation. In reported studies, up to 5<sup> </sup>Mb of sequence has been captured on the 350K array, with 60%&#8211;75%<sup> </sup>of sequencing reads mapping to targeted regions; other reads<sup> </sup>mapping to nontargeted regions reflect nonspecific capture.<sup> </sup>In development by NimbleGen is the use of an array of 2.1 <font face="arial,helvetica">x</font><sup> </sup>10<sup>6</sup> features for capturing larger genomic regions. Agilent&#8217;s<sup> </sup>solution-based technology uses oligonucleotides up to 170 bases<sup> </sup>in length, with each end containing sequences for universal<sup> </sup>PCR priming and with primer sites containing a restriction endonuclease&#8211;recognition<sup> </sup>sequence. The oligonucleotide library is amplified by the PCR,<sup> </sup>digested with restriction enzymes, and ligated to adapters containing<sup> </sup>the T7 polymerase promoter site. In vitro transcription is performed<sup> </sup>with biotinylated UTP to generate single-stranded biotinylated<sup> </sup>cRNA capture sequences. For capture, 3 µg of fragmented,<sup> </sup>denatured genomic DNA is hybridized with cRNA sequences for<sup> </sup>24 h in solution. After hybridization, duplexes consisting of<sup> </sup>single-stranded DNA and cRNA are bound to streptavidin-coated<sup> </sup>magnetic beads; the cRNA is then enzymatically digested, leaving<sup> </sup>enriched single-stranded DNA that is subsequently processed<sup> </sup>for NGS. An alternative enrichment approach developed by RainDance<sup> </sup>Technologies (<a href="http://www.raindancetechnologies.com/">http://www.raindancetechnologies.com</a>) uses a novel<sup> </sup>microfluidics technology in which individual pairs of PCR primers<sup> </sup>for the genomic regions of interest are segregated in water<sup> </sup>in emulsion droplets and then pooled to create a "primer library."<sup> </sup>Separately, emulsion droplets containing genomic DNA and PCR<sup> </sup>reagents are prepared. Two separate droplet streams are created,<sup> </sup>one with primer-library droplets and the other with droplets<sup> </sup>containing genomic DNA/PCR reagents. The 2 streams are merged<sup> </sup>and primer-library droplets and genomic DNA/PCR reagent droplets<sup> </sup>are paired in a 1:1 ratio. As paired droplets proceed through<sup> </sup>the microfluidic channel, they pass an electrical impulse that<sup> </sup>causes them to physically coalesce. The coalesced droplets containing<sup> </sup>individual primer pairs and genomic DNA/PCR reagents are deposited<sup> </sup>in a 96-well plate and amplified by the PCR. After amplification,<sup> </sup>the emulsions are disrupted, and the amplicons are pooled and<sup> </sup>processed for NGS.<sup> </sup></p><p>

<strong>metagenomics</strong><br>

NGS has had a tremendous impact on the study of microbial diversity<sup> </sup>in environmental and clinical samples. Operationally, genomic<sup> </sup>DNA is extracted from the sample of interest, converted to an<sup> </sup>NGS library and sequenced. The sequence output is aligned to<sup> </sup>known reference sequences for microorganisms that are predicted<sup> </sup>to be present in the sample. Closely related species can be<sup> </sup>discerned, and more distantly related species can be inferred.<sup> </sup>In addition, de novo assembly of the data set can yield information<sup> </sup>to support the presence of known and potentially new species.<sup> </sup>Qualitative genomic information is obtained, and analysis of<sup> </sup>the relative abundance of the sequence reads can be used to<sup> </sup>derive quantitative information on individual microbial species.<sup> </sup>To date, most NGS-based metagenomic analyses have used the 454<sup> </sup>technology and its associated longer read lengths to facilitate<sup> </sup>alignment to microbial reference genomes and for de novo assembly<sup> </sup>of previously uncharacterized microbial genomes. Examples of<sup> </sup>metagenomic studies include the analysis of microbial populations<sup> </sup>in the ocean(<a href="#R34">34</a>)(<a href="#R35">35</a>) and soil(<a href="#R36">36</a>), the identification of a novel<sup> </sup>arenavirus in transplantation patients(<a href="#R37">37</a>), and the characterization<sup> </sup>of microflora present in the human oral cavity(<a href="#R38">38</a>) and the guts<sup> </sup>of obese and lean twins(<a href="#R39">39</a>).<sup> </sup></p><p>

<strong>transcriptome sequencing</strong><br>

NGS has provided a powerful new approach, termed "RNA-Seq,"<sup> </sup>for mapping and quantifying transcripts in biological samples.<sup> </sup>Total, ribosomal RNA&#8211;depleted, or poly(A)+ RNA is isolated<sup> </sup>and converted to cDNA. A typical protocol would involve the<sup> </sup>generation of first-strand cDNA via random hexamer&#8211;primed<sup> </sup>reverse transcription and subsequent generation of second-strand<sup> </sup>cDNA with RNase H and DNA polymerase. The cDNA is then fragmented<sup> </sup>and ligated to NGS adapters. For small RNAs such as microRNAs<sup> </sup>(miRNAs) and short interfering RNAs, preferential isolation<sup> </sup>via a small RNA&#8211;enrichment method, size selection on an<sup> </sup>electrophoresis gel, or a combination of these approaches is<sup> </sup>commonly used. RNA ligase is used to join adapter sequences<sup> </sup>to the RNA; this step is often followed by a PCR amplification<sup> </sup>step before NGS processing. After sequencing, reads are aligned<sup> </sup>to a reference genome, compared with known transcript sequences,<sup> </sup>or assembled de novo to construct a genome-scale transcription<sup> </sup>map. Although RNA-Seq is in its early stages as a technology,<sup> </sup>it has already shown some advantages over gene expression arrays(<a href="#R40">40</a>).<sup> </sup>First, arrays depend on tiling existing genomic sequences, whereas<sup> </sup>RNA-Seq is not constrained by this limitation, allowing characterization<sup> </sup>of transcription without prior knowledge of the genomic sites<sup> </sup>of transcription origin. RNA-Seq is capable of single-base resolution<sup> </sup>and, compared with arrays, demonstrates a greater ability to<sup> </sup>distinguish RNA isoforms, determine allelic expression, and<sup> </sup>reveal sequence variants. Expression levels are deduced from<sup> </sup>the total number of reads that map to the exons of a gene, normalized<sup> </sup>by the length of exons that can be uniquely mapped. Results<sup> </sup>obtained with this approach have shown close correlation with<sup> </sup>those of quantitative PCR and RNA-spiking experiments. The dynamic<sup> </sup>range of RNA-Seq for determining expression levels is 3&#8211;4<sup> </sup>orders of magnitude, compared with 2 orders of magnitude for<sup> </sup>expression arrays. In this context, RNA-Seq has shown improved<sup> </sup>performance for the quantitative detection of both highly produced<sup> </sup>transcripts and transcripts produced at low levels. RNA-Seq<sup> </sup>is being used to confirm and revise gene annotation, including<sup> </sup>5' and 3', and exon/intron boundaries; the latter is achieved<sup> </sup>by mapping reads to exon junctions defined by GT-AG splicing<sup> </sup>consensus sites. Both qualitative and quantitative information<sup> </sup>regarding splicing diversity can be deduced. RNA-Seq has been<sup> </sup>applied to a variety of organisms, including <i>Saccharomyces cerevisiae</i>,<sup> </sup><i>Arabidopsis thaliana</i>, mice, and human cells(<a href="#R40">40</a>)(<a href="#R41">41</a>)(<a href="#R42">42</a>)(<a href="#R43">43</a>)(<a href="#R44">44</a>)(<a href="#R45">45</a>)(<a href="#R46">46</a>)(<a href="#R47">47</a>)(<a href="#R48">48</a>)(<a href="#R49">49</a>)(<a href="#R50">50</a>)(<a href="#R51">51</a>).<sup> </sup></p><p>

<strong>mapping of dna-binding proteins and chromatin analysis</strong><br>

The delineation of regulatory proteins associated with genomes<sup> </sup>was substantially accelerated by the introduction of chromatin<sup> </sup>immunoprecipitation and microarray hybridization (ChIP-on-chip)<sup> </sup>technology(<a href="#R52">52</a>). In this approach, proteins in contact with genomic<sup> </sup>DNA are chemically cross-linked (typically with mild formaldehyde<sup> </sup>treatment) to their binding sites, and the DNA is fragmented<sup> </sup>by sonication or digestion with micrococcal nuclease. The proteins<sup> </sup>cross-linked with DNA are immunoprecipitated with antibodies<sup> </sup>specific for the proteins of interest. The DNA in the immunoprecipitate<sup> </sup>is purified and hybridized to an oligonucleotide array consisting<sup> </sup>of sequences from the genome, allowing identification of the<sup> </sup>protein-binding sites. This approach has been successfully used<sup> </sup>to identify binding sites for transcription factors and histone<sup> </sup>proteins. ChIP-on-chip technology is now being supplanted in<sup> </sup>a variety of experimental settings with ChIP-Seq, in which the<sup> </sup>DNA harvested from the immunoprecipitate is converted into a<sup> </sup>library for NGS. The obtained reads are mapped to the reference<sup> </sup>genome of interest to generate a genome-wide protein-binding<sup> </sup>map(<a href="#R53">53</a>)(<a href="#R54">54</a>)(<a href="#R55">55</a>). Studies to date that have examined the genomic-binding<sup> </sup>sites of the human NRSF (neuron restrictive silencer factor)<sup> </sup>and STAT1 (signal transducer and activator of transcription<sup> </sup>1) proteins indicate the resolution of ChIP-Seq to be greater<sup> </sup>than for ChIP-on-chip, as evidenced by confirmation of previously<sup> </sup>identified binding sites and identification of novel binding<sup> </sup>sites(<a href="#R56">56</a>)(<a href="#R57">57</a>). Analogous to RNA-Seq, ChIP-Seq has the important<sup> </sup>advantage of not requiring prior knowledge of genomic locations<sup> </sup>of protein binding. In addition to the study of transcription<sup> </sup>factors, NGS is being used to map genomic methylation. One approach<sup> </sup>involves traditional bisulfite conversion of DNA followed by<sup> </sup>NGS, which has been applied to the study of entire genomes or<sup> </sup>genomic subregions(<a href="#R58">58</a>)(<a href="#R59">59</a>). Ongoing studies are attempting to<sup> </sup>develop a variant of ChIP-Seq in which genomic methylation is<sup> </sup>assayed by coupling immunoprecipitation with a monoclonal antibody<sup> </sup>directed against methylated cytosine and subsequent NGS(<a href="#R60">60</a>).<sup> </sup></p><p>

<a name="SEC3"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
NGS Data Analysis
</font></th></tr></tbody></table>
<table align="right" border="1" cellpadding="5"><tbody><tr><th align="left"><font size="-1">
<a href="#top"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Top<br></a>
<a href="#ABS"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Abstract<br></a>
<a href="#BDY"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Introduction<br></a>
<a href="#SEC1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Fundamentals of NGS Platforms<br></a>
<a href="#SEC2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">The Impact of NGS...<br></a>
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/dot.gif" border="0" height="9" hspace="5" width="11"><font color="#464c53">NGS Data Analysis</font><br>
<a href="#SEC4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">A Clinical Future for...<br></a>
<a href="#SEC5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Technologies on the Horizon<br></a>
<a href="#SEC6"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Conclusions<br></a>
<a href="#BIBL"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">References<br></a>
</font></th></tr></tbody></table>
&nbsp;<br>

NGS experiments generate unprecedented volumes of data, which<sup> </sup>present challenges and opportunities for data management, storage,<sup> </sup>and, most importantly, analysis(<a href="#R61">61</a>). NGS data begin as large<sup> </sup>sets of tiled fluorescence or luminescence images of the flow-cell<sup> </sup>surface recorded after each iterative sequencing step (Fig.<sup> </sup>4<a href="#F4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-down.gif" border="1" height="7" width="8"> </a>
). This volume of data requires a resource-intensive data-pipeline<sup> </sup>system for data storage, management, and processing. Data volumes<sup> </sup>generated during single runs of the 454 GS FLX, Illumina, and<sup> </sup>SOLiD instruments are approximately 15 GB, 1 TB, and 15 TB,<sup> </sup>respectively. The main processing feature of the data pipeline<sup> </sup>is the computationally intensive conversion of image data into<sup> </sup>sequence reads, known as base calling. First, individual beads<sup> </sup>or clusters are identified and localized in an image series.<sup> </sup>Image parameters such as intensity, background, and noise are<sup> </sup>then used in a platform-dependant algorithm to generate read<sup> </sup>sequences and error probability&#8211;related quality scores<sup> </sup>for each base. Although many researchers use the base calls<sup> </sup>generated by the platform-specific data-pipeline software, alternative<sup> </sup>base-calling programs that use more advanced software and statistical<sup> </sup>techniques have been developed. Features of these alternative<sup> </sup>programs include the incorporation of ambiguous bases into reads,<sup> </sup>improved removal of poor-quality bases from read ends(<a href="#R62">62</a>), and<sup> </sup>the use of data sets for software training(<a href="#R15">15</a>). Incorporation<sup> </sup>of these features has been shown to reduce read error and improve<sup> </sup>alignment, especially as platforms are pushed to generate longer<sup> </sup>reads. These advantages, however, must be weighed against the<sup> </sup>substantial computer resources required by the large volumes<sup> </sup>of image data.<sup> </sup></p><p>

<a name="F4"><!-- null --></a>
<br clear="all"></p><center><table cellpadding="0" cellspacing="0" width="95%"><tbody><tr bgcolor="#e1e1e1"><td><table cellpadding="2" cellspacing="2">
<tbody><tr bgcolor="#e1e1e1"><td align="center" bgcolor="#ffffff" valign="top">
<a href="http://www.clinchem.org/cgi/content/full/55/4/641/F4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/zcy0040992150004.gif" alt="Figure 4" border="2" height="200" hspace="10" vspace="5" width="163"></a><br>

<strong>View larger version</strong> (22K):<br>
<nobr><a href="http://www.clinchem.org/cgi/content/full/55/4/641/F4">[in this window]</a><br>
<a href="http://www.clinchem.org/cgi/content-nw/full/55/4/641/F4" onclick="startTarget('F4', 508, 640); this.href='/cgi/content-nw/full/55/4/641/F4'" onmouseover="window.status='View figure in a separate window'; return true" target="F4">[in a new window]</a><br>
<br>&nbsp;</nobr>
</td><td align="left" valign="top">
<b>Figure 4.</b> <b>Pseudocolor image from the Illumina flow cell.</b><p>
Each fluorescence signal originates from a clonally amplified template
cluster. Top panel illustrates 4 emission wavelengths of fluorescent
labels depicted in red, green, blue, and yellow. Images are processed
to identify individual clusters and to remove noise or interference.
The lower panel is a composite image of the 4 fluorescence channels.</p><p>


</p></td></tr></tbody></table>
</td></tr></tbody></table></center><p>


The quality values calculated during NGS base calling provide<sup> </sup>important information for alignment, assembly, and variant analysis.<sup> </sup>Although the calculation of quality varies between platforms,<sup> </sup>the calculations are all related to the historically relevant<sup> </sup>phred score, introduced in 1998 for Sanger sequence data(<a href="#R63">63</a>)(<a href="#R64">64</a>).<sup> </sup>The phred score quality value, <i>q</i>, uses a mathematical scale<sup> </sup>to convert the estimated probability of an incorrect call, <i>e</i>,<sup> </sup>to a log scale:
<sup> </sup><br clear="right"></p><p><table border="0" width="100%"><tbody><tr><td align="center">
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fd1.gif" alt="Formula" border="0"></td></tr></tbody></table><sup> </sup></p><p>

Miscall probabilities of 0.1 (10%), 0.01 (1%), and 0.001 (0.1%)<sup> </sup>yield phred scores of 10, 20, and 30, respectively. The NGS<sup> </sup>error rates estimated by quality values depend on several factors,<sup> </sup>including signal-to-noise levels, cross talk from nearby beads<sup> </sup>or clusters, and dephasing. Substantial effort has been made<sup> </sup>to understand and improve the accuracy of quality scores and<sup> </sup>the underlying error sources(<a href="#R10">10</a>)(<a href="#R14">14</a>), including inaccuracies<sup> </sup>in homopolymer run lengths on the 454 platform and base-substitution<sup> </sup>error biases with the Illumina format. Study of these error<sup> </sup>traits has led to examples of software that require no additional<sup> </sup>base calling but that improve quality-score accuracies and thus<sup> </sup>improve sequencing accuracy(<a href="#R65">65</a>)(<a href="#R66">66</a>). Quality values are an important<sup> </sup>tool for rejecting low-quality reads, trimming low-quality bases,<sup> </sup>improving alignment accuracy, and determining consensus-sequence<sup> </sup>and variant calls(<a href="#R67">67</a>).<sup> </sup></p><p>

Alignment and assembly are substantially more difficult for<sup> </sup>NGS data than for Sanger data because of the shorter reads lengths<sup> </sup>in the former. One limitation of short-read alignment and assembly<sup> </sup>is the inability to uniquely align large portions of a read<sup> </sup>set when the read length becomes too short. Similarly, the number<sup> </sup>of uniquely aligned reads is reduced when aligning to larger,<sup> </sup>more complex genomes or reference sequences because of their<sup> </sup>having a higher probability of repetitive sequences. A case<sup> </sup>in point is a modeling study that indicated that 97% of the<sup> </sup><i>E. coli</i> genome can be uniquely aligned with 18-bp reads but<sup> </sup>that only 90% of the human genome can be uniquely aligned with<sup> </sup>30-bp reads(<a href="#R68">68</a>)(<a href="#R69">69</a>). Unique alignment or assembly is reduced<sup> </sup>not only by the presence of repeat sequences but also by shared<sup> </sup>homologies within closely related gene families and pseudogenes.<sup> </sup>Nonunique read alignment is handled in software by read distribution<sup> </sup>between multiple alignment positions or leaving alignment gaps.<sup> </sup>De novo assembly will reject these reads, leading to shorter<sup> </sup>and more numerous assembled contigs. These factors are relevant<sup> </sup>when choosing an appropriate sequencing platform with its associated<sup> </sup>read length, particularly for de novo assembly(<a href="#R9">9</a>).<sup> </sup></p><p>

Error rates for individual NGS reads are higher than for Sanger<sup> </sup>sequencing. The higher accuracy of Sanger sequencing reflects<sup> </sup>not only the maturity of the chemistry but also the fact that<sup> </sup>a Sanger trace peak represents highly redundant, multiple terminated<sup> </sup>extension reactions. Accuracy in NGS is achieved by sequencing<sup> </sup>a given region multiple times, enabled by the massively parallel<sup> </sup>process, with each sequence contributing to "coverage" depth.<sup> </sup>Through this process, a "consensus" sequence is derived. To<sup> </sup>assemble, align, and analyze NGS data requires an adequate number<sup> </sup>of overlapping reads, or coverage. In practice, coverage across<sup> </sup>a sequenced region is variable, and factors other than the Poisson-like<sup> </sup>randomness of library preparation that may contribute to this<sup> </sup>variability include differential ligation of adapters to template<sup> </sup>sequences and differential amplification during clonal template<sup> </sup>generation(<a href="#R11">11</a>)(<a href="#R70">70</a>). Beyond sequence errors, inadequate coverage<sup> </sup>can cause failure to detect actual nucleotide variation, leading<sup> </sup>to false-negative results for heterozygotes(<a href="#R3">3</a>)(<a href="#R11">11</a>). Studies<sup> </sup>have shown that coverages of less than 20- to 30-fold begin<sup> </sup>to reduce the accuracy of single-nucleotide polymorphism calls<sup> </sup>in data on the 454 platform(<a href="#R65">65</a>). For the Illumina system, higher<sup> </sup>coverage depths (50- to 60-fold) have been used in an effort<sup> </sup>to improve short-read alignment, assembly, and accuracy, although<sup> </sup>coverage in the 20- to 30-fold range may be sufficient for certain<sup> </sup>resequencing applications(<a href="#R14">14</a>). As noted above, one comparative<sup> </sup>study of a yeast genome showed that the 454, Illumina, and SOLiD<sup> </sup>technologies all accurately detected single-nucleotide variations<sup> </sup>when the coverage depth was <img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/ge.gif" alt="&#8805;" border="0">15-fold per allele(<a href="#R20">20</a>). Coverage<sup> </sup>gaps can occur when sequences are not aligned because of substantial<sup> </sup>variance from a reference. Alignment of repetitive sequences<sup> </sup>in repeat regions of a target sequence can also affect the apparent<sup> </sup>coverage. Reads that align equally well at multiple sites can<sup> </sup>be randomly distributed to the sites or in some cases discarded,<sup> </sup>depending on the alignment software. In de novo&#8211;assembly<sup> </sup>software, reads with ambiguous alignments are typically discarded,<sup> </sup>yielding multiple aligned read groups, or contigs, with no information<sup> </sup>regarding relative order.<sup> </sup></p><p>

A large variety of software programs for alignment and assembly<sup> </sup>have been developed and made available to the research community<sup> </sup>(see Table 1<a href="#T1"><img alt="Up" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-up.gif" border="1" height="7" width="8">
</a>
 in the online Data Supplement). Most use the Linux<sup> </sup>operating system, and a few are available for Windows. Many<sup> </sup>require a 64-bit operating system and can use <img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/ge.gif" alt="&#8805;" border="0">16 MB of RAM and<sup> </sup>multiple central-processing unit cores. The range of data volumes,<sup> </sup>hardware, software packages, and settings leads to processing<sup> </sup>times from a few minutes to multiple hours, emphasizing the<sup> </sup>need for sufficient computational power. Although a growing<sup> </sup>set of variations in alignment and assembly algorithms are available,<sup> </sup>there remains the trade-off between speed and accuracy in which<sup> </sup>many but not all possible alignments are evaluated, with a balance<sup> </sup>having to be struck between ideal alignment and computational<sup> </sup>efficiency.<sup> </sup></p><p>

NGS software features vary with the application and in general<sup> </sup>may include alignment, de novo assembly, alignment viewing,<sup> </sup>and variant-discovery programs. In addition some NGS statistical<sup> </sup>data-analysis tools are being developed (such as JMP Genomics;<sup> </sup>SAS Institute). Software packages available for alignment and<sup> </sup>assembly to a reference sequence include Zoom(<a href="#R71">71</a>), MAQ(<a href="#R67">67</a>),<sup> </sup>Mosaik(<a href="#R72">72</a>), SOAP(<a href="#R73">73</a>), and SHRiMP (<a href="http://compbio.cs.toronto.edu/shrimp/">http://compbio.cs.toronto.edu/shrimp/</a>),<sup> </sup>which supports SOLiD color-space analysis. Software for de novo<sup> </sup>assembly includes Edina(<a href="#R70">70</a>), EULER-SR(<a href="#R74">74</a>), SHARCGS(<a href="#R75">75</a>), SSAKE(<a href="#R69">69</a>),<sup> </sup>Velvet(<a href="#R76">76</a>), and SOAPdenovo (<a href="http://soap.genomics.org.cn/">http://soap.genomics.org.cn/</a>). Recently<sup> </sup>released commercial software for alignment and de novo assembly<sup> </sup>includes packages from DNAStar (<a href="http://www.dnastar.com/">www.dnastar.com</a>), SoftGenetics<sup> </sup>(<a href="http://www.softgenetics.com/">www.softgenetics.com</a>), and CLC bio (<a href="http://www.clcbio.com/">www.clcbio.com</a>) that feature<sup> </sup>data viewers that allow the user to see read alignments, coverage<sup> </sup>depth, genome annotations, and variant analysis. Fig. 5<a href="#F5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-down.gif" border="1" height="7" width="8"> </a>
 presents<sup> </sup>some examples of NGS data viewed in 2 different software systems.<sup> </sup></p><p>

<a name="F5"><!-- null --></a>
<br clear="all"></p><center><table cellpadding="0" cellspacing="0" width="95%"><tbody><tr bgcolor="#e1e1e1"><td><table cellpadding="2" cellspacing="2">
<tbody><tr bgcolor="#e1e1e1"><td align="center" bgcolor="#ffffff" valign="top">
<a href="http://www.clinchem.org/cgi/content/full/55/4/641/F5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/zcy0040992150005.gif" alt="Figure 5" border="2" height="200" hspace="10" vspace="5" width="111"></a><br>

<strong>View larger version</strong> (48K):<br>
<nobr><a href="http://www.clinchem.org/cgi/content/full/55/4/641/F5">[in this window]</a><br>
<a href="http://www.clinchem.org/cgi/content-nw/full/55/4/641/F5" onclick="startTarget('F5', 395, 640); this.href='/cgi/content-nw/full/55/4/641/F5'" onmouseover="window.status='View figure in a separate window'; return true" target="F5">[in a new window]</a><br>
<br>&nbsp;</nobr>
</td><td align="left" valign="top">
<b>Figure 5.</b> <b>Examples of NGS data viewed in 2 different software systems.</b><p>

(A), Roche Amplicon Variant Analyzer software displaying GS FLX data from the <i>CFTR</i>
gene [cystic fibrosis transmembrane conductance regulator (ATP-binding
cassette sub-family C, member 7)]. Lower pane shows reference sequence
(green) above 18 of 68 aligned reads. Column highlighted in yellow and
blue shows a heterozygous single-nucleotide polymorphism (SNP). Single
T/A insertions (red) may represent errors. Upper pane shows percent
variation from reference (vertical bars) and coverage (pale blue line).
(B), DNAStar SeqMan Pro software displaying Illumina data from <i>Mycobacterium massiliense</i>.
Lower pane shows reference sequence above aligned reads. Green and red
arrows show direction of sequencing; base calls at variance with
reference are indicated (red). Three columns in agreement (red)
indicate presumptive SNPs. Other bases in red may be errors. Upper pane
shows read coverage, with relative alignment positions above the graph.
See the Acknowledgments for disclosure information on the <i>CFTR</i>-gene analysis performed on residual, deidentified DNA.</p><p>


</p></td></tr></tbody></table>
</td></tr></tbody></table></center><p>


RNA-Seq data analysis poses unique challenges and requires sequence<sup> </sup>alignment across spliced regions of transcripts as well as poly(A)<sup> </sup>tails. Current software has made strong inroads, however, with<sup> </sup>incorporation of motif recognition at splice junctions and identification<sup> </sup>of intron&#8211;exon borders through regions of low alignment<sup> </sup>coverage(<a href="#R41">41</a>). Deciphering multiple transcript isoforms involves<sup> </sup>mapping reads to known and putative splicing junctions and,<sup> </sup>in one approach, requires that each isoform be supported by<sup> </sup>multiple independent splice-junction reads with independent<sup> </sup>start sites(<a href="#R51">51</a>). ERANGE software has been used in the analysis<sup> </sup>of the mouse transcriptome(<a href="#R43">43</a>). ERANGE maps unique reads to<sup> </sup>their genomic site of origin and maps reads that match to several<sup> </sup>sites, or multireads, to a most likely site of origin. Reads<sup> </sup>that do not map to a known exon are grouped together by homology<sup> </sup>into candidate exons or parts of exons. The near proportional<sup> </sup>nature of NGS transcriptome data allows quantification of RNA<sup> </sup>production from the coverage of the assembled or aligned data.<sup> </sup>ERANGE uses normalized counts of unique reads, spliced reads,<sup> </sup>and multireads to quantify transcripts. Additional analytical<sup> </sup>considerations are needed for miRNA studies, including RNA secondary-structure<sup> </sup>analysis for hairpins, alignment to known miRNA databases, and<sup> </sup>searches within the NGS data set for complementary miRNA strands,<sup> </sup>as described in studies of developing rice grains(<a href="#R77">77</a>) and chicken<sup> </sup>embryos(<a href="#R78">78</a>).<sup> </sup></p><p>

Research with ChIP-Seq has led to analysis methods and software<sup> </sup>that exploit the advantages over ChIP-in-chip, namely a larger,<sup> </sup>more information-rich data set. The single-base resolution of<sup> </sup>the data allows improved estimation of binding-site positions<sup> </sup>in the programs QuEST(<a href="#R79">79</a>) and MACS(<a href="#R80">80</a>). Aligned data at the<sup> </sup>protein-binding regions typically have 2 characteristic offset<sup> </sup>peaks, each of which is populated by only forward or only reverse<sup> </sup>reads. These peaks are hallmarks of the immunoprecipitated short<sup> </sup>ChIP-Seq DNA fragments with a binding site near the center and<sup> </sup>are used by the software to estimate binding-site location near<sup> </sup>the mean peak position. Additional program features include<sup> </sup>advancements in statistical analysis to minimize miscalled binding<sup> </sup>sites, error probability estimation, and motif analysis (see<sup> </sup>Table 1<a href="#T1"><img alt="Up" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-up.gif" border="1" height="7" width="8">
</a>
 in the online Data Supplement).<sup> </sup></p><p>

<a name="SEC4"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
A Clinical Future for NGS
</font></th></tr></tbody></table>
<table align="right" border="1" cellpadding="5"><tbody><tr><th align="left"><font size="-1">
<a href="#top"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Top<br></a>
<a href="#ABS"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Abstract<br></a>
<a href="#BDY"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Introduction<br></a>
<a href="#SEC1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Fundamentals of NGS Platforms<br></a>
<a href="#SEC2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">The Impact of NGS...<br></a>
<a href="#SEC3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">NGS Data Analysis<br></a>
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/dot.gif" border="0" height="9" hspace="5" width="11"><font color="#464c53">A Clinical Future for...</font><br>
<a href="#SEC5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Technologies on the Horizon<br></a>
<a href="#SEC6"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Conclusions<br></a>
<a href="#BIBL"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">References<br></a>
</font></th></tr></tbody></table>
&nbsp;<br>

From the impact that NGS has made at the basic-research level,<sup> </sup>we can anticipate its translation into molecular diagnostics.<sup> </sup>Key issues that will need to be addressed in this transition<sup> </sup>will include complexity of technical procedures, robustness,<sup> </sup>accuracy, and cost. By all these measures, NGS platforms will<sup> </sup>benefit from continued process streamlining, automation, chemistry<sup> </sup>refinements, cost reductions, and improved data handling. The<sup> </sup>cost of NGS is currently substantial in terms of the investment<sup> </sup>in capital equipment (from approximately $600 000 for the Roche/454<sup> </sup>Life Science, Illumina, and Applied Biosystems SOLiD platforms<sup> </sup>to $1.35 million for the HeliScope platform) and costs of sequencing<sup> </sup>reagents (from approximately $3500&#8211;$4500 for the Illumina,<sup> </sup>Applied Biosystems, and Roche/454 platforms to $18 000 for the<sup> </sup>HeliScope platform). Nonetheless, the cost per base is substantially<sup> </sup>lower than for Sanger sequencing, and combined with the tremendous<sup> </sup>output, it is straightforward to see why genome centers, core<sup> </sup>facilities, and commercial contract-sequencing enterprises have<sup> </sup>readily adopted this new technology. Work flow considerations<sup> </sup>include the fact that preparation of a sample library requires<sup> </sup>multiple molecular biology steps and 2&#8211;4 days to complete,<sup> </sup>depending on the platform. In addition to the required molecular<sup> </sup>biology expertise, data analysis requires expertise in bioinformatics<sup> </sup>facilitated by a knowledge of Linux operating systems. Leveraging<sup> </sup>the high-throughput capacity of NGS platforms can be facilitated<sup> </sup>by analyzing multiple samples with separate flow-cell lanes<sup> </sup>or compartments. In addition, unique identifier sequences or<sup> </sup>"bar codes" can be ligated to individual samples, which can<sup> </sup>subsequently be pooled and sequenced. After sequencing, sequences<sup> </sup>of individual samples are derived by data deconvolution(<a href="#R81">81</a>)(<a href="#R82">82</a>)(<a href="#R83">83</a>).<sup> </sup></p><p>

The transition of NGS into clinical diagnostics is in the early<sup> </sup>stages of development in large reference laboratories and is<sup> </sup>being leveraged for applications that require large amounts<sup> </sup>of sequence information, relative quantification, and high-sensitivity<sup> </sup>detection. Examples that meet these criteria include the aforementioned<sup> </sup>detection of mutations in tumor cells from biopsies or in the<sup> </sup>circulation. In the area of mitochondrial disorders, NGS can<sup> </sup>be used to sequence the entire 16.5-kb mitochondrial genome,<sup> </sup>determine mutation heteroplasmy percentage, and analyze nuclear<sup> </sup>genes whose protein products affect mitochondrial metabolism&#8212;all<sup> </sup>in a single analytical run. In the authors&#8217; laboratory,<sup> </sup>sequencing of mycobacterial genomes is ongoing as an approach<sup> </sup>to refine organism identification and support clinical epidemiologic<sup> </sup>investigations. HIV quasi-species detection and relative quantification<sup> </sup>have been demonstrated and can be used to monitor emerging drug<sup> </sup>resistance(<a href="#R84">84</a>). For human genetics, there is an increasing need<sup> </sup>to analyze multiple genes that, when mutated, lead to overlapping<sup> </sup>physical findings and clinical phenotypes. For example, 16 different<sup> </sup>genes are implicated in the pathogenesis of hypertrophic cardiomyopathy(<a href="#R85">85</a>)(<a href="#R86">86</a>).<sup> </sup>For a comprehensive diagnostic evaluation in such settings,<sup> </sup>it will be necessary to sequence upwards of 100 000 to 200 000<sup> </sup>bp. The coupling of NGS with the genomic-enrichment techniques<sup> </sup>described above offers a promising approach to this technical<sup> </sup>challenge.<sup> </sup></p><p>

Recently, investigative groups led by Y.M. Dennis Lo and Stephen<sup> </sup>Quake have applied NGS to the detection of fetal chromosomal<sup> </sup>aneuploidy(<a href="#R87">87</a>)(<a href="#R88">88</a>). Prior work had demonstrated that cell-free<sup> </sup>fetal nucleic acids (DNA and RNA) are present in maternal blood<sup> </sup>during pregnancy, along with maternally derived cell-free nucleic<sup> </sup>acids. Several analytical approaches that use cell-free fetal<sup> </sup>nucleic acids have been developed to determine fetal aneuploidy,<sup> </sup>including the analysis of placental mRNA derived from the chromosomes<sup> </sup>of interest (e.g., chromosome 21) and the determination of relative<sup> </sup>chromosomal dosage via digital PCR analysis of a large number<sup> </sup>of target chromosomal loci compared with reference chromosomal<sup> </sup>loci(<a href="#R89">89</a>)(<a href="#R90">90</a>)(<a href="#R91">91</a>). Building on the concept of relative chromosome<sup> </sup>dosage, the Lo and Quake groups have independently shown the<sup> </sup>feasibility of converting cell-free DNA from maternal blood<sup> </sup>into an Illumina library, followed by sequencing and mapping<sup> </sup>the reads to the reference human genome. Counting the number<sup> </sup>of reads that map to each chromosome allows the relative dosage<sup> </sup>of each chromosome to be ascertained. If fetal aneuploidy is<sup> </sup>present, the number of sequence reads mapping to the affected<sup> </sup>chromosome would be expected to be statistically overrepresented<sup> </sup>in the data set. This expectation was confirmed in trisomy 21<sup> </sup>pregnancies, with additional supporting evidence obtained for<sup> </sup>trisomy 18 and 13 pregnancies. These studies open a new avenue<sup> </sup>for assessing fetal aneuploidy and provide a foundation for<sup> </sup>NGS-based analysis of cell-free DNA in both nonpathologic and<sup> </sup>pathophysiological states.<sup> </sup></p><p>

<a name="SEC5"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
Technologies on the Horizon
</font></th></tr></tbody></table>
<table align="right" border="1" cellpadding="5"><tbody><tr><th align="left"><font size="-1">
<a href="#top"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Top<br></a>
<a href="#ABS"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Abstract<br></a>
<a href="#BDY"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Introduction<br></a>
<a href="#SEC1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Fundamentals of NGS Platforms<br></a>
<a href="#SEC2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">The Impact of NGS...<br></a>
<a href="#SEC3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">NGS Data Analysis<br></a>
<a href="#SEC4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">A Clinical Future for...<br></a>
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/dot.gif" border="0" height="9" hspace="5" width="11"><font color="#464c53">Technologies on the Horizon</font><br>
<a href="#SEC6"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">Conclusions<br></a>
<a href="#BIBL"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">References<br></a>
</font></th></tr></tbody></table>
&nbsp;<br>

New single-molecule sequencing technologies in development may<sup> </sup>decrease sequencing time, reduce costs, and streamline sample<sup> </sup>preparation. Real-time sequencing by synthesis is being developed<sup> </sup>by VisiGen (<a href="http://www.visigenbio.com/">http://www.visigenbio.com</a>) and Pacific Biosciences<sup> </sup>(<a href="http://www.pacificbiosciences.com/">http://www.pacificbiosciences.com</a>). VisiGen&#8217;s approach<sup> </sup>uses DNA polymerase modified with a fluorescent donor molecule.<sup> </sup>Attached to a glass slide surface, the polymerase directs strand<sup> </sup>extension from primed DNA templates. Nucleotides are modified<sup> </sup>with fluorescent acceptor molecules, and light energy is used<sup> </sup>during incorporation to invoke fluorescence resonance energy<sup> </sup>transfer between polymerase and nucleotide fluorescent moieties,<sup> </sup>the latter being in the <img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/ggr.gif" alt="{gamma}" border="0">-phosphate position and cleaved away<sup> </sup>during incorporation. The company envisions its platform will<sup> </sup>consist of a massively parallel array of tethered DNA polymerases<sup> </sup>that will generate 1 <font face="arial,helvetica">x</font> 10<sup>6</sup> bp of sequence per second.<sup> </sup></p><p>

Pacific Biosciences performs single-molecule real-time sequencing<sup> </sup>and uses phospholinked fluorescently labeled dNTPs. DNA sequencing<sup> </sup>is performed in thousands of reaction wells 50&#8211;100 nm<sup> </sup>in diameter that are fabricated with a thin metal cladding film<sup> </sup>deposited on an optical waveguide consisting of a solid, transparent<sup> </sup>silicon dioxide substrate. Each reaction well is a nanophotonic<sup> </sup>chamber in which only the bottom third is visualized, producing<sup> </sup>a detection volume of approximately 20 zL (20 <font face="arial,helvetica">x</font> 10<sup>&#8211;21</sup><sup> </sup>L). DNA polymerase/template complexes are immobilized to the<sup> </sup>well bottoms, and 4 differently labeled dNTPs are added. As<sup> </sup>the DNA polymerase incorporates complementary nucleotides, each<sup> </sup>base is held within the detection volume for tens of milliseconds,<sup> </sup>orders of magnitude longer than the amount of time it takes<sup> </sup>for a nucleotide to diffuse in and out of the detection volume.<sup> </sup>Laser excitation enables the incorporation events in individual<sup> </sup>wells to be captured through the optical waveguide, with the<sup> </sup>fluorescent color detected reflecting the identity of the dNTP<sup> </sup>incorporated. For sequencing, Pacific Biosciences uses a modified<sup> </sup>phi29 DNA polymerase that has enhanced kinetic properties for<sup> </sup>incorporating the system&#8217;s phospholinked fluorescently<sup> </sup>labeled dNTPs. In addition, phi29 DNA polymerase is highly processive,<sup> </sup>with strand-displacement activity. By taking advantage of these<sup> </sup>properties, Pacific Biosciences has demonstrated sequencing<sup> </sup>reads exceeding 4000 bases when a circularized single-stranded<sup> </sup>DNA molecule is used as template. In this configuration, the<sup> </sup>phi29 DNA polymerase carries out multiple laps of DNA strand-displacement<sup> </sup>synthesis around the circular template. The mean DNA-synthesis<sup> </sup>rate was determined to be approximately 4 bases/s. The observed<sup> </sup>errors, including deletions, insertions, and mismatches can<sup> </sup>be addressed by developing a consensus sequence read derived<sup> </sup>from the multiple rounds of template sequencing. Further refinement<sup> </sup>of the chemistries and platform instrumentation are ongoing,<sup> </sup>with a 2010 target date for commercial launch(<a href="#R92">92</a>)(<a href="#R93">93</a>)(<a href="#R94">94</a>).<sup> </sup></p><p>

Farther out toward the horizon is sequencing based on monitoring<sup> </sup>the passage of DNA molecules through nanopores 2&#8211;5 nm<sup> </sup>or greater in diameter. Nanopores are being fabricated in inorganic<sup> </sup>membranes (solid-state nanopores), assembled from protein channels<sup> </sup>in lipid membranes, or configured in polymer-based nanofluidic<sup> </sup>channels. In some configurations, current is applied across<sup> </sup>nanopore membranes to drive the translocation of negatively<sup> </sup>charged DNA molecules through pores while monitoring changes<sup> </sup>in membrane electrical conductance measured in the picoampere<sup> </sup>range. NABsys (<a href="http://www.nabsys.com/">http://www.nabsys.com</a>) is pursuing a combination<sup> </sup>of nanopores with sequencing by hybridization in which single-stranded<sup> </sup>DNA molecules are hybridized with a library of hexamers of known<sup> </sup>sequence. The hybridized DNA is interrogated through a nanopore,<sup> </sup>with the current changes being different in regions of hexamer<sup> </sup>hybridization. The patterns of hybridization are used to map<sup> </sup>annealing regions and determine sequence. Oxford Nanopore Technologies<sup> </sup>(<a href="http://www.nanoporetech.com/">http://www.nanoporetech.com</a>) is developing nanopore-based sequencing<sup> </sup>that uses an <img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/agr.gif" alt="{alpha}" border="0">-hemolysin protein channel in reconstituted lipid<sup> </sup>bilayers. The nanopores are situated in individual array wells,<sup> </sup>and single DNA molecules are introduced into the wells and progressively<sup> </sup>digested by exonuclease. The released single-nucleotide bases<sup> </sup>enter the nanopore and alter the electrical current, creating<sup> </sup>a characteristic current change for each individual base(<a href="#R95">95</a>)(<a href="#R96">96</a>).<sup> </sup>Although the technology is seemingly futuristic, considerable<sup> </sup>NHGRI funding is being directed toward a variety of nanopore<sup> </sup>technologies under development as part of the goal of achieving<sup> </sup>the $1000 genome. For further descriptions of nanopore technologies,<sup> </sup>the reader is referred to recent reviews(<a href="#R97">97</a>)(<a href="#R98">98</a>).<sup> </sup></p><p>

<a name="SEC6"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
Conclusions
</font></th></tr></tbody></table>
<table align="right" border="1" cellpadding="5"><tbody><tr><th align="left"><font size="-1">
<a href="#top"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Top<br></a>
<a href="#ABS"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Abstract<br></a>
<a href="#BDY"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Introduction<br></a>
<a href="#SEC1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Fundamentals of NGS Platforms<br></a>
<a href="#SEC2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">The Impact of NGS...<br></a>
<a href="#SEC3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">NGS Data Analysis<br></a>
<a href="#SEC4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">A Clinical Future for...<br></a>
<a href="#SEC5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Technologies on the Horizon<br></a>
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/dot.gif" border="0" height="9" hspace="5" width="11"><font color="#464c53">Conclusions</font><br>
<a href="#BIBL"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/darrow.gif" border="0" height="9" hspace="5" width="11">References<br></a>
</font></th></tr></tbody></table>
&nbsp;<br>

The past few years have witnessed the emergence of NGS technologies<sup> </sup>that share a common basis, massively parallel sequencing of<sup> </sup>clonally amplified DNA molecules. In 2008, the first NGS platform<sup> </sup>based on single-molecule DNA sequencing was launched. On the<sup> </sup>horizon are real-time single-molecule DNA-sequencing technologies<sup> </sup>and approaches based on nanopores. NGS has had a substantial<sup> </sup>impact on basic genomics research in terms of scale and feasibility.<sup> </sup>Over the next several years, NGS is anticipated to transition<sup> </sup>into clinical-diagnostics use. Essential elements to make this<sup> </sup>transition successful will be the requirement of streamlining<sup> </sup>the processes, especially sample preparation, coupled with improvements<sup> </sup>in technology robustness and characterization of accuracy through<sup> </sup>validation studies. The large amounts of sequence-data output<sup> </sup>will pose a bioinformatics challenge for the clinical laboratory.<sup> </sup>In addition to data processing, the interpretation of sequencing<sup> </sup>results will require further characterization of the genomic<sup> </sup>variation present in the regions analyzed. Although considerable<sup> </sup>work lies ahead to implement NGS into clinical diagnostics,<sup> </sup>the potential applications are exciting and numerous.<sup> </sup></p><p>

<sup> </sup></p><p>



<a name="ACK"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
Acknowledgments
</font></th></tr></tbody></table>
&nbsp;<br>

<b>Author Contributions:</b> <i>All authors confirmed they have contributed<sup> </sup>to the intellectual content of this paper and have met the following<sup> </sup>3 requirements: (a) significant contributions to the conception<sup> </sup>and design, acquisition of data, or analysis and interpretation<sup> </sup>of data; (b) drafting or revising the article for intellectual<sup> </sup>content; and (c) final approval of the published article.</i><sup> </sup></p><p>

<b>Authors&#8217; Disclosures of Potential Conflicts of Interest:</b><sup> </sup><i>No authors declared any potential conflicts of interest.</i><sup> </sup></p><p>

<b>Role of Sponsor:</b> The funding organizations played a direct role<sup> </sup>in the preparation of the manuscript and in the final approval<sup> </sup>of the manuscript.<sup> </sup></p><p>

<b>Acknowledgments:</b> The analysis of the <i>CFTR</i> gene illustrated in<sup> </sup>Fig. 5<a href="#F5"><img alt="Up" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/fig-up.gif" border="1" height="7" width="8">
</a>
 was performed on residual, deidentified DNA under the<sup> </sup>approval of University of Utah Institutional Review Board, human<sup> </sup>subjects protocol number 7275.<sup> </sup></p><p>


<a name="FN"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
Footnotes
</font></th></tr></tbody></table>
&nbsp;<br>

<a name="FN1"><!-- null --></a>
<sup>1</sup> Nonstandard abbreviations: NGS, next-generation sequencing;<sup> </sup>NHGRI, National Human Genome Research Institute; dNTP, deoxynucleoside<sup> </sup>triphosphate; Mb, million base pairs; Gb, billion base pairs;<sup> </sup>miRNA, microRNA.<sup> </sup><a href="#RFN1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/back.gif" border="0" height="12" width="12"></a></p><p>
<a name="FN2"><!-- null --></a>
<sup>2</sup> S.A. Dames and J.D. Durtschi contributed equally to the review.<sup> </sup><a href="#RFN2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/back.gif" border="0" height="12" width="12"></a></p><p>




<a name="BIBL"><!-- null --></a>
<br clear="right"><table bgcolor="#e1e1e1" cellpadding="0" cellspacing="0" width="100%">
 <tbody><tr><td align="left" bgcolor="#ffffff" valign="middle" width="5%"><img alt="" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/rarrow.gif" height="21" hspace="5" width="10"></td>
 <th align="left" valign="middle" width="95%"><font size="+2">&nbsp;&nbsp;
References
</font></th></tr></tbody></table>
<table align="right" border="1" cellpadding="5"><tbody><tr><th align="left"><font size="-1">
<a href="#top"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Top<br></a>
<a href="#ABS"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Abstract<br></a>
<a href="#BDY"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Introduction<br></a>
<a href="#SEC1"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Fundamentals of NGS Platforms<br></a>
<a href="#SEC2"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">The Impact of NGS...<br></a>
<a href="#SEC3"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">NGS Data Analysis<br></a>
<a href="#SEC4"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">A Clinical Future for...<br></a>
<a href="#SEC5"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Technologies on the Horizon<br></a>
<a href="#SEC6"><img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/uarrow.gif" border="0" height="9" hspace="5" width="11">Conclusions<br></a>
<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/dot.gif" border="0" height="9" hspace="5" width="11"><font color="#464c53">References</font><br>
</font></th></tr></tbody></table>
&nbsp;<br>

</p><ol compact="compact">
<a name="R1"><!-- null --></a><li value="1"> Maxam AM, Gilbert W. A new method for sequencing DNA. Proc Natl Acad Sci U S A 1977;74:560-564.<!-- HIGHWIRE ID="55:4:641:1" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=74/2/560"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R2"><!-- null --></a></li><li value="2"> Sanger F, Nicklen S, Coulson AR. DNA sequencing with chain-terminating inhibitors. Proc Natl Acad Sci U S A 1977;74:5463-5467.<!-- HIGHWIRE ID="55:4:641:2" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=74/12/5463"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R3"><!-- null --></a></li><li value="3">
Wheeler DA, Srinivasan M, Egholm M, Shen Y, Chen L, McGuire A, et al.
The complete genome of an individual by massively parallel DNA
sequencing. Nature 2008;452:872-876.<!-- HIGHWIRE ID="55:4:641:3" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnature06884&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000255026000048&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18421352&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18421352&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R4"><!-- null --></a></li><li value="4">
Nyren P, Pettersson B, Uhlen M. Solid phase DNA minisequencing by an
enzymatic luminometric inorganic pyrophosphate detection assay. Anal
Biochem 1993;208:171-175.<!-- HIGHWIRE ID="55:4:641:4" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1006%2Fabio.1993.1024&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=A1993KF51800024&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=8382019&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=8382019&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R5"><!-- null --></a></li><li value="5">
Ronaghi M, Karamohamed S, Pettersson B, Uhlen M, Nyren P. Real-time DNA
sequencing using detection of pyrophosphate release. Anal Biochem
1996;242:84-89.<!-- HIGHWIRE ID="55:4:641:5" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1006%2Fabio.1996.0432&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=A1996VT72900014&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=8923969&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=8923969&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R6"><!-- null --></a></li><li value="6"> Ronaghi M, Uhlen M, Nyren P. A sequencing method based on real-time pyrophosphate. Science 1998;281:363-365.<!-- HIGHWIRE ID="55:4:641:6" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=FULL&amp;journalCode=sci&amp;resid=281/5375/363"><nobr>[<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R7"><!-- null --></a></li><li value="7"> Tawfik DS, Griffiths AD. Man-made cell-like compartments for molecular evolution. Nat Biotechnol 1998;16:652-656.<!-- HIGHWIRE ID="55:4:641:7" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnbt0798-652&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000074521800029&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=9661199&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=9661199&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R8"><!-- null --></a></li><li value="8">
Margulies M, Egholm M, Altman WE, Attiya S, Bader JS, Bemben LA, et al.
Genome sequencing in microfabricated high-density picolitre reactors.
Nature 2005;437:376-380.<!-- HIGHWIRE ID="55:4:641:8" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=16056220&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=16056220&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R9"><!-- null --></a></li><li value="9"> Pearson BM, Gaskin DJ, Segers RP, Wells JM, Nuijten PJ, van Vliet AH. The complete genome sequence of <i>Campylobacter jejuni</i> strain 81116 (NCTC11828). J Bacteriol 2007;189:8402-8403.<!-- HIGHWIRE ID="55:4:641:9" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=jb&amp;resid=189/22/8402"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R10"><!-- null --></a></li><li value="10">
Huse SM, Huber JA, Morrison HG, Sogin ML, Welch DM. Accuracy and
quality of massively parallel DNA pyrosequencing. Genome Biol
2007;8:R143.<!-- HIGHWIRE ID="55:4:641:10" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1186%2Fgb-2007-8-7-r143&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17659080&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17659080&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R11"><!-- null --></a></li><li value="11">
Bentley DR, Balasubramanian S, Swerdlow HP, Smith GP, Milton J, Brown
CG, et al. Accurate whole human genome sequencing using reversible
terminator chemistry. Nature 2008;456:53-59.<!-- HIGHWIRE ID="55:4:641:11" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnature07517&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000260674000039&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18987734&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18987734&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R12"><!-- null --></a></li><li value="12">
Korbel JO, Urban AE, Affourtit JP, Godwin B, Grubert F, Simons JF, et
al. Paired-end mapping reveals extensive structural variation in the
human genome. Science 2007;318:420-426.<!-- HIGHWIRE ID="55:4:641:12" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=318/5849/420"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R13"><!-- null --></a></li><li value="13">
Campbell PJ, Stephens PJ, Pleasance ED, O'Meara S, Li H, Santarius T,
et al. Identification of somatically acquired rearrangements in cancer
using genome-wide massively parallel paired-end sequencing. Nat Genet
2008;40:722-729.<!-- HIGHWIRE ID="55:4:641:13" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fng.128&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000256212000014&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18438408&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18438408&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R14"><!-- null --></a></li><li value="14">
Dohm JC, Lottaz C, Borodina T, Himmelbauer H. Substantial biases in
ultra-short read data sets from high-throughput DNA sequencing. Nucleic
Acids Res 2008;36:e105.<!-- HIGHWIRE ID="55:4:641:14" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=36/16/e105"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R15"><!-- null --></a></li><li value="15">
Erlich Y, Mitra PP, delaBastide M, McCombie WR, Hannon GJ. Alta-Cyclic:
a self-optimizing base caller for next-generation sequencing. Nat
Methods 2008;5:679-682.<!-- HIGHWIRE ID="55:4:641:15" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnmeth.1230&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000258077400010&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18604217&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18604217&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R16"><!-- null --></a></li><li value="16">
Quail MA, Kozarewa I, Smith F, Scally A, Stephens PJ, Durbin R, et al.
A large genome center&#8217;s improvements to the Illumina sequencing system.
Nat Methods 2008;5:1005-1010.<!-- HIGHWIRE ID="55:4:641:16" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnmeth.1270&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000261212700012&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=19034268&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=19034268&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R17"><!-- null --></a></li><li value="17">
Shendure J, Porreca GJ, Reppas NB, Lin X, McCutcheon JP, Rosenbaum AM,
et al. Accurate multiplex polony sequencing of an evolved bacterial
genome. Science 2005;309:1728-1732.<!-- HIGHWIRE ID="55:4:641:17" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=309/5741/1728"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R18"><!-- null --></a></li><li value="18">
Braslavsky I, Hebert B, Kartalov E, Quake SR. Sequence information can
be obtained from single DNA molecules. Proc Natl Acad Sci U S A
2003;100:3960-3964.<!-- HIGHWIRE ID="55:4:641:18" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=100/7/3960"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R19"><!-- null --></a></li><li value="19">
Harris TD, Buzby PR, Babcock H, Beer E, Bowers J, Braslavsky I, et al.
Single-molecule DNA sequencing of a viral genome. Science
2008;320:106-109.<!-- HIGHWIRE ID="55:4:641:19" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=320/5872/106"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R20"><!-- null --></a></li><li value="20">
Smith DR, Quinlan AR, Peckham HE, Makowsky K, Tao W, Woolf B, et al.
Rapid whole-genome mutational profiling using next-generation
sequencing technologies. Genome Res 2008;18:1638-1642.<!-- HIGHWIRE ID="55:4:641:20" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/10/1638"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R21"><!-- null --></a></li><li value="21">
Quinn NL, Levenkova N, Chow W, Bouffard P, Boroevich KA, Knight JR, et
al. Assessing the feasibility of GS FLX Pyrosequencing for sequencing
the Atlantic salmon genome. BMC Genomics 2008;9:404.<!-- HIGHWIRE ID="55:4:641:21" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1186%2F1471-2164-9-404&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18755037&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18755037&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R22"><!-- null --></a></li><li value="22">
Satkoski JA, Malhi R, Kanthaswamy S, Tito R, Malladi V, Smith D.
Pyrosequencing as a method for SNP identification in the rhesus macaque
<i>(Macaca mulatta)</i>. BMC Genomics 2008;9:256.<!-- HIGHWIRE ID="55:4:641:22" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1186%2F1471-2164-9-256&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18510772&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18510772&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R23"><!-- null --></a></li><li value="23"> Borneman AR, Forgan AH, Pretorius IS, Chambers PJ. Comparative genome analysis of a <i>Saccharomyces cerevisiae</i> wine strain. FEMS Yeast Res 2008;8:1185-1195.<!-- HIGHWIRE ID="55:4:641:23" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1111%2Fj.1567-1364.2008.00434.x&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000260114400021&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18778279&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18778279&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R24"><!-- null --></a></li><li value="24">
Wang J, Wang W, Li R, Li Y, Tian G, Goodman L, et al. The diploid
genome sequence of an Asian individual. Nature 2008;456:60-65.<!-- HIGHWIRE ID="55:4:641:24" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnature07484&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000260674000040&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18987735&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18987735&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R25"><!-- null --></a></li><li value="25">
Ley TJ, Mardis ER, Ding L, Fulton B, McLellan MD, Chen K, et al. DNA
sequencing of a cytogenetically normal acute myeloid leukaemia genome.
Nature 2008;456:66-72.<!-- HIGHWIRE ID="55:4:641:25" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnature07485&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000260674000041&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18987736&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18987736&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R26"><!-- null --></a></li><li value="26">
Kim PM, Lam HY, Urban AE, Korbel JO, Affourtit J, Grubert F, et al.
Analysis of copy number variants and segmental duplications in the
human genome: evidence for a change in the process of formation in
recent evolutionary history. Genome Res 2008;18:1865-1874.<!-- HIGHWIRE ID="55:4:641:26" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/12/1865"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R27"><!-- null --></a></li><li value="27">
Chen J, Kim YC, Jung YC, Xuan Z, Dworkin G, Zhang Y, et al. Scanning
the human genome at kilobase resolution. Genome Res 2008;18:751-762.<!-- HIGHWIRE ID="55:4:641:27" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/5/751"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R28"><!-- null --></a></li><li value="28">
Yeager M, Xiao N, Hayes RB, Bouffard P, Desany B, Burdett L, et al.
Comprehensive resequence analysis of a 136 kb region of human
chromosome 8q24 associated with prostate and colon cancers. Hum Genet
2008;124:161-170.<!-- HIGHWIRE ID="55:4:641:28" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1007%2Fs00439-008-0535-3&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000258759300006&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18704501&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18704501&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R29"><!-- null --></a></li><li value="29">
Ding L, Getz G, Wheeler DA, Mardis ER, McLellan MD, Cibulskis K, et al.
Somatic mutations affect key pathways in lung adenocarcinoma. Nature
2008;455:1069-1075.<!-- HIGHWIRE ID="55:4:641:29" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnature07423&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000260252600036&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18948947&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18948947&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R30"><!-- null --></a></li><li value="30">
Albert TJ, Molla MN, Muzny DM, Nazareth L, Wheeler D, Song X, et al.
Direct selection of human genomic loci by microarray hybridization. Nat
Methods 2007;4:903-905.<!-- HIGHWIRE ID="55:4:641:30" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnmeth1111&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000250575700011&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17934467&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17934467&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R31"><!-- null --></a></li><li value="31">
Hodges E, Xuan Z, Balija V, Kramer M, Molla MN, Smith SW, et al.
Genome-wide in situ exon capture for selective resequencing. Nat Genet
2007;39:1522-1527.<!-- HIGHWIRE ID="55:4:641:31" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fng.2007.42&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000251267400023&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17982454&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17982454&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R32"><!-- null --></a></li><li value="32">
Okou DT, Steinberg KM, Middle C, Cutler DJ, Albert TJ, Zwick ME.
Microarray-based genomic selection for high-throughput resequencing.
Nat Methods 2007;4:907-909.<!-- HIGHWIRE ID="55:4:641:32" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnmeth1109&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000250575700012&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17934469&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17934469&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R33"><!-- null --></a></li><li value="33">
Porreca GJ, Zhang K, Li JB, Xie B, Austin D, Vassallo SL, et al.
Multiplex amplification of large sets of human exons. Nat Methods
2007;4:931-936.<!-- HIGHWIRE ID="55:4:641:33" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnmeth1110&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000250575700018&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17934468&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17934468&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R34"><!-- null --></a></li><li value="34">
Huber JA, Mark Welch DB, Morrison HG, Huse SM, Neal PR, Butterfield DA,
Sogin ML. Microbial population structures in the deep marine biosphere.
Science 2007;318:97-100.<!-- HIGHWIRE ID="55:4:641:34" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=318/5847/97"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R35"><!-- null --></a></li><li value="35">
Sogin ML, Morrison HG, Huber JA, Mark Welch D, Huse SM, Neal PR, et al.
Microbial diversity in the deep sea and the underexplored "rare
biosphere.". Proc Natl Acad Sci U S A 2006;103:12115-12120.<!-- HIGHWIRE ID="55:4:641:35" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=103/32/12115"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R36"><!-- null --></a></li><li value="36">
Urich T, Lanzen A, Qi J, Huson DH, Schleper C, Schuster SC.
Simultaneous assessment of soil microbial community structure and
function through analysis of the meta-transcriptome. PLoS ONE
2008;3:e2527.<!-- HIGHWIRE ID="55:4:641:36" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1371%2Fjournal.pone.0002527&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18575584&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18575584&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R37"><!-- null --></a></li><li value="37">
Palacios G, Druce J, Du L, Tran T, Birch C, Briese T, et al. A new
arenavirus in a cluster of fatal transplant-associated diseases. N Engl
J Med 2008;358:991-998.<!-- HIGHWIRE ID="55:4:641:37" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=nejm&amp;resid=358/10/991"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R38"><!-- null --></a></li><li value="38">
Keijser BJ, Zaura E, Huse SM, van der Vossen JM, Schuren FH, Montijn
RC, et al. Pyrosequencing analysis of the oral microflora of healthy
adults. J Dent Res 2008;87:1016-1020.<!-- HIGHWIRE ID="55:4:641:38" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=spjdr&amp;resid=87/11/1016"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R39"><!-- null --></a></li><li value="39">
Turnbaugh PJ, Hamady M, Yatsunenko T, Cantarel BL, Duncan A, Ley RE, et
al. A core gut microbiome in obese and lean twins. Nature
2008;457:480-484.<!-- HIGHWIRE ID="55:4:641:39" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnature07540&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000262519200047&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=19043404&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=19043404&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R40"><!-- null --></a></li><li value="40"> Wang Z, Gerstein M, Snyder M. RNA-Seq: a revolutionary tool for transcriptomics. Nat Rev Genet 2009;10:57-63.<!-- HIGHWIRE ID="55:4:641:40" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnrg2484&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000261866500012&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=19015660&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=19015660&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R41"><!-- null --></a></li><li value="41">
Nagalakshmi U, Wang Z, Waern K, Shou C, Raha D, Gerstein M, Snyder M.
The transcriptional landscape of the yeast genome defined by RNA
sequencing. Science 2008;320:1344-1349.<!-- HIGHWIRE ID="55:4:641:41" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=320/5881/1344"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R42"><!-- null --></a></li><li value="42">
Wilhelm BT, Marguerat S, Watt S, Schubert F, Wood V, Goodhead I, et al.
Dynamic repertoire of a eukaryotic transcriptome surveyed at
single-nucleotide resolution. Nature 2008;453:1239-1243.<!-- HIGHWIRE ID="55:4:641:42" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnature07002&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000257075800043&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18488015&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18488015&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R43"><!-- null --></a></li><li value="43">
Mortazavi A, Williams BA, McCue K, Schaeffer L, Wold B. Mapping and
quantifying mammalian transcriptomes by RNA-Seq. Nat Methods
2008;5:621-628.<!-- HIGHWIRE ID="55:4:641:43" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnmeth.1226&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000257166700015&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18516045&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18516045&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R44"><!-- null --></a></li><li value="44">
Lister R, O'Malley RC, Tonti-Filippini J, Gregory BD, Berry CC, Millar
AH, Ecker JR. Highly integrated single-base resolution maps of the
epigenome in <i>Arabidopsis</i>. Cell 2008;133:523-536.<!-- HIGHWIRE ID="55:4:641:44" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1016%2Fj.cell.2008.03.029&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000255503900022&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18423832&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18423832&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R45"><!-- null --></a></li><li value="45">
Cloonan N, Forrest AR, Kolle G, Gardiner BB, Faulkner GJ, Brown MK, et
al. Stem cell transcriptome profiling via massive-scale mRNA
sequencing. Nat Methods 2008;5:613-619.<!-- HIGHWIRE ID="55:4:641:45" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnmeth.1223&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000257166700014&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18516046&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18516046&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R46"><!-- null --></a></li><li value="46">
Marioni JC, Mason CE, Mane SM, Stephens M, Gilad Y. RNA-seq: an
assessment of technical reproducibility and comparison with gene
expression arrays. Genome Res 2008;18:1509-1517.<!-- HIGHWIRE ID="55:4:641:46" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/9/1509"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R47"><!-- null --></a></li><li value="47">
Morin R, Bainbridge M, Fejes A, Hirst M, Krzywinski M, Pugh T, et al.
Profiling the HeLa S3 transcriptome using randomly primed cDNA and
massively parallel short-read sequencing. Biotechniques 2008;45:81-94.<!-- HIGHWIRE ID="55:4:641:47" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.2144%2F000112900&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18611170&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18611170&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R48"><!-- null --></a></li><li value="48">
Morin RD, O'Connor MD, Griffith M, Kuchenbauer F, Delaney A, Prabhu AL,
et al. Application of massively parallel sequencing to microRNA
profiling and discovery in human embryonic stem cells. Genome Res
2008;18:610-621.<!-- HIGHWIRE ID="55:4:641:48" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/4/610"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R49"><!-- null --></a></li><li value="49">
Emrich SJ, Barbazuk WB, Li L, Schnable PS. Gene discovery and
annotation using LCM-454 transcriptome sequencing. Genome Res
2007;17:69-73.<!-- HIGHWIRE ID="55:4:641:49" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=17/1/69"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R50"><!-- null --></a></li><li value="50">
Pan Q, Shai O, Lee LJ, Frey BJ, Blencowe BJ. Deep surveying of
alternative splicing complexity in the human transcriptome by
high-throughput sequencing. Nat Genet 2008;40:1413-1415.<!-- HIGHWIRE ID="55:4:641:50" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fng.259&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000261215900017&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18978789&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18978789&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R51"><!-- null --></a></li><li value="51">
Wang ET, Sandberg R, Luo S, Khrebtukova I, Zhang L, Mayr C, et al.
Alternative isoform regulation in human tissue transcriptomes. Nature
2008;456:470-476.<!-- HIGHWIRE ID="55:4:641:51" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnature07509&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000261170500031&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18978772&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18978772&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R52"><!-- null --></a></li><li value="52">
Ren B, Robert F, Wyrick JJ, Aparicio O, Jennings EG, Simon I, et al.
Genome-wide location and function of DNA binding proteins. Science
2000;290:2306-2309.<!-- HIGHWIRE ID="55:4:641:52" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=290/5500/2306"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R53"><!-- null --></a></li><li value="53">
Barski A, Cuddapah S, Cui K, Roh TY, Schones DE, Wang Z, et al.
High-resolution profiling of histone methylations in the human genome.
Cell 2007;129:823-837.<!-- HIGHWIRE ID="55:4:641:53" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1016%2Fj.cell.2007.05.009&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000246788300025&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17512414&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17512414&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R54"><!-- null --></a></li><li value="54"> Schones DE, Zhao K. Genome-wide approaches to studying chromatin modifications. Nat Rev Genet 2008;9:179-191.<!-- HIGHWIRE ID="55:4:641:54" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnrg2270&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000253239500011&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18250624&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18250624&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R55"><!-- null --></a></li><li value="55"> Valouev A, Ichikawa J, Tonthat T, Stuart J, Ranade S, Peckham H, et al. A high-resolution, nucleosome position map of <i>C. elegans</i> reveals a lack of universal sequence-dictated positioning. Genome Res 2008;18:1051-1063.<!-- HIGHWIRE ID="55:4:641:55" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/7/1051"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R56"><!-- null --></a></li><li value="56"> Johnson DS, Mortazavi A, Myers RM, Wold B. Genome-wide mapping of in vivo protein-DNA interactions. Science 2007;316:1497-1502.<!-- HIGHWIRE ID="55:4:641:56" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=316/5830/1497"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R57"><!-- null --></a></li><li value="57">
Robertson G, Hirst M, Bainbridge M, Bilenky M, Zhao Y, Zeng T, et al.
Genome-wide profiles of STAT1 DNA association using chromatin
immunoprecipitation and massively parallel sequencing. Nat Methods
2007;4:651-657.<!-- HIGHWIRE ID="55:4:641:57" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnmeth1068&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000248443900019&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17558387&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17558387&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R58"><!-- null --></a></li><li value="58">
Korshunova Y, Maloney RK, Lakey N, Citek RW, Bacher B, Budiman A, et
al. Massively parallel bisulphite pyrosequencing reveals the molecular
complexity of breast cancer-associated cytosine-methylation patterns
obtained from tissue and serum DNA. Genome Res 2008;18:19-29.<!-- HIGHWIRE ID="55:4:641:58" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/1/19"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R59"><!-- null --></a></li><li value="59"> Cokus SJ, Feng S, Zhang X, Chen Z, Merriman B, Haudenschild CD, et al. Shotgun bisulphite sequencing of the <i>Arabidopsis</i> genome reveals DNA methylation patterning. Nature 2008;452:215-219.<!-- HIGHWIRE ID="55:4:641:59" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnature06745&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000253925600040&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18278030&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18278030&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R60"><!-- null --></a></li><li value="60">
Marguerat S, Wilhelm BT, Bahler J. Next-generation sequencing:
applications beyond genomes. Biochem Soc Trans 2008;36:1091-1096.<!-- HIGHWIRE ID="55:4:641:60" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1042%2FBST0361091&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000260076300060&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18793195&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18793195&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R61"><!-- null --></a></li><li value="61"> Pop M, Salzberg SL. Bioinformatics challenges of new sequencing technology. Trends Genet 2008;24:142-149.<!-- HIGHWIRE ID="55:4:641:61" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=000254058200007&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18262676&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18262676&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R62"><!-- null --></a></li><li value="62">
Rougemont J, Amzallag A, Iseli C, Farinelli L, Xenarios I, Naef F.
Probabilistic base calling of Solexa sequencing data. BMC
Bioinformatics 2008;9:431.<!-- HIGHWIRE ID="55:4:641:62" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1186%2F1471-2105-9-431&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18851737&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18851737&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R63"><!-- null --></a></li><li value="63"> Ewing B, Green P. Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res 1998;8:186-194.<!-- HIGHWIRE ID="55:4:641:63" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=8/3/186"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R64"><!-- null --></a></li><li value="64">
Ewing B, Hillier L, Wendl MC, Green P. Base-calling of automated
sequencer traces using phred. I. Accuracy assessment. Genome Res
1998;8:175-185.<!-- HIGHWIRE ID="55:4:641:64" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=8/3/175"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R65"><!-- null --></a></li><li value="65">
Brockman W, Alvarez P, Young S, Garber M, Giannoukos G, Lee WL, et al.
Quality scores and SNP detection in sequencing-by-synthesis systems.
Genome Res 2008;18:763-770.<!-- HIGHWIRE ID="55:4:641:65" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/5/763"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R66"><!-- null --></a></li><li value="66">
Smith AD, Xuan Z, Zhang MQ. Using quality scores and longer reads
improves accuracy of Solexa read mapping. BMC Bioinformatics 2008;9:128.<!-- HIGHWIRE ID="55:4:641:66" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1186%2F1471-2105-9-128&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18307793&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18307793&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R67"><!-- null --></a></li><li value="67">
Li H, Ruan J, Durbin R. Mapping short DNA sequencing reads and calling
variants using mapping quality scores. Genome Res 2008;18:1851-1858.<!-- HIGHWIRE ID="55:4:641:67" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/11/1851"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R68"><!-- null --></a></li><li value="68">
Whiteford N, Haslam N, Weber G, Prugel-Bennett A, Essex JW, Roach PL,
et al. An analysis of the feasibility of short read sequencing. Nucleic
Acids Res 2005;33:e171.<!-- HIGHWIRE ID="55:4:641:68" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=33/19/e171"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R69"><!-- null --></a></li><li value="69">
Warren RL, Sutton GG, Jones SJ, Holt RA. Assembling millions of short
DNA sequences using SSAKE. Bioinformatics 2007;23:500-501.<!-- HIGHWIRE ID="55:4:641:69" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=23/4/500"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R70"><!-- null --></a></li><li value="70">
Hernandez D, Francois P, Farinelli L, Osteras M, Schrenzel J. De novo
bacterial genome sequencing: millions of very short reads assembled on
a desktop computer. Genome Res 2008;18:802-809.<!-- HIGHWIRE ID="55:4:641:70" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/5/802"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R71"><!-- null --></a></li><li value="71"> Lin H, Zhang Z, Zhang MQ, Ma B, Li M. ZOOM! Zillions Of Oligos Mapped. Bioinformatics 2008;24:2431-2437.<!-- HIGHWIRE ID="55:4:641:71" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=24/21/2431"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R72"><!-- null --></a></li><li value="72">
Smith DR, Quinlan AR, Peckham HE, Makowsky K, Tao W, Woolf B, et al.
Rapid whole-genome mutational profiling using next-generation
sequencing technologies. Genome Res 2008;18:1638-1642.<!-- HIGHWIRE ID="55:4:641:72" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/10/1638"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R73"><!-- null --></a></li><li value="73"> Li R, Li Y, Kristiansen K, Wang J. SOAP: short oligonucleotide alignment program. Bioinformatics 2008;24:713-714.<!-- HIGHWIRE ID="55:4:641:73" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=24/5/713"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R74"><!-- null --></a></li><li value="74"> Chaisson MJ, Pevzner PA. Short read fragment assembly of bacterial genomes. Genome Res 2008;18:324-330.<!-- HIGHWIRE ID="55:4:641:74" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/2/324"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R75"><!-- null --></a></li><li value="75">
Dohm JC, Lottaz C, Borodina T, Himmelbauer H. SHARCGS, a fast and
highly accurate short-read assembly algorithm for de novo genomic
sequencing. Genome Res 2007;17:1697-1706.<!-- HIGHWIRE ID="55:4:641:75" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=17/11/1697"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R76"><!-- null --></a></li><li value="76"> Zerbino DR, Birney E. Velvet: algorithms for de novo short read assembly using de Bruijn graphs. Genome Res 2008;18:821-829.<!-- HIGHWIRE ID="55:4:641:76" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/5/821"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R77"><!-- null --></a></li><li value="77">
Zhu QH, Spriggs A, Matthew L, Fan L, Kennedy G, Gubler F, Helliwell C.
A diverse set of microRNAs and microRNA-like small RNAs in developing
rice grains. Genome Res 2008;18:1456-1465.<!-- HIGHWIRE ID="55:4:641:77" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/9/1456"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R78"><!-- null --></a></li><li value="78">
Glazov EA, Cottee PA, Barris WC, Moore RJ, Dalrymple BP, Tizard ML. A
microRNA catalog of the developing chicken embryo identified by a deep
sequencing approach. Genome Res 2008;18:957-964.<!-- HIGHWIRE ID="55:4:641:78" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/6/957"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R79"><!-- null --></a></li><li value="79">
Valouev A, Johnson DS, Sundquist A, Medina C, Anton E, Batzoglou S, et
al. Genome-wide analysis of transcription factor binding sites based on
ChIP-Seq data. Nat Methods 2008;5:829-834.<!-- HIGHWIRE ID="55:4:641:79" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnmeth.1246&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000258912700017&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=19160518&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=19160518&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R80"><!-- null --></a></li><li value="80">
Zhang Y, Liu T, Meyer CA, Eeckhoute J, Johnson DS, Bernstein BE, et al.
Model-based analysis of ChIP-Seq (MACS). Genome Biol 2008;9:R137.<!-- HIGHWIRE ID="55:4:641:80" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1186%2Fgb-2008-9-9-r137&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18798982&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18798982&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R81"><!-- null --></a></li><li value="81">
Binladen J, Gilbert MT, Bollback JP, Panitz F, Bendixen C, Nielsen R,
Willerslev E. The use of coded PCR primers enables high-throughput
sequencing of multiple homolog amplification products by 454 parallel
sequencing. PLoS ONE 2007;2:e197.<!-- HIGHWIRE ID="55:4:641:81" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1371%2Fjournal.pone.0000197&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17299583&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17299583&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R82"><!-- null --></a></li><li value="82"> Meyer M, Stenzel U, Hofreiter M. Parallel tagged sequencing on the 454 platform. Nat Protoc 2008;3:267-278.<!-- HIGHWIRE ID="55:4:641:82" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnprot.2007.520&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000254137000014&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18274529&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18274529&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R83"><!-- null --></a></li><li value="83">
Meyer M, Stenzel U, Myles S, Prufer K, Hofreiter M. Targeted
high-throughput sequencing of tagged nucleic acid samples. Nucleic
Acids Res 2007;35:e97.<!-- HIGHWIRE ID="55:4:641:83" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=35/15/e97"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R84"><!-- null --></a></li><li value="84">
Wang C, Mitsuya Y, Gharizadeh B, Ronaghi M, Shafer RW. Characterization
of mutation spectra with ultra-deep pyrosequencing: application to
HIV-1 drug resistance. Genome Res 2007;17:1195-1201.<!-- HIGHWIRE ID="55:4:641:84" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=17/8/1195"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R85"><!-- null --></a></li><li value="85">
Fokstuen S, Lyle R, Munoz A, Gehrig C, Lerch R, Perrot A, et al. A DNA
resequencing array for pathogenic mutation detection in hypertrophic
cardiomyopathy. Hum Mutat 2008;29:879-885.<!-- HIGHWIRE ID="55:4:641:85" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1002%2Fhumu.20749&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000256609300013&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18409188&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18409188&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R86"><!-- null --></a></li><li value="86">
Morita H, Rehm HL, Menesses A, McDonough B, Roberts AE, Kucherlapati R,
et al. Shared genetic causes of cardiac hypertrophy in children and
adults. N Engl J Med 2008;358:1899-1908.<!-- HIGHWIRE ID="55:4:641:86" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=nejm&amp;resid=358/18/1899"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R87"><!-- null --></a></li><li value="87">
Chiu RW, Chan KC, Gao Y, Lau VY, Zheng W, Leung TY, et al. Noninvasive
prenatal diagnosis of fetal chromosomal aneuploidy by massively
parallel genomic sequencing of DNA in maternal plasma. Proc Natl Acad
Sci U S A 2008;105:20458-20463.<!-- HIGHWIRE ID="55:4:641:87" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=105/51/20458"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R88"><!-- null --></a></li><li value="88">
Fan HC, Blumenfeld YJ, Chitkara U, Hudgins L, Quake SR. Noninvasive
diagnosis of fetal aneuploidy by shotgun sequencing DNA from maternal
blood. Proc Natl Acad Sci U S A 2008;105:16266-16271.<!-- HIGHWIRE ID="55:4:641:88" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=105/42/16266"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R89"><!-- null --></a></li><li value="89"> Fan HC, Quake SR. Detection of aneuploidy with digital polymerase chain reaction. Anal Chem 2007;79:7576-7579.<!-- HIGHWIRE ID="55:4:641:89" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=17715994&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17715994&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R90"><!-- null --></a></li><li value="90">
Lo YM, Lun FM, Chan KC, Tsui NB, Chong KC, Lau TK, et al. Digital PCR
for the molecular detection of fetal chromosomal aneuploidy. Proc Natl
Acad Sci U S A 2007;104:13116-13121.<!-- HIGHWIRE ID="55:4:641:90" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=104/32/13116"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R91"><!-- null --></a></li><li value="91"> Dennis Lo YM, Chiu RW. Prenatal diagnosis: progress through plasma nucleic acids. Nat Rev Genet 2007;8:71-77.<!-- HIGHWIRE ID="55:4:641:91" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1038%2Fnrg1982&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000242934300017&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17146468&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17146468&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R92"><!-- null --></a></li><li value="92">
Korlach J, Marks PJ, Cicero RL, Gray JJ, Murphy DL, Roitman DB, et al.
Selective aluminum passivation for targeted immobilization of single
DNA polymerase molecules in zero-mode waveguide nanostructures. Proc
Natl Acad Sci U S A 2008;105:1176-1181.<!-- HIGHWIRE ID="55:4:641:92" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=105/4/1176"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R93"><!-- null --></a></li><li value="93">
Levene MJ, Korlach J, Turner SW, Foquet M, Craighead HG, Webb WW.
Zero-mode waveguides for single-molecule analysis at high
concentrations. Science 2003;299:682-686.<!-- HIGHWIRE ID="55:4:641:93" --><a href="http://www.clinchem.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=299/5607/682"><nobr>[Abstract/<font color="#cc0000">Free</font>&nbsp;Full&nbsp;Text]</nobr></a><!-- /HIGHWIRE --><a name="R94"><!-- null --></a></li><li value="94">
Eid J, Fehr A, Gray J, Luong K, Lyle J, Otto G, et al. Real-time DNA
sequencing from single polymerase molecules. Science 2008;323:133-138.<!-- HIGHWIRE ID="55:4:641:94" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=000262104100059&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=19023044&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=19023044&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R95"><!-- null --></a></li><li value="95">
Astier Y, Braha O, Bayley H. Toward single molecule DNA sequencing:
direct identification of ribonucleoside and deoxyribonucleoside
5'-monophosphates by using an engineered protein nanopore equipped with
a molecular adapter. J Am Chem Soc 2006;128:1705-1710.<!-- HIGHWIRE ID="55:4:641:95" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1021%2Fja057123%2B&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000235224700066&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=16448145&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=16448145&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R96"><!-- null --></a></li><li value="96">
Wu HC, Astier Y, Maglia G, Mikhailova E, Bayley H. Protein nanopores
with covalently attached molecular adapters. J Am Chem Soc
2007;129:16142-16148.<!-- HIGHWIRE ID="55:4:641:96" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1021%2Fja0761840&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000251974000068&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18047341&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18047341&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R97"><!-- null --></a></li><li value="97"> Gupta PK. Single-molecule DNA sequencing technologies for future genomics research. Trends Biotechnol 2008;26:602-611.<!-- HIGHWIRE ID="55:4:641:97" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1016%2Fj.tibtech.2008.07.003&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000260802600004&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=18722683&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=18722683&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --><a name="R98"><!-- null --></a></li><li value="98"> Rhee M, Burns MA. Nanopore sequencing technology: research trends and applications. Trends Biotechnol 2006;24:580-586.<!-- HIGHWIRE ID="55:4:641:98" --><a href="http://www.clinchem.org/cgi/external_ref?access_num=10.1016%2Fj.tibtech.2006.10.005&amp;link_type=DOI">[CrossRef]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=000242794400008&amp;link_type=ISI">[ISI]</a><a href="http://www.clinchem.org/cgi/external_ref?access_num=17055093&amp;link_type=MED">[Medline]</a>
<a href="http://www.clinchem.org/cgi/external_ref?access_num=17055093&amp;displayid=120699&amp;link_type=INFOTRIEVE">[Order article via Infotrieve]</a><!-- /HIGHWIRE --></li></ol>


		
        
        
	
	
	
	
	




 










<br clear="all">




  <font face="">
  
  </font>
<br clear="all">
<br clear="all">




	
	
	









<br clear="all">


	
	 
		
		
		  
			
	
		<table class="content_box_outer_table">
		
		
			<tbody><tr>
			<td>
		
	

<!-- beginning of inner table -->
	<table class="content_box_inner_table">
<!-- citation -->
	
		




	

	

	
		
			
<tbody><tr><td class="content_box_title_highlight" colspan="2">This Article</td></tr>

	

	<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr>
				
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				
				<strong><a href="http://www.clinchem.org/cgi/content/abstract/55/4/641">
				Abstract</a></strong>
				 					<img alt="Freely available" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/free.gif" border="0" height="7" width="32">
				
	
				</td></tr>
			
		
		
	

	
	
	

	
	
	


	
	
	

	
	
	

	
	
	
	
	
	
	
	
	

	
		
		
	
	
		
	

	

	
		
			
		
	

	

	
	 

	
	 

		
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
		<strong>
		
		
		
			
				
				
					<a href="http://www.clinchem.org/cgi/reprint/55/4/641">
		<strong>Full Text</strong> (PDF)</a>


				
			
		
		</strong>
		</td></tr>
	
	

	
	
	
	
	

	
	
	
	
	

	
	
	

	
	




	

	
		
	

	
		
	

	
		

	
		
		

			


	

			

			
		
	

			
	  
	

			

			
			

			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/content/full/clinchem.2008.112789/DC1">
				HTML Page - index.htslp</a></strong>
			</td></tr>
			
			

			
	
	

	
		
	









	
		
	
		
		
			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item"><strong>
			All Versions of this Article:<br>
		
			<a href="http://www.clinchem.org/cgi/content/short/clinchem.2008.112789v1">
					clinchem.2008.112789v1</a>
			
		<br>
	

	
		
		
			
					55/4/641
				
		&nbsp;&nbsp;&nbsp;<font color="#cc0000"><em>most recent</em></font></strong></td></tr>
	

	





	
	
	

	
		
	

	 

	
	
		  
    

	
	
	
	







		
	
		
			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/alerts/ctalert?alertType=citedby&amp;addAlert=cited_by&amp;saveAlert=no&amp;cited_by_criteria_resid=clinchem;55/4/641&amp;return_type=article&amp;return_url=http%3A%2F%2Fwww.clinchem.org%2Fcgi%2Fcontent%2Ffull%2F55%2F4%2F641">
				Alert me when this article is cited</a></strong>
			</td></tr>
	

		
		

		
		

		
		

			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/alerts/ctalert?alertType=correction&amp;addAlert=correction&amp;saveAlert=no&amp;correction_criteria_value=55/4/641&amp;return_type=article&amp;return_url=http%3A%2F%2Fwww.clinchem.org%2Fcgi%2Fcontent%2Ffull%2F55%2F4%2F641">
				Alert me if a correction is posted</a></strong>
			</td></tr>
		
























			
		
			
			
			
				
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
					<strong><a rel="nofollow" href="http://www.clinchem.org/cgi/citemap?id=clinchem;55/4/641">
					Citation Map</a></strong>
				</td></tr>
		
	

			
			
<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr><tr><td class="content_box_title" colspan="2">Services</td></tr>

	

	<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr>
			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/mailafriend?url=http%3A%2F%2Fwww.clinchem.org%2Fcgi%2Fcontent%2Ffull%2F55%2F4%2F641&amp;title=Next-Generation+Sequencing%3A+From+Basic+Research+to+Diagnostics">
				Email this article to a friend</a></strong>
			</td></tr>
			
			


	

	

	

		
		



			
			
			
		
	

		
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
		<strong>
		<a href="http://www.clinchem.org/cgi/search?qbe=clinchem;clinchem.2008.112789&amp;journalcode=clinchem&amp;minscore=5000">
		Similar articles in this journal</a>
		</strong>
		</td></tr>

		
		

			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/external_ref?access_num=19246620&amp;link_type=MED_NBRS">
				Similar articles in PubMed</a></strong>
			</td></tr>
		
	


		



			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/alerts/etoc">
				Alert me to new issues of the journal</a></strong>
			</td></tr>
		

		
		
	

		
		
		

			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
			<strong><a href="http://www.clinchem.org/cgi/citmgr?gca=clinchem;55/4/641">
			Download to citation manager</a></strong>
			</td></tr>
		
		
	


	
		
	

	



  			
		


	

	




	


			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/external_ref?&amp;link_type=PERMISSIONDIRECT" target="_new">
				<img src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/permissions.gif" alt="reprints &amp; permissions" border="0"></a></strong>
			</td></tr>	
			





		

		

	
	
	
		


	

	

	


	





	


	
		 
	

		
		

	

	
	















	
		
			
				
				
<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr><tr><td class="content_box_title" colspan="2">Google Scholar</td></tr>

	

	<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr>
						
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
							<strong><a target="_blank" href="http://scholar.google.com/scholar?q=%22author%3AK.%20V.+author%3AVoelkerding%22">
							Articles by Voelkerding, K. V.</a></strong>
						</td></tr>
				
				
			
				
			
				
				

						
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
							<strong><a target="_blank" href="http://scholar.google.com/scholar?q=%22author%3AJ.%20D.+author%3ADurtschi%22">
							Articles by Durtschi, J. D.</a></strong>
						</td></tr>
				
				
			
		
		
	
	
	
	

		
			
				
			
		
	










	





	
		
<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr><tr><td class="content_box_title" colspan="2">PubMed</td></tr>

	

	<tr><td class="content_box_space_between_sections" colspan="2"><img alt=" " src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/spacer.gif" height="1" width="200"></td></tr>
			
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
				<strong><a href="http://www.clinchem.org/cgi/external_ref?access_num=19246620&amp;link_type=PUBMED">
				PubMed Citation</a></strong>
			</td></tr>
		
	

	
		
		
			
				

						
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
							<strong><a target="_blank" href="http://www.clinchem.org/cgi/external_ref?access_num=Voelkerding+KV&amp;link_type=AUTHORSEARCH">
							Articles by Voelkerding, K. V.</a></strong>
						</td></tr>
				
			
				

						
		<tr><td class="content_box_arrow" valign="top" width="4"><img alt="Right arrow" src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/arrowTtrim.gif" border="0" height="11" width="4"></td><td class="content_box_item">
							<strong><a target="_blank" href="http://www.clinchem.org/cgi/external_ref?access_num=Durtschi+JD&amp;link_type=AUTHORSEARCH">
							Articles by Durtschi, J. D.</a></strong>
						</td></tr>
				
			
		

		
	

	















	
	


	












	


































	
		

	

		
			
		
	 
		











	

	






































	</tbody></table>
</td></tr></tbody></table> 


					
					
	

<br clear="all">








	
	<p>
	</p><hr align="left" noshade="noshade" width="450">
	
	<table cellpadding="2" cellspacing="2" width="450">
	<tbody><tr>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/"><font color="#ffffff" face="arial,helvetica" size="-2">HOME</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/help/"><font color="#ffffff" face="arial,helvetica" size="-2">HELP</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/cgi/feedback"><font color="#ffffff" face="arial,helvetica" size="-2">FEEDBACK</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/subscriptions/"><font color="#ffffff" face="arial,helvetica" size="-2">SUBSCRIPTIONS</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/contents-by-date.0.shtml"><font color="#ffffff" face="arial,helvetica" size="-2">ARCHIVE</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/search.dtl"><font color="#ffffff" face="arial,helvetica" size="-2">SEARCH</font></a></td>
	    <td align="center" bgcolor="#cc0000" nowrap="nowrap" valign="top"><a target="_top" style="text-decoration: none;" href="http://www.clinchem.org/content/vol55/issue4/"><font color="#ffffff" face="arial,helvetica" size="-2">TABLE OF CONTENTS</font></a></td>
	</tr>
	</tbody></table>
	
	






<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script><script src="Voelkerding%202009%20Next-Generation%20Sequencing%20-%20From%20Basic%20Research%20to%20Diagnostics_files/ga.js" type="text/javascript"></script>
<script type="text/javascript">
var pageTracker = _gat._getTracker("UA-5821081-1");
pageTracker._trackPageview();
</script>

	



</body></html>