<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head>


<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Characterization of mutation spectra with ultra-deep pyrosequencing: application to HIV-1 drug resistance</title>
</head><body>
<table border="1" bordercolor="#cccccc" width="794">
  <tbody><tr>
    <td width="784"><p>Supplemental Material for "<strong>Characterization of mutation spectra with ultra-deep pyrosequencing: application to HIV-1 drug resistance</strong>" by <em>Wang et al.</em></p>    </td>
  </tr>
  <tr>
    <td bgcolor="#ccffcc">&nbsp;</td>
  </tr>
  <tr>
    <td><p>Raw data: </p>
    <p>The <a href="http://www.ncbi.nlm.nih.gov/Traces/trace.cgi?cmd=show&amp;f=formats&amp;m=doc&amp;s=format">SFF</a> format raw output from GS20 platform and the reference sequences can be found <a href="http://dbpartners.stanford.edu/454/pub/download/">here</a>.</p>    </td>
  </tr>
  <tr>
    <td bgcolor="#ccffcc">&nbsp;</td>
  </tr>
  <tr>
    <td><p align="justify">Mapping results:</p>
    <p align="justify">Using
our newly developed asymmetric Smith-Waterman (ASW) algorithm, we
mapped GS20 reads onto reference sequences, which were determined with
conventional direct-PCR Sanger sequencing approach. Check <a href="http://dbpartners.stanford.edu/454/pub/download/">here</a> for mapping results.</p>
    <p align="justify"><img src="Wang%202007%20supplementary_files/mapping.gif" alt="Mapping legend" longdesc="http://dbpartners.stanford.edu/454/pub/img/mapping.gif" height="280" width="704"></p>
    <p align="justify">In addition, we provided a <a href="http://dbpartners.stanford.edu/454/pub/454.php">web interface</a>
to pinpoint each variant detected by 454 platform by specifying a
position of intererst. For the sample V11909, we sequenced 68 (31 1/100
cDNA dilution and 37 1/30 cDNA dilution) PCR products using Sanger
approach to confirm minor variants revealed by the 454 platform. The
confirmation results can be checked using the web interface by
selecting species name 'V11909C'. </p>
    <p align="justify">The colorful views of mapping results can be found <a href="http://dbpartners.stanford.edu/454/pub/download/">here</a>.
Those files named *_na_orign.pdf show nucleotide variants before
applying our filtering procedure and those named *_na_parsed.pdf show
nucleotide variants after applying the filtering procedure. The amino
acids variants files are named in a similar way. (Note in the amino
acids variants map, synonymous varaints are merged together. Only the
first 1kp sequence is shown in the nucleotide variants map.) </p>
    <p><img src="Wang%202007%20supplementary_files/legend.gif" alt="Legend" longdesc="http://dbpartners.stanford.edu/454/pub/img/legend.gif" height="364" width="703"> </p></td>
  </tr>
  
  <tr>
    <td bgcolor="#ccffcc">&nbsp;</td>
  </tr>
  <tr>
    <td><p>Contact Chunlin Wang <img src="Wang%202007%20supplementary_files/email.gif" alt="email address" longdesc="http://dbpartners.stanford.edu/454/pub/img/email.gif" height="26" width="143">if you need assistance using this site.</p>    </td>
  </tr>
</tbody></table>
</body></html>