<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Bioinformatics (Oxford, England)"><meta name="citation_title" content="Association mapping and fine mapping with TreeLD"><meta name="citation_authors" content="Sebastian Zöllner, Xiaoquan Wen, and  Jonathan K. Pritchard"><meta name="citation_date" content="2005 July 15"><meta name="citation_issue" content="14"><meta name="citation_volume" content="21"><meta name="citation_firstpage" content="3168"><meta name="citation_doi" content="10.1093/bioinformatics/bti453"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2587010&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010"><meta name="citation_pmid" content="15855250"><meta name="DC.Title" content="Association mapping and fine mapping with TreeLD"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Sebastian Zöllner"><meta name="DC.Contributor" content="Xiaoquan Wen"><meta name="DC.Contributor" content="Jonathan K. Pritchard"><meta name="DC.Date" content="2005 July 15"><meta name="DC.Identifier" content="10.1093/bioinformatics/bti453"><meta name="DC.Language" content="en"><link rel="stylesheet" href="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/pmcstatic.css" type="text/css"><link rel="stylesheet" href="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/pmcbase1.css" type="text/css"><link rel="stylesheet" href="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/pmcbody5.css" type="text/css"><link rel="stylesheet" href="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/pmcrefs1.css" type="text/css"><script type="text/javascript" src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/tileshop_pmc.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/utils.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/tileshop_pmc.js"></script><script src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/firebugx.js" type="text/javascript"></script><script src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/tile.js" type="text/javascript"></script><script src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/notify.js" type="text/javascript"></script><script src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/drag_n_drop.js" type="text/javascript"></script><script src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/scale_pmc.js" type="text/javascript"></script><script type="text/javascript" src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/tileshop_data_db.js"></script><script src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/remote_data_provider.js" type="text/javascript"></script><title>Association mapping and fine mapping with TreeLD</title></head><div FirebugVersion="1.3.3" style="display: none;" id="_firebugConsole"></div><body class="pmc-body"><div style="height: 0px;"><a id="top" name="top"></a></div><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr valign="top"><td colspan="3"><table style="border-collapse: collapse;" border="0" cellpadding="0" cellspacing="0"><tbody><tr><td valign="top" width="145"><img style="border-style: none;" usemap="#pmclogo" alt="pmc logo image" src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/pmc3_logo_v5.gif"><map name="pmclogo"><area shape="rect" coords="65,52,137,67" href="http://www.pubmedcentral.nih.gov/fprender.fcgi" alt="Journal List"><area shape="rect" coords="7,52,52,67" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0FCagx3u33yv83p7hFLw2adUCu7E6kLs8xSB1u5zs&amp;reftype=other&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?db=pmc" alt="Search"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="http://www.pubmedcentral.nih.gov/"></map></td><td style="padding-left: 4px;" height="75" valign="top"><div><img style="border-style: none;" src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/logo-nihpa.gif" alt="Logo of nihpa" usemap="#logo-imagemap" align="top"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="NIHPA banner" coords="0,0,499,56" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0vOEv45D0QRqOuXcVSDV9UkqNsgP6BDOBEGaHPAO7&amp;reftype=publisher&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://publicaccess.nih.gov" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="rect" alt="about author manuscripts" coords="0,57,249,74" href="http://www.pubmedcentral.nih.gov/about/authorms.html"><area shape="rect" alt="submit a manuscript" coords="250,57,499,74" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ruq05lAzoFJ70pmZlSdHD9Tw6WkZya_k6rb0OkJU&amp;reftype=publisher&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.nihms.nih.gov" target="pmc_ext" onclick="focuswin('pmc_ext')"></map></div></td></tr></tbody></table></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="3" width="100%"><tbody><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="http://www.pubmedcentral.nih.gov/fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0XiWhwND3PEwPew29wtISCB45U7TRDRXw7DQ4n5WS&amp;reftype=other&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CN%2FA&amp;TO=Entrez%7CSearch%7CNIHPA%20Author%20Manuscripts&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?db=pmc&amp;cmd=search&amp;term=nih%20author%20manuscript%5Bfilter%5D" class="navlink">NIHPA Author Manuscripts</a></div></td></tr><tr><td class="sidebar-cell" valign="top" width="145"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/square.gif" alt="&gt;" style="border-style: none;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2587010&amp;blobtype=pdf" class="sidefm-pmclink">PDF (178K)</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0Z7QTBd7fKDa70KAGvTMfcoG1CJHUhiJbOsO1cAI9&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15855250">PubMed record</option><option selected="selected" value="redirect3.cgi?&amp;&amp;auth=0-OGLYtI672xWx0Ue9qKud14DdrjMr28NHSaYMQKn&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=15855250">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0XutD-IagVu6qfeI1QRMuSq6BQOzUwusDc8RAMkf1&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=15855250">PubMed LinkOut</option></select><br><input class="sidefm-pmart" alt="GO" src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/go.gif" type="image"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0J3_9fRAHpTmdJzCkK3nSSO2kodMvTc8Nx07iskHK&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Z&amp;%23x000f6;llner+S%5Bauth%5D" class="sidefm-pmclink"> Zöllner, S.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0RCz2qYQ2C9XHpNvd_12itBEazHvx_FYmbk1HBQYr&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Wen+X%5Bauth%5D" class="sidefm-pmclink"> Wen, X.</a></div><div class="sidefm-pmclink-item"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0-StaKzj42Q3uzZQwUhd2W9M0wBcFpVb3S2rnvQH2&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=search&amp;db=PubMed&amp;term=%20Pritchard+JK%5Bauth%5D" class="sidefm-pmclink"> Pritchard, J.</a></div></div></div></td><td rowspan="103" background="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/wm-nihms.gif" height="300" width="20"><div style="width: 20px;"></div></td><td class="content-cell" valign="top"><div class="front-matter-section"><table cellpadding="0" cellspacing="0" width="100%"><tbody><tr style="vertical-align: top;"><td><div class="fm-citation">Bioinformatics.Author manuscript; available in PMC 2008 November 25.</div><div class="fm-citation">Published in final edited form as:</div><div style="margin-left: 5px;"><div class="fm-vol-iss-date"><a class="ext-reflink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ls7brnl2oahUjamiF0t4UfAtP79S-FNoK9gX6QkY&amp;reftype=publisher&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=15855250" target="pmc_ext" onclick="focuswin('pmc_ext')"><span class="citation-abbreviation">Bioinformatics. </span><span class="citation-publication-date">2005 July 15; </span><span class="citation-volume">21</span><span class="citation-issue">(14)</span><span class="citation-flpages">: 3168–3170. </span></a></div><span class="fm-vol-iss-date">Published online 2005 April 26. </span><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1093/bioinformatics/bti453.</span></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC2587010</div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS72780</span></div></td></tr></tbody></table><div class="fm-copyright"><a class="int-reflink" href="http://www.pubmedcentral.nih.gov/about/copyright.html">Copyright notice</a>  and <a class="int-reflink" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a></div><div class="fm-title">Association mapping and fine mapping with TreeLD</div><div class="contrib-group fm-author">Sebastian Zöllner,<sup><img src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/corrauth.gif" alt="corresponding author"></sup> Xiaoquan Wen, and  Jonathan K. Pritchard</div><div class="fm-affl">Department of Human Genetics, The University of Chicago, 920 East 58th Street–CLSC 507, Chicago, IL 60637, USA</div><div class="fm-footnote"><sup><img src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div class="fm-footnote"></div><div class="links-box"><div class="fm-footnote"><img src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> The publisher's final edited version of this article is available free at <a class="ext-reflink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0vci_zTsXfDcj6z6oXNz21izwGIxrL3GypK0yX57j&amp;reftype=publisher&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=15855250" target="pmc_ext" onclick="focuswin('pmc_ext')">Bioinformatics</a>.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id763584">Summary</div><div class="section-content"><!--article-meta--><div class="p p-first-last" id="P1"><span id="P1"></span>The
program package TreeLD implements a unified approach to association
mapping and fine mapping of complex trait loci and a novel approach to
visualizing association data, based on an inferred ancestry of the
sample. Fundamentally, the TreeLD approach is based on the idea that
the evidence for association at a particular position is contained in
the ancestral tree relating the sampled chromosomes at that position.
TreeLD provides an easy-to-use interface and can be applied to
case–control, TDT trio and quantitative trait data.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id763584" class="sidefm-pmclink" style="text-transform: none;">Summary</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/square.gif" alt="&gt;" style="border-style: none;"></span>INTRODUCTION</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id791126" class="sidefm-pmclink" style="text-transform: none;">OVERVIEW OF THE METHOD</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id563161" class="sidefm-pmclink" style="text-transform: none;">RESULTS FOR A QUANTITATIVE TRAIT</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id732627" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><span id="S2"></span><div class="head1 section-title" style="text-transform: none;" id="id791098"><a id="S2"></a>INTRODUCTION</div><div class="section-content"><span id="S2"></span><div class="p p-first-last" id="P2"><span id="P2"></span>Recent
advances in genotyping technology make it feasible to generate dense
single nucleotide polymorphism (SNP) data for use in linkage
disequilibrium (LD) mapping of complex trait loci. However, there is
currently a lack of effective computational tools for analyzing such
data. Thus, we announce the release of the program package TreeLD 1.0,
a tool for multipoint LD-analysis that applies a unified approach to
test for the presence of a disease locus and to fine-map its location.
Furthermore, the package provides a novel way of visualizing the
association signal in a sample. TreeLD can be applied to case–control
data and TDT trio data and it is also the first program of its kind
that can map quantitative trait loci (QTLs). The package has a
user-friendly point-and-click interface, allowing an in-depth
exploration of the data. Executables for any of Windows 2000/XP, Mac OS
X, or Linux and Solaris operating systems and extensive documentation
for TreeLD can be downloaded from the website given in the summary. The
source code is available from the authors upon request. In the
following, we describe the conceptual basis for the algorithm, give an
overview of the program and its features and describe its application
to a set of quantitative trait data.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id763584" class="sidefm-pmclink" style="text-transform: none;">Summary</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id791098" class="sidefm-pmclink" style="text-transform: none;">INTRODUCTION</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/square.gif" alt="&gt;" style="border-style: none;"></span>OVERVIEW OF THE METHOD</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id563161" class="sidefm-pmclink" style="text-transform: none;">RESULTS FOR A QUANTITATIVE TRAIT</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id732627" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><span id="S3"></span><div class="head1 section-title" style="text-transform: none;" id="id791126"><a id="S3"></a>OVERVIEW OF THE METHOD</div><div class="section-content"><span id="S3"></span><div class="p p-first" id="P3" cite="cite-id791135" pmids="15489534"><span id="P3"></span>The
algorithm employed in TreeLD makes use of the observation that
association between a phenotype and a marker is the result of shared
ancestry among cases in the vicinity of that marker. Indeed under
certain conditions, the ancestry of the disease locus incorporates the
entire evidence about the location of the disease mutation that is
contained in the marker data. Thus, it is natural to estimate the
ancestry of a region-of-interest based on the marker information and to
explore this ancestry for non-random distribution of phenotypes among
its terminal nodes, indicating the presence of a disease mutation (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 1</a>). For an in-depth description of these ideas and of the mathematical algorithm, please refer to <a href="#R5" rid="R5" class="cite-reflink bibr popnode">Zöllner and Pritchard (2005)</a>.</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F1" name="F1"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/picrender.gif" style="border: 1px solid ;" class="icon-reflink" alt="Fig. 1" title="Fig. 1"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Fig. 1</strong></a><div class="figure-table-caption-in-article"><span>Hypothetical
example of coalescent genealogies at two different loci for a
case–control sample of 10 chromosomes. Each circle at the bottom of the
tree represents a sampled chromosome; closed circles representing
chromosomes from cases and open</span><a class="side-caption" style="font-size: 100%;" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p" id="P4" cite="cite-id631585" pmids="14574645 11254454"><span id="P4"></span>TreeLD
is designed for analyzing a sample of unrelated individuals that have
either been classified as cases and controls or measured for a
quantitative trait of interest. TDT data can also be analyzed by
considering the non-transmitted chromosomes as sampled from unrelated
control individuals. The input file for TreeLD consists of one text
file containing the marker locations, phenotype information and marker
haplotypes for all individuals in the sample. The program in its
present form is designed for bi-allelic markers. Furthermore, it can
only be used on phased data; therefore, we suggest generating
haplotypes using an estimation method such as PHASE 2.1 (<a href="#R4" rid="R4" class="cite-reflink bibr popnode">Stephens <em>et al</em>., 2001</a>; <a href="#R3" rid="R3" class="cite-reflink bibr popnode">Stephens and Donnelly, 2003</a>). A Perl script that parses the output of PHASE 2.1 into the input format for TreeLD is included in our program package.</div><div class="p" id="P5"><span id="P5"></span>After
the input data are read in, TreeLD estimates the coalescent ancestry of
the sampled chromosomes at each of a series of positions (‘focal
points’) across the region-of-interest. The number and spacing of focal
points determines the resolution of the final analysis and is selected
by the user. Although the algorithm estimates the ancestry of the
chromosome at distinct focal points, the analysis makes full multipoint
use of the SNP data, since SNPs at varying distances contain
information about the ancestry at different levels in the tree. Our
model does not assume that the data can be described by a series of
distinct haplotype blocks, as in some other methods.</div><div class="p" id="P6"><span id="P6"></span>At
each focal point, the evidence for association is assessed using a
sample of trees that are consistent with the entire marker data, and
using a Markov chain Monte Carlo (MCMC) algorithm (<a href="#R2" rid="R2" class="cite-reflink bibr popnode">Gilks <em>et al</em>., 1996</a>)
to sample ancestries from the distribution Pr(Tree|Marker data). To
evaluate convergence of the MCMC, the program plots the probability of
the generated trees conditional on the marker data. After a
user-assigned burn-in period, a number of trees are sampled with a
fixed thinning interval. This set of likely ancestries is displayed for
the user to examine, providing a novel way to visualize the information
in the data. By identifying which clades carry an excess of case
chromosomes, the user can determine which affected individuals are most
likely to carry mutations at this locus and which are more likely to be
phenocopies.</div><div class="p" id="P7" cite="cite-id563087" pmids="7288891"><span id="P7"></span>For
a formal analysis, the likelihood of the phenotype data, given the set
of trees sampled at each focal point, can be computed using a peeling
algorithm (<a href="#R1" rid="R1" class="cite-reflink bibr popnode">Felsenstein, 1981</a>).
For this purpose, the user can either enter the penetrances of the
susceptibility and the wild-type alleles, or choose to integrate over a
set of penetrance parameters. For each tree sampled from the MCMC, the
program then calculates the probability of the observed phenotypes,
assuming that this tree represents the ancestry of a disease locus. By
averaging over all trees at one focal point, a term that is
proportional to the posterior probability of that focal point being the
locus of the disease mutation is calculated. The results can be
transformed into a Bayesian posterior distribution for the location of
the disease mutation(s) by normalizing over all focal points. Based on
this distribution, credible regions for the disease locus can be
generated and the maximum of the distribution can be chosen as a point
estimator (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 2</a>).</div><div style="border-style: solid; border-color: rgb(153, 153, 153) rgb(153, 153, 153) rgb(170, 170, 170) rgb(170, 170, 170); border-width: 1px; margin: 1em 2em 1em 1em;"><div style="border-style: solid; border-color: rgb(240, 240, 240) rgb(240, 240, 240) rgb(248, 248, 248) rgb(248, 248, 248); border-width: 3px 3px 1px 1px;"><a id="F2" name="F2"></a><table style="clear: both; width: 100%;" border="0" cellpadding="5" cellspacing="5"><tbody><tr align="left" valign="top"><td align="center" width="100"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/picrender_002.gif" style="border: 1px solid ;" class="icon-reflink" alt="Fig. 2" title="Fig. 2"></a></td><td><a class="side-caption" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Fig. 2</strong></a><div class="figure-table-caption-in-article"><span>Posterior
distribution for the location of a QTL. The graph displays the
posterior distribution, the vertical arrow indicates the true location
of the QTL, the dashed box contains the 80% credible region reaching
from 1485 to 1655 kb. The 95% credible</span><a class="side-caption" style="font-size: 100%;" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></tbody></table></div></div><div style="clear: both;"></div><div class="p" id="P8" cite="cite-id563123" pmids="15489534"><span id="P8"></span>To
perform a significance test for association based on the generated
likelihoods, two options are available in TreeLD: a likelihood-ratio
test based on a χ<sup>2</sup>-approximation which is fast but slightly
conservative and a more powerful permutation test that automatically
corrects for multiple tests. Applied to a set of 50 datasets, this
method has shown 25% higher power than a single marker test using
Pearson’s χ<sup>2</sup>-test for independence (<a href="#R5" rid="R5" class="cite-reflink bibr popnode">Zöllner and Pritchard, 2005</a>).</div><div class="p p-last" id="P9"><span id="P9"></span>Given
that some steps in the analysis are computationally intensive, it is
desirable to run TreeLD in parallel on multiple machines. To this end,
the front-end generates scripts that can be run independently on
separate processors. The largest dataset we have analyzed with TreeLD
consisted of 838 chromosomes typed at 36 markers. On a small cluster,
the program can be used to analyze datasets of up to 500 individuals
and up to 200 markers in a few days. Information about expected
runtimes can be found in the documentation file.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id763584" class="sidefm-pmclink" style="text-transform: none;">Summary</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id791098" class="sidefm-pmclink" style="text-transform: none;">INTRODUCTION</a></div></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id791126" class="sidefm-pmclink" style="text-transform: none;">OVERVIEW OF THE METHOD</a></div></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img src="Zollner%202008%20Association%20mapping%20and%20fine%20mapping%20with%20TreeLD_files/square.gif" alt="&gt;" style="border-style: none;"></span>RESULTS FOR A QUANTITATIVE TRAIT</a></div><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id732627" class="sidefm-pmclink" style="text-transform: none;">References</a></div></div></div></td><td class="content-cell"><span id="S4"></span><div class="head1 section-title" style="text-transform: none;" id="id563161"><a id="S4"></a>RESULTS FOR A QUANTITATIVE TRAIT</div><div class="section-content"><span id="S4"></span><div class="p p-first" id="P10"><span id="P10"></span>We
applied TreeLD to a quantitative trait dataset, generated by simulating
a coalescent ancestry for a sample of 100 diploid individuals and 100
bi-allelic markers distributed over 2 Mb. At a randomly selected
QTL-locus within those 2 Mb, we generated mutations in the ancestry
with a scaled mutation rate of ν = 0.4. Depending on the resulting
genotype, each individual in the sample was randomly assigned a
phenotype. These phenotypes were generated from a normal distribution
with a standard deviation of 2.0 and a mean of 0.0 for a homozygote
wild-type, 2.0 for a heterozygote and 8.0 for a homozygote mutant.</div><div class="p" id="P11" cite="cite-id732469" pmids="15489534"><span id="P11"></span>The generated dataset was analyzed with TreeLD, and after a burn-in of 3 × 10<sup>6</sup>
updates per focal point, we sampled 100 trees at each of 200 focal
points and generated the posterior distribution, based on those
ancestries (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2587010&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 2</a>).
The peak of the posterior distribution is within 10 kb of the real
functional variant, indicating that TreeLD is able to precisely
estimate the location of this QTL. Furthermore, most of the 2 Mb region
has essentially zero posterior probability of carrying the disease
mutation. Thus, the resulting credible region is a relatively narrow
window of 170 kb. This is consistent with earlier observations that
TreeLD generates precise estimates and narrow credible regions compared
with other methods of fine-mapping (<a href="#R5" rid="R5" class="cite-reflink bibr popnode">Zöllner and Pritchard, 2005</a>).</div><div class="p p-last" id="P12"><span id="P12"></span>In
summary, the package TreeLD provides a set of powerful tools for the
analysis of association data with an easy to use interface. The
structure of the algorithm facilitates extensions, and we plan to
release additional features for TreeLD to provide further options as
free tools to the research community.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145">&nbsp;</td><td class="content-cell"><span id="S6"></span><div class="head1 section-title" style="text-transform: none;" id="id732575"><a id="S6"></a>Acknowledgments</div><div class="section-content"><div class="sec"><div class="p" id="id732580">This work was supported by grant HG 2772 to J.K.P. from the National Institutes of Health.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145">&nbsp;</td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="footnotes">Footnotes</div><div class="section-content"><div class="fm-footnote" id="FN1"><div class="p p-first"><strong>Availability:</strong> The program TreeLD is available on the Internet from <span class="ext-reflink"><a href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=08Qf5s-DmSoJ7Kwp3UevHqvcexDnp2t2nqBiDHAV7&amp;reftype=extlink&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://pritch.bsd.uchicago.edu/software.html" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://pritch.bsd.uchicago.edu/software.html</a></span></div></div><div class="fm-footnote" id="FN2"><div class="p p-first"><strong>Contact:</strong> <span class="e_id732614"><a class="ext-reflink" href="mailto:szoellne@genetics.bsd.uchicago.edu">szoellne@genetics.bsd.uchicago.edu</a></span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id732614", '<a class="ext-reflink" href="' + reverseAndReplaceString('ude.ogacihcu.dsb.sciteneg/ta/enlleozs:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ude.ogacihcu.dsb.sciteneg/ta/enlleozs', '/at/','@') + '</a>')}catch(e){}
                                //--></script></div></div><div class="fm-footnote" id="FN3"><div class="p p-first">For permission, please email: <span class="e_id732624"><a class="ext-reflink" href="mailto:journals.permissions@oujournals.org">journals.permissions@oujournals.org</a></span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id732624", '<a class="ext-reflink" href="' + reverseAndReplaceString('gro.slanruojuo/ta/snoissimrep.slanruoj:otliam', '/at/', '@') + '">' + reverseAndReplaceString('gro.slanruojuo/ta/snoissimrep.slanruoj', '/at/','@') + '</a>')}catch(e){}
                                //--></script></div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><div style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id732627">References</div><div class="section-content"><div class="back-matter-section" id="reference-list"><ul class="back-ref-list" style="list-style-type: decimal;"><li id="R1"><div class="citation">Felsenstein J. Evolutionary trees from DNA sequences: a maximum likelihood approach. <span><span class="ref-journal">J Mol Evol. </span>1981;<span class="ref-vol">17</span>:368–376.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0bAMb-aiXuqnnLBTo_wpQS9-aDgiW_uK0wn-o4kN0&amp;reftype=pubmed&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7288891">PubMed</a>]</span></div></li><li id="R2"><div class="citation">Gilks
WR, Richardson S, Spiegelhalter DJ. Introducing Markov chain Monte
Carlo. In: Gilks WR, Richardson S, Spiegelhalter DJ, editors. <span class="ref-journal">Markov Chain Monte Carlo in Practice.</span> Chapman and Hall; London: 1996. pp. 1–19.</div></li><li id="R3"><div class="citation">Stephens M, Donnelly P. A comparison of Bayesian methods for haplotype reconstruction from population genotype data. <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">73</span>:1162–1169.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0xkpCduek7BHAO4Z-R08qJjzCfWAiCSFAmRsBBApu&amp;reftype=pubmed&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14574645">PubMed</a>]</span></div></li><li id="R4"><div class="citation">Stephens M, et al.  A new statistical method for haplotype reconstruction from population data. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">68</span>:978–989.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0xqp6G5470gvc_y0ji1tPkI4LSLbpm3HB6lwqgQ_P&amp;reftype=pubmed&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11254454">PubMed</a>]</span></div></li><li id="R5"><div class="citation">Zöllner S, Pritchard JK. Coalescent-based association mapping and fine mapping of complex trait loci. <span><span class="ref-journal">Genetics. </span>2005;<span class="ref-vol">169</span>:1071–1092.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="http://www.pubmedcentral.nih.gov/redirect3.cgi?&amp;&amp;auth=0ZoNBgdLQXIqvb3yMQDDToxenjyyy5xOfdyIivAIJ&amp;reftype=pubmed&amp;artid=2587010&amp;article-id=2587010&amp;iid=173756&amp;issue-id=173756&amp;jid=319&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15489534">PubMed</a>]</span></div></li></ul></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr>
    <div class="footer-link" id="fp-footer">
      <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.pubmedcentral.nih.gov/">PMC Home</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov/">NCBI</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov/">NIH</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="http://www.pubmedcentral.nih.gov/about/privacypolicy.html">Privacy Policy</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
          &nbsp;|&nbsp;
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></tbody></table><!--host:ipmc2-->

</body></html>